{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/AMIRMOHAMMAD-OSS/Deep-generative-transformer-based-model-for-de-novo-design-of-proteins/blob/main/Another_copy_of_llps.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#Liquid-Liquid Phase Separation (LLPS) Protein Scorer!\n",
        "\n",
        "This tool allows you to analyze protein sequences for their propensity to undergo phase separation:\n",
        "\n",
        "Input Options:\n",
        "Upload a single protein sequence directly into the designated box.\n",
        "Alternatively, load a FASTA or TXT file containing multiple protein sequences.\n",
        "\n",
        "To maintain efficiency, we advise against loading more than 100 sequences at once\n",
        "\n",
        "Specify the sequence ID or directory for your analysis.\n",
        "\n",
        "By setting the 'End_Sequence'  parameter,you can focus the scoring on sequences up to the specified endpoint.\n",
        "\n",
        "         \n",
        "\n",
        "\n",
        "\n",
        "#BLAST+ tool\n",
        "You can utilize the BLAST+ tool to run ncbi BLAST searches on either your protein sequence or your FASTA file against your dataset. This powerful tool allows you to compare sequences and identify homologous regions, aiding in various biological analyses.\n"
      ],
      "metadata": {
        "id": "XJ8r7M6CJzke"
      }
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "cellView": "form",
        "id": "hyzTHwf8dRBy",
        "collapsed": true,
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "353695cb-08e5-4494-ef3c-a1844e789536"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Everything looks ok! ‚ú®üç∞‚ú®\n"
          ]
        }
      ],
      "source": [
        "#@title Importing libraries\n",
        "from google.colab import output\n",
        "!git clone https://github.com/AMIRMOHAMMAD-OSS/classi\n",
        "output.clear()\n",
        "!pip install biopython\n",
        "output.clear()\n",
        "!apt-get update\n",
        "!apt-get install -y ncbi-blast+\n",
        "!pip install streamlit\n",
        "output.clear()\n",
        "\n",
        "!pip install py3Dmol\n",
        "output.clear()\n",
        "!pip install requests\n",
        "!pip install biotite\n",
        "!pip install stmol\n",
        "output.clear()\n",
        "!pip install command_runner\n",
        "output.clear()\n",
        "from command_runner import command_runner\n",
        "!pip install plotly\n",
        "!pip install --upgrade \"kaleido==0.1.*\"\n",
        "output.clear()\n",
        "import kaleido\n",
        "from Bio import SeqIO\n",
        "from Bio.Seq import Seq\n",
        "from Bio import pairwise2\n",
        "from google.colab import files\n",
        "import plotly.graph_objects as go\n",
        "from plotly.subplots import make_subplots\n",
        "import zipfile\n",
        "import shutil\n",
        "from transformers import AutoTokenizer\n",
        "from tqdm import tqdm\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "import numpy as np\n",
        "import time\n",
        "from collections import defaultdict\n",
        "import torch\n",
        "from torch.utils.data.dataloader import DataLoader\n",
        "import time\n",
        "import os\n",
        "import sys\n",
        "import json\n",
        "import random\n",
        "from ast import literal_eval\n",
        "import numpy as np\n",
        "import torch\n",
        "from transformers import AutoTokenizer\n",
        "from transformers import AutoTokenizer\n",
        "from tokenizers import Tokenizer, decoders, models, normalizers, pre_tokenizers, trainers\n",
        "from transformers import CanineTokenizer, CanineModel\n",
        "tokenizer = Tokenizer.from_file(\"classi/Trained_BPE2.json\")\n",
        "tokenizer.model_max_length = 256\n",
        "tokenizer\n",
        "\n",
        "from sklearn.preprocessing import StandardScaler as Sc\n",
        "from sklearn.decomposition import PCA\n",
        "from sklearn.model_selection import train_test_split as TTS\n",
        "import numpy as np\n",
        "\n",
        "import torch\n",
        "\n",
        "from sklearn.model_selection import train_test_split as TTS\n",
        "\n",
        "\n",
        "def edit(x):\n",
        "    for i in x:\n",
        "      if i in \"BJOUX\\Z_n\\n\":\n",
        "        x = x.replace(i,\"\")\n",
        "      else:\n",
        "        pass\n",
        "    return x\n",
        "\n",
        "def encode(x):\n",
        "  l = []\n",
        "  chars = tokenizer.get_vocab()\n",
        "  for i in x:\n",
        "    l.append(chars[i])\n",
        "  return l\n",
        "\n",
        "with open(\"classi/training.txt\",\"r\",encoding=\"utf-8\") as f:\n",
        "  text = f.read()\n",
        "  k = \"#\".join([i for i in edit(text).split(\"<|edoftext|>\")])\n",
        "chars = tokenizer.get_vocab()\n",
        "vocab_size = len(tokenizer.get_vocab())\n",
        "\n",
        "with open(\"classi/validation.txt\",\"r\",encoding=\"utf-8\") as f:\n",
        "  text = f.read()\n",
        "  l = \"#\".join([i for i in edit(text).split(\"<|edoftext|>\")])\n",
        "\n",
        "def Padding(x,PAD = 0,max_len = 512):\n",
        "  l = []\n",
        "  max_len = max(512,len(max(x,key = lambda x: len(x))))\n",
        "  for i in x:\n",
        "    a = i\n",
        "    if len(a) < max_len:\n",
        "      a = a + [PAD for i in range(max_len-len(a))]\n",
        "    l.append(a)\n",
        "  return np.array(l).reshape(len(x),max_len)\n",
        "with open(\"classi/negative_dataset.txt\") as x:\n",
        "  c = x.read()\n",
        "Filtered_train  = [i for i in k.split(\"#\") if len(i)<= 512][1:]\n",
        "Filtered_val = [i for i in l.split(\"#\") if len(i)<= 512][1:]\n",
        "Filtered_neg = [i for i in c.split(\"\\n\") if len(i)<= 512]\n",
        "pos = Filtered_train + Filtered_val\n",
        "pos2 = list(set([i for i in k.split(\"#\")][1:] +  [i for i in l.split(\"#\")][1:]))\n",
        "with open(\"DrLLPS.fasta\",\"w\") as w:\n",
        "  for i in range(len(pos2)):\n",
        "    w.write(\">seq{i}\\n\".format(i=i))\n",
        "    w.write(pos2[i]+\"\\n\")\n",
        "  w.close()\n",
        "files.download(\"DrLLPS.fasta\")\n",
        "output.clear()\n",
        "print(\"Everything looks ok! ‚ú®üç∞‚ú®\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "cellView": "form",
        "id": "Yc9VXFLYN-x6"
      },
      "outputs": [],
      "source": [
        "#@title Loading models and requierments\n",
        "\n",
        "def set_seed(seed):\n",
        "    random.seed(seed)\n",
        "    np.random.seed(seed)\n",
        "    torch.manual_seed(seed)\n",
        "    torch.cuda.manual_seed_all(seed)\n",
        "\n",
        "def setup_logging(config):\n",
        "    work_dir = config.system.work_dir\n",
        "    os.makedirs(work_dir, exist_ok=True)\n",
        "    with open(os.path.join(work_dir, 'args.txt'), 'w') as f:\n",
        "        f.write(' '.join(sys.argv))\n",
        "    with open(os.path.join(work_dir, 'config.json'), 'w') as f:\n",
        "        f.write(json.dumps(config.to_dict(), indent=4))\n",
        "\n",
        "class CfgNode:\n",
        "    def __init__(self, **kwargs):\n",
        "        self.__dict__.update(kwargs)\n",
        "\n",
        "    def __str__(self):\n",
        "        return self._str_helper(0)\n",
        "\n",
        "    def _str_helper(self, indent):\n",
        "        parts = []\n",
        "        for k, v in self.__dict__.items():\n",
        "            if isinstance(v, CfgNode):\n",
        "                parts.append(\"%s:\\n\" % k)\n",
        "                parts.append(v._str_helper(indent + 1))\n",
        "            else:\n",
        "                parts.append(\"%s: %s\\n\" % (k, v))\n",
        "        parts = [' ' * (indent * 4) + p for p in parts]\n",
        "        return \"\".join(parts)\n",
        "\n",
        "    def to_dict(self):\n",
        "        return { k: v.to_dict() if isinstance(v, CfgNode) else v for k, v in self.__dict__.items() }\n",
        "\n",
        "    def merge_from_dict(self, d):\n",
        "        self.__dict__.update(d)\n",
        "\n",
        "    def merge_from_args(self, args):\n",
        "\n",
        "        for arg in args:\n",
        "            keyval = arg.split('=')\n",
        "            assert len(keyval) == 2, \"expecting each override arg to be of form --arg=value, got %s\" % arg\n",
        "            key, val = keyval\n",
        "            try:\n",
        "                val = literal_eval(val)\n",
        "            except ValueError:\n",
        "                pass\n",
        "\n",
        "            assert key[:2] == '--'\n",
        "            key = key[2:]\n",
        "            keys = key.split('.')\n",
        "            obj = self\n",
        "            for k in keys[:-1]:\n",
        "                obj = getattr(obj, k)\n",
        "            leaf_key = keys[-1]\n",
        "            assert hasattr(obj, leaf_key), f\"{key} is not an attribute that exists in the config\"\n",
        "            print(\"command line overwriting config attribute %s with %s\" % (key, val))\n",
        "            setattr(obj, leaf_key, val)\n",
        "\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "\n",
        "class SiLU(nn.Module):\n",
        "   def forward(self, x):\n",
        "        return x*F.sigmoid(x)\n",
        "\n",
        "class NY(nn.Module):\n",
        "  def forward(self,x):\n",
        "    return 3*torch.tanh(0.3*x)\n",
        "\n",
        "class NewGELU(nn.Module):\n",
        "    def forward(self, x):\n",
        "        return 0.5 * x * (1.0 + torch.tanh(math.sqrt(2.0 / math.pi) * (x + 0.044715 * torch.pow(x, 3.0))))\n",
        "\n",
        "class SelfAttention(nn.Module):\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        assert config.n_embd % config.n_head == 0\n",
        "        self.c_attn = nn.Linear(config.n_embd, 3 * config.n_embd)\n",
        "        self.c_proj = nn.Linear(config.n_embd, config.n_embd)\n",
        "        self.attn_dropout = nn.Dropout(config.attn_pdrop)\n",
        "        self.resid_dropout = nn.Dropout(config.resid_pdrop)\n",
        "        self.register_buffer(\"bias\", torch.tril(torch.ones(config.block_size, config.block_size))\n",
        "                                     .view(1, 1, config.block_size, config.block_size))\n",
        "        self.n_head = config.n_head\n",
        "        self.n_embd = config.n_embd\n",
        "\n",
        "    def forward(self, x):\n",
        "        B, T, C = x.size()\n",
        "        q, k ,v  = self.c_attn(x).split(self.n_embd, dim=2)\n",
        "        k = k.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        q = q.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        v = v.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        att = (q @ k.transpose(-2, -1)) * (1.0 / math.sqrt(k.size(-1)))\n",
        "        att = att.masked_fill(self.bias[:,:,:T,:T] == 0, float('-inf'))\n",
        "        att = F.softmax(att, dim=-1)\n",
        "        att = self.attn_dropout(att)\n",
        "        y = att @ v\n",
        "        y = y.transpose(1, 2).contiguous().view(B, T, C)\n",
        "        y = self.resid_dropout(self.c_proj(y))\n",
        "        return y\n",
        "\n",
        "class Block(nn.Module):\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.ln_1 = nn.LayerNorm(config.n_embd)\n",
        "        self.attn = SelfAttention(config)\n",
        "        self.ln_2 = nn.LayerNorm(config.n_embd)\n",
        "        self.mlp = nn.ModuleDict(dict(\n",
        "            c_fc    = nn.Linear(config.n_embd, 4 * config.n_embd),\n",
        "            c_proj  = nn.Linear(4 * config.n_embd, config.n_embd),\n",
        "            act     = NewGELU(),\n",
        "            dropout = nn.Dropout(config.resid_pdrop),\n",
        "        ))\n",
        "        m = self.mlp\n",
        "        self.mlpf = lambda x: m.dropout(m.c_proj(m.act(m.c_fc(x))))\n",
        "    def forward(self, x):\n",
        "        x = x + self.attn(self.ln_1(x))\n",
        "        x = x + self.mlpf(self.ln_2(x))\n",
        "        return x\n",
        "\n",
        "class ClassifierII(nn.Module):\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.model_type = 'gpt'\n",
        "        C.n_layer = None\n",
        "        C.n_head = None\n",
        "        C.n_embd =  None\n",
        "        C.vocab_size = len(chars)\n",
        "        C.max_length = 512\n",
        "        C.embd_pdrop = 0.1\n",
        "        C.resid_pdrop = 0.1\n",
        "        C.attn_pdrop = 0.1\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.max_length = config.block_size\n",
        "        self.soft = nn.Softmax(1)\n",
        "        self.config = self.get_default_config()\n",
        "        self.device = \"cuda\"\n",
        "        self.model_states = {'h':{'n_layer': 48, 'n_head': 25, 'n_embd': 1600},\n",
        "                                    'g':{'n_layer': 12, '': 12, 'n_embd': 768},\n",
        "                'f':   {'n_layer': 24, 'n_head': 16, 'n_embd': 1024},\n",
        "                'e':   {'n_layer': 36, 'n_head': 20, 'n_embd': 1280},\n",
        "                'd':{'n_layer': 8, 'n_head': 16, 'n_embd': 512},\n",
        "                'c':{'n_layer': 6, 'n_head': 6, 'n_embd': 192},\n",
        "                'b':{'n_layer': 4, 'n_head': 4, 'n_embd': 128},\n",
        "                'a':{'n_layer': 3, 'n_head': 3, 'n_embd': 48}}\n",
        "\n",
        "        type_ = config.model_type is not None\n",
        "        #assert type_ in \"abcdefgh\"\n",
        "        p = all([config.n_layer is not None, config.n_head is not None, config.n_embd is not None])\n",
        "        #assert type_ == True and p == True\n",
        "        if type_:\n",
        "            config.merge_from_dict(self.model_states[config.model_type])\n",
        "        self.closs = nn.BCELoss()\n",
        "        self.ny = NY()\n",
        "        self.l = nn.Linear(512,1,64)\n",
        "        self.transformer = nn.ModuleDict(dict(\n",
        "            wte = nn.Embedding(config.vocab_size, config.n_embd),\n",
        "            wpe = nn.Embedding(config.max_length, config.n_embd),\n",
        "            drop = nn.Dropout(config.embd_pdrop),\n",
        "            h = nn.ModuleList([Block(config) for _ in range(config.n_layer)]),\n",
        "            ln_f = nn.LayerNorm(config.n_embd),))\n",
        "        self.classifier_head = nn.Sequential(#nn.Tanh(),\n",
        "                                             nn.Linear(config.n_embd, 2)\n",
        "                                             )\n",
        "        self.apply(self._init_weights)\n",
        "        for pn, p in self.named_parameters():\n",
        "            if pn.endswith('c_proj.weight'):\n",
        "                torch.nn.init.normal_(p, mean=0.0, std=0.02/math.sqrt(2 * config.n_layer))\n",
        "\n",
        "        n_params = sum(p.numel() for p in self.transformer.parameters())\n",
        "        print(\"[ Number of trainable parameters: %.2fM ]\" % (n_params/1e6,))\n",
        "\n",
        "    def _init_weights(self, module):\n",
        "        if isinstance(module, nn.Linear):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "            if module.bias is not None:\n",
        "                torch.nn.init.zeros_(module.bias)\n",
        "        elif isinstance(module, nn.Embedding):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "        elif isinstance(module, nn.LayerNorm):\n",
        "            torch.nn.init.zeros_(module.bias)\n",
        "            torch.nn.init.ones_(module.weight)\n",
        "\n",
        "    @classmethod\n",
        "    def from_pretrained(cls, model_type):\n",
        "        from transformers import GPT2LMHeadModel\n",
        "        config = cls.get_default_config()\n",
        "        config.model_type = model_type\n",
        "        config.vocab_size = 25\n",
        "        config.max_length = 512\n",
        "        model = ClassifierII(config)\n",
        "        sd = model.state_dict()\n",
        "        model_hf = GPT2LMHeadModel.from_pretrained(model_type)\n",
        "        sd_hf = model_hf.state_dict()\n",
        "        keys = [k for k in sd_hf if not k.endswith('attn.masked_bias')]\n",
        "        transposed = ['attn.c_attn.weight', 'attn.c_proj.weight', 'mlp.c_fc.weight', 'mlp.c_proj.weight']\n",
        "        assert len(keys) == len(sd)\n",
        "        for k in keys:\n",
        "            if any(k.endswith(w) for w in transposed):\n",
        "                assert sd_hf[k].shape[::-1] == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k].t())\n",
        "            else:\n",
        "                assert sd_hf[k].shape == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k])\n",
        "        return model\n",
        "\n",
        "    def configure_optimizers(self, train_config):\n",
        "        decay = set()\n",
        "        no_decay = set()\n",
        "        whitelist_weight_modules = (torch.nn.Linear, )\n",
        "        blacklist_weight_modules = (torch.nn.LayerNorm, torch.nn.Embedding)\n",
        "        for mn, m in self.named_modules():\n",
        "            for pn, p in m.named_parameters():\n",
        "                fpn = '%s.%s' % (mn, pn) if mn else pn\n",
        "                if pn.endswith('bias'):\n",
        "                    no_decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, whitelist_weight_modules):\n",
        "                    decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, blacklist_weight_modules):\n",
        "                    no_decay.add(fpn)\n",
        "        param_dict = {pn: p for pn, p in self.named_parameters()}\n",
        "        inter_params = decay & no_decay\n",
        "        union_params = decay | no_decay\n",
        "        assert len(inter_params) == 0, \"parameters %s made it into both decay/no_decay sets!\" % (str(inter_params), )\n",
        "        assert len(param_dict.keys() - union_params) == 0, \"parameters %s were not separated into either decay/no_decay set!\" \\\n",
        "                                                    % (str(param_dict.keys() - union_params), )\n",
        "        optim_groups = [\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(decay))], \"weight_decay\": train_config.weight_decay},\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(no_decay))], \"weight_decay\": 0.0},]\n",
        "        optimizer = torch.optim.AdamW(optim_groups, lr=train_config.learning_rate, betas=train_config.betas)\n",
        "        return optimizer\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict_proba(self,idx,j =\"None\"):\n",
        "      sigmoid = nn.Sigmoid()\n",
        "      soft = nn.Softmax(dim=1)\n",
        "      ny = NY()\n",
        "      si = SiLU()\n",
        "      self.eval()\n",
        "      x,_ = self.forward(idx)\n",
        "      if j == \"None\":\n",
        "        return x[:,0:1]\n",
        "      elif j == \"soft\":\n",
        "        return self.soft(x)[:,0:1]\n",
        "      else:\n",
        "        return sigmoid(x)[:,0:1]\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict(self,idx):\n",
        "      g = torch.zeros((idx.shape[0],1))\n",
        "      e = -1\n",
        "      for i in self.predict_proba(idx):\n",
        "        e+=1\n",
        "        if i[0].item()>=0.5:\n",
        "          g[e] = 1\n",
        "      return g\n",
        "\n",
        "    def forward(self, idx, targets=None):\n",
        "        device = idx.device\n",
        "        b, t = idx.size()\n",
        "        pos = torch.arange(0, t, dtype=torch.long, device=device).unsqueeze(0)\n",
        "        tok_emb = self.transformer.wte(idx)\n",
        "        pos_emb = self.transformer.wpe(pos)\n",
        "        x = self.transformer.drop(tok_emb + pos_emb)\n",
        "        for block in self.transformer.h:\n",
        "            x = block(x)\n",
        "        x = self.transformer.ln_f(x)\n",
        "        logits = self.classifier_head(x).view(b,2,t).to(device)\n",
        "        logits = self.l(logits)\n",
        "        logits = F.sigmoid(self.ny(logits).view(b,2).mean(1).view(b,1))\n",
        "        loss = None\n",
        "        if targets is not None:\n",
        "            loss = F.binary_cross_entropy(logits,targets)\n",
        "        return logits, loss\n",
        "\n",
        "\n",
        "class Trainer:\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.device = 'cuda'\n",
        "        C.num_workers = 4\n",
        "        C.max_iters = None\n",
        "        C.batch_size = 128\n",
        "        C.max_length = 512\n",
        "        C.learning_rate = 8e-4\n",
        "        C.betas = (0.9, 0.95)\n",
        "        C.weight_decay = 0.1\n",
        "        C.grad_norm_clip = 1.0\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config, model, train_dataset,val_dataset):\n",
        "        self.config = config\n",
        "        self.model = model\n",
        "        self.optimizer = None\n",
        "        self.train_dataset = train_dataset\n",
        "        self.val_dataset = val_dataset\n",
        "        self.callbacks = defaultdict(list)\n",
        "        self.device ='cuda'\n",
        "        self.model = self.model.to(self.device)\n",
        "        self.iter_num = 0\n",
        "        self.iter_time = 0.0\n",
        "        self.iter_dt = 0.0\n",
        "\n",
        "    def add_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent].append(callback)\n",
        "\n",
        "    def set_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent] = [callback]\n",
        "\n",
        "    def trigger_callbacks(self, onevent: str):\n",
        "        for callback in self.callbacks.get(onevent, []):\n",
        "            callback(self)\n",
        "\n",
        "    def run(self):\n",
        "        model, config = self.model, self.config\n",
        "        self.optimizer = model.configure_optimizers(config)\n",
        "        batch_size = self.config.batch_size\n",
        "        def get_batch(mode):\n",
        "            batch_size = 64\n",
        "            if mode == \"train\":\n",
        "              data = train_dataset\n",
        "              pos_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]),513))\n",
        "              neg_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]),513))\n",
        "              N = np.random.randint(batch_size)\n",
        "              ix_pos = np.random.randint(pos_train.shape[0]-N)\n",
        "              ix_neg = np.random.randint(neg_train.shape[0]-batch_size-N)\n",
        "              pos_data = pos_train[ix_pos:ix_pos+N,:]\n",
        "              neg_data = neg_train[ix_neg:ix_neg+batch_size-N,:]\n",
        "              data = torch.concat((pos_data,neg_data))\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,1), device=\"cuda\")\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o] = 1\n",
        "                #else:\n",
        "                  #targets[o][1] = 1\n",
        "            else:\n",
        "              data = val_dataset\n",
        "              ix = np.random.randint(data.shape[0]-batch_size)\n",
        "              data = data[ix:ix+batch_size,:]\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,1), device=\"cuda\")\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o] = 1\n",
        "                #else:\n",
        "                 # targets[o][1] = 1\n",
        "            targets = targets.view(batch_size,1).to(\"cuda\")\n",
        "            seq = seq.view(batch_size,512).to(\"cuda\")\n",
        "            return seq, targets\n",
        "\n",
        "        @torch.no_grad\n",
        "        def cross_val():\n",
        "          model.eval()\n",
        "          out = []\n",
        "          for i in [\"train\",\"val\"]:\n",
        "            losses = torch.zeros(200+1)\n",
        "            for k in range(200):\n",
        "              X,Y = get_batch(i)\n",
        "              logits,loss = model(X,Y)\n",
        "              losses[k]=loss.item()\n",
        "              out1 = losses.mean()\n",
        "            out.append(out1)\n",
        "          model.train()\n",
        "          return out\n",
        "        losses = cross_val()\n",
        "        LOSS = [losses]\n",
        "        print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        model.train()\n",
        "        for epoch in range(20):\n",
        "          print(\"[epoch {o}] \\n\".format(o = epoch))\n",
        "          iters = 200\n",
        "          for i in range(iters):\n",
        "            if i == 0:\n",
        "              Y = \"| =\"\n",
        "            elif i == iters -1 :\n",
        "              Y = \"=> 100% |\"\n",
        "            else:\n",
        "              if i%(int(iters/50)) == 0 :\n",
        "                Y = \"=\"\n",
        "              else:\n",
        "                Y = \"\"\n",
        "            print(\"{y}\".format(y = Y),end=\"\")\n",
        "            x, y = get_batch(\"train\")\n",
        "            logits, self.loss = model(x, y)\n",
        "            model.zero_grad(set_to_none=True)\n",
        "            self.loss.backward()\n",
        "            torch.nn.utils.clip_grad_norm_(model.parameters(), config.grad_norm_clip)\n",
        "            self.optimizer.step()\n",
        "          losses = cross_val()\n",
        "          LOSS.append(losses)\n",
        "          print(LOSS)\n",
        "          print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        PATH = \"model {h}\".format(h = model_config.model_type)\n",
        "        torch.save(model.state_dict(), PATH)\n",
        "\n",
        "def set_seed(seed):\n",
        "    random.seed(seed)\n",
        "    np.random.seed(seed)\n",
        "    torch.manual_seed(seed)\n",
        "    torch.cuda.manual_seed_all(seed)\n",
        "\n",
        "def setup_logging(config):\n",
        "    work_dir = config.system.work_dir\n",
        "    os.makedirs(work_dir, exist_ok=True)\n",
        "    with open(os.path.join(work_dir, 'args.txt'), 'w') as f:\n",
        "        f.write(' '.join(sys.argv))\n",
        "    with open(os.path.join(work_dir, 'config.json'), 'w') as f:\n",
        "        f.write(json.dumps(config.to_dict(), indent=4))\n",
        "\n",
        "class CfgNode:\n",
        "    def __init__(self, **kwargs):\n",
        "        self.__dict__.update(kwargs)\n",
        "\n",
        "    def __str__(self):\n",
        "        return self._str_helper(0)\n",
        "\n",
        "    def _str_helper(self, indent):\n",
        "        parts = []\n",
        "        for k, v in self.__dict__.items():\n",
        "            if isinstance(v, CfgNode):\n",
        "                parts.append(\"%s:\\n\" % k)\n",
        "                parts.append(v._str_helper(indent + 1))\n",
        "            else:\n",
        "                parts.append(\"%s: %s\\n\" % (k, v))\n",
        "        parts = [' ' * (indent * 4) + p for p in parts]\n",
        "        return \"\".join(parts)\n",
        "\n",
        "    def to_dict(self):\n",
        "        return { k: v.to_dict() if isinstance(v, CfgNode) else v for k, v in self.__dict__.items() }\n",
        "\n",
        "    def merge_from_dict(self, d):\n",
        "        self.__dict__.update(d)\n",
        "\n",
        "    def merge_from_args(self, args):\n",
        "\n",
        "        for arg in args:\n",
        "            keyval = arg.split('=')\n",
        "            assert len(keyval) == 2, \"expecting each override arg to be of form --arg=value, got %s\" % arg\n",
        "            key, val = keyval\n",
        "            try:\n",
        "                val = literal_eval(val)\n",
        "            except ValueError:\n",
        "                pass\n",
        "\n",
        "            assert key[:2] == '--'\n",
        "            key = key[2:]\n",
        "            keys = key.split('.')\n",
        "            obj = self\n",
        "            for k in keys[:-1]:\n",
        "                obj = getattr(obj, k)\n",
        "            leaf_key = keys[-1]\n",
        "            assert hasattr(obj, leaf_key), f\"{key} is not an attribute that exists in the config\"\n",
        "            print(\"command line overwriting config attribute %s with %s\" % (key, val))\n",
        "            setattr(obj, leaf_key, val)\n",
        "\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "\n",
        "class SiLU(nn.Module):\n",
        "   def forward(self, x):\n",
        "        return x*F.sigmoid(x)\n",
        "\n",
        "class NY(nn.Module):\n",
        "  def forward(self,x):\n",
        "    return 3*torch.tanh(0.3*x)\n",
        "\n",
        "class NewGELU(nn.Module):\n",
        "    def forward(self, x):\n",
        "        return 0.5 * x * (1.0 + torch.tanh(math.sqrt(2.0 / math.pi) * (x + 0.044715 * torch.pow(x, 3.0))))\n",
        "\n",
        "class SelfAttention(nn.Module):\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        assert config.n_embd % config.n_head == 0\n",
        "        self.c_attn = nn.Linear(config.n_embd, 3 * config.n_embd)\n",
        "        self.c_proj = nn.Linear(config.n_embd, config.n_embd)\n",
        "        self.attn_dropout = nn.Dropout(config.attn_pdrop)\n",
        "        self.resid_dropout = nn.Dropout(config.resid_pdrop)\n",
        "        self.register_buffer(\"bias\", torch.tril(torch.ones(config.block_size, config.block_size))\n",
        "                                     .view(1, 1, config.block_size, config.block_size))\n",
        "        self.n_head = config.n_head\n",
        "        self.n_embd = config.n_embd\n",
        "\n",
        "    def forward(self, x):\n",
        "        B, T, C = x.size()\n",
        "        q, k ,v  = self.c_attn(x).split(self.n_embd, dim=2)\n",
        "        k = k.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        q = q.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        v = v.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        att = (q @ k.transpose(-2, -1)) * (1.0 / math.sqrt(k.size(-1)))\n",
        "        att = att.masked_fill(self.bias[:,:,:T,:T] == 0, float('-inf'))\n",
        "        att = F.softmax(att, dim=-1)\n",
        "        att = self.attn_dropout(att)\n",
        "        y = att @ v\n",
        "        y = y.transpose(1, 2).contiguous().view(B, T, C)\n",
        "        y = self.resid_dropout(self.c_proj(y))\n",
        "        return y\n",
        "\n",
        "class Block(nn.Module):\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.ln_1 = nn.LayerNorm(config.n_embd)\n",
        "        self.attn = SelfAttention(config)\n",
        "        self.ln_2 = nn.LayerNorm(config.n_embd)\n",
        "        self.mlp = nn.ModuleDict(dict(\n",
        "            c_fc    = nn.Linear(config.n_embd, 4 * config.n_embd),\n",
        "            c_proj  = nn.Linear(4 * config.n_embd, config.n_embd),\n",
        "            act     = NewGELU(),\n",
        "            dropout = nn.Dropout(config.resid_pdrop),\n",
        "        ))\n",
        "        m = self.mlp\n",
        "        self.mlpf = lambda x: m.dropout(m.c_proj(m.act(m.c_fc(x))))\n",
        "    def forward(self, x):\n",
        "        x = x + self.attn(self.ln_1(x))\n",
        "        x = x + self.mlpf(self.ln_2(x))\n",
        "        return x\n",
        "\n",
        "class ClassifierI(nn.Module):\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.model_type = 'gpt'\n",
        "        C.n_layer = None\n",
        "        C.n_head = None\n",
        "        C.n_embd =  None\n",
        "        C.vocab_size = len(chars)\n",
        "        C.max_length = 512\n",
        "        C.embd_pdrop = 0.1\n",
        "        C.resid_pdrop = 0.1\n",
        "        C.attn_pdrop = 0.1\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.max_length = config.block_size\n",
        "        self.soft = nn.Softmax(1)\n",
        "        self.config = self.get_default_config()\n",
        "        self.device = \"cpu\"\n",
        "        self.model_states = {'h':{'n_layer': 48, 'n_head': 25, 'n_embd': 1600},\n",
        "                                    'g':{'n_layer': 12, '': 12, 'n_embd': 768},\n",
        "                'f':   {'n_layer': 24, 'n_head': 16, 'n_embd': 1024},\n",
        "                'e':   {'n_layer': 36, 'n_head': 20, 'n_embd': 1280},\n",
        "                'd':{'n_layer': 8, 'n_head': 16, 'n_embd': 512},\n",
        "                'c':{'n_layer': 6, 'n_head': 6, 'n_embd': 192},\n",
        "                'b':{'n_layer': 4, 'n_head': 4, 'n_embd': 128},\n",
        "                'a':{'n_layer': 3, 'n_head': 3, 'n_embd': 48}}\n",
        "\n",
        "        type_ = config.model_type is not None\n",
        "        #assert type_ in \"abcdefgh\"\n",
        "        p = all([config.n_layer is not None, config.n_head is not None, config.n_embd is not None])\n",
        "        #assert type_ == True and p == True\n",
        "        if type_:\n",
        "            config.merge_from_dict(self.model_states[config.model_type])\n",
        "        self.closs = nn.BCELoss()\n",
        "        self.ny = NY()\n",
        "        #self.l = nn.Linear(512,1,64)\n",
        "        self.transformer = nn.ModuleDict(dict(\n",
        "            wte = nn.Embedding(config.vocab_size, config.n_embd),\n",
        "            wpe = nn.Embedding(config.max_length, config.n_embd),\n",
        "            drop = nn.Dropout(config.embd_pdrop),\n",
        "            h = nn.ModuleList([Block(config) for _ in range(config.n_layer)]),\n",
        "            ln_f = nn.LayerNorm(config.n_embd),))\n",
        "        self.classifier_head = nn.Sequential(#nn.Tanh(),\n",
        "                                             nn.Linear(config.n_embd, 2)\n",
        "                                             )\n",
        "        self.apply(self._init_weights)\n",
        "        for pn, p in self.named_parameters():\n",
        "            if pn.endswith('c_proj.weight'):\n",
        "                torch.nn.init.normal_(p, mean=0.0, std=0.02/math.sqrt(2 * config.n_layer))\n",
        "\n",
        "        n_params = sum(p.numel() for p in self.transformer.parameters())\n",
        "        #print(\"[ Number of trainable parameters: %.2fM ]\" % (n_params/1e6,))\n",
        "\n",
        "    def _init_weights(self, module):\n",
        "        if isinstance(module, nn.Linear):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "            if module.bias is not None:\n",
        "                torch.nn.init.zeros_(module.bias)\n",
        "        elif isinstance(module, nn.Embedding):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "        elif isinstance(module, nn.LayerNorm):\n",
        "            torch.nn.init.zeros_(module.bias)\n",
        "            torch.nn.init.ones_(module.weight)\n",
        "\n",
        "    @classmethod\n",
        "    def from_pretrained(cls, model_type):\n",
        "        from transformers import GPT2LMHeadModel\n",
        "        config = cls.get_default_config()\n",
        "        config.model_type = model_type\n",
        "        config.vocab_size = 25\n",
        "        config.max_length = 512\n",
        "        model = ClassifierI(config)\n",
        "        sd = model.state_dict()\n",
        "        model_hf = GPT2LMHeadModel.from_pretrained(model_type)\n",
        "        sd_hf = model_hf.state_dict()\n",
        "        keys = [k for k in sd_hf if not k.endswith('attn.masked_bias')]\n",
        "        transposed = ['attn.c_attn.weight', 'attn.c_proj.weight', 'mlp.c_fc.weight', 'mlp.c_proj.weight']\n",
        "        assert len(keys) == len(sd)\n",
        "        for k in keys:\n",
        "            if any(k.endswith(w) for w in transposed):\n",
        "                assert sd_hf[k].shape[::-1] == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k].t())\n",
        "            else:\n",
        "                assert sd_hf[k].shape == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k])\n",
        "        return model\n",
        "\n",
        "    def configure_optimizers(self, train_config):\n",
        "        decay = set()\n",
        "        no_decay = set()\n",
        "        whitelist_weight_modules = (torch.nn.Linear, )\n",
        "        blacklist_weight_modules = (torch.nn.LayerNorm, torch.nn.Embedding)\n",
        "        for mn, m in self.named_modules():\n",
        "            for pn, p in m.named_parameters():\n",
        "                fpn = '%s.%s' % (mn, pn) if mn else pn\n",
        "                if pn.endswith('bias'):\n",
        "                    no_decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, whitelist_weight_modules):\n",
        "                    decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, blacklist_weight_modules):\n",
        "                    no_decay.add(fpn)\n",
        "        param_dict = {pn: p for pn, p in self.named_parameters()}\n",
        "        inter_params = decay & no_decay\n",
        "        union_params = decay | no_decay\n",
        "        assert len(inter_params) == 0, \"parameters %s made it into both decay/no_decay sets!\" % (str(inter_params), )\n",
        "        assert len(param_dict.keys() - union_params) == 0, \"parameters %s were not separated into either decay/no_decay set!\" \\\n",
        "                                                    % (str(param_dict.keys() - union_params), )\n",
        "        optim_groups = [\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(decay))], \"weight_decay\": train_config.weight_decay},\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(no_decay))], \"weight_decay\": 0.0},]\n",
        "        optimizer = torch.optim.AdamW(optim_groups, lr=train_config.learning_rate, betas=train_config.betas)\n",
        "        return optimizer\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict_proba(self,idx,j =\"None\"):\n",
        "      sigmoid = nn.Sigmoid()\n",
        "      soft = nn.Softmax(dim=1)\n",
        "      ny = NY()\n",
        "      si = SiLU()\n",
        "      self.eval()\n",
        "      x,_ = self.forward(idx)\n",
        "      if j == \"None\":\n",
        "        return x[:,0:1]\n",
        "      elif j == \"soft\":\n",
        "        return self.soft(x)[:,0:1]\n",
        "      else:\n",
        "        return sigmoid(x)[:,0:1]\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict(self,idx):\n",
        "      g = torch.zeros((idx.shape[0],1))\n",
        "      e = -1\n",
        "      for i in self.predict_proba(idx):\n",
        "        e+=1\n",
        "        if i[0].item()>=0.5:\n",
        "          g[e] = 1\n",
        "      return g\n",
        "\n",
        "    def forward(self, idx, targets=None):\n",
        "        device = idx.device\n",
        "        b, t = idx.size()\n",
        "        pos = torch.arange(0, t, dtype=torch.long, device=device).unsqueeze(0)\n",
        "        tok_emb = self.transformer.wte(idx)\n",
        "        pos_emb = self.transformer.wpe(pos)\n",
        "        x = self.transformer.drop(tok_emb + pos_emb)\n",
        "        for block in self.transformer.h:\n",
        "            x = block(x)\n",
        "        x = self.transformer.ln_f(x)\n",
        "        logits = self.ny(self.classifier_head(x)).mean(1).to(device)\n",
        "        #logits = self.l(logits)\n",
        "        # = F.sigmoid(logits.view(b,2).mean(1).view(b,1))\n",
        "        loss = None\n",
        "        if targets is not None:\n",
        "            loss = F.cross_entropy(logits,targets,ignore_index = -1)\n",
        "        return logits, loss\n",
        "\n",
        "\n",
        "class Trainer:\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.device = 'cuda'\n",
        "        C.num_workers = 4\n",
        "        C.max_iters = None\n",
        "        C.batch_size = 64\n",
        "        C.max_length = 512\n",
        "        C.learning_rate = 8e-4\n",
        "        C.betas = (0.9, 0.95)\n",
        "        C.weight_decay = 0.1\n",
        "        C.grad_norm_clip = 1.0\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config, model, train_dataset,val_dataset):\n",
        "        self.config = config\n",
        "        self.model = model\n",
        "        self.optimizer = None\n",
        "        self.train_dataset = train_dataset\n",
        "        self.val_dataset = val_dataset\n",
        "        self.callbacks = defaultdict(list)\n",
        "        self.device ='cuda'\n",
        "        self.model = self.model.to(self.device)\n",
        "        self.iter_num = 0\n",
        "        self.iter_time = 0.0\n",
        "        self.iter_dt = 0.0\n",
        "\n",
        "    def add_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent].append(callback)\n",
        "\n",
        "    def set_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent] = [callback]\n",
        "\n",
        "    def trigger_callbacks(self, onevent: str):\n",
        "        for callback in self.callbacks.get(onevent, []):\n",
        "            callback(self)\n",
        "\n",
        "    def run(self):\n",
        "        model, config = self.model, self.config\n",
        "        self.optimizer = model.configure_optimizers(config)\n",
        "        batch_size = self.config.batch_size\n",
        "        def get_batch(mode):\n",
        "            batch_size = 64\n",
        "            if mode == \"train\":\n",
        "              data = train_dataset\n",
        "              pos_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]),513))\n",
        "              neg_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]),513))\n",
        "              N = np.random.randint(batch_size)\n",
        "              ix_pos = np.random.randint(pos_train.shape[0]-N)\n",
        "              ix_neg = np.random.randint(neg_train.shape[0]-batch_size-N)\n",
        "              pos_data = pos_train[ix_pos:ix_pos+N,:]\n",
        "              neg_data = neg_train[ix_neg:ix_neg+batch_size-N,:]\n",
        "              data = torch.concat((pos_data,neg_data))\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,2), device=self.device)\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o][0] = 1\n",
        "                else:\n",
        "                  targets[o][1] = 1\n",
        "            else:\n",
        "              data = val_dataset\n",
        "              ix = np.random.randint(data.shape[0]-batch_size)\n",
        "              data = data[ix:ix+batch_size,:]\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,2), device=self.device)\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o][0] = 1\n",
        "                else:\n",
        "                  targets[o][1] = 1\n",
        "            targets = targets.view(batch_size,2).to(self.device)\n",
        "            seq = seq.view(batch_size,512).to(self.device)\n",
        "            return seq, targets\n",
        "\n",
        "        @torch.no_grad\n",
        "        def cross_val():\n",
        "          model.eval()\n",
        "          out = []\n",
        "          for i in [\"train\",\"val\"]:\n",
        "            losses = torch.zeros(200+1)\n",
        "            for k in range(200):\n",
        "              X,Y = get_batch(i)\n",
        "              logits,loss = model(X,Y)\n",
        "              losses[k]=loss.item()\n",
        "              out1 = losses.mean()\n",
        "            out.append(out1)\n",
        "          model.train()\n",
        "          return out\n",
        "        losses = cross_val()\n",
        "        LOSS = [losses]\n",
        "        print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        model.train()\n",
        "        for epoch in range(20):\n",
        "          print(\"[epoch {o}] \\n\".format(o = epoch))\n",
        "          iters = 200\n",
        "          for i in range(iters):\n",
        "            if i == 0:\n",
        "              Y = \"| =\"\n",
        "            elif i == iters -1 :\n",
        "              Y = \"=> 100% |\"\n",
        "            else:\n",
        "              if i%(int(iters/50)) == 0 :\n",
        "                Y = \"=\"\n",
        "              else:\n",
        "                Y = \"\"\n",
        "            print(\"{y}\".format(y = Y),end=\"\")\n",
        "            x, y = get_batch(\"train\")\n",
        "            logits, self.loss = model(x, y)\n",
        "            model.zero_grad(set_to_none=True)\n",
        "            self.loss.backward()\n",
        "            torch.nn.utils.clip_grad_norm_(model.parameters(), config.grad_norm_clip)\n",
        "            self.optimizer.step()\n",
        "          losses = cross_val()\n",
        "          LOSS.append(losses)\n",
        "          print(LOSS)\n",
        "          print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        PATH = \"model {h}\".format(h = model_config.model_type)\n",
        "        torch.save(model.state_dict(), PATH)\n",
        "class Transformer():\n",
        "  def __init__(self,mode):\n",
        "    self.mode = mode\n",
        "    self.device = torch.device(\"cuda\")\n",
        "    self.vocab_dict = tokenizer.get_vocab()\n",
        "\n",
        "  def classifier(self):\n",
        "    device = self.device\n",
        "    model_config = ClassifierI.get_default_config()\n",
        "    model_config.vocab_size = 25\n",
        "    model_config.block_size = 512\n",
        "    if self.mode == \"b\":\n",
        "      model_config.model_type = 'b'\n",
        "      model2 = ClassifierI(model_config)\n",
        "      model2.load_state_dict(torch.load(\"classi/model_b\",map_location=self.device))\n",
        "    else:\n",
        "      model_config.model_type = 'c'\n",
        "      model2 = ClassifierI(model_config)\n",
        "      model2.load_state_dict(torch.load(\"classi/model_c\",map_location = self.device))\n",
        "    model2.to(device)\n",
        "    return model2\n",
        "\n",
        "  def Encode(self, i):\n",
        "        def encode_char(x):\n",
        "            return [self.vocab_dict[char] for char in x]\n",
        "\n",
        "        def pad_sequences(x, PAD=0, max_len=512):\n",
        "            return np.array([seq + [PAD] * (max_len - len(seq)) for seq in x])\n",
        "\n",
        "        encoded = list(map(encode_char, i))\n",
        "        padded = pad_sequences(encoded)\n",
        "        return torch.tensor(padded).to(self.device)\n",
        "\n",
        "  def Decode(self,i):\n",
        "    def decode(k):\n",
        "      l = [j for j in k if j != 0 ]\n",
        "      seq = [chars[j] for j in l]\n",
        "      return \"\".join(seq)\n",
        "    H = list(map(decode,i))\n",
        "    G = []\n",
        "    for i in H:\n",
        "      if len(i)>512:\n",
        "        u = H.index(i)\n",
        "        for j in range(len(i)-512):\n",
        "          G.append(i[j:j+512])\n",
        "      else:\n",
        "        G.append(i)\n",
        "    return G,u\n",
        "  def Mean(self,i):\n",
        "    I = list(i.ravel())\n",
        "    sig = lambda x:np.e**(0*x)\n",
        "    Sum = [sig(np.abs(x-int(len(I)/2))) for x in range(len(I))]\n",
        "    wei = np.array([Sum[i]/sum(Sum) for i in range(len(I))]).reshape(len(I),1)\n",
        "    return np.sum(wei*i)\n",
        "\n",
        "  def enhancer(self,x):\n",
        "    wei = np.array([np.exp((i[0]-1)/(i[0])) for i in x])\n",
        "    return np.sum(np.array([i[0] for i in x])*wei)/np.sum(wei)\n",
        "\n",
        "  def predict_proba(self,i):\n",
        "    if len(i[0])>512:\n",
        "       l = [self.Encode([i[0][j:j+512]]) for j in range(len(i[0])-512)]\n",
        "       if len(l)>700:\n",
        "        L = [torch.concat(tuple(l[i*700:(i+1)*700])) for i in range(len(l)//700)] + [torch.concat(tuple(l[(len(l)//700)*700:(len(l)//700)*700+len(l)%700]))]\n",
        "        T = [self.classifier().predict_proba(i,\"sig\").tolist() for i in L ]\n",
        "        return self.enhancer(np.concatenate(tuple([i for i in T])))\n",
        "       else:\n",
        "         t = torch.concat(tuple(l))\n",
        "         return self.enhancer(self.classifier().predict_proba(t,\"sig\").tolist())\n",
        "    else:\n",
        "       seq = self.Encode(i)\n",
        "       U = np.array(self.classifier().predict_proba(seq,\"sig\").tolist())\n",
        "       out = np.array(U)\n",
        "       return out.reshape(len(i),1)\n",
        "\n",
        "model2 = Transformer(\"b\")\n",
        "model3 = Transformer(\"c\")\n",
        "\n",
        "\n",
        "import functools as FUNC\n",
        "from numba import njit\n",
        "\n",
        "class PPA():\n",
        "    def __init__(self, seq, model):\n",
        "        self.model = model\n",
        "        self.seq = seq\n",
        "        self.len = len(self.seq)\n",
        "        self.idx = None\n",
        "\n",
        "    def attention_mask(self):\n",
        "        idx2 = self.splicer(self.seq)[1]\n",
        "        k = self.T()\n",
        "        mask = [[True if len(k[0]) > idx2[j] - i > -1 else False for i in range(len(k[j]))] for j in range(len(k))]\n",
        "        return mask\n",
        "\n",
        "    def mean(self, mask, spliced):\n",
        "        u = [np.mean(self.model.predict_proba([spliced[i][j] for j in range(len(spliced[i])) if mask[i][j]])) for i in trange(len(spliced))]\n",
        "        return u\n",
        "\n",
        "    def splice_scorer(self, i):\n",
        "        lmin = len(i) // 2\n",
        "        return [i[j:j + lmin] for j in range(len(i) - lmin + 1)]\n",
        "\n",
        "    def splicer(self, i):\n",
        "        idx = self.idx\n",
        "        o, Before = [], []\n",
        "        for j in range(len(i)):\n",
        "            n_b, n_a = j, len(i) - j - 1\n",
        "            if n_b <= idx // 2:\n",
        "                before = i[max(0, j - idx // 2):j]\n",
        "                after = i[j + 1:j + idx - len(before)]\n",
        "            elif n_a <= idx // 2:\n",
        "                before = i[j - idx + n_a + 1:j]\n",
        "                after = i[j + 1:]\n",
        "            else:\n",
        "                before = i[j - idx // 2:j]\n",
        "                after = i[j + 1:j + idx // 2 + 1]\n",
        "            o.append(before + i[j] + after)\n",
        "            Before.append(len(before))\n",
        "        return o, Before\n",
        "\n",
        "    def T(self):\n",
        "        return list(map(self.splice_scorer, self.splicer(self.seq)[0]))\n",
        "\n",
        "    def Out(self):\n",
        "        s = self.s\n",
        "        def d(x):\n",
        "            if s > 0.7:\n",
        "                return x * np.exp(-1.2 * (x - s))\n",
        "            else:\n",
        "                return x if x > 0.7 else x\n",
        "\n",
        "        return list(map(d, self.mean(self.attention_mask(), self.T())))\n",
        "def show(i):\n",
        "  values = np.array(i).reshape(1, len(i))\n",
        "  y = range(len(i))\n",
        "  Y = [(j,1) for j in range(len(i)) if np.mean([i[j],i[min((j+1),len(i)-1)],i[max(0,(j-1))]])>0.6]\n",
        "  Y2 = [(j,1) for j in range(len(i)) if np.mean([i[j],i[min((j+1),len(i)-1)],i[min((j+2),len(i)-1)],i[min((j+3),len(i)-1)],i[min((j+4),len(i)-1)],i[min((j+5),len(i)-1)],i[max(0,(j-1))],i[max(0,(j-2))],i[max(0,(j-3))],i[max(0,(j-4))],i[max(0,(j-5))]])>0.85]\n",
        "  threshold = 0.5\n",
        "  above_threshold = np.maximum(values - threshold, 0)\n",
        "  below_threshold = np.minimum(values, threshold)\n",
        "  fig, (ax3 , ax4) = plt.subplots(2,1,sharex = True,sharey = True)\n",
        "  ax3.bar(y, below_threshold.ravel(), 0.9, color=\"lightblue\")\n",
        "  ax3.bar(y, above_threshold.ravel(), 0.9, color=\"lightgreen\",\n",
        "               bottom=below_threshold.ravel())\n",
        "  ax4.broken_barh(Y, (0.7, 0.03),\n",
        "               facecolors=( 'red'))\n",
        "  ax4.plot([0., y[-1]],[0.715,0.715],\"-b\")\n",
        "  for k in Y:\n",
        "    ax4.plot([k[0],k[0]+1],[0.715,0.715],\"-r\")\n",
        "\n",
        "  ax4.text(0,0.8 , \"LLPS-susceptible regions\",fontsize = 20)\n",
        "  ax3.plot([0., y[-1]], [threshold, threshold], \"k--\")\n",
        "  ax4.set_ylim([0,1])\n",
        "  ax3.set_ylim([0,1])\n",
        "\n",
        "  plt.subplots_adjust(\n",
        "                    wspace=0.4,\n",
        "                    hspace=0)\n",
        "  fig.set_figwidth(20)\n",
        "  fig.set_figheight(15)\n",
        "  plt.show()\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 49,
      "metadata": {
        "id": "_R4AYlL7PfO2",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "cellView": "form",
        "outputId": "29b3582c-14af-4004-b3e3-c5f8d12d7c4c"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Score : [[0.82050598]]\n"
          ]
        }
      ],
      "source": [
        "#@title Loading sequence , then hit `Runtime` -> `Run all`\n",
        "#@markdown - Load the sequence of your protein :\n",
        "Sequence = \"MGRKKKKVDKPWCWYCNREFDDEKILIQHQKAKHFKCHICHKKLFSGPGLSIHCMQVHKETIDKIPAAVH\\\\nGRDNIHVEIYGMQGIPSGAYRGAADEEPDEKRSRMDNGPPMPTPMPFPQHFPFPGMPPMPSGPPPPSMAY\\\\nGMPPMPSGMMPPRGMPGAYPPPRGYPPAPAPGVYMPPPGMPGAYPPPRMPIGHGPPGGPPMPGPPQRSRF\\\\nDQPDGGDRWGPPMRGVPRTPPYENEHEHQDYQAPDDYNQGGYDDRFREGDRGGPPGGSRFDQVKAELKEE\\\\nIVEPNNGPPTTSSGAAAASAAAVVASKLGSRTRIVYPDDHSLSLEERRVKMLFEKRSNPYH\" #@param {type:\"string\"}\n",
        "ID = None #@param {type:\"string\"}\n",
        "Directory = None #@param {type:\"string\"}\n",
        "#@markdown  - Use `End_sequence` to focus the scoring on sequences up to the specified endpoint\n",
        "\n",
        "End_sequence = 40 #@param {type:\"slider\" , min : 0 , max: 1000 , step : 1}\n",
        "Plot = True #@param {type: \"boolean\"}\n",
        "\n",
        "#@markdown - Specify the path of `FASTA format file` of your sequences.\n",
        "Fasta_file = \"SD.txt\" #@param {type:\"string\"}\n",
        "import pandas as pd\n",
        "#assert [Sequence == None , Fasta_file == None].count(True) == 1\n",
        "\n",
        "def enh(x):\n",
        "  wei = np.array([np.exp((i-1)/(i)) for i in x])\n",
        "  return np.sum(np.array(x)*wei)/np.sum(wei)\n",
        "\n",
        "def edit(i):\n",
        "  for j in i:\n",
        "    if j in \"\\t\\nBJOUXZ, _@#$%^& *n({}[])\\/\\\\\":\n",
        "      i = i.replace(j,\"\")\n",
        "  return i\n",
        "\n",
        "\n",
        "def SW(s):\n",
        "  if len(s)> 700:\n",
        "    L= [i for i in  [s[i*700:(i+1)*700] for i in range(len(s)//700+1)] if len(i)>0]\n",
        "    H = []\n",
        "    for i in range(len(L)):\n",
        "      H.append(model3.predict_proba(L[i]))\n",
        "    return H\n",
        "  else:\n",
        "    return model3.predict_proba(s)\n",
        "\n",
        "\n",
        "def Score1(i,Sc):\n",
        "    i = i-1\n",
        "    if i<= L:\n",
        "      return sum([Sc[j] for j in range(i+1)])/(i+1)\n",
        "\n",
        "    elif L+1<=i<n-L:\n",
        "       return sum([Sc[j] for j in range(i-L,i+1)])/(L+1)\n",
        "\n",
        "    else:\n",
        "       u = (n-(i)+1)\n",
        "       return sum([Sc[j] for j in range(i-L,n-L+1)])/(u)\n",
        "Sequence = edit(Sequence)\n",
        "\n",
        "u = model3.predict_proba([Sequence])\n",
        "\n",
        "def d(x):\n",
        "    if u>0.7:\n",
        "        return x*np.exp(-1.2*(x-u))\n",
        "    else:\n",
        "        if x > 0.7 :\n",
        "            return x\n",
        "        else:\n",
        "            return x\n",
        "\n",
        "\n",
        "if Sequence != None and Sequence !=\"\":\n",
        "  Sequence = edit(Sequence)\n",
        "  k = max(5,min(50,int(np.ceil(0.1*len(Sequence)))))\n",
        "  L = k//2\n",
        "  if Plot == True:\n",
        "    n = len(Sequence)\n",
        "    S = [Sequence[i:i+L] for i in range(len(Sequence)-L+1)]\n",
        "    Sc = SW(S)\n",
        "    Sc1 = Sc[0]\n",
        "    for i in Sc[1:]:\n",
        "      Sc1 = np.concatenate((Sc1,i))\n",
        "\n",
        "    p = list(map(d,[Score1(i,Sc1) for i in range(1,n+1)]))\n",
        "    if type(p[0]) != np.float64:\n",
        "      p = [i[0] for i in p]\n",
        "    if type(p[0]) != np.float64:\n",
        "      p = [i[0] for i in p]\n",
        "\n",
        "    u = model3.predict_proba([Sequence])\n",
        "    i = p\n",
        "    values = np.array(i).reshape(1, len(i))\n",
        "    below_threshold = [k if k<= 0.5 else 0.5 for k in i]\n",
        "    above_threshold = [0 if k<= 0.5 else k-0.5 for k in i]\n",
        "    threshold = 0.5\n",
        "    y = list(range(len(i)))\n",
        "    Y = [(j, 1) for j in range(len(i)) if np.mean([i[j], i[min((j + 1), len(i) - 1)], i[max(0, (j - 1))]]) > 0.6]\n",
        "    Y2 = [(j, 1) for j in range(len(i)) if np.mean([i[j], i[min((j + 1), len(i) - 1)], i[min((j + 2), len(i) - 1)],\n",
        "                                               i[min((j + 3), len(i) - 1)], i[min((j + 4), len(i) - 1)],\n",
        "                                               i[min((j + 5), len(i) - 1)], i[max(0, (j - 1))],\n",
        "                                               i[max(0, (j - 2))], i[max(0, (j - 3))], i[max(0, (j - 4))],\n",
        "                                               i[max(0, (j - 5))]]) > 0.85]\n",
        "    #if n>= 1500:\n",
        "      #c1 = \"blue\"\n",
        "      #c2 = \"green\"\n",
        "    #else:\n",
        "    c1 = \"lightblue\"\n",
        "    c2 = \"lightgreen\"\n",
        "\n",
        "    fig1 = make_subplots(rows=1, cols=1, shared_xaxes=True, shared_yaxes=True, vertical_spacing=0.02)\n",
        "    fig1.add_trace(go.Bar(x=y, y=below_threshold, marker_color= c1, name='Below Threshold', width= 0.6), row=1, col=1)\n",
        "    fig1.add_trace(go.Bar(x=y, y=above_threshold, marker_color=c2, name='Above Threshold', base=below_threshold, width= 0.6), row=1, col=1)\n",
        "\n",
        "    fig1.add_trace(go.Scatter(x=[0, len(i) - 1], y=[threshold, threshold], mode='lines', line=dict(color='black', dash='dash'), showlegend=False), row=1, col=1)\n",
        "    for start, duration in Y:\n",
        "        fig1.add_trace(go.Scatter(x=[start, start + duration], y=[0.01, 0.01], mode='lines', line=dict(color='red', width=7), showlegend=False), row=1, col=1)\n",
        "    fig1.update_layout(height=800, width=1700, title_text=\"LLPS Prediction Plot\")\n",
        "    score = str(((2*u+2*enh(p))/4))\n",
        "    print(\"Score : \"+score)\n",
        "  else:\n",
        "    score = u\n",
        "    print(\"Score : \"+str(score))\n",
        "\n",
        "\n",
        "else:\n",
        "  if os.path.exists(Fasta_file):\n",
        "    with open(Fasta_file) as f:\n",
        "      U = f.read().split(\">\")\n",
        "      w = len(U)\n",
        "    j = U[1:]\n",
        "    D = {\"id\":[i[:i.index(\"\\n\")] if \"\\n\" in i else i for i in j],\"seq\":[edit(i[i.index(\"\\n\"):]) if \"\\n\" in i else None for i in j],\"score\":[]}\n",
        "    #D[\"score\"] = [i[0][0] if type(i) != np.float64 else i for i in  [model3.predict_proba([s]) for s in tqdm(D[\"seq\"])]]\n",
        "    for Sequence in tqdm(D[\"seq\"]):\n",
        "      if Sequence == None :\n",
        "        D[\"score\"].append(\"_\")\n",
        "      else:\n",
        "        Sequence = edit(Sequence)\n",
        "        k = max(5,min(50,int(np.ceil(0.1*len(Sequence)))))\n",
        "        L = k//2\n",
        "        n = len(Sequence)\n",
        "        S = [Sequence[i:i+L] for i in range(len(Sequence)-L+1)]\n",
        "        if len(S)%700 == 0:\n",
        "          D[\"score\"].append(\"_\")\n",
        "        else:\n",
        "          Sc = SW(S)\n",
        "          Sc1 = Sc[0]\n",
        "          for i in Sc[1:]:\n",
        "            Sc1 = np.concatenate((Sc1,i))\n",
        "          u = model3.predict_proba([Sequence])\n",
        "          p = list(map(d,[Score1(i,Sc1) for i in range(1,n+1)]))\n",
        "          D[\"score\"].append(str(((2*u+2*enh(p))/4)))\n",
        "      if len(D[\"score\"])%1000 == 0:\n",
        "        print(D[\"score\"])\n",
        "\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title plot\n",
        "if Plot == True:\n",
        "  fig1.show()\n",
        "import matplotlib.pyplot as plt\n",
        "import streamlit as st\n",
        "from stmol import showmol\n",
        "import py3Dmol\n",
        "import requests\n",
        "import biotite.structure.io as bsio\n",
        "import certifi\n",
        "import matplotlib.colors as mcolors\n",
        "import urllib3\n",
        "\n",
        "urllib3.disable_warnings()\n",
        "\n",
        "st.sidebar.title('üéà ESMFold')\n",
        "st.sidebar.write('[*ESMFold*](https://esmatlas.com/about) is an end-to-end single sequence protein structure predictor based on the ESM-2 language model. For more information, read the [research article](https://www.biorxiv.org/content/10.1101/2022.07.20.500902v2) and the [news article](https://www.nature.com/articles/d41586-022-03539-1) published in *Nature*.')\n",
        "\n",
        "def render_mol(pdb):\n",
        "    pdbview = py3Dmol.view()\n",
        "    pdbview.addModel(pdb, 'pdb')\n",
        "    pdbview.setStyle({'cartoon': {'color': 'spectrum'}})\n",
        "    pdbview.setBackgroundColor('white')  # ('0xeeeeee')\n",
        "    pdbview.zoomTo()\n",
        "    pdbview.zoom(2, 800)\n",
        "    pdbview.spin(True)\n",
        "    showmol(pdbview, height=500, width=800)\n",
        "\n",
        "\n",
        "DEFAULT_SEQ = Sequence\n",
        "show_mainchains = True #@param {type: \"boolean\"}\n",
        "show_sidechains = True #@param {type: \"boolean\"}\n",
        "Color_Style =  \"yellow_blue_spectrum\" #@param [\"red_blue_spectrum\",\"green_magenta_spectrum\",\"yellow_blue_spectrum\",\"cyan_red_spectrum\"]\n",
        "\n",
        "import streamlit as st\n",
        "import requests\n",
        "import biotite.structure.io as bsio\n",
        "import py3Dmol\n",
        "\n",
        "DEFAULT_SEQ = Sequence\n",
        "\n",
        "txt = st.sidebar.text_area('Input sequence', DEFAULT_SEQ, height=275)\n",
        "\n",
        "def update(sequence,name):\n",
        "    headers = {\n",
        "        'Content-Type': 'application/x-www-form-urlencoded',\n",
        "    }\n",
        "\n",
        "    def get_pdb(sequence):\n",
        "        response = requests.post('https://api.esmatlas.com/foldSequence/v1/pdb/', headers=headers, data=sequence, verify=False)\n",
        "        if response.status_code != 200:\n",
        "            st.error(f\"Failed to get prediction: {response.status_code}\")\n",
        "            return None\n",
        "        pdb_string = response.content.decode('utf-8')\n",
        "        if not pdb_string or \"ATOM\" not in pdb_string:\n",
        "            st.error(\"Received an invalid PDB file.\")\n",
        "            return None\n",
        "        return pdb_string\n",
        "\n",
        "    def save_and_render_pdb(pdb_string, file_name):\n",
        "        with open(file_name, 'w') as f:\n",
        "            f.write(pdb_string)\n",
        "        try:\n",
        "            struct = bsio.load_structure(file_name, extra_fields=[\"b_factor\"])\n",
        "            b_value = round(struct.b_factor.mean(), 4)\n",
        "        except Exception as e:\n",
        "            st.error(f\"Error loading structure: {e}\")\n",
        "            return\n",
        "        st.subheader('Visualization of predicted protein structure')\n",
        "        render_mol(pdb_string)\n",
        "        st.subheader('plDDT')\n",
        "        st.write('plDDT is a per-residue estimate of the confidence in prediction on a scale from 0-100.')\n",
        "        st.info(f'plDDT: {b_value}')\n",
        "        st.download_button(label=\"Download PDB\", data=pdb_string, file_name=file_name, mime='text/plain')\n",
        "\n",
        "    def render_mol(pdb_string,neme):\n",
        "        view = py3Dmol.view(width=1000, height=800)\n",
        "        view.addModel(pdb_string, 'pdb')\n",
        "        F1 = Get_color(p)\n",
        "        F1.i = F1.J.index(Color_Style)\n",
        "\n",
        "        custom_colors=F1.spectrum()\n",
        "\n",
        "        for i, color in enumerate(custom_colors[1:-1]):\n",
        "          view.setStyle({ 'resi': i+1}, {'cartoon': {'color': color}})\n",
        "        #view.setStyle({'cartoon': {'color': 'spectrum'}})\n",
        "        view.zoomTo()\n",
        "        view.show()\n",
        "        html_content = view._make_html()\n",
        "        with open(name+'.html', 'w') as file:\n",
        "          file.write(html_content)\n",
        "        files.download(name+'.html')\n",
        "\n",
        "\n",
        "    chunk_size = 400\n",
        "    chunks = [sequence[i:i + chunk_size] for i in range(0, len(sequence), chunk_size)]\n",
        "\n",
        "    combined_pdb_string = \"\"\n",
        "    for i, chunk in enumerate(chunks):\n",
        "        st.write(f\"Processing chunk {i + 1} of {len(chunks)}\")\n",
        "        pdb_string = get_pdb(chunk)\n",
        "        if pdb_string:\n",
        "            combined_pdb_string += pdb_string\n",
        "\n",
        "    if combined_pdb_string:\n",
        "        render_mol(combined_pdb_string,name)\n",
        "        with open('combined_predicted.pdb', 'w') as f:\n",
        "            f.write(combined_pdb_string)\n",
        "        st.download_button(label=\"Download Combined PDB\", data=combined_pdb_string, file_name='combined_predicted.pdb', mime='text/plain')\n",
        "\n",
        "predict = st.sidebar.button('Predict', on_click=update)\n",
        "\n",
        "if not predict:\n",
        "    st.warning('üëà Enter protein sequence data!')\n",
        "\n",
        "class Get_color():\n",
        "  def __init__(self,values):\n",
        "    self.J = [\"red_blue_spectrum\",\"green_magenta_spectrum\",\"yellow_blue_spectrum\",\"cyan_red_spectrum\"]\n",
        "    self.i = None\n",
        "    self.values = values\n",
        "\n",
        "  def get_red_blue_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=min(values), vmax=max(values))\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"RedBlue\", [\"blue\", \"red\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "  def get_green_magenta_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=min(values), vmax=max(values))\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"GreenMagenta\", [\"green\", \"magenta\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "  def get_yellow_blue_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=min(values), vmax=max(values))\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"YellowBlue\", [\"yellow\", \"blue\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "\n",
        "  def get_cyan_red_spectrum(self,values):\n",
        "      norm = mcolors.Normalize(vmin=min(values), vmax=max(values))\n",
        "      cmap = mcolors.LinearSegmentedColormap.from_list(\"CyanRed\", [\"cyan\", \"red\"])\n",
        "      spectrum = [mcolors.to_hex(cmap(norm(value))) for value in values]\n",
        "      return spectrum\n",
        "\n",
        "  def spectrum(self):\n",
        "    if self.i == 0 : return self.get_red_blue_spectrum(self.values)\n",
        "    if self.i == 1 : return self.get_green_magenta_spectrum(self.values)\n",
        "    if self.i == 2 : return self.get_yellow_blue_spectrum(self.values)\n",
        "    if self.i == 3 : return self.get_cyan_red_spectrum(self.values)\n",
        "update(txt,\"A\")\n",
        "\n",
        "\n",
        "# Create a color bar legend using matplotlib\n",
        "fig, ax = plt.subplots(figsize=(6, 1))\n",
        "fig.subplots_adjust(bottom=0.5)\n",
        "\n",
        "cmap = plt.cm.Reds\n",
        "norm = mcolors.Normalize(vmin=0, vmax=1)\n",
        "fig.colorbar(plt.cm.ScalarMappable(norm=norm, cmap=cmap), cax=ax, orientation='horizontal', label='Predicted LLPS Score')\n",
        "plt.show()\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n"
      ],
      "metadata": {
        "id": "QXTbh0DQZm69",
        "cellView": "form",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "fcbcffd3-179d-4b38-a0c4-1d02ab8da2e3"
      },
      "execution_count": 50,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<html>\n",
              "<head><meta charset=\"utf-8\" /></head>\n",
              "<body>\n",
              "    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax && window.MathJax.Hub && window.MathJax.Hub.Config) {window.MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n",
              "        <script charset=\"utf-8\" src=\"https://cdn.plot.ly/plotly-2.24.1.min.js\"></script>                <div id=\"654db578-7a94-4a4d-9254-df474eac4dde\" class=\"plotly-graph-div\" style=\"height:800px; width:1700px;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"654db578-7a94-4a4d-9254-df474eac4dde\")) {                    Plotly.newPlot(                        \"654db578-7a94-4a4d-9254-df474eac4dde\",                        [{\"marker\":{\"color\":\"lightblue\"},\"name\":\"Below Threshold\",\"width\":0.6,\"x\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131,132,133,134,135,136,137,138,139,140,141,142,143,144,145,146,147,148,149,150,151,152,153,154,155,156,157,158,159,160,161,162,163,164,165,166,167,168,169,170,171,172,173,174,175,176,177,178,179,180,181,182,183,184,185,186,187,188,189,190,191,192,193,194,195,196,197,198,199,200,201,202,203,204,205,206,207,208,209,210,211,212,213,214,215,216,217,218,219,220,221,222,223,224,225,226,227,228,229,230,231,232,233,234,235,236,237,238,239,240,241,242,243,244,245,246,247,248,249,250,251,252,253,254,255,256,257,258,259,260,261,262,263,264,265,266,267,268,269,270,271,272,273,274,275,276,277,278,279,280,281,282,283,284,285,286,287,288,289,290,291,292,293,294,295,296,297,298,299,300,301,302,303,304,305,306,307,308,309,310,311,312,313,314,315,316,317,318,319,320,321,322,323,324,325,326,327,328,329,330,331,332,333,334,335,336,337,338,339,340],\"y\":[0.1441957691060967,0.5,0.5,0.5,0.4653767777032518,0.42645634987745146,0.4245323823775458,0.39668216334340334,0.37237023465242586,0.35215489047420345,0.33763860411254326,0.34237598920848805,0.3283981762342421,0.41075462002883956,0.493012069311906,0.47532544772129887,0.45942187542445023,0.4450227824370181,0.44478196690368693,0.3660326853635046,0.36515151463224654,0.365178816202485,0.3647383767359384,0.362652602386772,0.349180601074655,0.34774357759862023,0.3487052590823452,0.3906923503823818,0.38899036582826646,0.38812110832236824,0.4557239515243683,0.38999762954900186,0.29509025784095805,0.37301047160506573,0.44378264345720386,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.44238391657486437,0.45747381743095844,0.37377963528707564,0.37363254915679234,0.3724363515652536,0.2753594718539187,0.2753094901439069,0.2620950815112726,0.2625946268851752,0.26505661813987463,0.2660741183045638,0.2666307283965601,0.2666239663684831,0.2688423814583718,0.3211842259776935,0.3213544755087171,0.32129043616338643,0.32137541578042195,0.3163049372252466,0.22341392414240027,0.22326832527676957,0.22294213974604263,0.2228818493069784,0.2229303585545434,0.22273473513689324,0.3208642447227853,0.320147236740141,0.3179529086420577,0.410534088920696,0.49248959362241784,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.4896323027047704,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.4689518504419567,0.3854811371127678,0.3883615017984799,0.3880395218015078,0.38530543530188716,0.3745278153921799,0.37424844123336354,0.37307110761881546,0.37273854939187745,0.34384218914010284,0.34440471991909316,0.34384516198152704,0.3445070008481848,0.34408150063741055,0.343639639059632,0.34312155492961294,0.3428239969307352,0.342559307986305,0.34776947151972554,0.3586356819284374,0.25898759505604124,0.2663658041538897,0.27477713110129415,0.1474478028236056,0.14812264451456855,0.14907150756858115,0.15076791162057984,0.1530108473461387,0.1530445491726454,0.1562670687580644,0.15429701234890478,0.15850132559983593,0.16470471530259542,0.17557616481113886,0.19655517178227921],\"type\":\"bar\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"base\":[0.1441957691060967,0.5,0.5,0.5,0.4653767777032518,0.42645634987745146,0.4245323823775458,0.39668216334340334,0.37237023465242586,0.35215489047420345,0.33763860411254326,0.34237598920848805,0.3283981762342421,0.41075462002883956,0.493012069311906,0.47532544772129887,0.45942187542445023,0.4450227824370181,0.44478196690368693,0.3660326853635046,0.36515151463224654,0.365178816202485,0.3647383767359384,0.362652602386772,0.349180601074655,0.34774357759862023,0.3487052590823452,0.3906923503823818,0.38899036582826646,0.38812110832236824,0.4557239515243683,0.38999762954900186,0.29509025784095805,0.37301047160506573,0.44378264345720386,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.44238391657486437,0.45747381743095844,0.37377963528707564,0.37363254915679234,0.3724363515652536,0.2753594718539187,0.2753094901439069,0.2620950815112726,0.2625946268851752,0.26505661813987463,0.2660741183045638,0.2666307283965601,0.2666239663684831,0.2688423814583718,0.3211842259776935,0.3213544755087171,0.32129043616338643,0.32137541578042195,0.3163049372252466,0.22341392414240027,0.22326832527676957,0.22294213974604263,0.2228818493069784,0.2229303585545434,0.22273473513689324,0.3208642447227853,0.320147236740141,0.3179529086420577,0.410534088920696,0.49248959362241784,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.4896323027047704,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.4689518504419567,0.3854811371127678,0.3883615017984799,0.3880395218015078,0.38530543530188716,0.3745278153921799,0.37424844123336354,0.37307110761881546,0.37273854939187745,0.34384218914010284,0.34440471991909316,0.34384516198152704,0.3445070008481848,0.34408150063741055,0.343639639059632,0.34312155492961294,0.3428239969307352,0.342559307986305,0.34776947151972554,0.3586356819284374,0.25898759505604124,0.2663658041538897,0.27477713110129415,0.1474478028236056,0.14812264451456855,0.14907150756858115,0.15076791162057984,0.1530108473461387,0.1530445491726454,0.1562670687580644,0.15429701234890478,0.15850132559983593,0.16470471530259542,0.17557616481113886,0.19655517178227921],\"marker\":{\"color\":\"lightgreen\"},\"name\":\"Above Threshold\",\"width\":0.6,\"x\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131,132,133,134,135,136,137,138,139,140,141,142,143,144,145,146,147,148,149,150,151,152,153,154,155,156,157,158,159,160,161,162,163,164,165,166,167,168,169,170,171,172,173,174,175,176,177,178,179,180,181,182,183,184,185,186,187,188,189,190,191,192,193,194,195,196,197,198,199,200,201,202,203,204,205,206,207,208,209,210,211,212,213,214,215,216,217,218,219,220,221,222,223,224,225,226,227,228,229,230,231,232,233,234,235,236,237,238,239,240,241,242,243,244,245,246,247,248,249,250,251,252,253,254,255,256,257,258,259,260,261,262,263,264,265,266,267,268,269,270,271,272,273,274,275,276,277,278,279,280,281,282,283,284,285,286,287,288,289,290,291,292,293,294,295,296,297,298,299,300,301,302,303,304,305,306,307,308,309,310,311,312,313,314,315,316,317,318,319,320,321,322,323,324,325,326,327,328,329,330,331,332,333,334,335,336,337,338,339,340],\"y\":[0,0.23726897784773338,0.1126284596507443,0.02586345190192696,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.018680801642858902,0.018806461621059478,0.020008150477778708,0.08627010264583479,0.08606331915895749,0.09377877990870065,0.09377260539239152,0.0935192493611029,0.09357822416392036,0.09252502488203496,0.06371187272126311,0.06398420065914556,0.05845034882658917,0.0027353872726081363,0.0012956145804734476,0.0011248080823124118,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.06327156581618054,0.1231527686743139,0.09060019472698255,0.09067982657866591,0.1480734902550861,0.19644295480160223,0.23727739780286505,0.23313390991202776,0.23319459439351298,0.26782551104582986,0.2959147071744994,0.3168348484385505,0.33176616237576184,0.3194996269568825,0.33615719004939815,0.3426608201804384,0.3371012661306281,0.3370802222303254,0.33715048286116955,0.32098347934139504,0.33710595167369306,0.3380913798064591,0.3380637175299678,0.33811655425906606,0.3223885389821688,0.33821297459085686,0.34946447922889723,0.3494745212064134,0.34945591705569,0.34142865278893986,0.3296626481465066,0.3395557410601484,0.3395291458620924,0.3454220629523044,0.3497534574836215,0.3497004814798682,0.34970004518266007,0.35672709709037853,0.3567569486315554,0.359980661534426,0.35998788106310786,0.35998848310064924,0.3603059841716725,0.360305161697663,0.3603070524670524,0.36030545909424905,0.36030453083447667,0.3584277680551766,0.35380206336508413,0.35315381475562235,0.35313975146911825,0.3530959275010003,0.35308759684569613,0.35304303132779813,0.35304454694440524,0.3530477814712899,0.3580241608139675,0.35801763958541466,0.35802709579407677,0.3580092240847005,0.35801979756204294,0.3580194438836699,0.3580110379446416,0.35804591064891633,0.358045742718343,0.35801267872840825,0.3580145732676593,0.3580276107755267,0.35804096942890473,0.35803852606345066,0.35983796254153266,0.35983546293668256,0.35983593971059147,0.35984864580725706,0.3564767719118358,0.35648226876424416,0.35937770340509223,0.36053717835331467,0.3605379024175783,0.3605372386591912,0.3589997826150517,0.35902034554878837,0.35902134975248423,0.3590029278196889,0.35900023895058664,0.35899831982456987,0.3587341673164657,0.35871184219765806,0.3602076309770461,0.36020822276307185,0.3602077156617116,0.3575266406870973,0.35751688806189796,0.35750611555411704,0.3567488725767365,0.3600329329144307,0.36003112622954836,0.3600324894520228,0.35993131100515763,0.3599319859608968,0.35993042009493403,0.35993738045682955,0.3563267354799714,0.3563460902015413,0.35677398128810967,0.35019848873742176,0.3503769796581476,0.34487649471639725,0.3319607675591165,0.3307855077441997,0.33087288422030503,0.3309657843381302,0.34393282929888724,0.3439745604505916,0.34399351897940234,0.3440478615662613,0.35309711791216947,0.35311095168605855,0.35311855382788226,0.35313172241939617,0.3585256391073074,0.3585224304491633,0.3585066160300693,0.3604985789349493,0.35868881007223385,0.357001062756905,0.35694382841219896,0.36013980979763127,0.3600727060325907,0.35672416062990764,0.34983621335870174,0.33910085839591964,0.32409972645651364,0.3240149639937018,0.32396550122869294,0.30344591955015154,0.27720173953942795,0.2446171479123287,0.2446359033742006,0.24460489029799914,0.24363367582737194,0.20395929877056307,0.2399923137269906,0.27196057774344207,0.27288773586930526,0.27000975299249497,0.23726007680548222,0.24159150179112476,0.27399986018693545,0.30010663058554055,0.32069317377388573,0.32070777756956725,0.3206985247350451,0.336985812573125,0.34863525203210743,0.35613816076809457,0.3560715044456061,0.3560628265529293,0.35021655095389514,0.3558958028868132,0.35585774616576993,0.3504018635252627,0.35692261318374263,0.3573785270260107,0.3576090087983498,0.3601752901330706,0.3601747396207423,0.3601764656567026,0.3601835670181174,0.3601865385786356,0.36018488712990093,0.36018420499328074,0.3571978066970576,0.3572085631753885,0.3526854218738531,0.3527200770493544,0.35775061914063133,0.3586146721656023,0.3567645712563354,0.3596248097925221,0.35963211073585033,0.3596458728709979,0.36048462570582396,0.36048333458596904,0.3595636845044786,0.35953253133599583,0.3551920426034191,0.34713523208571706,0.3347604250769278,0.3177482300301787,0.3340784153224359,0.3167209819864354,0.31148303595840043,0.28727484674036674,0.2570266604273185,0.22000146823187305,0.23688304973645535,0.23686735334867037,0.19566893341816782,0.14659476642566827,0.08977228608502086,0.029934361156482847,0.06006121383204288,0,0.05912199037698773,0.12028946878141555,0.17026093296496525,0.214263975260222,0.21359655121206522,0.2501430172399288,0.25023276636731173,0.25137183594844503,0.2824159266551677,0.2825845296513785,0.2516985989427576,0.2137030988285753,0.22598075733691103,0.2260847283364218,0.22568725310500715,0.2219769641782915,0.20316596750790317,0.2030436386490796,0.1572938639790985,0.10183160955608028,0.04125220331942003,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],\"type\":\"bar\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"black\",\"dash\":\"dash\"},\"mode\":\"lines\",\"showlegend\":false,\"x\":[0,340],\"y\":[0.5,0.5],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[2,3],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[83,84],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[84,85],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[85,86],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[86,87],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[87,88],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[88,89],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[89,90],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[90,91],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[91,92],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[92,93],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[93,94],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[94,95],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[95,96],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[96,97],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[97,98],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[98,99],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[99,100],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[100,101],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[101,102],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[102,103],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[103,104],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[104,105],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[105,106],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[106,107],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[107,108],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[108,109],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[109,110],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[110,111],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[111,112],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[112,113],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[113,114],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[114,115],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[115,116],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[116,117],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[117,118],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[118,119],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[119,120],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[120,121],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[121,122],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[122,123],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[123,124],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[124,125],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[125,126],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[126,127],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[127,128],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[128,129],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[129,130],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[130,131],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[131,132],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[132,133],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[133,134],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[134,135],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[135,136],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[136,137],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[137,138],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[138,139],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[139,140],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[140,141],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[141,142],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[142,143],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[143,144],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[144,145],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[145,146],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[146,147],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[147,148],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[148,149],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[149,150],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[150,151],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[151,152],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[152,153],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[153,154],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[154,155],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[155,156],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[156,157],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[157,158],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[158,159],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[159,160],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[160,161],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[161,162],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[162,163],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[163,164],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[164,165],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[165,166],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[166,167],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[167,168],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[168,169],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[169,170],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[170,171],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[171,172],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[172,173],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[173,174],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[174,175],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[175,176],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[176,177],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[177,178],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[178,179],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[179,180],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[180,181],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[181,182],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[182,183],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[183,184],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[184,185],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[185,186],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[186,187],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[187,188],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[188,189],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[189,190],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[190,191],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[191,192],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[192,193],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[193,194],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[194,195],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[195,196],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[196,197],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[197,198],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[198,199],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[199,200],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[200,201],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[201,202],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[202,203],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[203,204],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[204,205],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[205,206],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[206,207],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[207,208],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[208,209],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[209,210],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[210,211],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[211,212],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[212,213],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[213,214],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[214,215],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[215,216],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[216,217],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[217,218],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[218,219],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[219,220],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[220,221],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[221,222],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[222,223],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[223,224],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[224,225],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[225,226],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[226,227],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[227,228],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[228,229],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[229,230],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[230,231],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[231,232],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[232,233],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[233,234],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[234,235],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[235,236],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[236,237],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[237,238],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[238,239],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[239,240],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[240,241],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[241,242],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[242,243],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[243,244],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[244,245],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[245,246],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[246,247],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[247,248],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[248,249],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[249,250],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[250,251],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[251,252],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[252,253],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[253,254],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[254,255],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[255,256],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[256,257],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[257,258],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[258,259],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[259,260],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[260,261],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[261,262],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[262,263],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[263,264],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[264,265],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[265,266],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[266,267],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[267,268],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[268,269],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[269,270],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[270,271],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[271,272],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[272,273],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[273,274],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[274,275],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[275,276],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[276,277],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[277,278],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[278,279],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[279,280],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[280,281],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[286,287],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[287,288],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[288,289],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[289,290],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[290,291],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[291,292],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[292,293],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[293,294],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[294,295],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[295,296],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[296,297],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[297,298],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[298,299],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[299,300],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[300,301],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[301,302],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[302,303],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[303,304],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[304,305],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"}],                        {\"template\":{\"data\":{\"candlestick\":[{\"decreasing\":{\"line\":{\"color\":\"#000033\"}},\"increasing\":{\"line\":{\"color\":\"#000032\"}},\"type\":\"candlestick\"}],\"contourcarpet\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"contourcarpet\"}],\"contour\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"contour\"}],\"heatmap\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"heatmap\"}],\"histogram2d\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"histogram2d\"}],\"icicle\":[{\"textfont\":{\"color\":\"white\"},\"type\":\"icicle\"}],\"sankey\":[{\"textfont\":{\"color\":\"#000036\"},\"type\":\"sankey\"}],\"scatter\":[{\"marker\":{\"line\":{\"width\":0}},\"type\":\"scatter\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#000038\"},\"font\":{\"color\":\"#000037\"},\"line\":{\"color\":\"#000039\"}},\"header\":{\"fill\":{\"color\":\"#000040\"},\"font\":{\"color\":\"#000036\"},\"line\":{\"color\":\"#000039\"}},\"type\":\"table\"}],\"waterfall\":[{\"connector\":{\"line\":{\"color\":\"#000036\",\"width\":2}},\"decreasing\":{\"marker\":{\"color\":\"#000033\"}},\"increasing\":{\"marker\":{\"color\":\"#000032\"}},\"totals\":{\"marker\":{\"color\":\"#000034\"}},\"type\":\"waterfall\"}]},\"layout\":{\"coloraxis\":{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]]},\"colorscale\":{\"diverging\":[[0.0,\"#000021\"],[0.1,\"#000022\"],[0.2,\"#000023\"],[0.3,\"#000024\"],[0.4,\"#000025\"],[0.5,\"#000026\"],[0.6,\"#000027\"],[0.7,\"#000028\"],[0.8,\"#000029\"],[0.9,\"#000030\"],[1.0,\"#000031\"]],\"sequential\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"sequentialminus\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]]},\"colorway\":[\"#000001\",\"#000002\",\"#000003\",\"#000004\",\"#000005\",\"#000006\",\"#000007\",\"#000008\",\"#000009\",\"#000010\"]}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0]},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0]},\"title\":{\"text\":\"LLPS Prediction Plot\"},\"height\":800,\"width\":1700},                        {\"responsive\": true}                    ).then(function(){\n",
              "                            \n",
              "var gd = document.getElementById('654db578-7a94-4a4d-9254-df474eac4dde');\n",
              "var x = new MutationObserver(function (mutations, observer) {{\n",
              "        var display = window.getComputedStyle(gd).display;\n",
              "        if (!display || display === 'none') {{\n",
              "            console.log([gd, 'removed!']);\n",
              "            Plotly.purge(gd);\n",
              "            observer.disconnect();\n",
              "        }}\n",
              "}});\n",
              "\n",
              "// Listen for the removal of the full notebook cells\n",
              "var notebookContainer = gd.closest('#notebook-container');\n",
              "if (notebookContainer) {{\n",
              "    x.observe(notebookContainer, {childList: true});\n",
              "}}\n",
              "\n",
              "// Listen for the clearing of the current output cell\n",
              "var outputEl = gd.closest('.output');\n",
              "if (outputEl) {{\n",
              "    x.observe(outputEl, {childList: true});\n",
              "}}\n",
              "\n",
              "                        })                };                            </script>        </div>\n",
              "</body>\n",
              "</html>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17207123026924806\"  style=\"position: relative; width: 1000px; height: 800px;\">\n        <p id=\"3dmolwarning_17207123026924806\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.2.0/3Dmol-min.js');\n}\n\nvar viewer_17207123026924806 = null;\nvar warn = document.getElementById(\"3dmolwarning_17207123026924806\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17207123026924806 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17207123026924806\"),{backgroundColor:\"white\"});\nviewer_17207123026924806.zoomTo();\n\tviewer_17207123026924806.addModel(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   MET A   1       0.319   0.252 -46.800  1.00  0.76           N  \\nATOM      2  CA  MET A   1      -0.203   0.998 -45.658  1.00  0.81           C  \\nATOM      3  C   MET A   1       0.586   0.679 -44.393  1.00  0.77           C  \\nATOM      4  CB  MET A   1      -1.685   0.686 -45.444  1.00  0.69           C  \\nATOM      5  O   MET A   1       0.552  -0.452 -43.903  1.00  0.67           O  \\nATOM      6  CG  MET A   1      -2.596   1.278 -46.507  1.00  0.62           C  \\nATOM      7  SD  MET A   1      -4.360   0.849 -46.238  1.00  0.68           S  \\nATOM      8  CE  MET A   1      -5.122   1.749 -47.618  1.00  0.63           C  \\nATOM      9  N   GLY A   2       1.736   1.505 -43.986  1.00  0.84           N  \\nATOM     10  CA  GLY A   2       2.492   1.362 -42.752  1.00  0.86           C  \\nATOM     11  C   GLY A   2       1.617   1.361 -41.513  1.00  0.84           C  \\nATOM     12  O   GLY A   2       0.842   2.294 -41.293  1.00  0.77           O  \\nATOM     13  N   ARG A   3       1.282   0.132 -40.795  1.00  0.84           N  \\nATOM     14  CA  ARG A   3       0.625   0.143 -39.492  1.00  0.85           C  \\nATOM     15  C   ARG A   3       1.236   1.203 -38.580  1.00  0.83           C  \\nATOM     16  CB  ARG A   3       0.719  -1.234 -38.831  1.00  0.81           C  \\nATOM     17  O   ARG A   3       2.411   1.115 -38.218  1.00  0.77           O  \\nATOM     18  CG  ARG A   3      -0.331  -2.222 -39.315  1.00  0.79           C  \\nATOM     19  CD  ARG A   3      -0.214  -3.562 -38.603  1.00  0.77           C  \\nATOM     20  NE  ARG A   3      -1.137  -4.549 -39.157  1.00  0.77           N  \\nATOM     21  NH1 ARG A   3      -0.479  -6.259 -37.751  1.00  0.62           N  \\nATOM     22  NH2 ARG A   3      -2.115  -6.621 -39.315  1.00  0.57           N  \\nATOM     23  CZ  ARG A   3      -1.241  -5.807 -38.740  1.00  0.76           C  \\nATOM     24  N   LYS A   4       0.539   2.338 -38.420  1.00  0.83           N  \\nATOM     25  CA  LYS A   4       0.941   3.279 -37.379  1.00  0.84           C  \\nATOM     26  C   LYS A   4       1.275   2.551 -36.080  1.00  0.82           C  \\nATOM     27  CB  LYS A   4      -0.160   4.311 -37.134  1.00  0.80           C  \\nATOM     28  O   LYS A   4       0.421   1.872 -35.505  1.00  0.78           O  \\nATOM     29  CG  LYS A   4      -0.304   5.340 -38.246  1.00  0.74           C  \\nATOM     30  CD  LYS A   4      -1.389   6.360 -37.927  1.00  0.72           C  \\nATOM     31  CE  LYS A   4      -1.542   7.383 -39.045  1.00  0.74           C  \\nATOM     32  NZ  LYS A   4      -2.650   8.345 -38.767  1.00  0.68           N  \\nATOM     33  N   LYS A   5       2.518   2.445 -35.703  1.00  0.79           N  \\nATOM     34  CA  LYS A   5       2.886   1.957 -34.377  1.00  0.80           C  \\nATOM     35  C   LYS A   5       2.107   2.687 -33.286  1.00  0.79           C  \\nATOM     36  CB  LYS A   5       4.389   2.118 -34.144  1.00  0.76           C  \\nATOM     37  O   LYS A   5       2.154   3.916 -33.199  1.00  0.75           O  \\nATOM     38  CG  LYS A   5       5.243   1.100 -34.886  1.00  0.73           C  \\nATOM     39  CD  LYS A   5       6.724   1.287 -34.584  1.00  0.72           C  \\nATOM     40  CE  LYS A   5       7.580   0.273 -35.332  1.00  0.72           C  \\nATOM     41  NZ  LYS A   5       9.035   0.461 -35.050  1.00  0.64           N  \\nATOM     42  N   LYS A   6       1.193   1.975 -32.649  1.00  0.77           N  \\nATOM     43  CA  LYS A   6       0.572   2.536 -31.453  1.00  0.78           C  \\nATOM     44  C   LYS A   6       1.622   3.104 -30.502  1.00  0.77           C  \\nATOM     45  CB  LYS A   6      -0.266   1.477 -30.736  1.00  0.74           C  \\nATOM     46  O   LYS A   6       2.547   2.397 -30.098  1.00  0.74           O  \\nATOM     47  CG  LYS A   6      -1.516   2.026 -30.064  1.00  0.69           C  \\nATOM     48  CD  LYS A   6      -2.347   0.915 -29.434  1.00  0.68           C  \\nATOM     49  CE  LYS A   6      -3.565   1.469 -28.709  1.00  0.63           C  \\nATOM     50  NZ  LYS A   6      -4.348   0.390 -28.036  1.00  0.57           N  \\nATOM     51  N   LYS A   7       1.645   4.412 -30.346  1.00  0.79           N  \\nATOM     52  CA  LYS A   7       2.417   5.011 -29.262  1.00  0.79           C  \\nATOM     53  C   LYS A   7       2.086   4.357 -27.924  1.00  0.79           C  \\nATOM     54  CB  LYS A   7       2.160   6.517 -29.188  1.00  0.76           C  \\nATOM     55  O   LYS A   7       0.944   4.418 -27.463  1.00  0.76           O  \\nATOM     56  CG  LYS A   7       3.275   7.303 -28.516  1.00  0.72           C  \\nATOM     57  CD  LYS A   7       3.005   8.802 -28.555  1.00  0.70           C  \\nATOM     58  CE  LYS A   7       4.065   9.582 -27.790  1.00  0.66           C  \\nATOM     59  NZ  LYS A   7       3.731  11.036 -27.707  1.00  0.60           N  \\nATOM     60  N   VAL A   8       2.941   3.471 -27.397  1.00  0.77           N  \\nATOM     61  CA  VAL A   8       2.815   2.987 -26.026  1.00  0.77           C  \\nATOM     62  C   VAL A   8       3.165   4.106 -25.049  1.00  0.77           C  \\nATOM     63  CB  VAL A   8       3.714   1.757 -25.773  1.00  0.74           C  \\nATOM     64  O   VAL A   8       4.207   4.753 -25.184  1.00  0.74           O  \\nATOM     65  CG1 VAL A   8       3.467   1.185 -24.379  1.00  0.69           C  \\nATOM     66  CG2 VAL A   8       3.474   0.692 -26.841  1.00  0.69           C  \\nATOM     67  N   ASP A   9       2.162   4.482 -24.260  1.00  0.79           N  \\nATOM     68  CA  ASP A   9       2.381   5.498 -23.235  1.00  0.79           C  \\nATOM     69  C   ASP A   9       3.495   5.081 -22.277  1.00  0.81           C  \\nATOM     70  CB  ASP A   9       1.091   5.764 -22.457  1.00  0.77           C  \\nATOM     71  O   ASP A   9       3.508   3.950 -21.788  1.00  0.79           O  \\nATOM     72  CG  ASP A   9      -0.034   6.284 -23.335  1.00  0.73           C  \\nATOM     73  OD1 ASP A   9       0.221   7.135 -24.214  1.00  0.72           O  \\nATOM     74  OD2 ASP A   9      -1.187   5.840 -23.145  1.00  0.74           O  \\nATOM     75  N   LYS A  10       4.565   5.912 -22.204  1.00  0.83           N  \\nATOM     76  CA  LYS A  10       5.703   5.678 -21.319  1.00  0.84           C  \\nATOM     77  C   LYS A  10       5.267   5.657 -19.857  1.00  0.84           C  \\nATOM     78  CB  LYS A  10       6.776   6.747 -21.531  1.00  0.82           C  \\nATOM     79  O   LYS A  10       4.481   6.503 -19.426  1.00  0.83           O  \\nATOM     80  CG  LYS A  10       7.525   6.619 -22.849  1.00  0.81           C  \\nATOM     81  CD  LYS A  10       8.658   7.632 -22.949  1.00  0.80           C  \\nATOM     82  CE  LYS A  10       9.465   7.446 -24.227  1.00  0.76           C  \\nATOM     83  NZ  LYS A  10      10.738   8.226 -24.197  1.00  0.73           N  \\nATOM     84  N   PRO A  11       5.711   4.655 -19.024  1.00  0.84           N  \\nATOM     85  CA  PRO A  11       5.436   4.737 -17.588  1.00  0.85           C  \\nATOM     86  C   PRO A  11       6.125   5.926 -16.922  1.00  0.85           C  \\nATOM     87  CB  PRO A  11       5.986   3.415 -17.047  1.00  0.84           C  \\nATOM     88  O   PRO A  11       7.226   6.311 -17.325  1.00  0.84           O  \\nATOM     89  CG  PRO A  11       7.106   3.061 -17.971  1.00  0.83           C  \\nATOM     90  CD  PRO A  11       6.765   3.556 -19.347  1.00  0.83           C  \\nATOM     91  N   TRP A  12       5.538   6.529 -15.970  1.00  0.84           N  \\nATOM     92  CA  TRP A  12       6.088   7.701 -15.296  1.00  0.85           C  \\nATOM     93  C   TRP A  12       6.010   7.543 -13.781  1.00  0.85           C  \\nATOM     94  CB  TRP A  12       5.346   8.968 -15.729  1.00  0.83           C  \\nATOM     95  O   TRP A  12       5.153   6.818 -13.269  1.00  0.84           O  \\nATOM     96  CG  TRP A  12       3.887   8.967 -15.387  1.00  0.79           C  \\nATOM     97  CD1 TRP A  12       2.869   8.425 -16.121  1.00  0.75           C  \\nATOM     98  CD2 TRP A  12       3.282   9.540 -14.224  1.00  0.77           C  \\nATOM     99  CE2 TRP A  12       1.892   9.307 -14.318  1.00  0.78           C  \\nATOM    100  CE3 TRP A  12       3.780  10.228 -13.109  1.00  0.80           C  \\nATOM    101  NE1 TRP A  12       1.666   8.626 -15.484  1.00  0.79           N  \\nATOM    102  CH2 TRP A  12       1.508  10.409 -12.259  1.00  0.77           C  \\nATOM    103  CZ2 TRP A  12       0.994   9.739 -13.338  1.00  0.78           C  \\nATOM    104  CZ3 TRP A  12       2.886  10.656 -12.135  1.00  0.78           C  \\nATOM    105  N   CYS A  13       6.861   8.172 -13.001  1.00  0.86           N  \\nATOM    106  CA  CYS A  13       6.957   8.134 -11.546  1.00  0.86           C  \\nATOM    107  C   CYS A  13       6.028   9.163 -10.912  1.00  0.86           C  \\nATOM    108  CB  CYS A  13       8.395   8.387 -11.097  1.00  0.85           C  \\nATOM    109  O   CYS A  13       6.276  10.367 -11.000  1.00  0.84           O  \\nATOM    110  SG  CYS A  13       8.618   8.337  -9.305  1.00  0.90           S  \\nATOM    111  N   TRP A  14       5.058   8.716 -10.126  1.00  0.85           N  \\nATOM    112  CA  TRP A  14       4.069   9.567  -9.473  1.00  0.86           C  \\nATOM    113  C   TRP A  14       4.737  10.524  -8.492  1.00  0.86           C  \\nATOM    114  CB  TRP A  14       3.024   8.717  -8.745  1.00  0.83           C  \\nATOM    115  O   TRP A  14       4.359  11.695  -8.401  1.00  0.83           O  \\nATOM    116  CG  TRP A  14       1.959   9.518  -8.058  1.00  0.75           C  \\nATOM    117  CD1 TRP A  14       0.813  10.014  -8.614  1.00  0.68           C  \\nATOM    118  CD2 TRP A  14       1.945   9.920  -6.685  1.00  0.70           C  \\nATOM    119  CE2 TRP A  14       0.759  10.657  -6.477  1.00  0.70           C  \\nATOM    120  CE3 TRP A  14       2.822   9.727  -5.608  1.00  0.75           C  \\nATOM    121  NE1 TRP A  14       0.086  10.700  -7.668  1.00  0.77           N  \\nATOM    122  CH2 TRP A  14       1.301  10.998  -4.198  1.00  0.71           C  \\nATOM    123  CZ2 TRP A  14       0.426  11.202  -5.234  1.00  0.74           C  \\nATOM    124  CZ3 TRP A  14       2.489  10.270  -4.373  1.00  0.72           C  \\nATOM    125  N   TYR A  15       5.804  10.168  -7.929  1.00  0.87           N  \\nATOM    126  CA  TYR A  15       6.447  10.922  -6.859  1.00  0.88           C  \\nATOM    127  C   TYR A  15       7.257  12.084  -7.421  1.00  0.88           C  \\nATOM    128  CB  TYR A  15       7.351  10.010  -6.025  1.00  0.87           C  \\nATOM    129  O   TYR A  15       7.540  13.053  -6.712  1.00  0.86           O  \\nATOM    130  CG  TYR A  15       6.611   8.894  -5.329  1.00  0.86           C  \\nATOM    131  CD1 TYR A  15       5.834   9.145  -4.200  1.00  0.84           C  \\nATOM    132  CD2 TYR A  15       6.687   7.587  -5.798  1.00  0.85           C  \\nATOM    133  CE1 TYR A  15       5.152   8.119  -3.554  1.00  0.87           C  \\nATOM    134  CE2 TYR A  15       6.009   6.553  -5.159  1.00  0.87           C  \\nATOM    135  OH  TYR A  15       4.572   5.810  -3.405  1.00  0.79           O  \\nATOM    136  CZ  TYR A  15       5.245   6.829  -4.040  1.00  0.87           C  \\nATOM    137  N   CYS A  16       7.756  12.110  -8.812  1.00  0.89           N  \\nATOM    138  CA  CYS A  16       8.626  13.181  -9.284  1.00  0.88           C  \\nATOM    139  C   CYS A  16       8.300  13.553 -10.726  1.00  0.87           C  \\nATOM    140  CB  CYS A  16      10.093  12.768  -9.174  1.00  0.87           C  \\nATOM    141  O   CYS A  16       8.977  14.391 -11.323  1.00  0.85           O  \\nATOM    142  SG  CYS A  16      10.552  11.410 -10.272  1.00  0.89           S  \\nATOM    143  N   ASN A  17       7.388  12.978 -11.403  1.00  0.84           N  \\nATOM    144  CA  ASN A  17       6.871  13.244 -12.741  1.00  0.84           C  \\nATOM    145  C   ASN A  17       7.875  12.847 -13.820  1.00  0.84           C  \\nATOM    146  CB  ASN A  17       6.492  14.720 -12.887  1.00  0.81           C  \\nATOM    147  O   ASN A  17       7.800  13.328 -14.952  1.00  0.82           O  \\nATOM    148  CG  ASN A  17       5.286  15.098 -12.049  1.00  0.76           C  \\nATOM    149  ND2 ASN A  17       5.157  16.383 -11.742  1.00  0.72           N  \\nATOM    150  OD1 ASN A  17       4.478  14.242 -11.679  1.00  0.72           O  \\nATOM    151  N   ARG A  18       8.819  12.093 -13.467  1.00  0.88           N  \\nATOM    152  CA  ARG A  18       9.807  11.689 -14.463  1.00  0.88           C  \\nATOM    153  C   ARG A  18       9.305  10.505 -15.283  1.00  0.88           C  \\nATOM    154  CB  ARG A  18      11.136  11.337 -13.791  1.00  0.87           C  \\nATOM    155  O   ARG A  18       8.780   9.537 -14.728  1.00  0.87           O  \\nATOM    156  CG  ARG A  18      12.259  11.035 -14.770  1.00  0.83           C  \\nATOM    157  CD  ARG A  18      13.610  10.959 -14.072  1.00  0.82           C  \\nATOM    158  NE  ARG A  18      14.671  10.561 -14.992  1.00  0.79           N  \\nATOM    159  NH1 ARG A  18      16.355  10.641 -13.412  1.00  0.72           N  \\nATOM    160  NH2 ARG A  18      16.834  10.053 -15.576  1.00  0.70           N  \\nATOM    161  CZ  ARG A  18      15.951  10.420 -14.658  1.00  0.78           C  \\nATOM    162  N   GLU A  19       9.453  10.497 -16.477  1.00  0.85           N  \\nATOM    163  CA  GLU A  19       9.134   9.437 -17.429  1.00  0.85           C  \\nATOM    164  C   GLU A  19      10.282   8.438 -17.547  1.00  0.86           C  \\nATOM    165  CB  GLU A  19       8.809  10.029 -18.803  1.00  0.83           C  \\nATOM    166  O   GLU A  19      11.452   8.813 -17.447  1.00  0.85           O  \\nATOM    167  CG  GLU A  19       7.511  10.823 -18.839  1.00  0.77           C  \\nATOM    168  CD  GLU A  19       7.140  11.303 -20.233  1.00  0.75           C  \\nATOM    169  OE1 GLU A  19       7.923  11.074 -21.182  1.00  0.74           O  \\nATOM    170  OE2 GLU A  19       6.057  11.915 -20.376  1.00  0.69           O  \\nATOM    171  N   PHE A  20       9.948   7.235 -17.910  1.00  0.87           N  \\nATOM    172  CA  PHE A  20      10.867   6.118 -18.095  1.00  0.88           C  \\nATOM    173  C   PHE A  20      10.487   5.303 -19.326  1.00  0.86           C  \\nATOM    174  CB  PHE A  20      10.879   5.220 -16.854  1.00  0.86           C  \\nATOM    175  O   PHE A  20       9.322   5.282 -19.728  1.00  0.84           O  \\nATOM    176  CG  PHE A  20      11.379   5.908 -15.612  1.00  0.86           C  \\nATOM    177  CD1 PHE A  20      12.725   5.865 -15.271  1.00  0.84           C  \\nATOM    178  CD2 PHE A  20      10.501   6.597 -14.786  1.00  0.85           C  \\nATOM    179  CE1 PHE A  20      13.190   6.501 -14.122  1.00  0.87           C  \\nATOM    180  CE2 PHE A  20      10.958   7.235 -13.636  1.00  0.87           C  \\nATOM    181  CZ  PHE A  20      12.303   7.185 -13.305  1.00  0.86           C  \\nATOM    182  N   ASP A  21      11.462   4.695 -19.909  1.00  0.86           N  \\nATOM    183  CA  ASP A  21      11.266   3.959 -21.154  1.00  0.86           C  \\nATOM    184  C   ASP A  21      10.561   2.628 -20.900  1.00  0.85           C  \\nATOM    185  CB  ASP A  21      12.607   3.719 -21.852  1.00  0.84           C  \\nATOM    186  O   ASP A  21       9.805   2.149 -21.747  1.00  0.83           O  \\nATOM    187  CG  ASP A  21      13.280   5.004 -22.301  1.00  0.80           C  \\nATOM    188  OD1 ASP A  21      12.599   5.881 -22.876  1.00  0.79           O  \\nATOM    189  OD2 ASP A  21      14.503   5.140 -22.081  1.00  0.80           O  \\nATOM    190  N   ASP A  22      10.833   2.138 -19.629  1.00  0.85           N  \\nATOM    191  CA  ASP A  22      10.293   0.841 -19.233  1.00  0.85           C  \\nATOM    192  C   ASP A  22       9.946   0.821 -17.747  1.00  0.86           C  \\nATOM    193  CB  ASP A  22      11.288  -0.276 -19.556  1.00  0.83           C  \\nATOM    194  O   ASP A  22      10.516   1.580 -16.960  1.00  0.85           O  \\nATOM    195  CG  ASP A  22      10.863  -1.626 -19.007  1.00  0.76           C  \\nATOM    196  OD1 ASP A  22       9.650  -1.930 -19.006  1.00  0.72           O  \\nATOM    197  OD2 ASP A  22      11.748  -2.392 -18.567  1.00  0.76           O  \\nATOM    198  N   GLU A  23       8.950  -0.051 -17.386  1.00  0.84           N  \\nATOM    199  CA  GLU A  23       8.467  -0.214 -16.018  1.00  0.85           C  \\nATOM    200  C   GLU A  23       9.583  -0.688 -15.092  1.00  0.86           C  \\nATOM    201  CB  GLU A  23       7.295  -1.197 -15.976  1.00  0.82           C  \\nATOM    202  O   GLU A  23       9.635  -0.298 -13.924  1.00  0.85           O  \\nATOM    203  CG  GLU A  23       6.595  -1.261 -14.626  1.00  0.74           C  \\nATOM    204  CD  GLU A  23       5.387  -2.186 -14.619  1.00  0.72           C  \\nATOM    205  OE1 GLU A  23       5.098  -2.813 -15.663  1.00  0.70           O  \\nATOM    206  OE2 GLU A  23       4.725  -2.282 -13.562  1.00  0.66           O  \\nATOM    207  N   LYS A  24      10.556  -1.397 -15.609  1.00  0.85           N  \\nATOM    208  CA  LYS A  24      11.661  -1.898 -14.797  1.00  0.86           C  \\nATOM    209  C   LYS A  24      12.576  -0.761 -14.352  1.00  0.86           C  \\nATOM    210  CB  LYS A  24      12.464  -2.945 -15.571  1.00  0.83           C  \\nATOM    211  O   LYS A  24      12.987  -0.706 -13.191  1.00  0.86           O  \\nATOM    212  CG  LYS A  24      13.526  -3.649 -14.740  1.00  0.77           C  \\nATOM    213  CD  LYS A  24      14.267  -4.702 -15.554  1.00  0.75           C  \\nATOM    214  CE  LYS A  24      15.380  -5.354 -14.746  1.00  0.70           C  \\nATOM    215  NZ  LYS A  24      16.154  -6.337 -15.562  1.00  0.66           N  \\nATOM    216  N   ILE A  25      12.823   0.139 -15.257  1.00  0.89           N  \\nATOM    217  CA  ILE A  25      13.645   1.309 -14.969  1.00  0.89           C  \\nATOM    218  C   ILE A  25      12.906   2.233 -14.003  1.00  0.89           C  \\nATOM    219  CB  ILE A  25      14.017   2.071 -16.260  1.00  0.88           C  \\nATOM    220  O   ILE A  25      13.497   2.748 -13.051  1.00  0.88           O  \\nATOM    221  CG1 ILE A  25      14.780   1.152 -17.221  1.00  0.85           C  \\nATOM    222  CG2 ILE A  25      14.836   3.323 -15.933  1.00  0.83           C  \\nATOM    223  CD1 ILE A  25      15.087   1.786 -18.571  1.00  0.81           C  \\nATOM    224  N   LEU A  26      11.556   2.318 -14.162  1.00  0.88           N  \\nATOM    225  CA  LEU A  26      10.726   3.091 -13.245  1.00  0.88           C  \\nATOM    226  C   LEU A  26      10.789   2.512 -11.836  1.00  0.88           C  \\nATOM    227  CB  LEU A  26       9.275   3.124 -13.732  1.00  0.87           C  \\nATOM    228  O   LEU A  26      11.047   3.237 -10.872  1.00  0.87           O  \\nATOM    229  CG  LEU A  26       8.246   3.697 -12.756  1.00  0.83           C  \\nATOM    230  CD1 LEU A  26       8.573   5.152 -12.435  1.00  0.80           C  \\nATOM    231  CD2 LEU A  26       6.839   3.574 -13.331  1.00  0.80           C  \\nATOM    232  N   ILE A  27      10.767   1.171 -11.612  1.00  0.87           N  \\nATOM    233  CA  ILE A  27      10.816   0.491 -10.322  1.00  0.88           C  \\nATOM    234  C   ILE A  27      12.197   0.672  -9.696  1.00  0.88           C  \\nATOM    235  CB  ILE A  27      10.482  -1.011 -10.461  1.00  0.87           C  \\nATOM    236  O   ILE A  27      12.311   0.983  -8.508  1.00  0.88           O  \\nATOM    237  CG1 ILE A  27       9.002  -1.199 -10.812  1.00  0.85           C  \\nATOM    238  CG2 ILE A  27      10.841  -1.765  -9.177  1.00  0.84           C  \\nATOM    239  CD1 ILE A  27       8.654  -2.600 -11.295  1.00  0.82           C  \\nATOM    240  N   GLN A  28      13.266   0.602 -10.514  1.00  0.89           N  \\nATOM    241  CA  GLN A  28      14.624   0.816 -10.024  1.00  0.90           C  \\nATOM    242  C   GLN A  28      14.829   2.263  -9.586  1.00  0.90           C  \\nATOM    243  CB  GLN A  28      15.648   0.442 -11.097  1.00  0.87           C  \\nATOM    244  O   GLN A  28      15.460   2.523  -8.559  1.00  0.89           O  \\nATOM    245  CG  GLN A  28      15.725  -1.052 -11.380  1.00  0.78           C  \\nATOM    246  CD  GLN A  28      16.116  -1.862 -10.158  1.00  0.76           C  \\nATOM    247  NE2 GLN A  28      15.686  -3.118 -10.116  1.00  0.72           N  \\nATOM    248  OE1 GLN A  28      16.800  -1.362  -9.259  1.00  0.77           O  \\nATOM    249  N   HIS A  29      14.226   3.222 -10.323  1.00  0.91           N  \\nATOM    250  CA  HIS A  29      14.222   4.634  -9.957  1.00  0.92           C  \\nATOM    251  C   HIS A  29      13.508   4.857  -8.629  1.00  0.92           C  \\nATOM    252  CB  HIS A  29      13.562   5.469 -11.056  1.00  0.91           C  \\nATOM    253  O   HIS A  29      14.045   5.514  -7.733  1.00  0.91           O  \\nATOM    254  CG  HIS A  29      13.248   6.870 -10.636  1.00  0.89           C  \\nATOM    255  CD2 HIS A  29      12.101   7.426 -10.180  1.00  0.87           C  \\nATOM    256  ND1 HIS A  29      14.183   7.883 -10.666  1.00  0.85           N  \\nATOM    257  CE1 HIS A  29      13.622   9.004 -10.244  1.00  0.87           C  \\nATOM    258  NE2 HIS A  29      12.359   8.754  -9.943  1.00  0.89           N  \\nATOM    259  N   GLN A  30      12.388   4.282  -8.458  1.00  0.91           N  \\nATOM    260  CA  GLN A  30      11.617   4.445  -7.229  1.00  0.91           C  \\nATOM    261  C   GLN A  30      12.377   3.892  -6.026  1.00  0.91           C  \\nATOM    262  CB  GLN A  30      10.257   3.757  -7.352  1.00  0.90           C  \\nATOM    263  O   GLN A  30      12.549   4.586  -5.022  1.00  0.90           O  \\nATOM    264  CG  GLN A  30       9.275   4.493  -8.253  1.00  0.87           C  \\nATOM    265  CD  GLN A  30       7.997   3.711  -8.491  1.00  0.84           C  \\nATOM    266  NE2 GLN A  30       6.891   4.424  -8.677  1.00  0.85           N  \\nATOM    267  OE1 GLN A  30       8.003   2.476  -8.509  1.00  0.86           O  \\nATOM    268  N   LYS A  31      13.043   2.725  -6.115  1.00  0.91           N  \\nATOM    269  CA  LYS A  31      13.869   2.119  -5.074  1.00  0.92           C  \\nATOM    270  C   LYS A  31      15.067   3.004  -4.739  1.00  0.91           C  \\nATOM    271  CB  LYS A  31      14.347   0.733  -5.507  1.00  0.90           C  \\nATOM    272  O   LYS A  31      15.431   3.147  -3.570  1.00  0.90           O  \\nATOM    273  CG  LYS A  31      13.247  -0.318  -5.544  1.00  0.85           C  \\nATOM    274  CD  LYS A  31      13.763  -1.649  -6.076  1.00  0.83           C  \\nATOM    275  CE  LYS A  31      12.677  -2.717  -6.060  1.00  0.84           C  \\nATOM    276  NZ  LYS A  31      13.187  -4.032  -6.550  1.00  0.81           N  \\nATOM    277  N   ALA A  32      15.678   3.589  -5.749  1.00  0.94           N  \\nATOM    278  CA  ALA A  32      16.936   4.311  -5.577  1.00  0.94           C  \\nATOM    279  C   ALA A  32      16.689   5.728  -5.067  1.00  0.93           C  \\nATOM    280  CB  ALA A  32      17.712   4.349  -6.891  1.00  0.92           C  \\nATOM    281  O   ALA A  32      17.504   6.277  -4.322  1.00  0.92           O  \\nATOM    282  N   LYS A  33      15.534   6.306  -5.475  1.00  0.94           N  \\nATOM    283  CA  LYS A  33      15.316   7.721  -5.191  1.00  0.94           C  \\nATOM    284  C   LYS A  33      14.292   7.907  -4.075  1.00  0.94           C  \\nATOM    285  CB  LYS A  33      14.858   8.457  -6.451  1.00  0.92           C  \\nATOM    286  O   LYS A  33      14.494   8.717  -3.168  1.00  0.92           O  \\nATOM    287  CG  LYS A  33      14.936   9.973  -6.345  1.00  0.80           C  \\nATOM    288  CD  LYS A  33      14.257  10.652  -7.528  1.00  0.75           C  \\nATOM    289  CE  LYS A  33      14.229  12.165  -7.364  1.00  0.72           C  \\nATOM    290  NZ  LYS A  33      13.345  12.815  -8.378  1.00  0.66           N  \\nATOM    291  N   HIS A  34      13.175   7.221  -4.112  1.00  0.93           N  \\nATOM    292  CA  HIS A  34      12.029   7.478  -3.246  1.00  0.93           C  \\nATOM    293  C   HIS A  34      12.010   6.519  -2.060  1.00  0.93           C  \\nATOM    294  CB  HIS A  34      10.724   7.362  -4.036  1.00  0.93           C  \\nATOM    295  O   HIS A  34      11.587   6.891  -0.963  1.00  0.91           O  \\nATOM    296  CG  HIS A  34      10.606   8.357  -5.146  1.00  0.92           C  \\nATOM    297  CD2 HIS A  34      10.553   8.189  -6.489  1.00  0.89           C  \\nATOM    298  ND1 HIS A  34      10.530   9.715  -4.924  1.00  0.89           N  \\nATOM    299  CE1 HIS A  34      10.434  10.340  -6.085  1.00  0.90           C  \\nATOM    300  NE2 HIS A  34      10.446   9.437  -7.051  1.00  0.91           N  \\nATOM    301  N   PHE A  35      12.495   5.310  -2.224  1.00  0.93           N  \\nATOM    302  CA  PHE A  35      12.411   4.317  -1.160  1.00  0.94           C  \\nATOM    303  C   PHE A  35      13.801   3.943  -0.657  1.00  0.93           C  \\nATOM    304  CB  PHE A  35      11.676   3.065  -1.648  1.00  0.92           C  \\nATOM    305  O   PHE A  35      14.049   2.789  -0.304  1.00  0.90           O  \\nATOM    306  CG  PHE A  35      10.323   3.349  -2.242  1.00  0.91           C  \\nATOM    307  CD1 PHE A  35       9.505   4.336  -1.705  1.00  0.87           C  \\nATOM    308  CD2 PHE A  35       9.868   2.628  -3.339  1.00  0.88           C  \\nATOM    309  CE1 PHE A  35       8.253   4.601  -2.254  1.00  0.88           C  \\nATOM    310  CE2 PHE A  35       8.617   2.887  -3.892  1.00  0.89           C  \\nATOM    311  CZ  PHE A  35       7.811   3.874  -3.347  1.00  0.88           C  \\nATOM    312  N   LYS A  36      14.722   4.682  -0.585  1.00  0.94           N  \\nATOM    313  CA  LYS A  36      16.054   4.473  -0.023  1.00  0.95           C  \\nATOM    314  C   LYS A  36      16.141   5.019   1.400  1.00  0.94           C  \\nATOM    315  CB  LYS A  36      17.118   5.131  -0.902  1.00  0.94           C  \\nATOM    316  O   LYS A  36      15.890   6.204   1.632  1.00  0.93           O  \\nATOM    317  CG  LYS A  36      18.546   4.778  -0.515  1.00  0.91           C  \\nATOM    318  CD  LYS A  36      19.558   5.496  -1.398  1.00  0.90           C  \\nATOM    319  CE  LYS A  36      20.988   5.185  -0.978  1.00  0.92           C  \\nATOM    320  NZ  LYS A  36      21.982   5.978  -1.762  1.00  0.91           N  \\nATOM    321  N   CYS A  37      16.587   4.227   2.311  1.00  0.93           N  \\nATOM    322  CA  CYS A  37      16.860   4.709   3.660  1.00  0.95           C  \\nATOM    323  C   CYS A  37      17.964   5.760   3.651  1.00  0.93           C  \\nATOM    324  CB  CYS A  37      17.256   3.550   4.575  1.00  0.91           C  \\nATOM    325  O   CYS A  37      19.032   5.540   3.077  1.00  0.90           O  \\nATOM    326  SG  CYS A  37      17.625   4.053   6.270  1.00  0.85           S  \\nATOM    327  N   HIS A  38      17.814   6.846   4.301  1.00  0.94           N  \\nATOM    328  CA  HIS A  38      18.803   7.918   4.327  1.00  0.94           C  \\nATOM    329  C   HIS A  38      19.985   7.556   5.220  1.00  0.93           C  \\nATOM    330  CB  HIS A  38      18.165   9.224   4.804  1.00  0.91           C  \\nATOM    331  O   HIS A  38      21.101   8.031   5.002  1.00  0.91           O  \\nATOM    332  CG  HIS A  38      17.623   9.154   6.196  1.00  0.83           C  \\nATOM    333  CD2 HIS A  38      18.150   9.553   7.377  1.00  0.78           C  \\nATOM    334  ND1 HIS A  38      16.388   8.614   6.487  1.00  0.76           N  \\nATOM    335  CE1 HIS A  38      16.180   8.686   7.791  1.00  0.79           C  \\nATOM    336  NE2 HIS A  38      17.233   9.252   8.354  1.00  0.83           N  \\nATOM    337  N   ILE A  39      19.811   6.557   6.161  1.00  0.94           N  \\nATOM    338  CA  ILE A  39      20.836   6.215   7.142  1.00  0.95           C  \\nATOM    339  C   ILE A  39      21.739   5.118   6.584  1.00  0.94           C  \\nATOM    340  CB  ILE A  39      20.209   5.764   8.480  1.00  0.94           C  \\nATOM    341  O   ILE A  39      22.954   5.303   6.472  1.00  0.92           O  \\nATOM    342  CG1 ILE A  39      19.347   6.885   9.071  1.00  0.89           C  \\nATOM    343  CG2 ILE A  39      21.296   5.331   9.467  1.00  0.87           C  \\nATOM    344  CD1 ILE A  39      18.534   6.465  10.289  1.00  0.87           C  \\nATOM    345  N   CYS A  40      21.225   3.998   6.101  1.00  0.94           N  \\nATOM    346  CA  CYS A  40      22.032   2.847   5.713  1.00  0.94           C  \\nATOM    347  C   CYS A  40      22.002   2.644   4.203  1.00  0.93           C  \\nATOM    348  CB  CYS A  40      21.536   1.583   6.416  1.00  0.92           C  \\nATOM    349  O   CYS A  40      22.541   1.659   3.694  1.00  0.91           O  \\nATOM    350  SG  CYS A  40      19.902   1.040   5.870  1.00  0.95           S  \\nATOM    351  N   HIS A  41      21.300   3.383   3.500  1.00  0.92           N  \\nATOM    352  CA  HIS A  41      21.214   3.450   2.045  1.00  0.92           C  \\nATOM    353  C   HIS A  41      20.568   2.193   1.473  1.00  0.90           C  \\nATOM    354  CB  HIS A  41      22.603   3.650   1.435  1.00  0.89           C  \\nATOM    355  O   HIS A  41      20.666   1.931   0.272  1.00  0.87           O  \\nATOM    356  CG  HIS A  41      23.265   4.923   1.857  1.00  0.81           C  \\nATOM    357  CD2 HIS A  41      24.401   5.144   2.561  1.00  0.77           C  \\nATOM    358  ND1 HIS A  41      22.751   6.165   1.555  1.00  0.78           N  \\nATOM    359  CE1 HIS A  41      23.544   7.098   2.056  1.00  0.77           C  \\nATOM    360  NE2 HIS A  41      24.553   6.504   2.671  1.00  0.83           N  \\nATOM    361  N   LYS A  42      19.979   1.410   2.350  1.00  0.95           N  \\nATOM    362  CA  LYS A  42      19.206   0.252   1.912  1.00  0.95           C  \\nATOM    363  C   LYS A  42      18.030   0.675   1.036  1.00  0.95           C  \\nATOM    364  CB  LYS A  42      18.703  -0.544   3.117  1.00  0.95           C  \\nATOM    365  O   LYS A  42      17.258   1.560   1.409  1.00  0.94           O  \\nATOM    366  CG  LYS A  42      18.043  -1.867   2.753  1.00  0.93           C  \\nATOM    367  CD  LYS A  42      17.771  -2.714   3.989  1.00  0.90           C  \\nATOM    368  CE  LYS A  42      17.131  -4.047   3.625  1.00  0.91           C  \\nATOM    369  NZ  LYS A  42      16.930  -4.912   4.825  1.00  0.90           N  \\nATOM    370  N   LYS A  43      17.830   0.088  -0.122  1.00  0.92           N  \\nATOM    371  CA  LYS A  43      16.725   0.281  -1.057  1.00  0.93           C  \\nATOM    372  C   LYS A  43      15.561  -0.651  -0.733  1.00  0.91           C  \\nATOM    373  CB  LYS A  43      17.192   0.055  -2.496  1.00  0.90           C  \\nATOM    374  O   LYS A  43      15.749  -1.862  -0.596  1.00  0.87           O  \\nATOM    375  CG  LYS A  43      18.314   0.984  -2.935  1.00  0.85           C  \\nATOM    376  CD  LYS A  43      18.772   0.675  -4.355  1.00  0.83           C  \\nATOM    377  CE  LYS A  43      19.955   1.542  -4.763  1.00  0.87           C  \\nATOM    378  NZ  LYS A  43      20.473   1.172  -6.114  1.00  0.85           N  \\nATOM    379  N   LEU A  44      14.382  -0.154  -0.716  1.00  0.92           N  \\nATOM    380  CA  LEU A  44      13.137  -0.848  -0.406  1.00  0.92           C  \\nATOM    381  C   LEU A  44      12.152  -0.742  -1.565  1.00  0.90           C  \\nATOM    382  CB  LEU A  44      12.509  -0.278   0.868  1.00  0.91           C  \\nATOM    383  O   LEU A  44      12.447  -0.108  -2.580  1.00  0.87           O  \\nATOM    384  CG  LEU A  44      13.305  -0.468   2.161  1.00  0.88           C  \\nATOM    385  CD1 LEU A  44      13.100   0.725   3.089  1.00  0.84           C  \\nATOM    386  CD2 LEU A  44      12.901  -1.766   2.852  1.00  0.84           C  \\nATOM    387  N   PHE A  45      11.036  -1.459  -1.455  1.00  0.88           N  \\nATOM    388  CA  PHE A  45      10.129  -1.593  -2.588  1.00  0.88           C  \\nATOM    389  C   PHE A  45       8.965  -0.616  -2.470  1.00  0.86           C  \\nATOM    390  CB  PHE A  45       9.601  -3.028  -2.688  1.00  0.85           C  \\nATOM    391  O   PHE A  45       8.282  -0.334  -3.456  1.00  0.83           O  \\nATOM    392  CG  PHE A  45      10.605  -4.008  -3.233  1.00  0.79           C  \\nATOM    393  CD1 PHE A  45      10.778  -4.158  -4.604  1.00  0.75           C  \\nATOM    394  CD2 PHE A  45      11.378  -4.779  -2.374  1.00  0.76           C  \\nATOM    395  CE1 PHE A  45      11.706  -5.063  -5.111  1.00  0.75           C  \\nATOM    396  CE2 PHE A  45      12.308  -5.686  -2.874  1.00  0.75           C  \\nATOM    397  CZ  PHE A  45      12.470  -5.827  -4.243  1.00  0.74           C  \\nATOM    398  N   SER A  46       8.687  -0.033  -1.246  1.00  0.88           N  \\nATOM    399  CA  SER A  46       7.503   0.786  -1.006  1.00  0.90           C  \\nATOM    400  C   SER A  46       7.757   1.816   0.090  1.00  0.90           C  \\nATOM    401  CB  SER A  46       6.311  -0.093  -0.625  1.00  0.87           C  \\nATOM    402  O   SER A  46       8.721   1.698   0.849  1.00  0.89           O  \\nATOM    403  OG  SER A  46       6.519  -0.702   0.638  1.00  0.79           O  \\nATOM    404  N   GLY A  47       6.883   2.890   0.174  1.00  0.90           N  \\nATOM    405  CA  GLY A  47       6.883   3.855   1.261  1.00  0.91           C  \\nATOM    406  C   GLY A  47       6.714   3.217   2.627  1.00  0.91           C  \\nATOM    407  O   GLY A  47       7.569   3.373   3.501  1.00  0.90           O  \\nATOM    408  N   PRO A  48       5.660   2.448   2.827  1.00  0.90           N  \\nATOM    409  CA  PRO A  48       5.489   1.768   4.113  1.00  0.91           C  \\nATOM    410  C   PRO A  48       6.692   0.906   4.489  1.00  0.91           C  \\nATOM    411  CB  PRO A  48       4.246   0.903   3.891  1.00  0.89           C  \\nATOM    412  O   PRO A  48       7.127   0.916   5.643  1.00  0.90           O  \\nATOM    413  CG  PRO A  48       3.436   1.649   2.880  1.00  0.86           C  \\nATOM    414  CD  PRO A  48       4.374   2.294   1.900  1.00  0.85           C  \\nATOM    415  N   GLY A  49       7.266   0.158   3.553  1.00  0.91           N  \\nATOM    416  CA  GLY A  49       8.467  -0.627   3.792  1.00  0.92           C  \\nATOM    417  C   GLY A  49       9.638   0.205   4.277  1.00  0.92           C  \\nATOM    418  O   GLY A  49      10.395  -0.227   5.149  1.00  0.91           O  \\nATOM    419  N   LEU A  50       9.841   1.390   3.745  1.00  0.94           N  \\nATOM    420  CA  LEU A  50      10.850   2.349   4.181  1.00  0.94           C  \\nATOM    421  C   LEU A  50      10.561   2.835   5.597  1.00  0.94           C  \\nATOM    422  CB  LEU A  50      10.906   3.541   3.221  1.00  0.94           C  \\nATOM    423  O   LEU A  50      11.458   2.865   6.442  1.00  0.93           O  \\nATOM    424  CG  LEU A  50      11.865   4.671   3.599  1.00  0.92           C  \\nATOM    425  CD1 LEU A  50      13.293   4.145   3.693  1.00  0.89           C  \\nATOM    426  CD2 LEU A  50      11.775   5.809   2.588  1.00  0.89           C  \\nATOM    427  N   SER A  51       9.215   3.157   5.937  1.00  0.94           N  \\nATOM    428  CA  SER A  51       8.818   3.583   7.275  1.00  0.94           C  \\nATOM    429  C   SER A  51       9.110   2.501   8.309  1.00  0.94           C  \\nATOM    430  CB  SER A  51       7.331   3.940   7.304  1.00  0.93           C  \\nATOM    431  O   SER A  51       9.841   2.737   9.273  1.00  0.92           O  \\nATOM    432  OG  SER A  51       6.942   4.369   8.598  1.00  0.86           O  \\nATOM    433  N   ILE A  52       8.724   1.270   8.110  1.00  0.93           N  \\nATOM    434  CA  ILE A  52       8.922   0.135   9.006  1.00  0.94           C  \\nATOM    435  C   ILE A  52      10.416  -0.119   9.193  1.00  0.94           C  \\nATOM    436  CB  ILE A  52       8.225  -1.136   8.471  1.00  0.93           C  \\nATOM    437  O   ILE A  52      10.882  -0.319  10.317  1.00  0.93           O  \\nATOM    438  CG1 ILE A  52       6.702  -0.981   8.547  1.00  0.88           C  \\nATOM    439  CG2 ILE A  52       8.689  -2.372   9.246  1.00  0.87           C  \\nATOM    440  CD1 ILE A  52       5.932  -2.037   7.765  1.00  0.84           C  \\nATOM    441  N   HIS A  53      11.186  -0.112   8.080  1.00  0.93           N  \\nATOM    442  CA  HIS A  53      12.631  -0.296   8.154  1.00  0.94           C  \\nATOM    443  C   HIS A  53      13.279   0.777   9.022  1.00  0.94           C  \\nATOM    444  CB  HIS A  53      13.246  -0.280   6.753  1.00  0.94           C  \\nATOM    445  O   HIS A  53      14.101   0.469   9.888  1.00  0.93           O  \\nATOM    446  CG  HIS A  53      14.741  -0.238   6.754  1.00  0.91           C  \\nATOM    447  CD2 HIS A  53      15.602   0.790   6.566  1.00  0.87           C  \\nATOM    448  ND1 HIS A  53      15.516  -1.358   6.965  1.00  0.87           N  \\nATOM    449  CE1 HIS A  53      16.793  -1.019   6.908  1.00  0.89           C  \\nATOM    450  NE2 HIS A  53      16.872   0.279   6.667  1.00  0.92           N  \\nATOM    451  N   CYS A  54      12.941   2.031   8.799  1.00  0.95           N  \\nATOM    452  CA  CYS A  54      13.525   3.104   9.596  1.00  0.95           C  \\nATOM    453  C   CYS A  54      13.175   2.940  11.071  1.00  0.95           C  \\nATOM    454  CB  CYS A  54      13.042   4.466   9.098  1.00  0.94           C  \\nATOM    455  O   CYS A  54      14.058   2.974  11.930  1.00  0.94           O  \\nATOM    456  SG  CYS A  54      13.768   4.962   7.520  1.00  0.94           S  \\nATOM    457  N   MET A  55      11.926   2.643  11.381  1.00  0.94           N  \\nATOM    458  CA  MET A  55      11.462   2.477  12.755  1.00  0.94           C  \\nATOM    459  C   MET A  55      12.147   1.288  13.422  1.00  0.93           C  \\nATOM    460  CB  MET A  55       9.944   2.294  12.793  1.00  0.91           C  \\nATOM    461  O   MET A  55      12.598   1.387  14.564  1.00  0.91           O  \\nATOM    462  CG  MET A  55       9.365   2.253  14.198  1.00  0.79           C  \\nATOM    463  SD  MET A  55       7.560   1.925  14.206  1.00  0.77           S  \\nATOM    464  CE  MET A  55       7.000   3.188  13.029  1.00  0.69           C  \\nATOM    465  N   GLN A  56      12.262   0.235  12.760  1.00  0.94           N  \\nATOM    466  CA  GLN A  56      12.769  -0.994  13.362  1.00  0.94           C  \\nATOM    467  C   GLN A  56      14.290  -0.961  13.479  1.00  0.94           C  \\nATOM    468  CB  GLN A  56      12.332  -2.212  12.547  1.00  0.93           C  \\nATOM    469  O   GLN A  56      14.850  -1.405  14.484  1.00  0.93           O  \\nATOM    470  CG  GLN A  56      10.853  -2.548  12.689  1.00  0.87           C  \\nATOM    471  CD  GLN A  56      10.456  -3.788  11.911  1.00  0.82           C  \\nATOM    472  NE2 GLN A  56       9.250  -4.285  12.163  1.00  0.85           N  \\nATOM    473  OE1 GLN A  56      11.226  -4.295  11.089  1.00  0.87           O  \\nATOM    474  N   VAL A  57      15.046  -0.451  12.453  1.00  0.94           N  \\nATOM    475  CA  VAL A  57      16.495  -0.581  12.350  1.00  0.95           C  \\nATOM    476  C   VAL A  57      17.168   0.669  12.914  1.00  0.94           C  \\nATOM    477  CB  VAL A  57      16.942  -0.815  10.890  1.00  0.93           C  \\nATOM    478  O   VAL A  57      18.188   0.576  13.601  1.00  0.93           O  \\nATOM    479  CG1 VAL A  57      18.459  -0.982  10.809  1.00  0.81           C  \\nATOM    480  CG2 VAL A  57      16.235  -2.036  10.304  1.00  0.78           C  \\nATOM    481  N   HIS A  58      16.691   1.827  12.671  1.00  0.94           N  \\nATOM    482  CA  HIS A  58      17.377   3.073  12.997  1.00  0.94           C  \\nATOM    483  C   HIS A  58      16.690   3.794  14.152  1.00  0.92           C  \\nATOM    484  CB  HIS A  58      17.440   3.986  11.771  1.00  0.93           C  \\nATOM    485  O   HIS A  58      17.215   4.781  14.672  1.00  0.89           O  \\nATOM    486  CG  HIS A  58      18.196   3.398  10.623  1.00  0.89           C  \\nATOM    487  CD2 HIS A  58      17.804   3.096   9.362  1.00  0.84           C  \\nATOM    488  ND1 HIS A  58      19.526   3.047  10.709  1.00  0.82           N  \\nATOM    489  CE1 HIS A  58      19.921   2.555   9.546  1.00  0.86           C  \\nATOM    490  NE2 HIS A  58      18.895   2.574   8.712  1.00  0.89           N  \\nATOM    491  N   LYS A  59      15.489   3.377  14.624  1.00  0.90           N  \\nATOM    492  CA  LYS A  59      14.624   4.028  15.604  1.00  0.90           C  \\nATOM    493  C   LYS A  59      14.246   5.436  15.153  1.00  0.87           C  \\nATOM    494  CB  LYS A  59      15.306   4.081  16.971  1.00  0.84           C  \\nATOM    495  O   LYS A  59      14.058   6.329  15.982  1.00  0.83           O  \\nATOM    496  CG  LYS A  59      15.793   2.729  17.473  1.00  0.75           C  \\nATOM    497  CD  LYS A  59      14.673   1.697  17.475  1.00  0.73           C  \\nATOM    498  CE  LYS A  59      15.143   0.361  18.033  1.00  0.74           C  \\nATOM    499  NZ  LYS A  59      14.147  -0.724  17.786  1.00  0.73           N  \\nATOM    500  N   GLU A  60      14.086   5.631  13.992  1.00  0.94           N  \\nATOM    501  CA  GLU A  60      13.657   6.902  13.417  1.00  0.94           C  \\nATOM    502  C   GLU A  60      12.349   6.746  12.647  1.00  0.93           C  \\nATOM    503  CB  GLU A  60      14.743   7.470  12.499  1.00  0.92           C  \\nATOM    504  O   GLU A  60      12.153   5.756  11.940  1.00  0.91           O  \\nATOM    505  CG  GLU A  60      15.927   8.069  13.245  1.00  0.80           C  \\nATOM    506  CD  GLU A  60      16.907   8.789  12.333  1.00  0.75           C  \\nATOM    507  OE1 GLU A  60      16.603   8.965  11.132  1.00  0.79           O  \\nATOM    508  OE2 GLU A  60      17.990   9.181  12.825  1.00  0.77           O  \\nATOM    509  N   THR A  61      11.526   7.678  12.872  1.00  0.93           N  \\nATOM    510  CA  THR A  61      10.265   7.753  12.143  1.00  0.93           C  \\nATOM    511  C   THR A  61      10.413   8.616  10.893  1.00  0.92           C  \\nATOM    512  CB  THR A  61       9.140   8.319  13.030  1.00  0.90           C  \\nATOM    513  O   THR A  61      10.883   9.754  10.970  1.00  0.90           O  \\nATOM    514  CG2 THR A  61       8.706   7.304  14.082  1.00  0.75           C  \\nATOM    515  OG1 THR A  61       9.608   9.503  13.688  1.00  0.78           O  \\nATOM    516  N   ILE A  62      10.010   8.165   9.799  1.00  0.94           N  \\nATOM    517  CA  ILE A  62       9.892   8.875   8.530  1.00  0.94           C  \\nATOM    518  C   ILE A  62       8.434   8.882   8.076  1.00  0.92           C  \\nATOM    519  CB  ILE A  62      10.788   8.241   7.442  1.00  0.91           C  \\nATOM    520  O   ILE A  62       7.820   7.824   7.921  1.00  0.88           O  \\nATOM    521  CG1 ILE A  62      12.264   8.326   7.850  1.00  0.74           C  \\nATOM    522  CG2 ILE A  62      10.555   8.918   6.088  1.00  0.73           C  \\nATOM    523  CD1 ILE A  62      13.223   7.730   6.829  1.00  0.72           C  \\nATOM    524  N   ASP A  63       7.858  10.039   7.942  1.00  0.92           N  \\nATOM    525  CA  ASP A  63       6.458  10.296   7.618  1.00  0.92           C  \\nATOM    526  C   ASP A  63       6.302  10.730   6.162  1.00  0.91           C  \\nATOM    527  CB  ASP A  63       5.879  11.363   8.550  1.00  0.88           C  \\nATOM    528  O   ASP A  63       5.187  10.775   5.639  1.00  0.89           O  \\nATOM    529  CG  ASP A  63       6.648  12.672   8.503  1.00  0.76           C  \\nATOM    530  OD1 ASP A  63       7.839  12.666   8.125  1.00  0.70           O  \\nATOM    531  OD2 ASP A  63       6.057  13.718   8.850  1.00  0.79           O  \\nATOM    532  N   LYS A  64       7.380  10.960   5.508  1.00  0.94           N  \\nATOM    533  CA  LYS A  64       7.338  11.501   4.152  1.00  0.94           C  \\nATOM    534  C   LYS A  64       8.274  10.732   3.224  1.00  0.94           C  \\nATOM    535  CB  LYS A  64       7.706  12.985   4.155  1.00  0.93           C  \\nATOM    536  O   LYS A  64       9.369  10.335   3.628  1.00  0.94           O  \\nATOM    537  CG  LYS A  64       6.703  13.870   4.880  1.00  0.89           C  \\nATOM    538  CD  LYS A  64       7.085  15.342   4.785  1.00  0.86           C  \\nATOM    539  CE  LYS A  64       6.080  16.229   5.507  1.00  0.86           C  \\nATOM    540  NZ  LYS A  64       6.445  17.674   5.410  1.00  0.83           N  \\nATOM    541  N   ILE A  65       7.887  10.491   1.996  1.00  0.91           N  \\nATOM    542  CA  ILE A  65       8.696   9.859   0.959  1.00  0.92           C  \\nATOM    543  C   ILE A  65       9.788  10.822   0.499  1.00  0.91           C  \\nATOM    544  CB  ILE A  65       7.830   9.418  -0.242  1.00  0.89           C  \\nATOM    545  O   ILE A  65       9.495  11.911   0.001  1.00  0.89           O  \\nATOM    546  CG1 ILE A  65       6.811   8.358   0.195  1.00  0.79           C  \\nATOM    547  CG2 ILE A  65       8.712   8.893  -1.379  1.00  0.77           C  \\nATOM    548  CD1 ILE A  65       5.924   7.851  -0.933  1.00  0.75           C  \\nATOM    549  N   PRO A  66      11.038  10.561   0.594  1.00  0.92           N  \\nATOM    550  CA  PRO A  66      12.121  11.423   0.115  1.00  0.93           C  \\nATOM    551  C   PRO A  66      12.029  11.710  -1.382  1.00  0.93           C  \\nATOM    552  CB  PRO A  66      13.382  10.617   0.439  1.00  0.91           C  \\nATOM    553  O   PRO A  66      11.670  10.825  -2.163  1.00  0.92           O  \\nATOM    554  CG  PRO A  66      12.950   9.633   1.478  1.00  0.86           C  \\nATOM    555  CD  PRO A  66      11.489   9.343   1.282  1.00  0.85           C  \\nATOM    556  N   ALA A  67      12.313  12.886  -1.758  1.00  0.93           N  \\nATOM    557  CA  ALA A  67      12.457  13.393  -3.120  1.00  0.93           C  \\nATOM    558  C   ALA A  67      11.156  13.241  -3.903  1.00  0.92           C  \\nATOM    559  CB  ALA A  67      13.595  12.673  -3.839  1.00  0.91           C  \\nATOM    560  O   ALA A  67      11.173  13.125  -5.130  1.00  0.89           O  \\nATOM    561  N   ALA A  68      10.019  13.274  -3.216  1.00  0.90           N  \\nATOM    562  CA  ALA A  68       8.698  13.382  -3.829  1.00  0.90           C  \\nATOM    563  C   ALA A  68       8.334  14.840  -4.096  1.00  0.89           C  \\nATOM    564  CB  ALA A  68       7.644  12.728  -2.939  1.00  0.88           C  \\nATOM    565  O   ALA A  68       8.877  15.748  -3.462  1.00  0.87           O  \\nATOM    566  N   VAL A  69       7.451  15.131  -5.021  1.00  0.88           N  \\nATOM    567  CA  VAL A  69       6.836  16.435  -5.243  1.00  0.88           C  \\nATOM    568  C   VAL A  69       5.929  16.786  -4.066  1.00  0.87           C  \\nATOM    569  CB  VAL A  69       6.034  16.468  -6.563  1.00  0.85           C  \\nATOM    570  O   VAL A  69       5.318  15.902  -3.460  1.00  0.86           O  \\nATOM    571  CG1 VAL A  69       5.350  17.821  -6.746  1.00  0.75           C  \\nATOM    572  CG2 VAL A  69       6.947  16.162  -7.749  1.00  0.75           C  \\nATOM    573  N   HIS A  70       5.843  18.078  -3.729  1.00  0.84           N  \\nATOM    574  CA  HIS A  70       4.976  18.556  -2.658  1.00  0.86           C  \\nATOM    575  C   HIS A  70       3.545  18.063  -2.845  1.00  0.84           C  \\nATOM    576  CB  HIS A  70       5.000  20.085  -2.592  1.00  0.81           C  \\nATOM    577  O   HIS A  70       3.010  18.105  -3.955  1.00  0.81           O  \\nATOM    578  CG  HIS A  70       4.265  20.646  -1.416  1.00  0.73           C  \\nATOM    579  CD2 HIS A  70       3.085  21.306  -1.338  1.00  0.70           C  \\nATOM    580  ND1 HIS A  70       4.745  20.558  -0.128  1.00  0.68           N  \\nATOM    581  CE1 HIS A  70       3.889  21.140   0.695  1.00  0.69           C  \\nATOM    582  NE2 HIS A  70       2.874  21.602  -0.014  1.00  0.67           N  \\nATOM    583  N   GLY A  71       2.917  17.468  -1.772  1.00  0.86           N  \\nATOM    584  CA  GLY A  71       1.572  16.917  -1.758  1.00  0.86           C  \\nATOM    585  C   GLY A  71       1.515  15.469  -2.207  1.00  0.84           C  \\nATOM    586  O   GLY A  71       0.477  14.816  -2.086  1.00  0.81           O  \\nATOM    587  N   ARG A  72       2.641  14.815  -2.579  1.00  0.87           N  \\nATOM    588  CA  ARG A  72       2.730  13.412  -2.969  1.00  0.87           C  \\nATOM    589  C   ARG A  72       3.769  12.675  -2.130  1.00  0.86           C  \\nATOM    590  CB  ARG A  72       3.073  13.288  -4.455  1.00  0.86           C  \\nATOM    591  O   ARG A  72       4.335  11.674  -2.573  1.00  0.84           O  \\nATOM    592  CG  ARG A  72       2.054  13.937  -5.379  1.00  0.84           C  \\nATOM    593  CD  ARG A  72       2.354  13.645  -6.843  1.00  0.82           C  \\nATOM    594  NE  ARG A  72       1.493  14.417  -7.733  1.00  0.78           N  \\nATOM    595  NH1 ARG A  72       2.591  13.799  -9.669  1.00  0.68           N  \\nATOM    596  NH2 ARG A  72       0.785  15.207  -9.771  1.00  0.66           N  \\nATOM    597  CZ  ARG A  72       1.625  14.473  -9.056  1.00  0.82           C  \\nATOM    598  N   ASP A  73       4.022  13.176  -1.009  1.00  0.88           N  \\nATOM    599  CA  ASP A  73       5.073  12.697  -0.116  1.00  0.90           C  \\nATOM    600  C   ASP A  73       4.499  11.786   0.968  1.00  0.88           C  \\nATOM    601  CB  ASP A  73       5.811  13.875   0.524  1.00  0.86           C  \\nATOM    602  O   ASP A  73       5.232  11.311   1.838  1.00  0.84           O  \\nATOM    603  CG  ASP A  73       4.880  14.852   1.219  1.00  0.80           C  \\nATOM    604  OD1 ASP A  73       3.675  14.888   0.887  1.00  0.77           O  \\nATOM    605  OD2 ASP A  73       5.355  15.595   2.105  1.00  0.83           O  \\nATOM    606  N   ASN A  74       3.089  11.560   0.881  1.00  0.88           N  \\nATOM    607  CA  ASN A  74       2.473  10.639   1.831  1.00  0.89           C  \\nATOM    608  C   ASN A  74       3.060   9.235   1.711  1.00  0.88           C  \\nATOM    609  CB  ASN A  74       0.956  10.599   1.634  1.00  0.86           C  \\nATOM    610  O   ASN A  74       2.952   8.600   0.660  1.00  0.86           O  \\nATOM    611  CG  ASN A  74       0.247   9.814   2.719  1.00  0.79           C  \\nATOM    612  ND2 ASN A  74      -1.062  10.009   2.832  1.00  0.78           N  \\nATOM    613  OD1 ASN A  74       0.869   9.039   3.450  1.00  0.77           O  \\nATOM    614  N   ILE A  75       3.623   8.722   2.815  1.00  0.85           N  \\nATOM    615  CA  ILE A  75       4.422   7.503   2.877  1.00  0.87           C  \\nATOM    616  C   ILE A  75       3.508   6.282   2.804  1.00  0.81           C  \\nATOM    617  CB  ILE A  75       5.281   7.454   4.160  1.00  0.80           C  \\nATOM    618  O   ILE A  75       3.943   5.195   2.417  1.00  0.75           O  \\nATOM    619  CG1 ILE A  75       6.428   6.451   3.998  1.00  0.65           C  \\nATOM    620  CG2 ILE A  75       4.416   7.106   5.375  1.00  0.63           C  \\nATOM    621  CD1 ILE A  75       7.558   6.633   5.002  1.00  0.65           C  \\nATOM    622  N   HIS A  76       2.153   6.550   3.157  1.00  0.83           N  \\nATOM    623  CA  HIS A  76       1.156   5.487   3.204  1.00  0.83           C  \\nATOM    624  C   HIS A  76       0.540   5.248   1.829  1.00  0.80           C  \\nATOM    625  CB  HIS A  76       0.061   5.823   4.218  1.00  0.79           C  \\nATOM    626  O   HIS A  76      -0.219   4.294   1.641  1.00  0.76           O  \\nATOM    627  CG  HIS A  76       0.568   5.995   5.614  1.00  0.73           C  \\nATOM    628  CD2 HIS A  76       0.516   7.054   6.455  1.00  0.70           C  \\nATOM    629  ND1 HIS A  76       1.226   4.993   6.295  1.00  0.69           N  \\nATOM    630  CE1 HIS A  76       1.556   5.431   7.499  1.00  0.69           C  \\nATOM    631  NE2 HIS A  76       1.137   6.679   7.621  1.00  0.72           N  \\nATOM    632  N   VAL A  77       0.779   5.979   0.994  1.00  0.80           N  \\nATOM    633  CA  VAL A  77       0.251   5.826  -0.358  1.00  0.78           C  \\nATOM    634  C   VAL A  77       1.203   4.970  -1.191  1.00  0.77           C  \\nATOM    635  CB  VAL A  77       0.030   7.195  -1.039  1.00  0.76           C  \\nATOM    636  O   VAL A  77       2.345   5.364  -1.440  1.00  0.73           O  \\nATOM    637  CG1 VAL A  77      -0.547   7.013  -2.442  1.00  0.72           C  \\nATOM    638  CG2 VAL A  77      -0.889   8.071  -0.190  1.00  0.72           C  \\nATOM    639  N   GLU A  78       0.756   3.760  -1.652  1.00  0.72           N  \\nATOM    640  CA  GLU A  78       1.436   2.824  -2.543  1.00  0.72           C  \\nATOM    641  C   GLU A  78       1.022   3.043  -3.995  1.00  0.69           C  \\nATOM    642  CB  GLU A  78       1.147   1.379  -2.126  1.00  0.66           C  \\nATOM    643  O   GLU A  78      -0.035   2.575  -4.422  1.00  0.64           O  \\nATOM    644  CG  GLU A  78       1.772   0.986  -0.795  1.00  0.62           C  \\nATOM    645  CD  GLU A  78       1.598  -0.487  -0.462  1.00  0.61           C  \\nATOM    646  OE1 GLU A  78       0.833  -1.182  -1.168  1.00  0.66           O  \\nATOM    647  OE2 GLU A  78       2.232  -0.949   0.512  1.00  0.65           O  \\nATOM    648  N   ILE A  79       1.777   3.716  -4.702  1.00  0.73           N  \\nATOM    649  CA  ILE A  79       1.616   3.920  -6.137  1.00  0.73           C  \\nATOM    650  C   ILE A  79       2.880   3.472  -6.867  1.00  0.72           C  \\nATOM    651  CB  ILE A  79       1.301   5.397  -6.466  1.00  0.69           C  \\nATOM    652  O   ILE A  79       3.937   4.091  -6.728  1.00  0.68           O  \\nATOM    653  CG1 ILE A  79       0.050   5.858  -5.709  1.00  0.63           C  \\nATOM    654  CG2 ILE A  79       1.130   5.589  -7.976  1.00  0.63           C  \\nATOM    655  CD1 ILE A  79      -0.269   7.336  -5.885  1.00  0.63           C  \\nATOM    656  N   TYR A  80       2.934   2.361  -7.638  1.00  0.63           N  \\nATOM    657  CA  TYR A  80       4.045   1.683  -8.296  1.00  0.62           C  \\nATOM    658  C   TYR A  80       4.321   2.291  -9.666  1.00  0.61           C  \\nATOM    659  CB  TYR A  80       3.754   0.186  -8.438  1.00  0.57           C  \\nATOM    660  O   TYR A  80       5.460   2.284 -10.137  1.00  0.58           O  \\nATOM    661  CG  TYR A  80       3.697  -0.550  -7.122  1.00  0.53           C  \\nATOM    662  CD1 TYR A  80       4.863  -0.972  -6.486  1.00  0.50           C  \\nATOM    663  CD2 TYR A  80       2.478  -0.825  -6.511  1.00  0.52           C  \\nATOM    664  CE1 TYR A  80       4.815  -1.651  -5.274  1.00  0.51           C  \\nATOM    665  CE2 TYR A  80       2.419  -1.504  -5.298  1.00  0.49           C  \\nATOM    666  OH  TYR A  80       3.538  -2.585  -3.488  1.00  0.47           O  \\nATOM    667  CZ  TYR A  80       3.591  -1.912  -4.688  1.00  0.51           C  \\nATOM    668  N   GLY A  81       3.426   3.096 -10.219  1.00  0.66           N  \\nATOM    669  CA  GLY A  81       3.453   3.611 -11.579  1.00  0.64           C  \\nATOM    670  C   GLY A  81       2.070   3.836 -12.159  1.00  0.64           C  \\nATOM    671  O   GLY A  81       1.107   3.182 -11.755  1.00  0.59           O  \\nATOM    672  N   MET A  82       1.920   4.939 -12.938  1.00  0.67           N  \\nATOM    673  CA  MET A  82       0.767   5.272 -13.770  1.00  0.67           C  \\nATOM    674  C   MET A  82       1.104   5.128 -15.250  1.00  0.67           C  \\nATOM    675  CB  MET A  82       0.287   6.695 -13.480  1.00  0.61           C  \\nATOM    676  O   MET A  82       2.220   5.441 -15.671  1.00  0.64           O  \\nATOM    677  CG  MET A  82      -0.258   6.883 -12.073  1.00  0.55           C  \\nATOM    678  SD  MET A  82      -1.440   8.282 -11.959  1.00  0.56           S  \\nATOM    679  CE  MET A  82      -0.668   9.449 -13.114  1.00  0.41           C  \\nATOM    680  N   GLN A  83       0.252   4.309 -15.822  1.00  0.67           N  \\nATOM    681  CA  GLN A  83       0.170   4.287 -17.278  1.00  0.66           C  \\nATOM    682  C   GLN A  83      -0.819   5.331 -17.789  1.00  0.66           C  \\nATOM    683  CB  GLN A  83      -0.232   2.897 -17.773  1.00  0.60           C  \\nATOM    684  O   GLN A  83      -1.863   5.559 -17.173  1.00  0.60           O  \\nATOM    685  CG  GLN A  83       0.821   1.826 -17.521  1.00  0.55           C  \\nATOM    686  CD  GLN A  83       0.459   0.490 -18.140  1.00  0.58           C  \\nATOM    687  NE2 GLN A  83       1.357  -0.483 -18.018  1.00  0.43           N  \\nATOM    688  OE1 GLN A  83      -0.618   0.332 -18.723  1.00  0.60           O  \\nATOM    689  N   GLY A  84      -0.475   6.173 -18.559  1.00  0.57           N  \\nATOM    690  CA  GLY A  84      -1.487   7.029 -19.156  1.00  0.58           C  \\nATOM    691  C   GLY A  84      -1.707   8.319 -18.389  1.00  0.55           C  \\nATOM    692  O   GLY A  84      -1.292   8.439 -17.234  1.00  0.50           O  \\nATOM    693  N   ILE A  85      -1.655   9.660 -19.118  1.00  0.45           N  \\nATOM    694  CA  ILE A  85      -1.559  11.067 -19.490  1.00  0.44           C  \\nATOM    695  C   ILE A  85      -0.308  11.681 -18.867  1.00  0.45           C  \\nATOM    696  CB  ILE A  85      -2.816  11.853 -19.055  1.00  0.40           C  \\nATOM    697  O   ILE A  85      -0.170  11.716 -17.642  1.00  0.42           O  \\nATOM    698  CG1 ILE A  85      -4.078  11.211 -19.643  1.00  0.36           C  \\nATOM    699  CG2 ILE A  85      -2.705  13.323 -19.472  1.00  0.37           C  \\nATOM    700  CD1 ILE A  85      -5.375  11.866 -19.189  1.00  0.32           C  \\nATOM    701  N   PRO A  86       0.793  11.970 -19.688  1.00  0.44           N  \\nATOM    702  CA  PRO A  86       2.063  12.670 -19.892  1.00  0.44           C  \\nATOM    703  C   PRO A  86       1.901  14.188 -19.925  1.00  0.44           C  \\nATOM    704  CB  PRO A  86       2.542  12.143 -21.247  1.00  0.40           C  \\nATOM    705  O   PRO A  86       0.943  14.698 -20.512  1.00  0.42           O  \\nATOM    706  CG  PRO A  86       1.434  11.262 -21.728  1.00  0.39           C  \\nATOM    707  CD  PRO A  86       0.364  11.223 -20.674  1.00  0.43           C  \\nATOM    708  N   SER A  87       2.454  14.985 -18.952  1.00  0.41           N  \\nATOM    709  CA  SER A  87       3.254  16.169 -18.657  1.00  0.41           C  \\nATOM    710  C   SER A  87       2.508  17.446 -19.027  1.00  0.41           C  \\nATOM    711  CB  SER A  87       4.588  16.114 -19.401  1.00  0.38           C  \\nATOM    712  O   SER A  87       2.757  18.506 -18.449  1.00  0.38           O  \\nATOM    713  OG  SER A  87       4.428  15.521 -20.678  1.00  0.37           O  \\nATOM    714  N   GLY A  88       1.301  17.511 -19.305  1.00  0.44           N  \\nATOM    715  CA  GLY A  88       0.392  18.623 -19.533  1.00  0.43           C  \\nATOM    716  C   GLY A  88      -0.853  18.563 -18.668  1.00  0.44           C  \\nATOM    717  O   GLY A  88      -1.616  19.530 -18.601  1.00  0.41           O  \\nATOM    718  N   ALA A  89      -0.945  17.504 -17.871  1.00  0.42           N  \\nATOM    719  CA  ALA A  89      -2.178  17.551 -17.089  1.00  0.42           C  \\nATOM    720  C   ALA A  89      -1.957  18.268 -15.760  1.00  0.42           C  \\nATOM    721  CB  ALA A  89      -2.708  16.140 -16.847  1.00  0.39           C  \\nATOM    722  O   ALA A  89      -2.901  18.801 -15.171  1.00  0.39           O  \\nATOM    723  N   TYR A  90      -0.698  18.693 -15.572  1.00  0.44           N  \\nATOM    724  CA  TYR A  90      -0.474  19.523 -14.393  1.00  0.45           C  \\nATOM    725  C   TYR A  90       0.233  20.821 -14.766  1.00  0.44           C  \\nATOM    726  CB  TYR A  90       0.348  18.763 -13.348  1.00  0.41           C  \\nATOM    727  O   TYR A  90       1.315  21.115 -14.254  1.00  0.41           O  \\nATOM    728  CG  TYR A  90      -0.480  17.868 -12.458  1.00  0.39           C  \\nATOM    729  CD1 TYR A  90      -1.132  18.379 -11.338  1.00  0.36           C  \\nATOM    730  CD2 TYR A  90      -0.610  16.512 -12.734  1.00  0.38           C  \\nATOM    731  CE1 TYR A  90      -1.895  17.558 -10.514  1.00  0.38           C  \\nATOM    732  CE2 TYR A  90      -1.370  15.681 -11.916  1.00  0.40           C  \\nATOM    733  OH  TYR A  90      -2.761  15.396  -9.998  1.00  0.31           O  \\nATOM    734  CZ  TYR A  90      -2.008  16.213 -10.810  1.00  0.34           C  \\nATOM    735  N   ARG A  91       0.360  21.172 -15.938  1.00  0.44           N  \\nATOM    736  CA  ARG A  91       0.713  22.549 -16.271  1.00  0.44           C  \\nATOM    737  C   ARG A  91      -0.279  23.532 -15.659  1.00  0.44           C  \\nATOM    738  CB  ARG A  91       0.768  22.737 -17.789  1.00  0.41           C  \\nATOM    739  O   ARG A  91      -0.193  24.738 -15.897  1.00  0.41           O  \\nATOM    740  CG  ARG A  91       2.077  22.288 -18.418  1.00  0.40           C  \\nATOM    741  CD  ARG A  91       2.129  22.612 -19.904  1.00  0.43           C  \\nATOM    742  NE  ARG A  91       3.154  21.830 -20.591  1.00  0.40           N  \\nATOM    743  NH1 ARG A  91       3.409  23.317 -22.340  1.00  0.32           N  \\nATOM    744  NH2 ARG A  91       4.659  21.397 -22.271  1.00  0.28           N  \\nATOM    745  CZ  ARG A  91       3.738  22.183 -21.732  1.00  0.38           C  \\nATOM    746  N   GLY A  92      -0.974  23.240 -14.566  1.00  0.42           N  \\nATOM    747  CA  GLY A  92      -1.790  24.273 -13.948  1.00  0.42           C  \\nATOM    748  C   GLY A  92      -1.593  24.371 -12.447  1.00  0.42           C  \\nATOM    749  O   GLY A  92      -2.170  25.244 -11.796  1.00  0.39           O  \\nATOM    750  N   ALA A  93      -0.241  24.127 -11.875  1.00  0.42           N  \\nATOM    751  CA  ALA A  93      -0.161  24.657 -10.516  1.00  0.44           C  \\nATOM    752  C   ALA A  93       1.263  24.567  -9.975  1.00  0.42           C  \\nATOM    753  CB  ALA A  93      -1.125  23.913  -9.596  1.00  0.39           C  \\nATOM    754  O   ALA A  93       1.542  25.020  -8.863  1.00  0.40           O  \\nATOM    755  N   ALA A  94       2.407  24.677 -10.880  1.00  0.44           N  \\nATOM    756  CA  ALA A  94       3.676  24.999 -10.234  1.00  0.45           C  \\nATOM    757  C   ALA A  94       4.717  25.441 -11.259  1.00  0.44           C  \\nATOM    758  CB  ALA A  94       4.190  23.799  -9.443  1.00  0.41           C  \\nATOM    759  O   ALA A  94       5.897  25.585 -10.932  1.00  0.42           O  \\nATOM    760  N   ASP A  95       4.188  26.193 -12.348  1.00  0.45           N  \\nATOM    761  CA  ASP A  95       5.117  27.036 -13.094  1.00  0.46           C  \\nATOM    762  C   ASP A  95       4.420  28.288 -13.621  1.00  0.45           C  \\nATOM    763  CB  ASP A  95       5.740  26.254 -14.251  1.00  0.41           C  \\nATOM    764  O   ASP A  95       3.935  28.304 -14.754  1.00  0.42           O  \\nATOM    765  CG  ASP A  95       6.748  25.216 -13.791  1.00  0.40           C  \\nATOM    766  OD1 ASP A  95       7.530  25.496 -12.856  1.00  0.42           O  \\nATOM    767  OD2 ASP A  95       6.764  24.108 -14.369  1.00  0.44           O  \\nATOM    768  N   GLU A  96       3.445  28.896 -12.862  1.00  0.42           N  \\nATOM    769  CA  GLU A  96       3.431  30.352 -12.968  1.00  0.43           C  \\nATOM    770  C   GLU A  96       4.822  30.935 -12.733  1.00  0.42           C  \\nATOM    771  CB  GLU A  96       2.434  30.954 -11.975  1.00  0.39           C  \\nATOM    772  O   GLU A  96       5.307  30.960 -11.600  1.00  0.40           O  \\nATOM    773  CG  GLU A  96       0.977  30.695 -12.333  1.00  0.38           C  \\nATOM    774  CD  GLU A  96      -0.002  31.403 -11.411  1.00  0.42           C  \\nATOM    775  OE1 GLU A  96       0.424  31.920 -10.354  1.00  0.46           O  \\nATOM    776  OE2 GLU A  96      -1.207  31.440 -11.748  1.00  0.44           O  \\nATOM    777  N   GLU A  97       5.705  30.718 -13.551  1.00  0.46           N  \\nATOM    778  CA  GLU A  97       6.376  31.966 -13.901  1.00  0.47           C  \\nATOM    779  C   GLU A  97       5.414  32.939 -14.578  1.00  0.46           C  \\nATOM    780  CB  GLU A  97       7.576  31.695 -14.812  1.00  0.44           C  \\nATOM    781  O   GLU A  97       4.724  32.574 -15.532  1.00  0.44           O  \\nATOM    782  CG  GLU A  97       8.918  31.756 -14.096  1.00  0.44           C  \\nATOM    783  CD  GLU A  97      10.105  31.575 -15.028  1.00  0.49           C  \\nATOM    784  OE1 GLU A  97       9.933  30.993 -16.123  1.00  0.51           O  \\nATOM    785  OE2 GLU A  97      11.216  32.018 -14.661  1.00  0.46           O  \\nATOM    786  N   PRO A  98       4.712  33.816 -13.862  1.00  0.48           N  \\nATOM    787  CA  PRO A  98       4.154  35.000 -14.520  1.00  0.49           C  \\nATOM    788  C   PRO A  98       4.969  35.439 -15.735  1.00  0.48           C  \\nATOM    789  CB  PRO A  98       4.193  36.066 -13.422  1.00  0.45           C  \\nATOM    790  O   PRO A  98       6.087  35.940 -15.584  1.00  0.46           O  \\nATOM    791  CG  PRO A  98       5.127  35.518 -12.391  1.00  0.44           C  \\nATOM    792  CD  PRO A  98       5.344  34.058 -12.672  1.00  0.48           C  \\nATOM    793  N   ASP A  99       5.029  34.787 -16.827  1.00  0.44           N  \\nATOM    794  CA  ASP A  99       5.511  35.816 -17.743  1.00  0.48           C  \\nATOM    795  C   ASP A  99       5.890  35.214 -19.095  1.00  0.44           C  \\nATOM    796  CB  ASP A  99       6.710  36.551 -17.141  1.00  0.41           C  \\nATOM    797  O   ASP A  99       6.850  34.447 -19.192  1.00  0.43           O  \\nATOM    798  CG  ASP A  99       6.454  38.034 -16.934  1.00  0.43           C  \\nATOM    799  OD1 ASP A  99       5.491  38.574 -17.519  1.00  0.44           O  \\nATOM    800  OD2 ASP A  99       7.224  38.668 -16.180  1.00  0.45           O  \\nATOM    801  N   GLU A 100       4.738  34.982 -20.024  1.00  0.49           N  \\nATOM    802  CA  GLU A 100       4.836  35.518 -21.379  1.00  0.50           C  \\nATOM    803  C   GLU A 100       4.317  34.516 -22.407  1.00  0.49           C  \\nATOM    804  CB  GLU A 100       6.281  35.903 -21.702  1.00  0.46           C  \\nATOM    805  O   GLU A 100       4.621  34.626 -23.597  1.00  0.47           O  \\nATOM    806  CG  GLU A 100       6.587  37.378 -21.489  1.00  0.45           C  \\nATOM    807  CD  GLU A 100       7.986  37.772 -21.936  1.00  0.49           C  \\nATOM    808  OE1 GLU A 100       8.872  36.889 -22.005  1.00  0.50           O  \\nATOM    809  OE2 GLU A 100       8.198  38.972 -22.219  1.00  0.45           O  \\nATOM    810  N   LYS A 101       3.172  33.841 -22.071  1.00  0.50           N  \\nATOM    811  CA  LYS A 101       2.411  33.473 -23.262  1.00  0.50           C  \\nATOM    812  C   LYS A 101       0.914  33.667 -23.040  1.00  0.50           C  \\nATOM    813  CB  LYS A 101       2.699  32.024 -23.657  1.00  0.46           C  \\nATOM    814  O   LYS A 101       0.215  32.735 -22.636  1.00  0.47           O  \\nATOM    815  CG  LYS A 101       3.940  31.852 -24.520  1.00  0.44           C  \\nATOM    816  CD  LYS A 101       4.187  30.387 -24.858  1.00  0.47           C  \\nATOM    817  CE  LYS A 101       5.444  30.211 -25.699  1.00  0.40           C  \\nATOM    818  NZ  LYS A 101       5.814  28.772 -25.848  1.00  0.38           N  \\nATOM    819  N   ARG A 102       0.573  34.823 -22.352  1.00  0.49           N  \\nATOM    820  CA  ARG A 102      -0.573  35.684 -22.627  1.00  0.49           C  \\nATOM    821  C   ARG A 102      -0.518  36.232 -24.049  1.00  0.49           C  \\nATOM    822  CB  ARG A 102      -0.633  36.837 -21.623  1.00  0.46           C  \\nATOM    823  O   ARG A 102       0.207  37.190 -24.323  1.00  0.46           O  \\nATOM    824  CG  ARG A 102      -1.312  36.477 -20.311  1.00  0.45           C  \\nATOM    825  CD  ARG A 102      -1.373  37.666 -19.363  1.00  0.48           C  \\nATOM    826  NE  ARG A 102      -2.004  37.312 -18.094  1.00  0.43           N  \\nATOM    827  NH1 ARG A 102      -1.962  39.457 -17.237  1.00  0.33           N  \\nATOM    828  NH2 ARG A 102      -2.845  37.740 -16.000  1.00  0.29           N  \\nATOM    829  CZ  ARG A 102      -2.269  38.170 -17.113  1.00  0.41           C  \\nATOM    830  N   SER A 103      -0.423  35.374 -25.023  1.00  0.50           N  \\nATOM    831  CA  SER A 103      -0.821  35.950 -26.304  1.00  0.52           C  \\nATOM    832  C   SER A 103      -1.182  34.862 -27.310  1.00  0.50           C  \\nATOM    833  CB  SER A 103       0.297  36.827 -26.869  1.00  0.46           C  \\nATOM    834  O   SER A 103      -0.309  34.135 -27.787  1.00  0.48           O  \\nATOM    835  OG  SER A 103       0.363  38.063 -26.180  1.00  0.48           O  \\nATOM    836  N   ARG A 104      -2.344  34.052 -26.961  1.00  0.55           N  \\nATOM    837  CA  ARG A 104      -3.249  33.759 -28.067  1.00  0.55           C  \\nATOM    838  C   ARG A 104      -4.326  32.765 -27.644  1.00  0.55           C  \\nATOM    839  CB  ARG A 104      -2.473  33.211 -29.267  1.00  0.51           C  \\nATOM    840  O   ARG A 104      -4.178  31.558 -27.845  1.00  0.52           O  \\nATOM    841  CG  ARG A 104      -2.443  34.150 -30.462  1.00  0.50           C  \\nATOM    842  CD  ARG A 104      -1.629  33.574 -31.612  1.00  0.52           C  \\nATOM    843  NE  ARG A 104      -1.621  34.466 -32.767  1.00  0.44           N  \\nATOM    844  NH1 ARG A 104      -0.225  33.140 -34.043  1.00  0.33           N  \\nATOM    845  NH2 ARG A 104      -1.017  35.119 -34.885  1.00  0.28           N  \\nATOM    846  CZ  ARG A 104      -0.955  34.240 -33.896  1.00  0.44           C  \\nATOM    847  N   MET A 105      -5.080  33.126 -26.635  1.00  0.56           N  \\nATOM    848  CA  MET A 105      -6.464  32.689 -26.474  1.00  0.57           C  \\nATOM    849  C   MET A 105      -7.406  33.553 -27.306  1.00  0.56           C  \\nATOM    850  CB  MET A 105      -6.876  32.735 -25.002  1.00  0.51           C  \\nATOM    851  O   MET A 105      -7.746  34.668 -26.909  1.00  0.54           O  \\nATOM    852  CG  MET A 105      -6.360  31.562 -24.184  1.00  0.50           C  \\nATOM    853  SD  MET A 105      -7.091  31.501 -22.503  1.00  0.52           S  \\nATOM    854  CE  MET A 105      -6.615  29.822 -22.007  1.00  0.43           C  \\nATOM    855  N   ASP A 106      -7.275  33.546 -28.528  1.00  0.52           N  \\nATOM    856  CA  ASP A 106      -8.165  33.855 -29.644  1.00  0.55           C  \\nATOM    857  C   ASP A 106      -9.182  32.737 -29.861  1.00  0.52           C  \\nATOM    858  CB  ASP A 106      -7.359  34.089 -30.923  1.00  0.48           C  \\nATOM    859  O   ASP A 106      -8.811  31.607 -30.186  1.00  0.50           O  \\nATOM    860  CG  ASP A 106      -7.732  35.379 -31.631  1.00  0.49           C  \\nATOM    861  OD1 ASP A 106      -8.770  35.985 -31.288  1.00  0.48           O  \\nATOM    862  OD2 ASP A 106      -6.982  35.792 -32.543  1.00  0.51           O  \\nATOM    863  N   ASN A 107     -10.380  32.613 -29.052  1.00  0.59           N  \\nATOM    864  CA  ASN A 107     -11.699  32.371 -29.625  1.00  0.60           C  \\nATOM    865  C   ASN A 107     -12.210  30.974 -29.285  1.00  0.59           C  \\nATOM    866  CB  ASN A 107     -11.671  32.572 -31.142  1.00  0.54           C  \\nATOM    867  O   ASN A 107     -12.269  30.101 -30.154  1.00  0.56           O  \\nATOM    868  CG  ASN A 107     -11.679  34.035 -31.538  1.00  0.52           C  \\nATOM    869  ND2 ASN A 107     -11.097  34.340 -32.692  1.00  0.49           N  \\nATOM    870  OD1 ASN A 107     -12.203  34.884 -30.812  1.00  0.53           O  \\nATOM    871  N   GLY A 108     -12.073  30.583 -27.961  1.00  0.60           N  \\nATOM    872  CA  GLY A 108     -13.144  29.669 -27.596  1.00  0.60           C  \\nATOM    873  C   GLY A 108     -14.267  30.341 -26.830  1.00  0.59           C  \\nATOM    874  O   GLY A 108     -14.027  31.267 -26.052  1.00  0.57           O  \\nATOM    875  N   PRO A 109     -15.439  30.339 -27.373  1.00  0.60           N  \\nATOM    876  CA  PRO A 109     -16.659  30.795 -26.702  1.00  0.59           C  \\nATOM    877  C   PRO A 109     -16.618  30.578 -25.191  1.00  0.60           C  \\nATOM    878  CB  PRO A 109     -17.755  29.943 -27.346  1.00  0.56           C  \\nATOM    879  O   PRO A 109     -16.061  29.582 -24.722  1.00  0.57           O  \\nATOM    880  CG  PRO A 109     -17.063  28.689 -27.773  1.00  0.54           C  \\nATOM    881  CD  PRO A 109     -15.622  29.010 -28.050  1.00  0.55           C  \\nATOM    882  N   PRO A 110     -16.603  31.679 -24.299  1.00  0.59           N  \\nATOM    883  CA  PRO A 110     -16.786  31.679 -22.846  1.00  0.58           C  \\nATOM    884  C   PRO A 110     -17.306  30.344 -22.316  1.00  0.59           C  \\nATOM    885  CB  PRO A 110     -17.810  32.793 -22.619  1.00  0.55           C  \\nATOM    886  O   PRO A 110     -18.318  29.835 -22.805  1.00  0.57           O  \\nATOM    887  CG  PRO A 110     -18.589  32.860 -23.893  1.00  0.54           C  \\nATOM    888  CD  PRO A 110     -17.705  32.402 -25.018  1.00  0.54           C  \\nATOM    889  N   MET A 111     -16.518  29.372 -21.759  1.00  0.63           N  \\nATOM    890  CA  MET A 111     -17.068  28.307 -20.924  1.00  0.64           C  \\nATOM    891  C   MET A 111     -18.303  28.789 -20.171  1.00  0.63           C  \\nATOM    892  CB  MET A 111     -16.016  27.802 -19.935  1.00  0.59           C  \\nATOM    893  O   MET A 111     -18.294  29.873 -19.585  1.00  0.60           O  \\nATOM    894  CG  MET A 111     -14.990  26.867 -20.556  1.00  0.57           C  \\nATOM    895  SD  MET A 111     -15.545  25.117 -20.559  1.00  0.55           S  \\nATOM    896  CE  MET A 111     -14.931  24.585 -18.936  1.00  0.48           C  \\nATOM    897  N   PRO A 112     -19.481  28.351 -20.524  1.00  0.60           N  \\nATOM    898  CA  PRO A 112     -20.698  28.674 -19.776  1.00  0.60           C  \\nATOM    899  C   PRO A 112     -20.492  28.624 -18.264  1.00  0.60           C  \\nATOM    900  CB  PRO A 112     -21.686  27.595 -20.228  1.00  0.56           C  \\nATOM    901  O   PRO A 112     -19.881  27.684 -17.750  1.00  0.57           O  \\nATOM    902  CG  PRO A 112     -20.830  26.438 -20.630  1.00  0.55           C  \\nATOM    903  CD  PRO A 112     -19.507  26.966 -21.107  1.00  0.55           C  \\nATOM    904  N   THR A 113     -20.275  29.807 -17.461  1.00  0.62           N  \\nATOM    905  CA  THR A 113     -20.480  30.022 -16.033  1.00  0.62           C  \\nATOM    906  C   THR A 113     -21.146  28.806 -15.394  1.00  0.61           C  \\nATOM    907  CB  THR A 113     -21.336  31.275 -15.773  1.00  0.58           C  \\nATOM    908  O   THR A 113     -22.148  28.301 -15.904  1.00  0.59           O  \\nATOM    909  CG2 THR A 113     -20.468  32.456 -15.352  1.00  0.55           C  \\nATOM    910  OG1 THR A 113     -22.043  31.620 -16.971  1.00  0.57           O  \\nATOM    911  N   PRO A 114     -20.443  27.861 -14.669  1.00  0.63           N  \\nATOM    912  CA  PRO A 114     -21.224  26.867 -13.929  1.00  0.62           C  \\nATOM    913  C   PRO A 114     -22.634  27.351 -13.598  1.00  0.62           C  \\nATOM    914  CB  PRO A 114     -20.406  26.656 -12.652  1.00  0.58           C  \\nATOM    915  O   PRO A 114     -22.799  28.405 -12.977  1.00  0.59           O  \\nATOM    916  CG  PRO A 114     -19.639  27.927 -12.475  1.00  0.56           C  \\nATOM    917  CD  PRO A 114     -19.488  28.584 -13.817  1.00  0.57           C  \\nATOM    918  N   MET A 115     -23.583  27.019 -14.384  1.00  0.59           N  \\nATOM    919  CA  MET A 115     -24.986  27.155 -14.005  1.00  0.60           C  \\nATOM    920  C   MET A 115     -25.179  26.869 -12.520  1.00  0.59           C  \\nATOM    921  CB  MET A 115     -25.861  26.215 -14.837  1.00  0.56           C  \\nATOM    922  O   MET A 115     -24.622  25.906 -11.991  1.00  0.57           O  \\nATOM    923  CG  MET A 115     -26.074  26.680 -16.268  1.00  0.55           C  \\nATOM    924  SD  MET A 115     -27.789  27.256 -16.572  1.00  0.53           S  \\nATOM    925  CE  MET A 115     -28.507  25.754 -17.295  1.00  0.47           C  \\nATOM    926  N   PRO A 116     -25.351  27.919 -11.573  1.00  0.60           N  \\nATOM    927  CA  PRO A 116     -25.891  27.588 -10.252  1.00  0.60           C  \\nATOM    928  C   PRO A 116     -26.636  26.255 -10.237  1.00  0.60           C  \\nATOM    929  CB  PRO A 116     -26.845  28.749  -9.960  1.00  0.56           C  \\nATOM    930  O   PRO A 116     -27.543  26.041 -11.045  1.00  0.58           O  \\nATOM    931  CG  PRO A 116     -27.322  29.199 -11.303  1.00  0.55           C  \\nATOM    932  CD  PRO A 116     -26.264  28.875 -12.318  1.00  0.56           C  \\nATOM    933  N   PHE A 117     -25.956  25.055  -9.917  1.00  0.62           N  \\nATOM    934  CA  PHE A 117     -26.813  23.938  -9.538  1.00  0.62           C  \\nATOM    935  C   PHE A 117     -28.219  24.423  -9.207  1.00  0.61           C  \\nATOM    936  CB  PHE A 117     -26.221  23.187  -8.341  1.00  0.58           C  \\nATOM    937  O   PHE A 117     -28.393  25.326  -8.386  1.00  0.59           O  \\nATOM    938  CG  PHE A 117     -25.160  22.188  -8.716  1.00  0.56           C  \\nATOM    939  CD1 PHE A 117     -25.506  20.913  -9.147  1.00  0.53           C  \\nATOM    940  CD2 PHE A 117     -23.815  22.524  -8.636  1.00  0.53           C  \\nATOM    941  CE1 PHE A 117     -24.526  19.987  -9.495  1.00  0.50           C  \\nATOM    942  CE2 PHE A 117     -22.829  21.604  -8.982  1.00  0.51           C  \\nATOM    943  CZ  PHE A 117     -23.187  20.335  -9.410  1.00  0.47           C  \\nATOM    944  N   PRO A 118     -29.093  24.330 -10.062  1.00  0.55           N  \\nATOM    945  CA  PRO A 118     -30.489  24.531  -9.668  1.00  0.54           C  \\nATOM    946  C   PRO A 118     -30.821  23.889  -8.322  1.00  0.55           C  \\nATOM    947  CB  PRO A 118     -31.275  23.864 -10.799  1.00  0.51           C  \\nATOM    948  O   PRO A 118     -30.674  22.675  -8.161  1.00  0.53           O  \\nATOM    949  CG  PRO A 118     -30.345  22.833 -11.353  1.00  0.50           C  \\nATOM    950  CD  PRO A 118     -28.938  23.219 -10.998  1.00  0.51           C  \\nATOM    951  N   GLN A 119     -30.603  24.579  -7.110  1.00  0.58           N  \\nATOM    952  CA  GLN A 119     -31.528  24.482  -5.986  1.00  0.58           C  \\nATOM    953  C   GLN A 119     -32.882  23.937  -6.432  1.00  0.57           C  \\nATOM    954  CB  GLN A 119     -31.706  25.846  -5.318  1.00  0.54           C  \\nATOM    955  O   GLN A 119     -33.490  24.461  -7.368  1.00  0.55           O  \\nATOM    956  CG  GLN A 119     -30.485  26.312  -4.536  1.00  0.53           C  \\nATOM    957  CD  GLN A 119     -30.706  27.641  -3.839  1.00  0.55           C  \\nATOM    958  NE2 GLN A 119     -29.742  28.049  -3.021  1.00  0.44           N  \\nATOM    959  OE1 GLN A 119     -31.735  28.296  -4.034  1.00  0.55           O  \\nATOM    960  N   HIS A 120     -32.937  22.640  -6.661  1.00  0.58           N  \\nATOM    961  CA  HIS A 120     -34.241  22.052  -6.375  1.00  0.59           C  \\nATOM    962  C   HIS A 120     -34.960  21.657  -7.661  1.00  0.58           C  \\nATOM    963  CB  HIS A 120     -35.103  23.025  -5.569  1.00  0.53           C  \\nATOM    964  O   HIS A 120     -35.505  22.514  -8.361  1.00  0.55           O  \\nATOM    965  CG  HIS A 120     -34.826  22.997  -4.100  1.00  0.52           C  \\nATOM    966  CD2 HIS A 120     -34.269  23.921  -3.282  1.00  0.50           C  \\nATOM    967  ND1 HIS A 120     -35.129  21.912  -3.306  1.00  0.51           N  \\nATOM    968  CE1 HIS A 120     -34.772  22.172  -2.060  1.00  0.47           C  \\nATOM    969  NE2 HIS A 120     -34.247  23.384  -2.018  1.00  0.43           N  \\nATOM    970  N   PHE A 121     -34.576  20.679  -8.402  1.00  0.56           N  \\nATOM    971  CA  PHE A 121     -35.644  20.057  -9.175  1.00  0.57           C  \\nATOM    972  C   PHE A 121     -36.284  18.916  -8.394  1.00  0.56           C  \\nATOM    973  CB  PHE A 121     -35.110  19.542 -10.515  1.00  0.53           C  \\nATOM    974  O   PHE A 121     -35.603  17.964  -8.005  1.00  0.53           O  \\nATOM    975  CG  PHE A 121     -35.120  20.574 -11.610  1.00  0.51           C  \\nATOM    976  CD1 PHE A 121     -36.267  20.802 -12.360  1.00  0.48           C  \\nATOM    977  CD2 PHE A 121     -33.980  21.318 -11.890  1.00  0.49           C  \\nATOM    978  CE1 PHE A 121     -36.279  21.757 -13.374  1.00  0.47           C  \\nATOM    979  CE2 PHE A 121     -33.985  22.274 -12.901  1.00  0.49           C  \\nATOM    980  CZ  PHE A 121     -35.135  22.491 -13.643  1.00  0.42           C  \\nATOM    981  N   PRO A 122     -37.363  19.263  -7.733  1.00  0.58           N  \\nATOM    982  CA  PRO A 122     -38.351  18.237  -7.394  1.00  0.57           C  \\nATOM    983  C   PRO A 122     -38.436  17.130  -8.442  1.00  0.59           C  \\nATOM    984  CB  PRO A 122     -39.663  19.022  -7.322  1.00  0.54           C  \\nATOM    985  O   PRO A 122     -38.750  17.398  -9.604  1.00  0.56           O  \\nATOM    986  CG  PRO A 122     -39.504  20.126  -8.316  1.00  0.52           C  \\nATOM    987  CD  PRO A 122     -38.043  20.451  -8.438  1.00  0.53           C  \\nATOM    988  N   PHE A 123     -37.513  16.039  -8.506  1.00  0.59           N  \\nATOM    989  CA  PHE A 123     -38.024  14.825  -9.132  1.00  0.61           C  \\nATOM    990  C   PHE A 123     -39.546  14.783  -9.068  1.00  0.60           C  \\nATOM    991  CB  PHE A 123     -37.435  13.582  -8.457  1.00  0.55           C  \\nATOM    992  O   PHE A 123     -40.132  14.925  -7.993  1.00  0.58           O  \\nATOM    993  CG  PHE A 123     -36.085  13.184  -8.989  1.00  0.53           C  \\nATOM    994  CD1 PHE A 123     -35.976  12.417 -10.142  1.00  0.50           C  \\nATOM    995  CD2 PHE A 123     -34.925  13.579  -8.336  1.00  0.51           C  \\nATOM    996  CE1 PHE A 123     -34.727  12.047 -10.637  1.00  0.50           C  \\nATOM    997  CE2 PHE A 123     -33.673  13.213  -8.825  1.00  0.50           C  \\nATOM    998  CZ  PHE A 123     -33.577  12.447  -9.975  1.00  0.46           C  \\nATOM    999  N   PRO A 124     -40.235  15.115 -10.089  1.00  0.58           N  \\nATOM   1000  CA  PRO A 124     -41.640  14.702 -10.056  1.00  0.58           C  \\nATOM   1001  C   PRO A 124     -41.847  13.373  -9.334  1.00  0.58           C  \\nATOM   1002  CB  PRO A 124     -42.006  14.581 -11.538  1.00  0.54           C  \\nATOM   1003  O   PRO A 124     -41.454  12.321  -9.844  1.00  0.56           O  \\nATOM   1004  CG  PRO A 124     -40.695  14.469 -12.247  1.00  0.53           C  \\nATOM   1005  CD  PRO A 124     -39.606  14.906 -11.310  1.00  0.55           C  \\nATOM   1006  N   GLY A 125     -42.012  13.437  -8.013  1.00  0.56           N  \\nATOM   1007  CA  GLY A 125     -42.662  12.444  -7.172  1.00  0.56           C  \\nATOM   1008  C   GLY A 125     -42.099  12.393  -5.764  1.00  0.55           C  \\nATOM   1009  O   GLY A 125     -42.450  11.508  -4.981  1.00  0.53           O  \\nATOM   1010  N   MET A 126     -41.263  13.364  -5.352  1.00  0.57           N  \\nATOM   1011  CA  MET A 126     -41.118  13.526  -3.908  1.00  0.59           C  \\nATOM   1012  C   MET A 126     -41.855  14.770  -3.424  1.00  0.56           C  \\nATOM   1013  CB  MET A 126     -39.640  13.610  -3.522  1.00  0.52           C  \\nATOM   1014  O   MET A 126     -41.706  15.849  -4.000  1.00  0.54           O  \\nATOM   1015  CG  MET A 126     -38.977  12.255  -3.335  1.00  0.51           C  \\nATOM   1016  SD  MET A 126     -37.300  12.388  -2.602  1.00  0.55           S  \\nATOM   1017  CE  MET A 126     -37.384  11.060  -1.368  1.00  0.45           C  \\nATOM   1018  N   PRO A 127     -42.951  14.642  -2.736  1.00  0.59           N  \\nATOM   1019  CA  PRO A 127     -43.554  15.736  -1.971  1.00  0.58           C  \\nATOM   1020  C   PRO A 127     -42.515  16.632  -1.301  1.00  0.59           C  \\nATOM   1021  CB  PRO A 127     -44.403  15.008  -0.925  1.00  0.55           C  \\nATOM   1022  O   PRO A 127     -41.437  16.161  -0.929  1.00  0.57           O  \\nATOM   1023  CG  PRO A 127     -43.737  13.682  -0.748  1.00  0.53           C  \\nATOM   1024  CD  PRO A 127     -43.014  13.338  -2.019  1.00  0.54           C  \\nATOM   1025  N   PRO A 128     -42.466  17.826  -1.689  1.00  0.60           N  \\nATOM   1026  CA  PRO A 128     -41.705  18.850  -0.971  1.00  0.59           C  \\nATOM   1027  C   PRO A 128     -41.374  18.443   0.463  1.00  0.60           C  \\nATOM   1028  CB  PRO A 128     -42.639  20.063  -0.992  1.00  0.56           C  \\nATOM   1029  O   PRO A 128     -42.277  18.132   1.244  1.00  0.58           O  \\nATOM   1030  CG  PRO A 128     -44.014  19.484  -1.080  1.00  0.54           C  \\nATOM   1031  CD  PRO A 128     -43.930  18.158  -1.780  1.00  0.55           C  \\nATOM   1032  N   MET A 129     -40.132  18.097   0.854  1.00  0.63           N  \\nATOM   1033  CA  MET A 129     -39.705  18.071   2.250  1.00  0.64           C  \\nATOM   1034  C   MET A 129     -40.043  19.385   2.947  1.00  0.63           C  \\nATOM   1035  CB  MET A 129     -38.203  17.798   2.350  1.00  0.59           C  \\nATOM   1036  O   MET A 129     -39.793  20.462   2.404  1.00  0.60           O  \\nATOM   1037  CG  MET A 129     -37.828  16.344   2.112  1.00  0.56           C  \\nATOM   1038  SD  MET A 129     -37.508  15.436   3.674  1.00  0.55           S  \\nATOM   1039  CE  MET A 129     -35.697  15.524   3.730  1.00  0.47           C  \\nATOM   1040  N   PRO A 130     -40.975  19.501   3.845  1.00  0.62           N  \\nATOM   1041  CA  PRO A 130     -41.283  20.629   4.727  1.00  0.61           C  \\nATOM   1042  C   PRO A 130     -40.032  21.274   5.319  1.00  0.61           C  \\nATOM   1043  CB  PRO A 130     -42.142  19.995   5.824  1.00  0.57           C  \\nATOM   1044  O   PRO A 130     -39.176  20.577   5.871  1.00  0.59           O  \\nATOM   1045  CG  PRO A 130     -41.702  18.567   5.876  1.00  0.55           C  \\nATOM   1046  CD  PRO A 130     -41.165  18.184   4.527  1.00  0.56           C  \\nATOM   1047  N   SER A 131     -39.570  22.483   4.876  1.00  0.62           N  \\nATOM   1048  CA  SER A 131     -38.798  23.535   5.530  1.00  0.62           C  \\nATOM   1049  C   SER A 131     -39.110  23.604   7.021  1.00  0.61           C  \\nATOM   1050  CB  SER A 131     -39.077  24.890   4.879  1.00  0.56           C  \\nATOM   1051  O   SER A 131     -40.130  24.168   7.422  1.00  0.58           O  \\nATOM   1052  OG  SER A 131     -40.166  24.798   3.976  1.00  0.57           O  \\nATOM   1053  N   GLY A 132     -38.800  22.562   7.736  1.00  0.59           N  \\nATOM   1054  CA  GLY A 132     -38.781  22.705   9.183  1.00  0.59           C  \\nATOM   1055  C   GLY A 132     -37.421  23.098   9.727  1.00  0.58           C  \\nATOM   1056  O   GLY A 132     -36.392  22.781   9.127  1.00  0.55           O  \\nATOM   1057  N   PRO A 133     -37.348  24.247  10.482  1.00  0.61           N  \\nATOM   1058  CA  PRO A 133     -36.146  24.673  11.202  1.00  0.60           C  \\nATOM   1059  C   PRO A 133     -35.239  23.504  11.580  1.00  0.61           C  \\nATOM   1060  CB  PRO A 133     -36.707  25.355  12.452  1.00  0.56           C  \\nATOM   1061  O   PRO A 133     -35.727  22.413  11.886  1.00  0.58           O  \\nATOM   1062  CG  PRO A 133     -38.034  24.706  12.678  1.00  0.54           C  \\nATOM   1063  CD  PRO A 133     -38.581  24.263  11.351  1.00  0.56           C  \\nATOM   1064  N   PRO A 134     -34.033  23.416  11.043  1.00  0.58           N  \\nATOM   1065  CA  PRO A 134     -33.050  22.452  11.544  1.00  0.58           C  \\nATOM   1066  C   PRO A 134     -33.266  22.099  13.014  1.00  0.58           C  \\nATOM   1067  CB  PRO A 134     -31.715  23.174  11.345  1.00  0.54           C  \\nATOM   1068  O   PRO A 134     -33.598  22.973  13.820  1.00  0.56           O  \\nATOM   1069  CG  PRO A 134     -32.058  24.628  11.383  1.00  0.52           C  \\nATOM   1070  CD  PRO A 134     -33.474  24.797  10.911  1.00  0.54           C  \\nATOM   1071  N   PRO A 135     -33.496  20.928  13.373  1.00  0.58           N  \\nATOM   1072  CA  PRO A 135     -33.553  20.533  14.782  1.00  0.57           C  \\nATOM   1073  C   PRO A 135     -32.371  21.062  15.592  1.00  0.58           C  \\nATOM   1074  CB  PRO A 135     -33.530  19.003  14.721  1.00  0.53           C  \\nATOM   1075  O   PRO A 135     -31.263  21.186  15.064  1.00  0.56           O  \\nATOM   1076  CG  PRO A 135     -32.895  18.683  13.406  1.00  0.51           C  \\nATOM   1077  CD  PRO A 135     -33.088  19.854  12.486  1.00  0.53           C  \\nATOM   1078  N   PRO A 136     -32.578  21.910  16.754  1.00  0.56           N  \\nATOM   1079  CA  PRO A 136     -31.646  22.371  17.786  1.00  0.55           C  \\nATOM   1080  C   PRO A 136     -30.528  21.368  18.061  1.00  0.55           C  \\nATOM   1081  CB  PRO A 136     -32.540  22.543  19.017  1.00  0.51           C  \\nATOM   1082  O   PRO A 136     -30.798  20.198  18.345  1.00  0.52           O  \\nATOM   1083  CG  PRO A 136     -33.743  21.702  18.736  1.00  0.49           C  \\nATOM   1084  CD  PRO A 136     -33.792  21.416  17.262  1.00  0.52           C  \\nATOM   1085  N   SER A 137     -29.350  21.370  17.388  1.00  0.56           N  \\nATOM   1086  CA  SER A 137     -27.984  21.057  17.796  1.00  0.57           C  \\nATOM   1087  C   SER A 137     -27.955  20.400  19.172  1.00  0.55           C  \\nATOM   1088  CB  SER A 137     -27.124  22.321  17.807  1.00  0.51           C  \\nATOM   1089  O   SER A 137     -28.285  21.035  20.175  1.00  0.53           O  \\nATOM   1090  OG  SER A 137     -27.226  23.010  16.572  1.00  0.52           O  \\nATOM   1091  N   MET A 138     -28.383  19.249  19.280  1.00  0.55           N  \\nATOM   1092  CA  MET A 138     -28.113  18.480  20.492  1.00  0.56           C  \\nATOM   1093  C   MET A 138     -26.623  18.484  20.818  1.00  0.54           C  \\nATOM   1094  CB  MET A 138     -28.611  17.041  20.339  1.00  0.49           C  \\nATOM   1095  O   MET A 138     -25.814  17.957  20.051  1.00  0.51           O  \\nATOM   1096  CG  MET A 138     -30.126  16.916  20.323  1.00  0.47           C  \\nATOM   1097  SD  MET A 138     -30.706  15.321  21.022  1.00  0.53           S  \\nATOM   1098  CE  MET A 138     -32.419  15.315  20.425  1.00  0.42           C  \\nATOM   1099  N   ALA A 139     -26.072  19.504  21.490  1.00  0.56           N  \\nATOM   1100  CA  ALA A 139     -25.014  19.672  22.483  1.00  0.57           C  \\nATOM   1101  C   ALA A 139     -24.686  18.346  23.165  1.00  0.56           C  \\nATOM   1102  CB  ALA A 139     -25.420  20.714  23.522  1.00  0.52           C  \\nATOM   1103  O   ALA A 139     -25.444  17.875  24.016  1.00  0.54           O  \\nATOM   1104  N   TYR A 140     -24.119  17.354  22.479  1.00  0.56           N  \\nATOM   1105  CA  TYR A 140     -23.419  16.328  23.243  1.00  0.58           C  \\nATOM   1106  C   TYR A 140     -22.102  16.862  23.794  1.00  0.55           C  \\nATOM   1107  CB  TYR A 140     -23.158  15.094  22.374  1.00  0.51           C  \\nATOM   1108  O   TYR A 140     -21.240  17.311  23.035  1.00  0.52           O  \\nATOM   1109  CG  TYR A 140     -24.349  14.175  22.251  1.00  0.50           C  \\nATOM   1110  CD1 TYR A 140     -24.599  13.195  23.210  1.00  0.47           C  \\nATOM   1111  CD2 TYR A 140     -25.226  14.282  21.177  1.00  0.48           C  \\nATOM   1112  CE1 TYR A 140     -25.694  12.345  23.101  1.00  0.45           C  \\nATOM   1113  CE2 TYR A 140     -26.324  13.437  21.058  1.00  0.47           C  \\nATOM   1114  OH  TYR A 140     -27.636  11.634  21.910  1.00  0.40           O  \\nATOM   1115  CZ  TYR A 140     -26.550  12.473  22.023  1.00  0.41           C  \\nATOM   1116  N   GLY A 141     -22.118  17.402  25.057  1.00  0.55           N  \\nATOM   1117  CA  GLY A 141     -21.154  17.579  26.131  1.00  0.55           C  \\nATOM   1118  C   GLY A 141     -19.882  16.778  25.928  1.00  0.55           C  \\nATOM   1119  O   GLY A 141     -19.900  15.547  26.000  1.00  0.53           O  \\nATOM   1120  N   MET A 142     -18.880  17.137  25.128  1.00  0.56           N  \\nATOM   1121  CA  MET A 142     -17.485  16.810  25.409  1.00  0.58           C  \\nATOM   1122  C   MET A 142     -17.197  16.889  26.904  1.00  0.56           C  \\nATOM   1123  CB  MET A 142     -16.548  17.749  24.646  1.00  0.53           C  \\nATOM   1124  O   MET A 142     -17.542  17.877  27.556  1.00  0.55           O  \\nATOM   1125  CG  MET A 142     -16.435  17.431  23.164  1.00  0.52           C  \\nATOM   1126  SD  MET A 142     -14.995  18.252  22.376  1.00  0.54           S  \\nATOM   1127  CE  MET A 142     -14.160  16.812  21.656  1.00  0.46           C  \\nATOM   1128  N   PRO A 143     -17.124  15.712  27.465  1.00  0.59           N  \\nATOM   1129  CA  PRO A 143     -16.530  15.828  28.799  1.00  0.58           C  \\nATOM   1130  C   PRO A 143     -15.313  16.751  28.825  1.00  0.59           C  \\nATOM   1131  CB  PRO A 143     -16.130  14.389  29.133  1.00  0.55           C  \\nATOM   1132  O   PRO A 143     -14.580  16.839  27.837  1.00  0.56           O  \\nATOM   1133  CG  PRO A 143     -15.929  13.730  27.806  1.00  0.53           C  \\nATOM   1134  CD  PRO A 143     -16.759  14.453  26.785  1.00  0.55           C  \\nATOM   1135  N   PRO A 144     -15.434  17.782  29.666  1.00  0.59           N  \\nATOM   1136  CA  PRO A 144     -14.327  18.710  29.907  1.00  0.58           C  \\nATOM   1137  C   PRO A 144     -12.967  18.017  29.915  1.00  0.59           C  \\nATOM   1138  CB  PRO A 144     -14.650  19.294  31.285  1.00  0.54           C  \\nATOM   1139  O   PRO A 144     -12.777  17.029  30.630  1.00  0.57           O  \\nATOM   1140  CG  PRO A 144     -15.446  18.232  31.973  1.00  0.52           C  \\nATOM   1141  CD  PRO A 144     -16.159  17.420  30.930  1.00  0.54           C  \\nATOM   1142  N   MET A 145     -12.119  18.067  28.997  1.00  0.61           N  \\nATOM   1143  CA  MET A 145     -10.680  17.826  29.049  1.00  0.63           C  \\nATOM   1144  C   MET A 145     -10.091  18.328  30.362  1.00  0.61           C  \\nATOM   1145  CB  MET A 145      -9.979  18.502  27.869  1.00  0.58           C  \\nATOM   1146  O   MET A 145     -10.432  19.418  30.825  1.00  0.59           O  \\nATOM   1147  CG  MET A 145     -10.215  17.809  26.537  1.00  0.56           C  \\nATOM   1148  SD  MET A 145      -8.807  16.743  26.040  1.00  0.55           S  \\nATOM   1149  CE  MET A 145      -7.945  17.860  24.900  1.00  0.47           C  \\nATOM   1150  N   PRO A 146      -9.839  17.436  31.219  1.00  0.58           N  \\nATOM   1151  CA  PRO A 146      -9.122  17.800  32.443  1.00  0.58           C  \\nATOM   1152  C   PRO A 146      -8.104  18.917  32.221  1.00  0.58           C  \\nATOM   1153  CB  PRO A 146      -8.424  16.497  32.842  1.00  0.54           C  \\nATOM   1154  O   PRO A 146      -7.331  18.870  31.261  1.00  0.56           O  \\nATOM   1155  CG  PRO A 146      -8.339  15.706  31.577  1.00  0.53           C  \\nATOM   1156  CD  PRO A 146      -9.359  16.239  30.612  1.00  0.55           C  \\nATOM   1157  N   SER A 147      -8.320  20.176  32.735  1.00  0.59           N  \\nATOM   1158  CA  SER A 147      -7.470  21.320  33.046  1.00  0.59           C  \\nATOM   1159  C   SER A 147      -6.090  20.871  33.515  1.00  0.58           C  \\nATOM   1160  CB  SER A 147      -8.121  22.198  34.115  1.00  0.54           C  \\nATOM   1161  O   SER A 147      -5.946  20.344  34.620  1.00  0.55           O  \\nATOM   1162  OG  SER A 147      -9.455  22.516  33.760  1.00  0.57           O  \\nATOM   1163  N   GLY A 148      -5.145  20.458  32.557  1.00  0.56           N  \\nATOM   1164  CA  GLY A 148      -3.736  20.484  32.915  1.00  0.56           C  \\nATOM   1165  C   GLY A 148      -2.839  19.872  31.854  1.00  0.55           C  \\nATOM   1166  O   GLY A 148      -1.651  19.649  32.092  1.00  0.52           O  \\nATOM   1167  N   MET A 149      -3.210  19.911  30.584  1.00  0.57           N  \\nATOM   1168  CA  MET A 149      -2.117  19.693  29.641  1.00  0.58           C  \\nATOM   1169  C   MET A 149      -2.260  20.599  28.423  1.00  0.56           C  \\nATOM   1170  CB  MET A 149      -2.069  18.229  29.201  1.00  0.51           C  \\nATOM   1171  O   MET A 149      -3.128  20.377  27.577  1.00  0.53           O  \\nATOM   1172  CG  MET A 149      -1.144  17.365  30.043  1.00  0.49           C  \\nATOM   1173  SD  MET A 149      -1.263  15.584  29.617  1.00  0.53           S  \\nATOM   1174  CE  MET A 149       0.472  15.091  29.816  1.00  0.43           C  \\nATOM   1175  N   MET A 150      -1.913  21.905  28.535  1.00  0.56           N  \\nATOM   1176  CA  MET A 150      -1.523  22.813  27.460  1.00  0.57           C  \\nATOM   1177  C   MET A 150      -0.886  22.047  26.305  1.00  0.56           C  \\nATOM   1178  CB  MET A 150      -0.554  23.877  27.979  1.00  0.52           C  \\nATOM   1179  O   MET A 150       0.063  21.287  26.507  1.00  0.54           O  \\nATOM   1180  CG  MET A 150      -1.233  25.005  28.739  1.00  0.51           C  \\nATOM   1181  SD  MET A 150      -0.158  26.483  28.912  1.00  0.52           S  \\nATOM   1182  CE  MET A 150      -1.299  27.775  28.346  1.00  0.43           C  \\nATOM   1183  N   PRO A 151      -1.602  21.764  25.281  1.00  0.58           N  \\nATOM   1184  CA  PRO A 151      -0.922  21.382  24.041  1.00  0.58           C  \\nATOM   1185  C   PRO A 151       0.220  22.328  23.678  1.00  0.58           C  \\nATOM   1186  CB  PRO A 151      -2.037  21.445  22.993  1.00  0.54           C  \\nATOM   1187  O   PRO A 151       0.113  23.539  23.887  1.00  0.56           O  \\nATOM   1188  CG  PRO A 151      -2.994  22.467  23.516  1.00  0.52           C  \\nATOM   1189  CD  PRO A 151      -2.904  22.482  25.015  1.00  0.54           C  \\nATOM   1190  N   PRO A 152       1.377  21.875  23.753  1.00  0.57           N  \\nATOM   1191  CA  PRO A 152       2.507  22.701  23.322  1.00  0.56           C  \\nATOM   1192  C   PRO A 152       2.163  23.597  22.134  1.00  0.57           C  \\nATOM   1193  CB  PRO A 152       3.571  21.669  22.937  1.00  0.52           C  \\nATOM   1194  O   PRO A 152       1.550  23.139  21.167  1.00  0.55           O  \\nATOM   1195  CG  PRO A 152       2.806  20.414  22.667  1.00  0.51           C  \\nATOM   1196  CD  PRO A 152       1.469  20.515  23.344  1.00  0.53           C  \\nATOM   1197  N   ARG A 153       2.021  24.880  22.303  1.00  0.58           N  \\nATOM   1198  CA  ARG A 153       2.160  26.017  21.399  1.00  0.58           C  \\nATOM   1199  C   ARG A 153       3.149  25.709  20.280  1.00  0.57           C  \\nATOM   1200  CB  ARG A 153       2.607  27.263  22.167  1.00  0.54           C  \\nATOM   1201  O   ARG A 153       4.359  25.657  20.511  1.00  0.55           O  \\nATOM   1202  CG  ARG A 153       1.463  28.169  22.591  1.00  0.53           C  \\nATOM   1203  CD  ARG A 153       1.965  29.412  23.313  1.00  0.55           C  \\nATOM   1204  NE  ARG A 153       0.876  30.134  23.963  1.00  0.45           N  \\nATOM   1205  NH1 ARG A 153       1.925  32.192  23.897  1.00  0.34           N  \\nATOM   1206  NH2 ARG A 153      -0.163  31.994  24.821  1.00  0.29           N  \\nATOM   1207  CZ  ARG A 153       0.882  31.438  24.226  1.00  0.48           C  \\nATOM   1208  N   GLY A 154       2.737  24.940  19.098  1.00  0.53           N  \\nATOM   1209  CA  GLY A 154       3.569  24.998  17.907  1.00  0.54           C  \\nATOM   1210  C   GLY A 154       3.316  23.853  16.946  1.00  0.53           C  \\nATOM   1211  O   GLY A 154       3.879  22.768  17.102  1.00  0.50           O  \\nATOM   1212  N   MET A 155       2.017  23.658  16.508  1.00  0.55           N  \\nATOM   1213  CA  MET A 155       1.837  23.146  15.152  1.00  0.57           C  \\nATOM   1214  C   MET A 155       0.690  23.864  14.447  1.00  0.55           C  \\nATOM   1215  CB  MET A 155       1.575  21.639  15.176  1.00  0.50           C  \\nATOM   1216  O   MET A 155      -0.442  23.860  14.933  1.00  0.53           O  \\nATOM   1217  CG  MET A 155       2.789  20.812  15.564  1.00  0.49           C  \\nATOM   1218  SD  MET A 155       2.739  19.117  14.863  1.00  0.52           S  \\nATOM   1219  CE  MET A 155       4.277  18.441  15.547  1.00  0.43           C  \\nATOM   1220  N   PRO A 156       0.903  24.811  13.629  1.00  0.55           N  \\nATOM   1221  CA  PRO A 156       0.079  25.478  12.619  1.00  0.55           C  \\nATOM   1222  C   PRO A 156      -0.575  24.496  11.649  1.00  0.54           C  \\nATOM   1223  CB  PRO A 156       1.076  26.382  11.891  1.00  0.49           C  \\nATOM   1224  O   PRO A 156       0.122  23.780  10.926  1.00  0.51           O  \\nATOM   1225  CG  PRO A 156       2.419  25.819  12.227  1.00  0.48           C  \\nATOM   1226  CD  PRO A 156       2.267  24.893  13.400  1.00  0.52           C  \\nATOM   1227  N   GLY A 157      -1.815  23.733  12.007  1.00  0.52           N  \\nATOM   1228  CA  GLY A 157      -2.762  23.365  10.967  1.00  0.53           C  \\nATOM   1229  C   GLY A 157      -3.493  22.068  11.259  1.00  0.51           C  \\nATOM   1230  O   GLY A 157      -2.950  20.981  11.046  1.00  0.49           O  \\nATOM   1231  N   ALA A 158      -4.300  22.074  12.263  1.00  0.53           N  \\nATOM   1232  CA  ALA A 158      -5.440  21.166  12.354  1.00  0.54           C  \\nATOM   1233  C   ALA A 158      -6.751  21.905  12.098  1.00  0.53           C  \\nATOM   1234  CB  ALA A 158      -5.474  20.489  13.722  1.00  0.48           C  \\nATOM   1235  O   ALA A 158      -7.177  22.727  12.913  1.00  0.51           O  \\nATOM   1236  N   TYR A 159      -7.073  22.195  10.778  1.00  0.55           N  \\nATOM   1237  CA  TYR A 159      -8.399  22.590  10.318  1.00  0.56           C  \\nATOM   1238  C   TYR A 159      -9.464  21.631  10.836  1.00  0.55           C  \\nATOM   1239  CB  TYR A 159      -8.445  22.642   8.788  1.00  0.51           C  \\nATOM   1240  O   TYR A 159      -9.328  20.412  10.705  1.00  0.53           O  \\nATOM   1241  CG  TYR A 159      -7.997  23.964   8.213  1.00  0.50           C  \\nATOM   1242  CD1 TYR A 159      -8.892  25.022   8.069  1.00  0.47           C  \\nATOM   1243  CD2 TYR A 159      -6.680  24.158   7.811  1.00  0.48           C  \\nATOM   1244  CE1 TYR A 159      -8.485  26.241   7.537  1.00  0.46           C  \\nATOM   1245  CE2 TYR A 159      -6.261  25.372   7.278  1.00  0.48           C  \\nATOM   1246  OH  TYR A 159      -6.761  27.612   6.619  1.00  0.44           O  \\nATOM   1247  CZ  TYR A 159      -7.169  26.407   7.146  1.00  0.43           C  \\nATOM   1248  N   PRO A 160     -10.351  22.103  11.762  1.00  0.54           N  \\nATOM   1249  CA  PRO A 160     -11.607  21.424  12.091  1.00  0.53           C  \\nATOM   1250  C   PRO A 160     -12.431  21.076  10.853  1.00  0.54           C  \\nATOM   1251  CB  PRO A 160     -12.340  22.443  12.966  1.00  0.50           C  \\nATOM   1252  O   PRO A 160     -12.465  21.849   9.892  1.00  0.52           O  \\nATOM   1253  CG  PRO A 160     -11.825  23.772  12.518  1.00  0.48           C  \\nATOM   1254  CD  PRO A 160     -10.424  23.594  12.008  1.00  0.51           C  \\nATOM   1255  N   PRO A 161     -12.791  19.853  10.488  1.00  0.52           N  \\nATOM   1256  CA  PRO A 161     -13.719  19.473   9.420  1.00  0.52           C  \\nATOM   1257  C   PRO A 161     -14.911  20.420   9.310  1.00  0.53           C  \\nATOM   1258  CB  PRO A 161     -14.172  18.070   9.830  1.00  0.48           C  \\nATOM   1259  O   PRO A 161     -15.395  20.930  10.324  1.00  0.51           O  \\nATOM   1260  CG  PRO A 161     -14.027  18.040  11.317  1.00  0.47           C  \\nATOM   1261  CD  PRO A 161     -12.937  18.996  11.709  1.00  0.50           C  \\nATOM   1262  N   PRO A 162     -15.148  21.157   8.135  1.00  0.52           N  \\nATOM   1263  CA  PRO A 162     -16.356  21.936   7.853  1.00  0.52           C  \\nATOM   1264  C   PRO A 162     -17.615  21.308   8.444  1.00  0.52           C  \\nATOM   1265  CB  PRO A 162     -16.418  21.948   6.323  1.00  0.48           C  \\nATOM   1266  O   PRO A 162     -17.793  20.089   8.374  1.00  0.50           O  \\nATOM   1267  CG  PRO A 162     -15.495  20.853   5.895  1.00  0.46           C  \\nATOM   1268  CD  PRO A 162     -14.639  20.465   7.066  1.00  0.48           C  \\nATOM   1269  N   ARG A 163     -18.229  21.812   9.515  1.00  0.50           N  \\nATOM   1270  CA  ARG A 163     -19.591  21.769  10.039  1.00  0.50           C  \\nATOM   1271  C   ARG A 163     -20.586  21.392   8.947  1.00  0.49           C  \\nATOM   1272  CB  ARG A 163     -19.973  23.117  10.654  1.00  0.46           C  \\nATOM   1273  O   ARG A 163     -20.810  22.162   8.011  1.00  0.47           O  \\nATOM   1274  CG  ARG A 163     -19.414  23.337  12.050  1.00  0.45           C  \\nATOM   1275  CD  ARG A 163     -19.883  24.659  12.643  1.00  0.48           C  \\nATOM   1276  NE  ARG A 163     -19.107  25.025  13.825  1.00  0.43           N  \\nATOM   1277  NH1 ARG A 163     -20.797  26.255  14.808  1.00  0.33           N  \\nATOM   1278  NH2 ARG A 163     -18.769  26.046  15.854  1.00  0.28           N  \\nATOM   1279  CZ  ARG A 163     -19.559  25.774  14.826  1.00  0.42           C  \\nATOM   1280  N   GLY A 164     -20.726  20.193   8.494  1.00  0.49           N  \\nATOM   1281  CA  GLY A 164     -21.865  19.599   7.813  1.00  0.49           C  \\nATOM   1282  C   GLY A 164     -21.526  18.302   7.103  1.00  0.48           C  \\nATOM   1283  O   GLY A 164     -22.358  17.747   6.382  1.00  0.46           O  \\nATOM   1284  N   TYR A 165     -20.373  17.610   7.553  1.00  0.45           N  \\nATOM   1285  CA  TYR A 165     -20.113  16.182   7.414  1.00  0.46           C  \\nATOM   1286  C   TYR A 165     -20.093  15.496   8.775  1.00  0.45           C  \\nATOM   1287  CB  TYR A 165     -18.784  15.945   6.690  1.00  0.42           C  \\nATOM   1288  O   TYR A 165     -19.551  16.038   9.742  1.00  0.43           O  \\nATOM   1289  CG  TYR A 165     -18.901  15.962   5.186  1.00  0.41           C  \\nATOM   1290  CD1 TYR A 165     -19.192  14.799   4.476  1.00  0.40           C  \\nATOM   1291  CD2 TYR A 165     -18.719  17.141   4.470  1.00  0.41           C  \\nATOM   1292  CE1 TYR A 165     -19.299  14.810   3.089  1.00  0.41           C  \\nATOM   1293  CE2 TYR A 165     -18.823  17.164   3.083  1.00  0.42           C  \\nATOM   1294  OH  TYR A 165     -19.218  16.012   1.030  1.00  0.38           O  \\nATOM   1295  CZ  TYR A 165     -19.113  15.996   2.403  1.00  0.37           C  \\nATOM   1296  N   PRO A 166     -21.047  14.785   9.177  1.00  0.48           N  \\nATOM   1297  CA  PRO A 166     -20.774  13.958  10.355  1.00  0.48           C  \\nATOM   1298  C   PRO A 166     -19.283  13.846  10.663  1.00  0.49           C  \\nATOM   1299  CB  PRO A 166     -21.357  12.596   9.971  1.00  0.45           C  \\nATOM   1300  O   PRO A 166     -18.456  13.860   9.748  1.00  0.47           O  \\nATOM   1301  CG  PRO A 166     -21.331  12.576   8.477  1.00  0.43           C  \\nATOM   1302  CD  PRO A 166     -21.353  13.993   7.981  1.00  0.46           C  \\nATOM   1303  N   PRO A 167     -18.803  14.462  11.794  1.00  0.47           N  \\nATOM   1304  CA  PRO A 167     -17.417  14.228  12.209  1.00  0.46           C  \\nATOM   1305  C   PRO A 167     -16.779  13.041  11.491  1.00  0.47           C  \\nATOM   1306  CB  PRO A 167     -17.540  13.957  13.710  1.00  0.44           C  \\nATOM   1307  O   PRO A 167     -17.429  12.010  11.297  1.00  0.45           O  \\nATOM   1308  CG  PRO A 167     -18.943  13.477  13.899  1.00  0.41           C  \\nATOM   1309  CD  PRO A 167     -19.785  14.015  12.778  1.00  0.43           C  \\nATOM   1310  N   ALA A 168     -15.861  13.171  10.499  1.00  0.46           N  \\nATOM   1311  CA  ALA A 168     -14.973  12.062  10.159  1.00  0.46           C  \\nATOM   1312  C   ALA A 168     -14.881  11.061  11.306  1.00  0.46           C  \\nATOM   1313  CB  ALA A 168     -13.584  12.585   9.800  1.00  0.44           C  \\nATOM   1314  O   ALA A 168     -14.880  11.447  12.477  1.00  0.44           O  \\nATOM   1315  N   PRO A 169     -15.451   9.914  11.333  1.00  0.48           N  \\nATOM   1316  CA  PRO A 169     -15.144   8.924  12.368  1.00  0.47           C  \\nATOM   1317  C   PRO A 169     -13.769   9.137  12.998  1.00  0.48           C  \\nATOM   1318  CB  PRO A 169     -15.197   7.595  11.610  1.00  0.44           C  \\nATOM   1319  O   PRO A 169     -12.821   9.517  12.307  1.00  0.46           O  \\nATOM   1320  CG  PRO A 169     -14.963   7.960  10.180  1.00  0.43           C  \\nATOM   1321  CD  PRO A 169     -15.338   9.402   9.989  1.00  0.45           C  \\nATOM   1322  N   ALA A 170     -13.557   9.805  14.305  1.00  0.50           N  \\nATOM   1323  CA  ALA A 170     -12.339   9.853  15.109  1.00  0.51           C  \\nATOM   1324  C   ALA A 170     -11.245   8.980  14.499  1.00  0.50           C  \\nATOM   1325  CB  ALA A 170     -12.629   9.413  16.541  1.00  0.48           C  \\nATOM   1326  O   ALA A 170     -11.518   7.879  14.017  1.00  0.49           O  \\nATOM   1327  N   PRO A 171     -10.155   9.381  13.841  1.00  0.51           N  \\nATOM   1328  CA  PRO A 171      -9.030   8.506  13.502  1.00  0.51           C  \\nATOM   1329  C   PRO A 171      -8.923   7.297  14.429  1.00  0.51           C  \\nATOM   1330  CB  PRO A 171      -7.812   9.421  13.656  1.00  0.48           C  \\nATOM   1331  O   PRO A 171      -8.693   7.454  15.631  1.00  0.49           O  \\nATOM   1332  CG  PRO A 171      -8.322  10.613  14.399  1.00  0.47           C  \\nATOM   1333  CD  PRO A 171      -9.823  10.570  14.398  1.00  0.50           C  \\nATOM   1334  N   GLY A 172      -9.912   6.302  14.245  1.00  0.47           N  \\nATOM   1335  CA  GLY A 172      -9.931   4.984  14.860  1.00  0.48           C  \\nATOM   1336  C   GLY A 172     -11.313   4.358  14.885  1.00  0.47           C  \\nATOM   1337  O   GLY A 172     -11.503   3.286  15.462  1.00  0.44           O  \\nATOM   1338  N   VAL A 173     -12.275   4.798  14.048  1.00  0.50           N  \\nATOM   1339  CA  VAL A 173     -13.457   4.000  13.739  1.00  0.50           C  \\nATOM   1340  C   VAL A 173     -13.561   3.793  12.229  1.00  0.50           C  \\nATOM   1341  CB  VAL A 173     -14.746   4.663  14.275  1.00  0.47           C  \\nATOM   1342  O   VAL A 173     -14.014   4.682  11.504  1.00  0.48           O  \\nATOM   1343  CG1 VAL A 173     -15.949   3.743  14.076  1.00  0.43           C  \\nATOM   1344  CG2 VAL A 173     -14.584   5.025  15.751  1.00  0.44           C  \\nATOM   1345  N   TYR A 174     -12.676   3.051  11.561  1.00  0.55           N  \\nATOM   1346  CA  TYR A 174     -12.944   2.395  10.286  1.00  0.56           C  \\nATOM   1347  C   TYR A 174     -14.374   1.869  10.233  1.00  0.55           C  \\nATOM   1348  CB  TYR A 174     -11.957   1.248  10.054  1.00  0.51           C  \\nATOM   1349  O   TYR A 174     -14.727   0.940  10.963  1.00  0.53           O  \\nATOM   1350  CG  TYR A 174     -10.923   1.541   8.994  1.00  0.49           C  \\nATOM   1351  CD1 TYR A 174     -11.164   1.240   7.656  1.00  0.46           C  \\nATOM   1352  CD2 TYR A 174      -9.703   2.120   9.329  1.00  0.47           C  \\nATOM   1353  CE1 TYR A 174     -10.214   1.507   6.676  1.00  0.45           C  \\nATOM   1354  CE2 TYR A 174      -8.745   2.391   8.357  1.00  0.48           C  \\nATOM   1355  OH  TYR A 174      -8.065   2.349   6.070  1.00  0.40           O  \\nATOM   1356  CZ  TYR A 174      -9.009   2.082   7.036  1.00  0.41           C  \\nATOM   1357  N   MET A 175     -15.371   2.660   9.910  1.00  0.52           N  \\nATOM   1358  CA  MET A 175     -16.585   2.087   9.336  1.00  0.53           C  \\nATOM   1359  C   MET A 175     -16.260   1.245   8.106  1.00  0.52           C  \\nATOM   1360  CB  MET A 175     -17.579   3.189   8.968  1.00  0.49           C  \\nATOM   1361  O   MET A 175     -15.640   1.734   7.161  1.00  0.51           O  \\nATOM   1362  CG  MET A 175     -18.276   3.812  10.167  1.00  0.48           C  \\nATOM   1363  SD  MET A 175     -19.795   4.729   9.700  1.00  0.52           S  \\nATOM   1364  CE  MET A 175     -20.999   3.372   9.696  1.00  0.44           C  \\nATOM   1365  N   PRO A 176     -16.195  -0.029   8.165  1.00  0.54           N  \\nATOM   1366  CA  PRO A 176     -16.190  -0.813   6.928  1.00  0.53           C  \\nATOM   1367  C   PRO A 176     -17.214  -0.315   5.910  1.00  0.54           C  \\nATOM   1368  CB  PRO A 176     -16.537  -2.226   7.405  1.00  0.50           C  \\nATOM   1369  O   PRO A 176     -18.272   0.192   6.290  1.00  0.53           O  \\nATOM   1370  CG  PRO A 176     -17.339  -2.023   8.650  1.00  0.49           C  \\nATOM   1371  CD  PRO A 176     -16.900  -0.740   9.296  1.00  0.51           C  \\nATOM   1372  N   PRO A 177     -16.902   0.165   4.672  1.00  0.54           N  \\nATOM   1373  CA  PRO A 177     -17.897   0.390   3.621  1.00  0.53           C  \\nATOM   1374  C   PRO A 177     -19.091  -0.555   3.726  1.00  0.54           C  \\nATOM   1375  CB  PRO A 177     -17.111   0.134   2.332  1.00  0.50           C  \\nATOM   1376  O   PRO A 177     -18.935  -1.711   4.129  1.00  0.52           O  \\nATOM   1377  CG  PRO A 177     -15.886  -0.602   2.771  1.00  0.48           C  \\nATOM   1378  CD  PRO A 177     -15.741  -0.443   4.257  1.00  0.49           C  \\nATOM   1379  N   PRO A 178     -20.393  -0.200   3.899  1.00  0.50           N  \\nATOM   1380  CA  PRO A 178     -21.567  -1.068   3.776  1.00  0.49           C  \\nATOM   1381  C   PRO A 178     -21.373  -2.180   2.748  1.00  0.50           C  \\nATOM   1382  CB  PRO A 178     -22.672  -0.104   3.336  1.00  0.47           C  \\nATOM   1383  O   PRO A 178     -21.230  -1.905   1.554  1.00  0.49           O  \\nATOM   1384  CG  PRO A 178     -21.952   1.134   2.910  1.00  0.46           C  \\nATOM   1385  CD  PRO A 178     -20.523   1.035   3.362  1.00  0.49           C  \\nATOM   1386  N   GLY A 179     -20.833  -3.316   3.073  1.00  0.51           N  \\nATOM   1387  CA  GLY A 179     -20.875  -4.561   2.322  1.00  0.51           C  \\nATOM   1388  C   GLY A 179     -19.772  -5.528   2.708  1.00  0.51           C  \\nATOM   1389  O   GLY A 179     -19.529  -6.513   2.007  1.00  0.48           O  \\nATOM   1390  N   MET A 180     -19.028  -5.300   3.866  1.00  0.51           N  \\nATOM   1391  CA  MET A 180     -18.287  -6.381   4.512  1.00  0.53           C  \\nATOM   1392  C   MET A 180     -19.150  -7.085   5.554  1.00  0.51           C  \\nATOM   1393  CB  MET A 180     -17.012  -5.845   5.164  1.00  0.47           C  \\nATOM   1394  O   MET A 180     -19.769  -6.433   6.396  1.00  0.49           O  \\nATOM   1395  CG  MET A 180     -15.924  -5.474   4.170  1.00  0.46           C  \\nATOM   1396  SD  MET A 180     -14.241  -5.857   4.794  1.00  0.53           S  \\nATOM   1397  CE  MET A 180     -13.236  -4.992   3.555  1.00  0.42           C  \\nATOM   1398  N   PRO A 181     -19.759  -8.277   5.391  1.00  0.50           N  \\nATOM   1399  CA  PRO A 181     -20.280  -9.169   6.429  1.00  0.50           C  \\nATOM   1400  C   PRO A 181     -19.403  -9.198   7.678  1.00  0.50           C  \\nATOM   1401  CB  PRO A 181     -20.296 -10.537   5.742  1.00  0.46           C  \\nATOM   1402  O   PRO A 181     -18.276  -9.697   7.634  1.00  0.48           O  \\nATOM   1403  CG  PRO A 181     -19.325 -10.409   4.613  1.00  0.45           C  \\nATOM   1404  CD  PRO A 181     -19.093  -8.949   4.347  1.00  0.48           C  \\nATOM   1405  N   GLY A 182     -19.310  -8.245   8.482  1.00  0.48           N  \\nATOM   1406  CA  GLY A 182     -19.205  -8.013   9.914  1.00  0.49           C  \\nATOM   1407  C   GLY A 182     -19.546  -6.591  10.316  1.00  0.48           C  \\nATOM   1408  O   GLY A 182     -18.699  -5.699  10.240  1.00  0.47           O  \\nATOM   1409  N   ALA A 183     -20.592  -6.115   9.716  1.00  0.46           N  \\nATOM   1410  CA  ALA A 183     -21.325  -4.960  10.229  1.00  0.47           C  \\nATOM   1411  C   ALA A 183     -21.901  -5.248  11.613  1.00  0.46           C  \\nATOM   1412  CB  ALA A 183     -22.439  -4.564   9.264  1.00  0.43           C  \\nATOM   1413  O   ALA A 183     -22.629  -6.227  11.798  1.00  0.44           O  \\nATOM   1414  N   TYR A 184     -21.114  -5.205  12.715  1.00  0.50           N  \\nATOM   1415  CA  TYR A 184     -21.838  -4.886  13.941  1.00  0.51           C  \\nATOM   1416  C   TYR A 184     -23.198  -4.274  13.628  1.00  0.50           C  \\nATOM   1417  CB  TYR A 184     -21.021  -3.927  14.813  1.00  0.47           C  \\nATOM   1418  O   TYR A 184     -23.297  -3.346  12.822  1.00  0.48           O  \\nATOM   1419  CG  TYR A 184     -20.037  -4.622  15.722  1.00  0.47           C  \\nATOM   1420  CD1 TYR A 184     -20.428  -5.098  16.971  1.00  0.44           C  \\nATOM   1421  CD2 TYR A 184     -18.713  -4.802  15.334  1.00  0.45           C  \\nATOM   1422  CE1 TYR A 184     -19.524  -5.737  17.813  1.00  0.45           C  \\nATOM   1423  CE2 TYR A 184     -17.801  -5.440  16.168  1.00  0.46           C  \\nATOM   1424  OH  TYR A 184     -17.315  -6.535  18.233  1.00  0.42           O  \\nATOM   1425  CZ  TYR A 184     -18.215  -5.903  17.403  1.00  0.41           C  \\nATOM   1426  N   PRO A 185     -24.386  -5.093  13.568  1.00  0.51           N  \\nATOM   1427  CA  PRO A 185     -25.704  -4.462  13.670  1.00  0.49           C  \\nATOM   1428  C   PRO A 185     -25.647  -3.078  14.312  1.00  0.51           C  \\nATOM   1429  CB  PRO A 185     -26.494  -5.438  14.546  1.00  0.48           C  \\nATOM   1430  O   PRO A 185     -24.887  -2.863  15.260  1.00  0.49           O  \\nATOM   1431  CG  PRO A 185     -25.463  -6.118  15.388  1.00  0.46           C  \\nATOM   1432  CD  PRO A 185     -24.163  -6.124  14.636  1.00  0.47           C  \\nATOM   1433  N   PRO A 186     -25.759  -2.004  13.571  1.00  0.53           N  \\nATOM   1434  CA  PRO A 186     -26.105  -0.733  14.212  1.00  0.52           C  \\nATOM   1435  C   PRO A 186     -26.744  -0.921  15.586  1.00  0.53           C  \\nATOM   1436  CB  PRO A 186     -27.095  -0.101  13.230  1.00  0.50           C  \\nATOM   1437  O   PRO A 186     -27.374  -1.950  15.842  1.00  0.51           O  \\nATOM   1438  CG  PRO A 186     -27.719  -1.258  12.520  1.00  0.48           C  \\nATOM   1439  CD  PRO A 186     -26.735  -2.392  12.497  1.00  0.50           C  \\nATOM   1440  N   PRO A 187     -26.180  -0.500  16.690  1.00  0.53           N  \\nATOM   1441  CA  PRO A 187     -26.988  -0.496  17.911  1.00  0.53           C  \\nATOM   1442  C   PRO A 187     -28.487  -0.565  17.627  1.00  0.53           C  \\nATOM   1443  CB  PRO A 187     -26.619   0.834  18.574  1.00  0.50           C  \\nATOM   1444  O   PRO A 187     -29.000   0.202  16.808  1.00  0.51           O  \\nATOM   1445  CG  PRO A 187     -26.024   1.656  17.477  1.00  0.48           C  \\nATOM   1446  CD  PRO A 187     -25.659   0.743  16.341  1.00  0.50           C  \\nATOM   1447  N   ARG A 188     -29.080  -1.822  17.560  1.00  0.52           N  \\nATOM   1448  CA  ARG A 188     -30.513  -1.893  17.823  1.00  0.51           C  \\nATOM   1449  C   ARG A 188     -30.995  -0.655  18.572  1.00  0.51           C  \\nATOM   1450  CB  ARG A 188     -30.851  -3.154  18.623  1.00  0.48           C  \\nATOM   1451  O   ARG A 188     -30.609  -0.428  19.720  1.00  0.49           O  \\nATOM   1452  CG  ARG A 188     -30.968  -4.409  17.774  1.00  0.47           C  \\nATOM   1453  CD  ARG A 188     -31.303  -5.631  18.617  1.00  0.48           C  \\nATOM   1454  NE  ARG A 188     -31.393  -6.840  17.803  1.00  0.43           N  \\nATOM   1455  NH1 ARG A 188     -32.641  -8.031  19.339  1.00  0.34           N  \\nATOM   1456  NH2 ARG A 188     -32.046  -8.994  17.345  1.00  0.30           N  \\nATOM   1457  CZ  ARG A 188     -32.026  -7.952  18.164  1.00  0.45           C  \\nATOM   1458  N   MET A 189     -31.354   0.468  17.980  1.00  0.56           N  \\nATOM   1459  CA  MET A 189     -32.414   1.300  18.541  1.00  0.57           C  \\nATOM   1460  C   MET A 189     -33.571   0.443  19.044  1.00  0.55           C  \\nATOM   1461  CB  MET A 189     -32.919   2.301  17.500  1.00  0.52           C  \\nATOM   1462  O   MET A 189     -33.931  -0.552  18.414  1.00  0.53           O  \\nATOM   1463  CG  MET A 189     -31.915   3.389  17.157  1.00  0.51           C  \\nATOM   1464  SD  MET A 189     -32.674   5.059  17.113  1.00  0.52           S  \\nATOM   1465  CE  MET A 189     -32.219   5.573  15.434  1.00  0.44           C  \\nATOM   1466  N   PRO A 190     -33.720   0.315  20.338  1.00  0.57           N  \\nATOM   1467  CA  PRO A 190     -34.942  -0.309  20.851  1.00  0.56           C  \\nATOM   1468  C   PRO A 190     -36.095  -0.258  19.850  1.00  0.57           C  \\nATOM   1469  CB  PRO A 190     -35.265   0.520  22.097  1.00  0.52           C  \\nATOM   1470  O   PRO A 190     -36.404   0.808  19.312  1.00  0.54           O  \\nATOM   1471  CG  PRO A 190     -34.673   1.865  21.826  1.00  0.50           C  \\nATOM   1472  CD  PRO A 190     -33.561   1.704  20.829  1.00  0.52           C  \\nATOM   1473  N   ILE A 191     -36.380  -1.302  19.046  1.00  0.53           N  \\nATOM   1474  CA  ILE A 191     -37.658  -1.748  18.501  1.00  0.54           C  \\nATOM   1475  C   ILE A 191     -38.800  -1.225  19.370  1.00  0.53           C  \\nATOM   1476  CB  ILE A 191     -37.723  -3.288  18.400  1.00  0.49           C  \\nATOM   1477  O   ILE A 191     -38.997  -1.693  20.494  1.00  0.51           O  \\nATOM   1478  CG1 ILE A 191     -36.584  -3.814  17.519  1.00  0.47           C  \\nATOM   1479  CG2 ILE A 191     -39.084  -3.737  17.861  1.00  0.48           C  \\nATOM   1480  CD1 ILE A 191     -36.407  -5.325  17.572  1.00  0.47           C  \\nATOM   1481  N   GLY A 192     -39.118  -0.031  19.263  1.00  0.55           N  \\nATOM   1482  CA  GLY A 192     -40.494   0.381  19.492  1.00  0.56           C  \\nATOM   1483  C   GLY A 192     -41.484  -0.305  18.571  1.00  0.54           C  \\nATOM   1484  O   GLY A 192     -41.296  -0.325  17.353  1.00  0.52           O  \\nATOM   1485  N   HIS A 193     -42.085  -1.506  18.910  1.00  0.55           N  \\nATOM   1486  CA  HIS A 193     -43.431  -1.952  18.566  1.00  0.57           C  \\nATOM   1487  C   HIS A 193     -44.086  -1.005  17.566  1.00  0.54           C  \\nATOM   1488  CB  HIS A 193     -44.294  -2.068  19.823  1.00  0.50           C  \\nATOM   1489  O   HIS A 193     -44.516   0.091  17.932  1.00  0.52           O  \\nATOM   1490  CG  HIS A 193     -43.996  -3.280  20.646  1.00  0.49           C  \\nATOM   1491  CD2 HIS A 193     -43.904  -3.444  21.987  1.00  0.49           C  \\nATOM   1492  ND1 HIS A 193     -43.748  -4.516  20.088  1.00  0.52           N  \\nATOM   1493  CE1 HIS A 193     -43.516  -5.390  21.054  1.00  0.47           C  \\nATOM   1494  NE2 HIS A 193     -43.604  -4.764  22.215  1.00  0.43           N  \\nATOM   1495  N   GLY A 194     -43.834  -1.118  16.264  1.00  0.54           N  \\nATOM   1496  CA  GLY A 194     -44.748  -0.568  15.276  1.00  0.55           C  \\nATOM   1497  C   GLY A 194     -45.670  -1.611  14.671  1.00  0.54           C  \\nATOM   1498  O   GLY A 194     -45.263  -2.753  14.450  1.00  0.51           O  \\nATOM   1499  N   PRO A 195     -47.010  -1.503  14.750  1.00  0.58           N  \\nATOM   1500  CA  PRO A 195     -48.156  -2.141  14.098  1.00  0.58           C  \\nATOM   1501  C   PRO A 195     -47.809  -2.719  12.728  1.00  0.57           C  \\nATOM   1502  CB  PRO A 195     -49.168  -1.000  13.969  1.00  0.51           C  \\nATOM   1503  O   PRO A 195     -46.961  -2.170  12.019  1.00  0.54           O  \\nATOM   1504  CG  PRO A 195     -48.338   0.241  13.890  1.00  0.50           C  \\nATOM   1505  CD  PRO A 195     -47.059   0.007  14.642  1.00  0.54           C  \\nATOM   1506  N   PRO A 196     -48.048  -4.043  12.262  1.00  0.62           N  \\nATOM   1507  CA  PRO A 196     -48.251  -4.961  11.140  1.00  0.62           C  \\nATOM   1508  C   PRO A 196     -48.489  -4.235   9.818  1.00  0.60           C  \\nATOM   1509  CB  PRO A 196     -49.491  -5.758  11.555  1.00  0.54           C  \\nATOM   1510  O   PRO A 196     -49.549  -3.636   9.620  1.00  0.56           O  \\nATOM   1511  CG  PRO A 196     -50.220  -4.862  12.504  1.00  0.52           C  \\nATOM   1512  CD  PRO A 196     -49.242  -3.876  13.077  1.00  0.56           C  \\nATOM   1513  N   GLY A 197     -47.348  -3.842   9.034  1.00  0.63           N  \\nATOM   1514  CA  GLY A 197     -47.184  -3.731   7.594  1.00  0.64           C  \\nATOM   1515  C   GLY A 197     -45.733  -3.631   7.164  1.00  0.62           C  \\nATOM   1516  O   GLY A 197     -45.044  -2.667   7.504  1.00  0.58           O  \\nATOM   1517  N   GLY A 198     -45.061  -4.731   6.992  1.00  0.60           N  \\nATOM   1518  CA  GLY A 198     -43.822  -5.332   6.523  1.00  0.62           C  \\nATOM   1519  C   GLY A 198     -43.184  -4.566   5.379  1.00  0.60           C  \\nATOM   1520  O   GLY A 198     -43.856  -4.219   4.406  1.00  0.58           O  \\nATOM   1521  N   PRO A 199     -42.081  -3.802   5.601  1.00  0.64           N  \\nATOM   1522  CA  PRO A 199     -41.128  -3.332   4.592  1.00  0.63           C  \\nATOM   1523  C   PRO A 199     -40.738  -4.422   3.596  1.00  0.63           C  \\nATOM   1524  CB  PRO A 199     -39.920  -2.897   5.426  1.00  0.58           C  \\nATOM   1525  O   PRO A 199     -40.683  -5.601   3.955  1.00  0.60           O  \\nATOM   1526  CG  PRO A 199     -39.982  -3.739   6.659  1.00  0.55           C  \\nATOM   1527  CD  PRO A 199     -41.420  -4.075   6.934  1.00  0.57           C  \\nATOM   1528  N   PRO A 200     -40.913  -4.136   2.248  1.00  0.65           N  \\nATOM   1529  CA  PRO A 200     -40.409  -4.973   1.156  1.00  0.65           C  \\nATOM   1530  C   PRO A 200     -39.109  -5.690   1.513  1.00  0.64           C  \\nATOM   1531  CB  PRO A 200     -40.186  -3.972   0.019  1.00  0.59           C  \\nATOM   1532  O   PRO A 200     -38.199  -5.079   2.079  1.00  0.60           O  \\nATOM   1533  CG  PRO A 200     -39.935  -2.667   0.704  1.00  0.57           C  \\nATOM   1534  CD  PRO A 200     -40.663  -2.669   2.017  1.00  0.58           C  \\nATOM   1535  N   MET A 201     -39.074  -7.076   1.504  1.00  0.64           N  \\nATOM   1536  CA  MET A 201     -38.044  -8.108   1.570  1.00  0.66           C  \\nATOM   1537  C   MET A 201     -36.769  -7.650   0.870  1.00  0.64           C  \\nATOM   1538  CB  MET A 201     -38.545  -9.411   0.943  1.00  0.60           C  \\nATOM   1539  O   MET A 201     -36.819  -7.155  -0.258  1.00  0.61           O  \\nATOM   1540  CG  MET A 201     -39.539 -10.166   1.812  1.00  0.57           C  \\nATOM   1541  SD  MET A 201     -38.717 -11.316   2.982  1.00  0.55           S  \\nATOM   1542  CE  MET A 201     -38.676 -12.823   1.972  1.00  0.48           C  \\nATOM   1543  N   PRO A 202     -35.713  -7.188   1.489  1.00  0.65           N  \\nATOM   1544  CA  PRO A 202     -34.362  -7.058   0.937  1.00  0.64           C  \\nATOM   1545  C   PRO A 202     -34.006  -8.191  -0.023  1.00  0.65           C  \\nATOM   1546  CB  PRO A 202     -33.469  -7.093   2.180  1.00  0.59           C  \\nATOM   1547  O   PRO A 202     -34.147  -9.366   0.324  1.00  0.61           O  \\nATOM   1548  CG  PRO A 202     -34.263  -7.843   3.200  1.00  0.58           C  \\nATOM   1549  CD  PRO A 202     -35.720  -7.710   2.862  1.00  0.59           C  \\nATOM   1550  N   GLY A 203     -34.022  -7.956  -1.295  1.00  0.65           N  \\nATOM   1551  CA  GLY A 203     -33.428  -8.771  -2.343  1.00  0.66           C  \\nATOM   1552  C   GLY A 203     -32.163  -9.482  -1.901  1.00  0.63           C  \\nATOM   1553  O   GLY A 203     -31.497  -9.047  -0.959  1.00  0.60           O  \\nATOM   1554  N   PRO A 204     -31.966 -10.793  -2.305  1.00  0.64           N  \\nATOM   1555  CA  PRO A 204     -30.808 -11.628  -1.982  1.00  0.64           C  \\nATOM   1556  C   PRO A 204     -29.515 -10.824  -1.861  1.00  0.64           C  \\nATOM   1557  CB  PRO A 204     -30.736 -12.603  -3.160  1.00  0.60           C  \\nATOM   1558  O   PRO A 204     -29.296  -9.882  -2.626  1.00  0.61           O  \\nATOM   1559  CG  PRO A 204     -31.347 -11.862  -4.306  1.00  0.59           C  \\nATOM   1560  CD  PRO A 204     -32.357 -10.892  -3.762  1.00  0.60           C  \\nATOM   1561  N   PRO A 205     -28.884 -10.675  -0.780  1.00  0.63           N  \\nATOM   1562  CA  PRO A 205     -27.535 -10.132  -0.605  1.00  0.62           C  \\nATOM   1563  C   PRO A 205     -26.604 -10.480  -1.763  1.00  0.63           C  \\nATOM   1564  CB  PRO A 205     -27.059 -10.786   0.695  1.00  0.58           C  \\nATOM   1565  O   PRO A 205     -26.473 -11.653  -2.124  1.00  0.60           O  \\nATOM   1566  CG  PRO A 205     -27.831 -12.062   0.786  1.00  0.57           C  \\nATOM   1567  CD  PRO A 205     -29.115 -11.899   0.024  1.00  0.58           C  \\nATOM   1568  N   GLN A 206     -26.135  -9.535  -2.700  1.00  0.65           N  \\nATOM   1569  CA  GLN A 206     -25.018  -9.615  -3.635  1.00  0.65           C  \\nATOM   1570  C   GLN A 206     -23.752 -10.104  -2.938  1.00  0.64           C  \\nATOM   1571  CB  GLN A 206     -24.766  -8.256  -4.289  1.00  0.60           C  \\nATOM   1572  O   GLN A 206     -23.281  -9.480  -1.985  1.00  0.60           O  \\nATOM   1573  CG  GLN A 206     -25.676  -7.966  -5.476  1.00  0.58           C  \\nATOM   1574  CD  GLN A 206     -25.418  -6.605  -6.094  1.00  0.58           C  \\nATOM   1575  NE2 GLN A 206     -26.080  -6.327  -7.212  1.00  0.50           N  \\nATOM   1576  OE1 GLN A 206     -24.631  -5.810  -5.572  1.00  0.61           O  \\nATOM   1577  N   ARG A 207     -23.421 -11.417  -3.037  1.00  0.64           N  \\nATOM   1578  CA  ARG A 207     -22.187 -12.131  -2.727  1.00  0.64           C  \\nATOM   1579  C   ARG A 207     -20.966 -11.337  -3.179  1.00  0.64           C  \\nATOM   1580  CB  ARG A 207     -22.186 -13.513  -3.383  1.00  0.60           C  \\nATOM   1581  O   ARG A 207     -20.825 -11.027  -4.364  1.00  0.62           O  \\nATOM   1582  CG  ARG A 207     -23.148 -14.500  -2.742  1.00  0.59           C  \\nATOM   1583  CD  ARG A 207     -23.050 -15.879  -3.380  1.00  0.60           C  \\nATOM   1584  NE  ARG A 207     -24.015 -16.810  -2.803  1.00  0.51           N  \\nATOM   1585  NH1 ARG A 207     -22.909 -18.714  -3.503  1.00  0.43           N  \\nATOM   1586  NH2 ARG A 207     -24.860 -18.889  -2.313  1.00  0.40           N  \\nATOM   1587  CZ  ARG A 207     -23.926 -18.135  -2.874  1.00  0.58           C  \\nATOM   1588  N   SER A 208     -20.225 -10.558  -2.407  1.00  0.65           N  \\nATOM   1589  CA  SER A 208     -18.888 -10.019  -2.633  1.00  0.65           C  \\nATOM   1590  C   SER A 208     -17.920 -11.110  -3.077  1.00  0.64           C  \\nATOM   1591  CB  SER A 208     -18.361  -9.341  -1.368  1.00  0.60           C  \\nATOM   1592  O   SER A 208     -17.928 -12.215  -2.529  1.00  0.60           O  \\nATOM   1593  OG  SER A 208     -16.997  -8.985  -1.519  1.00  0.58           O  \\nATOM   1594  N   ARG A 209     -17.487 -11.207  -4.239  1.00  0.63           N  \\nATOM   1595  CA  ARG A 209     -16.425 -12.003  -4.846  1.00  0.63           C  \\nATOM   1596  C   ARG A 209     -15.260 -12.189  -3.881  1.00  0.62           C  \\nATOM   1597  CB  ARG A 209     -15.935 -11.349  -6.140  1.00  0.58           C  \\nATOM   1598  O   ARG A 209     -14.253 -12.812  -4.227  1.00  0.60           O  \\nATOM   1599  CG  ARG A 209     -16.585 -11.905  -7.396  1.00  0.56           C  \\nATOM   1600  CD  ARG A 209     -16.020 -11.264  -8.655  1.00  0.57           C  \\nATOM   1601  NE  ARG A 209     -16.823 -11.585  -9.832  1.00  0.45           N  \\nATOM   1602  NH1 ARG A 209     -15.347 -10.659 -11.348  1.00  0.35           N  \\nATOM   1603  NH2 ARG A 209     -17.291 -11.629 -12.080  1.00  0.31           N  \\nATOM   1604  CZ  ARG A 209     -16.485 -11.290 -11.084  1.00  0.51           C  \\nATOM   1605  N   PHE A 210     -15.429 -11.868  -2.531  1.00  0.61           N  \\nATOM   1606  CA  PHE A 210     -14.368 -12.064  -1.550  1.00  0.61           C  \\nATOM   1607  C   PHE A 210     -14.914 -12.705  -0.281  1.00  0.60           C  \\nATOM   1608  CB  PHE A 210     -13.691 -10.731  -1.216  1.00  0.55           C  \\nATOM   1609  O   PHE A 210     -14.289 -12.628   0.779  1.00  0.57           O  \\nATOM   1610  CG  PHE A 210     -12.901 -10.148  -2.356  1.00  0.53           C  \\nATOM   1611  CD1 PHE A 210     -11.675 -10.693  -2.721  1.00  0.50           C  \\nATOM   1612  CD2 PHE A 210     -13.383  -9.055  -3.064  1.00  0.52           C  \\nATOM   1613  CE1 PHE A 210     -10.942 -10.156  -3.776  1.00  0.51           C  \\nATOM   1614  CE2 PHE A 210     -12.656  -8.512  -4.119  1.00  0.50           C  \\nATOM   1615  CZ  PHE A 210     -11.435  -9.064  -4.473  1.00  0.48           C  \\nATOM   1616  N   ASP A 211     -16.045 -13.238  -0.330  1.00  0.59           N  \\nATOM   1617  CA  ASP A 211     -16.494 -14.112   0.750  1.00  0.59           C  \\nATOM   1618  C   ASP A 211     -15.891 -15.509   0.612  1.00  0.58           C  \\nATOM   1619  CB  ASP A 211     -18.021 -14.198   0.771  1.00  0.54           C  \\nATOM   1620  O   ASP A 211     -16.295 -16.281  -0.260  1.00  0.56           O  \\nATOM   1621  CG  ASP A 211     -18.680 -12.915   1.246  1.00  0.52           C  \\nATOM   1622  OD1 ASP A 211     -18.177 -12.288   2.202  1.00  0.53           O  \\nATOM   1623  OD2 ASP A 211     -19.715 -12.529   0.659  1.00  0.59           O  \\nATOM   1624  N   GLN A 212     -14.616 -15.742   0.954  1.00  0.60           N  \\nATOM   1625  CA  GLN A 212     -14.099 -17.048   1.352  1.00  0.61           C  \\nATOM   1626  C   GLN A 212     -14.958 -17.667   2.451  1.00  0.60           C  \\nATOM   1627  CB  GLN A 212     -12.649 -16.931   1.822  1.00  0.56           C  \\nATOM   1628  O   GLN A 212     -15.300 -16.999   3.429  1.00  0.57           O  \\nATOM   1629  CG  GLN A 212     -11.641 -16.820   0.686  1.00  0.54           C  \\nATOM   1630  CD  GLN A 212     -10.205 -16.924   1.163  1.00  0.55           C  \\nATOM   1631  NE2 GLN A 212      -9.307 -17.307   0.262  1.00  0.45           N  \\nATOM   1632  OE1 GLN A 212      -9.904 -16.662   2.332  1.00  0.57           O  \\nATOM   1633  N   PRO A 213     -15.639 -18.791   2.168  1.00  0.57           N  \\nATOM   1634  CA  PRO A 213     -16.337 -19.657   3.121  1.00  0.56           C  \\nATOM   1635  C   PRO A 213     -15.542 -19.883   4.405  1.00  0.57           C  \\nATOM   1636  CB  PRO A 213     -16.509 -20.967   2.348  1.00  0.53           C  \\nATOM   1637  O   PRO A 213     -14.388 -20.316   4.353  1.00  0.55           O  \\nATOM   1638  CG  PRO A 213     -15.406 -20.963   1.339  1.00  0.51           C  \\nATOM   1639  CD  PRO A 213     -14.973 -19.543   1.112  1.00  0.52           C  \\nATOM   1640  N   ASP A 214     -15.717 -19.166   5.398  1.00  0.56           N  \\nATOM   1641  CA  ASP A 214     -15.525 -19.603   6.777  1.00  0.57           C  \\nATOM   1642  C   ASP A 214     -16.040 -21.027   6.979  1.00  0.56           C  \\nATOM   1643  CB  ASP A 214     -16.226 -18.648   7.746  1.00  0.53           C  \\nATOM   1644  O   ASP A 214     -17.224 -21.299   6.773  1.00  0.54           O  \\nATOM   1645  CG  ASP A 214     -15.255 -17.812   8.561  1.00  0.53           C  \\nATOM   1646  OD1 ASP A 214     -14.092 -18.233   8.743  1.00  0.52           O  \\nATOM   1647  OD2 ASP A 214     -15.658 -16.725   9.029  1.00  0.58           O  \\nATOM   1648  N   GLY A 215     -15.337 -22.012   6.761  1.00  0.55           N  \\nATOM   1649  CA  GLY A 215     -15.331 -23.307   7.422  1.00  0.55           C  \\nATOM   1650  C   GLY A 215     -16.450 -23.467   8.433  1.00  0.54           C  \\nATOM   1651  O   GLY A 215     -16.282 -23.142   9.610  1.00  0.52           O  \\nATOM   1652  N   GLY A 216     -17.707 -23.425   7.809  1.00  0.49           N  \\nATOM   1653  CA  GLY A 216     -18.786 -24.140   8.471  1.00  0.51           C  \\nATOM   1654  C   GLY A 216     -19.639 -24.952   7.516  1.00  0.49           C  \\nATOM   1655  O   GLY A 216     -20.176 -24.416   6.544  1.00  0.48           O  \\nATOM   1656  N   ASP A 217     -19.355 -26.248   7.342  1.00  0.57           N  \\nATOM   1657  CA  ASP A 217     -20.314 -27.327   7.124  1.00  0.58           C  \\nATOM   1658  C   ASP A 217     -21.749 -26.811   7.198  1.00  0.56           C  \\nATOM   1659  CB  ASP A 217     -20.104 -28.446   8.146  1.00  0.53           C  \\nATOM   1660  O   ASP A 217     -22.281 -26.595   8.289  1.00  0.54           O  \\nATOM   1661  CG  ASP A 217     -19.175 -29.539   7.646  1.00  0.53           C  \\nATOM   1662  OD1 ASP A 217     -19.101 -29.763   6.419  1.00  0.53           O  \\nATOM   1663  OD2 ASP A 217     -18.514 -30.185   8.488  1.00  0.57           O  \\nATOM   1664  N   ARG A 218     -22.128 -25.959   6.301  1.00  0.55           N  \\nATOM   1665  CA  ARG A 218     -23.574 -25.937   6.105  1.00  0.54           C  \\nATOM   1666  C   ARG A 218     -23.926 -25.969   4.621  1.00  0.55           C  \\nATOM   1667  CB  ARG A 218     -24.186 -24.698   6.762  1.00  0.50           C  \\nATOM   1668  O   ARG A 218     -23.645 -25.017   3.890  1.00  0.52           O  \\nATOM   1669  CG  ARG A 218     -24.291 -24.792   8.276  1.00  0.48           C  \\nATOM   1670  CD  ARG A 218     -25.022 -23.594   8.866  1.00  0.50           C  \\nATOM   1671  NE  ARG A 218     -25.062 -23.652  10.324  1.00  0.43           N  \\nATOM   1672  NH1 ARG A 218     -26.889 -22.254  10.536  1.00  0.33           N  \\nATOM   1673  NH2 ARG A 218     -25.894 -23.137  12.402  1.00  0.28           N  \\nATOM   1674  CZ  ARG A 218     -25.948 -23.014  11.084  1.00  0.41           C  \\nATOM   1675  N   TRP A 219     -24.143 -27.183   4.055  1.00  0.55           N  \\nATOM   1676  CA  TRP A 219     -25.219 -27.594   3.159  1.00  0.61           C  \\nATOM   1677  C   TRP A 219     -26.231 -26.469   2.972  1.00  0.55           C  \\nATOM   1678  CB  TRP A 219     -25.920 -28.844   3.698  1.00  0.51           C  \\nATOM   1679  O   TRP A 219     -27.177 -26.340   3.753  1.00  0.52           O  \\nATOM   1680  CG  TRP A 219     -25.144 -30.111   3.500  1.00  0.49           C  \\nATOM   1681  CD1 TRP A 219     -24.162 -30.611   4.310  1.00  0.45           C  \\nATOM   1682  CD2 TRP A 219     -25.282 -31.037   2.419  1.00  0.46           C  \\nATOM   1683  CE2 TRP A 219     -24.352 -32.077   2.637  1.00  0.45           C  \\nATOM   1684  CE3 TRP A 219     -26.105 -31.090   1.285  1.00  0.59           C  \\nATOM   1685  NE1 TRP A 219     -23.682 -31.793   3.796  1.00  0.56           N  \\nATOM   1686  CH2 TRP A 219     -25.038 -33.185   0.661  1.00  0.53           C  \\nATOM   1687  CZ2 TRP A 219     -24.222 -33.159   1.762  1.00  0.58           C  \\nATOM   1688  CZ3 TRP A 219     -25.973 -32.166   0.416  1.00  0.49           C  \\nATOM   1689  N   GLY A 220     -25.843 -25.297   2.388  1.00  0.56           N  \\nATOM   1690  CA  GLY A 220     -26.943 -24.540   1.811  1.00  0.56           C  \\nATOM   1691  C   GLY A 220     -27.302 -24.987   0.406  1.00  0.56           C  \\nATOM   1692  O   GLY A 220     -26.422 -25.327  -0.387  1.00  0.53           O  \\nATOM   1693  N   PRO A 221     -28.642 -25.339   0.239  1.00  0.58           N  \\nATOM   1694  CA  PRO A 221     -29.296 -25.598  -1.047  1.00  0.57           C  \\nATOM   1695  C   PRO A 221     -28.811 -24.665  -2.154  1.00  0.58           C  \\nATOM   1696  CB  PRO A 221     -30.777 -25.363  -0.743  1.00  0.54           C  \\nATOM   1697  O   PRO A 221     -28.523 -23.493  -1.896  1.00  0.55           O  \\nATOM   1698  CG  PRO A 221     -30.784 -24.433   0.427  1.00  0.52           C  \\nATOM   1699  CD  PRO A 221     -29.492 -24.595   1.174  1.00  0.54           C  \\nATOM   1700  N   PRO A 222     -28.336 -25.152  -3.290  1.00  0.59           N  \\nATOM   1701  CA  PRO A 222     -27.964 -24.570  -4.582  1.00  0.58           C  \\nATOM   1702  C   PRO A 222     -28.814 -23.357  -4.950  1.00  0.58           C  \\nATOM   1703  CB  PRO A 222     -28.197 -25.719  -5.566  1.00  0.54           C  \\nATOM   1704  O   PRO A 222     -30.034 -23.475  -5.097  1.00  0.56           O  \\nATOM   1705  CG  PRO A 222     -29.287 -26.535  -4.950  1.00  0.52           C  \\nATOM   1706  CD  PRO A 222     -29.243 -26.343  -3.461  1.00  0.54           C  \\nATOM   1707  N   MET A 223     -28.429 -22.171  -4.670  1.00  0.57           N  \\nATOM   1708  CA  MET A 223     -28.933 -20.919  -5.228  1.00  0.59           C  \\nATOM   1709  C   MET A 223     -29.106 -21.028  -6.739  1.00  0.57           C  \\nATOM   1710  CB  MET A 223     -27.990 -19.763  -4.891  1.00  0.53           C  \\nATOM   1711  O   MET A 223     -28.155 -21.344  -7.457  1.00  0.55           O  \\nATOM   1712  CG  MET A 223     -28.128 -19.255  -3.465  1.00  0.52           C  \\nATOM   1713  SD  MET A 223     -27.657 -17.489  -3.299  1.00  0.55           S  \\nATOM   1714  CE  MET A 223     -28.629 -17.035  -1.836  1.00  0.47           C  \\nATOM   1715  N   ARG A 224     -30.306 -21.164  -7.317  1.00  0.61           N  \\nATOM   1716  CA  ARG A 224     -31.049 -21.044  -8.567  1.00  0.61           C  \\nATOM   1717  C   ARG A 224     -30.870 -19.659  -9.180  1.00  0.59           C  \\nATOM   1718  CB  ARG A 224     -32.535 -21.328  -8.339  1.00  0.56           C  \\nATOM   1719  O   ARG A 224     -31.284 -18.657  -8.593  1.00  0.56           O  \\nATOM   1720  CG  ARG A 224     -32.856 -22.799  -8.125  1.00  0.55           C  \\nATOM   1721  CD  ARG A 224     -34.355 -23.039  -8.014  1.00  0.57           C  \\nATOM   1722  NE  ARG A 224     -34.649 -24.351  -7.444  1.00  0.47           N  \\nATOM   1723  NH1 ARG A 224     -36.512 -24.732  -8.757  1.00  0.37           N  \\nATOM   1724  NH2 ARG A 224     -35.843 -26.301  -7.225  1.00  0.33           N  \\nATOM   1725  CZ  ARG A 224     -35.667 -25.125  -7.810  1.00  0.52           C  \\nATOM   1726  N   GLY A 225     -29.746 -19.450 -10.207  1.00  0.55           N  \\nATOM   1727  CA  GLY A 225     -29.643 -18.885 -11.543  1.00  0.56           C  \\nATOM   1728  C   GLY A 225     -29.349 -17.397 -11.540  1.00  0.55           C  \\nATOM   1729  O   GLY A 225     -30.234 -16.584 -11.263  1.00  0.52           O  \\nATOM   1730  N   VAL A 226     -28.144 -16.881 -11.282  1.00  0.58           N  \\nATOM   1731  CA  VAL A 226     -27.531 -15.741 -11.955  1.00  0.59           C  \\nATOM   1732  C   VAL A 226     -27.041 -16.160 -13.340  1.00  0.58           C  \\nATOM   1733  CB  VAL A 226     -26.363 -15.155 -11.131  1.00  0.54           C  \\nATOM   1734  O   VAL A 226     -26.341 -17.166 -13.479  1.00  0.55           O  \\nATOM   1735  CG1 VAL A 226     -25.885 -13.835 -11.733  1.00  0.48           C  \\nATOM   1736  CG2 VAL A 226     -26.783 -14.961  -9.675  1.00  0.49           C  \\nATOM   1737  N   PRO A 227     -27.660 -15.649 -14.392  1.00  0.57           N  \\nATOM   1738  CA  PRO A 227     -27.169 -15.889 -15.751  1.00  0.57           C  \\nATOM   1739  C   PRO A 227     -25.644 -15.901 -15.834  1.00  0.56           C  \\nATOM   1740  CB  PRO A 227     -27.749 -14.717 -16.548  1.00  0.53           C  \\nATOM   1741  O   PRO A 227     -24.986 -15.001 -15.306  1.00  0.54           O  \\nATOM   1742  CG  PRO A 227     -28.336 -13.808 -15.517  1.00  0.52           C  \\nATOM   1743  CD  PRO A 227     -28.246 -14.483 -14.178  1.00  0.54           C  \\nATOM   1744  N   ARG A 228     -24.974 -17.109 -15.816  1.00  0.54           N  \\nATOM   1745  CA  ARG A 228     -23.711 -17.527 -16.417  1.00  0.54           C  \\nATOM   1746  C   ARG A 228     -23.302 -16.583 -17.542  1.00  0.54           C  \\nATOM   1747  CB  ARG A 228     -23.814 -18.959 -16.946  1.00  0.51           C  \\nATOM   1748  O   ARG A 228     -24.004 -16.467 -18.549  1.00  0.52           O  \\nATOM   1749  CG  ARG A 228     -23.460 -20.022 -15.918  1.00  0.50           C  \\nATOM   1750  CD  ARG A 228     -23.576 -21.426 -16.496  1.00  0.51           C  \\nATOM   1751  NE  ARG A 228     -23.209 -22.444 -15.516  1.00  0.46           N  \\nATOM   1752  NH1 ARG A 228     -23.182 -24.188 -17.031  1.00  0.35           N  \\nATOM   1753  NH2 ARG A 228     -22.697 -24.573 -14.824  1.00  0.30           N  \\nATOM   1754  CZ  ARG A 228     -23.030 -23.733 -15.792  1.00  0.44           C  \\nATOM   1755  N   THR A 229     -22.607 -15.504 -17.392  1.00  0.57           N  \\nATOM   1756  CA  THR A 229     -21.765 -15.123 -18.521  1.00  0.58           C  \\nATOM   1757  C   THR A 229     -21.442 -16.337 -19.388  1.00  0.56           C  \\nATOM   1758  CB  THR A 229     -20.456 -14.466 -18.046  1.00  0.53           C  \\nATOM   1759  O   THR A 229     -21.028 -17.379 -18.876  1.00  0.54           O  \\nATOM   1760  CG2 THR A 229     -20.586 -12.947 -18.009  1.00  0.48           C  \\nATOM   1761  OG1 THR A 229     -20.141 -14.939 -16.730  1.00  0.50           O  \\nATOM   1762  N   PRO A 230     -22.153 -16.561 -20.456  1.00  0.54           N  \\nATOM   1763  CA  PRO A 230     -21.734 -17.598 -21.403  1.00  0.53           C  \\nATOM   1764  C   PRO A 230     -20.234 -17.877 -21.346  1.00  0.53           C  \\nATOM   1765  CB  PRO A 230     -22.130 -17.014 -22.762  1.00  0.50           C  \\nATOM   1766  O   PRO A 230     -19.445 -16.974 -21.053  1.00  0.52           O  \\nATOM   1767  CG  PRO A 230     -22.298 -15.550 -22.515  1.00  0.49           C  \\nATOM   1768  CD  PRO A 230     -22.431 -15.328 -21.035  1.00  0.51           C  \\nATOM   1769  N   PRO A 231     -19.729 -19.107 -20.929  1.00  0.53           N  \\nATOM   1770  CA  PRO A 231     -18.368 -19.537 -21.259  1.00  0.52           C  \\nATOM   1771  C   PRO A 231     -17.804 -18.820 -22.484  1.00  0.53           C  \\nATOM   1772  CB  PRO A 231     -18.528 -21.035 -21.529  1.00  0.49           C  \\nATOM   1773  O   PRO A 231     -18.469 -18.742 -23.519  1.00  0.51           O  \\nATOM   1774  CG  PRO A 231     -19.989 -21.225 -21.779  1.00  0.47           C  \\nATOM   1775  CD  PRO A 231     -20.719 -20.007 -21.290  1.00  0.50           C  \\nATOM   1776  N   TYR A 232     -17.256 -17.574 -22.441  1.00  0.53           N  \\nATOM   1777  CA  TYR A 232     -16.282 -17.484 -23.524  1.00  0.54           C  \\nATOM   1778  C   TYR A 232     -15.802 -18.868 -23.942  1.00  0.52           C  \\nATOM   1779  CB  TYR A 232     -15.088 -16.622 -23.102  1.00  0.49           C  \\nATOM   1780  O   TYR A 232     -15.185 -19.584 -23.149  1.00  0.50           O  \\nATOM   1781  CG  TYR A 232     -15.335 -15.138 -23.229  1.00  0.48           C  \\nATOM   1782  CD1 TYR A 232     -15.129 -14.479 -24.439  1.00  0.46           C  \\nATOM   1783  CD2 TYR A 232     -15.775 -14.393 -22.140  1.00  0.47           C  \\nATOM   1784  CE1 TYR A 232     -15.354 -13.111 -24.560  1.00  0.47           C  \\nATOM   1785  CE2 TYR A 232     -16.003 -13.025 -22.250  1.00  0.47           C  \\nATOM   1786  OH  TYR A 232     -16.014 -11.040 -23.576  1.00  0.46           O  \\nATOM   1787  CZ  TYR A 232     -15.790 -12.394 -23.462  1.00  0.42           C  \\nATOM   1788  N   GLU A 233     -16.508 -19.741 -24.657  1.00  0.54           N  \\nATOM   1789  CA  GLU A 233     -15.942 -20.561 -25.723  1.00  0.54           C  \\nATOM   1790  C   GLU A 233     -14.736 -19.880 -26.362  1.00  0.53           C  \\nATOM   1791  CB  GLU A 233     -17.000 -20.865 -26.787  1.00  0.50           C  \\nATOM   1792  O   GLU A 233     -14.886 -18.906 -27.103  1.00  0.51           O  \\nATOM   1793  CG  GLU A 233     -17.956 -21.985 -26.403  1.00  0.49           C  \\nATOM   1794  CD  GLU A 233     -19.109 -22.152 -27.380  1.00  0.51           C  \\nATOM   1795  OE1 GLU A 233     -19.559 -21.141 -27.966  1.00  0.54           O  \\nATOM   1796  OE2 GLU A 233     -19.567 -23.302 -27.560  1.00  0.50           O  \\nATOM   1797  N   ASN A 234     -13.654 -19.634 -25.658  1.00  0.52           N  \\nATOM   1798  CA  ASN A 234     -12.401 -19.707 -26.403  1.00  0.53           C  \\nATOM   1799  C   ASN A 234     -12.169 -21.102 -26.976  1.00  0.52           C  \\nATOM   1800  CB  ASN A 234     -11.224 -19.296 -25.515  1.00  0.48           C  \\nATOM   1801  O   ASN A 234     -11.881 -22.043 -26.233  1.00  0.50           O  \\nATOM   1802  CG  ASN A 234     -10.706 -17.909 -25.838  1.00  0.47           C  \\nATOM   1803  ND2 ASN A 234      -9.894 -17.356 -24.945  1.00  0.47           N  \\nATOM   1804  OD1 ASN A 234     -11.033 -17.338 -26.882  1.00  0.48           O  \\nATOM   1805  N   GLU A 235     -12.881 -21.595 -27.851  1.00  0.53           N  \\nATOM   1806  CA  GLU A 235     -12.325 -22.342 -28.975  1.00  0.53           C  \\nATOM   1807  C   GLU A 235     -10.914 -21.866 -29.310  1.00  0.53           C  \\nATOM   1808  CB  GLU A 235     -13.228 -22.217 -30.204  1.00  0.50           C  \\nATOM   1809  O   GLU A 235     -10.740 -20.831 -29.957  1.00  0.50           O  \\nATOM   1810  CG  GLU A 235     -14.481 -23.078 -30.137  1.00  0.49           C  \\nATOM   1811  CD  GLU A 235     -15.371 -22.939 -31.362  1.00  0.52           C  \\nATOM   1812  OE1 GLU A 235     -15.453 -21.826 -31.929  1.00  0.53           O  \\nATOM   1813  OE2 GLU A 235     -15.992 -23.951 -31.757  1.00  0.49           O  \\nATOM   1814  N   HIS A 236     -10.062 -21.897 -28.354  1.00  0.49           N  \\nATOM   1815  CA  HIS A 236      -8.675 -22.189 -28.698  1.00  0.50           C  \\nATOM   1816  C   HIS A 236      -8.354 -23.665 -28.488  1.00  0.49           C  \\nATOM   1817  CB  HIS A 236      -7.724 -21.322 -27.870  1.00  0.45           C  \\nATOM   1818  O   HIS A 236      -8.382 -24.156 -27.357  1.00  0.46           O  \\nATOM   1819  CG  HIS A 236      -7.608 -19.916 -28.367  1.00  0.44           C  \\nATOM   1820  CD2 HIS A 236      -7.860 -18.734 -27.758  1.00  0.42           C  \\nATOM   1821  ND1 HIS A 236      -7.187 -19.611 -29.643  1.00  0.50           N  \\nATOM   1822  CE1 HIS A 236      -7.185 -18.298 -29.797  1.00  0.44           C  \\nATOM   1823  NE2 HIS A 236      -7.590 -17.742 -28.668  1.00  0.40           N  \\nATOM   1824  N   GLU A 237      -8.735 -24.491 -29.359  1.00  0.52           N  \\nATOM   1825  CA  GLU A 237      -7.946 -25.543 -29.991  1.00  0.52           C  \\nATOM   1826  C   GLU A 237      -6.756 -25.939 -29.121  1.00  0.52           C  \\nATOM   1827  CB  GLU A 237      -7.462 -25.097 -31.373  1.00  0.49           C  \\nATOM   1828  O   GLU A 237      -5.858 -25.129 -28.883  1.00  0.50           O  \\nATOM   1829  CG  GLU A 237      -8.069 -25.888 -32.522  1.00  0.47           C  \\nATOM   1830  CD  GLU A 237      -7.694 -25.342 -33.890  1.00  0.50           C  \\nATOM   1831  OE1 GLU A 237      -7.385 -24.133 -33.998  1.00  0.52           O  \\nATOM   1832  OE2 GLU A 237      -7.709 -26.129 -34.863  1.00  0.48           O  \\nATOM   1833  N   HIS A 238      -6.825 -26.721 -27.926  1.00  0.51           N  \\nATOM   1834  CA  HIS A 238      -5.835 -27.746 -27.619  1.00  0.52           C  \\nATOM   1835  C   HIS A 238      -4.930 -28.014 -28.817  1.00  0.50           C  \\nATOM   1836  CB  HIS A 238      -6.522 -29.041 -27.181  1.00  0.48           C  \\nATOM   1837  O   HIS A 238      -5.309 -28.743 -29.736  1.00  0.48           O  \\nATOM   1838  CG  HIS A 238      -6.596 -29.208 -25.696  1.00  0.48           C  \\nATOM   1839  CD2 HIS A 238      -7.659 -29.326 -24.866  1.00  0.47           C  \\nATOM   1840  ND1 HIS A 238      -5.474 -29.267 -24.898  1.00  0.51           N  \\nATOM   1841  CE1 HIS A 238      -5.845 -29.416 -23.637  1.00  0.46           C  \\nATOM   1842  NE2 HIS A 238      -7.166 -29.454 -23.591  1.00  0.43           N  \\nATOM   1843  N   GLN A 239      -4.209 -27.058 -29.241  1.00  0.48           N  \\nATOM   1844  CA  GLN A 239      -2.898 -27.409 -29.776  1.00  0.49           C  \\nATOM   1845  C   GLN A 239      -1.902 -27.685 -28.654  1.00  0.48           C  \\nATOM   1846  CB  GLN A 239      -2.372 -26.295 -30.683  1.00  0.46           C  \\nATOM   1847  O   GLN A 239      -1.727 -26.860 -27.755  1.00  0.47           O  \\nATOM   1848  CG  GLN A 239      -2.931 -26.339 -32.099  1.00  0.46           C  \\nATOM   1849  CD  GLN A 239      -2.403 -25.217 -32.973  1.00  0.48           C  \\nATOM   1850  NE2 GLN A 239      -2.880 -25.157 -34.212  1.00  0.40           N  \\nATOM   1851  OE1 GLN A 239      -1.575 -24.411 -32.539  1.00  0.51           O  \\nATOM   1852  N   ASP A 240      -1.788 -28.836 -28.141  1.00  0.51           N  \\nATOM   1853  CA  ASP A 240      -0.488 -29.375 -27.757  1.00  0.52           C  \\nATOM   1854  C   ASP A 240       0.581 -28.284 -27.754  1.00  0.50           C  \\nATOM   1855  CB  ASP A 240      -0.078 -30.510 -28.698  1.00  0.48           C  \\nATOM   1856  O   ASP A 240       1.020 -27.836 -28.815  1.00  0.49           O  \\nATOM   1857  CG  ASP A 240      -0.335 -31.888 -28.113  1.00  0.48           C  \\nATOM   1858  OD1 ASP A 240      -0.316 -32.036 -26.872  1.00  0.49           O  \\nATOM   1859  OD2 ASP A 240      -0.554 -32.834 -28.900  1.00  0.55           O  \\nATOM   1860  N   TYR A 241       0.629 -27.259 -26.966  1.00  0.46           N  \\nATOM   1861  CA  TYR A 241       1.905 -26.608 -26.691  1.00  0.48           C  \\nATOM   1862  C   TYR A 241       2.991 -27.637 -26.402  1.00  0.46           C  \\nATOM   1863  CB  TYR A 241       1.773 -25.642 -25.509  1.00  0.44           C  \\nATOM   1864  O   TYR A 241       3.073 -28.166 -25.291  1.00  0.45           O  \\nATOM   1865  CG  TYR A 241       2.197 -24.229 -25.830  1.00  0.44           C  \\nATOM   1866  CD1 TYR A 241       3.504 -23.805 -25.602  1.00  0.42           C  \\nATOM   1867  CD2 TYR A 241       1.292 -23.316 -26.360  1.00  0.43           C  \\nATOM   1868  CE1 TYR A 241       3.899 -22.504 -25.894  1.00  0.43           C  \\nATOM   1869  CE2 TYR A 241       1.675 -22.012 -26.655  1.00  0.46           C  \\nATOM   1870  OH  TYR A 241       3.364 -20.326 -26.711  1.00  0.38           O  \\nATOM   1871  CZ  TYR A 241       2.979 -21.616 -26.420  1.00  0.37           C  \\nATOM   1872  N   GLN A 242       3.376 -28.335 -27.341  1.00  0.47           N  \\nATOM   1873  CA  GLN A 242       4.798 -28.631 -27.474  1.00  0.47           C  \\nATOM   1874  C   GLN A 242       5.640 -27.370 -27.301  1.00  0.47           C  \\nATOM   1875  CB  GLN A 242       5.089 -29.274 -28.831  1.00  0.44           C  \\nATOM   1876  O   GLN A 242       5.501 -26.414 -28.066  1.00  0.45           O  \\nATOM   1877  CG  GLN A 242       4.686 -30.740 -28.915  1.00  0.44           C  \\nATOM   1878  CD  GLN A 242       5.006 -31.360 -30.263  1.00  0.47           C  \\nATOM   1879  NE2 GLN A 242       5.021 -32.687 -30.316  1.00  0.41           N  \\nATOM   1880  OE1 GLN A 242       5.238 -30.651 -31.247  1.00  0.51           O  \\nATOM   1881  N   ALA A 243       6.046 -26.804 -26.193  1.00  0.52           N  \\nATOM   1882  CA  ALA A 243       7.264 -26.001 -26.270  1.00  0.52           C  \\nATOM   1883  C   ALA A 243       7.965 -26.195 -27.612  1.00  0.51           C  \\nATOM   1884  CB  ALA A 243       8.208 -26.356 -25.124  1.00  0.49           C  \\nATOM   1885  O   ALA A 243       8.156 -27.328 -28.061  1.00  0.50           O  \\nATOM   1886  N   PRO A 244       7.745 -25.391 -28.575  1.00  0.48           N  \\nATOM   1887  CA  PRO A 244       8.633 -25.533 -29.732  1.00  0.47           C  \\nATOM   1888  C   PRO A 244      10.089 -25.760 -29.334  1.00  0.48           C  \\nATOM   1889  CB  PRO A 244       8.468 -24.201 -30.468  1.00  0.44           C  \\nATOM   1890  O   PRO A 244      10.637 -25.005 -28.526  1.00  0.45           O  \\nATOM   1891  CG  PRO A 244       7.848 -23.284 -29.464  1.00  0.41           C  \\nATOM   1892  CD  PRO A 244       7.332 -24.108 -28.320  1.00  0.44           C  \\nATOM   1893  N   ASP A 245      10.586 -26.873 -29.108  1.00  0.52           N  \\nATOM   1894  CA  ASP A 245      11.923 -27.155 -29.623  1.00  0.52           C  \\nATOM   1895  C   ASP A 245      12.247 -26.271 -30.825  1.00  0.51           C  \\nATOM   1896  CB  ASP A 245      12.049 -28.631 -30.006  1.00  0.49           C  \\nATOM   1897  O   ASP A 245      11.705 -26.471 -31.913  1.00  0.48           O  \\nATOM   1898  CG  ASP A 245      12.327 -29.532 -28.816  1.00  0.49           C  \\nATOM   1899  OD1 ASP A 245      13.145 -29.159 -27.947  1.00  0.50           O  \\nATOM   1900  OD2 ASP A 245      11.726 -30.625 -28.747  1.00  0.56           O  \\nATOM   1901  N   ASP A 246      12.032 -24.838 -30.716  1.00  0.49           N  \\nATOM   1902  CA  ASP A 246      12.744 -24.063 -31.727  1.00  0.50           C  \\nATOM   1903  C   ASP A 246      12.877 -22.601 -31.306  1.00  0.49           C  \\nATOM   1904  CB  ASP A 246      12.031 -24.158 -33.077  1.00  0.46           C  \\nATOM   1905  O   ASP A 246      12.038 -21.769 -31.658  1.00  0.46           O  \\nATOM   1906  CG  ASP A 246      12.650 -25.191 -34.003  1.00  0.45           C  \\nATOM   1907  OD1 ASP A 246      13.850 -25.505 -33.851  1.00  0.47           O  \\nATOM   1908  OD2 ASP A 246      11.931 -25.693 -34.895  1.00  0.51           O  \\nATOM   1909  N   TYR A 247      13.512 -22.278 -30.174  1.00  0.55           N  \\nATOM   1910  CA  TYR A 247      14.473 -21.183 -30.110  1.00  0.56           C  \\nATOM   1911  C   TYR A 247      15.580 -21.368 -31.141  1.00  0.55           C  \\nATOM   1912  CB  TYR A 247      15.078 -21.081 -28.706  1.00  0.52           C  \\nATOM   1913  O   TYR A 247      16.576 -22.047 -30.877  1.00  0.53           O  \\nATOM   1914  CG  TYR A 247      14.492 -19.967 -27.873  1.00  0.50           C  \\nATOM   1915  CD1 TYR A 247      15.041 -18.687 -27.901  1.00  0.49           C  \\nATOM   1916  CD2 TYR A 247      13.390 -20.193 -27.055  1.00  0.49           C  \\nATOM   1917  CE1 TYR A 247      14.506 -17.658 -27.132  1.00  0.49           C  \\nATOM   1918  CE2 TYR A 247      12.847 -19.172 -26.283  1.00  0.49           C  \\nATOM   1919  OH  TYR A 247      12.876 -16.895 -25.566  1.00  0.49           O  \\nATOM   1920  CZ  TYR A 247      13.410 -17.910 -26.329  1.00  0.44           C  \\nATOM   1921  N   ASN A 248      15.363 -21.555 -32.278  1.00  0.47           N  \\nATOM   1922  CA  ASN A 248      16.215 -21.333 -33.442  1.00  0.51           C  \\nATOM   1923  C   ASN A 248      16.335 -19.849 -33.774  1.00  0.47           C  \\nATOM   1924  CB  ASN A 248      15.686 -22.108 -34.651  1.00  0.45           C  \\nATOM   1925  O   ASN A 248      15.335 -19.194 -34.076  1.00  0.45           O  \\nATOM   1926  CG  ASN A 248      16.730 -22.282 -35.736  1.00  0.45           C  \\nATOM   1927  ND2 ASN A 248      16.291 -22.683 -36.923  1.00  0.39           N  \\nATOM   1928  OD1 ASN A 248      17.922 -22.058 -35.509  1.00  0.45           O  \\nATOM   1929  N   GLN A 249      17.337 -19.043 -33.133  1.00  0.53           N  \\nATOM   1930  CA  GLN A 249      18.307 -18.217 -33.846  1.00  0.53           C  \\nATOM   1931  C   GLN A 249      17.630 -17.019 -34.504  1.00  0.52           C  \\nATOM   1932  CB  GLN A 249      19.047 -19.045 -34.897  1.00  0.49           C  \\nATOM   1933  O   GLN A 249      17.121 -17.123 -35.622  1.00  0.50           O  \\nATOM   1934  CG  GLN A 249      20.105 -19.974 -34.316  1.00  0.49           C  \\nATOM   1935  CD  GLN A 249      20.841 -20.764 -35.382  1.00  0.53           C  \\nATOM   1936  NE2 GLN A 249      21.564 -21.796 -34.960  1.00  0.43           N  \\nATOM   1937  OE1 GLN A 249      20.759 -20.451 -36.574  1.00  0.55           O  \\nATOM   1938  N   GLY A 250      17.098 -16.128 -33.779  1.00  0.51           N  \\nATOM   1939  CA  GLY A 250      17.066 -14.811 -34.395  1.00  0.51           C  \\nATOM   1940  C   GLY A 250      17.833 -13.765 -33.608  1.00  0.51           C  \\nATOM   1941  O   GLY A 250      17.499 -13.479 -32.456  1.00  0.48           O  \\nATOM   1942  N   GLY A 251      19.260 -13.766 -33.719  1.00  0.52           N  \\nATOM   1943  CA  GLY A 251      20.244 -12.702 -33.834  1.00  0.52           C  \\nATOM   1944  C   GLY A 251      19.961 -11.525 -32.920  1.00  0.52           C  \\nATOM   1945  O   GLY A 251      19.050 -10.736 -33.180  1.00  0.50           O  \\nATOM   1946  N   TYR A 252      20.059 -11.545 -31.581  1.00  0.55           N  \\nATOM   1947  CA  TYR A 252      20.663 -10.422 -30.871  1.00  0.56           C  \\nATOM   1948  C   TYR A 252      21.998 -10.037 -31.496  1.00  0.55           C  \\nATOM   1949  CB  TYR A 252      20.858 -10.763 -29.391  1.00  0.52           C  \\nATOM   1950  O   TYR A 252      22.937 -10.837 -31.513  1.00  0.53           O  \\nATOM   1951  CG  TYR A 252      19.742 -10.272 -28.502  1.00  0.50           C  \\nATOM   1952  CD1 TYR A 252      19.771  -8.990 -27.958  1.00  0.47           C  \\nATOM   1953  CD2 TYR A 252      18.656 -11.089 -28.204  1.00  0.48           C  \\nATOM   1954  CE1 TYR A 252      18.745  -8.534 -27.137  1.00  0.48           C  \\nATOM   1955  CE2 TYR A 252      17.625 -10.643 -27.384  1.00  0.50           C  \\nATOM   1956  OH  TYR A 252      16.660  -8.919 -26.044  1.00  0.47           O  \\nATOM   1957  CZ  TYR A 252      17.678  -9.366 -26.856  1.00  0.42           C  \\nATOM   1958  N   ASP A 253      22.073  -9.449 -32.516  1.00  0.55           N  \\nATOM   1959  CA  ASP A 253      23.167  -8.651 -33.061  1.00  0.55           C  \\nATOM   1960  C   ASP A 253      24.113  -8.189 -31.955  1.00  0.55           C  \\nATOM   1961  CB  ASP A 253      22.622  -7.443 -33.825  1.00  0.52           C  \\nATOM   1962  O   ASP A 253      23.776  -7.296 -31.175  1.00  0.53           O  \\nATOM   1963  CG  ASP A 253      23.529  -7.001 -34.960  1.00  0.52           C  \\nATOM   1964  OD1 ASP A 253      24.694  -7.449 -35.020  1.00  0.52           O  \\nATOM   1965  OD2 ASP A 253      23.076  -6.194 -35.800  1.00  0.57           O  \\nATOM   1966  N   ASP A 254      25.117  -8.949 -31.440  1.00  0.53           N  \\nATOM   1967  CA  ASP A 254      26.439  -8.870 -30.827  1.00  0.53           C  \\nATOM   1968  C   ASP A 254      27.271  -7.756 -31.458  1.00  0.53           C  \\nATOM   1969  CB  ASP A 254      27.170 -10.208 -30.951  1.00  0.51           C  \\nATOM   1970  O   ASP A 254      28.287  -8.023 -32.104  1.00  0.52           O  \\nATOM   1971  CG  ASP A 254      27.420 -10.874 -29.610  1.00  0.51           C  \\nATOM   1972  OD1 ASP A 254      27.381 -10.183 -28.569  1.00  0.52           O  \\nATOM   1973  OD2 ASP A 254      27.663 -12.100 -29.593  1.00  0.56           O  \\nATOM   1974  N   ARG A 255      26.728  -6.623 -31.811  1.00  0.55           N  \\nATOM   1975  CA  ARG A 255      27.563  -5.480 -32.165  1.00  0.54           C  \\nATOM   1976  C   ARG A 255      27.672  -4.500 -31.001  1.00  0.54           C  \\nATOM   1977  CB  ARG A 255      27.006  -4.767 -33.399  1.00  0.51           C  \\nATOM   1978  O   ARG A 255      28.151  -3.377 -31.170  1.00  0.52           O  \\nATOM   1979  CG  ARG A 255      27.201  -5.537 -34.695  1.00  0.49           C  \\nATOM   1980  CD  ARG A 255      26.691  -4.756 -35.898  1.00  0.50           C  \\nATOM   1981  NE  ARG A 255      25.617  -5.466 -36.587  1.00  0.44           N  \\nATOM   1982  NH1 ARG A 255      25.381  -3.901 -38.269  1.00  0.34           N  \\nATOM   1983  NH2 ARG A 255      24.054  -5.769 -38.241  1.00  0.30           N  \\nATOM   1984  CZ  ARG A 255      25.020  -5.044 -37.698  1.00  0.44           C  \\nATOM   1985  N   PHE A 256      27.488  -4.946 -29.786  1.00  0.53           N  \\nATOM   1986  CA  PHE A 256      28.034  -4.226 -28.642  1.00  0.54           C  \\nATOM   1987  C   PHE A 256      29.162  -5.019 -27.991  1.00  0.53           C  \\nATOM   1988  CB  PHE A 256      26.936  -3.935 -27.614  1.00  0.49           C  \\nATOM   1989  O   PHE A 256      29.286  -5.040 -26.765  1.00  0.50           O  \\nATOM   1990  CG  PHE A 256      26.198  -2.647 -27.863  1.00  0.47           C  \\nATOM   1991  CD1 PHE A 256      26.705  -1.439 -27.401  1.00  0.44           C  \\nATOM   1992  CD2 PHE A 256      24.997  -2.646 -28.560  1.00  0.45           C  \\nATOM   1993  CE1 PHE A 256      26.024  -0.246 -27.630  1.00  0.45           C  \\nATOM   1994  CE2 PHE A 256      24.310  -1.457 -28.792  1.00  0.47           C  \\nATOM   1995  CZ  PHE A 256      24.825  -0.258 -28.326  1.00  0.40           C  \\nATOM   1996  N   ARG A 257      29.958  -5.782 -28.781  1.00  0.55           N  \\nATOM   1997  CA  ARG A 257      31.232  -6.275 -28.268  1.00  0.55           C  \\nATOM   1998  C   ARG A 257      32.343  -6.097 -29.297  1.00  0.55           C  \\nATOM   1999  CB  ARG A 257      31.118  -7.748 -27.870  1.00  0.51           C  \\nATOM   2000  O   ARG A 257      32.510  -6.931 -30.189  1.00  0.52           O  \\nATOM   2001  CG  ARG A 257      30.701  -7.964 -26.424  1.00  0.50           C  \\nATOM   2002  CD  ARG A 257      30.689  -9.441 -26.055  1.00  0.52           C  \\nATOM   2003  NE  ARG A 257      30.453  -9.639 -24.628  1.00  0.45           N  \\nATOM   2004  NH1 ARG A 257      30.466 -11.947 -24.735  1.00  0.35           N  \\nATOM   2005  NH2 ARG A 257      30.137 -10.889 -22.728  1.00  0.31           N  \\nATOM   2006  CZ  ARG A 257      30.353 -10.825 -24.033  1.00  0.46           C  \\nATOM   2007  N   GLU A 258      32.661  -4.877 -29.729  1.00  0.53           N  \\nATOM   2008  CA  GLU A 258      33.960  -4.546 -30.309  1.00  0.54           C  \\nATOM   2009  C   GLU A 258      34.261  -3.056 -30.174  1.00  0.53           C  \\nATOM   2010  CB  GLU A 258      34.014  -4.963 -31.781  1.00  0.50           C  \\nATOM   2011  O   GLU A 258      33.796  -2.249 -30.981  1.00  0.51           O  \\nATOM   2012  CG  GLU A 258      34.723  -6.289 -32.019  1.00  0.50           C  \\nATOM   2013  CD  GLU A 258      34.663  -6.750 -33.466  1.00  0.53           C  \\nATOM   2014  OE1 GLU A 258      33.646  -6.483 -34.146  1.00  0.54           O  \\nATOM   2015  OE2 GLU A 258      35.640  -7.384 -33.924  1.00  0.50           O  \\nATOM   2016  N   GLY A 259      34.593  -2.666 -28.859  1.00  0.49           N  \\nATOM   2017  CA  GLY A 259      35.418  -1.472 -28.760  1.00  0.50           C  \\nATOM   2018  C   GLY A 259      36.450  -1.550 -27.651  1.00  0.49           C  \\nATOM   2019  O   GLY A 259      36.110  -1.820 -26.497  1.00  0.46           O  \\nATOM   2020  N   ASP A 260      37.626  -2.400 -27.892  1.00  0.47           N  \\nATOM   2021  CA  ASP A 260      38.985  -1.870 -27.819  1.00  0.49           C  \\nATOM   2022  C   ASP A 260      39.625  -2.185 -26.469  1.00  0.47           C  \\nATOM   2023  CB  ASP A 260      38.986  -0.360 -28.063  1.00  0.43           C  \\nATOM   2024  O   ASP A 260      39.156  -1.717 -25.430  1.00  0.45           O  \\nATOM   2025  CG  ASP A 260      40.274   0.138 -28.695  1.00  0.44           C  \\nATOM   2026  OD1 ASP A 260      41.148  -0.689 -29.031  1.00  0.45           O  \\nATOM   2027  OD2 ASP A 260      40.417   1.370 -28.855  1.00  0.49           O  \\nATOM   2028  N   ARG A 261      40.274  -3.344 -26.364  1.00  0.53           N  \\nATOM   2029  CA  ARG A 261      41.714  -3.555 -26.251  1.00  0.54           C  \\nATOM   2030  C   ARG A 261      42.328  -2.607 -25.227  1.00  0.52           C  \\nATOM   2031  CB  ARG A 261      42.393  -3.371 -27.610  1.00  0.50           C  \\nATOM   2032  O   ARG A 261      42.540  -1.427 -25.515  1.00  0.50           O  \\nATOM   2033  CG  ARG A 261      42.182  -4.533 -28.567  1.00  0.51           C  \\nATOM   2034  CD  ARG A 261      42.958  -4.344 -29.863  1.00  0.54           C  \\nATOM   2035  NE  ARG A 261      42.577  -5.332 -30.869  1.00  0.46           N  \\nATOM   2036  NH1 ARG A 261      44.295  -4.818 -32.324  1.00  0.35           N  \\nATOM   2037  NH2 ARG A 261      42.787  -6.460 -32.858  1.00  0.29           N  \\nATOM   2038  CZ  ARG A 261      43.221  -5.534 -32.015  1.00  0.44           C  \\nATOM   2039  N   GLY A 262      42.154  -2.900 -23.887  1.00  0.50           N  \\nATOM   2040  CA  GLY A 262      43.246  -2.548 -22.994  1.00  0.51           C  \\nATOM   2041  C   GLY A 262      42.773  -2.021 -21.652  1.00  0.50           C  \\nATOM   2042  O   GLY A 262      42.635  -0.810 -21.468  1.00  0.48           O  \\nATOM   2043  N   GLY A 263      41.885  -2.781 -20.990  1.00  0.47           N  \\nATOM   2044  CA  GLY A 263      41.885  -2.603 -19.547  1.00  0.49           C  \\nATOM   2045  C   GLY A 263      42.130  -3.892 -18.786  1.00  0.48           C  \\nATOM   2046  O   GLY A 263      41.648  -4.955 -19.184  1.00  0.47           O  \\nATOM   2047  N   PRO A 264      43.218  -4.003 -18.201  1.00  0.53           N  \\nATOM   2048  CA  PRO A 264      43.436  -4.991 -17.141  1.00  0.53           C  \\nATOM   2049  C   PRO A 264      42.131  -5.532 -16.562  1.00  0.53           C  \\nATOM   2050  CB  PRO A 264      44.219  -4.207 -16.085  1.00  0.50           C  \\nATOM   2051  O   PRO A 264      41.140  -4.803 -16.474  1.00  0.51           O  \\nATOM   2052  CG  PRO A 264      43.783  -2.788 -16.260  1.00  0.49           C  \\nATOM   2053  CD  PRO A 264      43.373  -2.589 -17.691  1.00  0.51           C  \\nATOM   2054  N   PRO A 265      41.727  -6.899 -16.693  1.00  0.55           N  \\nATOM   2055  CA  PRO A 265      40.614  -7.517 -15.970  1.00  0.55           C  \\nATOM   2056  C   PRO A 265      40.122  -6.665 -14.802  1.00  0.55           C  \\nATOM   2057  CB  PRO A 265      41.207  -8.837 -15.471  1.00  0.52           C  \\nATOM   2058  O   PRO A 265      40.808  -6.553 -13.783  1.00  0.53           O  \\nATOM   2059  CG  PRO A 265      42.682  -8.600 -15.425  1.00  0.50           C  \\nATOM   2060  CD  PRO A 265      43.008  -7.449 -16.333  1.00  0.51           C  \\nATOM   2061  N   GLY A 266      39.324  -5.650 -14.894  1.00  0.45           N  \\nATOM   2062  CA  GLY A 266      38.557  -4.670 -14.141  1.00  0.46           C  \\nATOM   2063  C   GLY A 266      37.152  -5.137 -13.810  1.00  0.45           C  \\nATOM   2064  O   GLY A 266      36.342  -5.371 -14.710  1.00  0.44           O  \\nATOM   2065  N   GLY A 267      36.984  -5.935 -12.618  1.00  0.49           N  \\nATOM   2066  CA  GLY A 267      36.016  -6.143 -11.553  1.00  0.50           C  \\nATOM   2067  C   GLY A 267      34.657  -5.539 -11.856  1.00  0.49           C  \\nATOM   2068  O   GLY A 267      34.558  -4.354 -12.181  1.00  0.48           O  \\nATOM   2069  N   SER A 268      33.814  -6.181 -12.697  1.00  0.53           N  \\nATOM   2070  CA  SER A 268      32.394  -5.888 -12.860  1.00  0.54           C  \\nATOM   2071  C   SER A 268      31.901  -4.923 -11.786  1.00  0.52           C  \\nATOM   2072  CB  SER A 268      31.571  -7.176 -12.815  1.00  0.50           C  \\nATOM   2073  O   SER A 268      32.171  -5.118 -10.599  1.00  0.51           O  \\nATOM   2074  OG  SER A 268      30.187  -6.893 -12.923  1.00  0.49           O  \\nATOM   2075  N   ARG A 269      31.899  -3.565 -11.942  1.00  0.52           N  \\nATOM   2076  CA  ARG A 269      31.395  -2.522 -11.054  1.00  0.51           C  \\nATOM   2077  C   ARG A 269      30.289  -3.059 -10.152  1.00  0.52           C  \\nATOM   2078  CB  ARG A 269      30.880  -1.330 -11.862  1.00  0.49           C  \\nATOM   2079  O   ARG A 269      29.973  -2.458  -9.123  1.00  0.50           O  \\nATOM   2080  CG  ARG A 269      31.806  -0.124 -11.840  1.00  0.47           C  \\nATOM   2081  CD  ARG A 269      31.232   1.042 -12.633  1.00  0.49           C  \\nATOM   2082  NE  ARG A 269      32.131   2.193 -12.626  1.00  0.43           N  \\nATOM   2083  NH1 ARG A 269      30.761   3.534 -13.915  1.00  0.33           N  \\nATOM   2084  NH2 ARG A 269      32.774   4.331 -13.164  1.00  0.28           N  \\nATOM   2085  CZ  ARG A 269      31.887   3.350 -13.235  1.00  0.44           C  \\nATOM   2086  N   PHE A 270      29.879  -4.241 -10.165  1.00  0.49           N  \\nATOM   2087  CA  PHE A 270      29.106  -4.957  -9.157  1.00  0.50           C  \\nATOM   2088  C   PHE A 270      30.020  -5.786  -8.262  1.00  0.49           C  \\nATOM   2089  CB  PHE A 270      28.061  -5.860  -9.820  1.00  0.46           C  \\nATOM   2090  O   PHE A 270      29.695  -6.040  -7.100  1.00  0.47           O  \\nATOM   2091  CG  PHE A 270      26.651  -5.348  -9.698  1.00  0.44           C  \\nATOM   2092  CD1 PHE A 270      25.894  -5.618  -8.565  1.00  0.41           C  \\nATOM   2093  CD2 PHE A 270      26.083  -4.595 -10.717  1.00  0.43           C  \\nATOM   2094  CE1 PHE A 270      24.589  -5.145  -8.449  1.00  0.43           C  \\nATOM   2095  CE2 PHE A 270      24.780  -4.119 -10.609  1.00  0.45           C  \\nATOM   2096  CZ  PHE A 270      24.034  -4.396  -9.475  1.00  0.38           C  \\nATOM   2097  N   ASP A 271      31.299  -6.136  -8.686  1.00  0.44           N  \\nATOM   2098  CA  ASP A 271      32.333  -6.704  -7.827  1.00  0.44           C  \\nATOM   2099  C   ASP A 271      33.105  -5.607  -7.097  1.00  0.44           C  \\nATOM   2100  CB  ASP A 271      33.294  -7.570  -8.643  1.00  0.42           C  \\nATOM   2101  O   ASP A 271      33.564  -5.808  -5.971  1.00  0.42           O  \\nATOM   2102  CG  ASP A 271      32.651  -8.846  -9.158  1.00  0.42           C  \\nATOM   2103  OD1 ASP A 271      31.917  -9.510  -8.395  1.00  0.45           O  \\nATOM   2104  OD2 ASP A 271      32.884  -9.192 -10.337  1.00  0.47           O  \\nATOM   2105  N   GLN A 272      32.999  -4.352  -7.505  1.00  0.45           N  \\nATOM   2106  CA  GLN A 272      33.639  -3.324  -6.691  1.00  0.43           C  \\nATOM   2107  C   GLN A 272      32.657  -2.727  -5.687  1.00  0.44           C  \\nATOM   2108  CB  GLN A 272      34.219  -2.221  -7.578  1.00  0.42           C  \\nATOM   2109  O   GLN A 272      33.049  -2.338  -4.585  1.00  0.42           O  \\nATOM   2110  CG  GLN A 272      35.593  -2.548  -8.147  1.00  0.40           C  \\nATOM   2111  CD  GLN A 272      36.141  -1.442  -9.029  1.00  0.43           C  \\nATOM   2112  NE2 GLN A 272      37.419  -1.541  -9.379  1.00  0.36           N  \\nATOM   2113  OE1 GLN A 272      35.422  -0.506  -9.393  1.00  0.45           O  \\nATOM   2114  N   VAL A 273      31.380  -3.022  -5.790  1.00  0.46           N  \\nATOM   2115  CA  VAL A 273      30.532  -2.721  -4.641  1.00  0.44           C  \\nATOM   2116  C   VAL A 273      30.412  -3.955  -3.750  1.00  0.46           C  \\nATOM   2117  CB  VAL A 273      29.130  -2.242  -5.079  1.00  0.43           C  \\nATOM   2118  O   VAL A 273      30.254  -3.837  -2.533  1.00  0.44           O  \\nATOM   2119  CG1 VAL A 273      28.257  -1.943  -3.862  1.00  0.41           C  \\nATOM   2120  CG2 VAL A 273      29.244  -1.010  -5.976  1.00  0.42           C  \\nATOM   2121  N   LYS A 274      30.837  -5.147  -4.212  1.00  0.46           N  \\nATOM   2122  CA  LYS A 274      31.123  -6.263  -3.315  1.00  0.44           C  \\nATOM   2123  C   LYS A 274      32.594  -6.280  -2.910  1.00  0.46           C  \\nATOM   2124  CB  LYS A 274      30.746  -7.592  -3.972  1.00  0.43           C  \\nATOM   2125  O   LYS A 274      32.947  -6.805  -1.852  1.00  0.43           O  \\nATOM   2126  CG  LYS A 274      29.471  -8.214  -3.423  1.00  0.41           C  \\nATOM   2127  CD  LYS A 274      29.171  -9.552  -4.087  1.00  0.45           C  \\nATOM   2128  CE  LYS A 274      27.897 -10.178  -3.535  1.00  0.38           C  \\nATOM   2129  NZ  LYS A 274      27.542 -11.438  -4.255  1.00  0.38           N  \\nATOM   2130  N   ALA A 275      33.465  -5.700  -3.613  1.00  0.45           N  \\nATOM   2131  CA  ALA A 275      34.860  -5.744  -3.182  1.00  0.44           C  \\nATOM   2132  C   ALA A 275      35.204  -4.537  -2.314  1.00  0.45           C  \\nATOM   2133  CB  ALA A 275      35.789  -5.808  -4.391  1.00  0.42           C  \\nATOM   2134  O   ALA A 275      35.982  -4.650  -1.363  1.00  0.42           O  \\nATOM   2135  N   GLU A 276      34.440  -3.554  -2.377  1.00  0.45           N  \\nATOM   2136  CA  GLU A 276      34.737  -2.490  -1.422  1.00  0.45           C  \\nATOM   2137  C   GLU A 276      33.792  -2.544  -0.225  1.00  0.45           C  \\nATOM   2138  CB  GLU A 276      34.651  -1.120  -2.099  1.00  0.42           C  \\nATOM   2139  O   GLU A 276      34.075  -1.958   0.822  1.00  0.43           O  \\nATOM   2140  CG  GLU A 276      35.913  -0.726  -2.853  1.00  0.40           C  \\nATOM   2141  CD  GLU A 276      35.789   0.605  -3.577  1.00  0.45           C  \\nATOM   2142  OE1 GLU A 276      34.653   1.109  -3.731  1.00  0.46           O  \\nATOM   2143  OE2 GLU A 276      36.836   1.148  -3.995  1.00  0.42           O  \\nATOM   2144  N   LEU A 277      32.981  -3.574  -0.179  1.00  0.46           N  \\nATOM   2145  CA  LEU A 277      32.485  -4.054   1.106  1.00  0.45           C  \\nATOM   2146  C   LEU A 277      33.315  -5.233   1.603  1.00  0.46           C  \\nATOM   2147  CB  LEU A 277      31.013  -4.461   0.995  1.00  0.43           C  \\nATOM   2148  O   LEU A 277      32.874  -5.984   2.475  1.00  0.44           O  \\nATOM   2149  CG  LEU A 277      29.984  -3.355   1.233  1.00  0.41           C  \\nATOM   2150  CD1 LEU A 277      28.711  -3.636   0.441  1.00  0.38           C  \\nATOM   2151  CD2 LEU A 277      29.676  -3.223   2.721  1.00  0.40           C  \\nATOM   2152  N   LYS A 278      34.383  -5.583   0.945  1.00  0.47           N  \\nATOM   2153  CA  LYS A 278      35.392  -6.431   1.572  1.00  0.47           C  \\nATOM   2154  C   LYS A 278      36.608  -5.615   2.001  1.00  0.47           C  \\nATOM   2155  CB  LYS A 278      35.821  -7.549   0.620  1.00  0.44           C  \\nATOM   2156  O   LYS A 278      37.476  -6.114   2.719  1.00  0.44           O  \\nATOM   2157  CG  LYS A 278      35.107  -8.872   0.857  1.00  0.43           C  \\nATOM   2158  CD  LYS A 278      35.584  -9.945  -0.113  1.00  0.47           C  \\nATOM   2159  CE  LYS A 278      34.907 -11.283   0.156  1.00  0.40           C  \\nATOM   2160  NZ  LYS A 278      35.376 -12.341  -0.788  1.00  0.38           N  \\nATOM   2161  N   GLU A 279      36.510  -4.403   2.420  1.00  0.49           N  \\nATOM   2162  CA  GLU A 279      37.608  -3.816   3.181  1.00  0.49           C  \\nATOM   2163  C   GLU A 279      37.117  -2.679   4.073  1.00  0.48           C  \\nATOM   2164  CB  GLU A 279      38.704  -3.309   2.239  1.00  0.45           C  \\nATOM   2165  O   GLU A 279      36.689  -1.635   3.577  1.00  0.46           O  \\nATOM   2166  CG  GLU A 279      39.626  -4.404   1.723  1.00  0.45           C  \\nATOM   2167  CD  GLU A 279      40.704  -3.887   0.784  1.00  0.49           C  \\nATOM   2168  OE1 GLU A 279      40.531  -2.790   0.206  1.00  0.50           O  \\nATOM   2169  OE2 GLU A 279      41.730  -4.586   0.624  1.00  0.45           O  \\nATOM   2170  N   GLU A 280      36.213  -3.038   4.984  1.00  0.51           N  \\nATOM   2171  CA  GLU A 280      36.273  -2.485   6.334  1.00  0.52           C  \\nATOM   2172  C   GLU A 280      35.286  -3.187   7.263  1.00  0.51           C  \\nATOM   2173  CB  GLU A 280      35.994  -0.980   6.311  1.00  0.48           C  \\nATOM   2174  O   GLU A 280      34.111  -2.819   7.321  1.00  0.49           O  \\nATOM   2175  CG  GLU A 280      37.161  -0.131   6.795  1.00  0.46           C  \\nATOM   2176  CD  GLU A 280      36.898   1.363   6.695  1.00  0.50           C  \\nATOM   2177  OE1 GLU A 280      35.921   1.764   6.021  1.00  0.51           O  \\nATOM   2178  OE2 GLU A 280      37.675   2.140   7.295  1.00  0.47           O  \\nATOM   2179  N   ILE A 281      35.223  -4.529   7.198  1.00  0.52           N  \\nATOM   2180  CA  ILE A 281      34.979  -5.199   8.470  1.00  0.52           C  \\nATOM   2181  C   ILE A 281      36.056  -4.802   9.478  1.00  0.52           C  \\nATOM   2182  CB  ILE A 281      34.942  -6.734   8.303  1.00  0.49           C  \\nATOM   2183  O   ILE A 281      37.177  -5.314   9.431  1.00  0.50           O  \\nATOM   2184  CG1 ILE A 281      33.832  -7.138   7.326  1.00  0.46           C  \\nATOM   2185  CG2 ILE A 281      34.756  -7.421   9.659  1.00  0.47           C  \\nATOM   2186  CD1 ILE A 281      33.848  -8.612   6.945  1.00  0.45           C  \\nATOM   2187  N   VAL A 282      36.169  -3.501   9.792  1.00  0.54           N  \\nATOM   2188  CA  VAL A 282      36.747  -3.216  11.101  1.00  0.55           C  \\nATOM   2189  C   VAL A 282      36.133  -4.142  12.149  1.00  0.54           C  \\nATOM   2190  CB  VAL A 282      36.537  -1.740  11.506  1.00  0.51           C  \\nATOM   2191  O   VAL A 282      34.914  -4.160  12.333  1.00  0.52           O  \\nATOM   2192  CG1 VAL A 282      37.257  -1.432  12.818  1.00  0.48           C  \\nATOM   2193  CG2 VAL A 282      37.021  -0.809  10.396  1.00  0.49           C  \\nATOM   2194  N   GLU A 283      36.814  -5.327  12.330  1.00  0.58           N  \\nATOM   2195  CA  GLU A 283      36.814  -6.164  13.526  1.00  0.59           C  \\nATOM   2196  C   GLU A 283      36.070  -5.487  14.673  1.00  0.59           C  \\nATOM   2197  CB  GLU A 283      38.247  -6.495  13.951  1.00  0.55           C  \\nATOM   2198  O   GLU A 283      36.370  -4.344  15.026  1.00  0.57           O  \\nATOM   2199  CG  GLU A 283      38.942  -7.498  13.042  1.00  0.54           C  \\nATOM   2200  CD  GLU A 283      40.187  -8.110  13.664  1.00  0.56           C  \\nATOM   2201  OE1 GLU A 283      40.803  -7.469  14.546  1.00  0.56           O  \\nATOM   2202  OE2 GLU A 283      40.550  -9.239  13.267  1.00  0.51           O  \\nATOM   2203  N   PRO A 284      34.782  -5.674  14.882  1.00  0.58           N  \\nATOM   2204  CA  PRO A 284      34.278  -5.424  16.234  1.00  0.57           C  \\nATOM   2205  C   PRO A 284      35.269  -5.837  17.320  1.00  0.58           C  \\nATOM   2206  CB  PRO A 284      33.009  -6.277  16.300  1.00  0.54           C  \\nATOM   2207  O   PRO A 284      35.594  -7.021  17.446  1.00  0.56           O  \\nATOM   2208  CG  PRO A 284      33.271  -7.424  15.379  1.00  0.52           C  \\nATOM   2209  CD  PRO A 284      34.248  -6.978  14.329  1.00  0.54           C  \\nATOM   2210  N   ASN A 285      36.199  -4.957  17.453  1.00  0.57           N  \\nATOM   2211  CA  ASN A 285      36.892  -5.016  18.735  1.00  0.59           C  \\nATOM   2212  C   ASN A 285      35.980  -5.538  19.842  1.00  0.57           C  \\nATOM   2213  CB  ASN A 285      37.445  -3.639  19.110  1.00  0.53           C  \\nATOM   2214  O   ASN A 285      35.021  -4.868  20.230  1.00  0.55           O  \\nATOM   2215  CG  ASN A 285      38.495  -3.709  20.201  1.00  0.52           C  \\nATOM   2216  ND2 ASN A 285      38.872  -2.554  20.736  1.00  0.51           N  \\nATOM   2217  OD1 ASN A 285      38.964  -4.792  20.560  1.00  0.52           O  \\nATOM   2218  N   ASN A 286      35.871  -6.822  19.921  1.00  0.58           N  \\nATOM   2219  CA  ASN A 286      35.836  -7.717  21.073  1.00  0.59           C  \\nATOM   2220  C   ASN A 286      36.358  -7.031  22.332  1.00  0.57           C  \\nATOM   2221  CB  ASN A 286      36.638  -8.989  20.789  1.00  0.53           C  \\nATOM   2222  O   ASN A 286      37.560  -7.056  22.604  1.00  0.55           O  \\nATOM   2223  CG  ASN A 286      35.754 -10.180  20.477  1.00  0.52           C  \\nATOM   2224  ND2 ASN A 286      36.364 -11.272  20.033  1.00  0.52           N  \\nATOM   2225  OD1 ASN A 286      34.531 -10.119  20.634  1.00  0.53           O  \\nATOM   2226  N   GLY A 287      35.687  -5.966  22.674  1.00  0.59           N  \\nATOM   2227  CA  GLY A 287      35.809  -5.721  24.102  1.00  0.59           C  \\nATOM   2228  C   GLY A 287      34.640  -6.260  24.903  1.00  0.58           C  \\nATOM   2229  O   GLY A 287      33.490  -6.179  24.465  1.00  0.55           O  \\nATOM   2230  N   PRO A 288      35.008  -7.325  25.689  1.00  0.61           N  \\nATOM   2231  CA  PRO A 288      34.079  -7.883  26.675  1.00  0.60           C  \\nATOM   2232  C   PRO A 288      33.074  -6.853  27.187  1.00  0.60           C  \\nATOM   2233  CB  PRO A 288      35.001  -8.353  27.802  1.00  0.56           C  \\nATOM   2234  O   PRO A 288      33.422  -5.683  27.369  1.00  0.57           O  \\nATOM   2235  CG  PRO A 288      36.188  -7.448  27.728  1.00  0.55           C  \\nATOM   2236  CD  PRO A 288      36.352  -6.992  26.307  1.00  0.57           C  \\nATOM   2237  N   PRO A 289      31.805  -7.000  26.887  1.00  0.62           N  \\nATOM   2238  CA  PRO A 289      30.768  -6.288  27.637  1.00  0.61           C  \\nATOM   2239  C   PRO A 289      31.232  -5.866  29.029  1.00  0.62           C  \\nATOM   2240  CB  PRO A 289      29.633  -7.311  27.726  1.00  0.58           C  \\nATOM   2241  O   PRO A 289      31.582  -6.716  29.851  1.00  0.60           O  \\nATOM   2242  CG  PRO A 289      30.314  -8.641  27.707  1.00  0.56           C  \\nATOM   2243  CD  PRO A 289      31.614  -8.501  26.968  1.00  0.57           C  \\nATOM   2244  N   THR A 290      31.786  -4.744  29.191  1.00  0.60           N  \\nATOM   2245  CA  THR A 290      31.776  -4.131  30.514  1.00  0.60           C  \\nATOM   2246  C   THR A 290      30.349  -3.997  31.037  1.00  0.58           C  \\nATOM   2247  CB  THR A 290      32.451  -2.747  30.494  1.00  0.56           C  \\nATOM   2248  O   THR A 290      29.522  -3.314  30.429  1.00  0.56           O  \\nATOM   2249  CG2 THR A 290      33.858  -2.813  31.078  1.00  0.52           C  \\nATOM   2250  OG1 THR A 290      32.528  -2.279  29.142  1.00  0.55           O  \\nATOM   2251  N   THR A 291      29.916  -5.004  31.776  1.00  0.63           N  \\nATOM   2252  CA  THR A 291      29.018  -5.148  32.916  1.00  0.63           C  \\nATOM   2253  C   THR A 291      29.143  -3.954  33.858  1.00  0.61           C  \\nATOM   2254  CB  THR A 291      29.304  -6.448  33.691  1.00  0.59           C  \\nATOM   2255  O   THR A 291      30.151  -3.808  34.553  1.00  0.58           O  \\nATOM   2256  CG2 THR A 291      28.505  -7.615  33.121  1.00  0.57           C  \\nATOM   2257  OG1 THR A 291      30.702  -6.752  33.603  1.00  0.60           O  \\nATOM   2258  N   SER A 292      28.627  -2.891  33.519  1.00  0.55           N  \\nATOM   2259  CA  SER A 292      28.298  -1.985  34.614  1.00  0.56           C  \\nATOM   2260  C   SER A 292      26.793  -1.930  34.853  1.00  0.54           C  \\nATOM   2261  CB  SER A 292      28.828  -0.580  34.324  1.00  0.51           C  \\nATOM   2262  O   SER A 292      26.021  -1.673  33.927  1.00  0.51           O  \\nATOM   2263  OG  SER A 292      28.192  -0.027  33.184  1.00  0.53           O  \\nATOM   2264  N   SER A 293      26.422  -2.658  35.857  1.00  0.55           N  \\nATOM   2265  CA  SER A 293      25.585  -2.599  37.051  1.00  0.56           C  \\nATOM   2266  C   SER A 293      24.817  -1.284  37.123  1.00  0.54           C  \\nATOM   2267  CB  SER A 293      26.435  -2.772  38.311  1.00  0.51           C  \\nATOM   2268  O   SER A 293      25.395  -0.237  37.420  1.00  0.52           O  \\nATOM   2269  OG  SER A 293      26.476  -1.570  39.059  1.00  0.56           O  \\nATOM   2270  N   GLY A 294      23.743  -1.183  36.278  1.00  0.49           N  \\nATOM   2271  CA  GLY A 294      22.728  -0.299  36.828  1.00  0.50           C  \\nATOM   2272  C   GLY A 294      21.314  -0.801  36.601  1.00  0.50           C  \\nATOM   2273  O   GLY A 294      20.878  -0.950  35.458  1.00  0.47           O  \\nATOM   2274  N   ALA A 295      20.838  -1.818  37.362  1.00  0.49           N  \\nATOM   2275  CA  ALA A 295      19.691  -2.277  38.140  1.00  0.51           C  \\nATOM   2276  C   ALA A 295      18.585  -1.226  38.159  1.00  0.49           C  \\nATOM   2277  CB  ALA A 295      20.118  -2.621  39.565  1.00  0.48           C  \\nATOM   2278  O   ALA A 295      18.727  -0.177  38.792  1.00  0.47           O  \\nATOM   2279  N   ALA A 296      18.047  -0.831  36.963  1.00  0.50           N  \\nATOM   2280  CA  ALA A 296      16.686  -0.325  37.125  1.00  0.51           C  \\nATOM   2281  C   ALA A 296      16.115   0.148  35.792  1.00  0.49           C  \\nATOM   2282  CB  ALA A 296      16.658   0.809  38.146  1.00  0.47           C  \\nATOM   2283  O   ALA A 296      14.949   0.542  35.714  1.00  0.47           O  \\nATOM   2284  N   ALA A 297      16.465  -0.512  34.619  1.00  0.48           N  \\nATOM   2285  CA  ALA A 297      15.645  -0.291  33.431  1.00  0.48           C  \\nATOM   2286  C   ALA A 297      15.660  -1.515  32.519  1.00  0.47           C  \\nATOM   2287  CB  ALA A 297      16.132   0.941  32.671  1.00  0.45           C  \\nATOM   2288  O   ALA A 297      15.277  -1.430  31.350  1.00  0.46           O  \\nATOM   2289  N   ALA A 298      16.139  -2.681  33.043  1.00  0.44           N  \\nATOM   2290  CA  ALA A 298      16.181  -3.988  32.394  1.00  0.43           C  \\nATOM   2291  C   ALA A 298      14.911  -4.785  32.681  1.00  0.43           C  \\nATOM   2292  CB  ALA A 298      17.410  -4.770  32.851  1.00  0.41           C  \\nATOM   2293  O   ALA A 298      14.668  -5.822  32.059  1.00  0.42           O  \\nATOM   2294  N   SER A 299      13.920  -4.226  33.301  1.00  0.44           N  \\nATOM   2295  CA  SER A 299      12.731  -5.028  33.573  1.00  0.45           C  \\nATOM   2296  C   SER A 299      11.592  -4.667  32.626  1.00  0.44           C  \\nATOM   2297  CB  SER A 299      12.280  -4.842  35.022  1.00  0.42           C  \\nATOM   2298  O   SER A 299      10.747  -5.508  32.314  1.00  0.43           O  \\nATOM   2299  OG  SER A 299      11.203  -5.711  35.329  1.00  0.43           O  \\nATOM   2300  N   ALA A 300      11.810  -3.782  31.677  1.00  0.45           N  \\nATOM   2301  CA  ALA A 300      10.673  -3.500  30.806  1.00  0.45           C  \\nATOM   2302  C   ALA A 300      11.014  -3.790  29.347  1.00  0.45           C  \\nATOM   2303  CB  ALA A 300      10.226  -2.049  30.967  1.00  0.42           C  \\nATOM   2304  O   ALA A 300      10.134  -4.128  28.552  1.00  0.43           O  \\nATOM   2305  N   ALA A 301      12.262  -4.064  29.018  1.00  0.44           N  \\nATOM   2306  CA  ALA A 301      12.732  -4.493  27.704  1.00  0.43           C  \\nATOM   2307  C   ALA A 301      12.754  -6.016  27.600  1.00  0.44           C  \\nATOM   2308  CB  ALA A 301      14.120  -3.924  27.423  1.00  0.42           C  \\nATOM   2309  O   ALA A 301      12.578  -6.572  26.513  1.00  0.42           O  \\nATOM   2310  N   ALA A 302      12.716  -6.739  28.741  1.00  0.43           N  \\nATOM   2311  CA  ALA A 302      12.759  -8.197  28.802  1.00  0.43           C  \\nATOM   2312  C   ALA A 302      11.354  -8.790  28.755  1.00  0.43           C  \\nATOM   2313  CB  ALA A 302      13.485  -8.655  30.065  1.00  0.42           C  \\nATOM   2314  O   ALA A 302      11.160  -9.905  28.265  1.00  0.42           O  \\nATOM   2315  N   VAL A 303      10.277  -7.959  28.962  1.00  0.44           N  \\nATOM   2316  CA  VAL A 303       8.924  -8.506  28.953  1.00  0.43           C  \\nATOM   2317  C   VAL A 303       8.288  -8.288  27.582  1.00  0.44           C  \\nATOM   2318  CB  VAL A 303       8.048  -7.869  30.055  1.00  0.41           C  \\nATOM   2319  O   VAL A 303       7.428  -9.065  27.160  1.00  0.42           O  \\nATOM   2320  CG1 VAL A 303       6.642  -8.468  30.039  1.00  0.39           C  \\nATOM   2321  CG2 VAL A 303       8.696  -8.055  31.425  1.00  0.40           C  \\nATOM   2322  N   VAL A 304       8.891  -7.513  26.714  1.00  0.45           N  \\nATOM   2323  CA  VAL A 304       8.337  -7.450  25.365  1.00  0.44           C  \\nATOM   2324  C   VAL A 304       9.118  -8.382  24.442  1.00  0.45           C  \\nATOM   2325  CB  VAL A 304       8.359  -6.008  24.810  1.00  0.43           C  \\nATOM   2326  O   VAL A 304       8.564  -8.919  23.479  1.00  0.44           O  \\nATOM   2327  CG1 VAL A 304       7.725  -5.955  23.421  1.00  0.40           C  \\nATOM   2328  CG2 VAL A 304       7.641  -5.058  25.765  1.00  0.41           C  \\nATOM   2329  N   ALA A 305      10.275  -8.872  24.866  1.00  0.45           N  \\nATOM   2330  CA  ALA A 305      11.052  -9.881  24.149  1.00  0.45           C  \\nATOM   2331  C   ALA A 305      10.648 -11.289  24.575  1.00  0.45           C  \\nATOM   2332  CB  ALA A 305      12.546  -9.666  24.382  1.00  0.43           C  \\nATOM   2333  O   ALA A 305      10.726 -12.230  23.782  1.00  0.44           O  \\nATOM   2334  N   SER A 306       9.957 -11.475  25.676  1.00  0.44           N  \\nATOM   2335  CA  SER A 306       9.628 -12.825  26.122  1.00  0.44           C  \\nATOM   2336  C   SER A 306       8.172 -13.167  25.821  1.00  0.44           C  \\nATOM   2337  CB  SER A 306       9.896 -12.975  27.620  1.00  0.41           C  \\nATOM   2338  O   SER A 306       7.814 -14.342  25.719  1.00  0.42           O  \\nATOM   2339  OG  SER A 306       8.857 -12.382  28.379  1.00  0.42           O  \\nATOM   2340  N   LYS A 307       7.389 -12.143  25.293  1.00  0.48           N  \\nATOM   2341  CA  LYS A 307       6.040 -12.531  24.892  1.00  0.47           C  \\nATOM   2342  C   LYS A 307       5.862 -12.421  23.381  1.00  0.47           C  \\nATOM   2343  CB  LYS A 307       4.998 -11.668  25.606  1.00  0.44           C  \\nATOM   2344  O   LYS A 307       4.769 -12.651  22.860  1.00  0.45           O  \\nATOM   2345  CG  LYS A 307       4.573 -12.207  26.964  1.00  0.43           C  \\nATOM   2346  CD  LYS A 307       3.482 -11.348  27.590  1.00  0.47           C  \\nATOM   2347  CE  LYS A 307       3.036 -11.903  28.936  1.00  0.37           C  \\nATOM   2348  NZ  LYS A 307       1.978 -11.054  29.562  1.00  0.36           N  \\nATOM   2349  N   LEU A 308       6.985 -12.265  22.644  1.00  0.48           N  \\nATOM   2350  CA  LEU A 308       7.165 -12.755  21.282  1.00  0.48           C  \\nATOM   2351  C   LEU A 308       7.951 -14.061  21.274  1.00  0.47           C  \\nATOM   2352  CB  LEU A 308       7.882 -11.708  20.426  1.00  0.45           C  \\nATOM   2353  O   LEU A 308       9.184 -14.048  21.286  1.00  0.46           O  \\nATOM   2354  CG  LEU A 308       6.997 -10.653  19.760  1.00  0.44           C  \\nATOM   2355  CD1 LEU A 308       7.687  -9.294  19.777  1.00  0.42           C  \\nATOM   2356  CD2 LEU A 308       6.654 -11.067  18.333  1.00  0.43           C  \\nATOM   2357  N   GLY A 309       8.178 -14.785  22.335  1.00  0.42           N  \\nATOM   2358  CA  GLY A 309       8.827 -16.068  22.552  1.00  0.44           C  \\nATOM   2359  C   GLY A 309       7.873 -17.243  22.461  1.00  0.42           C  \\nATOM   2360  O   GLY A 309       6.952 -17.366  23.271  1.00  0.40           O  \\nATOM   2361  N   SER A 310       7.174 -17.457  21.299  1.00  0.41           N  \\nATOM   2362  CA  SER A 310       6.710 -18.786  20.915  1.00  0.46           C  \\nATOM   2363  C   SER A 310       5.633 -18.707  19.839  1.00  0.41           C  \\nATOM   2364  CB  SER A 310       6.172 -19.539  22.132  1.00  0.39           C  \\nATOM   2365  O   SER A 310       4.528 -18.223  20.094  1.00  0.40           O  \\nATOM   2366  OG  SER A 310       5.151 -20.446  21.752  1.00  0.39           O  \\nATOM   2367  N   ARG A 311       5.975 -18.021  18.609  1.00  0.50           N  \\nATOM   2368  CA  ARG A 311       5.501 -18.546  17.333  1.00  0.50           C  \\nATOM   2369  C   ARG A 311       6.192 -17.851  16.165  1.00  0.49           C  \\nATOM   2370  CB  ARG A 311       3.984 -18.384  17.215  1.00  0.46           C  \\nATOM   2371  O   ARG A 311       5.541 -17.181  15.361  1.00  0.47           O  \\nATOM   2372  CG  ARG A 311       3.192 -19.384  18.042  1.00  0.45           C  \\nATOM   2373  CD  ARG A 311       1.693 -19.243  17.815  1.00  0.48           C  \\nATOM   2374  NE  ARG A 311       0.943 -20.294  18.495  1.00  0.41           N  \\nATOM   2375  NH1 ARG A 311      -1.117 -19.637  17.683  1.00  0.31           N  \\nATOM   2376  NH2 ARG A 311      -0.955 -21.449  19.078  1.00  0.26           N  \\nATOM   2377  CZ  ARG A 311      -0.375 -20.458  18.417  1.00  0.39           C  \\nATOM   2378  N   THR A 312       7.603 -17.677  16.256  1.00  0.47           N  \\nATOM   2379  CA  THR A 312       8.331 -17.609  14.994  1.00  0.48           C  \\nATOM   2380  C   THR A 312       8.864 -18.984  14.602  1.00  0.47           C  \\nATOM   2381  CB  THR A 312       9.498 -16.608  15.076  1.00  0.44           C  \\nATOM   2382  O   THR A 312       9.741 -19.531  15.274  1.00  0.45           O  \\nATOM   2383  CG2 THR A 312       9.093 -15.244  14.527  1.00  0.42           C  \\nATOM   2384  OG1 THR A 312       9.898 -16.460  16.444  1.00  0.39           O  \\nATOM   2385  N   ARG A 313       8.024 -19.978  14.435  1.00  0.48           N  \\nATOM   2386  CA  ARG A 313       8.503 -21.070  13.594  1.00  0.48           C  \\nATOM   2387  C   ARG A 313       9.437 -20.553  12.504  1.00  0.48           C  \\nATOM   2388  CB  ARG A 313       7.327 -21.820  12.964  1.00  0.45           C  \\nATOM   2389  O   ARG A 313       9.036 -19.738  11.671  1.00  0.45           O  \\nATOM   2390  CG  ARG A 313       7.047 -23.172  13.600  1.00  0.43           C  \\nATOM   2391  CD  ARG A 313       5.895 -23.891  12.912  1.00  0.46           C  \\nATOM   2392  NE  ARG A 313       5.620 -25.185  13.531  1.00  0.42           N  \\nATOM   2393  NH1 ARG A 313       3.937 -25.764  12.059  1.00  0.31           N  \\nATOM   2394  NH2 ARG A 313       4.530 -27.195  13.749  1.00  0.26           N  \\nATOM   2395  CZ  ARG A 313       4.696 -26.045  13.111  1.00  0.39           C  \\nATOM   2396  N   ILE A 314      10.731 -20.357  12.755  1.00  0.48           N  \\nATOM   2397  CA  ILE A 314      11.772 -20.387  11.733  1.00  0.47           C  \\nATOM   2398  C   ILE A 314      11.521 -21.549  10.775  1.00  0.48           C  \\nATOM   2399  CB  ILE A 314      13.178 -20.506  12.363  1.00  0.44           C  \\nATOM   2400  O   ILE A 314      11.452 -22.706  11.196  1.00  0.46           O  \\nATOM   2401  CG1 ILE A 314      13.436 -19.335  13.318  1.00  0.41           C  \\nATOM   2402  CG2 ILE A 314      14.253 -20.572  11.274  1.00  0.42           C  \\nATOM   2403  CD1 ILE A 314      14.729 -19.458  14.113  1.00  0.39           C  \\nATOM   2404  N   VAL A 315      10.766 -21.379   9.835  1.00  0.46           N  \\nATOM   2405  CA  VAL A 315      10.574 -22.016   8.536  1.00  0.49           C  \\nATOM   2406  C   VAL A 315      11.908 -22.098   7.799  1.00  0.47           C  \\nATOM   2407  CB  VAL A 315       9.537 -21.255   7.679  1.00  0.44           C  \\nATOM   2408  O   VAL A 315      12.665 -21.125   7.760  1.00  0.45           O  \\nATOM   2409  CG1 VAL A 315       9.187 -22.051   6.423  1.00  0.42           C  \\nATOM   2410  CG2 VAL A 315       8.281 -20.962   8.497  1.00  0.42           C  \\nATOM   2411  N   TYR A 316      12.702 -23.300   7.859  1.00  0.52           N  \\nATOM   2412  CA  TYR A 316      13.586 -24.137   7.055  1.00  0.53           C  \\nATOM   2413  C   TYR A 316      15.016 -23.612   7.091  1.00  0.53           C  \\nATOM   2414  CB  TYR A 316      13.091 -24.206   5.607  1.00  0.49           C  \\nATOM   2415  O   TYR A 316      15.244 -22.404   6.983  1.00  0.51           O  \\nATOM   2416  CG  TYR A 316      11.784 -24.942   5.447  1.00  0.48           C  \\nATOM   2417  CD1 TYR A 316      11.748 -26.334   5.385  1.00  0.45           C  \\nATOM   2418  CD2 TYR A 316      10.582 -24.249   5.355  1.00  0.46           C  \\nATOM   2419  CE1 TYR A 316      10.546 -27.017   5.235  1.00  0.45           C  \\nATOM   2420  CE2 TYR A 316       9.373 -24.922   5.206  1.00  0.48           C  \\nATOM   2421  OH  TYR A 316       8.172 -26.974   4.999  1.00  0.39           O  \\nATOM   2422  CZ  TYR A 316       9.365 -26.303   5.147  1.00  0.41           C  \\nATOM   2423  N   PRO A 317      16.005 -24.143   7.738  1.00  0.53           N  \\nATOM   2424  CA  PRO A 317      17.450 -23.925   7.844  1.00  0.51           C  \\nATOM   2425  C   PRO A 317      18.043 -23.281   6.593  1.00  0.53           C  \\nATOM   2426  CB  PRO A 317      18.005 -25.337   8.047  1.00  0.48           C  \\nATOM   2427  O   PRO A 317      19.007 -22.516   6.686  1.00  0.51           O  \\nATOM   2428  CG  PRO A 317      16.838 -26.243   7.820  1.00  0.46           C  \\nATOM   2429  CD  PRO A 317      15.595 -25.406   7.731  1.00  0.48           C  \\nATOM   2430  N   ASP A 318      17.435 -23.289   5.307  1.00  0.56           N  \\nATOM   2431  CA  ASP A 318      17.885 -22.813   4.003  1.00  0.56           C  \\nATOM   2432  C   ASP A 318      17.251 -21.466   3.661  1.00  0.56           C  \\nATOM   2433  CB  ASP A 318      17.560 -23.838   2.915  1.00  0.52           C  \\nATOM   2434  O   ASP A 318      17.399 -20.971   2.542  1.00  0.54           O  \\nATOM   2435  CG  ASP A 318      18.281 -25.160   3.112  1.00  0.50           C  \\nATOM   2436  OD1 ASP A 318      19.404 -25.167   3.661  1.00  0.51           O  \\nATOM   2437  OD2 ASP A 318      17.722 -26.205   2.713  1.00  0.53           O  \\nATOM   2438  N   ASP A 319      16.525 -20.736   4.583  1.00  0.56           N  \\nATOM   2439  CA  ASP A 319      15.743 -19.517   4.399  1.00  0.57           C  \\nATOM   2440  C   ASP A 319      16.638 -18.280   4.431  1.00  0.56           C  \\nATOM   2441  CB  ASP A 319      14.657 -19.409   5.471  1.00  0.52           C  \\nATOM   2442  O   ASP A 319      16.220 -17.194   4.024  1.00  0.55           O  \\nATOM   2443  CG  ASP A 319      13.278 -19.142   4.894  1.00  0.51           C  \\nATOM   2444  OD1 ASP A 319      13.156 -18.964   3.663  1.00  0.51           O  \\nATOM   2445  OD2 ASP A 319      12.305 -19.105   5.678  1.00  0.53           O  \\nATOM   2446  N   HIS A 320      17.976 -18.394   4.752  1.00  0.61           N  \\nATOM   2447  CA  HIS A 320      18.894 -17.266   4.643  1.00  0.61           C  \\nATOM   2448  C   HIS A 320      19.365 -17.075   3.205  1.00  0.61           C  \\nATOM   2449  CB  HIS A 320      20.097 -17.464   5.567  1.00  0.57           C  \\nATOM   2450  O   HIS A 320      19.919 -16.028   2.862  1.00  0.60           O  \\nATOM   2451  CG  HIS A 320      19.738 -17.512   7.018  1.00  0.55           C  \\nATOM   2452  CD2 HIS A 320      19.727 -18.541   7.898  1.00  0.51           C  \\nATOM   2453  ND1 HIS A 320      19.325 -16.400   7.720  1.00  0.49           N  \\nATOM   2454  CE1 HIS A 320      19.076 -16.745   8.972  1.00  0.48           C  \\nATOM   2455  NE2 HIS A 320      19.312 -18.039   9.106  1.00  0.44           N  \\nATOM   2456  N   SER A 321      19.080 -18.012   2.139  1.00  0.66           N  \\nATOM   2457  CA  SER A 321      19.461 -17.962   0.731  1.00  0.66           C  \\nATOM   2458  C   SER A 321      18.341 -17.380  -0.124  1.00  0.66           C  \\nATOM   2459  CB  SER A 321      19.826 -19.358   0.225  1.00  0.62           C  \\nATOM   2460  O   SER A 321      18.579 -16.932  -1.248  1.00  0.64           O  \\nATOM   2461  OG  SER A 321      18.743 -20.256   0.397  1.00  0.60           O  \\nATOM   2462  N   LEU A 322      17.048 -17.283   0.416  1.00  0.70           N  \\nATOM   2463  CA  LEU A 322      15.954 -16.797  -0.418  1.00  0.70           C  \\nATOM   2464  C   LEU A 322      15.627 -15.343  -0.092  1.00  0.70           C  \\nATOM   2465  CB  LEU A 322      14.708 -17.667  -0.228  1.00  0.67           C  \\nATOM   2466  O   LEU A 322      15.410 -14.997   1.071  1.00  0.68           O  \\nATOM   2467  CG  LEU A 322      14.761 -19.069  -0.836  1.00  0.64           C  \\nATOM   2468  CD1 LEU A 322      13.599 -19.913  -0.325  1.00  0.60           C  \\nATOM   2469  CD2 LEU A 322      14.744 -18.993  -2.359  1.00  0.61           C  \\nATOM   2470  N   SER A 323      15.655 -14.360  -1.044  1.00  0.76           N  \\nATOM   2471  CA  SER A 323      15.101 -13.013  -0.959  1.00  0.75           C  \\nATOM   2472  C   SER A 323      13.628 -13.044  -0.565  1.00  0.76           C  \\nATOM   2473  CB  SER A 323      15.265 -12.280  -2.291  1.00  0.73           C  \\nATOM   2474  O   SER A 323      12.966 -14.075  -0.701  1.00  0.72           O  \\nATOM   2475  OG  SER A 323      14.354 -12.779  -3.255  1.00  0.69           O  \\nATOM   2476  N   LEU A 324      12.976 -11.971   0.049  1.00  0.76           N  \\nATOM   2477  CA  LEU A 324      11.561 -11.850   0.387  1.00  0.76           C  \\nATOM   2478  C   LEU A 324      10.687 -12.118  -0.833  1.00  0.76           C  \\nATOM   2479  CB  LEU A 324      11.261 -10.459   0.949  1.00  0.73           C  \\nATOM   2480  O   LEU A 324       9.587 -12.661  -0.707  1.00  0.75           O  \\nATOM   2481  CG  LEU A 324      10.108 -10.366   1.951  1.00  0.67           C  \\nATOM   2482  CD1 LEU A 324      10.648 -10.180   3.365  1.00  0.62           C  \\nATOM   2483  CD2 LEU A 324       9.166  -9.227   1.578  1.00  0.63           C  \\nATOM   2484  N   GLU A 325      11.142 -11.700  -2.023  1.00  0.78           N  \\nATOM   2485  CA  GLU A 325      10.412 -11.927  -3.266  1.00  0.79           C  \\nATOM   2486  C   GLU A 325      10.334 -13.415  -3.597  1.00  0.78           C  \\nATOM   2487  CB  GLU A 325      11.065 -11.165  -4.422  1.00  0.75           C  \\nATOM   2488  O   GLU A 325       9.256 -13.936  -3.889  1.00  0.75           O  \\nATOM   2489  CG  GLU A 325      10.787  -9.669  -4.408  1.00  0.68           C  \\nATOM   2490  CD  GLU A 325      12.049  -8.824  -4.475  1.00  0.66           C  \\nATOM   2491  OE1 GLU A 325      13.162  -9.396  -4.508  1.00  0.70           O  \\nATOM   2492  OE2 GLU A 325      11.923  -7.579  -4.495  1.00  0.63           O  \\nATOM   2493  N   GLU A 326      11.520 -14.119  -3.530  1.00  0.78           N  \\nATOM   2494  CA  GLU A 326      11.565 -15.556  -3.783  1.00  0.77           C  \\nATOM   2495  C   GLU A 326      10.701 -16.320  -2.783  1.00  0.75           C  \\nATOM   2496  CB  GLU A 326      13.007 -16.067  -3.730  1.00  0.74           C  \\nATOM   2497  O   GLU A 326       9.972 -17.240  -3.160  1.00  0.72           O  \\nATOM   2498  CG  GLU A 326      13.846 -15.669  -4.935  1.00  0.70           C  \\nATOM   2499  CD  GLU A 326      15.329 -15.949  -4.754  1.00  0.69           C  \\nATOM   2500  OE1 GLU A 326      15.814 -15.924  -3.600  1.00  0.73           O  \\nATOM   2501  OE2 GLU A 326      16.012 -16.194  -5.773  1.00  0.68           O  \\nATOM   2502  N   ARG A 327      10.554 -15.777  -1.507  1.00  0.73           N  \\nATOM   2503  CA  ARG A 327       9.664 -16.364  -0.509  1.00  0.72           C  \\nATOM   2504  C   ARG A 327       8.203 -16.173  -0.900  1.00  0.72           C  \\nATOM   2505  CB  ARG A 327       9.923 -15.751   0.869  1.00  0.69           C  \\nATOM   2506  O   ARG A 327       7.400 -17.103  -0.794  1.00  0.70           O  \\nATOM   2507  CG  ARG A 327      10.875 -16.562   1.734  1.00  0.66           C  \\nATOM   2508  CD  ARG A 327      11.101 -15.907   3.090  1.00  0.64           C  \\nATOM   2509  NE  ARG A 327      12.485 -16.046   3.532  1.00  0.54           N  \\nATOM   2510  NH1 ARG A 327      12.085 -15.329   5.691  1.00  0.45           N  \\nATOM   2511  NH2 ARG A 327      14.203 -15.931   5.053  1.00  0.40           N  \\nATOM   2512  CZ  ARG A 327      12.921 -15.768   4.758  1.00  0.58           C  \\nATOM   2513  N   ARG A 328       7.800 -15.001  -1.358  1.00  0.78           N  \\nATOM   2514  CA  ARG A 328       6.430 -14.720  -1.777  1.00  0.78           C  \\nATOM   2515  C   ARG A 328       6.040 -15.573  -2.980  1.00  0.78           C  \\nATOM   2516  CB  ARG A 328       6.263 -13.236  -2.111  1.00  0.76           C  \\nATOM   2517  O   ARG A 328       4.944 -16.136  -3.018  1.00  0.75           O  \\nATOM   2518  CG  ARG A 328       5.535 -12.442  -1.038  1.00  0.71           C  \\nATOM   2519  CD  ARG A 328       5.473 -10.959  -1.379  1.00  0.69           C  \\nATOM   2520  NE  ARG A 328       4.642 -10.223  -0.431  1.00  0.57           N  \\nATOM   2521  NH1 ARG A 328       4.933  -8.171  -1.452  1.00  0.46           N  \\nATOM   2522  NH2 ARG A 328       3.634  -8.347   0.428  1.00  0.42           N  \\nATOM   2523  CZ  ARG A 328       4.405  -8.915  -0.487  1.00  0.64           C  \\nATOM   2524  N   VAL A 329       6.888 -15.656  -3.987  1.00  0.79           N  \\nATOM   2525  CA  VAL A 329       6.649 -16.489  -5.161  1.00  0.78           C  \\nATOM   2526  C   VAL A 329       6.501 -17.949  -4.738  1.00  0.77           C  \\nATOM   2527  CB  VAL A 329       7.785 -16.347  -6.198  1.00  0.75           C  \\nATOM   2528  O   VAL A 329       5.548 -18.623  -5.134  1.00  0.73           O  \\nATOM   2529  CG1 VAL A 329       7.625 -17.372  -7.319  1.00  0.69           C  \\nATOM   2530  CG2 VAL A 329       7.814 -14.929  -6.766  1.00  0.68           C  \\nATOM   2531  N   LYS A 330       7.370 -18.507  -3.879  1.00  0.73           N  \\nATOM   2532  CA  LYS A 330       7.283 -19.870  -3.362  1.00  0.72           C  \\nATOM   2533  C   LYS A 330       5.991 -20.079  -2.578  1.00  0.72           C  \\nATOM   2534  CB  LYS A 330       8.490 -20.187  -2.478  1.00  0.69           C  \\nATOM   2535  O   LYS A 330       5.286 -21.068  -2.787  1.00  0.69           O  \\nATOM   2536  CG  LYS A 330       8.747 -21.675  -2.291  1.00  0.65           C  \\nATOM   2537  CD  LYS A 330       9.987 -21.926  -1.444  1.00  0.64           C  \\nATOM   2538  CE  LYS A 330      10.181 -23.409  -1.160  1.00  0.59           C  \\nATOM   2539  NZ  LYS A 330      11.363 -23.656  -0.281  1.00  0.50           N  \\nATOM   2540  N   MET A 331       5.520 -19.145  -1.755  1.00  0.76           N  \\nATOM   2541  CA  MET A 331       4.244 -19.226  -1.050  1.00  0.76           C  \\nATOM   2542  C   MET A 331       3.078 -19.238  -2.033  1.00  0.76           C  \\nATOM   2543  CB  MET A 331       4.093 -18.059  -0.073  1.00  0.73           C  \\nATOM   2544  O   MET A 331       2.065 -19.897  -1.794  1.00  0.74           O  \\nATOM   2545  CG  MET A 331       4.880 -18.232   1.216  1.00  0.67           C  \\nATOM   2546  SD  MET A 331       4.526 -16.911   2.439  1.00  0.62           S  \\nATOM   2547  CE  MET A 331       2.713 -16.877   2.357  1.00  0.58           C  \\nATOM   2548  N   LEU A 332       3.121 -18.502  -3.097  1.00  0.76           N  \\nATOM   2549  CA  LEU A 332       2.053 -18.502  -4.090  1.00  0.77           C  \\nATOM   2550  C   LEU A 332       1.994 -19.837  -4.825  1.00  0.75           C  \\nATOM   2551  CB  LEU A 332       2.254 -17.362  -5.093  1.00  0.73           C  \\nATOM   2552  O   LEU A 332       0.909 -20.320  -5.157  1.00  0.73           O  \\nATOM   2553  CG  LEU A 332       1.879 -15.960  -4.610  1.00  0.68           C  \\nATOM   2554  CD1 LEU A 332       2.415 -14.907  -5.574  1.00  0.64           C  \\nATOM   2555  CD2 LEU A 332       0.367 -15.834  -4.458  1.00  0.66           C  \\nATOM   2556  N   PHE A 333       3.124 -20.451  -5.090  1.00  0.72           N  \\nATOM   2557  CA  PHE A 333       3.198 -21.758  -5.732  1.00  0.73           C  \\nATOM   2558  C   PHE A 333       2.730 -22.855  -4.783  1.00  0.70           C  \\nATOM   2559  CB  PHE A 333       4.627 -22.046  -6.204  1.00  0.68           C  \\nATOM   2560  O   PHE A 333       1.947 -23.725  -5.170  1.00  0.67           O  \\nATOM   2561  CG  PHE A 333       4.879 -21.675  -7.640  1.00  0.62           C  \\nATOM   2562  CD1 PHE A 333       4.386 -22.463  -8.673  1.00  0.57           C  \\nATOM   2563  CD2 PHE A 333       5.608 -20.536  -7.958  1.00  0.59           C  \\nATOM   2564  CE1 PHE A 333       4.618 -22.122 -10.004  1.00  0.56           C  \\nATOM   2565  CE2 PHE A 333       5.843 -20.189  -9.285  1.00  0.55           C  \\nATOM   2566  CZ  PHE A 333       5.348 -20.983 -10.306  1.00  0.55           C  \\nATOM   2567  N   GLU A 334       3.186 -22.883  -3.559  1.00  0.71           N  \\nATOM   2568  CA  GLU A 334       2.754 -23.869  -2.573  1.00  0.71           C  \\nATOM   2569  C   GLU A 334       1.253 -23.771  -2.315  1.00  0.70           C  \\nATOM   2570  CB  GLU A 334       3.526 -23.694  -1.263  1.00  0.67           C  \\nATOM   2571  O   GLU A 334       0.562 -24.789  -2.243  1.00  0.68           O  \\nATOM   2572  CG  GLU A 334       4.977 -24.148  -1.338  1.00  0.63           C  \\nATOM   2573  CD  GLU A 334       5.763 -23.857  -0.070  1.00  0.63           C  \\nATOM   2574  OE1 GLU A 334       5.198 -23.253   0.869  1.00  0.65           O  \\nATOM   2575  OE2 GLU A 334       6.955 -24.235  -0.016  1.00  0.57           O  \\nATOM   2576  N   LYS A 335       0.670 -22.586  -2.210  1.00  0.72           N  \\nATOM   2577  CA  LYS A 335      -0.774 -22.429  -2.067  1.00  0.72           C  \\nATOM   2578  C   LYS A 335      -1.510 -22.975  -3.288  1.00  0.72           C  \\nATOM   2579  CB  LYS A 335      -1.136 -20.959  -1.851  1.00  0.68           C  \\nATOM   2580  O   LYS A 335      -2.654 -23.422  -3.181  1.00  0.71           O  \\nATOM   2581  CG  LYS A 335      -1.201 -20.547  -0.387  1.00  0.63           C  \\nATOM   2582  CD  LYS A 335      -1.647 -19.099  -0.234  1.00  0.63           C  \\nATOM   2583  CE  LYS A 335      -1.713 -18.685   1.230  1.00  0.55           C  \\nATOM   2584  NZ  LYS A 335      -2.174 -17.273   1.386  1.00  0.48           N  \\nATOM   2585  N   ARG A 336      -0.891 -22.864  -4.414  1.00  0.70           N  \\nATOM   2586  CA  ARG A 336      -1.499 -23.438  -5.610  1.00  0.70           C  \\nATOM   2587  C   ARG A 336      -1.348 -24.955  -5.626  1.00  0.69           C  \\nATOM   2588  CB  ARG A 336      -0.877 -22.836  -6.872  1.00  0.65           C  \\nATOM   2589  O   ARG A 336      -2.176 -25.661  -6.206  1.00  0.67           O  \\nATOM   2590  CG  ARG A 336      -1.496 -21.513  -7.293  1.00  0.61           C  \\nATOM   2591  CD  ARG A 336      -0.910 -21.008  -8.604  1.00  0.61           C  \\nATOM   2592  NE  ARG A 336      -1.496 -19.730  -9.000  1.00  0.49           N  \\nATOM   2593  NH1 ARG A 336      -0.446 -19.650 -11.056  1.00  0.38           N  \\nATOM   2594  NH2 ARG A 336      -1.847 -17.954 -10.412  1.00  0.35           N  \\nATOM   2595  CZ  ARG A 336      -1.262 -19.114 -10.155  1.00  0.58           C  \\nATOM   2596  N   SER A 337      -0.153 -25.393  -5.099  1.00  0.64           N  \\nATOM   2597  CA  SER A 337       0.152 -26.818  -5.157  1.00  0.65           C  \\nATOM   2598  C   SER A 337      -0.551 -27.581  -4.039  1.00  0.62           C  \\nATOM   2599  CB  SER A 337       1.662 -27.047  -5.070  1.00  0.59           C  \\nATOM   2600  O   SER A 337      -0.742 -28.795  -4.133  1.00  0.59           O  \\nATOM   2601  OG  SER A 337       2.369 -25.964  -5.647  1.00  0.55           O  \\nATOM   2602  N   ASN A 338      -1.050 -26.848  -3.021  1.00  0.55           N  \\nATOM   2603  CA  ASN A 338      -1.749 -27.545  -1.947  1.00  0.56           C  \\nATOM   2604  C   ASN A 338      -3.104 -26.907  -1.653  1.00  0.55           C  \\nATOM   2605  CB  ASN A 338      -0.891 -27.575  -0.680  1.00  0.50           C  \\nATOM   2606  O   ASN A 338      -3.195 -25.972  -0.857  1.00  0.54           O  \\nATOM   2607  CG  ASN A 338      -1.312 -28.668   0.284  1.00  0.48           C  \\nATOM   2608  ND2 ASN A 338      -0.569 -28.815   1.375  1.00  0.49           N  \\nATOM   2609  OD1 ASN A 338      -2.296 -29.374   0.050  1.00  0.50           O  \\nATOM   2610  N   PRO A 339      -4.074 -27.066  -2.494  1.00  0.63           N  \\nATOM   2611  CA  PRO A 339      -5.412 -26.557  -2.183  1.00  0.62           C  \\nATOM   2612  C   PRO A 339      -5.992 -27.164  -0.908  1.00  0.63           C  \\nATOM   2613  CB  PRO A 339      -6.237 -26.963  -3.406  1.00  0.58           C  \\nATOM   2614  O   PRO A 339      -7.094 -26.797  -0.491  1.00  0.61           O  \\nATOM   2615  CG  PRO A 339      -5.554 -28.181  -3.940  1.00  0.55           C  \\nATOM   2616  CD  PRO A 339      -4.102 -28.116  -3.565  1.00  0.55           C  \\nATOM   2617  N   TYR A 340      -5.352 -28.098  -0.176  1.00  0.63           N  \\nATOM   2618  CA  TYR A 340      -5.885 -28.880   0.934  1.00  0.62           C  \\nATOM   2619  C   TYR A 340      -5.019 -28.721   2.179  1.00  0.60           C  \\nATOM   2620  CB  TYR A 340      -5.982 -30.360   0.552  1.00  0.57           C  \\nATOM   2621  O   TYR A 340      -5.048 -29.568   3.075  1.00  0.56           O  \\nATOM   2622  CG  TYR A 340      -6.944 -30.633  -0.579  1.00  0.54           C  \\nATOM   2623  CD1 TYR A 340      -8.321 -30.551  -0.383  1.00  0.50           C  \\nATOM   2624  CD2 TYR A 340      -6.479 -30.974  -1.845  1.00  0.53           C  \\nATOM   2625  CE1 TYR A 340      -9.212 -30.804  -1.421  1.00  0.53           C  \\nATOM   2626  CE2 TYR A 340      -7.361 -31.228  -2.890  1.00  0.52           C  \\nATOM   2627  OH  TYR A 340      -9.600 -31.391  -3.700  1.00  0.54           O  \\nATOM   2628  CZ  TYR A 340      -8.723 -31.140  -2.669  1.00  0.49           C  \\nATOM   2629  N   HIS A 341      -4.445 -27.686   2.416  1.00  0.44           N  \\nATOM   2630  CA  HIS A 341      -4.101 -27.432   3.810  1.00  0.49           C  \\nATOM   2631  C   HIS A 341      -5.024 -26.385   4.423  1.00  0.44           C  \\nATOM   2632  CB  HIS A 341      -2.644 -26.982   3.929  1.00  0.44           C  \\nATOM   2633  O   HIS A 341      -5.324 -25.369   3.792  1.00  0.43           O  \\nATOM   2634  CG  HIS A 341      -1.673 -28.114   4.036  1.00  0.45           C  \\nATOM   2635  CD2 HIS A 341      -0.684 -28.519   3.204  1.00  0.44           C  \\nATOM   2636  ND1 HIS A 341      -1.659 -28.984   5.105  1.00  0.45           N  \\nATOM   2637  CE1 HIS A 341      -0.700 -29.877   4.925  1.00  0.41           C  \\nATOM   2638  NE2 HIS A 341      -0.094 -29.617   3.779  1.00  0.41           N  \",\"pdb\");\n\tviewer_17207123026924806.setStyle({\"resi\": 1},{\"cartoon\": {\"color\": \"#2c2cd3\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 2},{\"cartoon\": {\"color\": \"#5858a7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 3},{\"cartoon\": {\"color\": \"#777788\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 4},{\"cartoon\": {\"color\": \"#8d8d72\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 5},{\"cartoon\": {\"color\": \"#9b9b64\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 6},{\"cartoon\": {\"color\": \"#9b9b64\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 7},{\"cartoon\": {\"color\": \"#a5a55a\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 8},{\"cartoon\": {\"color\": \"#aeae51\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 9},{\"cartoon\": {\"color\": \"#b5b54a\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 10},{\"cartoon\": {\"color\": \"#baba45\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 11},{\"cartoon\": {\"color\": \"#b9b946\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 12},{\"cartoon\": {\"color\": \"#bebe41\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 13},{\"cartoon\": {\"color\": \"#a0a05f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 14},{\"cartoon\": {\"color\": \"#83837c\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 15},{\"cartoon\": {\"color\": \"#898976\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 16},{\"cartoon\": {\"color\": \"#8f8f70\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 17},{\"cartoon\": {\"color\": \"#94946b\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 18},{\"cartoon\": {\"color\": \"#94946b\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 19},{\"cartoon\": {\"color\": \"#b0b04f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 20},{\"cartoon\": {\"color\": \"#b1b14e\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 21},{\"cartoon\": {\"color\": \"#b1b14e\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 22},{\"cartoon\": {\"color\": \"#b1b14e\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 23},{\"cartoon\": {\"color\": \"#b1b14e\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 24},{\"cartoon\": {\"color\": \"#b6b649\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 25},{\"cartoon\": {\"color\": \"#b7b748\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 26},{\"cartoon\": {\"color\": \"#b6b649\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 27},{\"cartoon\": {\"color\": \"#a7a758\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 28},{\"cartoon\": {\"color\": \"#a8a857\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 29},{\"cartoon\": {\"color\": \"#a8a857\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 30},{\"cartoon\": {\"color\": \"#90906f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 31},{\"cartoon\": {\"color\": \"#a8a857\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 32},{\"cartoon\": {\"color\": \"#caca35\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 33},{\"cartoon\": {\"color\": \"#aeae51\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 34},{\"cartoon\": {\"color\": \"#94946b\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 35},{\"cartoon\": {\"color\": \"#7a7a85\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 36},{\"cartoon\": {\"color\": \"#7a7a85\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 37},{\"cartoon\": {\"color\": \"#797986\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 38},{\"cartoon\": {\"color\": \"#62629d\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 39},{\"cartoon\": {\"color\": \"#62629d\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 40},{\"cartoon\": {\"color\": \"#5f5fa0\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 41},{\"cartoon\": {\"color\": \"#5f5fa0\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 42},{\"cartoon\": {\"color\": \"#5f5fa0\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 43},{\"cartoon\": {\"color\": \"#5f5fa0\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 44},{\"cartoon\": {\"color\": \"#5f5fa0\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 45},{\"cartoon\": {\"color\": \"#6a6a95\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 46},{\"cartoon\": {\"color\": \"#696996\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 47},{\"cartoon\": {\"color\": \"#6b6b94\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 48},{\"cartoon\": {\"color\": \"#7f7f80\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 49},{\"cartoon\": {\"color\": \"#80807f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 50},{\"cartoon\": {\"color\": \"#80807f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 51},{\"cartoon\": {\"color\": \"#95956a\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 52},{\"cartoon\": {\"color\": \"#90906f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 53},{\"cartoon\": {\"color\": \"#adad52\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 54},{\"cartoon\": {\"color\": \"#aeae51\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 55},{\"cartoon\": {\"color\": \"#aeae51\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 56},{\"cartoon\": {\"color\": \"#d1d12e\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 57},{\"cartoon\": {\"color\": \"#d1d12e\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 58},{\"cartoon\": {\"color\": \"#d5d52a\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 59},{\"cartoon\": {\"color\": \"#d5d52a\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 60},{\"cartoon\": {\"color\": \"#d4d42b\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 61},{\"cartoon\": {\"color\": \"#d4d42b\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 62},{\"cartoon\": {\"color\": \"#d4d42b\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 63},{\"cartoon\": {\"color\": \"#d4d42b\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 64},{\"cartoon\": {\"color\": \"#d3d32c\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 65},{\"cartoon\": {\"color\": \"#c0c03f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 66},{\"cartoon\": {\"color\": \"#c0c03f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 67},{\"cartoon\": {\"color\": \"#c0c03f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 68},{\"cartoon\": {\"color\": \"#c0c03f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 69},{\"cartoon\": {\"color\": \"#c2c23d\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 70},{\"cartoon\": {\"color\": \"#e3e31c\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 71},{\"cartoon\": {\"color\": \"#e3e31c\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 72},{\"cartoon\": {\"color\": \"#e3e31c\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 73},{\"cartoon\": {\"color\": \"#e3e31c\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 74},{\"cartoon\": {\"color\": \"#e3e31c\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 75},{\"cartoon\": {\"color\": \"#e3e31c\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 76},{\"cartoon\": {\"color\": \"#c0c03f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 77},{\"cartoon\": {\"color\": \"#c1c13e\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 78},{\"cartoon\": {\"color\": \"#c1c13e\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 79},{\"cartoon\": {\"color\": \"#a0a05f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 80},{\"cartoon\": {\"color\": \"#83837c\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 81},{\"cartoon\": {\"color\": \"#6a6a95\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 82},{\"cartoon\": {\"color\": \"#5454ab\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 83},{\"cartoon\": {\"color\": \"#60609f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 84},{\"cartoon\": {\"color\": \"#60609f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 85},{\"cartoon\": {\"color\": \"#4b4bb4\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 86},{\"cartoon\": {\"color\": \"#3a3ac5\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 87},{\"cartoon\": {\"color\": \"#2c2cd3\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 88},{\"cartoon\": {\"color\": \"#2d2dd2\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 89},{\"cartoon\": {\"color\": \"#2d2dd2\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 90},{\"cartoon\": {\"color\": \"#2121de\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 91},{\"cartoon\": {\"color\": \"#1717e8\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 92},{\"cartoon\": {\"color\": \"#0f0ff0\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 93},{\"cartoon\": {\"color\": \"#0a0af5\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 94},{\"cartoon\": {\"color\": \"#0e0ef1\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 95},{\"cartoon\": {\"color\": \"#0808f7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 96},{\"cartoon\": {\"color\": \"#0606f9\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 97},{\"cartoon\": {\"color\": \"#0808f7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 98},{\"cartoon\": {\"color\": \"#0808f7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 99},{\"cartoon\": {\"color\": \"#0808f7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 100},{\"cartoon\": {\"color\": \"#0e0ef1\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 101},{\"cartoon\": {\"color\": \"#0808f7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 102},{\"cartoon\": {\"color\": \"#0808f7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 103},{\"cartoon\": {\"color\": \"#0808f7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 104},{\"cartoon\": {\"color\": \"#0808f7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 105},{\"cartoon\": {\"color\": \"#0d0df2\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 106},{\"cartoon\": {\"color\": \"#0707f8\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 107},{\"cartoon\": {\"color\": \"#0303fc\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 108},{\"cartoon\": {\"color\": \"#0303fc\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 109},{\"cartoon\": {\"color\": \"#0303fc\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 110},{\"cartoon\": {\"color\": \"#0606f9\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 111},{\"cartoon\": {\"color\": \"#0b0bf4\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 112},{\"cartoon\": {\"color\": \"#0707f8\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 113},{\"cartoon\": {\"color\": \"#0707f8\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 114},{\"cartoon\": {\"color\": \"#0505fa\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 115},{\"cartoon\": {\"color\": \"#0303fc\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 116},{\"cartoon\": {\"color\": \"#0303fc\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 117},{\"cartoon\": {\"color\": \"#0303fc\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 118},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 119},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 120},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 121},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 122},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 123},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 124},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 125},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 126},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 127},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 128},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 129},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 130},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 131},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 132},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 133},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 134},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 135},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 136},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 137},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 138},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 139},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 140},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 141},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 142},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 143},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 144},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 145},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 146},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 147},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 148},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 149},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 150},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 151},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 152},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 153},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 154},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 155},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 156},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 157},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 158},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 159},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 160},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 161},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 162},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 163},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 164},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 165},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 166},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 167},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 168},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 169},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 170},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 171},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 172},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 173},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 174},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 175},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 176},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 177},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 178},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 179},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 180},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 181},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 182},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 183},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 184},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 185},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 186},{\"cartoon\": {\"color\": \"#0303fc\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 187},{\"cartoon\": {\"color\": \"#0303fc\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 188},{\"cartoon\": {\"color\": \"#0505fa\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 189},{\"cartoon\": {\"color\": \"#0a0af5\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 190},{\"cartoon\": {\"color\": \"#0a0af5\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 191},{\"cartoon\": {\"color\": \"#0a0af5\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 192},{\"cartoon\": {\"color\": \"#0a0af5\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 193},{\"cartoon\": {\"color\": \"#0505fa\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 194},{\"cartoon\": {\"color\": \"#0505fa\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 195},{\"cartoon\": {\"color\": \"#0505fa\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 196},{\"cartoon\": {\"color\": \"#0505fa\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 197},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 198},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 199},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 200},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 201},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 202},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 203},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 204},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 205},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 206},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 207},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 208},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 209},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 210},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 211},{\"cartoon\": {\"color\": \"#0303fc\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 212},{\"cartoon\": {\"color\": \"#0707f8\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 213},{\"cartoon\": {\"color\": \"#0d0df2\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 214},{\"cartoon\": {\"color\": \"#0d0df2\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 215},{\"cartoon\": {\"color\": \"#0d0df2\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 216},{\"cartoon\": {\"color\": \"#1414eb\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 217},{\"cartoon\": {\"color\": \"#1d1de2\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 218},{\"cartoon\": {\"color\": \"#2929d6\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 219},{\"cartoon\": {\"color\": \"#2929d6\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 220},{\"cartoon\": {\"color\": \"#2929d6\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 221},{\"cartoon\": {\"color\": \"#2929d6\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 222},{\"cartoon\": {\"color\": \"#3737c8\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 223},{\"cartoon\": {\"color\": \"#2b2bd4\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 224},{\"cartoon\": {\"color\": \"#1f1fe0\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 225},{\"cartoon\": {\"color\": \"#1f1fe0\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 226},{\"cartoon\": {\"color\": \"#2020df\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 227},{\"cartoon\": {\"color\": \"#2c2cd3\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 228},{\"cartoon\": {\"color\": \"#2a2ad5\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 229},{\"cartoon\": {\"color\": \"#1e1ee1\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 230},{\"cartoon\": {\"color\": \"#1515ea\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 231},{\"cartoon\": {\"color\": \"#0e0ef1\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 232},{\"cartoon\": {\"color\": \"#0e0ef1\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 233},{\"cartoon\": {\"color\": \"#0e0ef1\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 234},{\"cartoon\": {\"color\": \"#0808f7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 235},{\"cartoon\": {\"color\": \"#0404fb\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 236},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 237},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 238},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 239},{\"cartoon\": {\"color\": \"#0303fc\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 240},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 241},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 242},{\"cartoon\": {\"color\": \"#0303fc\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 243},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 244},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 245},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 246},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 247},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 248},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 249},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 250},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 251},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 252},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 253},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 254},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 255},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 256},{\"cartoon\": {\"color\": \"#0202fd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 257},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 258},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 259},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 260},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 261},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 262},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 263},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 264},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 265},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 266},{\"cartoon\": {\"color\": \"#0000ff\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 267},{\"cartoon\": {\"color\": \"#0101fe\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 268},{\"cartoon\": {\"color\": \"#0404fb\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 269},{\"cartoon\": {\"color\": \"#0909f6\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 270},{\"cartoon\": {\"color\": \"#0f0ff0\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 271},{\"cartoon\": {\"color\": \"#0909f6\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 272},{\"cartoon\": {\"color\": \"#0f0ff0\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 273},{\"cartoon\": {\"color\": \"#1111ee\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 274},{\"cartoon\": {\"color\": \"#1a1ae5\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 275},{\"cartoon\": {\"color\": \"#2424db\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 276},{\"cartoon\": {\"color\": \"#3232cd\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 277},{\"cartoon\": {\"color\": \"#2c2cd3\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 278},{\"cartoon\": {\"color\": \"#2c2cd3\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 279},{\"cartoon\": {\"color\": \"#3a3ac5\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 280},{\"cartoon\": {\"color\": \"#4c4cb3\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 281},{\"cartoon\": {\"color\": \"#60609f\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 282},{\"cartoon\": {\"color\": \"#767689\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 283},{\"cartoon\": {\"color\": \"#6b6b94\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 284},{\"cartoon\": {\"color\": \"#84847b\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 285},{\"cartoon\": {\"color\": \"#6b6b94\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 286},{\"cartoon\": {\"color\": \"#5555aa\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 287},{\"cartoon\": {\"color\": \"#4343bc\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 288},{\"cartoon\": {\"color\": \"#3434cb\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 289},{\"cartoon\": {\"color\": \"#3434cb\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 290},{\"cartoon\": {\"color\": \"#2727d8\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 291},{\"cartoon\": {\"color\": \"#2727d8\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 292},{\"cartoon\": {\"color\": \"#2727d8\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 293},{\"cartoon\": {\"color\": \"#1b1be4\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 294},{\"cartoon\": {\"color\": \"#1b1be4\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 295},{\"cartoon\": {\"color\": \"#2626d9\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 296},{\"cartoon\": {\"color\": \"#3434cb\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 297},{\"cartoon\": {\"color\": \"#3030cf\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 298},{\"cartoon\": {\"color\": \"#3030cf\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 299},{\"cartoon\": {\"color\": \"#3030cf\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 300},{\"cartoon\": {\"color\": \"#3131ce\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 301},{\"cartoon\": {\"color\": \"#3838c7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 302},{\"cartoon\": {\"color\": \"#3838c7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 303},{\"cartoon\": {\"color\": \"#4848b7\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 304},{\"cartoon\": {\"color\": \"#5c5ca3\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 305},{\"cartoon\": {\"color\": \"#72728d\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 306},{\"cartoon\": {\"color\": \"#8b8b74\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 307},{\"cartoon\": {\"color\": \"#a9a956\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 308},{\"cartoon\": {\"color\": \"#a8a857\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 309},{\"cartoon\": {\"color\": \"#a8a857\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 310},{\"cartoon\": {\"color\": \"#a9a956\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 311},{\"cartoon\": {\"color\": \"#adad52\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 312},{\"cartoon\": {\"color\": \"#adad52\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 313},{\"cartoon\": {\"color\": \"#aeae51\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 314},{\"cartoon\": {\"color\": \"#aeae51\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 315},{\"cartoon\": {\"color\": \"#b8b847\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 316},{\"cartoon\": {\"color\": \"#b8b847\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 317},{\"cartoon\": {\"color\": \"#b8b847\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 318},{\"cartoon\": {\"color\": \"#b8b847\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 319},{\"cartoon\": {\"color\": \"#b8b847\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 320},{\"cartoon\": {\"color\": \"#b8b847\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 321},{\"cartoon\": {\"color\": \"#b8b847\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 322},{\"cartoon\": {\"color\": \"#b9b946\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 323},{\"cartoon\": {\"color\": \"#b9b946\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 324},{\"cartoon\": {\"color\": \"#b7b748\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 325},{\"cartoon\": {\"color\": \"#b3b34c\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 326},{\"cartoon\": {\"color\": \"#d6d629\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 327},{\"cartoon\": {\"color\": \"#d4d42b\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 328},{\"cartoon\": {\"color\": \"#d1d12e\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 329},{\"cartoon\": {\"color\": \"#fefe01\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 330},{\"cartoon\": {\"color\": \"#fefe01\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 331},{\"cartoon\": {\"color\": \"#fefe01\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 332},{\"cartoon\": {\"color\": \"#fdfd02\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 333},{\"cartoon\": {\"color\": \"#fcfc03\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 334},{\"cartoon\": {\"color\": \"#fcfc03\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 335},{\"cartoon\": {\"color\": \"#fbfb04\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 336},{\"cartoon\": {\"color\": \"#fcfc03\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 337},{\"cartoon\": {\"color\": \"#fafa05\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 338},{\"cartoon\": {\"color\": \"#f8f807\"}});\n\tviewer_17207123026924806.setStyle({\"resi\": 339},{\"cartoon\": {\"color\": \"#f4f40b\"}});\n\tviewer_17207123026924806.zoomTo();\nviewer_17207123026924806.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_17207123026924806\"  style=\"position: relative; width: 1000px; height: 800px;\">\n",
              "        <p id=\"3dmolwarning_17207123026924806\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.2.0/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_17207123026924806 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_17207123026924806\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_17207123026924806 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17207123026924806\"),{backgroundColor:\"white\"});\n",
              "viewer_17207123026924806.zoomTo();\n",
              "\tviewer_17207123026924806.addModel(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   MET A   1       0.319   0.252 -46.800  1.00  0.76           N  \\nATOM      2  CA  MET A   1      -0.203   0.998 -45.658  1.00  0.81           C  \\nATOM      3  C   MET A   1       0.586   0.679 -44.393  1.00  0.77           C  \\nATOM      4  CB  MET A   1      -1.685   0.686 -45.444  1.00  0.69           C  \\nATOM      5  O   MET A   1       0.552  -0.452 -43.903  1.00  0.67           O  \\nATOM      6  CG  MET A   1      -2.596   1.278 -46.507  1.00  0.62           C  \\nATOM      7  SD  MET A   1      -4.360   0.849 -46.238  1.00  0.68           S  \\nATOM      8  CE  MET A   1      -5.122   1.749 -47.618  1.00  0.63           C  \\nATOM      9  N   GLY A   2       1.736   1.505 -43.986  1.00  0.84           N  \\nATOM     10  CA  GLY A   2       2.492   1.362 -42.752  1.00  0.86           C  \\nATOM     11  C   GLY A   2       1.617   1.361 -41.513  1.00  0.84           C  \\nATOM     12  O   GLY A   2       0.842   2.294 -41.293  1.00  0.77           O  \\nATOM     13  N   ARG A   3       1.282   0.132 -40.795  1.00  0.84           N  \\nATOM     14  CA  ARG A   3       0.625   0.143 -39.492  1.00  0.85           C  \\nATOM     15  C   ARG A   3       1.236   1.203 -38.580  1.00  0.83           C  \\nATOM     16  CB  ARG A   3       0.719  -1.234 -38.831  1.00  0.81           C  \\nATOM     17  O   ARG A   3       2.411   1.115 -38.218  1.00  0.77           O  \\nATOM     18  CG  ARG A   3      -0.331  -2.222 -39.315  1.00  0.79           C  \\nATOM     19  CD  ARG A   3      -0.214  -3.562 -38.603  1.00  0.77           C  \\nATOM     20  NE  ARG A   3      -1.137  -4.549 -39.157  1.00  0.77           N  \\nATOM     21  NH1 ARG A   3      -0.479  -6.259 -37.751  1.00  0.62           N  \\nATOM     22  NH2 ARG A   3      -2.115  -6.621 -39.315  1.00  0.57           N  \\nATOM     23  CZ  ARG A   3      -1.241  -5.807 -38.740  1.00  0.76           C  \\nATOM     24  N   LYS A   4       0.539   2.338 -38.420  1.00  0.83           N  \\nATOM     25  CA  LYS A   4       0.941   3.279 -37.379  1.00  0.84           C  \\nATOM     26  C   LYS A   4       1.275   2.551 -36.080  1.00  0.82           C  \\nATOM     27  CB  LYS A   4      -0.160   4.311 -37.134  1.00  0.80           C  \\nATOM     28  O   LYS A   4       0.421   1.872 -35.505  1.00  0.78           O  \\nATOM     29  CG  LYS A   4      -0.304   5.340 -38.246  1.00  0.74           C  \\nATOM     30  CD  LYS A   4      -1.389   6.360 -37.927  1.00  0.72           C  \\nATOM     31  CE  LYS A   4      -1.542   7.383 -39.045  1.00  0.74           C  \\nATOM     32  NZ  LYS A   4      -2.650   8.345 -38.767  1.00  0.68           N  \\nATOM     33  N   LYS A   5       2.518   2.445 -35.703  1.00  0.79           N  \\nATOM     34  CA  LYS A   5       2.886   1.957 -34.377  1.00  0.80           C  \\nATOM     35  C   LYS A   5       2.107   2.687 -33.286  1.00  0.79           C  \\nATOM     36  CB  LYS A   5       4.389   2.118 -34.144  1.00  0.76           C  \\nATOM     37  O   LYS A   5       2.154   3.916 -33.199  1.00  0.75           O  \\nATOM     38  CG  LYS A   5       5.243   1.100 -34.886  1.00  0.73           C  \\nATOM     39  CD  LYS A   5       6.724   1.287 -34.584  1.00  0.72           C  \\nATOM     40  CE  LYS A   5       7.580   0.273 -35.332  1.00  0.72           C  \\nATOM     41  NZ  LYS A   5       9.035   0.461 -35.050  1.00  0.64           N  \\nATOM     42  N   LYS A   6       1.193   1.975 -32.649  1.00  0.77           N  \\nATOM     43  CA  LYS A   6       0.572   2.536 -31.453  1.00  0.78           C  \\nATOM     44  C   LYS A   6       1.622   3.104 -30.502  1.00  0.77           C  \\nATOM     45  CB  LYS A   6      -0.266   1.477 -30.736  1.00  0.74           C  \\nATOM     46  O   LYS A   6       2.547   2.397 -30.098  1.00  0.74           O  \\nATOM     47  CG  LYS A   6      -1.516   2.026 -30.064  1.00  0.69           C  \\nATOM     48  CD  LYS A   6      -2.347   0.915 -29.434  1.00  0.68           C  \\nATOM     49  CE  LYS A   6      -3.565   1.469 -28.709  1.00  0.63           C  \\nATOM     50  NZ  LYS A   6      -4.348   0.390 -28.036  1.00  0.57           N  \\nATOM     51  N   LYS A   7       1.645   4.412 -30.346  1.00  0.79           N  \\nATOM     52  CA  LYS A   7       2.417   5.011 -29.262  1.00  0.79           C  \\nATOM     53  C   LYS A   7       2.086   4.357 -27.924  1.00  0.79           C  \\nATOM     54  CB  LYS A   7       2.160   6.517 -29.188  1.00  0.76           C  \\nATOM     55  O   LYS A   7       0.944   4.418 -27.463  1.00  0.76           O  \\nATOM     56  CG  LYS A   7       3.275   7.303 -28.516  1.00  0.72           C  \\nATOM     57  CD  LYS A   7       3.005   8.802 -28.555  1.00  0.70           C  \\nATOM     58  CE  LYS A   7       4.065   9.582 -27.790  1.00  0.66           C  \\nATOM     59  NZ  LYS A   7       3.731  11.036 -27.707  1.00  0.60           N  \\nATOM     60  N   VAL A   8       2.941   3.471 -27.397  1.00  0.77           N  \\nATOM     61  CA  VAL A   8       2.815   2.987 -26.026  1.00  0.77           C  \\nATOM     62  C   VAL A   8       3.165   4.106 -25.049  1.00  0.77           C  \\nATOM     63  CB  VAL A   8       3.714   1.757 -25.773  1.00  0.74           C  \\nATOM     64  O   VAL A   8       4.207   4.753 -25.184  1.00  0.74           O  \\nATOM     65  CG1 VAL A   8       3.467   1.185 -24.379  1.00  0.69           C  \\nATOM     66  CG2 VAL A   8       3.474   0.692 -26.841  1.00  0.69           C  \\nATOM     67  N   ASP A   9       2.162   4.482 -24.260  1.00  0.79           N  \\nATOM     68  CA  ASP A   9       2.381   5.498 -23.235  1.00  0.79           C  \\nATOM     69  C   ASP A   9       3.495   5.081 -22.277  1.00  0.81           C  \\nATOM     70  CB  ASP A   9       1.091   5.764 -22.457  1.00  0.77           C  \\nATOM     71  O   ASP A   9       3.508   3.950 -21.788  1.00  0.79           O  \\nATOM     72  CG  ASP A   9      -0.034   6.284 -23.335  1.00  0.73           C  \\nATOM     73  OD1 ASP A   9       0.221   7.135 -24.214  1.00  0.72           O  \\nATOM     74  OD2 ASP A   9      -1.187   5.840 -23.145  1.00  0.74           O  \\nATOM     75  N   LYS A  10       4.565   5.912 -22.204  1.00  0.83           N  \\nATOM     76  CA  LYS A  10       5.703   5.678 -21.319  1.00  0.84           C  \\nATOM     77  C   LYS A  10       5.267   5.657 -19.857  1.00  0.84           C  \\nATOM     78  CB  LYS A  10       6.776   6.747 -21.531  1.00  0.82           C  \\nATOM     79  O   LYS A  10       4.481   6.503 -19.426  1.00  0.83           O  \\nATOM     80  CG  LYS A  10       7.525   6.619 -22.849  1.00  0.81           C  \\nATOM     81  CD  LYS A  10       8.658   7.632 -22.949  1.00  0.80           C  \\nATOM     82  CE  LYS A  10       9.465   7.446 -24.227  1.00  0.76           C  \\nATOM     83  NZ  LYS A  10      10.738   8.226 -24.197  1.00  0.73           N  \\nATOM     84  N   PRO A  11       5.711   4.655 -19.024  1.00  0.84           N  \\nATOM     85  CA  PRO A  11       5.436   4.737 -17.588  1.00  0.85           C  \\nATOM     86  C   PRO A  11       6.125   5.926 -16.922  1.00  0.85           C  \\nATOM     87  CB  PRO A  11       5.986   3.415 -17.047  1.00  0.84           C  \\nATOM     88  O   PRO A  11       7.226   6.311 -17.325  1.00  0.84           O  \\nATOM     89  CG  PRO A  11       7.106   3.061 -17.971  1.00  0.83           C  \\nATOM     90  CD  PRO A  11       6.765   3.556 -19.347  1.00  0.83           C  \\nATOM     91  N   TRP A  12       5.538   6.529 -15.970  1.00  0.84           N  \\nATOM     92  CA  TRP A  12       6.088   7.701 -15.296  1.00  0.85           C  \\nATOM     93  C   TRP A  12       6.010   7.543 -13.781  1.00  0.85           C  \\nATOM     94  CB  TRP A  12       5.346   8.968 -15.729  1.00  0.83           C  \\nATOM     95  O   TRP A  12       5.153   6.818 -13.269  1.00  0.84           O  \\nATOM     96  CG  TRP A  12       3.887   8.967 -15.387  1.00  0.79           C  \\nATOM     97  CD1 TRP A  12       2.869   8.425 -16.121  1.00  0.75           C  \\nATOM     98  CD2 TRP A  12       3.282   9.540 -14.224  1.00  0.77           C  \\nATOM     99  CE2 TRP A  12       1.892   9.307 -14.318  1.00  0.78           C  \\nATOM    100  CE3 TRP A  12       3.780  10.228 -13.109  1.00  0.80           C  \\nATOM    101  NE1 TRP A  12       1.666   8.626 -15.484  1.00  0.79           N  \\nATOM    102  CH2 TRP A  12       1.508  10.409 -12.259  1.00  0.77           C  \\nATOM    103  CZ2 TRP A  12       0.994   9.739 -13.338  1.00  0.78           C  \\nATOM    104  CZ3 TRP A  12       2.886  10.656 -12.135  1.00  0.78           C  \\nATOM    105  N   CYS A  13       6.861   8.172 -13.001  1.00  0.86           N  \\nATOM    106  CA  CYS A  13       6.957   8.134 -11.546  1.00  0.86           C  \\nATOM    107  C   CYS A  13       6.028   9.163 -10.912  1.00  0.86           C  \\nATOM    108  CB  CYS A  13       8.395   8.387 -11.097  1.00  0.85           C  \\nATOM    109  O   CYS A  13       6.276  10.367 -11.000  1.00  0.84           O  \\nATOM    110  SG  CYS A  13       8.618   8.337  -9.305  1.00  0.90           S  \\nATOM    111  N   TRP A  14       5.058   8.716 -10.126  1.00  0.85           N  \\nATOM    112  CA  TRP A  14       4.069   9.567  -9.473  1.00  0.86           C  \\nATOM    113  C   TRP A  14       4.737  10.524  -8.492  1.00  0.86           C  \\nATOM    114  CB  TRP A  14       3.024   8.717  -8.745  1.00  0.83           C  \\nATOM    115  O   TRP A  14       4.359  11.695  -8.401  1.00  0.83           O  \\nATOM    116  CG  TRP A  14       1.959   9.518  -8.058  1.00  0.75           C  \\nATOM    117  CD1 TRP A  14       0.813  10.014  -8.614  1.00  0.68           C  \\nATOM    118  CD2 TRP A  14       1.945   9.920  -6.685  1.00  0.70           C  \\nATOM    119  CE2 TRP A  14       0.759  10.657  -6.477  1.00  0.70           C  \\nATOM    120  CE3 TRP A  14       2.822   9.727  -5.608  1.00  0.75           C  \\nATOM    121  NE1 TRP A  14       0.086  10.700  -7.668  1.00  0.77           N  \\nATOM    122  CH2 TRP A  14       1.301  10.998  -4.198  1.00  0.71           C  \\nATOM    123  CZ2 TRP A  14       0.426  11.202  -5.234  1.00  0.74           C  \\nATOM    124  CZ3 TRP A  14       2.489  10.270  -4.373  1.00  0.72           C  \\nATOM    125  N   TYR A  15       5.804  10.168  -7.929  1.00  0.87           N  \\nATOM    126  CA  TYR A  15       6.447  10.922  -6.859  1.00  0.88           C  \\nATOM    127  C   TYR A  15       7.257  12.084  -7.421  1.00  0.88           C  \\nATOM    128  CB  TYR A  15       7.351  10.010  -6.025  1.00  0.87           C  \\nATOM    129  O   TYR A  15       7.540  13.053  -6.712  1.00  0.86           O  \\nATOM    130  CG  TYR A  15       6.611   8.894  -5.329  1.00  0.86           C  \\nATOM    131  CD1 TYR A  15       5.834   9.145  -4.200  1.00  0.84           C  \\nATOM    132  CD2 TYR A  15       6.687   7.587  -5.798  1.00  0.85           C  \\nATOM    133  CE1 TYR A  15       5.152   8.119  -3.554  1.00  0.87           C  \\nATOM    134  CE2 TYR A  15       6.009   6.553  -5.159  1.00  0.87           C  \\nATOM    135  OH  TYR A  15       4.572   5.810  -3.405  1.00  0.79           O  \\nATOM    136  CZ  TYR A  15       5.245   6.829  -4.040  1.00  0.87           C  \\nATOM    137  N   CYS A  16       7.756  12.110  -8.812  1.00  0.89           N  \\nATOM    138  CA  CYS A  16       8.626  13.181  -9.284  1.00  0.88           C  \\nATOM    139  C   CYS A  16       8.300  13.553 -10.726  1.00  0.87           C  \\nATOM    140  CB  CYS A  16      10.093  12.768  -9.174  1.00  0.87           C  \\nATOM    141  O   CYS A  16       8.977  14.391 -11.323  1.00  0.85           O  \\nATOM    142  SG  CYS A  16      10.552  11.410 -10.272  1.00  0.89           S  \\nATOM    143  N   ASN A  17       7.388  12.978 -11.403  1.00  0.84           N  \\nATOM    144  CA  ASN A  17       6.871  13.244 -12.741  1.00  0.84           C  \\nATOM    145  C   ASN A  17       7.875  12.847 -13.820  1.00  0.84           C  \\nATOM    146  CB  ASN A  17       6.492  14.720 -12.887  1.00  0.81           C  \\nATOM    147  O   ASN A  17       7.800  13.328 -14.952  1.00  0.82           O  \\nATOM    148  CG  ASN A  17       5.286  15.098 -12.049  1.00  0.76           C  \\nATOM    149  ND2 ASN A  17       5.157  16.383 -11.742  1.00  0.72           N  \\nATOM    150  OD1 ASN A  17       4.478  14.242 -11.679  1.00  0.72           O  \\nATOM    151  N   ARG A  18       8.819  12.093 -13.467  1.00  0.88           N  \\nATOM    152  CA  ARG A  18       9.807  11.689 -14.463  1.00  0.88           C  \\nATOM    153  C   ARG A  18       9.305  10.505 -15.283  1.00  0.88           C  \\nATOM    154  CB  ARG A  18      11.136  11.337 -13.791  1.00  0.87           C  \\nATOM    155  O   ARG A  18       8.780   9.537 -14.728  1.00  0.87           O  \\nATOM    156  CG  ARG A  18      12.259  11.035 -14.770  1.00  0.83           C  \\nATOM    157  CD  ARG A  18      13.610  10.959 -14.072  1.00  0.82           C  \\nATOM    158  NE  ARG A  18      14.671  10.561 -14.992  1.00  0.79           N  \\nATOM    159  NH1 ARG A  18      16.355  10.641 -13.412  1.00  0.72           N  \\nATOM    160  NH2 ARG A  18      16.834  10.053 -15.576  1.00  0.70           N  \\nATOM    161  CZ  ARG A  18      15.951  10.420 -14.658  1.00  0.78           C  \\nATOM    162  N   GLU A  19       9.453  10.497 -16.477  1.00  0.85           N  \\nATOM    163  CA  GLU A  19       9.134   9.437 -17.429  1.00  0.85           C  \\nATOM    164  C   GLU A  19      10.282   8.438 -17.547  1.00  0.86           C  \\nATOM    165  CB  GLU A  19       8.809  10.029 -18.803  1.00  0.83           C  \\nATOM    166  O   GLU A  19      11.452   8.813 -17.447  1.00  0.85           O  \\nATOM    167  CG  GLU A  19       7.511  10.823 -18.839  1.00  0.77           C  \\nATOM    168  CD  GLU A  19       7.140  11.303 -20.233  1.00  0.75           C  \\nATOM    169  OE1 GLU A  19       7.923  11.074 -21.182  1.00  0.74           O  \\nATOM    170  OE2 GLU A  19       6.057  11.915 -20.376  1.00  0.69           O  \\nATOM    171  N   PHE A  20       9.948   7.235 -17.910  1.00  0.87           N  \\nATOM    172  CA  PHE A  20      10.867   6.118 -18.095  1.00  0.88           C  \\nATOM    173  C   PHE A  20      10.487   5.303 -19.326  1.00  0.86           C  \\nATOM    174  CB  PHE A  20      10.879   5.220 -16.854  1.00  0.86           C  \\nATOM    175  O   PHE A  20       9.322   5.282 -19.728  1.00  0.84           O  \\nATOM    176  CG  PHE A  20      11.379   5.908 -15.612  1.00  0.86           C  \\nATOM    177  CD1 PHE A  20      12.725   5.865 -15.271  1.00  0.84           C  \\nATOM    178  CD2 PHE A  20      10.501   6.597 -14.786  1.00  0.85           C  \\nATOM    179  CE1 PHE A  20      13.190   6.501 -14.122  1.00  0.87           C  \\nATOM    180  CE2 PHE A  20      10.958   7.235 -13.636  1.00  0.87           C  \\nATOM    181  CZ  PHE A  20      12.303   7.185 -13.305  1.00  0.86           C  \\nATOM    182  N   ASP A  21      11.462   4.695 -19.909  1.00  0.86           N  \\nATOM    183  CA  ASP A  21      11.266   3.959 -21.154  1.00  0.86           C  \\nATOM    184  C   ASP A  21      10.561   2.628 -20.900  1.00  0.85           C  \\nATOM    185  CB  ASP A  21      12.607   3.719 -21.852  1.00  0.84           C  \\nATOM    186  O   ASP A  21       9.805   2.149 -21.747  1.00  0.83           O  \\nATOM    187  CG  ASP A  21      13.280   5.004 -22.301  1.00  0.80           C  \\nATOM    188  OD1 ASP A  21      12.599   5.881 -22.876  1.00  0.79           O  \\nATOM    189  OD2 ASP A  21      14.503   5.140 -22.081  1.00  0.80           O  \\nATOM    190  N   ASP A  22      10.833   2.138 -19.629  1.00  0.85           N  \\nATOM    191  CA  ASP A  22      10.293   0.841 -19.233  1.00  0.85           C  \\nATOM    192  C   ASP A  22       9.946   0.821 -17.747  1.00  0.86           C  \\nATOM    193  CB  ASP A  22      11.288  -0.276 -19.556  1.00  0.83           C  \\nATOM    194  O   ASP A  22      10.516   1.580 -16.960  1.00  0.85           O  \\nATOM    195  CG  ASP A  22      10.863  -1.626 -19.007  1.00  0.76           C  \\nATOM    196  OD1 ASP A  22       9.650  -1.930 -19.006  1.00  0.72           O  \\nATOM    197  OD2 ASP A  22      11.748  -2.392 -18.567  1.00  0.76           O  \\nATOM    198  N   GLU A  23       8.950  -0.051 -17.386  1.00  0.84           N  \\nATOM    199  CA  GLU A  23       8.467  -0.214 -16.018  1.00  0.85           C  \\nATOM    200  C   GLU A  23       9.583  -0.688 -15.092  1.00  0.86           C  \\nATOM    201  CB  GLU A  23       7.295  -1.197 -15.976  1.00  0.82           C  \\nATOM    202  O   GLU A  23       9.635  -0.298 -13.924  1.00  0.85           O  \\nATOM    203  CG  GLU A  23       6.595  -1.261 -14.626  1.00  0.74           C  \\nATOM    204  CD  GLU A  23       5.387  -2.186 -14.619  1.00  0.72           C  \\nATOM    205  OE1 GLU A  23       5.098  -2.813 -15.663  1.00  0.70           O  \\nATOM    206  OE2 GLU A  23       4.725  -2.282 -13.562  1.00  0.66           O  \\nATOM    207  N   LYS A  24      10.556  -1.397 -15.609  1.00  0.85           N  \\nATOM    208  CA  LYS A  24      11.661  -1.898 -14.797  1.00  0.86           C  \\nATOM    209  C   LYS A  24      12.576  -0.761 -14.352  1.00  0.86           C  \\nATOM    210  CB  LYS A  24      12.464  -2.945 -15.571  1.00  0.83           C  \\nATOM    211  O   LYS A  24      12.987  -0.706 -13.191  1.00  0.86           O  \\nATOM    212  CG  LYS A  24      13.526  -3.649 -14.740  1.00  0.77           C  \\nATOM    213  CD  LYS A  24      14.267  -4.702 -15.554  1.00  0.75           C  \\nATOM    214  CE  LYS A  24      15.380  -5.354 -14.746  1.00  0.70           C  \\nATOM    215  NZ  LYS A  24      16.154  -6.337 -15.562  1.00  0.66           N  \\nATOM    216  N   ILE A  25      12.823   0.139 -15.257  1.00  0.89           N  \\nATOM    217  CA  ILE A  25      13.645   1.309 -14.969  1.00  0.89           C  \\nATOM    218  C   ILE A  25      12.906   2.233 -14.003  1.00  0.89           C  \\nATOM    219  CB  ILE A  25      14.017   2.071 -16.260  1.00  0.88           C  \\nATOM    220  O   ILE A  25      13.497   2.748 -13.051  1.00  0.88           O  \\nATOM    221  CG1 ILE A  25      14.780   1.152 -17.221  1.00  0.85           C  \\nATOM    222  CG2 ILE A  25      14.836   3.323 -15.933  1.00  0.83           C  \\nATOM    223  CD1 ILE A  25      15.087   1.786 -18.571  1.00  0.81           C  \\nATOM    224  N   LEU A  26      11.556   2.318 -14.162  1.00  0.88           N  \\nATOM    225  CA  LEU A  26      10.726   3.091 -13.245  1.00  0.88           C  \\nATOM    226  C   LEU A  26      10.789   2.512 -11.836  1.00  0.88           C  \\nATOM    227  CB  LEU A  26       9.275   3.124 -13.732  1.00  0.87           C  \\nATOM    228  O   LEU A  26      11.047   3.237 -10.872  1.00  0.87           O  \\nATOM    229  CG  LEU A  26       8.246   3.697 -12.756  1.00  0.83           C  \\nATOM    230  CD1 LEU A  26       8.573   5.152 -12.435  1.00  0.80           C  \\nATOM    231  CD2 LEU A  26       6.839   3.574 -13.331  1.00  0.80           C  \\nATOM    232  N   ILE A  27      10.767   1.171 -11.612  1.00  0.87           N  \\nATOM    233  CA  ILE A  27      10.816   0.491 -10.322  1.00  0.88           C  \\nATOM    234  C   ILE A  27      12.197   0.672  -9.696  1.00  0.88           C  \\nATOM    235  CB  ILE A  27      10.482  -1.011 -10.461  1.00  0.87           C  \\nATOM    236  O   ILE A  27      12.311   0.983  -8.508  1.00  0.88           O  \\nATOM    237  CG1 ILE A  27       9.002  -1.199 -10.812  1.00  0.85           C  \\nATOM    238  CG2 ILE A  27      10.841  -1.765  -9.177  1.00  0.84           C  \\nATOM    239  CD1 ILE A  27       8.654  -2.600 -11.295  1.00  0.82           C  \\nATOM    240  N   GLN A  28      13.266   0.602 -10.514  1.00  0.89           N  \\nATOM    241  CA  GLN A  28      14.624   0.816 -10.024  1.00  0.90           C  \\nATOM    242  C   GLN A  28      14.829   2.263  -9.586  1.00  0.90           C  \\nATOM    243  CB  GLN A  28      15.648   0.442 -11.097  1.00  0.87           C  \\nATOM    244  O   GLN A  28      15.460   2.523  -8.559  1.00  0.89           O  \\nATOM    245  CG  GLN A  28      15.725  -1.052 -11.380  1.00  0.78           C  \\nATOM    246  CD  GLN A  28      16.116  -1.862 -10.158  1.00  0.76           C  \\nATOM    247  NE2 GLN A  28      15.686  -3.118 -10.116  1.00  0.72           N  \\nATOM    248  OE1 GLN A  28      16.800  -1.362  -9.259  1.00  0.77           O  \\nATOM    249  N   HIS A  29      14.226   3.222 -10.323  1.00  0.91           N  \\nATOM    250  CA  HIS A  29      14.222   4.634  -9.957  1.00  0.92           C  \\nATOM    251  C   HIS A  29      13.508   4.857  -8.629  1.00  0.92           C  \\nATOM    252  CB  HIS A  29      13.562   5.469 -11.056  1.00  0.91           C  \\nATOM    253  O   HIS A  29      14.045   5.514  -7.733  1.00  0.91           O  \\nATOM    254  CG  HIS A  29      13.248   6.870 -10.636  1.00  0.89           C  \\nATOM    255  CD2 HIS A  29      12.101   7.426 -10.180  1.00  0.87           C  \\nATOM    256  ND1 HIS A  29      14.183   7.883 -10.666  1.00  0.85           N  \\nATOM    257  CE1 HIS A  29      13.622   9.004 -10.244  1.00  0.87           C  \\nATOM    258  NE2 HIS A  29      12.359   8.754  -9.943  1.00  0.89           N  \\nATOM    259  N   GLN A  30      12.388   4.282  -8.458  1.00  0.91           N  \\nATOM    260  CA  GLN A  30      11.617   4.445  -7.229  1.00  0.91           C  \\nATOM    261  C   GLN A  30      12.377   3.892  -6.026  1.00  0.91           C  \\nATOM    262  CB  GLN A  30      10.257   3.757  -7.352  1.00  0.90           C  \\nATOM    263  O   GLN A  30      12.549   4.586  -5.022  1.00  0.90           O  \\nATOM    264  CG  GLN A  30       9.275   4.493  -8.253  1.00  0.87           C  \\nATOM    265  CD  GLN A  30       7.997   3.711  -8.491  1.00  0.84           C  \\nATOM    266  NE2 GLN A  30       6.891   4.424  -8.677  1.00  0.85           N  \\nATOM    267  OE1 GLN A  30       8.003   2.476  -8.509  1.00  0.86           O  \\nATOM    268  N   LYS A  31      13.043   2.725  -6.115  1.00  0.91           N  \\nATOM    269  CA  LYS A  31      13.869   2.119  -5.074  1.00  0.92           C  \\nATOM    270  C   LYS A  31      15.067   3.004  -4.739  1.00  0.91           C  \\nATOM    271  CB  LYS A  31      14.347   0.733  -5.507  1.00  0.90           C  \\nATOM    272  O   LYS A  31      15.431   3.147  -3.570  1.00  0.90           O  \\nATOM    273  CG  LYS A  31      13.247  -0.318  -5.544  1.00  0.85           C  \\nATOM    274  CD  LYS A  31      13.763  -1.649  -6.076  1.00  0.83           C  \\nATOM    275  CE  LYS A  31      12.677  -2.717  -6.060  1.00  0.84           C  \\nATOM    276  NZ  LYS A  31      13.187  -4.032  -6.550  1.00  0.81           N  \\nATOM    277  N   ALA A  32      15.678   3.589  -5.749  1.00  0.94           N  \\nATOM    278  CA  ALA A  32      16.936   4.311  -5.577  1.00  0.94           C  \\nATOM    279  C   ALA A  32      16.689   5.728  -5.067  1.00  0.93           C  \\nATOM    280  CB  ALA A  32      17.712   4.349  -6.891  1.00  0.92           C  \\nATOM    281  O   ALA A  32      17.504   6.277  -4.322  1.00  0.92           O  \\nATOM    282  N   LYS A  33      15.534   6.306  -5.475  1.00  0.94           N  \\nATOM    283  CA  LYS A  33      15.316   7.721  -5.191  1.00  0.94           C  \\nATOM    284  C   LYS A  33      14.292   7.907  -4.075  1.00  0.94           C  \\nATOM    285  CB  LYS A  33      14.858   8.457  -6.451  1.00  0.92           C  \\nATOM    286  O   LYS A  33      14.494   8.717  -3.168  1.00  0.92           O  \\nATOM    287  CG  LYS A  33      14.936   9.973  -6.345  1.00  0.80           C  \\nATOM    288  CD  LYS A  33      14.257  10.652  -7.528  1.00  0.75           C  \\nATOM    289  CE  LYS A  33      14.229  12.165  -7.364  1.00  0.72           C  \\nATOM    290  NZ  LYS A  33      13.345  12.815  -8.378  1.00  0.66           N  \\nATOM    291  N   HIS A  34      13.175   7.221  -4.112  1.00  0.93           N  \\nATOM    292  CA  HIS A  34      12.029   7.478  -3.246  1.00  0.93           C  \\nATOM    293  C   HIS A  34      12.010   6.519  -2.060  1.00  0.93           C  \\nATOM    294  CB  HIS A  34      10.724   7.362  -4.036  1.00  0.93           C  \\nATOM    295  O   HIS A  34      11.587   6.891  -0.963  1.00  0.91           O  \\nATOM    296  CG  HIS A  34      10.606   8.357  -5.146  1.00  0.92           C  \\nATOM    297  CD2 HIS A  34      10.553   8.189  -6.489  1.00  0.89           C  \\nATOM    298  ND1 HIS A  34      10.530   9.715  -4.924  1.00  0.89           N  \\nATOM    299  CE1 HIS A  34      10.434  10.340  -6.085  1.00  0.90           C  \\nATOM    300  NE2 HIS A  34      10.446   9.437  -7.051  1.00  0.91           N  \\nATOM    301  N   PHE A  35      12.495   5.310  -2.224  1.00  0.93           N  \\nATOM    302  CA  PHE A  35      12.411   4.317  -1.160  1.00  0.94           C  \\nATOM    303  C   PHE A  35      13.801   3.943  -0.657  1.00  0.93           C  \\nATOM    304  CB  PHE A  35      11.676   3.065  -1.648  1.00  0.92           C  \\nATOM    305  O   PHE A  35      14.049   2.789  -0.304  1.00  0.90           O  \\nATOM    306  CG  PHE A  35      10.323   3.349  -2.242  1.00  0.91           C  \\nATOM    307  CD1 PHE A  35       9.505   4.336  -1.705  1.00  0.87           C  \\nATOM    308  CD2 PHE A  35       9.868   2.628  -3.339  1.00  0.88           C  \\nATOM    309  CE1 PHE A  35       8.253   4.601  -2.254  1.00  0.88           C  \\nATOM    310  CE2 PHE A  35       8.617   2.887  -3.892  1.00  0.89           C  \\nATOM    311  CZ  PHE A  35       7.811   3.874  -3.347  1.00  0.88           C  \\nATOM    312  N   LYS A  36      14.722   4.682  -0.585  1.00  0.94           N  \\nATOM    313  CA  LYS A  36      16.054   4.473  -0.023  1.00  0.95           C  \\nATOM    314  C   LYS A  36      16.141   5.019   1.400  1.00  0.94           C  \\nATOM    315  CB  LYS A  36      17.118   5.131  -0.902  1.00  0.94           C  \\nATOM    316  O   LYS A  36      15.890   6.204   1.632  1.00  0.93           O  \\nATOM    317  CG  LYS A  36      18.546   4.778  -0.515  1.00  0.91           C  \\nATOM    318  CD  LYS A  36      19.558   5.496  -1.398  1.00  0.90           C  \\nATOM    319  CE  LYS A  36      20.988   5.185  -0.978  1.00  0.92           C  \\nATOM    320  NZ  LYS A  36      21.982   5.978  -1.762  1.00  0.91           N  \\nATOM    321  N   CYS A  37      16.587   4.227   2.311  1.00  0.93           N  \\nATOM    322  CA  CYS A  37      16.860   4.709   3.660  1.00  0.95           C  \\nATOM    323  C   CYS A  37      17.964   5.760   3.651  1.00  0.93           C  \\nATOM    324  CB  CYS A  37      17.256   3.550   4.575  1.00  0.91           C  \\nATOM    325  O   CYS A  37      19.032   5.540   3.077  1.00  0.90           O  \\nATOM    326  SG  CYS A  37      17.625   4.053   6.270  1.00  0.85           S  \\nATOM    327  N   HIS A  38      17.814   6.846   4.301  1.00  0.94           N  \\nATOM    328  CA  HIS A  38      18.803   7.918   4.327  1.00  0.94           C  \\nATOM    329  C   HIS A  38      19.985   7.556   5.220  1.00  0.93           C  \\nATOM    330  CB  HIS A  38      18.165   9.224   4.804  1.00  0.91           C  \\nATOM    331  O   HIS A  38      21.101   8.031   5.002  1.00  0.91           O  \\nATOM    332  CG  HIS A  38      17.623   9.154   6.196  1.00  0.83           C  \\nATOM    333  CD2 HIS A  38      18.150   9.553   7.377  1.00  0.78           C  \\nATOM    334  ND1 HIS A  38      16.388   8.614   6.487  1.00  0.76           N  \\nATOM    335  CE1 HIS A  38      16.180   8.686   7.791  1.00  0.79           C  \\nATOM    336  NE2 HIS A  38      17.233   9.252   8.354  1.00  0.83           N  \\nATOM    337  N   ILE A  39      19.811   6.557   6.161  1.00  0.94           N  \\nATOM    338  CA  ILE A  39      20.836   6.215   7.142  1.00  0.95           C  \\nATOM    339  C   ILE A  39      21.739   5.118   6.584  1.00  0.94           C  \\nATOM    340  CB  ILE A  39      20.209   5.764   8.480  1.00  0.94           C  \\nATOM    341  O   ILE A  39      22.954   5.303   6.472  1.00  0.92           O  \\nATOM    342  CG1 ILE A  39      19.347   6.885   9.071  1.00  0.89           C  \\nATOM    343  CG2 ILE A  39      21.296   5.331   9.467  1.00  0.87           C  \\nATOM    344  CD1 ILE A  39      18.534   6.465  10.289  1.00  0.87           C  \\nATOM    345  N   CYS A  40      21.225   3.998   6.101  1.00  0.94           N  \\nATOM    346  CA  CYS A  40      22.032   2.847   5.713  1.00  0.94           C  \\nATOM    347  C   CYS A  40      22.002   2.644   4.203  1.00  0.93           C  \\nATOM    348  CB  CYS A  40      21.536   1.583   6.416  1.00  0.92           C  \\nATOM    349  O   CYS A  40      22.541   1.659   3.694  1.00  0.91           O  \\nATOM    350  SG  CYS A  40      19.902   1.040   5.870  1.00  0.95           S  \\nATOM    351  N   HIS A  41      21.300   3.383   3.500  1.00  0.92           N  \\nATOM    352  CA  HIS A  41      21.214   3.450   2.045  1.00  0.92           C  \\nATOM    353  C   HIS A  41      20.568   2.193   1.473  1.00  0.90           C  \\nATOM    354  CB  HIS A  41      22.603   3.650   1.435  1.00  0.89           C  \\nATOM    355  O   HIS A  41      20.666   1.931   0.272  1.00  0.87           O  \\nATOM    356  CG  HIS A  41      23.265   4.923   1.857  1.00  0.81           C  \\nATOM    357  CD2 HIS A  41      24.401   5.144   2.561  1.00  0.77           C  \\nATOM    358  ND1 HIS A  41      22.751   6.165   1.555  1.00  0.78           N  \\nATOM    359  CE1 HIS A  41      23.544   7.098   2.056  1.00  0.77           C  \\nATOM    360  NE2 HIS A  41      24.553   6.504   2.671  1.00  0.83           N  \\nATOM    361  N   LYS A  42      19.979   1.410   2.350  1.00  0.95           N  \\nATOM    362  CA  LYS A  42      19.206   0.252   1.912  1.00  0.95           C  \\nATOM    363  C   LYS A  42      18.030   0.675   1.036  1.00  0.95           C  \\nATOM    364  CB  LYS A  42      18.703  -0.544   3.117  1.00  0.95           C  \\nATOM    365  O   LYS A  42      17.258   1.560   1.409  1.00  0.94           O  \\nATOM    366  CG  LYS A  42      18.043  -1.867   2.753  1.00  0.93           C  \\nATOM    367  CD  LYS A  42      17.771  -2.714   3.989  1.00  0.90           C  \\nATOM    368  CE  LYS A  42      17.131  -4.047   3.625  1.00  0.91           C  \\nATOM    369  NZ  LYS A  42      16.930  -4.912   4.825  1.00  0.90           N  \\nATOM    370  N   LYS A  43      17.830   0.088  -0.122  1.00  0.92           N  \\nATOM    371  CA  LYS A  43      16.725   0.281  -1.057  1.00  0.93           C  \\nATOM    372  C   LYS A  43      15.561  -0.651  -0.733  1.00  0.91           C  \\nATOM    373  CB  LYS A  43      17.192   0.055  -2.496  1.00  0.90           C  \\nATOM    374  O   LYS A  43      15.749  -1.862  -0.596  1.00  0.87           O  \\nATOM    375  CG  LYS A  43      18.314   0.984  -2.935  1.00  0.85           C  \\nATOM    376  CD  LYS A  43      18.772   0.675  -4.355  1.00  0.83           C  \\nATOM    377  CE  LYS A  43      19.955   1.542  -4.763  1.00  0.87           C  \\nATOM    378  NZ  LYS A  43      20.473   1.172  -6.114  1.00  0.85           N  \\nATOM    379  N   LEU A  44      14.382  -0.154  -0.716  1.00  0.92           N  \\nATOM    380  CA  LEU A  44      13.137  -0.848  -0.406  1.00  0.92           C  \\nATOM    381  C   LEU A  44      12.152  -0.742  -1.565  1.00  0.90           C  \\nATOM    382  CB  LEU A  44      12.509  -0.278   0.868  1.00  0.91           C  \\nATOM    383  O   LEU A  44      12.447  -0.108  -2.580  1.00  0.87           O  \\nATOM    384  CG  LEU A  44      13.305  -0.468   2.161  1.00  0.88           C  \\nATOM    385  CD1 LEU A  44      13.100   0.725   3.089  1.00  0.84           C  \\nATOM    386  CD2 LEU A  44      12.901  -1.766   2.852  1.00  0.84           C  \\nATOM    387  N   PHE A  45      11.036  -1.459  -1.455  1.00  0.88           N  \\nATOM    388  CA  PHE A  45      10.129  -1.593  -2.588  1.00  0.88           C  \\nATOM    389  C   PHE A  45       8.965  -0.616  -2.470  1.00  0.86           C  \\nATOM    390  CB  PHE A  45       9.601  -3.028  -2.688  1.00  0.85           C  \\nATOM    391  O   PHE A  45       8.282  -0.334  -3.456  1.00  0.83           O  \\nATOM    392  CG  PHE A  45      10.605  -4.008  -3.233  1.00  0.79           C  \\nATOM    393  CD1 PHE A  45      10.778  -4.158  -4.604  1.00  0.75           C  \\nATOM    394  CD2 PHE A  45      11.378  -4.779  -2.374  1.00  0.76           C  \\nATOM    395  CE1 PHE A  45      11.706  -5.063  -5.111  1.00  0.75           C  \\nATOM    396  CE2 PHE A  45      12.308  -5.686  -2.874  1.00  0.75           C  \\nATOM    397  CZ  PHE A  45      12.470  -5.827  -4.243  1.00  0.74           C  \\nATOM    398  N   SER A  46       8.687  -0.033  -1.246  1.00  0.88           N  \\nATOM    399  CA  SER A  46       7.503   0.786  -1.006  1.00  0.90           C  \\nATOM    400  C   SER A  46       7.757   1.816   0.090  1.00  0.90           C  \\nATOM    401  CB  SER A  46       6.311  -0.093  -0.625  1.00  0.87           C  \\nATOM    402  O   SER A  46       8.721   1.698   0.849  1.00  0.89           O  \\nATOM    403  OG  SER A  46       6.519  -0.702   0.638  1.00  0.79           O  \\nATOM    404  N   GLY A  47       6.883   2.890   0.174  1.00  0.90           N  \\nATOM    405  CA  GLY A  47       6.883   3.855   1.261  1.00  0.91           C  \\nATOM    406  C   GLY A  47       6.714   3.217   2.627  1.00  0.91           C  \\nATOM    407  O   GLY A  47       7.569   3.373   3.501  1.00  0.90           O  \\nATOM    408  N   PRO A  48       5.660   2.448   2.827  1.00  0.90           N  \\nATOM    409  CA  PRO A  48       5.489   1.768   4.113  1.00  0.91           C  \\nATOM    410  C   PRO A  48       6.692   0.906   4.489  1.00  0.91           C  \\nATOM    411  CB  PRO A  48       4.246   0.903   3.891  1.00  0.89           C  \\nATOM    412  O   PRO A  48       7.127   0.916   5.643  1.00  0.90           O  \\nATOM    413  CG  PRO A  48       3.436   1.649   2.880  1.00  0.86           C  \\nATOM    414  CD  PRO A  48       4.374   2.294   1.900  1.00  0.85           C  \\nATOM    415  N   GLY A  49       7.266   0.158   3.553  1.00  0.91           N  \\nATOM    416  CA  GLY A  49       8.467  -0.627   3.792  1.00  0.92           C  \\nATOM    417  C   GLY A  49       9.638   0.205   4.277  1.00  0.92           C  \\nATOM    418  O   GLY A  49      10.395  -0.227   5.149  1.00  0.91           O  \\nATOM    419  N   LEU A  50       9.841   1.390   3.745  1.00  0.94           N  \\nATOM    420  CA  LEU A  50      10.850   2.349   4.181  1.00  0.94           C  \\nATOM    421  C   LEU A  50      10.561   2.835   5.597  1.00  0.94           C  \\nATOM    422  CB  LEU A  50      10.906   3.541   3.221  1.00  0.94           C  \\nATOM    423  O   LEU A  50      11.458   2.865   6.442  1.00  0.93           O  \\nATOM    424  CG  LEU A  50      11.865   4.671   3.599  1.00  0.92           C  \\nATOM    425  CD1 LEU A  50      13.293   4.145   3.693  1.00  0.89           C  \\nATOM    426  CD2 LEU A  50      11.775   5.809   2.588  1.00  0.89           C  \\nATOM    427  N   SER A  51       9.215   3.157   5.937  1.00  0.94           N  \\nATOM    428  CA  SER A  51       8.818   3.583   7.275  1.00  0.94           C  \\nATOM    429  C   SER A  51       9.110   2.501   8.309  1.00  0.94           C  \\nATOM    430  CB  SER A  51       7.331   3.940   7.304  1.00  0.93           C  \\nATOM    431  O   SER A  51       9.841   2.737   9.273  1.00  0.92           O  \\nATOM    432  OG  SER A  51       6.942   4.369   8.598  1.00  0.86           O  \\nATOM    433  N   ILE A  52       8.724   1.270   8.110  1.00  0.93           N  \\nATOM    434  CA  ILE A  52       8.922   0.135   9.006  1.00  0.94           C  \\nATOM    435  C   ILE A  52      10.416  -0.119   9.193  1.00  0.94           C  \\nATOM    436  CB  ILE A  52       8.225  -1.136   8.471  1.00  0.93           C  \\nATOM    437  O   ILE A  52      10.882  -0.319  10.317  1.00  0.93           O  \\nATOM    438  CG1 ILE A  52       6.702  -0.981   8.547  1.00  0.88           C  \\nATOM    439  CG2 ILE A  52       8.689  -2.372   9.246  1.00  0.87           C  \\nATOM    440  CD1 ILE A  52       5.932  -2.037   7.765  1.00  0.84           C  \\nATOM    441  N   HIS A  53      11.186  -0.112   8.080  1.00  0.93           N  \\nATOM    442  CA  HIS A  53      12.631  -0.296   8.154  1.00  0.94           C  \\nATOM    443  C   HIS A  53      13.279   0.777   9.022  1.00  0.94           C  \\nATOM    444  CB  HIS A  53      13.246  -0.280   6.753  1.00  0.94           C  \\nATOM    445  O   HIS A  53      14.101   0.469   9.888  1.00  0.93           O  \\nATOM    446  CG  HIS A  53      14.741  -0.238   6.754  1.00  0.91           C  \\nATOM    447  CD2 HIS A  53      15.602   0.790   6.566  1.00  0.87           C  \\nATOM    448  ND1 HIS A  53      15.516  -1.358   6.965  1.00  0.87           N  \\nATOM    449  CE1 HIS A  53      16.793  -1.019   6.908  1.00  0.89           C  \\nATOM    450  NE2 HIS A  53      16.872   0.279   6.667  1.00  0.92           N  \\nATOM    451  N   CYS A  54      12.941   2.031   8.799  1.00  0.95           N  \\nATOM    452  CA  CYS A  54      13.525   3.104   9.596  1.00  0.95           C  \\nATOM    453  C   CYS A  54      13.175   2.940  11.071  1.00  0.95           C  \\nATOM    454  CB  CYS A  54      13.042   4.466   9.098  1.00  0.94           C  \\nATOM    455  O   CYS A  54      14.058   2.974  11.930  1.00  0.94           O  \\nATOM    456  SG  CYS A  54      13.768   4.962   7.520  1.00  0.94           S  \\nATOM    457  N   MET A  55      11.926   2.643  11.381  1.00  0.94           N  \\nATOM    458  CA  MET A  55      11.462   2.477  12.755  1.00  0.94           C  \\nATOM    459  C   MET A  55      12.147   1.288  13.422  1.00  0.93           C  \\nATOM    460  CB  MET A  55       9.944   2.294  12.793  1.00  0.91           C  \\nATOM    461  O   MET A  55      12.598   1.387  14.564  1.00  0.91           O  \\nATOM    462  CG  MET A  55       9.365   2.253  14.198  1.00  0.79           C  \\nATOM    463  SD  MET A  55       7.560   1.925  14.206  1.00  0.77           S  \\nATOM    464  CE  MET A  55       7.000   3.188  13.029  1.00  0.69           C  \\nATOM    465  N   GLN A  56      12.262   0.235  12.760  1.00  0.94           N  \\nATOM    466  CA  GLN A  56      12.769  -0.994  13.362  1.00  0.94           C  \\nATOM    467  C   GLN A  56      14.290  -0.961  13.479  1.00  0.94           C  \\nATOM    468  CB  GLN A  56      12.332  -2.212  12.547  1.00  0.93           C  \\nATOM    469  O   GLN A  56      14.850  -1.405  14.484  1.00  0.93           O  \\nATOM    470  CG  GLN A  56      10.853  -2.548  12.689  1.00  0.87           C  \\nATOM    471  CD  GLN A  56      10.456  -3.788  11.911  1.00  0.82           C  \\nATOM    472  NE2 GLN A  56       9.250  -4.285  12.163  1.00  0.85           N  \\nATOM    473  OE1 GLN A  56      11.226  -4.295  11.089  1.00  0.87           O  \\nATOM    474  N   VAL A  57      15.046  -0.451  12.453  1.00  0.94           N  \\nATOM    475  CA  VAL A  57      16.495  -0.581  12.350  1.00  0.95           C  \\nATOM    476  C   VAL A  57      17.168   0.669  12.914  1.00  0.94           C  \\nATOM    477  CB  VAL A  57      16.942  -0.815  10.890  1.00  0.93           C  \\nATOM    478  O   VAL A  57      18.188   0.576  13.601  1.00  0.93           O  \\nATOM    479  CG1 VAL A  57      18.459  -0.982  10.809  1.00  0.81           C  \\nATOM    480  CG2 VAL A  57      16.235  -2.036  10.304  1.00  0.78           C  \\nATOM    481  N   HIS A  58      16.691   1.827  12.671  1.00  0.94           N  \\nATOM    482  CA  HIS A  58      17.377   3.073  12.997  1.00  0.94           C  \\nATOM    483  C   HIS A  58      16.690   3.794  14.152  1.00  0.92           C  \\nATOM    484  CB  HIS A  58      17.440   3.986  11.771  1.00  0.93           C  \\nATOM    485  O   HIS A  58      17.215   4.781  14.672  1.00  0.89           O  \\nATOM    486  CG  HIS A  58      18.196   3.398  10.623  1.00  0.89           C  \\nATOM    487  CD2 HIS A  58      17.804   3.096   9.362  1.00  0.84           C  \\nATOM    488  ND1 HIS A  58      19.526   3.047  10.709  1.00  0.82           N  \\nATOM    489  CE1 HIS A  58      19.921   2.555   9.546  1.00  0.86           C  \\nATOM    490  NE2 HIS A  58      18.895   2.574   8.712  1.00  0.89           N  \\nATOM    491  N   LYS A  59      15.489   3.377  14.624  1.00  0.90           N  \\nATOM    492  CA  LYS A  59      14.624   4.028  15.604  1.00  0.90           C  \\nATOM    493  C   LYS A  59      14.246   5.436  15.153  1.00  0.87           C  \\nATOM    494  CB  LYS A  59      15.306   4.081  16.971  1.00  0.84           C  \\nATOM    495  O   LYS A  59      14.058   6.329  15.982  1.00  0.83           O  \\nATOM    496  CG  LYS A  59      15.793   2.729  17.473  1.00  0.75           C  \\nATOM    497  CD  LYS A  59      14.673   1.697  17.475  1.00  0.73           C  \\nATOM    498  CE  LYS A  59      15.143   0.361  18.033  1.00  0.74           C  \\nATOM    499  NZ  LYS A  59      14.147  -0.724  17.786  1.00  0.73           N  \\nATOM    500  N   GLU A  60      14.086   5.631  13.992  1.00  0.94           N  \\nATOM    501  CA  GLU A  60      13.657   6.902  13.417  1.00  0.94           C  \\nATOM    502  C   GLU A  60      12.349   6.746  12.647  1.00  0.93           C  \\nATOM    503  CB  GLU A  60      14.743   7.470  12.499  1.00  0.92           C  \\nATOM    504  O   GLU A  60      12.153   5.756  11.940  1.00  0.91           O  \\nATOM    505  CG  GLU A  60      15.927   8.069  13.245  1.00  0.80           C  \\nATOM    506  CD  GLU A  60      16.907   8.789  12.333  1.00  0.75           C  \\nATOM    507  OE1 GLU A  60      16.603   8.965  11.132  1.00  0.79           O  \\nATOM    508  OE2 GLU A  60      17.990   9.181  12.825  1.00  0.77           O  \\nATOM    509  N   THR A  61      11.526   7.678  12.872  1.00  0.93           N  \\nATOM    510  CA  THR A  61      10.265   7.753  12.143  1.00  0.93           C  \\nATOM    511  C   THR A  61      10.413   8.616  10.893  1.00  0.92           C  \\nATOM    512  CB  THR A  61       9.140   8.319  13.030  1.00  0.90           C  \\nATOM    513  O   THR A  61      10.883   9.754  10.970  1.00  0.90           O  \\nATOM    514  CG2 THR A  61       8.706   7.304  14.082  1.00  0.75           C  \\nATOM    515  OG1 THR A  61       9.608   9.503  13.688  1.00  0.78           O  \\nATOM    516  N   ILE A  62      10.010   8.165   9.799  1.00  0.94           N  \\nATOM    517  CA  ILE A  62       9.892   8.875   8.530  1.00  0.94           C  \\nATOM    518  C   ILE A  62       8.434   8.882   8.076  1.00  0.92           C  \\nATOM    519  CB  ILE A  62      10.788   8.241   7.442  1.00  0.91           C  \\nATOM    520  O   ILE A  62       7.820   7.824   7.921  1.00  0.88           O  \\nATOM    521  CG1 ILE A  62      12.264   8.326   7.850  1.00  0.74           C  \\nATOM    522  CG2 ILE A  62      10.555   8.918   6.088  1.00  0.73           C  \\nATOM    523  CD1 ILE A  62      13.223   7.730   6.829  1.00  0.72           C  \\nATOM    524  N   ASP A  63       7.858  10.039   7.942  1.00  0.92           N  \\nATOM    525  CA  ASP A  63       6.458  10.296   7.618  1.00  0.92           C  \\nATOM    526  C   ASP A  63       6.302  10.730   6.162  1.00  0.91           C  \\nATOM    527  CB  ASP A  63       5.879  11.363   8.550  1.00  0.88           C  \\nATOM    528  O   ASP A  63       5.187  10.775   5.639  1.00  0.89           O  \\nATOM    529  CG  ASP A  63       6.648  12.672   8.503  1.00  0.76           C  \\nATOM    530  OD1 ASP A  63       7.839  12.666   8.125  1.00  0.70           O  \\nATOM    531  OD2 ASP A  63       6.057  13.718   8.850  1.00  0.79           O  \\nATOM    532  N   LYS A  64       7.380  10.960   5.508  1.00  0.94           N  \\nATOM    533  CA  LYS A  64       7.338  11.501   4.152  1.00  0.94           C  \\nATOM    534  C   LYS A  64       8.274  10.732   3.224  1.00  0.94           C  \\nATOM    535  CB  LYS A  64       7.706  12.985   4.155  1.00  0.93           C  \\nATOM    536  O   LYS A  64       9.369  10.335   3.628  1.00  0.94           O  \\nATOM    537  CG  LYS A  64       6.703  13.870   4.880  1.00  0.89           C  \\nATOM    538  CD  LYS A  64       7.085  15.342   4.785  1.00  0.86           C  \\nATOM    539  CE  LYS A  64       6.080  16.229   5.507  1.00  0.86           C  \\nATOM    540  NZ  LYS A  64       6.445  17.674   5.410  1.00  0.83           N  \\nATOM    541  N   ILE A  65       7.887  10.491   1.996  1.00  0.91           N  \\nATOM    542  CA  ILE A  65       8.696   9.859   0.959  1.00  0.92           C  \\nATOM    543  C   ILE A  65       9.788  10.822   0.499  1.00  0.91           C  \\nATOM    544  CB  ILE A  65       7.830   9.418  -0.242  1.00  0.89           C  \\nATOM    545  O   ILE A  65       9.495  11.911   0.001  1.00  0.89           O  \\nATOM    546  CG1 ILE A  65       6.811   8.358   0.195  1.00  0.79           C  \\nATOM    547  CG2 ILE A  65       8.712   8.893  -1.379  1.00  0.77           C  \\nATOM    548  CD1 ILE A  65       5.924   7.851  -0.933  1.00  0.75           C  \\nATOM    549  N   PRO A  66      11.038  10.561   0.594  1.00  0.92           N  \\nATOM    550  CA  PRO A  66      12.121  11.423   0.115  1.00  0.93           C  \\nATOM    551  C   PRO A  66      12.029  11.710  -1.382  1.00  0.93           C  \\nATOM    552  CB  PRO A  66      13.382  10.617   0.439  1.00  0.91           C  \\nATOM    553  O   PRO A  66      11.670  10.825  -2.163  1.00  0.92           O  \\nATOM    554  CG  PRO A  66      12.950   9.633   1.478  1.00  0.86           C  \\nATOM    555  CD  PRO A  66      11.489   9.343   1.282  1.00  0.85           C  \\nATOM    556  N   ALA A  67      12.313  12.886  -1.758  1.00  0.93           N  \\nATOM    557  CA  ALA A  67      12.457  13.393  -3.120  1.00  0.93           C  \\nATOM    558  C   ALA A  67      11.156  13.241  -3.903  1.00  0.92           C  \\nATOM    559  CB  ALA A  67      13.595  12.673  -3.839  1.00  0.91           C  \\nATOM    560  O   ALA A  67      11.173  13.125  -5.130  1.00  0.89           O  \\nATOM    561  N   ALA A  68      10.019  13.274  -3.216  1.00  0.90           N  \\nATOM    562  CA  ALA A  68       8.698  13.382  -3.829  1.00  0.90           C  \\nATOM    563  C   ALA A  68       8.334  14.840  -4.096  1.00  0.89           C  \\nATOM    564  CB  ALA A  68       7.644  12.728  -2.939  1.00  0.88           C  \\nATOM    565  O   ALA A  68       8.877  15.748  -3.462  1.00  0.87           O  \\nATOM    566  N   VAL A  69       7.451  15.131  -5.021  1.00  0.88           N  \\nATOM    567  CA  VAL A  69       6.836  16.435  -5.243  1.00  0.88           C  \\nATOM    568  C   VAL A  69       5.929  16.786  -4.066  1.00  0.87           C  \\nATOM    569  CB  VAL A  69       6.034  16.468  -6.563  1.00  0.85           C  \\nATOM    570  O   VAL A  69       5.318  15.902  -3.460  1.00  0.86           O  \\nATOM    571  CG1 VAL A  69       5.350  17.821  -6.746  1.00  0.75           C  \\nATOM    572  CG2 VAL A  69       6.947  16.162  -7.749  1.00  0.75           C  \\nATOM    573  N   HIS A  70       5.843  18.078  -3.729  1.00  0.84           N  \\nATOM    574  CA  HIS A  70       4.976  18.556  -2.658  1.00  0.86           C  \\nATOM    575  C   HIS A  70       3.545  18.063  -2.845  1.00  0.84           C  \\nATOM    576  CB  HIS A  70       5.000  20.085  -2.592  1.00  0.81           C  \\nATOM    577  O   HIS A  70       3.010  18.105  -3.955  1.00  0.81           O  \\nATOM    578  CG  HIS A  70       4.265  20.646  -1.416  1.00  0.73           C  \\nATOM    579  CD2 HIS A  70       3.085  21.306  -1.338  1.00  0.70           C  \\nATOM    580  ND1 HIS A  70       4.745  20.558  -0.128  1.00  0.68           N  \\nATOM    581  CE1 HIS A  70       3.889  21.140   0.695  1.00  0.69           C  \\nATOM    582  NE2 HIS A  70       2.874  21.602  -0.014  1.00  0.67           N  \\nATOM    583  N   GLY A  71       2.917  17.468  -1.772  1.00  0.86           N  \\nATOM    584  CA  GLY A  71       1.572  16.917  -1.758  1.00  0.86           C  \\nATOM    585  C   GLY A  71       1.515  15.469  -2.207  1.00  0.84           C  \\nATOM    586  O   GLY A  71       0.477  14.816  -2.086  1.00  0.81           O  \\nATOM    587  N   ARG A  72       2.641  14.815  -2.579  1.00  0.87           N  \\nATOM    588  CA  ARG A  72       2.730  13.412  -2.969  1.00  0.87           C  \\nATOM    589  C   ARG A  72       3.769  12.675  -2.130  1.00  0.86           C  \\nATOM    590  CB  ARG A  72       3.073  13.288  -4.455  1.00  0.86           C  \\nATOM    591  O   ARG A  72       4.335  11.674  -2.573  1.00  0.84           O  \\nATOM    592  CG  ARG A  72       2.054  13.937  -5.379  1.00  0.84           C  \\nATOM    593  CD  ARG A  72       2.354  13.645  -6.843  1.00  0.82           C  \\nATOM    594  NE  ARG A  72       1.493  14.417  -7.733  1.00  0.78           N  \\nATOM    595  NH1 ARG A  72       2.591  13.799  -9.669  1.00  0.68           N  \\nATOM    596  NH2 ARG A  72       0.785  15.207  -9.771  1.00  0.66           N  \\nATOM    597  CZ  ARG A  72       1.625  14.473  -9.056  1.00  0.82           C  \\nATOM    598  N   ASP A  73       4.022  13.176  -1.009  1.00  0.88           N  \\nATOM    599  CA  ASP A  73       5.073  12.697  -0.116  1.00  0.90           C  \\nATOM    600  C   ASP A  73       4.499  11.786   0.968  1.00  0.88           C  \\nATOM    601  CB  ASP A  73       5.811  13.875   0.524  1.00  0.86           C  \\nATOM    602  O   ASP A  73       5.232  11.311   1.838  1.00  0.84           O  \\nATOM    603  CG  ASP A  73       4.880  14.852   1.219  1.00  0.80           C  \\nATOM    604  OD1 ASP A  73       3.675  14.888   0.887  1.00  0.77           O  \\nATOM    605  OD2 ASP A  73       5.355  15.595   2.105  1.00  0.83           O  \\nATOM    606  N   ASN A  74       3.089  11.560   0.881  1.00  0.88           N  \\nATOM    607  CA  ASN A  74       2.473  10.639   1.831  1.00  0.89           C  \\nATOM    608  C   ASN A  74       3.060   9.235   1.711  1.00  0.88           C  \\nATOM    609  CB  ASN A  74       0.956  10.599   1.634  1.00  0.86           C  \\nATOM    610  O   ASN A  74       2.952   8.600   0.660  1.00  0.86           O  \\nATOM    611  CG  ASN A  74       0.247   9.814   2.719  1.00  0.79           C  \\nATOM    612  ND2 ASN A  74      -1.062  10.009   2.832  1.00  0.78           N  \\nATOM    613  OD1 ASN A  74       0.869   9.039   3.450  1.00  0.77           O  \\nATOM    614  N   ILE A  75       3.623   8.722   2.815  1.00  0.85           N  \\nATOM    615  CA  ILE A  75       4.422   7.503   2.877  1.00  0.87           C  \\nATOM    616  C   ILE A  75       3.508   6.282   2.804  1.00  0.81           C  \\nATOM    617  CB  ILE A  75       5.281   7.454   4.160  1.00  0.80           C  \\nATOM    618  O   ILE A  75       3.943   5.195   2.417  1.00  0.75           O  \\nATOM    619  CG1 ILE A  75       6.428   6.451   3.998  1.00  0.65           C  \\nATOM    620  CG2 ILE A  75       4.416   7.106   5.375  1.00  0.63           C  \\nATOM    621  CD1 ILE A  75       7.558   6.633   5.002  1.00  0.65           C  \\nATOM    622  N   HIS A  76       2.153   6.550   3.157  1.00  0.83           N  \\nATOM    623  CA  HIS A  76       1.156   5.487   3.204  1.00  0.83           C  \\nATOM    624  C   HIS A  76       0.540   5.248   1.829  1.00  0.80           C  \\nATOM    625  CB  HIS A  76       0.061   5.823   4.218  1.00  0.79           C  \\nATOM    626  O   HIS A  76      -0.219   4.294   1.641  1.00  0.76           O  \\nATOM    627  CG  HIS A  76       0.568   5.995   5.614  1.00  0.73           C  \\nATOM    628  CD2 HIS A  76       0.516   7.054   6.455  1.00  0.70           C  \\nATOM    629  ND1 HIS A  76       1.226   4.993   6.295  1.00  0.69           N  \\nATOM    630  CE1 HIS A  76       1.556   5.431   7.499  1.00  0.69           C  \\nATOM    631  NE2 HIS A  76       1.137   6.679   7.621  1.00  0.72           N  \\nATOM    632  N   VAL A  77       0.779   5.979   0.994  1.00  0.80           N  \\nATOM    633  CA  VAL A  77       0.251   5.826  -0.358  1.00  0.78           C  \\nATOM    634  C   VAL A  77       1.203   4.970  -1.191  1.00  0.77           C  \\nATOM    635  CB  VAL A  77       0.030   7.195  -1.039  1.00  0.76           C  \\nATOM    636  O   VAL A  77       2.345   5.364  -1.440  1.00  0.73           O  \\nATOM    637  CG1 VAL A  77      -0.547   7.013  -2.442  1.00  0.72           C  \\nATOM    638  CG2 VAL A  77      -0.889   8.071  -0.190  1.00  0.72           C  \\nATOM    639  N   GLU A  78       0.756   3.760  -1.652  1.00  0.72           N  \\nATOM    640  CA  GLU A  78       1.436   2.824  -2.543  1.00  0.72           C  \\nATOM    641  C   GLU A  78       1.022   3.043  -3.995  1.00  0.69           C  \\nATOM    642  CB  GLU A  78       1.147   1.379  -2.126  1.00  0.66           C  \\nATOM    643  O   GLU A  78      -0.035   2.575  -4.422  1.00  0.64           O  \\nATOM    644  CG  GLU A  78       1.772   0.986  -0.795  1.00  0.62           C  \\nATOM    645  CD  GLU A  78       1.598  -0.487  -0.462  1.00  0.61           C  \\nATOM    646  OE1 GLU A  78       0.833  -1.182  -1.168  1.00  0.66           O  \\nATOM    647  OE2 GLU A  78       2.232  -0.949   0.512  1.00  0.65           O  \\nATOM    648  N   ILE A  79       1.777   3.716  -4.702  1.00  0.73           N  \\nATOM    649  CA  ILE A  79       1.616   3.920  -6.137  1.00  0.73           C  \\nATOM    650  C   ILE A  79       2.880   3.472  -6.867  1.00  0.72           C  \\nATOM    651  CB  ILE A  79       1.301   5.397  -6.466  1.00  0.69           C  \\nATOM    652  O   ILE A  79       3.937   4.091  -6.728  1.00  0.68           O  \\nATOM    653  CG1 ILE A  79       0.050   5.858  -5.709  1.00  0.63           C  \\nATOM    654  CG2 ILE A  79       1.130   5.589  -7.976  1.00  0.63           C  \\nATOM    655  CD1 ILE A  79      -0.269   7.336  -5.885  1.00  0.63           C  \\nATOM    656  N   TYR A  80       2.934   2.361  -7.638  1.00  0.63           N  \\nATOM    657  CA  TYR A  80       4.045   1.683  -8.296  1.00  0.62           C  \\nATOM    658  C   TYR A  80       4.321   2.291  -9.666  1.00  0.61           C  \\nATOM    659  CB  TYR A  80       3.754   0.186  -8.438  1.00  0.57           C  \\nATOM    660  O   TYR A  80       5.460   2.284 -10.137  1.00  0.58           O  \\nATOM    661  CG  TYR A  80       3.697  -0.550  -7.122  1.00  0.53           C  \\nATOM    662  CD1 TYR A  80       4.863  -0.972  -6.486  1.00  0.50           C  \\nATOM    663  CD2 TYR A  80       2.478  -0.825  -6.511  1.00  0.52           C  \\nATOM    664  CE1 TYR A  80       4.815  -1.651  -5.274  1.00  0.51           C  \\nATOM    665  CE2 TYR A  80       2.419  -1.504  -5.298  1.00  0.49           C  \\nATOM    666  OH  TYR A  80       3.538  -2.585  -3.488  1.00  0.47           O  \\nATOM    667  CZ  TYR A  80       3.591  -1.912  -4.688  1.00  0.51           C  \\nATOM    668  N   GLY A  81       3.426   3.096 -10.219  1.00  0.66           N  \\nATOM    669  CA  GLY A  81       3.453   3.611 -11.579  1.00  0.64           C  \\nATOM    670  C   GLY A  81       2.070   3.836 -12.159  1.00  0.64           C  \\nATOM    671  O   GLY A  81       1.107   3.182 -11.755  1.00  0.59           O  \\nATOM    672  N   MET A  82       1.920   4.939 -12.938  1.00  0.67           N  \\nATOM    673  CA  MET A  82       0.767   5.272 -13.770  1.00  0.67           C  \\nATOM    674  C   MET A  82       1.104   5.128 -15.250  1.00  0.67           C  \\nATOM    675  CB  MET A  82       0.287   6.695 -13.480  1.00  0.61           C  \\nATOM    676  O   MET A  82       2.220   5.441 -15.671  1.00  0.64           O  \\nATOM    677  CG  MET A  82      -0.258   6.883 -12.073  1.00  0.55           C  \\nATOM    678  SD  MET A  82      -1.440   8.282 -11.959  1.00  0.56           S  \\nATOM    679  CE  MET A  82      -0.668   9.449 -13.114  1.00  0.41           C  \\nATOM    680  N   GLN A  83       0.252   4.309 -15.822  1.00  0.67           N  \\nATOM    681  CA  GLN A  83       0.170   4.287 -17.278  1.00  0.66           C  \\nATOM    682  C   GLN A  83      -0.819   5.331 -17.789  1.00  0.66           C  \\nATOM    683  CB  GLN A  83      -0.232   2.897 -17.773  1.00  0.60           C  \\nATOM    684  O   GLN A  83      -1.863   5.559 -17.173  1.00  0.60           O  \\nATOM    685  CG  GLN A  83       0.821   1.826 -17.521  1.00  0.55           C  \\nATOM    686  CD  GLN A  83       0.459   0.490 -18.140  1.00  0.58           C  \\nATOM    687  NE2 GLN A  83       1.357  -0.483 -18.018  1.00  0.43           N  \\nATOM    688  OE1 GLN A  83      -0.618   0.332 -18.723  1.00  0.60           O  \\nATOM    689  N   GLY A  84      -0.475   6.173 -18.559  1.00  0.57           N  \\nATOM    690  CA  GLY A  84      -1.487   7.029 -19.156  1.00  0.58           C  \\nATOM    691  C   GLY A  84      -1.707   8.319 -18.389  1.00  0.55           C  \\nATOM    692  O   GLY A  84      -1.292   8.439 -17.234  1.00  0.50           O  \\nATOM    693  N   ILE A  85      -1.655   9.660 -19.118  1.00  0.45           N  \\nATOM    694  CA  ILE A  85      -1.559  11.067 -19.490  1.00  0.44           C  \\nATOM    695  C   ILE A  85      -0.308  11.681 -18.867  1.00  0.45           C  \\nATOM    696  CB  ILE A  85      -2.816  11.853 -19.055  1.00  0.40           C  \\nATOM    697  O   ILE A  85      -0.170  11.716 -17.642  1.00  0.42           O  \\nATOM    698  CG1 ILE A  85      -4.078  11.211 -19.643  1.00  0.36           C  \\nATOM    699  CG2 ILE A  85      -2.705  13.323 -19.472  1.00  0.37           C  \\nATOM    700  CD1 ILE A  85      -5.375  11.866 -19.189  1.00  0.32           C  \\nATOM    701  N   PRO A  86       0.793  11.970 -19.688  1.00  0.44           N  \\nATOM    702  CA  PRO A  86       2.063  12.670 -19.892  1.00  0.44           C  \\nATOM    703  C   PRO A  86       1.901  14.188 -19.925  1.00  0.44           C  \\nATOM    704  CB  PRO A  86       2.542  12.143 -21.247  1.00  0.40           C  \\nATOM    705  O   PRO A  86       0.943  14.698 -20.512  1.00  0.42           O  \\nATOM    706  CG  PRO A  86       1.434  11.262 -21.728  1.00  0.39           C  \\nATOM    707  CD  PRO A  86       0.364  11.223 -20.674  1.00  0.43           C  \\nATOM    708  N   SER A  87       2.454  14.985 -18.952  1.00  0.41           N  \\nATOM    709  CA  SER A  87       3.254  16.169 -18.657  1.00  0.41           C  \\nATOM    710  C   SER A  87       2.508  17.446 -19.027  1.00  0.41           C  \\nATOM    711  CB  SER A  87       4.588  16.114 -19.401  1.00  0.38           C  \\nATOM    712  O   SER A  87       2.757  18.506 -18.449  1.00  0.38           O  \\nATOM    713  OG  SER A  87       4.428  15.521 -20.678  1.00  0.37           O  \\nATOM    714  N   GLY A  88       1.301  17.511 -19.305  1.00  0.44           N  \\nATOM    715  CA  GLY A  88       0.392  18.623 -19.533  1.00  0.43           C  \\nATOM    716  C   GLY A  88      -0.853  18.563 -18.668  1.00  0.44           C  \\nATOM    717  O   GLY A  88      -1.616  19.530 -18.601  1.00  0.41           O  \\nATOM    718  N   ALA A  89      -0.945  17.504 -17.871  1.00  0.42           N  \\nATOM    719  CA  ALA A  89      -2.178  17.551 -17.089  1.00  0.42           C  \\nATOM    720  C   ALA A  89      -1.957  18.268 -15.760  1.00  0.42           C  \\nATOM    721  CB  ALA A  89      -2.708  16.140 -16.847  1.00  0.39           C  \\nATOM    722  O   ALA A  89      -2.901  18.801 -15.171  1.00  0.39           O  \\nATOM    723  N   TYR A  90      -0.698  18.693 -15.572  1.00  0.44           N  \\nATOM    724  CA  TYR A  90      -0.474  19.523 -14.393  1.00  0.45           C  \\nATOM    725  C   TYR A  90       0.233  20.821 -14.766  1.00  0.44           C  \\nATOM    726  CB  TYR A  90       0.348  18.763 -13.348  1.00  0.41           C  \\nATOM    727  O   TYR A  90       1.315  21.115 -14.254  1.00  0.41           O  \\nATOM    728  CG  TYR A  90      -0.480  17.868 -12.458  1.00  0.39           C  \\nATOM    729  CD1 TYR A  90      -1.132  18.379 -11.338  1.00  0.36           C  \\nATOM    730  CD2 TYR A  90      -0.610  16.512 -12.734  1.00  0.38           C  \\nATOM    731  CE1 TYR A  90      -1.895  17.558 -10.514  1.00  0.38           C  \\nATOM    732  CE2 TYR A  90      -1.370  15.681 -11.916  1.00  0.40           C  \\nATOM    733  OH  TYR A  90      -2.761  15.396  -9.998  1.00  0.31           O  \\nATOM    734  CZ  TYR A  90      -2.008  16.213 -10.810  1.00  0.34           C  \\nATOM    735  N   ARG A  91       0.360  21.172 -15.938  1.00  0.44           N  \\nATOM    736  CA  ARG A  91       0.713  22.549 -16.271  1.00  0.44           C  \\nATOM    737  C   ARG A  91      -0.279  23.532 -15.659  1.00  0.44           C  \\nATOM    738  CB  ARG A  91       0.768  22.737 -17.789  1.00  0.41           C  \\nATOM    739  O   ARG A  91      -0.193  24.738 -15.897  1.00  0.41           O  \\nATOM    740  CG  ARG A  91       2.077  22.288 -18.418  1.00  0.40           C  \\nATOM    741  CD  ARG A  91       2.129  22.612 -19.904  1.00  0.43           C  \\nATOM    742  NE  ARG A  91       3.154  21.830 -20.591  1.00  0.40           N  \\nATOM    743  NH1 ARG A  91       3.409  23.317 -22.340  1.00  0.32           N  \\nATOM    744  NH2 ARG A  91       4.659  21.397 -22.271  1.00  0.28           N  \\nATOM    745  CZ  ARG A  91       3.738  22.183 -21.732  1.00  0.38           C  \\nATOM    746  N   GLY A  92      -0.974  23.240 -14.566  1.00  0.42           N  \\nATOM    747  CA  GLY A  92      -1.790  24.273 -13.948  1.00  0.42           C  \\nATOM    748  C   GLY A  92      -1.593  24.371 -12.447  1.00  0.42           C  \\nATOM    749  O   GLY A  92      -2.170  25.244 -11.796  1.00  0.39           O  \\nATOM    750  N   ALA A  93      -0.241  24.127 -11.875  1.00  0.42           N  \\nATOM    751  CA  ALA A  93      -0.161  24.657 -10.516  1.00  0.44           C  \\nATOM    752  C   ALA A  93       1.263  24.567  -9.975  1.00  0.42           C  \\nATOM    753  CB  ALA A  93      -1.125  23.913  -9.596  1.00  0.39           C  \\nATOM    754  O   ALA A  93       1.542  25.020  -8.863  1.00  0.40           O  \\nATOM    755  N   ALA A  94       2.407  24.677 -10.880  1.00  0.44           N  \\nATOM    756  CA  ALA A  94       3.676  24.999 -10.234  1.00  0.45           C  \\nATOM    757  C   ALA A  94       4.717  25.441 -11.259  1.00  0.44           C  \\nATOM    758  CB  ALA A  94       4.190  23.799  -9.443  1.00  0.41           C  \\nATOM    759  O   ALA A  94       5.897  25.585 -10.932  1.00  0.42           O  \\nATOM    760  N   ASP A  95       4.188  26.193 -12.348  1.00  0.45           N  \\nATOM    761  CA  ASP A  95       5.117  27.036 -13.094  1.00  0.46           C  \\nATOM    762  C   ASP A  95       4.420  28.288 -13.621  1.00  0.45           C  \\nATOM    763  CB  ASP A  95       5.740  26.254 -14.251  1.00  0.41           C  \\nATOM    764  O   ASP A  95       3.935  28.304 -14.754  1.00  0.42           O  \\nATOM    765  CG  ASP A  95       6.748  25.216 -13.791  1.00  0.40           C  \\nATOM    766  OD1 ASP A  95       7.530  25.496 -12.856  1.00  0.42           O  \\nATOM    767  OD2 ASP A  95       6.764  24.108 -14.369  1.00  0.44           O  \\nATOM    768  N   GLU A  96       3.445  28.896 -12.862  1.00  0.42           N  \\nATOM    769  CA  GLU A  96       3.431  30.352 -12.968  1.00  0.43           C  \\nATOM    770  C   GLU A  96       4.822  30.935 -12.733  1.00  0.42           C  \\nATOM    771  CB  GLU A  96       2.434  30.954 -11.975  1.00  0.39           C  \\nATOM    772  O   GLU A  96       5.307  30.960 -11.600  1.00  0.40           O  \\nATOM    773  CG  GLU A  96       0.977  30.695 -12.333  1.00  0.38           C  \\nATOM    774  CD  GLU A  96      -0.002  31.403 -11.411  1.00  0.42           C  \\nATOM    775  OE1 GLU A  96       0.424  31.920 -10.354  1.00  0.46           O  \\nATOM    776  OE2 GLU A  96      -1.207  31.440 -11.748  1.00  0.44           O  \\nATOM    777  N   GLU A  97       5.705  30.718 -13.551  1.00  0.46           N  \\nATOM    778  CA  GLU A  97       6.376  31.966 -13.901  1.00  0.47           C  \\nATOM    779  C   GLU A  97       5.414  32.939 -14.578  1.00  0.46           C  \\nATOM    780  CB  GLU A  97       7.576  31.695 -14.812  1.00  0.44           C  \\nATOM    781  O   GLU A  97       4.724  32.574 -15.532  1.00  0.44           O  \\nATOM    782  CG  GLU A  97       8.918  31.756 -14.096  1.00  0.44           C  \\nATOM    783  CD  GLU A  97      10.105  31.575 -15.028  1.00  0.49           C  \\nATOM    784  OE1 GLU A  97       9.933  30.993 -16.123  1.00  0.51           O  \\nATOM    785  OE2 GLU A  97      11.216  32.018 -14.661  1.00  0.46           O  \\nATOM    786  N   PRO A  98       4.712  33.816 -13.862  1.00  0.48           N  \\nATOM    787  CA  PRO A  98       4.154  35.000 -14.520  1.00  0.49           C  \\nATOM    788  C   PRO A  98       4.969  35.439 -15.735  1.00  0.48           C  \\nATOM    789  CB  PRO A  98       4.193  36.066 -13.422  1.00  0.45           C  \\nATOM    790  O   PRO A  98       6.087  35.940 -15.584  1.00  0.46           O  \\nATOM    791  CG  PRO A  98       5.127  35.518 -12.391  1.00  0.44           C  \\nATOM    792  CD  PRO A  98       5.344  34.058 -12.672  1.00  0.48           C  \\nATOM    793  N   ASP A  99       5.029  34.787 -16.827  1.00  0.44           N  \\nATOM    794  CA  ASP A  99       5.511  35.816 -17.743  1.00  0.48           C  \\nATOM    795  C   ASP A  99       5.890  35.214 -19.095  1.00  0.44           C  \\nATOM    796  CB  ASP A  99       6.710  36.551 -17.141  1.00  0.41           C  \\nATOM    797  O   ASP A  99       6.850  34.447 -19.192  1.00  0.43           O  \\nATOM    798  CG  ASP A  99       6.454  38.034 -16.934  1.00  0.43           C  \\nATOM    799  OD1 ASP A  99       5.491  38.574 -17.519  1.00  0.44           O  \\nATOM    800  OD2 ASP A  99       7.224  38.668 -16.180  1.00  0.45           O  \\nATOM    801  N   GLU A 100       4.738  34.982 -20.024  1.00  0.49           N  \\nATOM    802  CA  GLU A 100       4.836  35.518 -21.379  1.00  0.50           C  \\nATOM    803  C   GLU A 100       4.317  34.516 -22.407  1.00  0.49           C  \\nATOM    804  CB  GLU A 100       6.281  35.903 -21.702  1.00  0.46           C  \\nATOM    805  O   GLU A 100       4.621  34.626 -23.597  1.00  0.47           O  \\nATOM    806  CG  GLU A 100       6.587  37.378 -21.489  1.00  0.45           C  \\nATOM    807  CD  GLU A 100       7.986  37.772 -21.936  1.00  0.49           C  \\nATOM    808  OE1 GLU A 100       8.872  36.889 -22.005  1.00  0.50           O  \\nATOM    809  OE2 GLU A 100       8.198  38.972 -22.219  1.00  0.45           O  \\nATOM    810  N   LYS A 101       3.172  33.841 -22.071  1.00  0.50           N  \\nATOM    811  CA  LYS A 101       2.411  33.473 -23.262  1.00  0.50           C  \\nATOM    812  C   LYS A 101       0.914  33.667 -23.040  1.00  0.50           C  \\nATOM    813  CB  LYS A 101       2.699  32.024 -23.657  1.00  0.46           C  \\nATOM    814  O   LYS A 101       0.215  32.735 -22.636  1.00  0.47           O  \\nATOM    815  CG  LYS A 101       3.940  31.852 -24.520  1.00  0.44           C  \\nATOM    816  CD  LYS A 101       4.187  30.387 -24.858  1.00  0.47           C  \\nATOM    817  CE  LYS A 101       5.444  30.211 -25.699  1.00  0.40           C  \\nATOM    818  NZ  LYS A 101       5.814  28.772 -25.848  1.00  0.38           N  \\nATOM    819  N   ARG A 102       0.573  34.823 -22.352  1.00  0.49           N  \\nATOM    820  CA  ARG A 102      -0.573  35.684 -22.627  1.00  0.49           C  \\nATOM    821  C   ARG A 102      -0.518  36.232 -24.049  1.00  0.49           C  \\nATOM    822  CB  ARG A 102      -0.633  36.837 -21.623  1.00  0.46           C  \\nATOM    823  O   ARG A 102       0.207  37.190 -24.323  1.00  0.46           O  \\nATOM    824  CG  ARG A 102      -1.312  36.477 -20.311  1.00  0.45           C  \\nATOM    825  CD  ARG A 102      -1.373  37.666 -19.363  1.00  0.48           C  \\nATOM    826  NE  ARG A 102      -2.004  37.312 -18.094  1.00  0.43           N  \\nATOM    827  NH1 ARG A 102      -1.962  39.457 -17.237  1.00  0.33           N  \\nATOM    828  NH2 ARG A 102      -2.845  37.740 -16.000  1.00  0.29           N  \\nATOM    829  CZ  ARG A 102      -2.269  38.170 -17.113  1.00  0.41           C  \\nATOM    830  N   SER A 103      -0.423  35.374 -25.023  1.00  0.50           N  \\nATOM    831  CA  SER A 103      -0.821  35.950 -26.304  1.00  0.52           C  \\nATOM    832  C   SER A 103      -1.182  34.862 -27.310  1.00  0.50           C  \\nATOM    833  CB  SER A 103       0.297  36.827 -26.869  1.00  0.46           C  \\nATOM    834  O   SER A 103      -0.309  34.135 -27.787  1.00  0.48           O  \\nATOM    835  OG  SER A 103       0.363  38.063 -26.180  1.00  0.48           O  \\nATOM    836  N   ARG A 104      -2.344  34.052 -26.961  1.00  0.55           N  \\nATOM    837  CA  ARG A 104      -3.249  33.759 -28.067  1.00  0.55           C  \\nATOM    838  C   ARG A 104      -4.326  32.765 -27.644  1.00  0.55           C  \\nATOM    839  CB  ARG A 104      -2.473  33.211 -29.267  1.00  0.51           C  \\nATOM    840  O   ARG A 104      -4.178  31.558 -27.845  1.00  0.52           O  \\nATOM    841  CG  ARG A 104      -2.443  34.150 -30.462  1.00  0.50           C  \\nATOM    842  CD  ARG A 104      -1.629  33.574 -31.612  1.00  0.52           C  \\nATOM    843  NE  ARG A 104      -1.621  34.466 -32.767  1.00  0.44           N  \\nATOM    844  NH1 ARG A 104      -0.225  33.140 -34.043  1.00  0.33           N  \\nATOM    845  NH2 ARG A 104      -1.017  35.119 -34.885  1.00  0.28           N  \\nATOM    846  CZ  ARG A 104      -0.955  34.240 -33.896  1.00  0.44           C  \\nATOM    847  N   MET A 105      -5.080  33.126 -26.635  1.00  0.56           N  \\nATOM    848  CA  MET A 105      -6.464  32.689 -26.474  1.00  0.57           C  \\nATOM    849  C   MET A 105      -7.406  33.553 -27.306  1.00  0.56           C  \\nATOM    850  CB  MET A 105      -6.876  32.735 -25.002  1.00  0.51           C  \\nATOM    851  O   MET A 105      -7.746  34.668 -26.909  1.00  0.54           O  \\nATOM    852  CG  MET A 105      -6.360  31.562 -24.184  1.00  0.50           C  \\nATOM    853  SD  MET A 105      -7.091  31.501 -22.503  1.00  0.52           S  \\nATOM    854  CE  MET A 105      -6.615  29.822 -22.007  1.00  0.43           C  \\nATOM    855  N   ASP A 106      -7.275  33.546 -28.528  1.00  0.52           N  \\nATOM    856  CA  ASP A 106      -8.165  33.855 -29.644  1.00  0.55           C  \\nATOM    857  C   ASP A 106      -9.182  32.737 -29.861  1.00  0.52           C  \\nATOM    858  CB  ASP A 106      -7.359  34.089 -30.923  1.00  0.48           C  \\nATOM    859  O   ASP A 106      -8.811  31.607 -30.186  1.00  0.50           O  \\nATOM    860  CG  ASP A 106      -7.732  35.379 -31.631  1.00  0.49           C  \\nATOM    861  OD1 ASP A 106      -8.770  35.985 -31.288  1.00  0.48           O  \\nATOM    862  OD2 ASP A 106      -6.982  35.792 -32.543  1.00  0.51           O  \\nATOM    863  N   ASN A 107     -10.380  32.613 -29.052  1.00  0.59           N  \\nATOM    864  CA  ASN A 107     -11.699  32.371 -29.625  1.00  0.60           C  \\nATOM    865  C   ASN A 107     -12.210  30.974 -29.285  1.00  0.59           C  \\nATOM    866  CB  ASN A 107     -11.671  32.572 -31.142  1.00  0.54           C  \\nATOM    867  O   ASN A 107     -12.269  30.101 -30.154  1.00  0.56           O  \\nATOM    868  CG  ASN A 107     -11.679  34.035 -31.538  1.00  0.52           C  \\nATOM    869  ND2 ASN A 107     -11.097  34.340 -32.692  1.00  0.49           N  \\nATOM    870  OD1 ASN A 107     -12.203  34.884 -30.812  1.00  0.53           O  \\nATOM    871  N   GLY A 108     -12.073  30.583 -27.961  1.00  0.60           N  \\nATOM    872  CA  GLY A 108     -13.144  29.669 -27.596  1.00  0.60           C  \\nATOM    873  C   GLY A 108     -14.267  30.341 -26.830  1.00  0.59           C  \\nATOM    874  O   GLY A 108     -14.027  31.267 -26.052  1.00  0.57           O  \\nATOM    875  N   PRO A 109     -15.439  30.339 -27.373  1.00  0.60           N  \\nATOM    876  CA  PRO A 109     -16.659  30.795 -26.702  1.00  0.59           C  \\nATOM    877  C   PRO A 109     -16.618  30.578 -25.191  1.00  0.60           C  \\nATOM    878  CB  PRO A 109     -17.755  29.943 -27.346  1.00  0.56           C  \\nATOM    879  O   PRO A 109     -16.061  29.582 -24.722  1.00  0.57           O  \\nATOM    880  CG  PRO A 109     -17.063  28.689 -27.773  1.00  0.54           C  \\nATOM    881  CD  PRO A 109     -15.622  29.010 -28.050  1.00  0.55           C  \\nATOM    882  N   PRO A 110     -16.603  31.679 -24.299  1.00  0.59           N  \\nATOM    883  CA  PRO A 110     -16.786  31.679 -22.846  1.00  0.58           C  \\nATOM    884  C   PRO A 110     -17.306  30.344 -22.316  1.00  0.59           C  \\nATOM    885  CB  PRO A 110     -17.810  32.793 -22.619  1.00  0.55           C  \\nATOM    886  O   PRO A 110     -18.318  29.835 -22.805  1.00  0.57           O  \\nATOM    887  CG  PRO A 110     -18.589  32.860 -23.893  1.00  0.54           C  \\nATOM    888  CD  PRO A 110     -17.705  32.402 -25.018  1.00  0.54           C  \\nATOM    889  N   MET A 111     -16.518  29.372 -21.759  1.00  0.63           N  \\nATOM    890  CA  MET A 111     -17.068  28.307 -20.924  1.00  0.64           C  \\nATOM    891  C   MET A 111     -18.303  28.789 -20.171  1.00  0.63           C  \\nATOM    892  CB  MET A 111     -16.016  27.802 -19.935  1.00  0.59           C  \\nATOM    893  O   MET A 111     -18.294  29.873 -19.585  1.00  0.60           O  \\nATOM    894  CG  MET A 111     -14.990  26.867 -20.556  1.00  0.57           C  \\nATOM    895  SD  MET A 111     -15.545  25.117 -20.559  1.00  0.55           S  \\nATOM    896  CE  MET A 111     -14.931  24.585 -18.936  1.00  0.48           C  \\nATOM    897  N   PRO A 112     -19.481  28.351 -20.524  1.00  0.60           N  \\nATOM    898  CA  PRO A 112     -20.698  28.674 -19.776  1.00  0.60           C  \\nATOM    899  C   PRO A 112     -20.492  28.624 -18.264  1.00  0.60           C  \\nATOM    900  CB  PRO A 112     -21.686  27.595 -20.228  1.00  0.56           C  \\nATOM    901  O   PRO A 112     -19.881  27.684 -17.750  1.00  0.57           O  \\nATOM    902  CG  PRO A 112     -20.830  26.438 -20.630  1.00  0.55           C  \\nATOM    903  CD  PRO A 112     -19.507  26.966 -21.107  1.00  0.55           C  \\nATOM    904  N   THR A 113     -20.275  29.807 -17.461  1.00  0.62           N  \\nATOM    905  CA  THR A 113     -20.480  30.022 -16.033  1.00  0.62           C  \\nATOM    906  C   THR A 113     -21.146  28.806 -15.394  1.00  0.61           C  \\nATOM    907  CB  THR A 113     -21.336  31.275 -15.773  1.00  0.58           C  \\nATOM    908  O   THR A 113     -22.148  28.301 -15.904  1.00  0.59           O  \\nATOM    909  CG2 THR A 113     -20.468  32.456 -15.352  1.00  0.55           C  \\nATOM    910  OG1 THR A 113     -22.043  31.620 -16.971  1.00  0.57           O  \\nATOM    911  N   PRO A 114     -20.443  27.861 -14.669  1.00  0.63           N  \\nATOM    912  CA  PRO A 114     -21.224  26.867 -13.929  1.00  0.62           C  \\nATOM    913  C   PRO A 114     -22.634  27.351 -13.598  1.00  0.62           C  \\nATOM    914  CB  PRO A 114     -20.406  26.656 -12.652  1.00  0.58           C  \\nATOM    915  O   PRO A 114     -22.799  28.405 -12.977  1.00  0.59           O  \\nATOM    916  CG  PRO A 114     -19.639  27.927 -12.475  1.00  0.56           C  \\nATOM    917  CD  PRO A 114     -19.488  28.584 -13.817  1.00  0.57           C  \\nATOM    918  N   MET A 115     -23.583  27.019 -14.384  1.00  0.59           N  \\nATOM    919  CA  MET A 115     -24.986  27.155 -14.005  1.00  0.60           C  \\nATOM    920  C   MET A 115     -25.179  26.869 -12.520  1.00  0.59           C  \\nATOM    921  CB  MET A 115     -25.861  26.215 -14.837  1.00  0.56           C  \\nATOM    922  O   MET A 115     -24.622  25.906 -11.991  1.00  0.57           O  \\nATOM    923  CG  MET A 115     -26.074  26.680 -16.268  1.00  0.55           C  \\nATOM    924  SD  MET A 115     -27.789  27.256 -16.572  1.00  0.53           S  \\nATOM    925  CE  MET A 115     -28.507  25.754 -17.295  1.00  0.47           C  \\nATOM    926  N   PRO A 116     -25.351  27.919 -11.573  1.00  0.60           N  \\nATOM    927  CA  PRO A 116     -25.891  27.588 -10.252  1.00  0.60           C  \\nATOM    928  C   PRO A 116     -26.636  26.255 -10.237  1.00  0.60           C  \\nATOM    929  CB  PRO A 116     -26.845  28.749  -9.960  1.00  0.56           C  \\nATOM    930  O   PRO A 116     -27.543  26.041 -11.045  1.00  0.58           O  \\nATOM    931  CG  PRO A 116     -27.322  29.199 -11.303  1.00  0.55           C  \\nATOM    932  CD  PRO A 116     -26.264  28.875 -12.318  1.00  0.56           C  \\nATOM    933  N   PHE A 117     -25.956  25.055  -9.917  1.00  0.62           N  \\nATOM    934  CA  PHE A 117     -26.813  23.938  -9.538  1.00  0.62           C  \\nATOM    935  C   PHE A 117     -28.219  24.423  -9.207  1.00  0.61           C  \\nATOM    936  CB  PHE A 117     -26.221  23.187  -8.341  1.00  0.58           C  \\nATOM    937  O   PHE A 117     -28.393  25.326  -8.386  1.00  0.59           O  \\nATOM    938  CG  PHE A 117     -25.160  22.188  -8.716  1.00  0.56           C  \\nATOM    939  CD1 PHE A 117     -25.506  20.913  -9.147  1.00  0.53           C  \\nATOM    940  CD2 PHE A 117     -23.815  22.524  -8.636  1.00  0.53           C  \\nATOM    941  CE1 PHE A 117     -24.526  19.987  -9.495  1.00  0.50           C  \\nATOM    942  CE2 PHE A 117     -22.829  21.604  -8.982  1.00  0.51           C  \\nATOM    943  CZ  PHE A 117     -23.187  20.335  -9.410  1.00  0.47           C  \\nATOM    944  N   PRO A 118     -29.093  24.330 -10.062  1.00  0.55           N  \\nATOM    945  CA  PRO A 118     -30.489  24.531  -9.668  1.00  0.54           C  \\nATOM    946  C   PRO A 118     -30.821  23.889  -8.322  1.00  0.55           C  \\nATOM    947  CB  PRO A 118     -31.275  23.864 -10.799  1.00  0.51           C  \\nATOM    948  O   PRO A 118     -30.674  22.675  -8.161  1.00  0.53           O  \\nATOM    949  CG  PRO A 118     -30.345  22.833 -11.353  1.00  0.50           C  \\nATOM    950  CD  PRO A 118     -28.938  23.219 -10.998  1.00  0.51           C  \\nATOM    951  N   GLN A 119     -30.603  24.579  -7.110  1.00  0.58           N  \\nATOM    952  CA  GLN A 119     -31.528  24.482  -5.986  1.00  0.58           C  \\nATOM    953  C   GLN A 119     -32.882  23.937  -6.432  1.00  0.57           C  \\nATOM    954  CB  GLN A 119     -31.706  25.846  -5.318  1.00  0.54           C  \\nATOM    955  O   GLN A 119     -33.490  24.461  -7.368  1.00  0.55           O  \\nATOM    956  CG  GLN A 119     -30.485  26.312  -4.536  1.00  0.53           C  \\nATOM    957  CD  GLN A 119     -30.706  27.641  -3.839  1.00  0.55           C  \\nATOM    958  NE2 GLN A 119     -29.742  28.049  -3.021  1.00  0.44           N  \\nATOM    959  OE1 GLN A 119     -31.735  28.296  -4.034  1.00  0.55           O  \\nATOM    960  N   HIS A 120     -32.937  22.640  -6.661  1.00  0.58           N  \\nATOM    961  CA  HIS A 120     -34.241  22.052  -6.375  1.00  0.59           C  \\nATOM    962  C   HIS A 120     -34.960  21.657  -7.661  1.00  0.58           C  \\nATOM    963  CB  HIS A 120     -35.103  23.025  -5.569  1.00  0.53           C  \\nATOM    964  O   HIS A 120     -35.505  22.514  -8.361  1.00  0.55           O  \\nATOM    965  CG  HIS A 120     -34.826  22.997  -4.100  1.00  0.52           C  \\nATOM    966  CD2 HIS A 120     -34.269  23.921  -3.282  1.00  0.50           C  \\nATOM    967  ND1 HIS A 120     -35.129  21.912  -3.306  1.00  0.51           N  \\nATOM    968  CE1 HIS A 120     -34.772  22.172  -2.060  1.00  0.47           C  \\nATOM    969  NE2 HIS A 120     -34.247  23.384  -2.018  1.00  0.43           N  \\nATOM    970  N   PHE A 121     -34.576  20.679  -8.402  1.00  0.56           N  \\nATOM    971  CA  PHE A 121     -35.644  20.057  -9.175  1.00  0.57           C  \\nATOM    972  C   PHE A 121     -36.284  18.916  -8.394  1.00  0.56           C  \\nATOM    973  CB  PHE A 121     -35.110  19.542 -10.515  1.00  0.53           C  \\nATOM    974  O   PHE A 121     -35.603  17.964  -8.005  1.00  0.53           O  \\nATOM    975  CG  PHE A 121     -35.120  20.574 -11.610  1.00  0.51           C  \\nATOM    976  CD1 PHE A 121     -36.267  20.802 -12.360  1.00  0.48           C  \\nATOM    977  CD2 PHE A 121     -33.980  21.318 -11.890  1.00  0.49           C  \\nATOM    978  CE1 PHE A 121     -36.279  21.757 -13.374  1.00  0.47           C  \\nATOM    979  CE2 PHE A 121     -33.985  22.274 -12.901  1.00  0.49           C  \\nATOM    980  CZ  PHE A 121     -35.135  22.491 -13.643  1.00  0.42           C  \\nATOM    981  N   PRO A 122     -37.363  19.263  -7.733  1.00  0.58           N  \\nATOM    982  CA  PRO A 122     -38.351  18.237  -7.394  1.00  0.57           C  \\nATOM    983  C   PRO A 122     -38.436  17.130  -8.442  1.00  0.59           C  \\nATOM    984  CB  PRO A 122     -39.663  19.022  -7.322  1.00  0.54           C  \\nATOM    985  O   PRO A 122     -38.750  17.398  -9.604  1.00  0.56           O  \\nATOM    986  CG  PRO A 122     -39.504  20.126  -8.316  1.00  0.52           C  \\nATOM    987  CD  PRO A 122     -38.043  20.451  -8.438  1.00  0.53           C  \\nATOM    988  N   PHE A 123     -37.513  16.039  -8.506  1.00  0.59           N  \\nATOM    989  CA  PHE A 123     -38.024  14.825  -9.132  1.00  0.61           C  \\nATOM    990  C   PHE A 123     -39.546  14.783  -9.068  1.00  0.60           C  \\nATOM    991  CB  PHE A 123     -37.435  13.582  -8.457  1.00  0.55           C  \\nATOM    992  O   PHE A 123     -40.132  14.925  -7.993  1.00  0.58           O  \\nATOM    993  CG  PHE A 123     -36.085  13.184  -8.989  1.00  0.53           C  \\nATOM    994  CD1 PHE A 123     -35.976  12.417 -10.142  1.00  0.50           C  \\nATOM    995  CD2 PHE A 123     -34.925  13.579  -8.336  1.00  0.51           C  \\nATOM    996  CE1 PHE A 123     -34.727  12.047 -10.637  1.00  0.50           C  \\nATOM    997  CE2 PHE A 123     -33.673  13.213  -8.825  1.00  0.50           C  \\nATOM    998  CZ  PHE A 123     -33.577  12.447  -9.975  1.00  0.46           C  \\nATOM    999  N   PRO A 124     -40.235  15.115 -10.089  1.00  0.58           N  \\nATOM   1000  CA  PRO A 124     -41.640  14.702 -10.056  1.00  0.58           C  \\nATOM   1001  C   PRO A 124     -41.847  13.373  -9.334  1.00  0.58           C  \\nATOM   1002  CB  PRO A 124     -42.006  14.581 -11.538  1.00  0.54           C  \\nATOM   1003  O   PRO A 124     -41.454  12.321  -9.844  1.00  0.56           O  \\nATOM   1004  CG  PRO A 124     -40.695  14.469 -12.247  1.00  0.53           C  \\nATOM   1005  CD  PRO A 124     -39.606  14.906 -11.310  1.00  0.55           C  \\nATOM   1006  N   GLY A 125     -42.012  13.437  -8.013  1.00  0.56           N  \\nATOM   1007  CA  GLY A 125     -42.662  12.444  -7.172  1.00  0.56           C  \\nATOM   1008  C   GLY A 125     -42.099  12.393  -5.764  1.00  0.55           C  \\nATOM   1009  O   GLY A 125     -42.450  11.508  -4.981  1.00  0.53           O  \\nATOM   1010  N   MET A 126     -41.263  13.364  -5.352  1.00  0.57           N  \\nATOM   1011  CA  MET A 126     -41.118  13.526  -3.908  1.00  0.59           C  \\nATOM   1012  C   MET A 126     -41.855  14.770  -3.424  1.00  0.56           C  \\nATOM   1013  CB  MET A 126     -39.640  13.610  -3.522  1.00  0.52           C  \\nATOM   1014  O   MET A 126     -41.706  15.849  -4.000  1.00  0.54           O  \\nATOM   1015  CG  MET A 126     -38.977  12.255  -3.335  1.00  0.51           C  \\nATOM   1016  SD  MET A 126     -37.300  12.388  -2.602  1.00  0.55           S  \\nATOM   1017  CE  MET A 126     -37.384  11.060  -1.368  1.00  0.45           C  \\nATOM   1018  N   PRO A 127     -42.951  14.642  -2.736  1.00  0.59           N  \\nATOM   1019  CA  PRO A 127     -43.554  15.736  -1.971  1.00  0.58           C  \\nATOM   1020  C   PRO A 127     -42.515  16.632  -1.301  1.00  0.59           C  \\nATOM   1021  CB  PRO A 127     -44.403  15.008  -0.925  1.00  0.55           C  \\nATOM   1022  O   PRO A 127     -41.437  16.161  -0.929  1.00  0.57           O  \\nATOM   1023  CG  PRO A 127     -43.737  13.682  -0.748  1.00  0.53           C  \\nATOM   1024  CD  PRO A 127     -43.014  13.338  -2.019  1.00  0.54           C  \\nATOM   1025  N   PRO A 128     -42.466  17.826  -1.689  1.00  0.60           N  \\nATOM   1026  CA  PRO A 128     -41.705  18.850  -0.971  1.00  0.59           C  \\nATOM   1027  C   PRO A 128     -41.374  18.443   0.463  1.00  0.60           C  \\nATOM   1028  CB  PRO A 128     -42.639  20.063  -0.992  1.00  0.56           C  \\nATOM   1029  O   PRO A 128     -42.277  18.132   1.244  1.00  0.58           O  \\nATOM   1030  CG  PRO A 128     -44.014  19.484  -1.080  1.00  0.54           C  \\nATOM   1031  CD  PRO A 128     -43.930  18.158  -1.780  1.00  0.55           C  \\nATOM   1032  N   MET A 129     -40.132  18.097   0.854  1.00  0.63           N  \\nATOM   1033  CA  MET A 129     -39.705  18.071   2.250  1.00  0.64           C  \\nATOM   1034  C   MET A 129     -40.043  19.385   2.947  1.00  0.63           C  \\nATOM   1035  CB  MET A 129     -38.203  17.798   2.350  1.00  0.59           C  \\nATOM   1036  O   MET A 129     -39.793  20.462   2.404  1.00  0.60           O  \\nATOM   1037  CG  MET A 129     -37.828  16.344   2.112  1.00  0.56           C  \\nATOM   1038  SD  MET A 129     -37.508  15.436   3.674  1.00  0.55           S  \\nATOM   1039  CE  MET A 129     -35.697  15.524   3.730  1.00  0.47           C  \\nATOM   1040  N   PRO A 130     -40.975  19.501   3.845  1.00  0.62           N  \\nATOM   1041  CA  PRO A 130     -41.283  20.629   4.727  1.00  0.61           C  \\nATOM   1042  C   PRO A 130     -40.032  21.274   5.319  1.00  0.61           C  \\nATOM   1043  CB  PRO A 130     -42.142  19.995   5.824  1.00  0.57           C  \\nATOM   1044  O   PRO A 130     -39.176  20.577   5.871  1.00  0.59           O  \\nATOM   1045  CG  PRO A 130     -41.702  18.567   5.876  1.00  0.55           C  \\nATOM   1046  CD  PRO A 130     -41.165  18.184   4.527  1.00  0.56           C  \\nATOM   1047  N   SER A 131     -39.570  22.483   4.876  1.00  0.62           N  \\nATOM   1048  CA  SER A 131     -38.798  23.535   5.530  1.00  0.62           C  \\nATOM   1049  C   SER A 131     -39.110  23.604   7.021  1.00  0.61           C  \\nATOM   1050  CB  SER A 131     -39.077  24.890   4.879  1.00  0.56           C  \\nATOM   1051  O   SER A 131     -40.130  24.168   7.422  1.00  0.58           O  \\nATOM   1052  OG  SER A 131     -40.166  24.798   3.976  1.00  0.57           O  \\nATOM   1053  N   GLY A 132     -38.800  22.562   7.736  1.00  0.59           N  \\nATOM   1054  CA  GLY A 132     -38.781  22.705   9.183  1.00  0.59           C  \\nATOM   1055  C   GLY A 132     -37.421  23.098   9.727  1.00  0.58           C  \\nATOM   1056  O   GLY A 132     -36.392  22.781   9.127  1.00  0.55           O  \\nATOM   1057  N   PRO A 133     -37.348  24.247  10.482  1.00  0.61           N  \\nATOM   1058  CA  PRO A 133     -36.146  24.673  11.202  1.00  0.60           C  \\nATOM   1059  C   PRO A 133     -35.239  23.504  11.580  1.00  0.61           C  \\nATOM   1060  CB  PRO A 133     -36.707  25.355  12.452  1.00  0.56           C  \\nATOM   1061  O   PRO A 133     -35.727  22.413  11.886  1.00  0.58           O  \\nATOM   1062  CG  PRO A 133     -38.034  24.706  12.678  1.00  0.54           C  \\nATOM   1063  CD  PRO A 133     -38.581  24.263  11.351  1.00  0.56           C  \\nATOM   1064  N   PRO A 134     -34.033  23.416  11.043  1.00  0.58           N  \\nATOM   1065  CA  PRO A 134     -33.050  22.452  11.544  1.00  0.58           C  \\nATOM   1066  C   PRO A 134     -33.266  22.099  13.014  1.00  0.58           C  \\nATOM   1067  CB  PRO A 134     -31.715  23.174  11.345  1.00  0.54           C  \\nATOM   1068  O   PRO A 134     -33.598  22.973  13.820  1.00  0.56           O  \\nATOM   1069  CG  PRO A 134     -32.058  24.628  11.383  1.00  0.52           C  \\nATOM   1070  CD  PRO A 134     -33.474  24.797  10.911  1.00  0.54           C  \\nATOM   1071  N   PRO A 135     -33.496  20.928  13.373  1.00  0.58           N  \\nATOM   1072  CA  PRO A 135     -33.553  20.533  14.782  1.00  0.57           C  \\nATOM   1073  C   PRO A 135     -32.371  21.062  15.592  1.00  0.58           C  \\nATOM   1074  CB  PRO A 135     -33.530  19.003  14.721  1.00  0.53           C  \\nATOM   1075  O   PRO A 135     -31.263  21.186  15.064  1.00  0.56           O  \\nATOM   1076  CG  PRO A 135     -32.895  18.683  13.406  1.00  0.51           C  \\nATOM   1077  CD  PRO A 135     -33.088  19.854  12.486  1.00  0.53           C  \\nATOM   1078  N   PRO A 136     -32.578  21.910  16.754  1.00  0.56           N  \\nATOM   1079  CA  PRO A 136     -31.646  22.371  17.786  1.00  0.55           C  \\nATOM   1080  C   PRO A 136     -30.528  21.368  18.061  1.00  0.55           C  \\nATOM   1081  CB  PRO A 136     -32.540  22.543  19.017  1.00  0.51           C  \\nATOM   1082  O   PRO A 136     -30.798  20.198  18.345  1.00  0.52           O  \\nATOM   1083  CG  PRO A 136     -33.743  21.702  18.736  1.00  0.49           C  \\nATOM   1084  CD  PRO A 136     -33.792  21.416  17.262  1.00  0.52           C  \\nATOM   1085  N   SER A 137     -29.350  21.370  17.388  1.00  0.56           N  \\nATOM   1086  CA  SER A 137     -27.984  21.057  17.796  1.00  0.57           C  \\nATOM   1087  C   SER A 137     -27.955  20.400  19.172  1.00  0.55           C  \\nATOM   1088  CB  SER A 137     -27.124  22.321  17.807  1.00  0.51           C  \\nATOM   1089  O   SER A 137     -28.285  21.035  20.175  1.00  0.53           O  \\nATOM   1090  OG  SER A 137     -27.226  23.010  16.572  1.00  0.52           O  \\nATOM   1091  N   MET A 138     -28.383  19.249  19.280  1.00  0.55           N  \\nATOM   1092  CA  MET A 138     -28.113  18.480  20.492  1.00  0.56           C  \\nATOM   1093  C   MET A 138     -26.623  18.484  20.818  1.00  0.54           C  \\nATOM   1094  CB  MET A 138     -28.611  17.041  20.339  1.00  0.49           C  \\nATOM   1095  O   MET A 138     -25.814  17.957  20.051  1.00  0.51           O  \\nATOM   1096  CG  MET A 138     -30.126  16.916  20.323  1.00  0.47           C  \\nATOM   1097  SD  MET A 138     -30.706  15.321  21.022  1.00  0.53           S  \\nATOM   1098  CE  MET A 138     -32.419  15.315  20.425  1.00  0.42           C  \\nATOM   1099  N   ALA A 139     -26.072  19.504  21.490  1.00  0.56           N  \\nATOM   1100  CA  ALA A 139     -25.014  19.672  22.483  1.00  0.57           C  \\nATOM   1101  C   ALA A 139     -24.686  18.346  23.165  1.00  0.56           C  \\nATOM   1102  CB  ALA A 139     -25.420  20.714  23.522  1.00  0.52           C  \\nATOM   1103  O   ALA A 139     -25.444  17.875  24.016  1.00  0.54           O  \\nATOM   1104  N   TYR A 140     -24.119  17.354  22.479  1.00  0.56           N  \\nATOM   1105  CA  TYR A 140     -23.419  16.328  23.243  1.00  0.58           C  \\nATOM   1106  C   TYR A 140     -22.102  16.862  23.794  1.00  0.55           C  \\nATOM   1107  CB  TYR A 140     -23.158  15.094  22.374  1.00  0.51           C  \\nATOM   1108  O   TYR A 140     -21.240  17.311  23.035  1.00  0.52           O  \\nATOM   1109  CG  TYR A 140     -24.349  14.175  22.251  1.00  0.50           C  \\nATOM   1110  CD1 TYR A 140     -24.599  13.195  23.210  1.00  0.47           C  \\nATOM   1111  CD2 TYR A 140     -25.226  14.282  21.177  1.00  0.48           C  \\nATOM   1112  CE1 TYR A 140     -25.694  12.345  23.101  1.00  0.45           C  \\nATOM   1113  CE2 TYR A 140     -26.324  13.437  21.058  1.00  0.47           C  \\nATOM   1114  OH  TYR A 140     -27.636  11.634  21.910  1.00  0.40           O  \\nATOM   1115  CZ  TYR A 140     -26.550  12.473  22.023  1.00  0.41           C  \\nATOM   1116  N   GLY A 141     -22.118  17.402  25.057  1.00  0.55           N  \\nATOM   1117  CA  GLY A 141     -21.154  17.579  26.131  1.00  0.55           C  \\nATOM   1118  C   GLY A 141     -19.882  16.778  25.928  1.00  0.55           C  \\nATOM   1119  O   GLY A 141     -19.900  15.547  26.000  1.00  0.53           O  \\nATOM   1120  N   MET A 142     -18.880  17.137  25.128  1.00  0.56           N  \\nATOM   1121  CA  MET A 142     -17.485  16.810  25.409  1.00  0.58           C  \\nATOM   1122  C   MET A 142     -17.197  16.889  26.904  1.00  0.56           C  \\nATOM   1123  CB  MET A 142     -16.548  17.749  24.646  1.00  0.53           C  \\nATOM   1124  O   MET A 142     -17.542  17.877  27.556  1.00  0.55           O  \\nATOM   1125  CG  MET A 142     -16.435  17.431  23.164  1.00  0.52           C  \\nATOM   1126  SD  MET A 142     -14.995  18.252  22.376  1.00  0.54           S  \\nATOM   1127  CE  MET A 142     -14.160  16.812  21.656  1.00  0.46           C  \\nATOM   1128  N   PRO A 143     -17.124  15.712  27.465  1.00  0.59           N  \\nATOM   1129  CA  PRO A 143     -16.530  15.828  28.799  1.00  0.58           C  \\nATOM   1130  C   PRO A 143     -15.313  16.751  28.825  1.00  0.59           C  \\nATOM   1131  CB  PRO A 143     -16.130  14.389  29.133  1.00  0.55           C  \\nATOM   1132  O   PRO A 143     -14.580  16.839  27.837  1.00  0.56           O  \\nATOM   1133  CG  PRO A 143     -15.929  13.730  27.806  1.00  0.53           C  \\nATOM   1134  CD  PRO A 143     -16.759  14.453  26.785  1.00  0.55           C  \\nATOM   1135  N   PRO A 144     -15.434  17.782  29.666  1.00  0.59           N  \\nATOM   1136  CA  PRO A 144     -14.327  18.710  29.907  1.00  0.58           C  \\nATOM   1137  C   PRO A 144     -12.967  18.017  29.915  1.00  0.59           C  \\nATOM   1138  CB  PRO A 144     -14.650  19.294  31.285  1.00  0.54           C  \\nATOM   1139  O   PRO A 144     -12.777  17.029  30.630  1.00  0.57           O  \\nATOM   1140  CG  PRO A 144     -15.446  18.232  31.973  1.00  0.52           C  \\nATOM   1141  CD  PRO A 144     -16.159  17.420  30.930  1.00  0.54           C  \\nATOM   1142  N   MET A 145     -12.119  18.067  28.997  1.00  0.61           N  \\nATOM   1143  CA  MET A 145     -10.680  17.826  29.049  1.00  0.63           C  \\nATOM   1144  C   MET A 145     -10.091  18.328  30.362  1.00  0.61           C  \\nATOM   1145  CB  MET A 145      -9.979  18.502  27.869  1.00  0.58           C  \\nATOM   1146  O   MET A 145     -10.432  19.418  30.825  1.00  0.59           O  \\nATOM   1147  CG  MET A 145     -10.215  17.809  26.537  1.00  0.56           C  \\nATOM   1148  SD  MET A 145      -8.807  16.743  26.040  1.00  0.55           S  \\nATOM   1149  CE  MET A 145      -7.945  17.860  24.900  1.00  0.47           C  \\nATOM   1150  N   PRO A 146      -9.839  17.436  31.219  1.00  0.58           N  \\nATOM   1151  CA  PRO A 146      -9.122  17.800  32.443  1.00  0.58           C  \\nATOM   1152  C   PRO A 146      -8.104  18.917  32.221  1.00  0.58           C  \\nATOM   1153  CB  PRO A 146      -8.424  16.497  32.842  1.00  0.54           C  \\nATOM   1154  O   PRO A 146      -7.331  18.870  31.261  1.00  0.56           O  \\nATOM   1155  CG  PRO A 146      -8.339  15.706  31.577  1.00  0.53           C  \\nATOM   1156  CD  PRO A 146      -9.359  16.239  30.612  1.00  0.55           C  \\nATOM   1157  N   SER A 147      -8.320  20.176  32.735  1.00  0.59           N  \\nATOM   1158  CA  SER A 147      -7.470  21.320  33.046  1.00  0.59           C  \\nATOM   1159  C   SER A 147      -6.090  20.871  33.515  1.00  0.58           C  \\nATOM   1160  CB  SER A 147      -8.121  22.198  34.115  1.00  0.54           C  \\nATOM   1161  O   SER A 147      -5.946  20.344  34.620  1.00  0.55           O  \\nATOM   1162  OG  SER A 147      -9.455  22.516  33.760  1.00  0.57           O  \\nATOM   1163  N   GLY A 148      -5.145  20.458  32.557  1.00  0.56           N  \\nATOM   1164  CA  GLY A 148      -3.736  20.484  32.915  1.00  0.56           C  \\nATOM   1165  C   GLY A 148      -2.839  19.872  31.854  1.00  0.55           C  \\nATOM   1166  O   GLY A 148      -1.651  19.649  32.092  1.00  0.52           O  \\nATOM   1167  N   MET A 149      -3.210  19.911  30.584  1.00  0.57           N  \\nATOM   1168  CA  MET A 149      -2.117  19.693  29.641  1.00  0.58           C  \\nATOM   1169  C   MET A 149      -2.260  20.599  28.423  1.00  0.56           C  \\nATOM   1170  CB  MET A 149      -2.069  18.229  29.201  1.00  0.51           C  \\nATOM   1171  O   MET A 149      -3.128  20.377  27.577  1.00  0.53           O  \\nATOM   1172  CG  MET A 149      -1.144  17.365  30.043  1.00  0.49           C  \\nATOM   1173  SD  MET A 149      -1.263  15.584  29.617  1.00  0.53           S  \\nATOM   1174  CE  MET A 149       0.472  15.091  29.816  1.00  0.43           C  \\nATOM   1175  N   MET A 150      -1.913  21.905  28.535  1.00  0.56           N  \\nATOM   1176  CA  MET A 150      -1.523  22.813  27.460  1.00  0.57           C  \\nATOM   1177  C   MET A 150      -0.886  22.047  26.305  1.00  0.56           C  \\nATOM   1178  CB  MET A 150      -0.554  23.877  27.979  1.00  0.52           C  \\nATOM   1179  O   MET A 150       0.063  21.287  26.507  1.00  0.54           O  \\nATOM   1180  CG  MET A 150      -1.233  25.005  28.739  1.00  0.51           C  \\nATOM   1181  SD  MET A 150      -0.158  26.483  28.912  1.00  0.52           S  \\nATOM   1182  CE  MET A 150      -1.299  27.775  28.346  1.00  0.43           C  \\nATOM   1183  N   PRO A 151      -1.602  21.764  25.281  1.00  0.58           N  \\nATOM   1184  CA  PRO A 151      -0.922  21.382  24.041  1.00  0.58           C  \\nATOM   1185  C   PRO A 151       0.220  22.328  23.678  1.00  0.58           C  \\nATOM   1186  CB  PRO A 151      -2.037  21.445  22.993  1.00  0.54           C  \\nATOM   1187  O   PRO A 151       0.113  23.539  23.887  1.00  0.56           O  \\nATOM   1188  CG  PRO A 151      -2.994  22.467  23.516  1.00  0.52           C  \\nATOM   1189  CD  PRO A 151      -2.904  22.482  25.015  1.00  0.54           C  \\nATOM   1190  N   PRO A 152       1.377  21.875  23.753  1.00  0.57           N  \\nATOM   1191  CA  PRO A 152       2.507  22.701  23.322  1.00  0.56           C  \\nATOM   1192  C   PRO A 152       2.163  23.597  22.134  1.00  0.57           C  \\nATOM   1193  CB  PRO A 152       3.571  21.669  22.937  1.00  0.52           C  \\nATOM   1194  O   PRO A 152       1.550  23.139  21.167  1.00  0.55           O  \\nATOM   1195  CG  PRO A 152       2.806  20.414  22.667  1.00  0.51           C  \\nATOM   1196  CD  PRO A 152       1.469  20.515  23.344  1.00  0.53           C  \\nATOM   1197  N   ARG A 153       2.021  24.880  22.303  1.00  0.58           N  \\nATOM   1198  CA  ARG A 153       2.160  26.017  21.399  1.00  0.58           C  \\nATOM   1199  C   ARG A 153       3.149  25.709  20.280  1.00  0.57           C  \\nATOM   1200  CB  ARG A 153       2.607  27.263  22.167  1.00  0.54           C  \\nATOM   1201  O   ARG A 153       4.359  25.657  20.511  1.00  0.55           O  \\nATOM   1202  CG  ARG A 153       1.463  28.169  22.591  1.00  0.53           C  \\nATOM   1203  CD  ARG A 153       1.965  29.412  23.313  1.00  0.55           C  \\nATOM   1204  NE  ARG A 153       0.876  30.134  23.963  1.00  0.45           N  \\nATOM   1205  NH1 ARG A 153       1.925  32.192  23.897  1.00  0.34           N  \\nATOM   1206  NH2 ARG A 153      -0.163  31.994  24.821  1.00  0.29           N  \\nATOM   1207  CZ  ARG A 153       0.882  31.438  24.226  1.00  0.48           C  \\nATOM   1208  N   GLY A 154       2.737  24.940  19.098  1.00  0.53           N  \\nATOM   1209  CA  GLY A 154       3.569  24.998  17.907  1.00  0.54           C  \\nATOM   1210  C   GLY A 154       3.316  23.853  16.946  1.00  0.53           C  \\nATOM   1211  O   GLY A 154       3.879  22.768  17.102  1.00  0.50           O  \\nATOM   1212  N   MET A 155       2.017  23.658  16.508  1.00  0.55           N  \\nATOM   1213  CA  MET A 155       1.837  23.146  15.152  1.00  0.57           C  \\nATOM   1214  C   MET A 155       0.690  23.864  14.447  1.00  0.55           C  \\nATOM   1215  CB  MET A 155       1.575  21.639  15.176  1.00  0.50           C  \\nATOM   1216  O   MET A 155      -0.442  23.860  14.933  1.00  0.53           O  \\nATOM   1217  CG  MET A 155       2.789  20.812  15.564  1.00  0.49           C  \\nATOM   1218  SD  MET A 155       2.739  19.117  14.863  1.00  0.52           S  \\nATOM   1219  CE  MET A 155       4.277  18.441  15.547  1.00  0.43           C  \\nATOM   1220  N   PRO A 156       0.903  24.811  13.629  1.00  0.55           N  \\nATOM   1221  CA  PRO A 156       0.079  25.478  12.619  1.00  0.55           C  \\nATOM   1222  C   PRO A 156      -0.575  24.496  11.649  1.00  0.54           C  \\nATOM   1223  CB  PRO A 156       1.076  26.382  11.891  1.00  0.49           C  \\nATOM   1224  O   PRO A 156       0.122  23.780  10.926  1.00  0.51           O  \\nATOM   1225  CG  PRO A 156       2.419  25.819  12.227  1.00  0.48           C  \\nATOM   1226  CD  PRO A 156       2.267  24.893  13.400  1.00  0.52           C  \\nATOM   1227  N   GLY A 157      -1.815  23.733  12.007  1.00  0.52           N  \\nATOM   1228  CA  GLY A 157      -2.762  23.365  10.967  1.00  0.53           C  \\nATOM   1229  C   GLY A 157      -3.493  22.068  11.259  1.00  0.51           C  \\nATOM   1230  O   GLY A 157      -2.950  20.981  11.046  1.00  0.49           O  \\nATOM   1231  N   ALA A 158      -4.300  22.074  12.263  1.00  0.53           N  \\nATOM   1232  CA  ALA A 158      -5.440  21.166  12.354  1.00  0.54           C  \\nATOM   1233  C   ALA A 158      -6.751  21.905  12.098  1.00  0.53           C  \\nATOM   1234  CB  ALA A 158      -5.474  20.489  13.722  1.00  0.48           C  \\nATOM   1235  O   ALA A 158      -7.177  22.727  12.913  1.00  0.51           O  \\nATOM   1236  N   TYR A 159      -7.073  22.195  10.778  1.00  0.55           N  \\nATOM   1237  CA  TYR A 159      -8.399  22.590  10.318  1.00  0.56           C  \\nATOM   1238  C   TYR A 159      -9.464  21.631  10.836  1.00  0.55           C  \\nATOM   1239  CB  TYR A 159      -8.445  22.642   8.788  1.00  0.51           C  \\nATOM   1240  O   TYR A 159      -9.328  20.412  10.705  1.00  0.53           O  \\nATOM   1241  CG  TYR A 159      -7.997  23.964   8.213  1.00  0.50           C  \\nATOM   1242  CD1 TYR A 159      -8.892  25.022   8.069  1.00  0.47           C  \\nATOM   1243  CD2 TYR A 159      -6.680  24.158   7.811  1.00  0.48           C  \\nATOM   1244  CE1 TYR A 159      -8.485  26.241   7.537  1.00  0.46           C  \\nATOM   1245  CE2 TYR A 159      -6.261  25.372   7.278  1.00  0.48           C  \\nATOM   1246  OH  TYR A 159      -6.761  27.612   6.619  1.00  0.44           O  \\nATOM   1247  CZ  TYR A 159      -7.169  26.407   7.146  1.00  0.43           C  \\nATOM   1248  N   PRO A 160     -10.351  22.103  11.762  1.00  0.54           N  \\nATOM   1249  CA  PRO A 160     -11.607  21.424  12.091  1.00  0.53           C  \\nATOM   1250  C   PRO A 160     -12.431  21.076  10.853  1.00  0.54           C  \\nATOM   1251  CB  PRO A 160     -12.340  22.443  12.966  1.00  0.50           C  \\nATOM   1252  O   PRO A 160     -12.465  21.849   9.892  1.00  0.52           O  \\nATOM   1253  CG  PRO A 160     -11.825  23.772  12.518  1.00  0.48           C  \\nATOM   1254  CD  PRO A 160     -10.424  23.594  12.008  1.00  0.51           C  \\nATOM   1255  N   PRO A 161     -12.791  19.853  10.488  1.00  0.52           N  \\nATOM   1256  CA  PRO A 161     -13.719  19.473   9.420  1.00  0.52           C  \\nATOM   1257  C   PRO A 161     -14.911  20.420   9.310  1.00  0.53           C  \\nATOM   1258  CB  PRO A 161     -14.172  18.070   9.830  1.00  0.48           C  \\nATOM   1259  O   PRO A 161     -15.395  20.930  10.324  1.00  0.51           O  \\nATOM   1260  CG  PRO A 161     -14.027  18.040  11.317  1.00  0.47           C  \\nATOM   1261  CD  PRO A 161     -12.937  18.996  11.709  1.00  0.50           C  \\nATOM   1262  N   PRO A 162     -15.148  21.157   8.135  1.00  0.52           N  \\nATOM   1263  CA  PRO A 162     -16.356  21.936   7.853  1.00  0.52           C  \\nATOM   1264  C   PRO A 162     -17.615  21.308   8.444  1.00  0.52           C  \\nATOM   1265  CB  PRO A 162     -16.418  21.948   6.323  1.00  0.48           C  \\nATOM   1266  O   PRO A 162     -17.793  20.089   8.374  1.00  0.50           O  \\nATOM   1267  CG  PRO A 162     -15.495  20.853   5.895  1.00  0.46           C  \\nATOM   1268  CD  PRO A 162     -14.639  20.465   7.066  1.00  0.48           C  \\nATOM   1269  N   ARG A 163     -18.229  21.812   9.515  1.00  0.50           N  \\nATOM   1270  CA  ARG A 163     -19.591  21.769  10.039  1.00  0.50           C  \\nATOM   1271  C   ARG A 163     -20.586  21.392   8.947  1.00  0.49           C  \\nATOM   1272  CB  ARG A 163     -19.973  23.117  10.654  1.00  0.46           C  \\nATOM   1273  O   ARG A 163     -20.810  22.162   8.011  1.00  0.47           O  \\nATOM   1274  CG  ARG A 163     -19.414  23.337  12.050  1.00  0.45           C  \\nATOM   1275  CD  ARG A 163     -19.883  24.659  12.643  1.00  0.48           C  \\nATOM   1276  NE  ARG A 163     -19.107  25.025  13.825  1.00  0.43           N  \\nATOM   1277  NH1 ARG A 163     -20.797  26.255  14.808  1.00  0.33           N  \\nATOM   1278  NH2 ARG A 163     -18.769  26.046  15.854  1.00  0.28           N  \\nATOM   1279  CZ  ARG A 163     -19.559  25.774  14.826  1.00  0.42           C  \\nATOM   1280  N   GLY A 164     -20.726  20.193   8.494  1.00  0.49           N  \\nATOM   1281  CA  GLY A 164     -21.865  19.599   7.813  1.00  0.49           C  \\nATOM   1282  C   GLY A 164     -21.526  18.302   7.103  1.00  0.48           C  \\nATOM   1283  O   GLY A 164     -22.358  17.747   6.382  1.00  0.46           O  \\nATOM   1284  N   TYR A 165     -20.373  17.610   7.553  1.00  0.45           N  \\nATOM   1285  CA  TYR A 165     -20.113  16.182   7.414  1.00  0.46           C  \\nATOM   1286  C   TYR A 165     -20.093  15.496   8.775  1.00  0.45           C  \\nATOM   1287  CB  TYR A 165     -18.784  15.945   6.690  1.00  0.42           C  \\nATOM   1288  O   TYR A 165     -19.551  16.038   9.742  1.00  0.43           O  \\nATOM   1289  CG  TYR A 165     -18.901  15.962   5.186  1.00  0.41           C  \\nATOM   1290  CD1 TYR A 165     -19.192  14.799   4.476  1.00  0.40           C  \\nATOM   1291  CD2 TYR A 165     -18.719  17.141   4.470  1.00  0.41           C  \\nATOM   1292  CE1 TYR A 165     -19.299  14.810   3.089  1.00  0.41           C  \\nATOM   1293  CE2 TYR A 165     -18.823  17.164   3.083  1.00  0.42           C  \\nATOM   1294  OH  TYR A 165     -19.218  16.012   1.030  1.00  0.38           O  \\nATOM   1295  CZ  TYR A 165     -19.113  15.996   2.403  1.00  0.37           C  \\nATOM   1296  N   PRO A 166     -21.047  14.785   9.177  1.00  0.48           N  \\nATOM   1297  CA  PRO A 166     -20.774  13.958  10.355  1.00  0.48           C  \\nATOM   1298  C   PRO A 166     -19.283  13.846  10.663  1.00  0.49           C  \\nATOM   1299  CB  PRO A 166     -21.357  12.596   9.971  1.00  0.45           C  \\nATOM   1300  O   PRO A 166     -18.456  13.860   9.748  1.00  0.47           O  \\nATOM   1301  CG  PRO A 166     -21.331  12.576   8.477  1.00  0.43           C  \\nATOM   1302  CD  PRO A 166     -21.353  13.993   7.981  1.00  0.46           C  \\nATOM   1303  N   PRO A 167     -18.803  14.462  11.794  1.00  0.47           N  \\nATOM   1304  CA  PRO A 167     -17.417  14.228  12.209  1.00  0.46           C  \\nATOM   1305  C   PRO A 167     -16.779  13.041  11.491  1.00  0.47           C  \\nATOM   1306  CB  PRO A 167     -17.540  13.957  13.710  1.00  0.44           C  \\nATOM   1307  O   PRO A 167     -17.429  12.010  11.297  1.00  0.45           O  \\nATOM   1308  CG  PRO A 167     -18.943  13.477  13.899  1.00  0.41           C  \\nATOM   1309  CD  PRO A 167     -19.785  14.015  12.778  1.00  0.43           C  \\nATOM   1310  N   ALA A 168     -15.861  13.171  10.499  1.00  0.46           N  \\nATOM   1311  CA  ALA A 168     -14.973  12.062  10.159  1.00  0.46           C  \\nATOM   1312  C   ALA A 168     -14.881  11.061  11.306  1.00  0.46           C  \\nATOM   1313  CB  ALA A 168     -13.584  12.585   9.800  1.00  0.44           C  \\nATOM   1314  O   ALA A 168     -14.880  11.447  12.477  1.00  0.44           O  \\nATOM   1315  N   PRO A 169     -15.451   9.914  11.333  1.00  0.48           N  \\nATOM   1316  CA  PRO A 169     -15.144   8.924  12.368  1.00  0.47           C  \\nATOM   1317  C   PRO A 169     -13.769   9.137  12.998  1.00  0.48           C  \\nATOM   1318  CB  PRO A 169     -15.197   7.595  11.610  1.00  0.44           C  \\nATOM   1319  O   PRO A 169     -12.821   9.517  12.307  1.00  0.46           O  \\nATOM   1320  CG  PRO A 169     -14.963   7.960  10.180  1.00  0.43           C  \\nATOM   1321  CD  PRO A 169     -15.338   9.402   9.989  1.00  0.45           C  \\nATOM   1322  N   ALA A 170     -13.557   9.805  14.305  1.00  0.50           N  \\nATOM   1323  CA  ALA A 170     -12.339   9.853  15.109  1.00  0.51           C  \\nATOM   1324  C   ALA A 170     -11.245   8.980  14.499  1.00  0.50           C  \\nATOM   1325  CB  ALA A 170     -12.629   9.413  16.541  1.00  0.48           C  \\nATOM   1326  O   ALA A 170     -11.518   7.879  14.017  1.00  0.49           O  \\nATOM   1327  N   PRO A 171     -10.155   9.381  13.841  1.00  0.51           N  \\nATOM   1328  CA  PRO A 171      -9.030   8.506  13.502  1.00  0.51           C  \\nATOM   1329  C   PRO A 171      -8.923   7.297  14.429  1.00  0.51           C  \\nATOM   1330  CB  PRO A 171      -7.812   9.421  13.656  1.00  0.48           C  \\nATOM   1331  O   PRO A 171      -8.693   7.454  15.631  1.00  0.49           O  \\nATOM   1332  CG  PRO A 171      -8.322  10.613  14.399  1.00  0.47           C  \\nATOM   1333  CD  PRO A 171      -9.823  10.570  14.398  1.00  0.50           C  \\nATOM   1334  N   GLY A 172      -9.912   6.302  14.245  1.00  0.47           N  \\nATOM   1335  CA  GLY A 172      -9.931   4.984  14.860  1.00  0.48           C  \\nATOM   1336  C   GLY A 172     -11.313   4.358  14.885  1.00  0.47           C  \\nATOM   1337  O   GLY A 172     -11.503   3.286  15.462  1.00  0.44           O  \\nATOM   1338  N   VAL A 173     -12.275   4.798  14.048  1.00  0.50           N  \\nATOM   1339  CA  VAL A 173     -13.457   4.000  13.739  1.00  0.50           C  \\nATOM   1340  C   VAL A 173     -13.561   3.793  12.229  1.00  0.50           C  \\nATOM   1341  CB  VAL A 173     -14.746   4.663  14.275  1.00  0.47           C  \\nATOM   1342  O   VAL A 173     -14.014   4.682  11.504  1.00  0.48           O  \\nATOM   1343  CG1 VAL A 173     -15.949   3.743  14.076  1.00  0.43           C  \\nATOM   1344  CG2 VAL A 173     -14.584   5.025  15.751  1.00  0.44           C  \\nATOM   1345  N   TYR A 174     -12.676   3.051  11.561  1.00  0.55           N  \\nATOM   1346  CA  TYR A 174     -12.944   2.395  10.286  1.00  0.56           C  \\nATOM   1347  C   TYR A 174     -14.374   1.869  10.233  1.00  0.55           C  \\nATOM   1348  CB  TYR A 174     -11.957   1.248  10.054  1.00  0.51           C  \\nATOM   1349  O   TYR A 174     -14.727   0.940  10.963  1.00  0.53           O  \\nATOM   1350  CG  TYR A 174     -10.923   1.541   8.994  1.00  0.49           C  \\nATOM   1351  CD1 TYR A 174     -11.164   1.240   7.656  1.00  0.46           C  \\nATOM   1352  CD2 TYR A 174      -9.703   2.120   9.329  1.00  0.47           C  \\nATOM   1353  CE1 TYR A 174     -10.214   1.507   6.676  1.00  0.45           C  \\nATOM   1354  CE2 TYR A 174      -8.745   2.391   8.357  1.00  0.48           C  \\nATOM   1355  OH  TYR A 174      -8.065   2.349   6.070  1.00  0.40           O  \\nATOM   1356  CZ  TYR A 174      -9.009   2.082   7.036  1.00  0.41           C  \\nATOM   1357  N   MET A 175     -15.371   2.660   9.910  1.00  0.52           N  \\nATOM   1358  CA  MET A 175     -16.585   2.087   9.336  1.00  0.53           C  \\nATOM   1359  C   MET A 175     -16.260   1.245   8.106  1.00  0.52           C  \\nATOM   1360  CB  MET A 175     -17.579   3.189   8.968  1.00  0.49           C  \\nATOM   1361  O   MET A 175     -15.640   1.734   7.161  1.00  0.51           O  \\nATOM   1362  CG  MET A 175     -18.276   3.812  10.167  1.00  0.48           C  \\nATOM   1363  SD  MET A 175     -19.795   4.729   9.700  1.00  0.52           S  \\nATOM   1364  CE  MET A 175     -20.999   3.372   9.696  1.00  0.44           C  \\nATOM   1365  N   PRO A 176     -16.195  -0.029   8.165  1.00  0.54           N  \\nATOM   1366  CA  PRO A 176     -16.190  -0.813   6.928  1.00  0.53           C  \\nATOM   1367  C   PRO A 176     -17.214  -0.315   5.910  1.00  0.54           C  \\nATOM   1368  CB  PRO A 176     -16.537  -2.226   7.405  1.00  0.50           C  \\nATOM   1369  O   PRO A 176     -18.272   0.192   6.290  1.00  0.53           O  \\nATOM   1370  CG  PRO A 176     -17.339  -2.023   8.650  1.00  0.49           C  \\nATOM   1371  CD  PRO A 176     -16.900  -0.740   9.296  1.00  0.51           C  \\nATOM   1372  N   PRO A 177     -16.902   0.165   4.672  1.00  0.54           N  \\nATOM   1373  CA  PRO A 177     -17.897   0.390   3.621  1.00  0.53           C  \\nATOM   1374  C   PRO A 177     -19.091  -0.555   3.726  1.00  0.54           C  \\nATOM   1375  CB  PRO A 177     -17.111   0.134   2.332  1.00  0.50           C  \\nATOM   1376  O   PRO A 177     -18.935  -1.711   4.129  1.00  0.52           O  \\nATOM   1377  CG  PRO A 177     -15.886  -0.602   2.771  1.00  0.48           C  \\nATOM   1378  CD  PRO A 177     -15.741  -0.443   4.257  1.00  0.49           C  \\nATOM   1379  N   PRO A 178     -20.393  -0.200   3.899  1.00  0.50           N  \\nATOM   1380  CA  PRO A 178     -21.567  -1.068   3.776  1.00  0.49           C  \\nATOM   1381  C   PRO A 178     -21.373  -2.180   2.748  1.00  0.50           C  \\nATOM   1382  CB  PRO A 178     -22.672  -0.104   3.336  1.00  0.47           C  \\nATOM   1383  O   PRO A 178     -21.230  -1.905   1.554  1.00  0.49           O  \\nATOM   1384  CG  PRO A 178     -21.952   1.134   2.910  1.00  0.46           C  \\nATOM   1385  CD  PRO A 178     -20.523   1.035   3.362  1.00  0.49           C  \\nATOM   1386  N   GLY A 179     -20.833  -3.316   3.073  1.00  0.51           N  \\nATOM   1387  CA  GLY A 179     -20.875  -4.561   2.322  1.00  0.51           C  \\nATOM   1388  C   GLY A 179     -19.772  -5.528   2.708  1.00  0.51           C  \\nATOM   1389  O   GLY A 179     -19.529  -6.513   2.007  1.00  0.48           O  \\nATOM   1390  N   MET A 180     -19.028  -5.300   3.866  1.00  0.51           N  \\nATOM   1391  CA  MET A 180     -18.287  -6.381   4.512  1.00  0.53           C  \\nATOM   1392  C   MET A 180     -19.150  -7.085   5.554  1.00  0.51           C  \\nATOM   1393  CB  MET A 180     -17.012  -5.845   5.164  1.00  0.47           C  \\nATOM   1394  O   MET A 180     -19.769  -6.433   6.396  1.00  0.49           O  \\nATOM   1395  CG  MET A 180     -15.924  -5.474   4.170  1.00  0.46           C  \\nATOM   1396  SD  MET A 180     -14.241  -5.857   4.794  1.00  0.53           S  \\nATOM   1397  CE  MET A 180     -13.236  -4.992   3.555  1.00  0.42           C  \\nATOM   1398  N   PRO A 181     -19.759  -8.277   5.391  1.00  0.50           N  \\nATOM   1399  CA  PRO A 181     -20.280  -9.169   6.429  1.00  0.50           C  \\nATOM   1400  C   PRO A 181     -19.403  -9.198   7.678  1.00  0.50           C  \\nATOM   1401  CB  PRO A 181     -20.296 -10.537   5.742  1.00  0.46           C  \\nATOM   1402  O   PRO A 181     -18.276  -9.697   7.634  1.00  0.48           O  \\nATOM   1403  CG  PRO A 181     -19.325 -10.409   4.613  1.00  0.45           C  \\nATOM   1404  CD  PRO A 181     -19.093  -8.949   4.347  1.00  0.48           C  \\nATOM   1405  N   GLY A 182     -19.310  -8.245   8.482  1.00  0.48           N  \\nATOM   1406  CA  GLY A 182     -19.205  -8.013   9.914  1.00  0.49           C  \\nATOM   1407  C   GLY A 182     -19.546  -6.591  10.316  1.00  0.48           C  \\nATOM   1408  O   GLY A 182     -18.699  -5.699  10.240  1.00  0.47           O  \\nATOM   1409  N   ALA A 183     -20.592  -6.115   9.716  1.00  0.46           N  \\nATOM   1410  CA  ALA A 183     -21.325  -4.960  10.229  1.00  0.47           C  \\nATOM   1411  C   ALA A 183     -21.901  -5.248  11.613  1.00  0.46           C  \\nATOM   1412  CB  ALA A 183     -22.439  -4.564   9.264  1.00  0.43           C  \\nATOM   1413  O   ALA A 183     -22.629  -6.227  11.798  1.00  0.44           O  \\nATOM   1414  N   TYR A 184     -21.114  -5.205  12.715  1.00  0.50           N  \\nATOM   1415  CA  TYR A 184     -21.838  -4.886  13.941  1.00  0.51           C  \\nATOM   1416  C   TYR A 184     -23.198  -4.274  13.628  1.00  0.50           C  \\nATOM   1417  CB  TYR A 184     -21.021  -3.927  14.813  1.00  0.47           C  \\nATOM   1418  O   TYR A 184     -23.297  -3.346  12.822  1.00  0.48           O  \\nATOM   1419  CG  TYR A 184     -20.037  -4.622  15.722  1.00  0.47           C  \\nATOM   1420  CD1 TYR A 184     -20.428  -5.098  16.971  1.00  0.44           C  \\nATOM   1421  CD2 TYR A 184     -18.713  -4.802  15.334  1.00  0.45           C  \\nATOM   1422  CE1 TYR A 184     -19.524  -5.737  17.813  1.00  0.45           C  \\nATOM   1423  CE2 TYR A 184     -17.801  -5.440  16.168  1.00  0.46           C  \\nATOM   1424  OH  TYR A 184     -17.315  -6.535  18.233  1.00  0.42           O  \\nATOM   1425  CZ  TYR A 184     -18.215  -5.903  17.403  1.00  0.41           C  \\nATOM   1426  N   PRO A 185     -24.386  -5.093  13.568  1.00  0.51           N  \\nATOM   1427  CA  PRO A 185     -25.704  -4.462  13.670  1.00  0.49           C  \\nATOM   1428  C   PRO A 185     -25.647  -3.078  14.312  1.00  0.51           C  \\nATOM   1429  CB  PRO A 185     -26.494  -5.438  14.546  1.00  0.48           C  \\nATOM   1430  O   PRO A 185     -24.887  -2.863  15.260  1.00  0.49           O  \\nATOM   1431  CG  PRO A 185     -25.463  -6.118  15.388  1.00  0.46           C  \\nATOM   1432  CD  PRO A 185     -24.163  -6.124  14.636  1.00  0.47           C  \\nATOM   1433  N   PRO A 186     -25.759  -2.004  13.571  1.00  0.53           N  \\nATOM   1434  CA  PRO A 186     -26.105  -0.733  14.212  1.00  0.52           C  \\nATOM   1435  C   PRO A 186     -26.744  -0.921  15.586  1.00  0.53           C  \\nATOM   1436  CB  PRO A 186     -27.095  -0.101  13.230  1.00  0.50           C  \\nATOM   1437  O   PRO A 186     -27.374  -1.950  15.842  1.00  0.51           O  \\nATOM   1438  CG  PRO A 186     -27.719  -1.258  12.520  1.00  0.48           C  \\nATOM   1439  CD  PRO A 186     -26.735  -2.392  12.497  1.00  0.50           C  \\nATOM   1440  N   PRO A 187     -26.180  -0.500  16.690  1.00  0.53           N  \\nATOM   1441  CA  PRO A 187     -26.988  -0.496  17.911  1.00  0.53           C  \\nATOM   1442  C   PRO A 187     -28.487  -0.565  17.627  1.00  0.53           C  \\nATOM   1443  CB  PRO A 187     -26.619   0.834  18.574  1.00  0.50           C  \\nATOM   1444  O   PRO A 187     -29.000   0.202  16.808  1.00  0.51           O  \\nATOM   1445  CG  PRO A 187     -26.024   1.656  17.477  1.00  0.48           C  \\nATOM   1446  CD  PRO A 187     -25.659   0.743  16.341  1.00  0.50           C  \\nATOM   1447  N   ARG A 188     -29.080  -1.822  17.560  1.00  0.52           N  \\nATOM   1448  CA  ARG A 188     -30.513  -1.893  17.823  1.00  0.51           C  \\nATOM   1449  C   ARG A 188     -30.995  -0.655  18.572  1.00  0.51           C  \\nATOM   1450  CB  ARG A 188     -30.851  -3.154  18.623  1.00  0.48           C  \\nATOM   1451  O   ARG A 188     -30.609  -0.428  19.720  1.00  0.49           O  \\nATOM   1452  CG  ARG A 188     -30.968  -4.409  17.774  1.00  0.47           C  \\nATOM   1453  CD  ARG A 188     -31.303  -5.631  18.617  1.00  0.48           C  \\nATOM   1454  NE  ARG A 188     -31.393  -6.840  17.803  1.00  0.43           N  \\nATOM   1455  NH1 ARG A 188     -32.641  -8.031  19.339  1.00  0.34           N  \\nATOM   1456  NH2 ARG A 188     -32.046  -8.994  17.345  1.00  0.30           N  \\nATOM   1457  CZ  ARG A 188     -32.026  -7.952  18.164  1.00  0.45           C  \\nATOM   1458  N   MET A 189     -31.354   0.468  17.980  1.00  0.56           N  \\nATOM   1459  CA  MET A 189     -32.414   1.300  18.541  1.00  0.57           C  \\nATOM   1460  C   MET A 189     -33.571   0.443  19.044  1.00  0.55           C  \\nATOM   1461  CB  MET A 189     -32.919   2.301  17.500  1.00  0.52           C  \\nATOM   1462  O   MET A 189     -33.931  -0.552  18.414  1.00  0.53           O  \\nATOM   1463  CG  MET A 189     -31.915   3.389  17.157  1.00  0.51           C  \\nATOM   1464  SD  MET A 189     -32.674   5.059  17.113  1.00  0.52           S  \\nATOM   1465  CE  MET A 189     -32.219   5.573  15.434  1.00  0.44           C  \\nATOM   1466  N   PRO A 190     -33.720   0.315  20.338  1.00  0.57           N  \\nATOM   1467  CA  PRO A 190     -34.942  -0.309  20.851  1.00  0.56           C  \\nATOM   1468  C   PRO A 190     -36.095  -0.258  19.850  1.00  0.57           C  \\nATOM   1469  CB  PRO A 190     -35.265   0.520  22.097  1.00  0.52           C  \\nATOM   1470  O   PRO A 190     -36.404   0.808  19.312  1.00  0.54           O  \\nATOM   1471  CG  PRO A 190     -34.673   1.865  21.826  1.00  0.50           C  \\nATOM   1472  CD  PRO A 190     -33.561   1.704  20.829  1.00  0.52           C  \\nATOM   1473  N   ILE A 191     -36.380  -1.302  19.046  1.00  0.53           N  \\nATOM   1474  CA  ILE A 191     -37.658  -1.748  18.501  1.00  0.54           C  \\nATOM   1475  C   ILE A 191     -38.800  -1.225  19.370  1.00  0.53           C  \\nATOM   1476  CB  ILE A 191     -37.723  -3.288  18.400  1.00  0.49           C  \\nATOM   1477  O   ILE A 191     -38.997  -1.693  20.494  1.00  0.51           O  \\nATOM   1478  CG1 ILE A 191     -36.584  -3.814  17.519  1.00  0.47           C  \\nATOM   1479  CG2 ILE A 191     -39.084  -3.737  17.861  1.00  0.48           C  \\nATOM   1480  CD1 ILE A 191     -36.407  -5.325  17.572  1.00  0.47           C  \\nATOM   1481  N   GLY A 192     -39.118  -0.031  19.263  1.00  0.55           N  \\nATOM   1482  CA  GLY A 192     -40.494   0.381  19.492  1.00  0.56           C  \\nATOM   1483  C   GLY A 192     -41.484  -0.305  18.571  1.00  0.54           C  \\nATOM   1484  O   GLY A 192     -41.296  -0.325  17.353  1.00  0.52           O  \\nATOM   1485  N   HIS A 193     -42.085  -1.506  18.910  1.00  0.55           N  \\nATOM   1486  CA  HIS A 193     -43.431  -1.952  18.566  1.00  0.57           C  \\nATOM   1487  C   HIS A 193     -44.086  -1.005  17.566  1.00  0.54           C  \\nATOM   1488  CB  HIS A 193     -44.294  -2.068  19.823  1.00  0.50           C  \\nATOM   1489  O   HIS A 193     -44.516   0.091  17.932  1.00  0.52           O  \\nATOM   1490  CG  HIS A 193     -43.996  -3.280  20.646  1.00  0.49           C  \\nATOM   1491  CD2 HIS A 193     -43.904  -3.444  21.987  1.00  0.49           C  \\nATOM   1492  ND1 HIS A 193     -43.748  -4.516  20.088  1.00  0.52           N  \\nATOM   1493  CE1 HIS A 193     -43.516  -5.390  21.054  1.00  0.47           C  \\nATOM   1494  NE2 HIS A 193     -43.604  -4.764  22.215  1.00  0.43           N  \\nATOM   1495  N   GLY A 194     -43.834  -1.118  16.264  1.00  0.54           N  \\nATOM   1496  CA  GLY A 194     -44.748  -0.568  15.276  1.00  0.55           C  \\nATOM   1497  C   GLY A 194     -45.670  -1.611  14.671  1.00  0.54           C  \\nATOM   1498  O   GLY A 194     -45.263  -2.753  14.450  1.00  0.51           O  \\nATOM   1499  N   PRO A 195     -47.010  -1.503  14.750  1.00  0.58           N  \\nATOM   1500  CA  PRO A 195     -48.156  -2.141  14.098  1.00  0.58           C  \\nATOM   1501  C   PRO A 195     -47.809  -2.719  12.728  1.00  0.57           C  \\nATOM   1502  CB  PRO A 195     -49.168  -1.000  13.969  1.00  0.51           C  \\nATOM   1503  O   PRO A 195     -46.961  -2.170  12.019  1.00  0.54           O  \\nATOM   1504  CG  PRO A 195     -48.338   0.241  13.890  1.00  0.50           C  \\nATOM   1505  CD  PRO A 195     -47.059   0.007  14.642  1.00  0.54           C  \\nATOM   1506  N   PRO A 196     -48.048  -4.043  12.262  1.00  0.62           N  \\nATOM   1507  CA  PRO A 196     -48.251  -4.961  11.140  1.00  0.62           C  \\nATOM   1508  C   PRO A 196     -48.489  -4.235   9.818  1.00  0.60           C  \\nATOM   1509  CB  PRO A 196     -49.491  -5.758  11.555  1.00  0.54           C  \\nATOM   1510  O   PRO A 196     -49.549  -3.636   9.620  1.00  0.56           O  \\nATOM   1511  CG  PRO A 196     -50.220  -4.862  12.504  1.00  0.52           C  \\nATOM   1512  CD  PRO A 196     -49.242  -3.876  13.077  1.00  0.56           C  \\nATOM   1513  N   GLY A 197     -47.348  -3.842   9.034  1.00  0.63           N  \\nATOM   1514  CA  GLY A 197     -47.184  -3.731   7.594  1.00  0.64           C  \\nATOM   1515  C   GLY A 197     -45.733  -3.631   7.164  1.00  0.62           C  \\nATOM   1516  O   GLY A 197     -45.044  -2.667   7.504  1.00  0.58           O  \\nATOM   1517  N   GLY A 198     -45.061  -4.731   6.992  1.00  0.60           N  \\nATOM   1518  CA  GLY A 198     -43.822  -5.332   6.523  1.00  0.62           C  \\nATOM   1519  C   GLY A 198     -43.184  -4.566   5.379  1.00  0.60           C  \\nATOM   1520  O   GLY A 198     -43.856  -4.219   4.406  1.00  0.58           O  \\nATOM   1521  N   PRO A 199     -42.081  -3.802   5.601  1.00  0.64           N  \\nATOM   1522  CA  PRO A 199     -41.128  -3.332   4.592  1.00  0.63           C  \\nATOM   1523  C   PRO A 199     -40.738  -4.422   3.596  1.00  0.63           C  \\nATOM   1524  CB  PRO A 199     -39.920  -2.897   5.426  1.00  0.58           C  \\nATOM   1525  O   PRO A 199     -40.683  -5.601   3.955  1.00  0.60           O  \\nATOM   1526  CG  PRO A 199     -39.982  -3.739   6.659  1.00  0.55           C  \\nATOM   1527  CD  PRO A 199     -41.420  -4.075   6.934  1.00  0.57           C  \\nATOM   1528  N   PRO A 200     -40.913  -4.136   2.248  1.00  0.65           N  \\nATOM   1529  CA  PRO A 200     -40.409  -4.973   1.156  1.00  0.65           C  \\nATOM   1530  C   PRO A 200     -39.109  -5.690   1.513  1.00  0.64           C  \\nATOM   1531  CB  PRO A 200     -40.186  -3.972   0.019  1.00  0.59           C  \\nATOM   1532  O   PRO A 200     -38.199  -5.079   2.079  1.00  0.60           O  \\nATOM   1533  CG  PRO A 200     -39.935  -2.667   0.704  1.00  0.57           C  \\nATOM   1534  CD  PRO A 200     -40.663  -2.669   2.017  1.00  0.58           C  \\nATOM   1535  N   MET A 201     -39.074  -7.076   1.504  1.00  0.64           N  \\nATOM   1536  CA  MET A 201     -38.044  -8.108   1.570  1.00  0.66           C  \\nATOM   1537  C   MET A 201     -36.769  -7.650   0.870  1.00  0.64           C  \\nATOM   1538  CB  MET A 201     -38.545  -9.411   0.943  1.00  0.60           C  \\nATOM   1539  O   MET A 201     -36.819  -7.155  -0.258  1.00  0.61           O  \\nATOM   1540  CG  MET A 201     -39.539 -10.166   1.812  1.00  0.57           C  \\nATOM   1541  SD  MET A 201     -38.717 -11.316   2.982  1.00  0.55           S  \\nATOM   1542  CE  MET A 201     -38.676 -12.823   1.972  1.00  0.48           C  \\nATOM   1543  N   PRO A 202     -35.713  -7.188   1.489  1.00  0.65           N  \\nATOM   1544  CA  PRO A 202     -34.362  -7.058   0.937  1.00  0.64           C  \\nATOM   1545  C   PRO A 202     -34.006  -8.191  -0.023  1.00  0.65           C  \\nATOM   1546  CB  PRO A 202     -33.469  -7.093   2.180  1.00  0.59           C  \\nATOM   1547  O   PRO A 202     -34.147  -9.366   0.324  1.00  0.61           O  \\nATOM   1548  CG  PRO A 202     -34.263  -7.843   3.200  1.00  0.58           C  \\nATOM   1549  CD  PRO A 202     -35.720  -7.710   2.862  1.00  0.59           C  \\nATOM   1550  N   GLY A 203     -34.022  -7.956  -1.295  1.00  0.65           N  \\nATOM   1551  CA  GLY A 203     -33.428  -8.771  -2.343  1.00  0.66           C  \\nATOM   1552  C   GLY A 203     -32.163  -9.482  -1.901  1.00  0.63           C  \\nATOM   1553  O   GLY A 203     -31.497  -9.047  -0.959  1.00  0.60           O  \\nATOM   1554  N   PRO A 204     -31.966 -10.793  -2.305  1.00  0.64           N  \\nATOM   1555  CA  PRO A 204     -30.808 -11.628  -1.982  1.00  0.64           C  \\nATOM   1556  C   PRO A 204     -29.515 -10.824  -1.861  1.00  0.64           C  \\nATOM   1557  CB  PRO A 204     -30.736 -12.603  -3.160  1.00  0.60           C  \\nATOM   1558  O   PRO A 204     -29.296  -9.882  -2.626  1.00  0.61           O  \\nATOM   1559  CG  PRO A 204     -31.347 -11.862  -4.306  1.00  0.59           C  \\nATOM   1560  CD  PRO A 204     -32.357 -10.892  -3.762  1.00  0.60           C  \\nATOM   1561  N   PRO A 205     -28.884 -10.675  -0.780  1.00  0.63           N  \\nATOM   1562  CA  PRO A 205     -27.535 -10.132  -0.605  1.00  0.62           C  \\nATOM   1563  C   PRO A 205     -26.604 -10.480  -1.763  1.00  0.63           C  \\nATOM   1564  CB  PRO A 205     -27.059 -10.786   0.695  1.00  0.58           C  \\nATOM   1565  O   PRO A 205     -26.473 -11.653  -2.124  1.00  0.60           O  \\nATOM   1566  CG  PRO A 205     -27.831 -12.062   0.786  1.00  0.57           C  \\nATOM   1567  CD  PRO A 205     -29.115 -11.899   0.024  1.00  0.58           C  \\nATOM   1568  N   GLN A 206     -26.135  -9.535  -2.700  1.00  0.65           N  \\nATOM   1569  CA  GLN A 206     -25.018  -9.615  -3.635  1.00  0.65           C  \\nATOM   1570  C   GLN A 206     -23.752 -10.104  -2.938  1.00  0.64           C  \\nATOM   1571  CB  GLN A 206     -24.766  -8.256  -4.289  1.00  0.60           C  \\nATOM   1572  O   GLN A 206     -23.281  -9.480  -1.985  1.00  0.60           O  \\nATOM   1573  CG  GLN A 206     -25.676  -7.966  -5.476  1.00  0.58           C  \\nATOM   1574  CD  GLN A 206     -25.418  -6.605  -6.094  1.00  0.58           C  \\nATOM   1575  NE2 GLN A 206     -26.080  -6.327  -7.212  1.00  0.50           N  \\nATOM   1576  OE1 GLN A 206     -24.631  -5.810  -5.572  1.00  0.61           O  \\nATOM   1577  N   ARG A 207     -23.421 -11.417  -3.037  1.00  0.64           N  \\nATOM   1578  CA  ARG A 207     -22.187 -12.131  -2.727  1.00  0.64           C  \\nATOM   1579  C   ARG A 207     -20.966 -11.337  -3.179  1.00  0.64           C  \\nATOM   1580  CB  ARG A 207     -22.186 -13.513  -3.383  1.00  0.60           C  \\nATOM   1581  O   ARG A 207     -20.825 -11.027  -4.364  1.00  0.62           O  \\nATOM   1582  CG  ARG A 207     -23.148 -14.500  -2.742  1.00  0.59           C  \\nATOM   1583  CD  ARG A 207     -23.050 -15.879  -3.380  1.00  0.60           C  \\nATOM   1584  NE  ARG A 207     -24.015 -16.810  -2.803  1.00  0.51           N  \\nATOM   1585  NH1 ARG A 207     -22.909 -18.714  -3.503  1.00  0.43           N  \\nATOM   1586  NH2 ARG A 207     -24.860 -18.889  -2.313  1.00  0.40           N  \\nATOM   1587  CZ  ARG A 207     -23.926 -18.135  -2.874  1.00  0.58           C  \\nATOM   1588  N   SER A 208     -20.225 -10.558  -2.407  1.00  0.65           N  \\nATOM   1589  CA  SER A 208     -18.888 -10.019  -2.633  1.00  0.65           C  \\nATOM   1590  C   SER A 208     -17.920 -11.110  -3.077  1.00  0.64           C  \\nATOM   1591  CB  SER A 208     -18.361  -9.341  -1.368  1.00  0.60           C  \\nATOM   1592  O   SER A 208     -17.928 -12.215  -2.529  1.00  0.60           O  \\nATOM   1593  OG  SER A 208     -16.997  -8.985  -1.519  1.00  0.58           O  \\nATOM   1594  N   ARG A 209     -17.487 -11.207  -4.239  1.00  0.63           N  \\nATOM   1595  CA  ARG A 209     -16.425 -12.003  -4.846  1.00  0.63           C  \\nATOM   1596  C   ARG A 209     -15.260 -12.189  -3.881  1.00  0.62           C  \\nATOM   1597  CB  ARG A 209     -15.935 -11.349  -6.140  1.00  0.58           C  \\nATOM   1598  O   ARG A 209     -14.253 -12.812  -4.227  1.00  0.60           O  \\nATOM   1599  CG  ARG A 209     -16.585 -11.905  -7.396  1.00  0.56           C  \\nATOM   1600  CD  ARG A 209     -16.020 -11.264  -8.655  1.00  0.57           C  \\nATOM   1601  NE  ARG A 209     -16.823 -11.585  -9.832  1.00  0.45           N  \\nATOM   1602  NH1 ARG A 209     -15.347 -10.659 -11.348  1.00  0.35           N  \\nATOM   1603  NH2 ARG A 209     -17.291 -11.629 -12.080  1.00  0.31           N  \\nATOM   1604  CZ  ARG A 209     -16.485 -11.290 -11.084  1.00  0.51           C  \\nATOM   1605  N   PHE A 210     -15.429 -11.868  -2.531  1.00  0.61           N  \\nATOM   1606  CA  PHE A 210     -14.368 -12.064  -1.550  1.00  0.61           C  \\nATOM   1607  C   PHE A 210     -14.914 -12.705  -0.281  1.00  0.60           C  \\nATOM   1608  CB  PHE A 210     -13.691 -10.731  -1.216  1.00  0.55           C  \\nATOM   1609  O   PHE A 210     -14.289 -12.628   0.779  1.00  0.57           O  \\nATOM   1610  CG  PHE A 210     -12.901 -10.148  -2.356  1.00  0.53           C  \\nATOM   1611  CD1 PHE A 210     -11.675 -10.693  -2.721  1.00  0.50           C  \\nATOM   1612  CD2 PHE A 210     -13.383  -9.055  -3.064  1.00  0.52           C  \\nATOM   1613  CE1 PHE A 210     -10.942 -10.156  -3.776  1.00  0.51           C  \\nATOM   1614  CE2 PHE A 210     -12.656  -8.512  -4.119  1.00  0.50           C  \\nATOM   1615  CZ  PHE A 210     -11.435  -9.064  -4.473  1.00  0.48           C  \\nATOM   1616  N   ASP A 211     -16.045 -13.238  -0.330  1.00  0.59           N  \\nATOM   1617  CA  ASP A 211     -16.494 -14.112   0.750  1.00  0.59           C  \\nATOM   1618  C   ASP A 211     -15.891 -15.509   0.612  1.00  0.58           C  \\nATOM   1619  CB  ASP A 211     -18.021 -14.198   0.771  1.00  0.54           C  \\nATOM   1620  O   ASP A 211     -16.295 -16.281  -0.260  1.00  0.56           O  \\nATOM   1621  CG  ASP A 211     -18.680 -12.915   1.246  1.00  0.52           C  \\nATOM   1622  OD1 ASP A 211     -18.177 -12.288   2.202  1.00  0.53           O  \\nATOM   1623  OD2 ASP A 211     -19.715 -12.529   0.659  1.00  0.59           O  \\nATOM   1624  N   GLN A 212     -14.616 -15.742   0.954  1.00  0.60           N  \\nATOM   1625  CA  GLN A 212     -14.099 -17.048   1.352  1.00  0.61           C  \\nATOM   1626  C   GLN A 212     -14.958 -17.667   2.451  1.00  0.60           C  \\nATOM   1627  CB  GLN A 212     -12.649 -16.931   1.822  1.00  0.56           C  \\nATOM   1628  O   GLN A 212     -15.300 -16.999   3.429  1.00  0.57           O  \\nATOM   1629  CG  GLN A 212     -11.641 -16.820   0.686  1.00  0.54           C  \\nATOM   1630  CD  GLN A 212     -10.205 -16.924   1.163  1.00  0.55           C  \\nATOM   1631  NE2 GLN A 212      -9.307 -17.307   0.262  1.00  0.45           N  \\nATOM   1632  OE1 GLN A 212      -9.904 -16.662   2.332  1.00  0.57           O  \\nATOM   1633  N   PRO A 213     -15.639 -18.791   2.168  1.00  0.57           N  \\nATOM   1634  CA  PRO A 213     -16.337 -19.657   3.121  1.00  0.56           C  \\nATOM   1635  C   PRO A 213     -15.542 -19.883   4.405  1.00  0.57           C  \\nATOM   1636  CB  PRO A 213     -16.509 -20.967   2.348  1.00  0.53           C  \\nATOM   1637  O   PRO A 213     -14.388 -20.316   4.353  1.00  0.55           O  \\nATOM   1638  CG  PRO A 213     -15.406 -20.963   1.339  1.00  0.51           C  \\nATOM   1639  CD  PRO A 213     -14.973 -19.543   1.112  1.00  0.52           C  \\nATOM   1640  N   ASP A 214     -15.717 -19.166   5.398  1.00  0.56           N  \\nATOM   1641  CA  ASP A 214     -15.525 -19.603   6.777  1.00  0.57           C  \\nATOM   1642  C   ASP A 214     -16.040 -21.027   6.979  1.00  0.56           C  \\nATOM   1643  CB  ASP A 214     -16.226 -18.648   7.746  1.00  0.53           C  \\nATOM   1644  O   ASP A 214     -17.224 -21.299   6.773  1.00  0.54           O  \\nATOM   1645  CG  ASP A 214     -15.255 -17.812   8.561  1.00  0.53           C  \\nATOM   1646  OD1 ASP A 214     -14.092 -18.233   8.743  1.00  0.52           O  \\nATOM   1647  OD2 ASP A 214     -15.658 -16.725   9.029  1.00  0.58           O  \\nATOM   1648  N   GLY A 215     -15.337 -22.012   6.761  1.00  0.55           N  \\nATOM   1649  CA  GLY A 215     -15.331 -23.307   7.422  1.00  0.55           C  \\nATOM   1650  C   GLY A 215     -16.450 -23.467   8.433  1.00  0.54           C  \\nATOM   1651  O   GLY A 215     -16.282 -23.142   9.610  1.00  0.52           O  \\nATOM   1652  N   GLY A 216     -17.707 -23.425   7.809  1.00  0.49           N  \\nATOM   1653  CA  GLY A 216     -18.786 -24.140   8.471  1.00  0.51           C  \\nATOM   1654  C   GLY A 216     -19.639 -24.952   7.516  1.00  0.49           C  \\nATOM   1655  O   GLY A 216     -20.176 -24.416   6.544  1.00  0.48           O  \\nATOM   1656  N   ASP A 217     -19.355 -26.248   7.342  1.00  0.57           N  \\nATOM   1657  CA  ASP A 217     -20.314 -27.327   7.124  1.00  0.58           C  \\nATOM   1658  C   ASP A 217     -21.749 -26.811   7.198  1.00  0.56           C  \\nATOM   1659  CB  ASP A 217     -20.104 -28.446   8.146  1.00  0.53           C  \\nATOM   1660  O   ASP A 217     -22.281 -26.595   8.289  1.00  0.54           O  \\nATOM   1661  CG  ASP A 217     -19.175 -29.539   7.646  1.00  0.53           C  \\nATOM   1662  OD1 ASP A 217     -19.101 -29.763   6.419  1.00  0.53           O  \\nATOM   1663  OD2 ASP A 217     -18.514 -30.185   8.488  1.00  0.57           O  \\nATOM   1664  N   ARG A 218     -22.128 -25.959   6.301  1.00  0.55           N  \\nATOM   1665  CA  ARG A 218     -23.574 -25.937   6.105  1.00  0.54           C  \\nATOM   1666  C   ARG A 218     -23.926 -25.969   4.621  1.00  0.55           C  \\nATOM   1667  CB  ARG A 218     -24.186 -24.698   6.762  1.00  0.50           C  \\nATOM   1668  O   ARG A 218     -23.645 -25.017   3.890  1.00  0.52           O  \\nATOM   1669  CG  ARG A 218     -24.291 -24.792   8.276  1.00  0.48           C  \\nATOM   1670  CD  ARG A 218     -25.022 -23.594   8.866  1.00  0.50           C  \\nATOM   1671  NE  ARG A 218     -25.062 -23.652  10.324  1.00  0.43           N  \\nATOM   1672  NH1 ARG A 218     -26.889 -22.254  10.536  1.00  0.33           N  \\nATOM   1673  NH2 ARG A 218     -25.894 -23.137  12.402  1.00  0.28           N  \\nATOM   1674  CZ  ARG A 218     -25.948 -23.014  11.084  1.00  0.41           C  \\nATOM   1675  N   TRP A 219     -24.143 -27.183   4.055  1.00  0.55           N  \\nATOM   1676  CA  TRP A 219     -25.219 -27.594   3.159  1.00  0.61           C  \\nATOM   1677  C   TRP A 219     -26.231 -26.469   2.972  1.00  0.55           C  \\nATOM   1678  CB  TRP A 219     -25.920 -28.844   3.698  1.00  0.51           C  \\nATOM   1679  O   TRP A 219     -27.177 -26.340   3.753  1.00  0.52           O  \\nATOM   1680  CG  TRP A 219     -25.144 -30.111   3.500  1.00  0.49           C  \\nATOM   1681  CD1 TRP A 219     -24.162 -30.611   4.310  1.00  0.45           C  \\nATOM   1682  CD2 TRP A 219     -25.282 -31.037   2.419  1.00  0.46           C  \\nATOM   1683  CE2 TRP A 219     -24.352 -32.077   2.637  1.00  0.45           C  \\nATOM   1684  CE3 TRP A 219     -26.105 -31.090   1.285  1.00  0.59           C  \\nATOM   1685  NE1 TRP A 219     -23.682 -31.793   3.796  1.00  0.56           N  \\nATOM   1686  CH2 TRP A 219     -25.038 -33.185   0.661  1.00  0.53           C  \\nATOM   1687  CZ2 TRP A 219     -24.222 -33.159   1.762  1.00  0.58           C  \\nATOM   1688  CZ3 TRP A 219     -25.973 -32.166   0.416  1.00  0.49           C  \\nATOM   1689  N   GLY A 220     -25.843 -25.297   2.388  1.00  0.56           N  \\nATOM   1690  CA  GLY A 220     -26.943 -24.540   1.811  1.00  0.56           C  \\nATOM   1691  C   GLY A 220     -27.302 -24.987   0.406  1.00  0.56           C  \\nATOM   1692  O   GLY A 220     -26.422 -25.327  -0.387  1.00  0.53           O  \\nATOM   1693  N   PRO A 221     -28.642 -25.339   0.239  1.00  0.58           N  \\nATOM   1694  CA  PRO A 221     -29.296 -25.598  -1.047  1.00  0.57           C  \\nATOM   1695  C   PRO A 221     -28.811 -24.665  -2.154  1.00  0.58           C  \\nATOM   1696  CB  PRO A 221     -30.777 -25.363  -0.743  1.00  0.54           C  \\nATOM   1697  O   PRO A 221     -28.523 -23.493  -1.896  1.00  0.55           O  \\nATOM   1698  CG  PRO A 221     -30.784 -24.433   0.427  1.00  0.52           C  \\nATOM   1699  CD  PRO A 221     -29.492 -24.595   1.174  1.00  0.54           C  \\nATOM   1700  N   PRO A 222     -28.336 -25.152  -3.290  1.00  0.59           N  \\nATOM   1701  CA  PRO A 222     -27.964 -24.570  -4.582  1.00  0.58           C  \\nATOM   1702  C   PRO A 222     -28.814 -23.357  -4.950  1.00  0.58           C  \\nATOM   1703  CB  PRO A 222     -28.197 -25.719  -5.566  1.00  0.54           C  \\nATOM   1704  O   PRO A 222     -30.034 -23.475  -5.097  1.00  0.56           O  \\nATOM   1705  CG  PRO A 222     -29.287 -26.535  -4.950  1.00  0.52           C  \\nATOM   1706  CD  PRO A 222     -29.243 -26.343  -3.461  1.00  0.54           C  \\nATOM   1707  N   MET A 223     -28.429 -22.171  -4.670  1.00  0.57           N  \\nATOM   1708  CA  MET A 223     -28.933 -20.919  -5.228  1.00  0.59           C  \\nATOM   1709  C   MET A 223     -29.106 -21.028  -6.739  1.00  0.57           C  \\nATOM   1710  CB  MET A 223     -27.990 -19.763  -4.891  1.00  0.53           C  \\nATOM   1711  O   MET A 223     -28.155 -21.344  -7.457  1.00  0.55           O  \\nATOM   1712  CG  MET A 223     -28.128 -19.255  -3.465  1.00  0.52           C  \\nATOM   1713  SD  MET A 223     -27.657 -17.489  -3.299  1.00  0.55           S  \\nATOM   1714  CE  MET A 223     -28.629 -17.035  -1.836  1.00  0.47           C  \\nATOM   1715  N   ARG A 224     -30.306 -21.164  -7.317  1.00  0.61           N  \\nATOM   1716  CA  ARG A 224     -31.049 -21.044  -8.567  1.00  0.61           C  \\nATOM   1717  C   ARG A 224     -30.870 -19.659  -9.180  1.00  0.59           C  \\nATOM   1718  CB  ARG A 224     -32.535 -21.328  -8.339  1.00  0.56           C  \\nATOM   1719  O   ARG A 224     -31.284 -18.657  -8.593  1.00  0.56           O  \\nATOM   1720  CG  ARG A 224     -32.856 -22.799  -8.125  1.00  0.55           C  \\nATOM   1721  CD  ARG A 224     -34.355 -23.039  -8.014  1.00  0.57           C  \\nATOM   1722  NE  ARG A 224     -34.649 -24.351  -7.444  1.00  0.47           N  \\nATOM   1723  NH1 ARG A 224     -36.512 -24.732  -8.757  1.00  0.37           N  \\nATOM   1724  NH2 ARG A 224     -35.843 -26.301  -7.225  1.00  0.33           N  \\nATOM   1725  CZ  ARG A 224     -35.667 -25.125  -7.810  1.00  0.52           C  \\nATOM   1726  N   GLY A 225     -29.746 -19.450 -10.207  1.00  0.55           N  \\nATOM   1727  CA  GLY A 225     -29.643 -18.885 -11.543  1.00  0.56           C  \\nATOM   1728  C   GLY A 225     -29.349 -17.397 -11.540  1.00  0.55           C  \\nATOM   1729  O   GLY A 225     -30.234 -16.584 -11.263  1.00  0.52           O  \\nATOM   1730  N   VAL A 226     -28.144 -16.881 -11.282  1.00  0.58           N  \\nATOM   1731  CA  VAL A 226     -27.531 -15.741 -11.955  1.00  0.59           C  \\nATOM   1732  C   VAL A 226     -27.041 -16.160 -13.340  1.00  0.58           C  \\nATOM   1733  CB  VAL A 226     -26.363 -15.155 -11.131  1.00  0.54           C  \\nATOM   1734  O   VAL A 226     -26.341 -17.166 -13.479  1.00  0.55           O  \\nATOM   1735  CG1 VAL A 226     -25.885 -13.835 -11.733  1.00  0.48           C  \\nATOM   1736  CG2 VAL A 226     -26.783 -14.961  -9.675  1.00  0.49           C  \\nATOM   1737  N   PRO A 227     -27.660 -15.649 -14.392  1.00  0.57           N  \\nATOM   1738  CA  PRO A 227     -27.169 -15.889 -15.751  1.00  0.57           C  \\nATOM   1739  C   PRO A 227     -25.644 -15.901 -15.834  1.00  0.56           C  \\nATOM   1740  CB  PRO A 227     -27.749 -14.717 -16.548  1.00  0.53           C  \\nATOM   1741  O   PRO A 227     -24.986 -15.001 -15.306  1.00  0.54           O  \\nATOM   1742  CG  PRO A 227     -28.336 -13.808 -15.517  1.00  0.52           C  \\nATOM   1743  CD  PRO A 227     -28.246 -14.483 -14.178  1.00  0.54           C  \\nATOM   1744  N   ARG A 228     -24.974 -17.109 -15.816  1.00  0.54           N  \\nATOM   1745  CA  ARG A 228     -23.711 -17.527 -16.417  1.00  0.54           C  \\nATOM   1746  C   ARG A 228     -23.302 -16.583 -17.542  1.00  0.54           C  \\nATOM   1747  CB  ARG A 228     -23.814 -18.959 -16.946  1.00  0.51           C  \\nATOM   1748  O   ARG A 228     -24.004 -16.467 -18.549  1.00  0.52           O  \\nATOM   1749  CG  ARG A 228     -23.460 -20.022 -15.918  1.00  0.50           C  \\nATOM   1750  CD  ARG A 228     -23.576 -21.426 -16.496  1.00  0.51           C  \\nATOM   1751  NE  ARG A 228     -23.209 -22.444 -15.516  1.00  0.46           N  \\nATOM   1752  NH1 ARG A 228     -23.182 -24.188 -17.031  1.00  0.35           N  \\nATOM   1753  NH2 ARG A 228     -22.697 -24.573 -14.824  1.00  0.30           N  \\nATOM   1754  CZ  ARG A 228     -23.030 -23.733 -15.792  1.00  0.44           C  \\nATOM   1755  N   THR A 229     -22.607 -15.504 -17.392  1.00  0.57           N  \\nATOM   1756  CA  THR A 229     -21.765 -15.123 -18.521  1.00  0.58           C  \\nATOM   1757  C   THR A 229     -21.442 -16.337 -19.388  1.00  0.56           C  \\nATOM   1758  CB  THR A 229     -20.456 -14.466 -18.046  1.00  0.53           C  \\nATOM   1759  O   THR A 229     -21.028 -17.379 -18.876  1.00  0.54           O  \\nATOM   1760  CG2 THR A 229     -20.586 -12.947 -18.009  1.00  0.48           C  \\nATOM   1761  OG1 THR A 229     -20.141 -14.939 -16.730  1.00  0.50           O  \\nATOM   1762  N   PRO A 230     -22.153 -16.561 -20.456  1.00  0.54           N  \\nATOM   1763  CA  PRO A 230     -21.734 -17.598 -21.403  1.00  0.53           C  \\nATOM   1764  C   PRO A 230     -20.234 -17.877 -21.346  1.00  0.53           C  \\nATOM   1765  CB  PRO A 230     -22.130 -17.014 -22.762  1.00  0.50           C  \\nATOM   1766  O   PRO A 230     -19.445 -16.974 -21.053  1.00  0.52           O  \\nATOM   1767  CG  PRO A 230     -22.298 -15.550 -22.515  1.00  0.49           C  \\nATOM   1768  CD  PRO A 230     -22.431 -15.328 -21.035  1.00  0.51           C  \\nATOM   1769  N   PRO A 231     -19.729 -19.107 -20.929  1.00  0.53           N  \\nATOM   1770  CA  PRO A 231     -18.368 -19.537 -21.259  1.00  0.52           C  \\nATOM   1771  C   PRO A 231     -17.804 -18.820 -22.484  1.00  0.53           C  \\nATOM   1772  CB  PRO A 231     -18.528 -21.035 -21.529  1.00  0.49           C  \\nATOM   1773  O   PRO A 231     -18.469 -18.742 -23.519  1.00  0.51           O  \\nATOM   1774  CG  PRO A 231     -19.989 -21.225 -21.779  1.00  0.47           C  \\nATOM   1775  CD  PRO A 231     -20.719 -20.007 -21.290  1.00  0.50           C  \\nATOM   1776  N   TYR A 232     -17.256 -17.574 -22.441  1.00  0.53           N  \\nATOM   1777  CA  TYR A 232     -16.282 -17.484 -23.524  1.00  0.54           C  \\nATOM   1778  C   TYR A 232     -15.802 -18.868 -23.942  1.00  0.52           C  \\nATOM   1779  CB  TYR A 232     -15.088 -16.622 -23.102  1.00  0.49           C  \\nATOM   1780  O   TYR A 232     -15.185 -19.584 -23.149  1.00  0.50           O  \\nATOM   1781  CG  TYR A 232     -15.335 -15.138 -23.229  1.00  0.48           C  \\nATOM   1782  CD1 TYR A 232     -15.129 -14.479 -24.439  1.00  0.46           C  \\nATOM   1783  CD2 TYR A 232     -15.775 -14.393 -22.140  1.00  0.47           C  \\nATOM   1784  CE1 TYR A 232     -15.354 -13.111 -24.560  1.00  0.47           C  \\nATOM   1785  CE2 TYR A 232     -16.003 -13.025 -22.250  1.00  0.47           C  \\nATOM   1786  OH  TYR A 232     -16.014 -11.040 -23.576  1.00  0.46           O  \\nATOM   1787  CZ  TYR A 232     -15.790 -12.394 -23.462  1.00  0.42           C  \\nATOM   1788  N   GLU A 233     -16.508 -19.741 -24.657  1.00  0.54           N  \\nATOM   1789  CA  GLU A 233     -15.942 -20.561 -25.723  1.00  0.54           C  \\nATOM   1790  C   GLU A 233     -14.736 -19.880 -26.362  1.00  0.53           C  \\nATOM   1791  CB  GLU A 233     -17.000 -20.865 -26.787  1.00  0.50           C  \\nATOM   1792  O   GLU A 233     -14.886 -18.906 -27.103  1.00  0.51           O  \\nATOM   1793  CG  GLU A 233     -17.956 -21.985 -26.403  1.00  0.49           C  \\nATOM   1794  CD  GLU A 233     -19.109 -22.152 -27.380  1.00  0.51           C  \\nATOM   1795  OE1 GLU A 233     -19.559 -21.141 -27.966  1.00  0.54           O  \\nATOM   1796  OE2 GLU A 233     -19.567 -23.302 -27.560  1.00  0.50           O  \\nATOM   1797  N   ASN A 234     -13.654 -19.634 -25.658  1.00  0.52           N  \\nATOM   1798  CA  ASN A 234     -12.401 -19.707 -26.403  1.00  0.53           C  \\nATOM   1799  C   ASN A 234     -12.169 -21.102 -26.976  1.00  0.52           C  \\nATOM   1800  CB  ASN A 234     -11.224 -19.296 -25.515  1.00  0.48           C  \\nATOM   1801  O   ASN A 234     -11.881 -22.043 -26.233  1.00  0.50           O  \\nATOM   1802  CG  ASN A 234     -10.706 -17.909 -25.838  1.00  0.47           C  \\nATOM   1803  ND2 ASN A 234      -9.894 -17.356 -24.945  1.00  0.47           N  \\nATOM   1804  OD1 ASN A 234     -11.033 -17.338 -26.882  1.00  0.48           O  \\nATOM   1805  N   GLU A 235     -12.881 -21.595 -27.851  1.00  0.53           N  \\nATOM   1806  CA  GLU A 235     -12.325 -22.342 -28.975  1.00  0.53           C  \\nATOM   1807  C   GLU A 235     -10.914 -21.866 -29.310  1.00  0.53           C  \\nATOM   1808  CB  GLU A 235     -13.228 -22.217 -30.204  1.00  0.50           C  \\nATOM   1809  O   GLU A 235     -10.740 -20.831 -29.957  1.00  0.50           O  \\nATOM   1810  CG  GLU A 235     -14.481 -23.078 -30.137  1.00  0.49           C  \\nATOM   1811  CD  GLU A 235     -15.371 -22.939 -31.362  1.00  0.52           C  \\nATOM   1812  OE1 GLU A 235     -15.453 -21.826 -31.929  1.00  0.53           O  \\nATOM   1813  OE2 GLU A 235     -15.992 -23.951 -31.757  1.00  0.49           O  \\nATOM   1814  N   HIS A 236     -10.062 -21.897 -28.354  1.00  0.49           N  \\nATOM   1815  CA  HIS A 236      -8.675 -22.189 -28.698  1.00  0.50           C  \\nATOM   1816  C   HIS A 236      -8.354 -23.665 -28.488  1.00  0.49           C  \\nATOM   1817  CB  HIS A 236      -7.724 -21.322 -27.870  1.00  0.45           C  \\nATOM   1818  O   HIS A 236      -8.382 -24.156 -27.357  1.00  0.46           O  \\nATOM   1819  CG  HIS A 236      -7.608 -19.916 -28.367  1.00  0.44           C  \\nATOM   1820  CD2 HIS A 236      -7.860 -18.734 -27.758  1.00  0.42           C  \\nATOM   1821  ND1 HIS A 236      -7.187 -19.611 -29.643  1.00  0.50           N  \\nATOM   1822  CE1 HIS A 236      -7.185 -18.298 -29.797  1.00  0.44           C  \\nATOM   1823  NE2 HIS A 236      -7.590 -17.742 -28.668  1.00  0.40           N  \\nATOM   1824  N   GLU A 237      -8.735 -24.491 -29.359  1.00  0.52           N  \\nATOM   1825  CA  GLU A 237      -7.946 -25.543 -29.991  1.00  0.52           C  \\nATOM   1826  C   GLU A 237      -6.756 -25.939 -29.121  1.00  0.52           C  \\nATOM   1827  CB  GLU A 237      -7.462 -25.097 -31.373  1.00  0.49           C  \\nATOM   1828  O   GLU A 237      -5.858 -25.129 -28.883  1.00  0.50           O  \\nATOM   1829  CG  GLU A 237      -8.069 -25.888 -32.522  1.00  0.47           C  \\nATOM   1830  CD  GLU A 237      -7.694 -25.342 -33.890  1.00  0.50           C  \\nATOM   1831  OE1 GLU A 237      -7.385 -24.133 -33.998  1.00  0.52           O  \\nATOM   1832  OE2 GLU A 237      -7.709 -26.129 -34.863  1.00  0.48           O  \\nATOM   1833  N   HIS A 238      -6.825 -26.721 -27.926  1.00  0.51           N  \\nATOM   1834  CA  HIS A 238      -5.835 -27.746 -27.619  1.00  0.52           C  \\nATOM   1835  C   HIS A 238      -4.930 -28.014 -28.817  1.00  0.50           C  \\nATOM   1836  CB  HIS A 238      -6.522 -29.041 -27.181  1.00  0.48           C  \\nATOM   1837  O   HIS A 238      -5.309 -28.743 -29.736  1.00  0.48           O  \\nATOM   1838  CG  HIS A 238      -6.596 -29.208 -25.696  1.00  0.48           C  \\nATOM   1839  CD2 HIS A 238      -7.659 -29.326 -24.866  1.00  0.47           C  \\nATOM   1840  ND1 HIS A 238      -5.474 -29.267 -24.898  1.00  0.51           N  \\nATOM   1841  CE1 HIS A 238      -5.845 -29.416 -23.637  1.00  0.46           C  \\nATOM   1842  NE2 HIS A 238      -7.166 -29.454 -23.591  1.00  0.43           N  \\nATOM   1843  N   GLN A 239      -4.209 -27.058 -29.241  1.00  0.48           N  \\nATOM   1844  CA  GLN A 239      -2.898 -27.409 -29.776  1.00  0.49           C  \\nATOM   1845  C   GLN A 239      -1.902 -27.685 -28.654  1.00  0.48           C  \\nATOM   1846  CB  GLN A 239      -2.372 -26.295 -30.683  1.00  0.46           C  \\nATOM   1847  O   GLN A 239      -1.727 -26.860 -27.755  1.00  0.47           O  \\nATOM   1848  CG  GLN A 239      -2.931 -26.339 -32.099  1.00  0.46           C  \\nATOM   1849  CD  GLN A 239      -2.403 -25.217 -32.973  1.00  0.48           C  \\nATOM   1850  NE2 GLN A 239      -2.880 -25.157 -34.212  1.00  0.40           N  \\nATOM   1851  OE1 GLN A 239      -1.575 -24.411 -32.539  1.00  0.51           O  \\nATOM   1852  N   ASP A 240      -1.788 -28.836 -28.141  1.00  0.51           N  \\nATOM   1853  CA  ASP A 240      -0.488 -29.375 -27.757  1.00  0.52           C  \\nATOM   1854  C   ASP A 240       0.581 -28.284 -27.754  1.00  0.50           C  \\nATOM   1855  CB  ASP A 240      -0.078 -30.510 -28.698  1.00  0.48           C  \\nATOM   1856  O   ASP A 240       1.020 -27.836 -28.815  1.00  0.49           O  \\nATOM   1857  CG  ASP A 240      -0.335 -31.888 -28.113  1.00  0.48           C  \\nATOM   1858  OD1 ASP A 240      -0.316 -32.036 -26.872  1.00  0.49           O  \\nATOM   1859  OD2 ASP A 240      -0.554 -32.834 -28.900  1.00  0.55           O  \\nATOM   1860  N   TYR A 241       0.629 -27.259 -26.966  1.00  0.46           N  \\nATOM   1861  CA  TYR A 241       1.905 -26.608 -26.691  1.00  0.48           C  \\nATOM   1862  C   TYR A 241       2.991 -27.637 -26.402  1.00  0.46           C  \\nATOM   1863  CB  TYR A 241       1.773 -25.642 -25.509  1.00  0.44           C  \\nATOM   1864  O   TYR A 241       3.073 -28.166 -25.291  1.00  0.45           O  \\nATOM   1865  CG  TYR A 241       2.197 -24.229 -25.830  1.00  0.44           C  \\nATOM   1866  CD1 TYR A 241       3.504 -23.805 -25.602  1.00  0.42           C  \\nATOM   1867  CD2 TYR A 241       1.292 -23.316 -26.360  1.00  0.43           C  \\nATOM   1868  CE1 TYR A 241       3.899 -22.504 -25.894  1.00  0.43           C  \\nATOM   1869  CE2 TYR A 241       1.675 -22.012 -26.655  1.00  0.46           C  \\nATOM   1870  OH  TYR A 241       3.364 -20.326 -26.711  1.00  0.38           O  \\nATOM   1871  CZ  TYR A 241       2.979 -21.616 -26.420  1.00  0.37           C  \\nATOM   1872  N   GLN A 242       3.376 -28.335 -27.341  1.00  0.47           N  \\nATOM   1873  CA  GLN A 242       4.798 -28.631 -27.474  1.00  0.47           C  \\nATOM   1874  C   GLN A 242       5.640 -27.370 -27.301  1.00  0.47           C  \\nATOM   1875  CB  GLN A 242       5.089 -29.274 -28.831  1.00  0.44           C  \\nATOM   1876  O   GLN A 242       5.501 -26.414 -28.066  1.00  0.45           O  \\nATOM   1877  CG  GLN A 242       4.686 -30.740 -28.915  1.00  0.44           C  \\nATOM   1878  CD  GLN A 242       5.006 -31.360 -30.263  1.00  0.47           C  \\nATOM   1879  NE2 GLN A 242       5.021 -32.687 -30.316  1.00  0.41           N  \\nATOM   1880  OE1 GLN A 242       5.238 -30.651 -31.247  1.00  0.51           O  \\nATOM   1881  N   ALA A 243       6.046 -26.804 -26.193  1.00  0.52           N  \\nATOM   1882  CA  ALA A 243       7.264 -26.001 -26.270  1.00  0.52           C  \\nATOM   1883  C   ALA A 243       7.965 -26.195 -27.612  1.00  0.51           C  \\nATOM   1884  CB  ALA A 243       8.208 -26.356 -25.124  1.00  0.49           C  \\nATOM   1885  O   ALA A 243       8.156 -27.328 -28.061  1.00  0.50           O  \\nATOM   1886  N   PRO A 244       7.745 -25.391 -28.575  1.00  0.48           N  \\nATOM   1887  CA  PRO A 244       8.633 -25.533 -29.732  1.00  0.47           C  \\nATOM   1888  C   PRO A 244      10.089 -25.760 -29.334  1.00  0.48           C  \\nATOM   1889  CB  PRO A 244       8.468 -24.201 -30.468  1.00  0.44           C  \\nATOM   1890  O   PRO A 244      10.637 -25.005 -28.526  1.00  0.45           O  \\nATOM   1891  CG  PRO A 244       7.848 -23.284 -29.464  1.00  0.41           C  \\nATOM   1892  CD  PRO A 244       7.332 -24.108 -28.320  1.00  0.44           C  \\nATOM   1893  N   ASP A 245      10.586 -26.873 -29.108  1.00  0.52           N  \\nATOM   1894  CA  ASP A 245      11.923 -27.155 -29.623  1.00  0.52           C  \\nATOM   1895  C   ASP A 245      12.247 -26.271 -30.825  1.00  0.51           C  \\nATOM   1896  CB  ASP A 245      12.049 -28.631 -30.006  1.00  0.49           C  \\nATOM   1897  O   ASP A 245      11.705 -26.471 -31.913  1.00  0.48           O  \\nATOM   1898  CG  ASP A 245      12.327 -29.532 -28.816  1.00  0.49           C  \\nATOM   1899  OD1 ASP A 245      13.145 -29.159 -27.947  1.00  0.50           O  \\nATOM   1900  OD2 ASP A 245      11.726 -30.625 -28.747  1.00  0.56           O  \\nATOM   1901  N   ASP A 246      12.032 -24.838 -30.716  1.00  0.49           N  \\nATOM   1902  CA  ASP A 246      12.744 -24.063 -31.727  1.00  0.50           C  \\nATOM   1903  C   ASP A 246      12.877 -22.601 -31.306  1.00  0.49           C  \\nATOM   1904  CB  ASP A 246      12.031 -24.158 -33.077  1.00  0.46           C  \\nATOM   1905  O   ASP A 246      12.038 -21.769 -31.658  1.00  0.46           O  \\nATOM   1906  CG  ASP A 246      12.650 -25.191 -34.003  1.00  0.45           C  \\nATOM   1907  OD1 ASP A 246      13.850 -25.505 -33.851  1.00  0.47           O  \\nATOM   1908  OD2 ASP A 246      11.931 -25.693 -34.895  1.00  0.51           O  \\nATOM   1909  N   TYR A 247      13.512 -22.278 -30.174  1.00  0.55           N  \\nATOM   1910  CA  TYR A 247      14.473 -21.183 -30.110  1.00  0.56           C  \\nATOM   1911  C   TYR A 247      15.580 -21.368 -31.141  1.00  0.55           C  \\nATOM   1912  CB  TYR A 247      15.078 -21.081 -28.706  1.00  0.52           C  \\nATOM   1913  O   TYR A 247      16.576 -22.047 -30.877  1.00  0.53           O  \\nATOM   1914  CG  TYR A 247      14.492 -19.967 -27.873  1.00  0.50           C  \\nATOM   1915  CD1 TYR A 247      15.041 -18.687 -27.901  1.00  0.49           C  \\nATOM   1916  CD2 TYR A 247      13.390 -20.193 -27.055  1.00  0.49           C  \\nATOM   1917  CE1 TYR A 247      14.506 -17.658 -27.132  1.00  0.49           C  \\nATOM   1918  CE2 TYR A 247      12.847 -19.172 -26.283  1.00  0.49           C  \\nATOM   1919  OH  TYR A 247      12.876 -16.895 -25.566  1.00  0.49           O  \\nATOM   1920  CZ  TYR A 247      13.410 -17.910 -26.329  1.00  0.44           C  \\nATOM   1921  N   ASN A 248      15.363 -21.555 -32.278  1.00  0.47           N  \\nATOM   1922  CA  ASN A 248      16.215 -21.333 -33.442  1.00  0.51           C  \\nATOM   1923  C   ASN A 248      16.335 -19.849 -33.774  1.00  0.47           C  \\nATOM   1924  CB  ASN A 248      15.686 -22.108 -34.651  1.00  0.45           C  \\nATOM   1925  O   ASN A 248      15.335 -19.194 -34.076  1.00  0.45           O  \\nATOM   1926  CG  ASN A 248      16.730 -22.282 -35.736  1.00  0.45           C  \\nATOM   1927  ND2 ASN A 248      16.291 -22.683 -36.923  1.00  0.39           N  \\nATOM   1928  OD1 ASN A 248      17.922 -22.058 -35.509  1.00  0.45           O  \\nATOM   1929  N   GLN A 249      17.337 -19.043 -33.133  1.00  0.53           N  \\nATOM   1930  CA  GLN A 249      18.307 -18.217 -33.846  1.00  0.53           C  \\nATOM   1931  C   GLN A 249      17.630 -17.019 -34.504  1.00  0.52           C  \\nATOM   1932  CB  GLN A 249      19.047 -19.045 -34.897  1.00  0.49           C  \\nATOM   1933  O   GLN A 249      17.121 -17.123 -35.622  1.00  0.50           O  \\nATOM   1934  CG  GLN A 249      20.105 -19.974 -34.316  1.00  0.49           C  \\nATOM   1935  CD  GLN A 249      20.841 -20.764 -35.382  1.00  0.53           C  \\nATOM   1936  NE2 GLN A 249      21.564 -21.796 -34.960  1.00  0.43           N  \\nATOM   1937  OE1 GLN A 249      20.759 -20.451 -36.574  1.00  0.55           O  \\nATOM   1938  N   GLY A 250      17.098 -16.128 -33.779  1.00  0.51           N  \\nATOM   1939  CA  GLY A 250      17.066 -14.811 -34.395  1.00  0.51           C  \\nATOM   1940  C   GLY A 250      17.833 -13.765 -33.608  1.00  0.51           C  \\nATOM   1941  O   GLY A 250      17.499 -13.479 -32.456  1.00  0.48           O  \\nATOM   1942  N   GLY A 251      19.260 -13.766 -33.719  1.00  0.52           N  \\nATOM   1943  CA  GLY A 251      20.244 -12.702 -33.834  1.00  0.52           C  \\nATOM   1944  C   GLY A 251      19.961 -11.525 -32.920  1.00  0.52           C  \\nATOM   1945  O   GLY A 251      19.050 -10.736 -33.180  1.00  0.50           O  \\nATOM   1946  N   TYR A 252      20.059 -11.545 -31.581  1.00  0.55           N  \\nATOM   1947  CA  TYR A 252      20.663 -10.422 -30.871  1.00  0.56           C  \\nATOM   1948  C   TYR A 252      21.998 -10.037 -31.496  1.00  0.55           C  \\nATOM   1949  CB  TYR A 252      20.858 -10.763 -29.391  1.00  0.52           C  \\nATOM   1950  O   TYR A 252      22.937 -10.837 -31.513  1.00  0.53           O  \\nATOM   1951  CG  TYR A 252      19.742 -10.272 -28.502  1.00  0.50           C  \\nATOM   1952  CD1 TYR A 252      19.771  -8.990 -27.958  1.00  0.47           C  \\nATOM   1953  CD2 TYR A 252      18.656 -11.089 -28.204  1.00  0.48           C  \\nATOM   1954  CE1 TYR A 252      18.745  -8.534 -27.137  1.00  0.48           C  \\nATOM   1955  CE2 TYR A 252      17.625 -10.643 -27.384  1.00  0.50           C  \\nATOM   1956  OH  TYR A 252      16.660  -8.919 -26.044  1.00  0.47           O  \\nATOM   1957  CZ  TYR A 252      17.678  -9.366 -26.856  1.00  0.42           C  \\nATOM   1958  N   ASP A 253      22.073  -9.449 -32.516  1.00  0.55           N  \\nATOM   1959  CA  ASP A 253      23.167  -8.651 -33.061  1.00  0.55           C  \\nATOM   1960  C   ASP A 253      24.113  -8.189 -31.955  1.00  0.55           C  \\nATOM   1961  CB  ASP A 253      22.622  -7.443 -33.825  1.00  0.52           C  \\nATOM   1962  O   ASP A 253      23.776  -7.296 -31.175  1.00  0.53           O  \\nATOM   1963  CG  ASP A 253      23.529  -7.001 -34.960  1.00  0.52           C  \\nATOM   1964  OD1 ASP A 253      24.694  -7.449 -35.020  1.00  0.52           O  \\nATOM   1965  OD2 ASP A 253      23.076  -6.194 -35.800  1.00  0.57           O  \\nATOM   1966  N   ASP A 254      25.117  -8.949 -31.440  1.00  0.53           N  \\nATOM   1967  CA  ASP A 254      26.439  -8.870 -30.827  1.00  0.53           C  \\nATOM   1968  C   ASP A 254      27.271  -7.756 -31.458  1.00  0.53           C  \\nATOM   1969  CB  ASP A 254      27.170 -10.208 -30.951  1.00  0.51           C  \\nATOM   1970  O   ASP A 254      28.287  -8.023 -32.104  1.00  0.52           O  \\nATOM   1971  CG  ASP A 254      27.420 -10.874 -29.610  1.00  0.51           C  \\nATOM   1972  OD1 ASP A 254      27.381 -10.183 -28.569  1.00  0.52           O  \\nATOM   1973  OD2 ASP A 254      27.663 -12.100 -29.593  1.00  0.56           O  \\nATOM   1974  N   ARG A 255      26.728  -6.623 -31.811  1.00  0.55           N  \\nATOM   1975  CA  ARG A 255      27.563  -5.480 -32.165  1.00  0.54           C  \\nATOM   1976  C   ARG A 255      27.672  -4.500 -31.001  1.00  0.54           C  \\nATOM   1977  CB  ARG A 255      27.006  -4.767 -33.399  1.00  0.51           C  \\nATOM   1978  O   ARG A 255      28.151  -3.377 -31.170  1.00  0.52           O  \\nATOM   1979  CG  ARG A 255      27.201  -5.537 -34.695  1.00  0.49           C  \\nATOM   1980  CD  ARG A 255      26.691  -4.756 -35.898  1.00  0.50           C  \\nATOM   1981  NE  ARG A 255      25.617  -5.466 -36.587  1.00  0.44           N  \\nATOM   1982  NH1 ARG A 255      25.381  -3.901 -38.269  1.00  0.34           N  \\nATOM   1983  NH2 ARG A 255      24.054  -5.769 -38.241  1.00  0.30           N  \\nATOM   1984  CZ  ARG A 255      25.020  -5.044 -37.698  1.00  0.44           C  \\nATOM   1985  N   PHE A 256      27.488  -4.946 -29.786  1.00  0.53           N  \\nATOM   1986  CA  PHE A 256      28.034  -4.226 -28.642  1.00  0.54           C  \\nATOM   1987  C   PHE A 256      29.162  -5.019 -27.991  1.00  0.53           C  \\nATOM   1988  CB  PHE A 256      26.936  -3.935 -27.614  1.00  0.49           C  \\nATOM   1989  O   PHE A 256      29.286  -5.040 -26.765  1.00  0.50           O  \\nATOM   1990  CG  PHE A 256      26.198  -2.647 -27.863  1.00  0.47           C  \\nATOM   1991  CD1 PHE A 256      26.705  -1.439 -27.401  1.00  0.44           C  \\nATOM   1992  CD2 PHE A 256      24.997  -2.646 -28.560  1.00  0.45           C  \\nATOM   1993  CE1 PHE A 256      26.024  -0.246 -27.630  1.00  0.45           C  \\nATOM   1994  CE2 PHE A 256      24.310  -1.457 -28.792  1.00  0.47           C  \\nATOM   1995  CZ  PHE A 256      24.825  -0.258 -28.326  1.00  0.40           C  \\nATOM   1996  N   ARG A 257      29.958  -5.782 -28.781  1.00  0.55           N  \\nATOM   1997  CA  ARG A 257      31.232  -6.275 -28.268  1.00  0.55           C  \\nATOM   1998  C   ARG A 257      32.343  -6.097 -29.297  1.00  0.55           C  \\nATOM   1999  CB  ARG A 257      31.118  -7.748 -27.870  1.00  0.51           C  \\nATOM   2000  O   ARG A 257      32.510  -6.931 -30.189  1.00  0.52           O  \\nATOM   2001  CG  ARG A 257      30.701  -7.964 -26.424  1.00  0.50           C  \\nATOM   2002  CD  ARG A 257      30.689  -9.441 -26.055  1.00  0.52           C  \\nATOM   2003  NE  ARG A 257      30.453  -9.639 -24.628  1.00  0.45           N  \\nATOM   2004  NH1 ARG A 257      30.466 -11.947 -24.735  1.00  0.35           N  \\nATOM   2005  NH2 ARG A 257      30.137 -10.889 -22.728  1.00  0.31           N  \\nATOM   2006  CZ  ARG A 257      30.353 -10.825 -24.033  1.00  0.46           C  \\nATOM   2007  N   GLU A 258      32.661  -4.877 -29.729  1.00  0.53           N  \\nATOM   2008  CA  GLU A 258      33.960  -4.546 -30.309  1.00  0.54           C  \\nATOM   2009  C   GLU A 258      34.261  -3.056 -30.174  1.00  0.53           C  \\nATOM   2010  CB  GLU A 258      34.014  -4.963 -31.781  1.00  0.50           C  \\nATOM   2011  O   GLU A 258      33.796  -2.249 -30.981  1.00  0.51           O  \\nATOM   2012  CG  GLU A 258      34.723  -6.289 -32.019  1.00  0.50           C  \\nATOM   2013  CD  GLU A 258      34.663  -6.750 -33.466  1.00  0.53           C  \\nATOM   2014  OE1 GLU A 258      33.646  -6.483 -34.146  1.00  0.54           O  \\nATOM   2015  OE2 GLU A 258      35.640  -7.384 -33.924  1.00  0.50           O  \\nATOM   2016  N   GLY A 259      34.593  -2.666 -28.859  1.00  0.49           N  \\nATOM   2017  CA  GLY A 259      35.418  -1.472 -28.760  1.00  0.50           C  \\nATOM   2018  C   GLY A 259      36.450  -1.550 -27.651  1.00  0.49           C  \\nATOM   2019  O   GLY A 259      36.110  -1.820 -26.497  1.00  0.46           O  \\nATOM   2020  N   ASP A 260      37.626  -2.400 -27.892  1.00  0.47           N  \\nATOM   2021  CA  ASP A 260      38.985  -1.870 -27.819  1.00  0.49           C  \\nATOM   2022  C   ASP A 260      39.625  -2.185 -26.469  1.00  0.47           C  \\nATOM   2023  CB  ASP A 260      38.986  -0.360 -28.063  1.00  0.43           C  \\nATOM   2024  O   ASP A 260      39.156  -1.717 -25.430  1.00  0.45           O  \\nATOM   2025  CG  ASP A 260      40.274   0.138 -28.695  1.00  0.44           C  \\nATOM   2026  OD1 ASP A 260      41.148  -0.689 -29.031  1.00  0.45           O  \\nATOM   2027  OD2 ASP A 260      40.417   1.370 -28.855  1.00  0.49           O  \\nATOM   2028  N   ARG A 261      40.274  -3.344 -26.364  1.00  0.53           N  \\nATOM   2029  CA  ARG A 261      41.714  -3.555 -26.251  1.00  0.54           C  \\nATOM   2030  C   ARG A 261      42.328  -2.607 -25.227  1.00  0.52           C  \\nATOM   2031  CB  ARG A 261      42.393  -3.371 -27.610  1.00  0.50           C  \\nATOM   2032  O   ARG A 261      42.540  -1.427 -25.515  1.00  0.50           O  \\nATOM   2033  CG  ARG A 261      42.182  -4.533 -28.567  1.00  0.51           C  \\nATOM   2034  CD  ARG A 261      42.958  -4.344 -29.863  1.00  0.54           C  \\nATOM   2035  NE  ARG A 261      42.577  -5.332 -30.869  1.00  0.46           N  \\nATOM   2036  NH1 ARG A 261      44.295  -4.818 -32.324  1.00  0.35           N  \\nATOM   2037  NH2 ARG A 261      42.787  -6.460 -32.858  1.00  0.29           N  \\nATOM   2038  CZ  ARG A 261      43.221  -5.534 -32.015  1.00  0.44           C  \\nATOM   2039  N   GLY A 262      42.154  -2.900 -23.887  1.00  0.50           N  \\nATOM   2040  CA  GLY A 262      43.246  -2.548 -22.994  1.00  0.51           C  \\nATOM   2041  C   GLY A 262      42.773  -2.021 -21.652  1.00  0.50           C  \\nATOM   2042  O   GLY A 262      42.635  -0.810 -21.468  1.00  0.48           O  \\nATOM   2043  N   GLY A 263      41.885  -2.781 -20.990  1.00  0.47           N  \\nATOM   2044  CA  GLY A 263      41.885  -2.603 -19.547  1.00  0.49           C  \\nATOM   2045  C   GLY A 263      42.130  -3.892 -18.786  1.00  0.48           C  \\nATOM   2046  O   GLY A 263      41.648  -4.955 -19.184  1.00  0.47           O  \\nATOM   2047  N   PRO A 264      43.218  -4.003 -18.201  1.00  0.53           N  \\nATOM   2048  CA  PRO A 264      43.436  -4.991 -17.141  1.00  0.53           C  \\nATOM   2049  C   PRO A 264      42.131  -5.532 -16.562  1.00  0.53           C  \\nATOM   2050  CB  PRO A 264      44.219  -4.207 -16.085  1.00  0.50           C  \\nATOM   2051  O   PRO A 264      41.140  -4.803 -16.474  1.00  0.51           O  \\nATOM   2052  CG  PRO A 264      43.783  -2.788 -16.260  1.00  0.49           C  \\nATOM   2053  CD  PRO A 264      43.373  -2.589 -17.691  1.00  0.51           C  \\nATOM   2054  N   PRO A 265      41.727  -6.899 -16.693  1.00  0.55           N  \\nATOM   2055  CA  PRO A 265      40.614  -7.517 -15.970  1.00  0.55           C  \\nATOM   2056  C   PRO A 265      40.122  -6.665 -14.802  1.00  0.55           C  \\nATOM   2057  CB  PRO A 265      41.207  -8.837 -15.471  1.00  0.52           C  \\nATOM   2058  O   PRO A 265      40.808  -6.553 -13.783  1.00  0.53           O  \\nATOM   2059  CG  PRO A 265      42.682  -8.600 -15.425  1.00  0.50           C  \\nATOM   2060  CD  PRO A 265      43.008  -7.449 -16.333  1.00  0.51           C  \\nATOM   2061  N   GLY A 266      39.324  -5.650 -14.894  1.00  0.45           N  \\nATOM   2062  CA  GLY A 266      38.557  -4.670 -14.141  1.00  0.46           C  \\nATOM   2063  C   GLY A 266      37.152  -5.137 -13.810  1.00  0.45           C  \\nATOM   2064  O   GLY A 266      36.342  -5.371 -14.710  1.00  0.44           O  \\nATOM   2065  N   GLY A 267      36.984  -5.935 -12.618  1.00  0.49           N  \\nATOM   2066  CA  GLY A 267      36.016  -6.143 -11.553  1.00  0.50           C  \\nATOM   2067  C   GLY A 267      34.657  -5.539 -11.856  1.00  0.49           C  \\nATOM   2068  O   GLY A 267      34.558  -4.354 -12.181  1.00  0.48           O  \\nATOM   2069  N   SER A 268      33.814  -6.181 -12.697  1.00  0.53           N  \\nATOM   2070  CA  SER A 268      32.394  -5.888 -12.860  1.00  0.54           C  \\nATOM   2071  C   SER A 268      31.901  -4.923 -11.786  1.00  0.52           C  \\nATOM   2072  CB  SER A 268      31.571  -7.176 -12.815  1.00  0.50           C  \\nATOM   2073  O   SER A 268      32.171  -5.118 -10.599  1.00  0.51           O  \\nATOM   2074  OG  SER A 268      30.187  -6.893 -12.923  1.00  0.49           O  \\nATOM   2075  N   ARG A 269      31.899  -3.565 -11.942  1.00  0.52           N  \\nATOM   2076  CA  ARG A 269      31.395  -2.522 -11.054  1.00  0.51           C  \\nATOM   2077  C   ARG A 269      30.289  -3.059 -10.152  1.00  0.52           C  \\nATOM   2078  CB  ARG A 269      30.880  -1.330 -11.862  1.00  0.49           C  \\nATOM   2079  O   ARG A 269      29.973  -2.458  -9.123  1.00  0.50           O  \\nATOM   2080  CG  ARG A 269      31.806  -0.124 -11.840  1.00  0.47           C  \\nATOM   2081  CD  ARG A 269      31.232   1.042 -12.633  1.00  0.49           C  \\nATOM   2082  NE  ARG A 269      32.131   2.193 -12.626  1.00  0.43           N  \\nATOM   2083  NH1 ARG A 269      30.761   3.534 -13.915  1.00  0.33           N  \\nATOM   2084  NH2 ARG A 269      32.774   4.331 -13.164  1.00  0.28           N  \\nATOM   2085  CZ  ARG A 269      31.887   3.350 -13.235  1.00  0.44           C  \\nATOM   2086  N   PHE A 270      29.879  -4.241 -10.165  1.00  0.49           N  \\nATOM   2087  CA  PHE A 270      29.106  -4.957  -9.157  1.00  0.50           C  \\nATOM   2088  C   PHE A 270      30.020  -5.786  -8.262  1.00  0.49           C  \\nATOM   2089  CB  PHE A 270      28.061  -5.860  -9.820  1.00  0.46           C  \\nATOM   2090  O   PHE A 270      29.695  -6.040  -7.100  1.00  0.47           O  \\nATOM   2091  CG  PHE A 270      26.651  -5.348  -9.698  1.00  0.44           C  \\nATOM   2092  CD1 PHE A 270      25.894  -5.618  -8.565  1.00  0.41           C  \\nATOM   2093  CD2 PHE A 270      26.083  -4.595 -10.717  1.00  0.43           C  \\nATOM   2094  CE1 PHE A 270      24.589  -5.145  -8.449  1.00  0.43           C  \\nATOM   2095  CE2 PHE A 270      24.780  -4.119 -10.609  1.00  0.45           C  \\nATOM   2096  CZ  PHE A 270      24.034  -4.396  -9.475  1.00  0.38           C  \\nATOM   2097  N   ASP A 271      31.299  -6.136  -8.686  1.00  0.44           N  \\nATOM   2098  CA  ASP A 271      32.333  -6.704  -7.827  1.00  0.44           C  \\nATOM   2099  C   ASP A 271      33.105  -5.607  -7.097  1.00  0.44           C  \\nATOM   2100  CB  ASP A 271      33.294  -7.570  -8.643  1.00  0.42           C  \\nATOM   2101  O   ASP A 271      33.564  -5.808  -5.971  1.00  0.42           O  \\nATOM   2102  CG  ASP A 271      32.651  -8.846  -9.158  1.00  0.42           C  \\nATOM   2103  OD1 ASP A 271      31.917  -9.510  -8.395  1.00  0.45           O  \\nATOM   2104  OD2 ASP A 271      32.884  -9.192 -10.337  1.00  0.47           O  \\nATOM   2105  N   GLN A 272      32.999  -4.352  -7.505  1.00  0.45           N  \\nATOM   2106  CA  GLN A 272      33.639  -3.324  -6.691  1.00  0.43           C  \\nATOM   2107  C   GLN A 272      32.657  -2.727  -5.687  1.00  0.44           C  \\nATOM   2108  CB  GLN A 272      34.219  -2.221  -7.578  1.00  0.42           C  \\nATOM   2109  O   GLN A 272      33.049  -2.338  -4.585  1.00  0.42           O  \\nATOM   2110  CG  GLN A 272      35.593  -2.548  -8.147  1.00  0.40           C  \\nATOM   2111  CD  GLN A 272      36.141  -1.442  -9.029  1.00  0.43           C  \\nATOM   2112  NE2 GLN A 272      37.419  -1.541  -9.379  1.00  0.36           N  \\nATOM   2113  OE1 GLN A 272      35.422  -0.506  -9.393  1.00  0.45           O  \\nATOM   2114  N   VAL A 273      31.380  -3.022  -5.790  1.00  0.46           N  \\nATOM   2115  CA  VAL A 273      30.532  -2.721  -4.641  1.00  0.44           C  \\nATOM   2116  C   VAL A 273      30.412  -3.955  -3.750  1.00  0.46           C  \\nATOM   2117  CB  VAL A 273      29.130  -2.242  -5.079  1.00  0.43           C  \\nATOM   2118  O   VAL A 273      30.254  -3.837  -2.533  1.00  0.44           O  \\nATOM   2119  CG1 VAL A 273      28.257  -1.943  -3.862  1.00  0.41           C  \\nATOM   2120  CG2 VAL A 273      29.244  -1.010  -5.976  1.00  0.42           C  \\nATOM   2121  N   LYS A 274      30.837  -5.147  -4.212  1.00  0.46           N  \\nATOM   2122  CA  LYS A 274      31.123  -6.263  -3.315  1.00  0.44           C  \\nATOM   2123  C   LYS A 274      32.594  -6.280  -2.910  1.00  0.46           C  \\nATOM   2124  CB  LYS A 274      30.746  -7.592  -3.972  1.00  0.43           C  \\nATOM   2125  O   LYS A 274      32.947  -6.805  -1.852  1.00  0.43           O  \\nATOM   2126  CG  LYS A 274      29.471  -8.214  -3.423  1.00  0.41           C  \\nATOM   2127  CD  LYS A 274      29.171  -9.552  -4.087  1.00  0.45           C  \\nATOM   2128  CE  LYS A 274      27.897 -10.178  -3.535  1.00  0.38           C  \\nATOM   2129  NZ  LYS A 274      27.542 -11.438  -4.255  1.00  0.38           N  \\nATOM   2130  N   ALA A 275      33.465  -5.700  -3.613  1.00  0.45           N  \\nATOM   2131  CA  ALA A 275      34.860  -5.744  -3.182  1.00  0.44           C  \\nATOM   2132  C   ALA A 275      35.204  -4.537  -2.314  1.00  0.45           C  \\nATOM   2133  CB  ALA A 275      35.789  -5.808  -4.391  1.00  0.42           C  \\nATOM   2134  O   ALA A 275      35.982  -4.650  -1.363  1.00  0.42           O  \\nATOM   2135  N   GLU A 276      34.440  -3.554  -2.377  1.00  0.45           N  \\nATOM   2136  CA  GLU A 276      34.737  -2.490  -1.422  1.00  0.45           C  \\nATOM   2137  C   GLU A 276      33.792  -2.544  -0.225  1.00  0.45           C  \\nATOM   2138  CB  GLU A 276      34.651  -1.120  -2.099  1.00  0.42           C  \\nATOM   2139  O   GLU A 276      34.075  -1.958   0.822  1.00  0.43           O  \\nATOM   2140  CG  GLU A 276      35.913  -0.726  -2.853  1.00  0.40           C  \\nATOM   2141  CD  GLU A 276      35.789   0.605  -3.577  1.00  0.45           C  \\nATOM   2142  OE1 GLU A 276      34.653   1.109  -3.731  1.00  0.46           O  \\nATOM   2143  OE2 GLU A 276      36.836   1.148  -3.995  1.00  0.42           O  \\nATOM   2144  N   LEU A 277      32.981  -3.574  -0.179  1.00  0.46           N  \\nATOM   2145  CA  LEU A 277      32.485  -4.054   1.106  1.00  0.45           C  \\nATOM   2146  C   LEU A 277      33.315  -5.233   1.603  1.00  0.46           C  \\nATOM   2147  CB  LEU A 277      31.013  -4.461   0.995  1.00  0.43           C  \\nATOM   2148  O   LEU A 277      32.874  -5.984   2.475  1.00  0.44           O  \\nATOM   2149  CG  LEU A 277      29.984  -3.355   1.233  1.00  0.41           C  \\nATOM   2150  CD1 LEU A 277      28.711  -3.636   0.441  1.00  0.38           C  \\nATOM   2151  CD2 LEU A 277      29.676  -3.223   2.721  1.00  0.40           C  \\nATOM   2152  N   LYS A 278      34.383  -5.583   0.945  1.00  0.47           N  \\nATOM   2153  CA  LYS A 278      35.392  -6.431   1.572  1.00  0.47           C  \\nATOM   2154  C   LYS A 278      36.608  -5.615   2.001  1.00  0.47           C  \\nATOM   2155  CB  LYS A 278      35.821  -7.549   0.620  1.00  0.44           C  \\nATOM   2156  O   LYS A 278      37.476  -6.114   2.719  1.00  0.44           O  \\nATOM   2157  CG  LYS A 278      35.107  -8.872   0.857  1.00  0.43           C  \\nATOM   2158  CD  LYS A 278      35.584  -9.945  -0.113  1.00  0.47           C  \\nATOM   2159  CE  LYS A 278      34.907 -11.283   0.156  1.00  0.40           C  \\nATOM   2160  NZ  LYS A 278      35.376 -12.341  -0.788  1.00  0.38           N  \\nATOM   2161  N   GLU A 279      36.510  -4.403   2.420  1.00  0.49           N  \\nATOM   2162  CA  GLU A 279      37.608  -3.816   3.181  1.00  0.49           C  \\nATOM   2163  C   GLU A 279      37.117  -2.679   4.073  1.00  0.48           C  \\nATOM   2164  CB  GLU A 279      38.704  -3.309   2.239  1.00  0.45           C  \\nATOM   2165  O   GLU A 279      36.689  -1.635   3.577  1.00  0.46           O  \\nATOM   2166  CG  GLU A 279      39.626  -4.404   1.723  1.00  0.45           C  \\nATOM   2167  CD  GLU A 279      40.704  -3.887   0.784  1.00  0.49           C  \\nATOM   2168  OE1 GLU A 279      40.531  -2.790   0.206  1.00  0.50           O  \\nATOM   2169  OE2 GLU A 279      41.730  -4.586   0.624  1.00  0.45           O  \\nATOM   2170  N   GLU A 280      36.213  -3.038   4.984  1.00  0.51           N  \\nATOM   2171  CA  GLU A 280      36.273  -2.485   6.334  1.00  0.52           C  \\nATOM   2172  C   GLU A 280      35.286  -3.187   7.263  1.00  0.51           C  \\nATOM   2173  CB  GLU A 280      35.994  -0.980   6.311  1.00  0.48           C  \\nATOM   2174  O   GLU A 280      34.111  -2.819   7.321  1.00  0.49           O  \\nATOM   2175  CG  GLU A 280      37.161  -0.131   6.795  1.00  0.46           C  \\nATOM   2176  CD  GLU A 280      36.898   1.363   6.695  1.00  0.50           C  \\nATOM   2177  OE1 GLU A 280      35.921   1.764   6.021  1.00  0.51           O  \\nATOM   2178  OE2 GLU A 280      37.675   2.140   7.295  1.00  0.47           O  \\nATOM   2179  N   ILE A 281      35.223  -4.529   7.198  1.00  0.52           N  \\nATOM   2180  CA  ILE A 281      34.979  -5.199   8.470  1.00  0.52           C  \\nATOM   2181  C   ILE A 281      36.056  -4.802   9.478  1.00  0.52           C  \\nATOM   2182  CB  ILE A 281      34.942  -6.734   8.303  1.00  0.49           C  \\nATOM   2183  O   ILE A 281      37.177  -5.314   9.431  1.00  0.50           O  \\nATOM   2184  CG1 ILE A 281      33.832  -7.138   7.326  1.00  0.46           C  \\nATOM   2185  CG2 ILE A 281      34.756  -7.421   9.659  1.00  0.47           C  \\nATOM   2186  CD1 ILE A 281      33.848  -8.612   6.945  1.00  0.45           C  \\nATOM   2187  N   VAL A 282      36.169  -3.501   9.792  1.00  0.54           N  \\nATOM   2188  CA  VAL A 282      36.747  -3.216  11.101  1.00  0.55           C  \\nATOM   2189  C   VAL A 282      36.133  -4.142  12.149  1.00  0.54           C  \\nATOM   2190  CB  VAL A 282      36.537  -1.740  11.506  1.00  0.51           C  \\nATOM   2191  O   VAL A 282      34.914  -4.160  12.333  1.00  0.52           O  \\nATOM   2192  CG1 VAL A 282      37.257  -1.432  12.818  1.00  0.48           C  \\nATOM   2193  CG2 VAL A 282      37.021  -0.809  10.396  1.00  0.49           C  \\nATOM   2194  N   GLU A 283      36.814  -5.327  12.330  1.00  0.58           N  \\nATOM   2195  CA  GLU A 283      36.814  -6.164  13.526  1.00  0.59           C  \\nATOM   2196  C   GLU A 283      36.070  -5.487  14.673  1.00  0.59           C  \\nATOM   2197  CB  GLU A 283      38.247  -6.495  13.951  1.00  0.55           C  \\nATOM   2198  O   GLU A 283      36.370  -4.344  15.026  1.00  0.57           O  \\nATOM   2199  CG  GLU A 283      38.942  -7.498  13.042  1.00  0.54           C  \\nATOM   2200  CD  GLU A 283      40.187  -8.110  13.664  1.00  0.56           C  \\nATOM   2201  OE1 GLU A 283      40.803  -7.469  14.546  1.00  0.56           O  \\nATOM   2202  OE2 GLU A 283      40.550  -9.239  13.267  1.00  0.51           O  \\nATOM   2203  N   PRO A 284      34.782  -5.674  14.882  1.00  0.58           N  \\nATOM   2204  CA  PRO A 284      34.278  -5.424  16.234  1.00  0.57           C  \\nATOM   2205  C   PRO A 284      35.269  -5.837  17.320  1.00  0.58           C  \\nATOM   2206  CB  PRO A 284      33.009  -6.277  16.300  1.00  0.54           C  \\nATOM   2207  O   PRO A 284      35.594  -7.021  17.446  1.00  0.56           O  \\nATOM   2208  CG  PRO A 284      33.271  -7.424  15.379  1.00  0.52           C  \\nATOM   2209  CD  PRO A 284      34.248  -6.978  14.329  1.00  0.54           C  \\nATOM   2210  N   ASN A 285      36.199  -4.957  17.453  1.00  0.57           N  \\nATOM   2211  CA  ASN A 285      36.892  -5.016  18.735  1.00  0.59           C  \\nATOM   2212  C   ASN A 285      35.980  -5.538  19.842  1.00  0.57           C  \\nATOM   2213  CB  ASN A 285      37.445  -3.639  19.110  1.00  0.53           C  \\nATOM   2214  O   ASN A 285      35.021  -4.868  20.230  1.00  0.55           O  \\nATOM   2215  CG  ASN A 285      38.495  -3.709  20.201  1.00  0.52           C  \\nATOM   2216  ND2 ASN A 285      38.872  -2.554  20.736  1.00  0.51           N  \\nATOM   2217  OD1 ASN A 285      38.964  -4.792  20.560  1.00  0.52           O  \\nATOM   2218  N   ASN A 286      35.871  -6.822  19.921  1.00  0.58           N  \\nATOM   2219  CA  ASN A 286      35.836  -7.717  21.073  1.00  0.59           C  \\nATOM   2220  C   ASN A 286      36.358  -7.031  22.332  1.00  0.57           C  \\nATOM   2221  CB  ASN A 286      36.638  -8.989  20.789  1.00  0.53           C  \\nATOM   2222  O   ASN A 286      37.560  -7.056  22.604  1.00  0.55           O  \\nATOM   2223  CG  ASN A 286      35.754 -10.180  20.477  1.00  0.52           C  \\nATOM   2224  ND2 ASN A 286      36.364 -11.272  20.033  1.00  0.52           N  \\nATOM   2225  OD1 ASN A 286      34.531 -10.119  20.634  1.00  0.53           O  \\nATOM   2226  N   GLY A 287      35.687  -5.966  22.674  1.00  0.59           N  \\nATOM   2227  CA  GLY A 287      35.809  -5.721  24.102  1.00  0.59           C  \\nATOM   2228  C   GLY A 287      34.640  -6.260  24.903  1.00  0.58           C  \\nATOM   2229  O   GLY A 287      33.490  -6.179  24.465  1.00  0.55           O  \\nATOM   2230  N   PRO A 288      35.008  -7.325  25.689  1.00  0.61           N  \\nATOM   2231  CA  PRO A 288      34.079  -7.883  26.675  1.00  0.60           C  \\nATOM   2232  C   PRO A 288      33.074  -6.853  27.187  1.00  0.60           C  \\nATOM   2233  CB  PRO A 288      35.001  -8.353  27.802  1.00  0.56           C  \\nATOM   2234  O   PRO A 288      33.422  -5.683  27.369  1.00  0.57           O  \\nATOM   2235  CG  PRO A 288      36.188  -7.448  27.728  1.00  0.55           C  \\nATOM   2236  CD  PRO A 288      36.352  -6.992  26.307  1.00  0.57           C  \\nATOM   2237  N   PRO A 289      31.805  -7.000  26.887  1.00  0.62           N  \\nATOM   2238  CA  PRO A 289      30.768  -6.288  27.637  1.00  0.61           C  \\nATOM   2239  C   PRO A 289      31.232  -5.866  29.029  1.00  0.62           C  \\nATOM   2240  CB  PRO A 289      29.633  -7.311  27.726  1.00  0.58           C  \\nATOM   2241  O   PRO A 289      31.582  -6.716  29.851  1.00  0.60           O  \\nATOM   2242  CG  PRO A 289      30.314  -8.641  27.707  1.00  0.56           C  \\nATOM   2243  CD  PRO A 289      31.614  -8.501  26.968  1.00  0.57           C  \\nATOM   2244  N   THR A 290      31.786  -4.744  29.191  1.00  0.60           N  \\nATOM   2245  CA  THR A 290      31.776  -4.131  30.514  1.00  0.60           C  \\nATOM   2246  C   THR A 290      30.349  -3.997  31.037  1.00  0.58           C  \\nATOM   2247  CB  THR A 290      32.451  -2.747  30.494  1.00  0.56           C  \\nATOM   2248  O   THR A 290      29.522  -3.314  30.429  1.00  0.56           O  \\nATOM   2249  CG2 THR A 290      33.858  -2.813  31.078  1.00  0.52           C  \\nATOM   2250  OG1 THR A 290      32.528  -2.279  29.142  1.00  0.55           O  \\nATOM   2251  N   THR A 291      29.916  -5.004  31.776  1.00  0.63           N  \\nATOM   2252  CA  THR A 291      29.018  -5.148  32.916  1.00  0.63           C  \\nATOM   2253  C   THR A 291      29.143  -3.954  33.858  1.00  0.61           C  \\nATOM   2254  CB  THR A 291      29.304  -6.448  33.691  1.00  0.59           C  \\nATOM   2255  O   THR A 291      30.151  -3.808  34.553  1.00  0.58           O  \\nATOM   2256  CG2 THR A 291      28.505  -7.615  33.121  1.00  0.57           C  \\nATOM   2257  OG1 THR A 291      30.702  -6.752  33.603  1.00  0.60           O  \\nATOM   2258  N   SER A 292      28.627  -2.891  33.519  1.00  0.55           N  \\nATOM   2259  CA  SER A 292      28.298  -1.985  34.614  1.00  0.56           C  \\nATOM   2260  C   SER A 292      26.793  -1.930  34.853  1.00  0.54           C  \\nATOM   2261  CB  SER A 292      28.828  -0.580  34.324  1.00  0.51           C  \\nATOM   2262  O   SER A 292      26.021  -1.673  33.927  1.00  0.51           O  \\nATOM   2263  OG  SER A 292      28.192  -0.027  33.184  1.00  0.53           O  \\nATOM   2264  N   SER A 293      26.422  -2.658  35.857  1.00  0.55           N  \\nATOM   2265  CA  SER A 293      25.585  -2.599  37.051  1.00  0.56           C  \\nATOM   2266  C   SER A 293      24.817  -1.284  37.123  1.00  0.54           C  \\nATOM   2267  CB  SER A 293      26.435  -2.772  38.311  1.00  0.51           C  \\nATOM   2268  O   SER A 293      25.395  -0.237  37.420  1.00  0.52           O  \\nATOM   2269  OG  SER A 293      26.476  -1.570  39.059  1.00  0.56           O  \\nATOM   2270  N   GLY A 294      23.743  -1.183  36.278  1.00  0.49           N  \\nATOM   2271  CA  GLY A 294      22.728  -0.299  36.828  1.00  0.50           C  \\nATOM   2272  C   GLY A 294      21.314  -0.801  36.601  1.00  0.50           C  \\nATOM   2273  O   GLY A 294      20.878  -0.950  35.458  1.00  0.47           O  \\nATOM   2274  N   ALA A 295      20.838  -1.818  37.362  1.00  0.49           N  \\nATOM   2275  CA  ALA A 295      19.691  -2.277  38.140  1.00  0.51           C  \\nATOM   2276  C   ALA A 295      18.585  -1.226  38.159  1.00  0.49           C  \\nATOM   2277  CB  ALA A 295      20.118  -2.621  39.565  1.00  0.48           C  \\nATOM   2278  O   ALA A 295      18.727  -0.177  38.792  1.00  0.47           O  \\nATOM   2279  N   ALA A 296      18.047  -0.831  36.963  1.00  0.50           N  \\nATOM   2280  CA  ALA A 296      16.686  -0.325  37.125  1.00  0.51           C  \\nATOM   2281  C   ALA A 296      16.115   0.148  35.792  1.00  0.49           C  \\nATOM   2282  CB  ALA A 296      16.658   0.809  38.146  1.00  0.47           C  \\nATOM   2283  O   ALA A 296      14.949   0.542  35.714  1.00  0.47           O  \\nATOM   2284  N   ALA A 297      16.465  -0.512  34.619  1.00  0.48           N  \\nATOM   2285  CA  ALA A 297      15.645  -0.291  33.431  1.00  0.48           C  \\nATOM   2286  C   ALA A 297      15.660  -1.515  32.519  1.00  0.47           C  \\nATOM   2287  CB  ALA A 297      16.132   0.941  32.671  1.00  0.45           C  \\nATOM   2288  O   ALA A 297      15.277  -1.430  31.350  1.00  0.46           O  \\nATOM   2289  N   ALA A 298      16.139  -2.681  33.043  1.00  0.44           N  \\nATOM   2290  CA  ALA A 298      16.181  -3.988  32.394  1.00  0.43           C  \\nATOM   2291  C   ALA A 298      14.911  -4.785  32.681  1.00  0.43           C  \\nATOM   2292  CB  ALA A 298      17.410  -4.770  32.851  1.00  0.41           C  \\nATOM   2293  O   ALA A 298      14.668  -5.822  32.059  1.00  0.42           O  \\nATOM   2294  N   SER A 299      13.920  -4.226  33.301  1.00  0.44           N  \\nATOM   2295  CA  SER A 299      12.731  -5.028  33.573  1.00  0.45           C  \\nATOM   2296  C   SER A 299      11.592  -4.667  32.626  1.00  0.44           C  \\nATOM   2297  CB  SER A 299      12.280  -4.842  35.022  1.00  0.42           C  \\nATOM   2298  O   SER A 299      10.747  -5.508  32.314  1.00  0.43           O  \\nATOM   2299  OG  SER A 299      11.203  -5.711  35.329  1.00  0.43           O  \\nATOM   2300  N   ALA A 300      11.810  -3.782  31.677  1.00  0.45           N  \\nATOM   2301  CA  ALA A 300      10.673  -3.500  30.806  1.00  0.45           C  \\nATOM   2302  C   ALA A 300      11.014  -3.790  29.347  1.00  0.45           C  \\nATOM   2303  CB  ALA A 300      10.226  -2.049  30.967  1.00  0.42           C  \\nATOM   2304  O   ALA A 300      10.134  -4.128  28.552  1.00  0.43           O  \\nATOM   2305  N   ALA A 301      12.262  -4.064  29.018  1.00  0.44           N  \\nATOM   2306  CA  ALA A 301      12.732  -4.493  27.704  1.00  0.43           C  \\nATOM   2307  C   ALA A 301      12.754  -6.016  27.600  1.00  0.44           C  \\nATOM   2308  CB  ALA A 301      14.120  -3.924  27.423  1.00  0.42           C  \\nATOM   2309  O   ALA A 301      12.578  -6.572  26.513  1.00  0.42           O  \\nATOM   2310  N   ALA A 302      12.716  -6.739  28.741  1.00  0.43           N  \\nATOM   2311  CA  ALA A 302      12.759  -8.197  28.802  1.00  0.43           C  \\nATOM   2312  C   ALA A 302      11.354  -8.790  28.755  1.00  0.43           C  \\nATOM   2313  CB  ALA A 302      13.485  -8.655  30.065  1.00  0.42           C  \\nATOM   2314  O   ALA A 302      11.160  -9.905  28.265  1.00  0.42           O  \\nATOM   2315  N   VAL A 303      10.277  -7.959  28.962  1.00  0.44           N  \\nATOM   2316  CA  VAL A 303       8.924  -8.506  28.953  1.00  0.43           C  \\nATOM   2317  C   VAL A 303       8.288  -8.288  27.582  1.00  0.44           C  \\nATOM   2318  CB  VAL A 303       8.048  -7.869  30.055  1.00  0.41           C  \\nATOM   2319  O   VAL A 303       7.428  -9.065  27.160  1.00  0.42           O  \\nATOM   2320  CG1 VAL A 303       6.642  -8.468  30.039  1.00  0.39           C  \\nATOM   2321  CG2 VAL A 303       8.696  -8.055  31.425  1.00  0.40           C  \\nATOM   2322  N   VAL A 304       8.891  -7.513  26.714  1.00  0.45           N  \\nATOM   2323  CA  VAL A 304       8.337  -7.450  25.365  1.00  0.44           C  \\nATOM   2324  C   VAL A 304       9.118  -8.382  24.442  1.00  0.45           C  \\nATOM   2325  CB  VAL A 304       8.359  -6.008  24.810  1.00  0.43           C  \\nATOM   2326  O   VAL A 304       8.564  -8.919  23.479  1.00  0.44           O  \\nATOM   2327  CG1 VAL A 304       7.725  -5.955  23.421  1.00  0.40           C  \\nATOM   2328  CG2 VAL A 304       7.641  -5.058  25.765  1.00  0.41           C  \\nATOM   2329  N   ALA A 305      10.275  -8.872  24.866  1.00  0.45           N  \\nATOM   2330  CA  ALA A 305      11.052  -9.881  24.149  1.00  0.45           C  \\nATOM   2331  C   ALA A 305      10.648 -11.289  24.575  1.00  0.45           C  \\nATOM   2332  CB  ALA A 305      12.546  -9.666  24.382  1.00  0.43           C  \\nATOM   2333  O   ALA A 305      10.726 -12.230  23.782  1.00  0.44           O  \\nATOM   2334  N   SER A 306       9.957 -11.475  25.676  1.00  0.44           N  \\nATOM   2335  CA  SER A 306       9.628 -12.825  26.122  1.00  0.44           C  \\nATOM   2336  C   SER A 306       8.172 -13.167  25.821  1.00  0.44           C  \\nATOM   2337  CB  SER A 306       9.896 -12.975  27.620  1.00  0.41           C  \\nATOM   2338  O   SER A 306       7.814 -14.342  25.719  1.00  0.42           O  \\nATOM   2339  OG  SER A 306       8.857 -12.382  28.379  1.00  0.42           O  \\nATOM   2340  N   LYS A 307       7.389 -12.143  25.293  1.00  0.48           N  \\nATOM   2341  CA  LYS A 307       6.040 -12.531  24.892  1.00  0.47           C  \\nATOM   2342  C   LYS A 307       5.862 -12.421  23.381  1.00  0.47           C  \\nATOM   2343  CB  LYS A 307       4.998 -11.668  25.606  1.00  0.44           C  \\nATOM   2344  O   LYS A 307       4.769 -12.651  22.860  1.00  0.45           O  \\nATOM   2345  CG  LYS A 307       4.573 -12.207  26.964  1.00  0.43           C  \\nATOM   2346  CD  LYS A 307       3.482 -11.348  27.590  1.00  0.47           C  \\nATOM   2347  CE  LYS A 307       3.036 -11.903  28.936  1.00  0.37           C  \\nATOM   2348  NZ  LYS A 307       1.978 -11.054  29.562  1.00  0.36           N  \\nATOM   2349  N   LEU A 308       6.985 -12.265  22.644  1.00  0.48           N  \\nATOM   2350  CA  LEU A 308       7.165 -12.755  21.282  1.00  0.48           C  \\nATOM   2351  C   LEU A 308       7.951 -14.061  21.274  1.00  0.47           C  \\nATOM   2352  CB  LEU A 308       7.882 -11.708  20.426  1.00  0.45           C  \\nATOM   2353  O   LEU A 308       9.184 -14.048  21.286  1.00  0.46           O  \\nATOM   2354  CG  LEU A 308       6.997 -10.653  19.760  1.00  0.44           C  \\nATOM   2355  CD1 LEU A 308       7.687  -9.294  19.777  1.00  0.42           C  \\nATOM   2356  CD2 LEU A 308       6.654 -11.067  18.333  1.00  0.43           C  \\nATOM   2357  N   GLY A 309       8.178 -14.785  22.335  1.00  0.42           N  \\nATOM   2358  CA  GLY A 309       8.827 -16.068  22.552  1.00  0.44           C  \\nATOM   2359  C   GLY A 309       7.873 -17.243  22.461  1.00  0.42           C  \\nATOM   2360  O   GLY A 309       6.952 -17.366  23.271  1.00  0.40           O  \\nATOM   2361  N   SER A 310       7.174 -17.457  21.299  1.00  0.41           N  \\nATOM   2362  CA  SER A 310       6.710 -18.786  20.915  1.00  0.46           C  \\nATOM   2363  C   SER A 310       5.633 -18.707  19.839  1.00  0.41           C  \\nATOM   2364  CB  SER A 310       6.172 -19.539  22.132  1.00  0.39           C  \\nATOM   2365  O   SER A 310       4.528 -18.223  20.094  1.00  0.40           O  \\nATOM   2366  OG  SER A 310       5.151 -20.446  21.752  1.00  0.39           O  \\nATOM   2367  N   ARG A 311       5.975 -18.021  18.609  1.00  0.50           N  \\nATOM   2368  CA  ARG A 311       5.501 -18.546  17.333  1.00  0.50           C  \\nATOM   2369  C   ARG A 311       6.192 -17.851  16.165  1.00  0.49           C  \\nATOM   2370  CB  ARG A 311       3.984 -18.384  17.215  1.00  0.46           C  \\nATOM   2371  O   ARG A 311       5.541 -17.181  15.361  1.00  0.47           O  \\nATOM   2372  CG  ARG A 311       3.192 -19.384  18.042  1.00  0.45           C  \\nATOM   2373  CD  ARG A 311       1.693 -19.243  17.815  1.00  0.48           C  \\nATOM   2374  NE  ARG A 311       0.943 -20.294  18.495  1.00  0.41           N  \\nATOM   2375  NH1 ARG A 311      -1.117 -19.637  17.683  1.00  0.31           N  \\nATOM   2376  NH2 ARG A 311      -0.955 -21.449  19.078  1.00  0.26           N  \\nATOM   2377  CZ  ARG A 311      -0.375 -20.458  18.417  1.00  0.39           C  \\nATOM   2378  N   THR A 312       7.603 -17.677  16.256  1.00  0.47           N  \\nATOM   2379  CA  THR A 312       8.331 -17.609  14.994  1.00  0.48           C  \\nATOM   2380  C   THR A 312       8.864 -18.984  14.602  1.00  0.47           C  \\nATOM   2381  CB  THR A 312       9.498 -16.608  15.076  1.00  0.44           C  \\nATOM   2382  O   THR A 312       9.741 -19.531  15.274  1.00  0.45           O  \\nATOM   2383  CG2 THR A 312       9.093 -15.244  14.527  1.00  0.42           C  \\nATOM   2384  OG1 THR A 312       9.898 -16.460  16.444  1.00  0.39           O  \\nATOM   2385  N   ARG A 313       8.024 -19.978  14.435  1.00  0.48           N  \\nATOM   2386  CA  ARG A 313       8.503 -21.070  13.594  1.00  0.48           C  \\nATOM   2387  C   ARG A 313       9.437 -20.553  12.504  1.00  0.48           C  \\nATOM   2388  CB  ARG A 313       7.327 -21.820  12.964  1.00  0.45           C  \\nATOM   2389  O   ARG A 313       9.036 -19.738  11.671  1.00  0.45           O  \\nATOM   2390  CG  ARG A 313       7.047 -23.172  13.600  1.00  0.43           C  \\nATOM   2391  CD  ARG A 313       5.895 -23.891  12.912  1.00  0.46           C  \\nATOM   2392  NE  ARG A 313       5.620 -25.185  13.531  1.00  0.42           N  \\nATOM   2393  NH1 ARG A 313       3.937 -25.764  12.059  1.00  0.31           N  \\nATOM   2394  NH2 ARG A 313       4.530 -27.195  13.749  1.00  0.26           N  \\nATOM   2395  CZ  ARG A 313       4.696 -26.045  13.111  1.00  0.39           C  \\nATOM   2396  N   ILE A 314      10.731 -20.357  12.755  1.00  0.48           N  \\nATOM   2397  CA  ILE A 314      11.772 -20.387  11.733  1.00  0.47           C  \\nATOM   2398  C   ILE A 314      11.521 -21.549  10.775  1.00  0.48           C  \\nATOM   2399  CB  ILE A 314      13.178 -20.506  12.363  1.00  0.44           C  \\nATOM   2400  O   ILE A 314      11.452 -22.706  11.196  1.00  0.46           O  \\nATOM   2401  CG1 ILE A 314      13.436 -19.335  13.318  1.00  0.41           C  \\nATOM   2402  CG2 ILE A 314      14.253 -20.572  11.274  1.00  0.42           C  \\nATOM   2403  CD1 ILE A 314      14.729 -19.458  14.113  1.00  0.39           C  \\nATOM   2404  N   VAL A 315      10.766 -21.379   9.835  1.00  0.46           N  \\nATOM   2405  CA  VAL A 315      10.574 -22.016   8.536  1.00  0.49           C  \\nATOM   2406  C   VAL A 315      11.908 -22.098   7.799  1.00  0.47           C  \\nATOM   2407  CB  VAL A 315       9.537 -21.255   7.679  1.00  0.44           C  \\nATOM   2408  O   VAL A 315      12.665 -21.125   7.760  1.00  0.45           O  \\nATOM   2409  CG1 VAL A 315       9.187 -22.051   6.423  1.00  0.42           C  \\nATOM   2410  CG2 VAL A 315       8.281 -20.962   8.497  1.00  0.42           C  \\nATOM   2411  N   TYR A 316      12.702 -23.300   7.859  1.00  0.52           N  \\nATOM   2412  CA  TYR A 316      13.586 -24.137   7.055  1.00  0.53           C  \\nATOM   2413  C   TYR A 316      15.016 -23.612   7.091  1.00  0.53           C  \\nATOM   2414  CB  TYR A 316      13.091 -24.206   5.607  1.00  0.49           C  \\nATOM   2415  O   TYR A 316      15.244 -22.404   6.983  1.00  0.51           O  \\nATOM   2416  CG  TYR A 316      11.784 -24.942   5.447  1.00  0.48           C  \\nATOM   2417  CD1 TYR A 316      11.748 -26.334   5.385  1.00  0.45           C  \\nATOM   2418  CD2 TYR A 316      10.582 -24.249   5.355  1.00  0.46           C  \\nATOM   2419  CE1 TYR A 316      10.546 -27.017   5.235  1.00  0.45           C  \\nATOM   2420  CE2 TYR A 316       9.373 -24.922   5.206  1.00  0.48           C  \\nATOM   2421  OH  TYR A 316       8.172 -26.974   4.999  1.00  0.39           O  \\nATOM   2422  CZ  TYR A 316       9.365 -26.303   5.147  1.00  0.41           C  \\nATOM   2423  N   PRO A 317      16.005 -24.143   7.738  1.00  0.53           N  \\nATOM   2424  CA  PRO A 317      17.450 -23.925   7.844  1.00  0.51           C  \\nATOM   2425  C   PRO A 317      18.043 -23.281   6.593  1.00  0.53           C  \\nATOM   2426  CB  PRO A 317      18.005 -25.337   8.047  1.00  0.48           C  \\nATOM   2427  O   PRO A 317      19.007 -22.516   6.686  1.00  0.51           O  \\nATOM   2428  CG  PRO A 317      16.838 -26.243   7.820  1.00  0.46           C  \\nATOM   2429  CD  PRO A 317      15.595 -25.406   7.731  1.00  0.48           C  \\nATOM   2430  N   ASP A 318      17.435 -23.289   5.307  1.00  0.56           N  \\nATOM   2431  CA  ASP A 318      17.885 -22.813   4.003  1.00  0.56           C  \\nATOM   2432  C   ASP A 318      17.251 -21.466   3.661  1.00  0.56           C  \\nATOM   2433  CB  ASP A 318      17.560 -23.838   2.915  1.00  0.52           C  \\nATOM   2434  O   ASP A 318      17.399 -20.971   2.542  1.00  0.54           O  \\nATOM   2435  CG  ASP A 318      18.281 -25.160   3.112  1.00  0.50           C  \\nATOM   2436  OD1 ASP A 318      19.404 -25.167   3.661  1.00  0.51           O  \\nATOM   2437  OD2 ASP A 318      17.722 -26.205   2.713  1.00  0.53           O  \\nATOM   2438  N   ASP A 319      16.525 -20.736   4.583  1.00  0.56           N  \\nATOM   2439  CA  ASP A 319      15.743 -19.517   4.399  1.00  0.57           C  \\nATOM   2440  C   ASP A 319      16.638 -18.280   4.431  1.00  0.56           C  \\nATOM   2441  CB  ASP A 319      14.657 -19.409   5.471  1.00  0.52           C  \\nATOM   2442  O   ASP A 319      16.220 -17.194   4.024  1.00  0.55           O  \\nATOM   2443  CG  ASP A 319      13.278 -19.142   4.894  1.00  0.51           C  \\nATOM   2444  OD1 ASP A 319      13.156 -18.964   3.663  1.00  0.51           O  \\nATOM   2445  OD2 ASP A 319      12.305 -19.105   5.678  1.00  0.53           O  \\nATOM   2446  N   HIS A 320      17.976 -18.394   4.752  1.00  0.61           N  \\nATOM   2447  CA  HIS A 320      18.894 -17.266   4.643  1.00  0.61           C  \\nATOM   2448  C   HIS A 320      19.365 -17.075   3.205  1.00  0.61           C  \\nATOM   2449  CB  HIS A 320      20.097 -17.464   5.567  1.00  0.57           C  \\nATOM   2450  O   HIS A 320      19.919 -16.028   2.862  1.00  0.60           O  \\nATOM   2451  CG  HIS A 320      19.738 -17.512   7.018  1.00  0.55           C  \\nATOM   2452  CD2 HIS A 320      19.727 -18.541   7.898  1.00  0.51           C  \\nATOM   2453  ND1 HIS A 320      19.325 -16.400   7.720  1.00  0.49           N  \\nATOM   2454  CE1 HIS A 320      19.076 -16.745   8.972  1.00  0.48           C  \\nATOM   2455  NE2 HIS A 320      19.312 -18.039   9.106  1.00  0.44           N  \\nATOM   2456  N   SER A 321      19.080 -18.012   2.139  1.00  0.66           N  \\nATOM   2457  CA  SER A 321      19.461 -17.962   0.731  1.00  0.66           C  \\nATOM   2458  C   SER A 321      18.341 -17.380  -0.124  1.00  0.66           C  \\nATOM   2459  CB  SER A 321      19.826 -19.358   0.225  1.00  0.62           C  \\nATOM   2460  O   SER A 321      18.579 -16.932  -1.248  1.00  0.64           O  \\nATOM   2461  OG  SER A 321      18.743 -20.256   0.397  1.00  0.60           O  \\nATOM   2462  N   LEU A 322      17.048 -17.283   0.416  1.00  0.70           N  \\nATOM   2463  CA  LEU A 322      15.954 -16.797  -0.418  1.00  0.70           C  \\nATOM   2464  C   LEU A 322      15.627 -15.343  -0.092  1.00  0.70           C  \\nATOM   2465  CB  LEU A 322      14.708 -17.667  -0.228  1.00  0.67           C  \\nATOM   2466  O   LEU A 322      15.410 -14.997   1.071  1.00  0.68           O  \\nATOM   2467  CG  LEU A 322      14.761 -19.069  -0.836  1.00  0.64           C  \\nATOM   2468  CD1 LEU A 322      13.599 -19.913  -0.325  1.00  0.60           C  \\nATOM   2469  CD2 LEU A 322      14.744 -18.993  -2.359  1.00  0.61           C  \\nATOM   2470  N   SER A 323      15.655 -14.360  -1.044  1.00  0.76           N  \\nATOM   2471  CA  SER A 323      15.101 -13.013  -0.959  1.00  0.75           C  \\nATOM   2472  C   SER A 323      13.628 -13.044  -0.565  1.00  0.76           C  \\nATOM   2473  CB  SER A 323      15.265 -12.280  -2.291  1.00  0.73           C  \\nATOM   2474  O   SER A 323      12.966 -14.075  -0.701  1.00  0.72           O  \\nATOM   2475  OG  SER A 323      14.354 -12.779  -3.255  1.00  0.69           O  \\nATOM   2476  N   LEU A 324      12.976 -11.971   0.049  1.00  0.76           N  \\nATOM   2477  CA  LEU A 324      11.561 -11.850   0.387  1.00  0.76           C  \\nATOM   2478  C   LEU A 324      10.687 -12.118  -0.833  1.00  0.76           C  \\nATOM   2479  CB  LEU A 324      11.261 -10.459   0.949  1.00  0.73           C  \\nATOM   2480  O   LEU A 324       9.587 -12.661  -0.707  1.00  0.75           O  \\nATOM   2481  CG  LEU A 324      10.108 -10.366   1.951  1.00  0.67           C  \\nATOM   2482  CD1 LEU A 324      10.648 -10.180   3.365  1.00  0.62           C  \\nATOM   2483  CD2 LEU A 324       9.166  -9.227   1.578  1.00  0.63           C  \\nATOM   2484  N   GLU A 325      11.142 -11.700  -2.023  1.00  0.78           N  \\nATOM   2485  CA  GLU A 325      10.412 -11.927  -3.266  1.00  0.79           C  \\nATOM   2486  C   GLU A 325      10.334 -13.415  -3.597  1.00  0.78           C  \\nATOM   2487  CB  GLU A 325      11.065 -11.165  -4.422  1.00  0.75           C  \\nATOM   2488  O   GLU A 325       9.256 -13.936  -3.889  1.00  0.75           O  \\nATOM   2489  CG  GLU A 325      10.787  -9.669  -4.408  1.00  0.68           C  \\nATOM   2490  CD  GLU A 325      12.049  -8.824  -4.475  1.00  0.66           C  \\nATOM   2491  OE1 GLU A 325      13.162  -9.396  -4.508  1.00  0.70           O  \\nATOM   2492  OE2 GLU A 325      11.923  -7.579  -4.495  1.00  0.63           O  \\nATOM   2493  N   GLU A 326      11.520 -14.119  -3.530  1.00  0.78           N  \\nATOM   2494  CA  GLU A 326      11.565 -15.556  -3.783  1.00  0.77           C  \\nATOM   2495  C   GLU A 326      10.701 -16.320  -2.783  1.00  0.75           C  \\nATOM   2496  CB  GLU A 326      13.007 -16.067  -3.730  1.00  0.74           C  \\nATOM   2497  O   GLU A 326       9.972 -17.240  -3.160  1.00  0.72           O  \\nATOM   2498  CG  GLU A 326      13.846 -15.669  -4.935  1.00  0.70           C  \\nATOM   2499  CD  GLU A 326      15.329 -15.949  -4.754  1.00  0.69           C  \\nATOM   2500  OE1 GLU A 326      15.814 -15.924  -3.600  1.00  0.73           O  \\nATOM   2501  OE2 GLU A 326      16.012 -16.194  -5.773  1.00  0.68           O  \\nATOM   2502  N   ARG A 327      10.554 -15.777  -1.507  1.00  0.73           N  \\nATOM   2503  CA  ARG A 327       9.664 -16.364  -0.509  1.00  0.72           C  \\nATOM   2504  C   ARG A 327       8.203 -16.173  -0.900  1.00  0.72           C  \\nATOM   2505  CB  ARG A 327       9.923 -15.751   0.869  1.00  0.69           C  \\nATOM   2506  O   ARG A 327       7.400 -17.103  -0.794  1.00  0.70           O  \\nATOM   2507  CG  ARG A 327      10.875 -16.562   1.734  1.00  0.66           C  \\nATOM   2508  CD  ARG A 327      11.101 -15.907   3.090  1.00  0.64           C  \\nATOM   2509  NE  ARG A 327      12.485 -16.046   3.532  1.00  0.54           N  \\nATOM   2510  NH1 ARG A 327      12.085 -15.329   5.691  1.00  0.45           N  \\nATOM   2511  NH2 ARG A 327      14.203 -15.931   5.053  1.00  0.40           N  \\nATOM   2512  CZ  ARG A 327      12.921 -15.768   4.758  1.00  0.58           C  \\nATOM   2513  N   ARG A 328       7.800 -15.001  -1.358  1.00  0.78           N  \\nATOM   2514  CA  ARG A 328       6.430 -14.720  -1.777  1.00  0.78           C  \\nATOM   2515  C   ARG A 328       6.040 -15.573  -2.980  1.00  0.78           C  \\nATOM   2516  CB  ARG A 328       6.263 -13.236  -2.111  1.00  0.76           C  \\nATOM   2517  O   ARG A 328       4.944 -16.136  -3.018  1.00  0.75           O  \\nATOM   2518  CG  ARG A 328       5.535 -12.442  -1.038  1.00  0.71           C  \\nATOM   2519  CD  ARG A 328       5.473 -10.959  -1.379  1.00  0.69           C  \\nATOM   2520  NE  ARG A 328       4.642 -10.223  -0.431  1.00  0.57           N  \\nATOM   2521  NH1 ARG A 328       4.933  -8.171  -1.452  1.00  0.46           N  \\nATOM   2522  NH2 ARG A 328       3.634  -8.347   0.428  1.00  0.42           N  \\nATOM   2523  CZ  ARG A 328       4.405  -8.915  -0.487  1.00  0.64           C  \\nATOM   2524  N   VAL A 329       6.888 -15.656  -3.987  1.00  0.79           N  \\nATOM   2525  CA  VAL A 329       6.649 -16.489  -5.161  1.00  0.78           C  \\nATOM   2526  C   VAL A 329       6.501 -17.949  -4.738  1.00  0.77           C  \\nATOM   2527  CB  VAL A 329       7.785 -16.347  -6.198  1.00  0.75           C  \\nATOM   2528  O   VAL A 329       5.548 -18.623  -5.134  1.00  0.73           O  \\nATOM   2529  CG1 VAL A 329       7.625 -17.372  -7.319  1.00  0.69           C  \\nATOM   2530  CG2 VAL A 329       7.814 -14.929  -6.766  1.00  0.68           C  \\nATOM   2531  N   LYS A 330       7.370 -18.507  -3.879  1.00  0.73           N  \\nATOM   2532  CA  LYS A 330       7.283 -19.870  -3.362  1.00  0.72           C  \\nATOM   2533  C   LYS A 330       5.991 -20.079  -2.578  1.00  0.72           C  \\nATOM   2534  CB  LYS A 330       8.490 -20.187  -2.478  1.00  0.69           C  \\nATOM   2535  O   LYS A 330       5.286 -21.068  -2.787  1.00  0.69           O  \\nATOM   2536  CG  LYS A 330       8.747 -21.675  -2.291  1.00  0.65           C  \\nATOM   2537  CD  LYS A 330       9.987 -21.926  -1.444  1.00  0.64           C  \\nATOM   2538  CE  LYS A 330      10.181 -23.409  -1.160  1.00  0.59           C  \\nATOM   2539  NZ  LYS A 330      11.363 -23.656  -0.281  1.00  0.50           N  \\nATOM   2540  N   MET A 331       5.520 -19.145  -1.755  1.00  0.76           N  \\nATOM   2541  CA  MET A 331       4.244 -19.226  -1.050  1.00  0.76           C  \\nATOM   2542  C   MET A 331       3.078 -19.238  -2.033  1.00  0.76           C  \\nATOM   2543  CB  MET A 331       4.093 -18.059  -0.073  1.00  0.73           C  \\nATOM   2544  O   MET A 331       2.065 -19.897  -1.794  1.00  0.74           O  \\nATOM   2545  CG  MET A 331       4.880 -18.232   1.216  1.00  0.67           C  \\nATOM   2546  SD  MET A 331       4.526 -16.911   2.439  1.00  0.62           S  \\nATOM   2547  CE  MET A 331       2.713 -16.877   2.357  1.00  0.58           C  \\nATOM   2548  N   LEU A 332       3.121 -18.502  -3.097  1.00  0.76           N  \\nATOM   2549  CA  LEU A 332       2.053 -18.502  -4.090  1.00  0.77           C  \\nATOM   2550  C   LEU A 332       1.994 -19.837  -4.825  1.00  0.75           C  \\nATOM   2551  CB  LEU A 332       2.254 -17.362  -5.093  1.00  0.73           C  \\nATOM   2552  O   LEU A 332       0.909 -20.320  -5.157  1.00  0.73           O  \\nATOM   2553  CG  LEU A 332       1.879 -15.960  -4.610  1.00  0.68           C  \\nATOM   2554  CD1 LEU A 332       2.415 -14.907  -5.574  1.00  0.64           C  \\nATOM   2555  CD2 LEU A 332       0.367 -15.834  -4.458  1.00  0.66           C  \\nATOM   2556  N   PHE A 333       3.124 -20.451  -5.090  1.00  0.72           N  \\nATOM   2557  CA  PHE A 333       3.198 -21.758  -5.732  1.00  0.73           C  \\nATOM   2558  C   PHE A 333       2.730 -22.855  -4.783  1.00  0.70           C  \\nATOM   2559  CB  PHE A 333       4.627 -22.046  -6.204  1.00  0.68           C  \\nATOM   2560  O   PHE A 333       1.947 -23.725  -5.170  1.00  0.67           O  \\nATOM   2561  CG  PHE A 333       4.879 -21.675  -7.640  1.00  0.62           C  \\nATOM   2562  CD1 PHE A 333       4.386 -22.463  -8.673  1.00  0.57           C  \\nATOM   2563  CD2 PHE A 333       5.608 -20.536  -7.958  1.00  0.59           C  \\nATOM   2564  CE1 PHE A 333       4.618 -22.122 -10.004  1.00  0.56           C  \\nATOM   2565  CE2 PHE A 333       5.843 -20.189  -9.285  1.00  0.55           C  \\nATOM   2566  CZ  PHE A 333       5.348 -20.983 -10.306  1.00  0.55           C  \\nATOM   2567  N   GLU A 334       3.186 -22.883  -3.559  1.00  0.71           N  \\nATOM   2568  CA  GLU A 334       2.754 -23.869  -2.573  1.00  0.71           C  \\nATOM   2569  C   GLU A 334       1.253 -23.771  -2.315  1.00  0.70           C  \\nATOM   2570  CB  GLU A 334       3.526 -23.694  -1.263  1.00  0.67           C  \\nATOM   2571  O   GLU A 334       0.562 -24.789  -2.243  1.00  0.68           O  \\nATOM   2572  CG  GLU A 334       4.977 -24.148  -1.338  1.00  0.63           C  \\nATOM   2573  CD  GLU A 334       5.763 -23.857  -0.070  1.00  0.63           C  \\nATOM   2574  OE1 GLU A 334       5.198 -23.253   0.869  1.00  0.65           O  \\nATOM   2575  OE2 GLU A 334       6.955 -24.235  -0.016  1.00  0.57           O  \\nATOM   2576  N   LYS A 335       0.670 -22.586  -2.210  1.00  0.72           N  \\nATOM   2577  CA  LYS A 335      -0.774 -22.429  -2.067  1.00  0.72           C  \\nATOM   2578  C   LYS A 335      -1.510 -22.975  -3.288  1.00  0.72           C  \\nATOM   2579  CB  LYS A 335      -1.136 -20.959  -1.851  1.00  0.68           C  \\nATOM   2580  O   LYS A 335      -2.654 -23.422  -3.181  1.00  0.71           O  \\nATOM   2581  CG  LYS A 335      -1.201 -20.547  -0.387  1.00  0.63           C  \\nATOM   2582  CD  LYS A 335      -1.647 -19.099  -0.234  1.00  0.63           C  \\nATOM   2583  CE  LYS A 335      -1.713 -18.685   1.230  1.00  0.55           C  \\nATOM   2584  NZ  LYS A 335      -2.174 -17.273   1.386  1.00  0.48           N  \\nATOM   2585  N   ARG A 336      -0.891 -22.864  -4.414  1.00  0.70           N  \\nATOM   2586  CA  ARG A 336      -1.499 -23.438  -5.610  1.00  0.70           C  \\nATOM   2587  C   ARG A 336      -1.348 -24.955  -5.626  1.00  0.69           C  \\nATOM   2588  CB  ARG A 336      -0.877 -22.836  -6.872  1.00  0.65           C  \\nATOM   2589  O   ARG A 336      -2.176 -25.661  -6.206  1.00  0.67           O  \\nATOM   2590  CG  ARG A 336      -1.496 -21.513  -7.293  1.00  0.61           C  \\nATOM   2591  CD  ARG A 336      -0.910 -21.008  -8.604  1.00  0.61           C  \\nATOM   2592  NE  ARG A 336      -1.496 -19.730  -9.000  1.00  0.49           N  \\nATOM   2593  NH1 ARG A 336      -0.446 -19.650 -11.056  1.00  0.38           N  \\nATOM   2594  NH2 ARG A 336      -1.847 -17.954 -10.412  1.00  0.35           N  \\nATOM   2595  CZ  ARG A 336      -1.262 -19.114 -10.155  1.00  0.58           C  \\nATOM   2596  N   SER A 337      -0.153 -25.393  -5.099  1.00  0.64           N  \\nATOM   2597  CA  SER A 337       0.152 -26.818  -5.157  1.00  0.65           C  \\nATOM   2598  C   SER A 337      -0.551 -27.581  -4.039  1.00  0.62           C  \\nATOM   2599  CB  SER A 337       1.662 -27.047  -5.070  1.00  0.59           C  \\nATOM   2600  O   SER A 337      -0.742 -28.795  -4.133  1.00  0.59           O  \\nATOM   2601  OG  SER A 337       2.369 -25.964  -5.647  1.00  0.55           O  \\nATOM   2602  N   ASN A 338      -1.050 -26.848  -3.021  1.00  0.55           N  \\nATOM   2603  CA  ASN A 338      -1.749 -27.545  -1.947  1.00  0.56           C  \\nATOM   2604  C   ASN A 338      -3.104 -26.907  -1.653  1.00  0.55           C  \\nATOM   2605  CB  ASN A 338      -0.891 -27.575  -0.680  1.00  0.50           C  \\nATOM   2606  O   ASN A 338      -3.195 -25.972  -0.857  1.00  0.54           O  \\nATOM   2607  CG  ASN A 338      -1.312 -28.668   0.284  1.00  0.48           C  \\nATOM   2608  ND2 ASN A 338      -0.569 -28.815   1.375  1.00  0.49           N  \\nATOM   2609  OD1 ASN A 338      -2.296 -29.374   0.050  1.00  0.50           O  \\nATOM   2610  N   PRO A 339      -4.074 -27.066  -2.494  1.00  0.63           N  \\nATOM   2611  CA  PRO A 339      -5.412 -26.557  -2.183  1.00  0.62           C  \\nATOM   2612  C   PRO A 339      -5.992 -27.164  -0.908  1.00  0.63           C  \\nATOM   2613  CB  PRO A 339      -6.237 -26.963  -3.406  1.00  0.58           C  \\nATOM   2614  O   PRO A 339      -7.094 -26.797  -0.491  1.00  0.61           O  \\nATOM   2615  CG  PRO A 339      -5.554 -28.181  -3.940  1.00  0.55           C  \\nATOM   2616  CD  PRO A 339      -4.102 -28.116  -3.565  1.00  0.55           C  \\nATOM   2617  N   TYR A 340      -5.352 -28.098  -0.176  1.00  0.63           N  \\nATOM   2618  CA  TYR A 340      -5.885 -28.880   0.934  1.00  0.62           C  \\nATOM   2619  C   TYR A 340      -5.019 -28.721   2.179  1.00  0.60           C  \\nATOM   2620  CB  TYR A 340      -5.982 -30.360   0.552  1.00  0.57           C  \\nATOM   2621  O   TYR A 340      -5.048 -29.568   3.075  1.00  0.56           O  \\nATOM   2622  CG  TYR A 340      -6.944 -30.633  -0.579  1.00  0.54           C  \\nATOM   2623  CD1 TYR A 340      -8.321 -30.551  -0.383  1.00  0.50           C  \\nATOM   2624  CD2 TYR A 340      -6.479 -30.974  -1.845  1.00  0.53           C  \\nATOM   2625  CE1 TYR A 340      -9.212 -30.804  -1.421  1.00  0.53           C  \\nATOM   2626  CE2 TYR A 340      -7.361 -31.228  -2.890  1.00  0.52           C  \\nATOM   2627  OH  TYR A 340      -9.600 -31.391  -3.700  1.00  0.54           O  \\nATOM   2628  CZ  TYR A 340      -8.723 -31.140  -2.669  1.00  0.49           C  \\nATOM   2629  N   HIS A 341      -4.445 -27.686   2.416  1.00  0.44           N  \\nATOM   2630  CA  HIS A 341      -4.101 -27.432   3.810  1.00  0.49           C  \\nATOM   2631  C   HIS A 341      -5.024 -26.385   4.423  1.00  0.44           C  \\nATOM   2632  CB  HIS A 341      -2.644 -26.982   3.929  1.00  0.44           C  \\nATOM   2633  O   HIS A 341      -5.324 -25.369   3.792  1.00  0.43           O  \\nATOM   2634  CG  HIS A 341      -1.673 -28.114   4.036  1.00  0.45           C  \\nATOM   2635  CD2 HIS A 341      -0.684 -28.519   3.204  1.00  0.44           C  \\nATOM   2636  ND1 HIS A 341      -1.659 -28.984   5.105  1.00  0.45           N  \\nATOM   2637  CE1 HIS A 341      -0.700 -29.877   4.925  1.00  0.41           C  \\nATOM   2638  NE2 HIS A 341      -0.094 -29.617   3.779  1.00  0.41           N  \",\"pdb\");\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 1},{\"cartoon\": {\"color\": \"#2c2cd3\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 2},{\"cartoon\": {\"color\": \"#5858a7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 3},{\"cartoon\": {\"color\": \"#777788\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 4},{\"cartoon\": {\"color\": \"#8d8d72\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 5},{\"cartoon\": {\"color\": \"#9b9b64\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 6},{\"cartoon\": {\"color\": \"#9b9b64\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 7},{\"cartoon\": {\"color\": \"#a5a55a\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 8},{\"cartoon\": {\"color\": \"#aeae51\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 9},{\"cartoon\": {\"color\": \"#b5b54a\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 10},{\"cartoon\": {\"color\": \"#baba45\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 11},{\"cartoon\": {\"color\": \"#b9b946\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 12},{\"cartoon\": {\"color\": \"#bebe41\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 13},{\"cartoon\": {\"color\": \"#a0a05f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 14},{\"cartoon\": {\"color\": \"#83837c\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 15},{\"cartoon\": {\"color\": \"#898976\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 16},{\"cartoon\": {\"color\": \"#8f8f70\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 17},{\"cartoon\": {\"color\": \"#94946b\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 18},{\"cartoon\": {\"color\": \"#94946b\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 19},{\"cartoon\": {\"color\": \"#b0b04f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 20},{\"cartoon\": {\"color\": \"#b1b14e\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 21},{\"cartoon\": {\"color\": \"#b1b14e\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 22},{\"cartoon\": {\"color\": \"#b1b14e\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 23},{\"cartoon\": {\"color\": \"#b1b14e\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 24},{\"cartoon\": {\"color\": \"#b6b649\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 25},{\"cartoon\": {\"color\": \"#b7b748\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 26},{\"cartoon\": {\"color\": \"#b6b649\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 27},{\"cartoon\": {\"color\": \"#a7a758\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 28},{\"cartoon\": {\"color\": \"#a8a857\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 29},{\"cartoon\": {\"color\": \"#a8a857\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 30},{\"cartoon\": {\"color\": \"#90906f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 31},{\"cartoon\": {\"color\": \"#a8a857\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 32},{\"cartoon\": {\"color\": \"#caca35\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 33},{\"cartoon\": {\"color\": \"#aeae51\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 34},{\"cartoon\": {\"color\": \"#94946b\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 35},{\"cartoon\": {\"color\": \"#7a7a85\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 36},{\"cartoon\": {\"color\": \"#7a7a85\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 37},{\"cartoon\": {\"color\": \"#797986\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 38},{\"cartoon\": {\"color\": \"#62629d\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 39},{\"cartoon\": {\"color\": \"#62629d\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 40},{\"cartoon\": {\"color\": \"#5f5fa0\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 41},{\"cartoon\": {\"color\": \"#5f5fa0\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 42},{\"cartoon\": {\"color\": \"#5f5fa0\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 43},{\"cartoon\": {\"color\": \"#5f5fa0\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 44},{\"cartoon\": {\"color\": \"#5f5fa0\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 45},{\"cartoon\": {\"color\": \"#6a6a95\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 46},{\"cartoon\": {\"color\": \"#696996\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 47},{\"cartoon\": {\"color\": \"#6b6b94\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 48},{\"cartoon\": {\"color\": \"#7f7f80\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 49},{\"cartoon\": {\"color\": \"#80807f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 50},{\"cartoon\": {\"color\": \"#80807f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 51},{\"cartoon\": {\"color\": \"#95956a\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 52},{\"cartoon\": {\"color\": \"#90906f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 53},{\"cartoon\": {\"color\": \"#adad52\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 54},{\"cartoon\": {\"color\": \"#aeae51\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 55},{\"cartoon\": {\"color\": \"#aeae51\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 56},{\"cartoon\": {\"color\": \"#d1d12e\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 57},{\"cartoon\": {\"color\": \"#d1d12e\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 58},{\"cartoon\": {\"color\": \"#d5d52a\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 59},{\"cartoon\": {\"color\": \"#d5d52a\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 60},{\"cartoon\": {\"color\": \"#d4d42b\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 61},{\"cartoon\": {\"color\": \"#d4d42b\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 62},{\"cartoon\": {\"color\": \"#d4d42b\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 63},{\"cartoon\": {\"color\": \"#d4d42b\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 64},{\"cartoon\": {\"color\": \"#d3d32c\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 65},{\"cartoon\": {\"color\": \"#c0c03f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 66},{\"cartoon\": {\"color\": \"#c0c03f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 67},{\"cartoon\": {\"color\": \"#c0c03f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 68},{\"cartoon\": {\"color\": \"#c0c03f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 69},{\"cartoon\": {\"color\": \"#c2c23d\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 70},{\"cartoon\": {\"color\": \"#e3e31c\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 71},{\"cartoon\": {\"color\": \"#e3e31c\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 72},{\"cartoon\": {\"color\": \"#e3e31c\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 73},{\"cartoon\": {\"color\": \"#e3e31c\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 74},{\"cartoon\": {\"color\": \"#e3e31c\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 75},{\"cartoon\": {\"color\": \"#e3e31c\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 76},{\"cartoon\": {\"color\": \"#c0c03f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 77},{\"cartoon\": {\"color\": \"#c1c13e\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 78},{\"cartoon\": {\"color\": \"#c1c13e\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 79},{\"cartoon\": {\"color\": \"#a0a05f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 80},{\"cartoon\": {\"color\": \"#83837c\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 81},{\"cartoon\": {\"color\": \"#6a6a95\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 82},{\"cartoon\": {\"color\": \"#5454ab\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 83},{\"cartoon\": {\"color\": \"#60609f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 84},{\"cartoon\": {\"color\": \"#60609f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 85},{\"cartoon\": {\"color\": \"#4b4bb4\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 86},{\"cartoon\": {\"color\": \"#3a3ac5\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 87},{\"cartoon\": {\"color\": \"#2c2cd3\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 88},{\"cartoon\": {\"color\": \"#2d2dd2\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 89},{\"cartoon\": {\"color\": \"#2d2dd2\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 90},{\"cartoon\": {\"color\": \"#2121de\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 91},{\"cartoon\": {\"color\": \"#1717e8\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 92},{\"cartoon\": {\"color\": \"#0f0ff0\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 93},{\"cartoon\": {\"color\": \"#0a0af5\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 94},{\"cartoon\": {\"color\": \"#0e0ef1\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 95},{\"cartoon\": {\"color\": \"#0808f7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 96},{\"cartoon\": {\"color\": \"#0606f9\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 97},{\"cartoon\": {\"color\": \"#0808f7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 98},{\"cartoon\": {\"color\": \"#0808f7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 99},{\"cartoon\": {\"color\": \"#0808f7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 100},{\"cartoon\": {\"color\": \"#0e0ef1\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 101},{\"cartoon\": {\"color\": \"#0808f7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 102},{\"cartoon\": {\"color\": \"#0808f7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 103},{\"cartoon\": {\"color\": \"#0808f7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 104},{\"cartoon\": {\"color\": \"#0808f7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 105},{\"cartoon\": {\"color\": \"#0d0df2\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 106},{\"cartoon\": {\"color\": \"#0707f8\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 107},{\"cartoon\": {\"color\": \"#0303fc\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 108},{\"cartoon\": {\"color\": \"#0303fc\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 109},{\"cartoon\": {\"color\": \"#0303fc\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 110},{\"cartoon\": {\"color\": \"#0606f9\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 111},{\"cartoon\": {\"color\": \"#0b0bf4\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 112},{\"cartoon\": {\"color\": \"#0707f8\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 113},{\"cartoon\": {\"color\": \"#0707f8\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 114},{\"cartoon\": {\"color\": \"#0505fa\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 115},{\"cartoon\": {\"color\": \"#0303fc\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 116},{\"cartoon\": {\"color\": \"#0303fc\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 117},{\"cartoon\": {\"color\": \"#0303fc\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 118},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 119},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 120},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 121},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 122},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 123},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 124},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 125},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 126},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 127},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 128},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 129},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 130},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 131},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 132},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 133},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 134},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 135},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 136},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 137},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 138},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 139},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 140},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 141},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 142},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 143},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 144},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 145},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 146},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 147},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 148},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 149},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 150},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 151},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 152},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 153},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 154},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 155},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 156},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 157},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 158},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 159},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 160},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 161},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 162},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 163},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 164},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 165},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 166},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 167},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 168},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 169},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 170},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 171},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 172},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 173},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 174},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 175},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 176},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 177},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 178},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 179},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 180},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 181},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 182},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 183},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 184},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 185},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 186},{\"cartoon\": {\"color\": \"#0303fc\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 187},{\"cartoon\": {\"color\": \"#0303fc\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 188},{\"cartoon\": {\"color\": \"#0505fa\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 189},{\"cartoon\": {\"color\": \"#0a0af5\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 190},{\"cartoon\": {\"color\": \"#0a0af5\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 191},{\"cartoon\": {\"color\": \"#0a0af5\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 192},{\"cartoon\": {\"color\": \"#0a0af5\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 193},{\"cartoon\": {\"color\": \"#0505fa\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 194},{\"cartoon\": {\"color\": \"#0505fa\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 195},{\"cartoon\": {\"color\": \"#0505fa\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 196},{\"cartoon\": {\"color\": \"#0505fa\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 197},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 198},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 199},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 200},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 201},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 202},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 203},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 204},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 205},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 206},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 207},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 208},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 209},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 210},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 211},{\"cartoon\": {\"color\": \"#0303fc\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 212},{\"cartoon\": {\"color\": \"#0707f8\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 213},{\"cartoon\": {\"color\": \"#0d0df2\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 214},{\"cartoon\": {\"color\": \"#0d0df2\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 215},{\"cartoon\": {\"color\": \"#0d0df2\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 216},{\"cartoon\": {\"color\": \"#1414eb\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 217},{\"cartoon\": {\"color\": \"#1d1de2\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 218},{\"cartoon\": {\"color\": \"#2929d6\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 219},{\"cartoon\": {\"color\": \"#2929d6\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 220},{\"cartoon\": {\"color\": \"#2929d6\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 221},{\"cartoon\": {\"color\": \"#2929d6\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 222},{\"cartoon\": {\"color\": \"#3737c8\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 223},{\"cartoon\": {\"color\": \"#2b2bd4\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 224},{\"cartoon\": {\"color\": \"#1f1fe0\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 225},{\"cartoon\": {\"color\": \"#1f1fe0\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 226},{\"cartoon\": {\"color\": \"#2020df\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 227},{\"cartoon\": {\"color\": \"#2c2cd3\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 228},{\"cartoon\": {\"color\": \"#2a2ad5\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 229},{\"cartoon\": {\"color\": \"#1e1ee1\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 230},{\"cartoon\": {\"color\": \"#1515ea\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 231},{\"cartoon\": {\"color\": \"#0e0ef1\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 232},{\"cartoon\": {\"color\": \"#0e0ef1\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 233},{\"cartoon\": {\"color\": \"#0e0ef1\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 234},{\"cartoon\": {\"color\": \"#0808f7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 235},{\"cartoon\": {\"color\": \"#0404fb\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 236},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 237},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 238},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 239},{\"cartoon\": {\"color\": \"#0303fc\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 240},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 241},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 242},{\"cartoon\": {\"color\": \"#0303fc\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 243},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 244},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 245},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 246},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 247},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 248},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 249},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 250},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 251},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 252},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 253},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 254},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 255},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 256},{\"cartoon\": {\"color\": \"#0202fd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 257},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 258},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 259},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 260},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 261},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 262},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 263},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 264},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 265},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 266},{\"cartoon\": {\"color\": \"#0000ff\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 267},{\"cartoon\": {\"color\": \"#0101fe\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 268},{\"cartoon\": {\"color\": \"#0404fb\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 269},{\"cartoon\": {\"color\": \"#0909f6\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 270},{\"cartoon\": {\"color\": \"#0f0ff0\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 271},{\"cartoon\": {\"color\": \"#0909f6\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 272},{\"cartoon\": {\"color\": \"#0f0ff0\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 273},{\"cartoon\": {\"color\": \"#1111ee\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 274},{\"cartoon\": {\"color\": \"#1a1ae5\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 275},{\"cartoon\": {\"color\": \"#2424db\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 276},{\"cartoon\": {\"color\": \"#3232cd\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 277},{\"cartoon\": {\"color\": \"#2c2cd3\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 278},{\"cartoon\": {\"color\": \"#2c2cd3\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 279},{\"cartoon\": {\"color\": \"#3a3ac5\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 280},{\"cartoon\": {\"color\": \"#4c4cb3\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 281},{\"cartoon\": {\"color\": \"#60609f\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 282},{\"cartoon\": {\"color\": \"#767689\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 283},{\"cartoon\": {\"color\": \"#6b6b94\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 284},{\"cartoon\": {\"color\": \"#84847b\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 285},{\"cartoon\": {\"color\": \"#6b6b94\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 286},{\"cartoon\": {\"color\": \"#5555aa\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 287},{\"cartoon\": {\"color\": \"#4343bc\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 288},{\"cartoon\": {\"color\": \"#3434cb\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 289},{\"cartoon\": {\"color\": \"#3434cb\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 290},{\"cartoon\": {\"color\": \"#2727d8\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 291},{\"cartoon\": {\"color\": \"#2727d8\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 292},{\"cartoon\": {\"color\": \"#2727d8\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 293},{\"cartoon\": {\"color\": \"#1b1be4\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 294},{\"cartoon\": {\"color\": \"#1b1be4\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 295},{\"cartoon\": {\"color\": \"#2626d9\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 296},{\"cartoon\": {\"color\": \"#3434cb\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 297},{\"cartoon\": {\"color\": \"#3030cf\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 298},{\"cartoon\": {\"color\": \"#3030cf\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 299},{\"cartoon\": {\"color\": \"#3030cf\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 300},{\"cartoon\": {\"color\": \"#3131ce\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 301},{\"cartoon\": {\"color\": \"#3838c7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 302},{\"cartoon\": {\"color\": \"#3838c7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 303},{\"cartoon\": {\"color\": \"#4848b7\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 304},{\"cartoon\": {\"color\": \"#5c5ca3\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 305},{\"cartoon\": {\"color\": \"#72728d\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 306},{\"cartoon\": {\"color\": \"#8b8b74\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 307},{\"cartoon\": {\"color\": \"#a9a956\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 308},{\"cartoon\": {\"color\": \"#a8a857\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 309},{\"cartoon\": {\"color\": \"#a8a857\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 310},{\"cartoon\": {\"color\": \"#a9a956\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 311},{\"cartoon\": {\"color\": \"#adad52\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 312},{\"cartoon\": {\"color\": \"#adad52\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 313},{\"cartoon\": {\"color\": \"#aeae51\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 314},{\"cartoon\": {\"color\": \"#aeae51\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 315},{\"cartoon\": {\"color\": \"#b8b847\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 316},{\"cartoon\": {\"color\": \"#b8b847\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 317},{\"cartoon\": {\"color\": \"#b8b847\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 318},{\"cartoon\": {\"color\": \"#b8b847\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 319},{\"cartoon\": {\"color\": \"#b8b847\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 320},{\"cartoon\": {\"color\": \"#b8b847\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 321},{\"cartoon\": {\"color\": \"#b8b847\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 322},{\"cartoon\": {\"color\": \"#b9b946\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 323},{\"cartoon\": {\"color\": \"#b9b946\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 324},{\"cartoon\": {\"color\": \"#b7b748\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 325},{\"cartoon\": {\"color\": \"#b3b34c\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 326},{\"cartoon\": {\"color\": \"#d6d629\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 327},{\"cartoon\": {\"color\": \"#d4d42b\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 328},{\"cartoon\": {\"color\": \"#d1d12e\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 329},{\"cartoon\": {\"color\": \"#fefe01\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 330},{\"cartoon\": {\"color\": \"#fefe01\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 331},{\"cartoon\": {\"color\": \"#fefe01\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 332},{\"cartoon\": {\"color\": \"#fdfd02\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 333},{\"cartoon\": {\"color\": \"#fcfc03\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 334},{\"cartoon\": {\"color\": \"#fcfc03\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 335},{\"cartoon\": {\"color\": \"#fbfb04\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 336},{\"cartoon\": {\"color\": \"#fcfc03\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 337},{\"cartoon\": {\"color\": \"#fafa05\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 338},{\"cartoon\": {\"color\": \"#f8f807\"}});\n",
              "\tviewer_17207123026924806.setStyle({\"resi\": 339},{\"cartoon\": {\"color\": \"#f4f40b\"}});\n",
              "\tviewer_17207123026924806.zoomTo();\n",
              "viewer_17207123026924806.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_63caa569-4b21-4eed-89b5-9c627f6c2580\", \"A.html\", 248203)"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 600x100 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAfoAAABlCAYAAAClUiFrAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAAAWWElEQVR4nO3dfVTV9R0H8Pfl4V4gLgIaXECGCAFGpinpIUNGY4PcfNjOSVuGsNTWtJOD5UO5hmmgh7QsjkXDBO2wbGu4Y0ia2dRJuZ0hOI8yTEDTBrbOCh95vJ/9ofdyH34XuQRX+fl+ncO5/H7fp8/3w7184PrDn0ZEBERERKRKbjc7ACIiIho8LPREREQqxkJPRESkYiz0REREKsZCT0REpGIs9ERERCrGQk9ERKRiLPREREQq5jEYk7a1taGjo2MwpiYiIlI1rVYLLy+vAZtvwAt9W1sbIkeNQsv58wM9NRERkeoZDAY0NTUNWLEf8ELf0dGBlvPncfbkcfj5+l4/K4Dpf9o1/4e7pmPLR5s+5v+dVxSOrc+JUt9ex9uu04e1Hc0nRutjo9Gir02bZbxGo/J4y2PT50YHa1k+OpzH2LOmoz5K89uuYVQa21ubzbx28yms6XCfCsd2cUFhvn7E5TDHSvtU2P8N89bzPBK7NtiPVfpaO5rXtk3pdWa0ec2Yu14/b+ytr8LrzUEfpfnEZi3FWGxf032Jy8F817681rk1t6HnuGe4aTys1rROsfU5uzFW3yp64ri2pv1YsY1L4dg2HqOjvViuef2cbcrFHIVFH4s203nbp6QRNmuZPrEYZ/Oss3i0yLFtm8IeeuJS2N/1eE3xGG3jNK8jFn1t92nbV2GcQh9TTErjLfdidDDOuq9Fm8WaHRCUtbSgo6Pj1i30Jn56Pfz0+utHCt/8vkuht+qr8I3HdP6Ghd7JtftT6B0UrIEr9H3p299Cr7Av27G9/hBgM29f1rzRPpXmc7R2fwu9o8KsOJ/CGCfmsy/0CmO/S6FX3IPNa6aXwnzDQu/kfDcu9KIcx43i6kOh762gOizaGtu+9uPtC71C8dYor+10obctbkpF0sWFXpTigu2j2Mdj0Wa3BwdrGi0ebQuyfR+x6Ou4zXEfx4+O2qQPfRTnEeu2gcaL8YiIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxVjoiYiIVIyFnoiISMVY6ImIiFSMhZ6IiEjFWOiJiIhUjIWeiIhIxTwGa+ILFy8CItePpOdz0ynTJ2L5aNPHcrzdsfU5Uerb63jbdfqwtqP5xGh9bDRa9LVps4zXaFQeb3ls+tzoYC3LR4fzGHvWdNRHaX7bNYxKY3trs5nXbj6FNR3uU+HYLi4ozNePuBzmWGmfCvu/Yd56nkdi1wb7sUpfa0fz2rYpvc6MNq8Zc9fr54299VV4vTnoozSf2KylGIvta7ovcTmY79qX1zq35jb0HPcMN42H1ZrWKbY+ZzfG6ltFTxzX1rQfK7ZxKRzbxmN0tBfLNa+fs025mKOw6GPRZjpv+5Q0wmYt0ycW42yedRaPFjm2bVPYQ09cCvu7Hq8pHqNtnOZ1xKKv7T5t+yqMU+hjiklpvOVejA7GWfe1aLNYs6OnSA6YAS/0IgJfX1+Ex8QP9NRERESq5+vra/HD9Hc34IVeo9Hg0qVLOHv2LPz8/AZ6elJw4cIFhIeHM+cuxJy7HnPuesy565lyrtFoBmzOQXvr3s/Pj08MF2POXY85dz3m3PWY86GNF+MRERGpGAs9ERGRig14odfpdMjNzYVOpxvoqckB5tz1mHPXY85djzl3vcHIuUYG8tI+IiIiuqXwrXsiIiIVY6EnIiJSMRZ6IiIiFWOhJyIiUrF+FfpNmzZh1KhR8PLywuTJk/GPf/yj1/5/+tOfEBcXBy8vL4wdOxaVlZX9CvZ25kzOi4uLkZSUhICAAAQEBCA1NfWGXyOy5+zz3GT79u3QaDSYNWvW4AaoQs7m/Ntvv8XixYsREhICnU6HmJgYfn9xkrM537hxI2JjY+Ht7Y3w8HBkZ2ejra3NRdEObQcPHsT06dMRGhoKjUaDv/zlLzccs3//fkyYMAE6nQ7R0dEoLS11fmFx0vbt20Wr1cqWLVvk+PHjsnDhQvH395fz588r9q+qqhJ3d3cpKCiQEydOyG9/+1vx9PSUY8eOObv0bcvZnD/22GOyadMmqampkbq6OsnKypJhw4bJuXPnXBz50OVszk2ampokLCxMkpKSZObMma4JViWczXl7e7skJCTItGnT5NChQ9LU1CT79++X2tpaF0c+dDmb87KyMtHpdFJWViZNTU2yZ88eCQkJkezsbBdHPjRVVlbKypUrpby8XADIjh07eu3f2NgoPj4+kpOTIydOnJDCwkJxd3eX3bt3O7Wu04V+0qRJsnjxYvNxd3e3hIaGytq1axX7z549W3784x9bnZs8ebL88pe/dHbp25azObfV1dUler1etm7dOlghqk5/ct7V1SUPPPCAbN68WTIzM1noneRszt98800ZPXq0dHR0uCpE1XE254sXL5aHHnrI6lxOTo5MmTJlUONUo74U+mXLlkl8fLzVuTlz5khaWppTazn11n1HRweqq6uRmppqPufm5obU1FR89tlnimM+++wzq/4AkJaW5rA/WetPzm1duXIFnZ2dCAwMHKwwVaW/OV+9ejWCgoIwf/58V4SpKv3J+c6dO5GYmIjFixcjODgY99xzD/Lz89Hd3e2qsIe0/uT8gQceQHV1tfnt/cbGRlRWVmLatGkuifl2M1D106mb2nz99dfo7u5GcHCw1fng4GD8+9//VhzT0tKi2L+lpcWpQG9X/cm5reXLlyM0NNTuCUPK+pPzQ4cO4e2330Ztba0LIlSf/uS8sbERn3zyCebOnYvKykqcOnUKixYtQmdnJ3Jzc10R9pDWn5w/9thj+Prrr/Hggw9CRNDV1YWnnnoKzz//vCtCvu04qp8XLlzA1atX4e3t3ad5eNW9yq1btw7bt2/Hjh074OXldbPDUaWLFy8iIyMDxcXFGDFixM0O57ZhNBoRFBSE3//+95g4cSLmzJmDlStXoqio6GaHplr79+9Hfn4+3njjDRw5cgTl5eXYtWsX1qxZc7NDo1449Rv9iBEj4O7ujvPnz1udP3/+PAwGg+IYg8HgVH+y1p+cm6xfvx7r1q3Dxx9/jHvvvXcww1QVZ3Pe0NCA06dPY/r06eZzRqMRAODh4YH6+npERUUNbtBDXH+e5yEhIfD09IS7u7v53JgxY9DS0oKOjg5otdpBjXmo60/OX3jhBWRkZGDBggUAgLFjx+Ly5ct48sknsXLlSri58XfHgeSofvr5+fX5t3nAyd/otVotJk6ciH379pnPGY1G7Nu3D4mJiYpjEhMTrfoDwN69ex32J2v9yTkAFBQUYM2aNdi9ezcSEhJcEapqOJvzuLg4HDt2DLW1teaPGTNmICUlBbW1tQgPD3dl+ENSf57nU6ZMwalTp8w/VAHAyZMnERISwiLfB/3J+ZUrV+yKuekHLeFtUwbcgNVP564TvPbnGDqdTkpLS+XEiRPy5JNPir+/v7S0tIiISEZGhqxYscLcv6qqSjw8PGT9+vVSV1cnubm5/PM6Jzmb83Xr1olWq5X3339fmpubzR8XL168WVsYcpzNuS1ede88Z3P+xRdfiF6vl6efflrq6+uloqJCgoKC5KWXXrpZWxhynM15bm6u6PV6effdd6WxsVE++ugjiYqKktmzZ9+sLQwpFy9elJqaGqmpqREA8sorr0hNTY2cOXNGRERWrFghGRkZ5v6mP69bunSp1NXVyaZNm1zz53UiIoWFhfK9731PtFqtTJo0SQ4fPmxuS05OlszMTKv+f/zjHyUmJka0Wq3Ex8fLrl27+rPsbc2ZnEdERAgAu4/c3FzXBz6EOfs8t8RC3z/O5vzTTz+VyZMni06nk9GjR0teXp50dXW5OOqhzZmcd3Z2yqpVqyQqKkq8vLwkPDxcFi1aJN98843rAx+C/vrXvyp+bzblODMzU5KTk+3GjB8/XrRarYwePVpKSkqcXpe3qSUiIlIxXjlBRESkYiz0REREKsZCT0REpGIs9ERERCrGQk9ERKRiLPREREQqxkJPRESkYiz0RN9RVlYWZs2aZT7+/ve/j1//+tcuj2P//v3QaDT49ttvXb42Ed26WOhJlbKysqDRaKDRaKDVahEdHY3Vq1ejq6tr0NcuLy/v8928XF2cR40ahY0bNyq2nT59GhqNxuGtdktLS805dXNzw8iRI/GLX/wCX331lbnPgQMH8NBDDyEwMBA+Pj646667kJmZiY6ODocxHT16FDNmzEBQUBC8vLwwatQozJkzx2peIuo/FnpSrfT0dDQ3N+Pzzz/Hb37zG6xatQovv/yyYt/eCpGzAgMDodfrB2y+W4mfnx+am5tx7tw5FBcX48MPP0RGRgYA4MSJE0hPT0dCQgIOHjyIY8eOobCwEFqtFt3d3Yrz/fe//8UPfvADBAYGYs+ePairq0NJSQlCQ0Nx+fLlQdtHZ2fnoM1NdKthoSfV0ul0MBgMiIiIwK9+9SukpqZi586dAHrebs/Ly0NoaChiY2MBAGfPnsXs2bPh7++PwMBAzJw5E6dPnzbP2d3djZycHPj7+2P48OFYtmyZ3V27bN+6b29vx/LlyxEeHg6dTofo6Gi8/fbbOH36NFJSUgAAAQEB0Gg0yMrKAnDtLmJr165FZGQkvL29MW7cOLz//vtW61RWViImJgbe3t5ISUmxinOwaDQaGAwGhIaG4uGHH8YzzzyDjz/+GFevXsVHH30Eg8GAgoIC3HPPPYiKikJ6ejqKi4sd3lKzqqoKra2t2Lx5M+677z5ERkYiJSUFr776KiIjI839jh8/jp/85Cfw8/ODXq9HUlISGhoaAFzL1erVqzFy5EjodDqMHz8eu3fvNo81vVPx3nvvITk5GV5eXigrKwMAbN68GWPGjIGXlxfi4uLwxhtvDGL2iG4OFnq6bXh7e1v95r5v3z7U19dj7969qKioQGdnJ9LS0qDX6/G3v/0NVVVV8PX1RXp6unnchg0bUFpaii1btuDQoUP43//+hx07dvS67rx58/Duu+/i9ddfR11dHd566y34+voiPDwcf/7znwEA9fX1aG5uxmuvvQYAWLt2LbZt24aioiIcP34c2dnZePzxx3HgwAEA134g+dnPfobp06ejtrYWCxYswIoVKwYjbb3y9vaG0WhEV1cXDAYDmpubcfDgwT6PNxgM6Orqwo4dOxze5vTLL7/E1KlTodPp8Mknn6C6uhpPPPGE+Z9hXnvtNWzYsAHr16/Hv/71L6SlpWHGjBn4/PPPreZZsWIFlixZgrq6OqSlpaGsrAy/+93vkJeXh7q6OuTn5+OFF17A1q1b+58QolvRd7sXD9GtyfLucUajUfbu3Ss6nU6effZZc3twcLC0t7ebx7zzzjsSGxsrRqPRfK69vV28vb1lz549IiISEhIiBQUF5vbOzk4ZOXKk1Z3qkpOTZcmSJSIiUl9fLwBk7969inGa7mZlefevtrY28fHxkU8//dSq7/z58+XnP/+5iIg899xzcvfdd1u1L1++3G4uWxEREfLqq68qtjU1NQkAqampUWwvKSmRYcOGmY9PnjwpMTExkpCQICIiXV1dkpWVJQDEYDDIrFmzpLCwUFpbWx3GIyLy/PPPi4eHhwQGBkp6eroUFBSYb5Nq2mtkZKR0dHQojg8NDZW8vDyrc/fff78sWrTIal8bN2606hMVFSV/+MMfrM6tWbNGEhMTe42XaKjhb/SkWhUVFfD19YWXlxcefvhhzJkzB6tWrTK3jx07Flqt1nx89OhRnDp1Cnq9Hr6+vvD19UVgYCDa2trQ0NCA1tZWNDc3Y/LkyeYxHh4eSEhIcBhDbW0t3N3dkZyc3Oe4T506hStXruCHP/yhOQ5fX19s27bN/HZ1XV2dVRwAkJiY2Oc1+qu1tRW+vr7w8fFBbGwsgoODzW+Du7u7o6SkBOfOnUNBQQHCwsKQn5+P+Ph4NDc3O5wzLy8PLS0tKCoqQnx8PIqKihAXF4djx44BuJbDpKQkeHp62o29cOEC/vOf/2DKlClW56dMmYK6ujqrc5Zfp8uXL6OhoQHz58+3yvFLL71kzjGRWnjc7ACIBktKSgrefPNNaLVahIaGwsPD+ul+xx13WB1funQJEydONBcuS3feeWe/YnD0b9O9uXTpEgBg165dCAsLs2rT6XT9imOg6PV6HDlyBG5ubggJCVHcX1hYGDIyMpCRkYE1a9YgJiYGRUVFePHFFx3OO3z4cDzyyCN45JFHkJ+fj/vuuw/r16/H1q1b+5VDJZZfb1OOi4uL7X5gcnd3H5D1iG4VLPSkWnfccQeio6P73H/ChAl47733EBQUBD8/P8U+ISEh+Pvf/46pU6cCALq6ulBdXY0JEyYo9h87diyMRiMOHDiA1NRUu3bTOwqWV6Xffffd0Ol0+OKLLxy+EzBmzBjzhYUmhw8fvvEmvyM3NzenchoQEICQkBCnrqDXarWIiooyj7n33nuxdetWdHZ22v1W7+fnh9DQUFRVVVnlqqqqCpMmTXK4RnBwMEJDQ9HY2Ii5c+f2OTaioYiFnui6uXPn4uWXX8bMmTPNV3GfOXMG5eXlWLZsGUaOHIklS5Zg3bp1uOuuuxAXF4dXXnml17+BHzVqFDIzM/HEE0/g9ddfx7hx43DmzBl89dVXmD17NiIiIqDRaFBRUYFp06bB29sber0ezz77LLKzs2E0GvHggw+itbUVVVVV8PPzQ2ZmJp566ils2LABS5cuxYIFC1BdXY3S0tI+7fPLL7+0+1v5iIgI8+f19fV2Y+Lj428471tvvYXa2lr89Kc/RVRUFNra2rBt2zYcP34chYWFimMqKiqwfft2PProo4iJiYGI4IMPPkBlZSVKSkoAAE8//TQKCwvx6KOP4rnnnsOwYcNw+PBhTJo0CbGxsVi6dClyc3MRFRWF8ePHo6SkBLW1tYrvzFh68cUX8cwzz2DYsGFIT09He3s7/vnPf+Kbb75BTk7ODfdLNGTc7IsEiAaD5cV4zrQ3NzfLvHnzZMSIEaLT6WT06NGycOFC8wVlnZ2dsmTJEvHz8xN/f3/JycmRefPmObwYT0Tk6tWrkp2dLSEhIaLVaiU6Olq2bNlibl+9erUYDAbRaDSSmZkpItcuINy4caPExsaKp6en3HnnnZKWliYHDhwwj/vggw8kOjpadDqdJCUlyZYtW/p0MR4Au4933nnHfNGa0sfZs2ftLsazdeTIEXn88cclMjJSdDqdDB8+XKZOnSo7d+50OKahoUEWLlwoMTEx4u3tLf7+/nL//fdLSUmJVb+jR4/Kj370I/Hx8RG9Xi9JSUnS0NAgIiLd3d2yatUqCQsLE09PTxk3bpx8+OGH5rG9XWRYVlYm48ePF61WKwEBATJ16lQpLy93GC/RUKQRcfA3LURERDTk8ap7IiIiFWOhJyIiUjEWeiIiIhVjoSciIlIxFnoiIiIVY6EnIiJSMRZ6IiIiFWOhJyIiUjEWeiIiIhVjoSciIlIxFnoiIiIVY6EnIiJSsf8DMIiDYEQUuXYAAAAASUVORK5CYII=\n"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 46,
      "metadata": {
        "cellView": "form",
        "id": "1xeTkJCtV0Wt",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "cff30b15-c7ef-48de-eee6-cbdd76b6f7e2"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Files zipped successfully!\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_8c951a24-5096-4e5f-b172-839c60b7c6c2\", \"zipped_files.zip\", 133231)"
            ]
          },
          "metadata": {}
        }
      ],
      "source": [
        "#@title Downloading the results`\n",
        "\n",
        "if os.path.exists(\"d\"):\n",
        "    !rm -rf \"d\"\n",
        "if os.path.exists(\"text_dir\"):\n",
        "    !rm -rf \"text_dir\"\n",
        "\n",
        "!mkdir text_dir\n",
        "!mkdir d\n",
        "c = PPA(Sequence , model3)\n",
        "if Sequence != None and Sequence !=\"\" :\n",
        "  if ID == None:\n",
        "    c.name = \"Your peptide\"\n",
        "  with open(\"seq.txt\",\"w\") as w:\n",
        "          w.write(\"                                                 <<<<<<  LLPS prediction results  >>>>>>\\n\\n\\n\")\n",
        "          w.write(\"[Sequence id] : \" +c.name+\"\\n\\n\")\n",
        "          w.write(\"[Sequence length] : \"+str(len(c.seq))+\"\\n\\n\")\n",
        "          w.write(\"[Sequence_of_peptide] : \" +c.seq+\"\\n\\n\")\n",
        "          w.write(\"[Predicted_score] : \" +score +\"\\n\\n\")\n",
        "          w.close()\n",
        "\n",
        "\n",
        "\n",
        "  if c.name != None:\n",
        "    img_name = \"Info_of_sequenceID:\"+c.name+\".png\"\n",
        "    fig1.write_image(img_name)\n",
        "  else:\n",
        "    name = np.random.randint(1000000)\n",
        "    img_name = \"Info_of_sequenceID:\"+str(name)+\".png\"\n",
        "    fig1.write_image(img_name)\n",
        "\n",
        "  shutil.move(img_name, \"d\")\n",
        "  shutil.move(\"seq.txt\", \"text_dir\")\n",
        "  text_files_path = \"text_dir\"\n",
        "  png_files_path = \"d\"\n",
        "\n",
        "  zip_output_path = 'zipped_files.zip'\n",
        "\n",
        "  text_files = [f for f in os.listdir(text_files_path) if f.endswith('.txt')]\n",
        "  png_files = [f for f in os.listdir(png_files_path) if f.endswith('.png')]\n",
        "\n",
        "  with zipfile.ZipFile(zip_output_path, 'w') as zipf:\n",
        "    for text_file in text_files:\n",
        "      zipf.write(os.path.join(text_files_path, text_file), arcname=text_file)\n",
        "\n",
        "    for png_file in png_files:\n",
        "      zipf.write(os.path.join(png_files_path, png_file), arcname=png_file)\n",
        "\n",
        "  print(\"Files zipped successfully!\")\n",
        "\n",
        "\n",
        "else:\n",
        "  h = pd.DataFrame(D)\n",
        "  h.to_json(\"h.json\")\n",
        "  with open(\"results.txt\",\"w\") as w:\n",
        "    w.write(\"                                                 <<<<<<  LLPS prediction results  >>>>>>\\n\\n\\n\")\n",
        "    w.write(\"Number of sequences : \"+str(len(D[\"seq\"]))+\"\\n\\n\")\n",
        "    for i in range(len(D[\"id\"])):\n",
        "      w.write(\" [Sequence {i}]\".format(i = i)+\"\\n\\n\")\n",
        "      w.write(\"   Sequence id : \"+D[\"id\"][i]+\"\\n\\n\")\n",
        "      w.write(\"   Sequence length : \"+str(len(D[\"seq\"][i]))+\"\\n\\n\")\n",
        "      w.write(\"   Sequence : \" + D[\"seq\"][i]+\"\\n\\n\")\n",
        "      w.write(\"   Predicted Score : \" + str(D[\"score\"][i])+\"\\n\\n\")\n",
        "      w.write(\"-\"*150+\"\\n\\n\")\n",
        "    w.close()\n",
        "    shutil.move(\"h.json\", \"d\")\n",
        "    shutil.move(\"results.txt\", \"text_dir\")\n",
        "    text_files_path = \"text_dir\"\n",
        "    png_files_path = \"d\"\n",
        "    zip_output_path = 'zipped_files.zip'\n",
        "\n",
        "    text_files = [f for f in os.listdir(text_files_path) if f.endswith('.txt')]\n",
        "    png_files = [f for f in os.listdir(png_files_path) if f.endswith('.json')]\n",
        "\n",
        "    with zipfile.ZipFile(zip_output_path, 'w') as zipf:\n",
        "      for text_file in text_files:\n",
        "        zipf.write(os.path.join(text_files_path, text_file), arcname=text_file)\n",
        "\n",
        "      for png_file in png_files:\n",
        "        zipf.write(os.path.join(png_files_path, png_file), arcname=png_file)\n",
        "\n",
        "    print(\"Files zipped successfully!\")\n",
        "\n",
        "files.download(\"zipped_files.zip\")\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "##NCBI BLAST+\n",
        "To determine the BLAST database, you can either provide a FASTA file path to create your own database or use the URL link to NCBI's protein databases available on the NCBI page: https://ftp.ncbi.nlm.nih.gov/blast/db/\n",
        "\n",
        "The DrLLPS database has already been provided. By setting the Blast_Database to DrLLPS, the BLAST search will be conducted against the DrLLPS database.\n",
        "\n",
        "To load your sequences, you can either input a single sequence or upload a FASTA format file containing your sequences into the designated box\n",
        "\n",
        "as example :\n",
        "  **MAVRVAKMTEREKLLLKKYGNRRLYDTKRSAYVTLSQVADIIKQGVDVEVVDAKTQEDVTAFILTQIVLEQARRKNALLPAELLHLIIRYGEDLLQEFFQKYLEQTIHGYIQFKKAMDEQFSKWIDMQLNYSDMAHKTLSGLTRFPNIFSPPPGSPGTKKDKG**\n",
        "\n",
        "or\n",
        "\n",
        "**Your_file.fasta**\n",
        "\n"
      ],
      "metadata": {
        "id": "N1t2f4GlUJ7_"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#@title BLAST+ Tool\n",
        "\n",
        "\n",
        "import os\n",
        "#@markdown - Specify the sequence or the path of your FASTA format file in this box : like  `MAVRVAKMTEREKLL`  or `Your_file.fasta`\n",
        "\n",
        "Sequence = \"Sequences.txt\" #@param {type:\"string\"}\n",
        "#@markdown - Specify the fasta file or URL of your Database here :                    like  `https://ftp.ncbi.nlm.nih.gov/blast/db/swissprot.tar.gz`  or `Your_Database.fasta`\n",
        "\n",
        "Blast_Database = \"https://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/swissprot.gz\" #@param [\"DrLLPS\",\"https://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/swissprot.gz\"] {allow-input : true}\n",
        "Blast_result_format = \"1\" #@param [1,2,3,4,5,6,7]\n",
        "Blast_result_file_name = \"Blast_result.txt\" #@param {type:\"string\"}\n",
        "E_value_threshold = \"100\" #@param {type:\"string\"}\n",
        "Max_target_Sequences = 6 #@param {type:\"integer\"}\n",
        "\n",
        "if Blast_Database == \"DrLLPS\":\n",
        "  Blast_Database = \"DrLLPS.fasta\"\n",
        "\n",
        "import sys\n",
        "from os import path\n",
        "\n",
        "if path.exists(\"/content/bioinformatics_stockholm/\"):\n",
        "  pass\n",
        "else:\n",
        "  !git clone https://github.com/Intertangler/bioinformatics_stockholm\n",
        "sys.path.insert(0,'/content')\n",
        "\n",
        "import importlib.util\n",
        "spec = importlib.util.spec_from_file_location(\"msa\", \"/content/bioinformatics_stockholm/msa/msa.py\")\n",
        "msa = importlib.util.module_from_spec(spec)\n",
        "spec.loader.exec_module(msa)\n",
        "\n",
        "spec = importlib.util.spec_from_file_location(\"miniblast\", \"/content/bioinformatics_stockholm/Miniblast/miniblast.py\")\n",
        "miniblast = importlib.util.module_from_spec(spec)\n",
        "spec.loader.exec_module(miniblast)\n",
        "\n",
        "def Fasta(fasta_path):\n",
        "  from command_runner import command_runner\n",
        "  command = \"makeblastdb -in {k} -dbtype prot -out blastdb\".format(k =fasta_path )\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  print(\"-- Blast Database has been created -- \\n\")\n",
        "\n",
        "\n",
        "if \".fasta\" in Blast_Database or \".txt\" in Blast_Database:\n",
        "  if os.path.exists(Blast_Database):\n",
        "    Fasta(Blast_Database)\n",
        "  with open(Blast_Database) as w:\n",
        "    j = w.read().split(\">\")[1:]\n",
        "  Dict = {i[:i.index(\"\\n\")].split(\" \")[0] if \"\\n\" in i else i : edit(i[i.index(\"\\n\"):]) if \"\\n\" in i else None for i in j}\n",
        "\n",
        "else:\n",
        "  command = \"wget {k}\".format(k = Blast_Database)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  print(\"-- Blast Database has been downloaded -- \\n\")\n",
        "  exit_code, output = command_runner(\"gunzip {l}\".format(l = Blast_Database.split(\"/\")[-1]), shell=True, live_output=True)\n",
        "  Fasta(Blast_Database.split(\"/\")[-1][:-3])\n",
        "  db = miniblast.read_database_fasta(Blast_Database.split(\"/\")[-1][:-3])\n",
        "  Dict = {i.id :\"\".join(list(i.seq)) for i in db}\n"
      ],
      "metadata": {
        "cellView": "form",
        "id": "HC5GzDMDCOgN"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Blast Result\n",
        "\n",
        "blosum62 = {\n",
        "    ('W', 'F'): 1, ('L', 'R'): -2, ('S', 'P'): -1, ('V', 'T'): 0,\n",
        "    ('Q', 'Q'): 5, ('N', 'A'): -2, ('Z', 'Y'): -2, ('W', 'R'): -3,\n",
        "    ('Q', 'A'): -1, ('S', 'D'): 0, ('H', 'H'): 8, ('S', 'H'): -1,\n",
        "    ('H', 'D'): -1, ('L', 'N'): -3, ('W', 'A'): -3, ('Y', 'M'): -1,\n",
        "    ('G', 'R'): -2, ('Y', 'I'): -1, ('Y', 'E'): -2, ('B', 'Y'): -3,\n",
        "    ('Y', 'A'): -2, ('V', 'D'): -3, ('B', 'S'): 0, ('Y', 'Y'): 7,\n",
        "    ('G', 'N'): 0, ('E', 'C'): -4, ('Y', 'Q'): -1, ('Z', 'Z'): 4,\n",
        "    ('V', 'A'): 0, ('C', 'C'): 9, ('M', 'R'): -1, ('V', 'E'): -2,\n",
        "    ('T', 'N'): 0, ('P', 'P'): 7, ('V', 'I'): 3, ('V', 'S'): -2,\n",
        "    ('Z', 'P'): -1, ('V', 'M'): 1, ('T', 'F'): -2, ('V', 'Q'): -2,\n",
        "    ('K', 'K'): 5, ('P', 'D'): -1, ('I', 'H'): -3, ('I', 'D'): -3,\n",
        "    ('T', 'R'): -1, ('P', 'L'): -3, ('K', 'G'): -2, ('M', 'N'): -2,\n",
        "    ('P', 'H'): -2, ('F', 'Q'): -3, ('Z', 'G'): -2, ('X', 'L'): -1,\n",
        "    ('T', 'M'): -1, ('Z', 'C'): -3, ('X', 'H'): -1, ('D', 'R'): -2,\n",
        "    ('B', 'W'): -4, ('X', 'D'): -1, ('Z', 'K'): 1, ('F', 'A'): -2,\n",
        "    ('Z', 'W'): -3, ('F', 'E'): -3, ('D', 'N'): 1, ('B', 'K'): 0,\n",
        "    ('X', 'X'): -1, ('F', 'I'): 0, ('B', 'G'): -1, ('X', 'T'): 0,\n",
        "    ('F', 'M'): 0, ('B', 'C'): -3, ('Z', 'I'): -3, ('Z', 'V'): -2,\n",
        "    ('S', 'S'): 4, ('L', 'Q'): -2, ('W', 'E'): -3, ('Q', 'R'): 1,\n",
        "    ('N', 'N'): 6, ('W', 'M'): -1, ('Q', 'C'): -3, ('W', 'I'): -3,\n",
        "    ('S', 'C'): -1, ('L', 'A'): -1, ('S', 'G'): 0, ('L', 'E'): -3,\n",
        "    ('W', 'Q'): -2, ('H', 'G'): -2, ('S', 'K'): 0, ('Q', 'N'): 0,\n",
        "    ('N', 'R'): 0, ('H', 'C'): -3, ('Y', 'N'): -2, ('G', 'Q'): -2,\n",
        "    ('Y', 'F'): 3, ('C', 'A'): 0, ('V', 'L'): 1, ('G', 'E'): -2,\n",
        "    ('G', 'A'): 0, ('K', 'R'): 2, ('E', 'D'): 2, ('Y', 'R'): -2,\n",
        "    ('M', 'Q'): 0, ('T', 'I'): -1, ('C', 'D'): -3, ('V', 'F'): -1,\n",
        "    ('T', 'A'): 0, ('T', 'P'): -1, ('B', 'P'): -2, ('T', 'E'): -1,\n",
        "    ('V', 'N'): -3, ('P', 'G'): -2, ('M', 'A'): -1, ('K', 'H'): -1,\n",
        "    ('V', 'R'): -3, ('P', 'C'): -3, ('M', 'E'): -2, ('K', 'L'): -2,\n",
        "    ('V', 'V'): 4, ('M', 'I'): 1, ('T', 'Q'): -1, ('I', 'G'): -4,\n",
        "    ('P', 'K'): -1, ('M', 'M'): 5, ('K', 'D'): -1, ('I', 'C'): -1,\n",
        "    ('Z', 'D'): 1, ('F', 'R'): -3, ('X', 'K'): -1, ('Q', 'D'): 0,\n",
        "    ('X', 'G'): -1, ('Z', 'L'): -3, ('X', 'C'): -2, ('Z', 'H'): 0,\n",
        "    ('B', 'L'): -4, ('B', 'H'): 0, ('F', 'F'): 6, ('X', 'W'): -2,\n",
        "    ('B', 'D'): 4, ('D', 'A'): -2, ('S', 'L'): -2, ('X', 'S'): 0,\n",
        "    ('F', 'N'): -3, ('S', 'R'): -1, ('W', 'D'): -4, ('V', 'Y'): -1,\n",
        "    ('W', 'L'): -2, ('H', 'R'): 0, ('W', 'H'): -2, ('H', 'N'): 1,\n",
        "    ('W', 'T'): -2, ('T', 'T'): 5, ('S', 'F'): -2, ('W', 'P'): -4,\n",
        "    ('L', 'D'): -4, ('B', 'I'): -3, ('L', 'H'): -3, ('S', 'N'): 1,\n",
        "    ('B', 'T'): -1, ('L', 'L'): 4, ('Y', 'K'): -2, ('E', 'Q'): 2,\n",
        "    ('Y', 'G'): -3, ('Z', 'S'): 0, ('Y', 'C'): -2, ('G', 'D'): -1,\n",
        "    ('B', 'V'): -3, ('E', 'A'): -1, ('Y', 'W'): 2, ('E', 'E'): 5,\n",
        "    ('Y', 'S'): -2, ('C', 'N'): -3, ('V', 'C'): -1, ('T', 'H'): -2,\n",
        "    ('P', 'R'): -2, ('V', 'G'): -3, ('T', 'L'): -1, ('V', 'K'): -2,\n",
        "    ('K', 'Q'): 1, ('R', 'A'): -1, ('I', 'R'): -3, ('T', 'D'): -1,\n",
        "    ('P', 'F'): -4, ('I', 'N'): -3, ('K', 'I'): -3, ('M', 'D'): -3,\n",
        "    ('V', 'W'): -3, ('W', 'W'): 11, ('M', 'H'): -2, ('P', 'N'): -2,\n",
        "    ('K', 'A'): -1, ('M', 'L'): 2, ('K', 'E'): 1, ('Z', 'E'): 4,\n",
        "    ('X', 'N'): -1, ('Z', 'A'): -1, ('Z', 'M'): -1, ('X', 'F'): -1,\n",
        "    ('K', 'C'): -3, ('B', 'Q'): 0, ('X', 'B'): -1, ('B', 'M'): -3,\n",
        "    ('F', 'C'): -2, ('Z', 'Q'): 3, ('X', 'Z'): -1, ('F', 'G'): -3,\n",
        "    ('B', 'E'): 1, ('X', 'V'): -1, ('F', 'K'): -3, ('B', 'A'): -2,\n",
        "    ('X', 'R'): -1, ('D', 'D'): 6, ('W', 'G'): -2, ('Z', 'F'): -3,\n",
        "    ('S', 'Q'): 0, ('W', 'C'): -2, ('W', 'K'): -3, ('H', 'Q'): 0,\n",
        "    ('L', 'C'): -1, ('W', 'N'): -4, ('S', 'A'): 1, ('L', 'G'): -4,\n",
        "    ('W', 'S'): -3, ('S', 'E'): 0, ('H', 'E'): 0, ('S', 'I'): -2,\n",
        "    ('H', 'A'): -2, ('S', 'M'): -1, ('Y', 'L'): -1, ('Y', 'H'): 2,\n",
        "    ('Y', 'D'): -3, ('E', 'R'): 0, ('X', 'P'): -2, ('G', 'G'): 6,\n",
        "    ('G', 'C'): -3, ('E', 'N'): 0, ('Y', 'T'): -2, ('Y', 'P'): -3,\n",
        "    ('T', 'K'): -1, ('A', 'A'): 4, ('P', 'Q'): -1, ('T', 'C'): -1,\n",
        "    ('V', 'H'): -3, ('T', 'G'): -2, ('I', 'Q'): -3, ('Z', 'T'): -1,\n",
        "    ('C', 'R'): -3, ('V', 'P'): -2, ('P', 'E'): -1, ('M', 'C'): -1,\n",
        "    ('K', 'N'): 0, ('I', 'I'): 4, ('P', 'A'): -1, ('M', 'G'): -3,\n",
        "    ('T', 'S'): 1, ('I', 'E'): -3, ('P', 'M'): -2, ('M', 'K'): -1,\n",
        "    ('I', 'A'): -1, ('P', 'I'): -3, ('R', 'R'): 5, ('X', 'M'): -1,\n",
        "    ('L', 'I'): 2, ('X', 'I'): -1, ('Z', 'B'): 1, ('X', 'E'): -1,\n",
        "    ('Z', 'N'): 0, ('X', 'A'): 0, ('B', 'R'): -1, ('B', 'N'): 3,\n",
        "    ('F', 'D'): -3, ('X', 'Y'): -1, ('Z', 'R'): 0, ('F', 'H'): -1,\n",
        "    ('B', 'F'): -3, ('F', 'L'): 0, ('X', 'Q'): -1, ('B', 'B'): 4\n",
        "}\n",
        "\n",
        "Blosum62 = {}\n",
        "for i in blosum62:\n",
        "  Blosum62[i] = blosum62[i]\n",
        "  Blosum62[i[::-1]] = blosum62[i]\n",
        "if \".\" not in Sequence:\n",
        "  with open(\"sequence.txt\",\"w\") as w:\n",
        "    w.write(\">seq1\\n{o}\".format(o = Sequence))\n",
        "    w.close()\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = \"sequence.txt\",d = \"blastdb\"\n",
        "  ,f = Blast_result_format,o = Blast_result_file_name , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  files.download(Blast_result_file_name)\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = \"sequence.txt\",d = \"blastdb\"\n",
        "  ,f = 6 ,o = \"results_dict.txt\" , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "\n",
        "  with open(\"results_dict.txt\") as w:\n",
        "    y = w.read()\n",
        "  idx = [i.split(\"\\t\")[1] for i in y.split(\"\\nseq\")]\n",
        "  for i in idx:\n",
        "    target_sequence = Dict[i]\n",
        "    aln = pairwise2.align.localds(Sequence,target_sequence, match_dict = Blosum62, open = -3, extend = -1)[0]\n",
        "    print(\"Subject_Id : \"+i+\"\\n\")\n",
        "    print(pairwise2.format_alignment(align1=aln[0], align2=aln[1], score=aln[2], begin=aln[3], end=aln[4], full_sequences=True))\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "if \".\" in Sequence:\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = Sequence,d = \"blastdb\"\n",
        "  ,f = Blast_result_format,o = Blast_result_file_name , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  files.download(Blast_result_file_name)"
      ],
      "metadata": {
        "cellView": "form",
        "id": "Fd2CuI4MHUuM",
        "outputId": "6a0b5f9d-e893-4019-f6c6-29fa8a5529cb",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 17
        }
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_096f37ac-15df-4ad8-8aaf-2c9815fca5ab\", \"Blast_result.txt\", 783817)"
            ]
          },
          "metadata": {}
        }
      ]
    }
  ],
  "metadata": {
    "accelerator": "GPU",
    "colab": {
      "gpuType": "T4",
      "machine_shape": "hm",
      "provenance": [],
      "include_colab_link": true
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}