{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/AMIRMOHAMMAD-OSS/Deep-generative-transformer-based-model-for-de-novo-design-of-proteins/blob/main/LLPS_%2B.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#Liquid-Liquid Phase Separation (LLPS) Protein Scorer!\n",
        "\n",
        "This tool allows you to analyze protein sequences for their propensity to undergo phase separation:\n",
        "\n",
        "Input Options:\n",
        "Upload a single protein sequence directly into the designated box.\n",
        "Alternatively, load a FASTA or TXT file containing multiple protein sequences.\n",
        "\n",
        "To maintain efficiency, we advise against loading more than 100 sequences at once\n",
        "\n",
        "Specify the sequence ID or directory for your analysis.\n",
        "\n",
        "By setting the 'End_Sequence'  parameter,you can focus the scoring on sequences up to the specified endpoint.\n",
        "\n",
        "         \n",
        "\n",
        "\n",
        "\n",
        "#BLAST+ tool\n",
        "You can utilize the BLAST+ tool to run ncbi BLAST searches on either your protein sequence or your FASTA file against your dataset. This powerful tool allows you to compare sequences and identify homologous regions, aiding in various biological analyses.\n"
      ],
      "metadata": {
        "id": "XJ8r7M6CJzke"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "hyzTHwf8dRBy"
      },
      "outputs": [],
      "source": [
        "#@title Importing libraries\n",
        "!git clone https://github.com/AMIRMOHAMMAD-OSS/classi\n",
        "!pip install biopython\n",
        "!apt-get update\n",
        "!apt-get install -y ncbi-blast+\n",
        "!pip install streamlit\n",
        "!pip install py3Dmol\n",
        "!pip install requests\n",
        "!pip install biotite\n",
        "!pip install stmol\n",
        "!pip install command_runner\n",
        "from command_runner import command_runner\n",
        "!pip install plotly\n",
        "!pip install --upgrade \"kaleido==0.1.*\"\n",
        "import kaleido\n",
        "\n",
        "from Bio import SeqIO\n",
        "from Bio.Seq import Seq\n",
        "from Bio import pairwise2\n",
        "from google.colab import files\n",
        "import plotly.graph_objects as go\n",
        "from plotly.subplots import make_subplots\n",
        "import zipfile\n",
        "import shutil\n",
        "from transformers import AutoTokenizer\n",
        "from tqdm import tqdm\n",
        "#!pip install slack-sdk\n",
        "#from tqdm.contrib.slack import tqdm, trange\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "import numpy as np\n",
        "import time\n",
        "from collections import defaultdict\n",
        "import torch\n",
        "from torch.utils.data.dataloader import DataLoader\n",
        "import time\n",
        "import os\n",
        "import sys\n",
        "import json\n",
        "import random\n",
        "from ast import literal_eval\n",
        "import numpy as np\n",
        "import torch\n",
        "from transformers import AutoTokenizer\n",
        "from transformers import AutoTokenizer\n",
        "from tokenizers import Tokenizer, decoders, models, normalizers, pre_tokenizers, trainers\n",
        "from transformers import CanineTokenizer, CanineModel\n",
        "tokenizer = Tokenizer.from_file(\"classi/Trained_BPE2.json\")\n",
        "tokenizer.model_max_length = 256\n",
        "tokenizer\n",
        "\n",
        "from sklearn.preprocessing import StandardScaler as Sc\n",
        "from sklearn.decomposition import PCA\n",
        "from sklearn.model_selection import train_test_split as TTS\n",
        "import numpy as np\n",
        "\n",
        "import torch\n",
        "\n",
        "from sklearn.model_selection import train_test_split as TTS\n",
        "\n",
        "\n",
        "def edit(x):\n",
        "    for i in x:\n",
        "      if i in \"BJOUX\\Z_n\\n\":\n",
        "        x = x.replace(i,\"\")\n",
        "      else:\n",
        "        pass\n",
        "    return x\n",
        "\n",
        "def encode(x):\n",
        "  l = []\n",
        "  chars = tokenizer.get_vocab()\n",
        "  for i in x:\n",
        "    l.append(chars[i])\n",
        "  return l\n",
        "\n",
        "with open(\"classi/training.txt\",\"r\",encoding=\"utf-8\") as f:\n",
        "  text = f.read()\n",
        "  k = \"#\".join([i for i in edit(text).split(\"<|edoftext|>\")])\n",
        "chars = tokenizer.get_vocab()\n",
        "vocab_size = len(tokenizer.get_vocab())\n",
        "\n",
        "with open(\"classi/validation.txt\",\"r\",encoding=\"utf-8\") as f:\n",
        "  text = f.read()\n",
        "  l = \"#\".join([i for i in edit(text).split(\"<|edoftext|>\")])\n",
        "\n",
        "def Padding(x,PAD = 0,max_len = 512):\n",
        "  l = []\n",
        "  max_len = max(512,len(max(x,key = lambda x: len(x))))\n",
        "  for i in x:\n",
        "    a = i\n",
        "    if len(a) < max_len:\n",
        "      a = a + [PAD for i in range(max_len-len(a))]\n",
        "    l.append(a)\n",
        "  return np.array(l).reshape(len(x),max_len)\n",
        "with open(\"classi/negative_dataset.txt\") as x:\n",
        "  c = x.read()\n",
        "Filtered_train  = [i for i in k.split(\"#\") if len(i)<= 512][1:]\n",
        "Filtered_val = [i for i in l.split(\"#\") if len(i)<= 512][1:]\n",
        "Filtered_neg = [i for i in c.split(\"\\n\") if len(i)<= 512]\n",
        "pos = Filtered_train + Filtered_val\n",
        "pos2 = [i for i in k.split(\"#\")][1:] +  [i for i in l.split(\"#\")][1:]\n",
        "with open(\"DrLLPS.fasta\",\"w\") as w:\n",
        "  for i in range(len(pos2)):\n",
        "    w.write(\">seq{i}\\n\".format(i=i))\n",
        "    w.write(pos2[i]+\"\\n\")\n",
        "  w.close()\n",
        "\n",
        "files.download(\"DrLLPS.fasta\")\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "cellView": "form",
        "id": "Yc9VXFLYN-x6"
      },
      "outputs": [],
      "source": [
        "#@title Loading models and requierments\n",
        "\n",
        "def set_seed(seed):\n",
        "    random.seed(seed)\n",
        "    np.random.seed(seed)\n",
        "    torch.manual_seed(seed)\n",
        "    torch.cuda.manual_seed_all(seed)\n",
        "\n",
        "def setup_logging(config):\n",
        "    work_dir = config.system.work_dir\n",
        "    os.makedirs(work_dir, exist_ok=True)\n",
        "    with open(os.path.join(work_dir, 'args.txt'), 'w') as f:\n",
        "        f.write(' '.join(sys.argv))\n",
        "    with open(os.path.join(work_dir, 'config.json'), 'w') as f:\n",
        "        f.write(json.dumps(config.to_dict(), indent=4))\n",
        "\n",
        "class CfgNode:\n",
        "    def __init__(self, **kwargs):\n",
        "        self.__dict__.update(kwargs)\n",
        "\n",
        "    def __str__(self):\n",
        "        return self._str_helper(0)\n",
        "\n",
        "    def _str_helper(self, indent):\n",
        "        parts = []\n",
        "        for k, v in self.__dict__.items():\n",
        "            if isinstance(v, CfgNode):\n",
        "                parts.append(\"%s:\\n\" % k)\n",
        "                parts.append(v._str_helper(indent + 1))\n",
        "            else:\n",
        "                parts.append(\"%s: %s\\n\" % (k, v))\n",
        "        parts = [' ' * (indent * 4) + p for p in parts]\n",
        "        return \"\".join(parts)\n",
        "\n",
        "    def to_dict(self):\n",
        "        return { k: v.to_dict() if isinstance(v, CfgNode) else v for k, v in self.__dict__.items() }\n",
        "\n",
        "    def merge_from_dict(self, d):\n",
        "        self.__dict__.update(d)\n",
        "\n",
        "    def merge_from_args(self, args):\n",
        "\n",
        "        for arg in args:\n",
        "            keyval = arg.split('=')\n",
        "            assert len(keyval) == 2, \"expecting each override arg to be of form --arg=value, got %s\" % arg\n",
        "            key, val = keyval\n",
        "            try:\n",
        "                val = literal_eval(val)\n",
        "            except ValueError:\n",
        "                pass\n",
        "\n",
        "            assert key[:2] == '--'\n",
        "            key = key[2:]\n",
        "            keys = key.split('.')\n",
        "            obj = self\n",
        "            for k in keys[:-1]:\n",
        "                obj = getattr(obj, k)\n",
        "            leaf_key = keys[-1]\n",
        "            assert hasattr(obj, leaf_key), f\"{key} is not an attribute that exists in the config\"\n",
        "            print(\"command line overwriting config attribute %s with %s\" % (key, val))\n",
        "            setattr(obj, leaf_key, val)\n",
        "\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "\n",
        "class SiLU(nn.Module):\n",
        "   def forward(self, x):\n",
        "        return x*F.sigmoid(x)\n",
        "\n",
        "class NY(nn.Module):\n",
        "  def forward(self,x):\n",
        "    return 3*torch.tanh(0.3*x)\n",
        "\n",
        "class NewGELU(nn.Module):\n",
        "    def forward(self, x):\n",
        "        return 0.5 * x * (1.0 + torch.tanh(math.sqrt(2.0 / math.pi) * (x + 0.044715 * torch.pow(x, 3.0))))\n",
        "\n",
        "class SelfAttention(nn.Module):\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        assert config.n_embd % config.n_head == 0\n",
        "        self.c_attn = nn.Linear(config.n_embd, 3 * config.n_embd)\n",
        "        self.c_proj = nn.Linear(config.n_embd, config.n_embd)\n",
        "        self.attn_dropout = nn.Dropout(config.attn_pdrop)\n",
        "        self.resid_dropout = nn.Dropout(config.resid_pdrop)\n",
        "        self.register_buffer(\"bias\", torch.tril(torch.ones(config.block_size, config.block_size))\n",
        "                                     .view(1, 1, config.block_size, config.block_size))\n",
        "        self.n_head = config.n_head\n",
        "        self.n_embd = config.n_embd\n",
        "\n",
        "    def forward(self, x):\n",
        "        B, T, C = x.size()\n",
        "        q, k ,v  = self.c_attn(x).split(self.n_embd, dim=2)\n",
        "        k = k.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        q = q.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        v = v.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        att = (q @ k.transpose(-2, -1)) * (1.0 / math.sqrt(k.size(-1)))\n",
        "        att = att.masked_fill(self.bias[:,:,:T,:T] == 0, float('-inf'))\n",
        "        att = F.softmax(att, dim=-1)\n",
        "        att = self.attn_dropout(att)\n",
        "        y = att @ v\n",
        "        y = y.transpose(1, 2).contiguous().view(B, T, C)\n",
        "        y = self.resid_dropout(self.c_proj(y))\n",
        "        return y\n",
        "\n",
        "class Block(nn.Module):\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.ln_1 = nn.LayerNorm(config.n_embd)\n",
        "        self.attn = SelfAttention(config)\n",
        "        self.ln_2 = nn.LayerNorm(config.n_embd)\n",
        "        self.mlp = nn.ModuleDict(dict(\n",
        "            c_fc    = nn.Linear(config.n_embd, 4 * config.n_embd),\n",
        "            c_proj  = nn.Linear(4 * config.n_embd, config.n_embd),\n",
        "            act     = NewGELU(),\n",
        "            dropout = nn.Dropout(config.resid_pdrop),\n",
        "        ))\n",
        "        m = self.mlp\n",
        "        self.mlpf = lambda x: m.dropout(m.c_proj(m.act(m.c_fc(x))))\n",
        "    def forward(self, x):\n",
        "        x = x + self.attn(self.ln_1(x))\n",
        "        x = x + self.mlpf(self.ln_2(x))\n",
        "        return x\n",
        "\n",
        "class ClassifierII(nn.Module):\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.model_type = 'gpt'\n",
        "        C.n_layer = None\n",
        "        C.n_head = None\n",
        "        C.n_embd =  None\n",
        "        C.vocab_size = len(chars)\n",
        "        C.max_length = 512\n",
        "        C.embd_pdrop = 0.1\n",
        "        C.resid_pdrop = 0.1\n",
        "        C.attn_pdrop = 0.1\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.max_length = config.block_size\n",
        "        self.soft = nn.Softmax(1)\n",
        "        self.config = self.get_default_config()\n",
        "        self.device = \"cuda\"\n",
        "        self.model_states = {'h':{'n_layer': 48, 'n_head': 25, 'n_embd': 1600},\n",
        "                                    'g':{'n_layer': 12, '': 12, 'n_embd': 768},\n",
        "                'f':   {'n_layer': 24, 'n_head': 16, 'n_embd': 1024},\n",
        "                'e':   {'n_layer': 36, 'n_head': 20, 'n_embd': 1280},\n",
        "                'd':{'n_layer': 8, 'n_head': 16, 'n_embd': 512},\n",
        "                'c':{'n_layer': 6, 'n_head': 6, 'n_embd': 192},\n",
        "                'b':{'n_layer': 4, 'n_head': 4, 'n_embd': 128},\n",
        "                'a':{'n_layer': 3, 'n_head': 3, 'n_embd': 48}}\n",
        "\n",
        "        type_ = config.model_type is not None\n",
        "        #assert type_ in \"abcdefgh\"\n",
        "        p = all([config.n_layer is not None, config.n_head is not None, config.n_embd is not None])\n",
        "        #assert type_ == True and p == True\n",
        "        if type_:\n",
        "            config.merge_from_dict(self.model_states[config.model_type])\n",
        "        self.closs = nn.BCELoss()\n",
        "        self.ny = NY()\n",
        "        self.l = nn.Linear(512,1,64)\n",
        "        self.transformer = nn.ModuleDict(dict(\n",
        "            wte = nn.Embedding(config.vocab_size, config.n_embd),\n",
        "            wpe = nn.Embedding(config.max_length, config.n_embd),\n",
        "            drop = nn.Dropout(config.embd_pdrop),\n",
        "            h = nn.ModuleList([Block(config) for _ in range(config.n_layer)]),\n",
        "            ln_f = nn.LayerNorm(config.n_embd),))\n",
        "        self.classifier_head = nn.Sequential(#nn.Tanh(),\n",
        "                                             nn.Linear(config.n_embd, 2)\n",
        "                                             )\n",
        "        self.apply(self._init_weights)\n",
        "        for pn, p in self.named_parameters():\n",
        "            if pn.endswith('c_proj.weight'):\n",
        "                torch.nn.init.normal_(p, mean=0.0, std=0.02/math.sqrt(2 * config.n_layer))\n",
        "\n",
        "        n_params = sum(p.numel() for p in self.transformer.parameters())\n",
        "        print(\"[ Number of trainable parameters: %.2fM ]\" % (n_params/1e6,))\n",
        "\n",
        "    def _init_weights(self, module):\n",
        "        if isinstance(module, nn.Linear):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "            if module.bias is not None:\n",
        "                torch.nn.init.zeros_(module.bias)\n",
        "        elif isinstance(module, nn.Embedding):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "        elif isinstance(module, nn.LayerNorm):\n",
        "            torch.nn.init.zeros_(module.bias)\n",
        "            torch.nn.init.ones_(module.weight)\n",
        "\n",
        "    @classmethod\n",
        "    def from_pretrained(cls, model_type):\n",
        "        from transformers import GPT2LMHeadModel\n",
        "        config = cls.get_default_config()\n",
        "        config.model_type = model_type\n",
        "        config.vocab_size = 25\n",
        "        config.max_length = 512\n",
        "        model = ClassifierII(config)\n",
        "        sd = model.state_dict()\n",
        "        model_hf = GPT2LMHeadModel.from_pretrained(model_type)\n",
        "        sd_hf = model_hf.state_dict()\n",
        "        keys = [k for k in sd_hf if not k.endswith('attn.masked_bias')]\n",
        "        transposed = ['attn.c_attn.weight', 'attn.c_proj.weight', 'mlp.c_fc.weight', 'mlp.c_proj.weight']\n",
        "        assert len(keys) == len(sd)\n",
        "        for k in keys:\n",
        "            if any(k.endswith(w) for w in transposed):\n",
        "                assert sd_hf[k].shape[::-1] == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k].t())\n",
        "            else:\n",
        "                assert sd_hf[k].shape == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k])\n",
        "        return model\n",
        "\n",
        "    def configure_optimizers(self, train_config):\n",
        "        decay = set()\n",
        "        no_decay = set()\n",
        "        whitelist_weight_modules = (torch.nn.Linear, )\n",
        "        blacklist_weight_modules = (torch.nn.LayerNorm, torch.nn.Embedding)\n",
        "        for mn, m in self.named_modules():\n",
        "            for pn, p in m.named_parameters():\n",
        "                fpn = '%s.%s' % (mn, pn) if mn else pn\n",
        "                if pn.endswith('bias'):\n",
        "                    no_decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, whitelist_weight_modules):\n",
        "                    decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, blacklist_weight_modules):\n",
        "                    no_decay.add(fpn)\n",
        "        param_dict = {pn: p for pn, p in self.named_parameters()}\n",
        "        inter_params = decay & no_decay\n",
        "        union_params = decay | no_decay\n",
        "        assert len(inter_params) == 0, \"parameters %s made it into both decay/no_decay sets!\" % (str(inter_params), )\n",
        "        assert len(param_dict.keys() - union_params) == 0, \"parameters %s were not separated into either decay/no_decay set!\" \\\n",
        "                                                    % (str(param_dict.keys() - union_params), )\n",
        "        optim_groups = [\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(decay))], \"weight_decay\": train_config.weight_decay},\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(no_decay))], \"weight_decay\": 0.0},]\n",
        "        optimizer = torch.optim.AdamW(optim_groups, lr=train_config.learning_rate, betas=train_config.betas)\n",
        "        return optimizer\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict_proba(self,idx,j =\"None\"):\n",
        "      sigmoid = nn.Sigmoid()\n",
        "      soft = nn.Softmax(dim=1)\n",
        "      ny = NY()\n",
        "      si = SiLU()\n",
        "      self.eval()\n",
        "      x,_ = self.forward(idx)\n",
        "      if j == \"None\":\n",
        "        return x[:,0:1]\n",
        "      elif j == \"soft\":\n",
        "        return self.soft(x)[:,0:1]\n",
        "      else:\n",
        "        return sigmoid(x)[:,0:1]\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict(self,idx):\n",
        "      g = torch.zeros((idx.shape[0],1))\n",
        "      e = -1\n",
        "      for i in self.predict_proba(idx):\n",
        "        e+=1\n",
        "        if i[0].item()>=0.5:\n",
        "          g[e] = 1\n",
        "      return g\n",
        "\n",
        "    def forward(self, idx, targets=None):\n",
        "        device = idx.device\n",
        "        b, t = idx.size()\n",
        "        pos = torch.arange(0, t, dtype=torch.long, device=device).unsqueeze(0)\n",
        "        tok_emb = self.transformer.wte(idx)\n",
        "        pos_emb = self.transformer.wpe(pos)\n",
        "        x = self.transformer.drop(tok_emb + pos_emb)\n",
        "        for block in self.transformer.h:\n",
        "            x = block(x)\n",
        "        x = self.transformer.ln_f(x)\n",
        "        logits = self.classifier_head(x).view(b,2,t).to(device)\n",
        "        logits = self.l(logits)\n",
        "        logits = F.sigmoid(self.ny(logits).view(b,2).mean(1).view(b,1))\n",
        "        loss = None\n",
        "        if targets is not None:\n",
        "            loss = F.binary_cross_entropy(logits,targets)\n",
        "        return logits, loss\n",
        "\n",
        "\n",
        "class Trainer:\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.device = 'cuda'\n",
        "        C.num_workers = 4\n",
        "        C.max_iters = None\n",
        "        C.batch_size = 128\n",
        "        C.max_length = 512\n",
        "        C.learning_rate = 8e-4\n",
        "        C.betas = (0.9, 0.95)\n",
        "        C.weight_decay = 0.1\n",
        "        C.grad_norm_clip = 1.0\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config, model, train_dataset,val_dataset):\n",
        "        self.config = config\n",
        "        self.model = model\n",
        "        self.optimizer = None\n",
        "        self.train_dataset = train_dataset\n",
        "        self.val_dataset = val_dataset\n",
        "        self.callbacks = defaultdict(list)\n",
        "        self.device ='cuda'\n",
        "        self.model = self.model.to(self.device)\n",
        "        self.iter_num = 0\n",
        "        self.iter_time = 0.0\n",
        "        self.iter_dt = 0.0\n",
        "\n",
        "    def add_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent].append(callback)\n",
        "\n",
        "    def set_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent] = [callback]\n",
        "\n",
        "    def trigger_callbacks(self, onevent: str):\n",
        "        for callback in self.callbacks.get(onevent, []):\n",
        "            callback(self)\n",
        "\n",
        "    def run(self):\n",
        "        model, config = self.model, self.config\n",
        "        self.optimizer = model.configure_optimizers(config)\n",
        "        batch_size = self.config.batch_size\n",
        "        def get_batch(mode):\n",
        "            batch_size = 64\n",
        "            if mode == \"train\":\n",
        "              data = train_dataset\n",
        "              pos_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]),513))\n",
        "              neg_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]),513))\n",
        "              N = np.random.randint(batch_size)\n",
        "              ix_pos = np.random.randint(pos_train.shape[0]-N)\n",
        "              ix_neg = np.random.randint(neg_train.shape[0]-batch_size-N)\n",
        "              pos_data = pos_train[ix_pos:ix_pos+N,:]\n",
        "              neg_data = neg_train[ix_neg:ix_neg+batch_size-N,:]\n",
        "              data = torch.concat((pos_data,neg_data))\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,1), device=\"cuda\")\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o] = 1\n",
        "                #else:\n",
        "                  #targets[o][1] = 1\n",
        "            else:\n",
        "              data = val_dataset\n",
        "              ix = np.random.randint(data.shape[0]-batch_size)\n",
        "              data = data[ix:ix+batch_size,:]\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,1), device=\"cuda\")\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o] = 1\n",
        "                #else:\n",
        "                 # targets[o][1] = 1\n",
        "            targets = targets.view(batch_size,1).to(\"cuda\")\n",
        "            seq = seq.view(batch_size,512).to(\"cuda\")\n",
        "            return seq, targets\n",
        "\n",
        "        @torch.no_grad\n",
        "        def cross_val():\n",
        "          model.eval()\n",
        "          out = []\n",
        "          for i in [\"train\",\"val\"]:\n",
        "            losses = torch.zeros(200+1)\n",
        "            for k in range(200):\n",
        "              X,Y = get_batch(i)\n",
        "              logits,loss = model(X,Y)\n",
        "              losses[k]=loss.item()\n",
        "              out1 = losses.mean()\n",
        "            out.append(out1)\n",
        "          model.train()\n",
        "          return out\n",
        "        losses = cross_val()\n",
        "        LOSS = [losses]\n",
        "        print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        model.train()\n",
        "        for epoch in range(20):\n",
        "          print(\"[epoch {o}] \\n\".format(o = epoch))\n",
        "          iters = 200\n",
        "          for i in range(iters):\n",
        "            if i == 0:\n",
        "              Y = \"| =\"\n",
        "            elif i == iters -1 :\n",
        "              Y = \"=> 100% |\"\n",
        "            else:\n",
        "              if i%(int(iters/50)) == 0 :\n",
        "                Y = \"=\"\n",
        "              else:\n",
        "                Y = \"\"\n",
        "            print(\"{y}\".format(y = Y),end=\"\")\n",
        "            x, y = get_batch(\"train\")\n",
        "            logits, self.loss = model(x, y)\n",
        "            model.zero_grad(set_to_none=True)\n",
        "            self.loss.backward()\n",
        "            torch.nn.utils.clip_grad_norm_(model.parameters(), config.grad_norm_clip)\n",
        "            self.optimizer.step()\n",
        "          losses = cross_val()\n",
        "          LOSS.append(losses)\n",
        "          print(LOSS)\n",
        "          print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        PATH = \"model {h}\".format(h = model_config.model_type)\n",
        "        torch.save(model.state_dict(), PATH)\n",
        "\n",
        "def set_seed(seed):\n",
        "    random.seed(seed)\n",
        "    np.random.seed(seed)\n",
        "    torch.manual_seed(seed)\n",
        "    torch.cuda.manual_seed_all(seed)\n",
        "\n",
        "def setup_logging(config):\n",
        "    work_dir = config.system.work_dir\n",
        "    os.makedirs(work_dir, exist_ok=True)\n",
        "    with open(os.path.join(work_dir, 'args.txt'), 'w') as f:\n",
        "        f.write(' '.join(sys.argv))\n",
        "    with open(os.path.join(work_dir, 'config.json'), 'w') as f:\n",
        "        f.write(json.dumps(config.to_dict(), indent=4))\n",
        "\n",
        "class CfgNode:\n",
        "    def __init__(self, **kwargs):\n",
        "        self.__dict__.update(kwargs)\n",
        "\n",
        "    def __str__(self):\n",
        "        return self._str_helper(0)\n",
        "\n",
        "    def _str_helper(self, indent):\n",
        "        parts = []\n",
        "        for k, v in self.__dict__.items():\n",
        "            if isinstance(v, CfgNode):\n",
        "                parts.append(\"%s:\\n\" % k)\n",
        "                parts.append(v._str_helper(indent + 1))\n",
        "            else:\n",
        "                parts.append(\"%s: %s\\n\" % (k, v))\n",
        "        parts = [' ' * (indent * 4) + p for p in parts]\n",
        "        return \"\".join(parts)\n",
        "\n",
        "    def to_dict(self):\n",
        "        return { k: v.to_dict() if isinstance(v, CfgNode) else v for k, v in self.__dict__.items() }\n",
        "\n",
        "    def merge_from_dict(self, d):\n",
        "        self.__dict__.update(d)\n",
        "\n",
        "    def merge_from_args(self, args):\n",
        "\n",
        "        for arg in args:\n",
        "            keyval = arg.split('=')\n",
        "            assert len(keyval) == 2, \"expecting each override arg to be of form --arg=value, got %s\" % arg\n",
        "            key, val = keyval\n",
        "            try:\n",
        "                val = literal_eval(val)\n",
        "            except ValueError:\n",
        "                pass\n",
        "\n",
        "            assert key[:2] == '--'\n",
        "            key = key[2:]\n",
        "            keys = key.split('.')\n",
        "            obj = self\n",
        "            for k in keys[:-1]:\n",
        "                obj = getattr(obj, k)\n",
        "            leaf_key = keys[-1]\n",
        "            assert hasattr(obj, leaf_key), f\"{key} is not an attribute that exists in the config\"\n",
        "            print(\"command line overwriting config attribute %s with %s\" % (key, val))\n",
        "            setattr(obj, leaf_key, val)\n",
        "\n",
        "import math\n",
        "import torch\n",
        "import torch.nn as nn\n",
        "from torch.nn import functional as F\n",
        "\n",
        "class SiLU(nn.Module):\n",
        "   def forward(self, x):\n",
        "        return x*F.sigmoid(x)\n",
        "\n",
        "class NY(nn.Module):\n",
        "  def forward(self,x):\n",
        "    return 3*torch.tanh(0.3*x)\n",
        "\n",
        "class NewGELU(nn.Module):\n",
        "    def forward(self, x):\n",
        "        return 0.5 * x * (1.0 + torch.tanh(math.sqrt(2.0 / math.pi) * (x + 0.044715 * torch.pow(x, 3.0))))\n",
        "\n",
        "class SelfAttention(nn.Module):\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        assert config.n_embd % config.n_head == 0\n",
        "        self.c_attn = nn.Linear(config.n_embd, 3 * config.n_embd)\n",
        "        self.c_proj = nn.Linear(config.n_embd, config.n_embd)\n",
        "        self.attn_dropout = nn.Dropout(config.attn_pdrop)\n",
        "        self.resid_dropout = nn.Dropout(config.resid_pdrop)\n",
        "        self.register_buffer(\"bias\", torch.tril(torch.ones(config.block_size, config.block_size))\n",
        "                                     .view(1, 1, config.block_size, config.block_size))\n",
        "        self.n_head = config.n_head\n",
        "        self.n_embd = config.n_embd\n",
        "\n",
        "    def forward(self, x):\n",
        "        B, T, C = x.size()\n",
        "        q, k ,v  = self.c_attn(x).split(self.n_embd, dim=2)\n",
        "        k = k.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        q = q.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        v = v.view(B, T, self.n_head, C // self.n_head).transpose(1, 2)\n",
        "        att = (q @ k.transpose(-2, -1)) * (1.0 / math.sqrt(k.size(-1)))\n",
        "        att = att.masked_fill(self.bias[:,:,:T,:T] == 0, float('-inf'))\n",
        "        att = F.softmax(att, dim=-1)\n",
        "        att = self.attn_dropout(att)\n",
        "        y = att @ v\n",
        "        y = y.transpose(1, 2).contiguous().view(B, T, C)\n",
        "        y = self.resid_dropout(self.c_proj(y))\n",
        "        return y\n",
        "\n",
        "class Block(nn.Module):\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.ln_1 = nn.LayerNorm(config.n_embd)\n",
        "        self.attn = SelfAttention(config)\n",
        "        self.ln_2 = nn.LayerNorm(config.n_embd)\n",
        "        self.mlp = nn.ModuleDict(dict(\n",
        "            c_fc    = nn.Linear(config.n_embd, 4 * config.n_embd),\n",
        "            c_proj  = nn.Linear(4 * config.n_embd, config.n_embd),\n",
        "            act     = NewGELU(),\n",
        "            dropout = nn.Dropout(config.resid_pdrop),\n",
        "        ))\n",
        "        m = self.mlp\n",
        "        self.mlpf = lambda x: m.dropout(m.c_proj(m.act(m.c_fc(x))))\n",
        "    def forward(self, x):\n",
        "        x = x + self.attn(self.ln_1(x))\n",
        "        x = x + self.mlpf(self.ln_2(x))\n",
        "        return x\n",
        "\n",
        "class ClassifierI(nn.Module):\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.model_type = 'gpt'\n",
        "        C.n_layer = None\n",
        "        C.n_head = None\n",
        "        C.n_embd =  None\n",
        "        C.vocab_size = len(chars)\n",
        "        C.max_length = 512\n",
        "        C.embd_pdrop = 0.1\n",
        "        C.resid_pdrop = 0.1\n",
        "        C.attn_pdrop = 0.1\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config):\n",
        "        super().__init__()\n",
        "        self.max_length = config.block_size\n",
        "        self.soft = nn.Softmax(1)\n",
        "        self.config = self.get_default_config()\n",
        "        self.device = \"cpu\"\n",
        "        self.model_states = {'h':{'n_layer': 48, 'n_head': 25, 'n_embd': 1600},\n",
        "                                    'g':{'n_layer': 12, '': 12, 'n_embd': 768},\n",
        "                'f':   {'n_layer': 24, 'n_head': 16, 'n_embd': 1024},\n",
        "                'e':   {'n_layer': 36, 'n_head': 20, 'n_embd': 1280},\n",
        "                'd':{'n_layer': 8, 'n_head': 16, 'n_embd': 512},\n",
        "                'c':{'n_layer': 6, 'n_head': 6, 'n_embd': 192},\n",
        "                'b':{'n_layer': 4, 'n_head': 4, 'n_embd': 128},\n",
        "                'a':{'n_layer': 3, 'n_head': 3, 'n_embd': 48}}\n",
        "\n",
        "        type_ = config.model_type is not None\n",
        "        #assert type_ in \"abcdefgh\"\n",
        "        p = all([config.n_layer is not None, config.n_head is not None, config.n_embd is not None])\n",
        "        #assert type_ == True and p == True\n",
        "        if type_:\n",
        "            config.merge_from_dict(self.model_states[config.model_type])\n",
        "        self.closs = nn.BCELoss()\n",
        "        self.ny = NY()\n",
        "        #self.l = nn.Linear(512,1,64)\n",
        "        self.transformer = nn.ModuleDict(dict(\n",
        "            wte = nn.Embedding(config.vocab_size, config.n_embd),\n",
        "            wpe = nn.Embedding(config.max_length, config.n_embd),\n",
        "            drop = nn.Dropout(config.embd_pdrop),\n",
        "            h = nn.ModuleList([Block(config) for _ in range(config.n_layer)]),\n",
        "            ln_f = nn.LayerNorm(config.n_embd),))\n",
        "        self.classifier_head = nn.Sequential(#nn.Tanh(),\n",
        "                                             nn.Linear(config.n_embd, 2)\n",
        "                                             )\n",
        "        self.apply(self._init_weights)\n",
        "        for pn, p in self.named_parameters():\n",
        "            if pn.endswith('c_proj.weight'):\n",
        "                torch.nn.init.normal_(p, mean=0.0, std=0.02/math.sqrt(2 * config.n_layer))\n",
        "\n",
        "        n_params = sum(p.numel() for p in self.transformer.parameters())\n",
        "        #print(\"[ Number of trainable parameters: %.2fM ]\" % (n_params/1e6,))\n",
        "\n",
        "    def _init_weights(self, module):\n",
        "        if isinstance(module, nn.Linear):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "            if module.bias is not None:\n",
        "                torch.nn.init.zeros_(module.bias)\n",
        "        elif isinstance(module, nn.Embedding):\n",
        "            torch.nn.init.normal_(module.weight, mean=0.0, std=0.02)\n",
        "        elif isinstance(module, nn.LayerNorm):\n",
        "            torch.nn.init.zeros_(module.bias)\n",
        "            torch.nn.init.ones_(module.weight)\n",
        "\n",
        "    @classmethod\n",
        "    def from_pretrained(cls, model_type):\n",
        "        from transformers import GPT2LMHeadModel\n",
        "        config = cls.get_default_config()\n",
        "        config.model_type = model_type\n",
        "        config.vocab_size = 25\n",
        "        config.max_length = 512\n",
        "        model = ClassifierI(config)\n",
        "        sd = model.state_dict()\n",
        "        model_hf = GPT2LMHeadModel.from_pretrained(model_type)\n",
        "        sd_hf = model_hf.state_dict()\n",
        "        keys = [k for k in sd_hf if not k.endswith('attn.masked_bias')]\n",
        "        transposed = ['attn.c_attn.weight', 'attn.c_proj.weight', 'mlp.c_fc.weight', 'mlp.c_proj.weight']\n",
        "        assert len(keys) == len(sd)\n",
        "        for k in keys:\n",
        "            if any(k.endswith(w) for w in transposed):\n",
        "                assert sd_hf[k].shape[::-1] == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k].t())\n",
        "            else:\n",
        "                assert sd_hf[k].shape == sd[k].shape\n",
        "                with torch.no_grad():\n",
        "                    sd[k].copy_(sd_hf[k])\n",
        "        return model\n",
        "\n",
        "    def configure_optimizers(self, train_config):\n",
        "        decay = set()\n",
        "        no_decay = set()\n",
        "        whitelist_weight_modules = (torch.nn.Linear, )\n",
        "        blacklist_weight_modules = (torch.nn.LayerNorm, torch.nn.Embedding)\n",
        "        for mn, m in self.named_modules():\n",
        "            for pn, p in m.named_parameters():\n",
        "                fpn = '%s.%s' % (mn, pn) if mn else pn\n",
        "                if pn.endswith('bias'):\n",
        "                    no_decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, whitelist_weight_modules):\n",
        "                    decay.add(fpn)\n",
        "                elif pn.endswith('weight') and isinstance(m, blacklist_weight_modules):\n",
        "                    no_decay.add(fpn)\n",
        "        param_dict = {pn: p for pn, p in self.named_parameters()}\n",
        "        inter_params = decay & no_decay\n",
        "        union_params = decay | no_decay\n",
        "        assert len(inter_params) == 0, \"parameters %s made it into both decay/no_decay sets!\" % (str(inter_params), )\n",
        "        assert len(param_dict.keys() - union_params) == 0, \"parameters %s were not separated into either decay/no_decay set!\" \\\n",
        "                                                    % (str(param_dict.keys() - union_params), )\n",
        "        optim_groups = [\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(decay))], \"weight_decay\": train_config.weight_decay},\n",
        "            {\"params\": [param_dict[pn] for pn in sorted(list(no_decay))], \"weight_decay\": 0.0},]\n",
        "        optimizer = torch.optim.AdamW(optim_groups, lr=train_config.learning_rate, betas=train_config.betas)\n",
        "        return optimizer\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict_proba(self,idx,j =\"None\"):\n",
        "      sigmoid = nn.Sigmoid()\n",
        "      soft = nn.Softmax(dim=1)\n",
        "      ny = NY()\n",
        "      si = SiLU()\n",
        "      self.eval()\n",
        "      x,_ = self.forward(idx)\n",
        "      if j == \"None\":\n",
        "        return x[:,0:1]\n",
        "      elif j == \"soft\":\n",
        "        return self.soft(x)[:,0:1]\n",
        "      else:\n",
        "        return sigmoid(x)[:,0:1]\n",
        "\n",
        "    @torch.no_grad\n",
        "    def predict(self,idx):\n",
        "      g = torch.zeros((idx.shape[0],1))\n",
        "      e = -1\n",
        "      for i in self.predict_proba(idx):\n",
        "        e+=1\n",
        "        if i[0].item()>=0.5:\n",
        "          g[e] = 1\n",
        "      return g\n",
        "\n",
        "    def forward(self, idx, targets=None):\n",
        "        device = idx.device\n",
        "        b, t = idx.size()\n",
        "        pos = torch.arange(0, t, dtype=torch.long, device=device).unsqueeze(0)\n",
        "        tok_emb = self.transformer.wte(idx)\n",
        "        pos_emb = self.transformer.wpe(pos)\n",
        "        x = self.transformer.drop(tok_emb + pos_emb)\n",
        "        for block in self.transformer.h:\n",
        "            x = block(x)\n",
        "        x = self.transformer.ln_f(x)\n",
        "        logits = self.ny(self.classifier_head(x)).mean(1).to(device)\n",
        "        #logits = self.l(logits)\n",
        "        # = F.sigmoid(logits.view(b,2).mean(1).view(b,1))\n",
        "        loss = None\n",
        "        if targets is not None:\n",
        "            loss = F.cross_entropy(logits,targets,ignore_index = -1)\n",
        "        return logits, loss\n",
        "\n",
        "\n",
        "class Trainer:\n",
        "    @staticmethod\n",
        "    def get_default_config():\n",
        "        C = CfgNode()\n",
        "        C.device = 'cuda'\n",
        "        C.num_workers = 4\n",
        "        C.max_iters = None\n",
        "        C.batch_size = 64\n",
        "        C.max_length = 512\n",
        "        C.learning_rate = 8e-4\n",
        "        C.betas = (0.9, 0.95)\n",
        "        C.weight_decay = 0.1\n",
        "        C.grad_norm_clip = 1.0\n",
        "        return C\n",
        "\n",
        "    def __init__(self, config, model, train_dataset,val_dataset):\n",
        "        self.config = config\n",
        "        self.model = model\n",
        "        self.optimizer = None\n",
        "        self.train_dataset = train_dataset\n",
        "        self.val_dataset = val_dataset\n",
        "        self.callbacks = defaultdict(list)\n",
        "        self.device ='cuda'\n",
        "        self.model = self.model.to(self.device)\n",
        "        self.iter_num = 0\n",
        "        self.iter_time = 0.0\n",
        "        self.iter_dt = 0.0\n",
        "\n",
        "    def add_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent].append(callback)\n",
        "\n",
        "    def set_callback(self, onevent: str, callback):\n",
        "        self.callbacks[onevent] = [callback]\n",
        "\n",
        "    def trigger_callbacks(self, onevent: str):\n",
        "        for callback in self.callbacks.get(onevent, []):\n",
        "            callback(self)\n",
        "\n",
        "    def run(self):\n",
        "        model, config = self.model, self.config\n",
        "        self.optimizer = model.configure_optimizers(config)\n",
        "        batch_size = self.config.batch_size\n",
        "        def get_batch(mode):\n",
        "            batch_size = 64\n",
        "            if mode == \"train\":\n",
        "              data = train_dataset\n",
        "              pos_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 1]),513))\n",
        "              neg_train = torch.tensor(np.array([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]).reshape(len([data[i:i+1,:] for i in range(data.shape[0]) if data[i,-1].item() == 0]),513))\n",
        "              N = np.random.randint(batch_size)\n",
        "              ix_pos = np.random.randint(pos_train.shape[0]-N)\n",
        "              ix_neg = np.random.randint(neg_train.shape[0]-batch_size-N)\n",
        "              pos_data = pos_train[ix_pos:ix_pos+N,:]\n",
        "              neg_data = neg_train[ix_neg:ix_neg+batch_size-N,:]\n",
        "              data = torch.concat((pos_data,neg_data))\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,2), device=self.device)\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o][0] = 1\n",
        "                else:\n",
        "                  targets[o][1] = 1\n",
        "            else:\n",
        "              data = val_dataset\n",
        "              ix = np.random.randint(data.shape[0]-batch_size)\n",
        "              data = data[ix:ix+batch_size,:]\n",
        "              seq = data[:,:-1]\n",
        "              targets = torch.zeros((batch_size,2), device=self.device)\n",
        "              o = -1\n",
        "              for i in data:\n",
        "                o+=1\n",
        "                if i[-1].item() == 1:\n",
        "                  targets[o][0] = 1\n",
        "                else:\n",
        "                  targets[o][1] = 1\n",
        "            targets = targets.view(batch_size,2).to(self.device)\n",
        "            seq = seq.view(batch_size,512).to(self.device)\n",
        "            return seq, targets\n",
        "\n",
        "        @torch.no_grad\n",
        "        def cross_val():\n",
        "          model.eval()\n",
        "          out = []\n",
        "          for i in [\"train\",\"val\"]:\n",
        "            losses = torch.zeros(200+1)\n",
        "            for k in range(200):\n",
        "              X,Y = get_batch(i)\n",
        "              logits,loss = model(X,Y)\n",
        "              losses[k]=loss.item()\n",
        "              out1 = losses.mean()\n",
        "            out.append(out1)\n",
        "          model.train()\n",
        "          return out\n",
        "        losses = cross_val()\n",
        "        LOSS = [losses]\n",
        "        print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        model.train()\n",
        "        for epoch in range(20):\n",
        "          print(\"[epoch {o}] \\n\".format(o = epoch))\n",
        "          iters = 200\n",
        "          for i in range(iters):\n",
        "            if i == 0:\n",
        "              Y = \"| =\"\n",
        "            elif i == iters -1 :\n",
        "              Y = \"=> 100% |\"\n",
        "            else:\n",
        "              if i%(int(iters/50)) == 0 :\n",
        "                Y = \"=\"\n",
        "              else:\n",
        "                Y = \"\"\n",
        "            print(\"{y}\".format(y = Y),end=\"\")\n",
        "            x, y = get_batch(\"train\")\n",
        "            logits, self.loss = model(x, y)\n",
        "            model.zero_grad(set_to_none=True)\n",
        "            self.loss.backward()\n",
        "            torch.nn.utils.clip_grad_norm_(model.parameters(), config.grad_norm_clip)\n",
        "            self.optimizer.step()\n",
        "          losses = cross_val()\n",
        "          LOSS.append(losses)\n",
        "          print(LOSS)\n",
        "          print(\"\\n[train loss = {k}, val loss =  {j}]\\n\".format(k = losses[0],j = losses[1]))\n",
        "        PATH = \"model {h}\".format(h = model_config.model_type)\n",
        "        torch.save(model.state_dict(), PATH)\n",
        "class Transformer():\n",
        "  def __init__(self,mode):\n",
        "    self.mode = mode\n",
        "    self.device = torch.device(\"cuda\")\n",
        "    self.vocab_dict = tokenizer.get_vocab()\n",
        "\n",
        "  def classifier(self):\n",
        "    device = self.device\n",
        "    model_config = ClassifierI.get_default_config()\n",
        "    model_config.vocab_size = 25\n",
        "    model_config.block_size = 512\n",
        "    if self.mode == \"b\":\n",
        "      model_config.model_type = 'b'\n",
        "      model2 = ClassifierI(model_config)\n",
        "      model2.load_state_dict(torch.load(\"classi/model_b\",map_location=self.device))\n",
        "    else:\n",
        "      model_config.model_type = 'c'\n",
        "      model2 = ClassifierI(model_config)\n",
        "      model2.load_state_dict(torch.load(\"classi/model_c\",map_location = self.device))\n",
        "    model2.to(device)\n",
        "    return model2\n",
        "\n",
        "  def Encode(self, i):\n",
        "        def encode_char(x):\n",
        "            return [self.vocab_dict[char] for char in x]\n",
        "\n",
        "        def pad_sequences(x, PAD=0, max_len=512):\n",
        "            return np.array([seq + [PAD] * (max_len - len(seq)) for seq in x])\n",
        "\n",
        "        encoded = list(map(encode_char, i))\n",
        "        padded = pad_sequences(encoded)\n",
        "        return torch.tensor(padded).to(self.device)\n",
        "\n",
        "  def Decode(self,i):\n",
        "    def decode(k):\n",
        "      l = [j for j in k if j != 0 ]\n",
        "      seq = [chars[j] for j in l]\n",
        "      return \"\".join(seq)\n",
        "    H = list(map(decode,i))\n",
        "    G = []\n",
        "    for i in H:\n",
        "      if len(i)>512:\n",
        "        u = H.index(i)\n",
        "        for j in range(len(i)-512):\n",
        "          G.append(i[j:j+512])\n",
        "      else:\n",
        "        G.append(i)\n",
        "    return G,u\n",
        "  def Mean(self,i):\n",
        "    I = list(i.ravel())\n",
        "    sig = lambda x:np.e**(0*x)\n",
        "    Sum = [sig(np.abs(x-int(len(I)/2))) for x in range(len(I))]\n",
        "    wei = np.array([Sum[i]/sum(Sum) for i in range(len(I))]).reshape(len(I),1)\n",
        "    return np.sum(wei*i)\n",
        "\n",
        "  def enhancer(self,x):\n",
        "    wei = np.array([np.exp((i[0]-1)/(i[0])) for i in x])\n",
        "    return np.sum(np.array([i[0] for i in x])*wei)/np.sum(wei)\n",
        "\n",
        "  def predict_proba(self,i):\n",
        "    if len(i[0])>512:\n",
        "       l = [self.Encode([i[0][j:j+512]]) for j in range(len(i[0])-512)]\n",
        "       if len(l)>700:\n",
        "        L = [torch.concat(tuple(l[i*700:(i+1)*700])) for i in range(len(l)//700)] + [torch.concat(tuple(l[(len(l)//700)*700:(len(l)//700)*700+len(l)%700]))]\n",
        "        T = [self.classifier().predict_proba(i,\"sig\").tolist() for i in L ]\n",
        "        return self.enhancer(np.concatenate(tuple([i for i in T])))\n",
        "       else:\n",
        "         t = torch.concat(tuple(l))\n",
        "         return self.enhancer(self.classifier().predict_proba(t,\"sig\").tolist())\n",
        "    else:\n",
        "       seq = self.Encode(i)\n",
        "       U = np.array(self.classifier().predict_proba(seq,\"sig\").tolist())\n",
        "       out = np.array(U)\n",
        "       return out.reshape(len(i),1)\n",
        "\n",
        "model2 = Transformer(\"b\")\n",
        "model3 = Transformer(\"c\")\n",
        "\n",
        "\n",
        "import functools as FUNC\n",
        "from numba import njit\n",
        "\n",
        "class PPA():\n",
        "    def __init__(self, seq, model):\n",
        "        self.model = model\n",
        "        self.seq = seq\n",
        "        self.len = len(self.seq)\n",
        "        self.idx = None\n",
        "\n",
        "    def attention_mask(self):\n",
        "        idx2 = self.splicer(self.seq)[1]\n",
        "        k = self.T()\n",
        "        mask = [[True if len(k[0]) > idx2[j] - i > -1 else False for i in range(len(k[j]))] for j in range(len(k))]\n",
        "        return mask\n",
        "\n",
        "    def mean(self, mask, spliced):\n",
        "        u = [np.mean(self.model.predict_proba([spliced[i][j] for j in range(len(spliced[i])) if mask[i][j]])) for i in trange(len(spliced))]\n",
        "        return u\n",
        "\n",
        "    def splice_scorer(self, i):\n",
        "        lmin = len(i) // 2\n",
        "        return [i[j:j + lmin] for j in range(len(i) - lmin + 1)]\n",
        "\n",
        "    def splicer(self, i):\n",
        "        idx = self.idx\n",
        "        o, Before = [], []\n",
        "        for j in range(len(i)):\n",
        "            n_b, n_a = j, len(i) - j - 1\n",
        "            if n_b <= idx // 2:\n",
        "                before = i[max(0, j - idx // 2):j]\n",
        "                after = i[j + 1:j + idx - len(before)]\n",
        "            elif n_a <= idx // 2:\n",
        "                before = i[j - idx + n_a + 1:j]\n",
        "                after = i[j + 1:]\n",
        "            else:\n",
        "                before = i[j - idx // 2:j]\n",
        "                after = i[j + 1:j + idx // 2 + 1]\n",
        "            o.append(before + i[j] + after)\n",
        "            Before.append(len(before))\n",
        "        return o, Before\n",
        "\n",
        "    def T(self):\n",
        "        return list(map(self.splice_scorer, self.splicer(self.seq)[0]))\n",
        "\n",
        "    def Out(self):\n",
        "        s = self.s\n",
        "        def d(x):\n",
        "            if s > 0.7:\n",
        "                return x * np.exp(-1.2 * (x - s))\n",
        "            else:\n",
        "                return x if x > 0.7 else x\n",
        "\n",
        "        return list(map(d, self.mean(self.attention_mask(), self.T())))\n",
        "def show(i):\n",
        "  values = np.array(i).reshape(1, len(i))\n",
        "  y = range(len(i))\n",
        "  Y = [(j,1) for j in range(len(i)) if np.mean([i[j],i[min((j+1),len(i)-1)],i[max(0,(j-1))]])>0.6]\n",
        "  Y2 = [(j,1) for j in range(len(i)) if np.mean([i[j],i[min((j+1),len(i)-1)],i[min((j+2),len(i)-1)],i[min((j+3),len(i)-1)],i[min((j+4),len(i)-1)],i[min((j+5),len(i)-1)],i[max(0,(j-1))],i[max(0,(j-2))],i[max(0,(j-3))],i[max(0,(j-4))],i[max(0,(j-5))]])>0.85]\n",
        "  threshold = 0.5\n",
        "  above_threshold = np.maximum(values - threshold, 0)\n",
        "  below_threshold = np.minimum(values, threshold)\n",
        "  fig, (ax3 , ax4) = plt.subplots(2,1,sharex = True,sharey = True)\n",
        "  ax3.bar(y, below_threshold.ravel(), 0.9, color=\"lightblue\")\n",
        "  ax3.bar(y, above_threshold.ravel(), 0.9, color=\"lightgreen\",\n",
        "               bottom=below_threshold.ravel())\n",
        "  ax4.broken_barh(Y, (0.7, 0.03),\n",
        "               facecolors=( 'red'))\n",
        "  ax4.plot([0., y[-1]],[0.715,0.715],\"-b\")\n",
        "  for k in Y:\n",
        "    ax4.plot([k[0],k[0]+1],[0.715,0.715],\"-r\")\n",
        "\n",
        "  ax4.text(0,0.8 , \"LLPS-susceptible regions\",fontsize = 20)\n",
        "  ax3.plot([0., y[-1]], [threshold, threshold], \"k--\")\n",
        "  ax4.set_ylim([0,1])\n",
        "  ax3.set_ylim([0,1])\n",
        "\n",
        "  plt.subplots_adjust(\n",
        "                    wspace=0.4,\n",
        "                    hspace=0)\n",
        "  fig.set_figwidth(20)\n",
        "  fig.set_figheight(15)\n",
        "  plt.show()\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 35,
      "metadata": {
        "id": "_R4AYlL7PfO2",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "cellView": "form",
        "outputId": "3b75b503-10a3-4eb2-e160-d2b1d11d76f8"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Score : [[0.86435096]]\n"
          ]
        }
      ],
      "source": [
        "#@title Loading sequence , then hit `Runtime` -> `Run all`\n",
        "#@markdown - Load the sequence of your protein :\n",
        "Sequence = \"GIKEEAAAIKWEAAAIKEGASPEPQPKGIKEEAAAIKWEAAAIKEGASPEPQPKPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPKASPEPQPGIKEEAAAIKWEAAAIKEGASPEPQPKPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGKPSGQAKEIQWQAKEIQEQAKGGIKEEAAAIKWEAAAIKEGASPEPQPKPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKG\" #@param {type:\"string\"}\n",
        "ID = None #@param {type:\"string\"}\n",
        "Directory = None #@param {type:\"string\"}\n",
        "#@markdown  - Use `End_sequence` to focus the scoring on sequences up to the specified endpoint\n",
        "\n",
        "End_sequence = 51 #@param {type:\"slider\" , min : 0 , max: 1000 , step : 1}\n",
        "Plot = True #@param {type: \"boolean\"}\n",
        "\n",
        "#@markdown - Specify the path of `FASTA format file` of your sequences.\n",
        "Fasta_file = \"uniref_PHB_synthesis_2024_06_07.fasta\" #@param {type:\"string\"}\n",
        "import pandas as pd\n",
        "#assert [Sequence == None , Fasta_file == None].count(True) == 1\n",
        "\n",
        "def enh(x):\n",
        "  wei = np.array([np.exp((i-1)/(i)) for i in x])\n",
        "  return np.sum(np.array(x)*wei)/np.sum(wei)\n",
        "\n",
        "def edit(i):\n",
        "  for j in i:\n",
        "    if j in \"\\t\\nBJOUXZ, _@#$%^&*({}[])\":\n",
        "      i = i.replace(j,\"\")\n",
        "  return i\n",
        "\n",
        "\n",
        "def SW(s):\n",
        "  if len(s)>= 700:\n",
        "    L= [s[i*700:(i+1)*700] for i in range(len(s)//700+1)]\n",
        "    H = []\n",
        "    for i in range(len(L)):\n",
        "      H.append(model3.predict_proba(L[i]))\n",
        "    return H\n",
        "  else:\n",
        "    return model3.predict_proba(s)\n",
        "\n",
        "\n",
        "def Score1(i,Sc):\n",
        "    i = i-1\n",
        "    if i<= L:\n",
        "      return sum([Sc[j] for j in range(i+1)])/(i+1)\n",
        "\n",
        "    elif L+1<=i<n-L:\n",
        "       return sum([Sc[j] for j in range(i-L,i+1)])/(L+1)\n",
        "\n",
        "    else:\n",
        "       u = (n-(i)+1)\n",
        "       return sum([Sc[j] for j in range(i-L,n-L+1)])/(u)\n",
        "\n",
        "u = model3.predict_proba([Sequence])\n",
        "\n",
        "def d(x):\n",
        "    if u>0.7:\n",
        "        return x*np.exp(-1.2*(x-u))\n",
        "    else:\n",
        "        if x > 0.7 :\n",
        "            return x\n",
        "        else:\n",
        "            return x\n",
        "\n",
        "\n",
        "if Sequence != None and Sequence !=\"\":\n",
        "  Sequence = edit(Sequence)\n",
        "  k = max(5,min(50,int(np.ceil(0.1*len(Sequence)))))\n",
        "  L = k//2\n",
        "  if Plot == True:\n",
        "    n = len(Sequence)\n",
        "    S = [Sequence[i:i+L] for i in range(len(Sequence)-L+1)]\n",
        "    Sc = SW(S)\n",
        "    Sc1 = Sc[0]\n",
        "    for i in Sc[1:]:\n",
        "      Sc1 = np.concatenate((Sc1,i))\n",
        "\n",
        "    p = list(map(d,[Score1(i,Sc1) for i in range(1,n+1)]))\n",
        "    if type(p[0]) != np.float64:\n",
        "      p = [i[0][0] for i in p]\n",
        "    u = model3.predict_proba([Sequence])\n",
        "    i = p\n",
        "    values = np.array(i).reshape(1, len(i))\n",
        "    below_threshold = [k if k<= 0.5 else 0.5 for k in i]\n",
        "    above_threshold = [0 if k<= 0.5 else k-0.5 for k in i]\n",
        "    threshold = 0.5\n",
        "    y = list(range(len(i)))\n",
        "    Y = [(j, 1) for j in range(len(i)) if np.mean([i[j], i[min((j + 1), len(i) - 1)], i[max(0, (j - 1))]]) > 0.6]\n",
        "    Y2 = [(j, 1) for j in range(len(i)) if np.mean([i[j], i[min((j + 1), len(i) - 1)], i[min((j + 2), len(i) - 1)],\n",
        "                                               i[min((j + 3), len(i) - 1)], i[min((j + 4), len(i) - 1)],\n",
        "                                               i[min((j + 5), len(i) - 1)], i[max(0, (j - 1))],\n",
        "                                               i[max(0, (j - 2))], i[max(0, (j - 3))], i[max(0, (j - 4))],\n",
        "                                               i[max(0, (j - 5))]]) > 0.85]\n",
        "    if n>= 1500:\n",
        "      c1 = \"blue\"\n",
        "      c2 = \"green\"\n",
        "    else:\n",
        "      c1 = \"lightblue\"\n",
        "      c2 = \"lightgreen\"\n",
        "\n",
        "    fig = make_subplots(rows=1, cols=1, shared_xaxes=True, shared_yaxes=True, vertical_spacing=0.02)\n",
        "    fig.add_trace(go.Bar(x=y, y=below_threshold, marker_color= c1, name='Below Threshold', width= 0.6), row=1, col=1)\n",
        "    fig.add_trace(go.Bar(x=y, y=above_threshold, marker_color=c2, name='Above Threshold', base=below_threshold, width= 0.6), row=1, col=1)\n",
        "\n",
        "    fig.add_trace(go.Scatter(x=[0, len(i) - 1], y=[threshold, threshold], mode='lines', line=dict(color='black', dash='dash'), showlegend=False), row=1, col=1)\n",
        "    for start, duration in Y:\n",
        "        fig.add_trace(go.Scatter(x=[start, start + duration], y=[0.01, 0.01], mode='lines', line=dict(color='red', width=7), showlegend=False), row=1, col=1)\n",
        "    fig.update_layout(height=800, width=1700, title_text=\"LLPS Prediction Plot\")\n",
        "    score = str(((2*u+2*enh(p))/4))\n",
        "    print(\"Score : \"+score)\n",
        "  else:\n",
        "    score = u\n",
        "    print(\"Score : \"+str(score))\n",
        "\n",
        "\n",
        "else:\n",
        "  if os.path.exists(Fasta_file):\n",
        "    with open(Fasta_file) as f:\n",
        "      u = f.read().split(\">\")\n",
        "      w = len(u)\n",
        "\n",
        "    j = u[1:End_sequence+1]\n",
        "    D = {\"id\":[i[:i.index(\"\\n\")] if \"\\n\" in i else i for i in j],\"seq\":[edit(i[i.index(\"\\n\"):]) if \"\\n\" in i else None for i in j],\"score\":None}\n",
        "    D[\"score\"] = [i[0][0] if type(i) != np.float64 else i for i in  [model3.predict_proba([s]) for s in tqdm(D[\"seq\"])]]\n",
        "\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title plot\n",
        "if Plot == True:\n",
        "  fig.show()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 837
        },
        "id": "QXTbh0DQZm69",
        "outputId": "370d32ca-c6af-4c12-8081-5ed411b742f8",
        "cellView": "form"
      },
      "execution_count": 36,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<html>\n",
              "<head><meta charset=\"utf-8\" /></head>\n",
              "<body>\n",
              "    <div>            <script src=\"https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_SVG\"></script><script type=\"text/javascript\">if (window.MathJax && window.MathJax.Hub && window.MathJax.Hub.Config) {window.MathJax.Hub.Config({SVG: {font: \"STIX-Web\"}});}</script>                <script type=\"text/javascript\">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>\n",
              "        <script charset=\"utf-8\" src=\"https://cdn.plot.ly/plotly-2.24.1.min.js\"></script>                <div id=\"7442fa78-4f37-4ec3-9cbb-e9278858aaca\" class=\"plotly-graph-div\" style=\"height:800px; width:1700px;\"></div>            <script type=\"text/javascript\">                                    window.PLOTLYENV=window.PLOTLYENV || {};                                    if (document.getElementById(\"7442fa78-4f37-4ec3-9cbb-e9278858aaca\")) {                    Plotly.newPlot(                        \"7442fa78-4f37-4ec3-9cbb-e9278858aaca\",                        [{\"marker\":{\"color\":\"lightblue\"},\"name\":\"Below Threshold\",\"width\":0.6,\"x\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131,132,133,134,135,136,137,138,139,140,141,142,143,144,145,146,147,148,149,150,151,152,153,154,155,156,157,158,159,160,161,162,163,164,165,166,167,168,169,170,171,172,173,174,175,176,177,178,179,180,181,182,183,184,185,186,187,188,189,190,191,192,193,194,195,196,197,198,199,200,201,202,203,204,205,206,207,208,209,210,211,212,213,214,215,216,217,218,219,220,221,222,223,224,225,226,227,228,229,230,231,232,233,234,235,236,237,238,239,240,241,242,243,244,245,246,247,248,249,250,251,252,253,254,255,256,257,258,259,260,261,262,263,264,265,266,267,268,269,270,271,272,273,274,275,276,277,278,279,280,281,282,283,284,285,286,287,288,289,290,291,292,293,294,295,296,297,298,299,300,301,302,303,304,305,306,307,308,309,310,311,312,313,314,315,316,317,318,319,320,321,322,323,324,325,326,327,328,329,330,331,332,333,334,335,336,337,338,339,340,341,342,343,344,345,346,347,348,349,350,351],\"y\":[0.13922964681061026,0.246029386295127,0.2136857545607141,0.2797817199610416,0.25376535876691353,0.2374808994032837,0.22539849563973272,0.2162637852366293,0.21065921467678225,0.20440293566605267,0.19874033511050776,0.1944315246713017,0.34446956650604954,0.3315880511663831,0.3200786997443162,0.30994955087100007,0.4096154594883266,0.487772609396202,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.47496256196804076,0.38729464912760103,0.4706778587841827,0.47217177731436405,0.3858763624293667,0.38600797784351426,0.2889575350765487,0.38790167853283586,0.4745568701310183,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.44231199829611667,0.5,0.5,0.5,0.45408156160383045,0.45395497103140336,0.4607089846252747,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.47073465559602307,0.3913487785011084,0.39094159674076095,0.47422704915205227,0.476389329297906,0.47018014147335113,0.47037594238192876,0.38623952549601,0.4741398309343762,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5],\"type\":\"bar\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"base\":[0.13922964681061026,0.246029386295127,0.2136857545607141,0.2797817199610416,0.25376535876691353,0.2374808994032837,0.22539849563973272,0.2162637852366293,0.21065921467678225,0.20440293566605267,0.19874033511050776,0.1944315246713017,0.34446956650604954,0.3315880511663831,0.3200786997443162,0.30994955087100007,0.4096154594883266,0.487772609396202,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.47496256196804076,0.38729464912760103,0.4706778587841827,0.47217177731436405,0.3858763624293667,0.38600797784351426,0.2889575350765487,0.38790167853283586,0.4745568701310183,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.44231199829611667,0.5,0.5,0.5,0.45408156160383045,0.45395497103140336,0.4607089846252747,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.47073465559602307,0.3913487785011084,0.39094159674076095,0.47422704915205227,0.476389329297906,0.47018014147335113,0.47037594238192876,0.38623952549601,0.4741398309343762,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5],\"marker\":{\"color\":\"lightgreen\"},\"name\":\"Above Threshold\",\"width\":0.6,\"x\":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131,132,133,134,135,136,137,138,139,140,141,142,143,144,145,146,147,148,149,150,151,152,153,154,155,156,157,158,159,160,161,162,163,164,165,166,167,168,169,170,171,172,173,174,175,176,177,178,179,180,181,182,183,184,185,186,187,188,189,190,191,192,193,194,195,196,197,198,199,200,201,202,203,204,205,206,207,208,209,210,211,212,213,214,215,216,217,218,219,220,221,222,223,224,225,226,227,228,229,230,231,232,233,234,235,236,237,238,239,240,241,242,243,244,245,246,247,248,249,250,251,252,253,254,255,256,257,258,259,260,261,262,263,264,265,266,267,268,269,270,271,272,273,274,275,276,277,278,279,280,281,282,283,284,285,286,287,288,289,290,291,292,293,294,295,296,297,298,299,300,301,302,303,304,305,306,307,308,309,310,311,312,313,314,315,316,317,318,319,320,321,322,323,324,325,326,327,328,329,330,331,332,333,334,335,336,337,338,339,340,341,342,343,344,345,346,347,348,349,350,351],\"y\":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.050080675932075436,0.05048754630730634,0.04417926392149174,0.11090068195249925,0.10094859787949495,0.16006627812413032,0.1601618372283038,0.16073398374773407,0.1608333762472729,0.16021981509289684,0.16539598936847755,0.16552140234063362,0.17402746076566855,0.11730248941407706,0.1174997196960863,0.11772699560135136,0.11792263668286918,0.05160590452653979,0,0,0,0,0,0,0,0,0,0.05124071518630713,0.07496535148432026,0.1332724596087267,0.18912085971688386,0.22987581099746812,0.2713979447780286,0.30595869794039665,0.33436608771969056,0.35722313541548534,0.374954179337354,0.38825473255494425,0.39745272173790713,0.39502962486044246,0.4014834389949664,0.4046804117334706,0.40478426592995864,0.40213875924770737,0.4047735390490561,0.4050979076621739,0.40509789417025543,0.4044896940384115,0.4046034958050818,0.4046037286296952,0.4050103524913403,0.4050087071682231,0.4050094683963823,0.40501112665448125,0.40501242851921915,0.40501293648515946,0.4050131994276541,0.40501411878624494,0.40507689354893117,0.40509653562186854,0.4050964440879903,0.40509639499141026,0.405096269898776,0.40363112413711866,0.4047092964970471,0.40484945102838366,0.4048514011671771,0.40477229884952304,0.40469792258343973,0.40495623091643895,0.4049552415379396,0.40460063166653926,0.40459610588297246,0.40439321804876105,0.4041724844186194,0.40013950475195736,0.39256057323852434,0.38519023545530806,0.38620769013108736,0.3734144892043777,0.37324687028729253,0.35480435961830725,0.3312905723635421,0.3309799082713871,0.3050371531154862,0.2704141338220727,0.26927466286799406,0.3040919941833171,0.3040242028963577,0.26911358032990296,0.2650922167522761,0.2480283302197529,0.25233001344860573,0.25627753806620146,0.29328854823406914,0.32398790056422466,0.34155429880435517,0.3416626397284104,0.3614638934081258,0.361648401193689,0.37296507438921633,0.386826271686009,0.39648023622989936,0.40240365771590514,0.40493789663254387,0.4025623836026003,0.40021353377286717,0.4002328984268091,0.40420535805012614,0.4050607837358068,0.4046118253746033,0.4050077147584147,0.4050072780197834,0.4050105325883572,0.4050123470765835,0.40501242851921915,0.40501293648515946,0.4050131994276541,0.40501411878624494,0.4050768879653732,0.4040281278979352,0.404030318349862,0.39985823650378527,0.3998618647897889,0.40402869267698716,0.40227905467625336,0.3964442635070793,0.39637527250431215,0.38740986925717147,0.3738333300571337,0.37138190719726916,0.36829906811098023,0.34848394797617865,0.34664356017934794,0.3211525957729934,0.3162786038831935,0.28369326269756934,0.24849951607103737,0.20281247239670375,0.20865904339663344,0.15597458854535606,0.1554959779406021,0.09441472096158432,0.02410938646378702,0.024460898779283924,0.02261129952933627,0,0.01928395318882503,0.020741410713220665,0.01959193649478419,0,0,0,0.038995267136860634,0.05124071518630713,0.07496535148432026,0.1332724596087267,0.18912085971688386,0.22987581099746812,0.2713979447780286,0.30595869794039665,0.33436608771969056,0.35722313541548534,0.374954179337354,0.38825473255494425,0.39745272173790713,0.39502962486044246,0.4014834389949664,0.4046804117334706,0.40478426592995864,0.40213875924770737,0.4047735390490561,0.4050979076621739,0.40509789417025543,0.4044896940384115,0.4046034958050818,0.4046037286296952,0.4050103524913403,0.4050087071682231,0.4050094683963823,0.40501112665448125,0.40501242851921915,0.40501293648515946,0.4050131994276541,0.40501411878624494,0.40507689354893117,0.40509653562186854,0.4050964440879903,0.40509639499141026,0.405096269898776,0.40363112413711866,0.4047092964970471,0.40484945102838366,0.4048514011671771,0.40477229884952304,0.40469792258343973,0.40495623091643895,0.4049552415379396,0.40460063166653926,0.40459610588297246,0.40439321804876105,0.4041724844186194,0.40013950475195736,0.39256057323852434,0.38519023545530806,0.38620769013108736,0.3734144892043777,0.3730846590916357,0.35457944569017874,0.33098319964948586,0.3306720315234365,0.3046483845228668,0.2703364873698485,0.2702686948645021,0.30490918733631023,0.27012258009791956,0.22859425866209948,0.2235729579511606,0.22359223025130526,0.22825562860441462,0.23215129786352395,0.27266166193471486,0.3061802931877986,0.3260036549003068,0.2958116124861452,0.2931421537896822,0.26942670225586907,0.30430414244430626,0.3068886332911318,0.3348496091680516,0.3348509128333429,0.3344922310802688,0.3062126731322339,0.27153625308865714,0.27177740574855924,0.3059287807908213,0.33201783565067744,0.30308626932787197,0.26791409945366884,0.22623822029141805,0.1814403632069843,0.12600791308808068,0.1236192108896309,0.12341479241066122,0.12464181899827109,0.16288590801084002,0.10296530842692098,0.1037210525560559,0.0353647689431249,0.04685057610317134,0.04680719556760704,0.0467796176805233,0.047121584670790084,0.04668329955214712,0,0,0,0,0,0,0,0,0,0.0494953071767944,0.05124071518630713,0.07496535148432026,0.1332724596087267,0.18912085971688386,0.22987581099746812,0.2713979447780286,0.30595869794039665,0.33436608771969056,0.35722313541548534,0.374954179337354,0.38825473255494425,0.39745272173790713,0.39502962486044246,0.4014834389949664,0.4046804117334706,0.40478426592995864,0.40213875924770737,0.4047735390490561,0.4050979076621739,0.40509789417025543,0.4044896940384115,0.4046034958050818,0.4046037286296952,0.4050103524913403,0.4050087071682231,0.4050094683963823,0.40501112665448125,0.40501242851921915,0.40501293648515946,0.4050131994276541,0.40501411878624494,0.40507689354893117,0.40509653562186854,0.4050964440879903,0.40509639499141026,0.405096269898776,0.40363112413711866,0.4047092964970471,0.40484945102838366,0.4048514011671771,0.40477229884952304,0.40469792258343973,0.40495623091643895,0.4049552415379396,0.40460063166653926,0.40459610588297246,0.40439321804876105,0.4041724844186194,0.40013950475195736,0.39256057323852434,0.38519023545530806,0.38620769013108736,0.3734144892043777,0.3670515972885684,0.3591257174743916,0.3491817123117479,0.36370470492034723,0.3562137359722175,0.34318612191728914,0.32553745102202725,0.3480168614170168,0.3280066116745002,0.2984243847159136,0.32501958615653503,0.2845571106538347,0.2287042232191976,0.1242183425160529,0.19807995774538967,0.2907758963744501,0.3320069444403556],\"type\":\"bar\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"black\",\"dash\":\"dash\"},\"mode\":\"lines\",\"showlegend\":false,\"x\":[0,351],\"y\":[0.5,0.5],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[22,23],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[23,24],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[24,25],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[25,26],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[26,27],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[27,28],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[28,29],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[29,30],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[30,31],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[31,32],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[32,33],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[33,34],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[47,48],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[48,49],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[49,50],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[50,51],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[51,52],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[52,53],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[53,54],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[54,55],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[55,56],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[56,57],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[57,58],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[58,59],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[59,60],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[60,61],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[61,62],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[62,63],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[63,64],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[64,65],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[65,66],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[66,67],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[67,68],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[68,69],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[69,70],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[70,71],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[71,72],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[72,73],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[73,74],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[74,75],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[75,76],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[76,77],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[77,78],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[78,79],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[79,80],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[80,81],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[81,82],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[82,83],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[83,84],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[84,85],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[85,86],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[86,87],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[87,88],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[88,89],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[89,90],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[90,91],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[91,92],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[92,93],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[93,94],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[94,95],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[95,96],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[96,97],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[97,98],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[98,99],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[99,100],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[100,101],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[101,102],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[102,103],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[103,104],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[104,105],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[105,106],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[106,107],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[107,108],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[108,109],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[109,110],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[110,111],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[111,112],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[112,113],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[113,114],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[114,115],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[115,116],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[116,117],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[117,118],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[118,119],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[119,120],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[120,121],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[121,122],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[122,123],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[123,124],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[124,125],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[125,126],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[126,127],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[127,128],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[128,129],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[129,130],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[130,131],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[131,132],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[132,133],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[133,134],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[134,135],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[135,136],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[136,137],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[137,138],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[138,139],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[139,140],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[140,141],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[141,142],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[142,143],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[143,144],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[144,145],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[145,146],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[146,147],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[147,148],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[148,149],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[149,150],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[150,151],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[151,152],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[152,153],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[153,154],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[154,155],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[155,156],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[156,157],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[157,158],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[158,159],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[159,160],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[174,175],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[175,176],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[176,177],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[177,178],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[178,179],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[179,180],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[180,181],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[181,182],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[182,183],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[183,184],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[184,185],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[185,186],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[186,187],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[187,188],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[188,189],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[189,190],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[190,191],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[191,192],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[192,193],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[193,194],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[194,195],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[195,196],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[196,197],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[197,198],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[198,199],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[199,200],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[200,201],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[201,202],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[202,203],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[203,204],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[204,205],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[205,206],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[206,207],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[207,208],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[208,209],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[209,210],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[210,211],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[211,212],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[212,213],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[213,214],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[214,215],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[215,216],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[216,217],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[217,218],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[218,219],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[219,220],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[220,221],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[221,222],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[222,223],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[223,224],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[224,225],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[225,226],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[226,227],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[227,228],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[228,229],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[229,230],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[230,231],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[231,232],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[232,233],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[233,234],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[234,235],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[235,236],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[236,237],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[237,238],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[238,239],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[239,240],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[240,241],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[241,242],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[242,243],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[243,244],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[244,245],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[245,246],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[246,247],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[247,248],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[248,249],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[249,250],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[250,251],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[251,252],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[252,253],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[253,254],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[254,255],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[255,256],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[256,257],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[257,258],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[258,259],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[259,260],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[260,261],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[261,262],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[262,263],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[263,264],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[264,265],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[284,285],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[285,286],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[286,287],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[287,288],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[288,289],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[289,290],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[290,291],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[291,292],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[292,293],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[293,294],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[294,295],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[295,296],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[296,297],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[297,298],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[298,299],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[299,300],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[300,301],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[301,302],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[302,303],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[303,304],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[304,305],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[305,306],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[306,307],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[307,308],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[308,309],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[309,310],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[310,311],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[311,312],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[312,313],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[313,314],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[314,315],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[315,316],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[316,317],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[317,318],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[318,319],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[319,320],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[320,321],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[321,322],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[322,323],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[323,324],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[324,325],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[325,326],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[326,327],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[327,328],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[328,329],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[329,330],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[330,331],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[331,332],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[332,333],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[333,334],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[334,335],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[335,336],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[336,337],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[337,338],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[338,339],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[339,340],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[340,341],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[341,342],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[342,343],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[343,344],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[344,345],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[345,346],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[346,347],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[347,348],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[348,349],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[349,350],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[350,351],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"},{\"line\":{\"color\":\"red\",\"width\":7},\"mode\":\"lines\",\"showlegend\":false,\"x\":[351,352],\"y\":[0.01,0.01],\"type\":\"scatter\",\"xaxis\":\"x\",\"yaxis\":\"y\"}],                        {\"template\":{\"data\":{\"candlestick\":[{\"decreasing\":{\"line\":{\"color\":\"#000033\"}},\"increasing\":{\"line\":{\"color\":\"#000032\"}},\"type\":\"candlestick\"}],\"contourcarpet\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"contourcarpet\"}],\"contour\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"contour\"}],\"heatmap\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"heatmap\"}],\"histogram2d\":[{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"type\":\"histogram2d\"}],\"icicle\":[{\"textfont\":{\"color\":\"white\"},\"type\":\"icicle\"}],\"sankey\":[{\"textfont\":{\"color\":\"#000036\"},\"type\":\"sankey\"}],\"scatter\":[{\"marker\":{\"line\":{\"width\":0}},\"type\":\"scatter\"}],\"table\":[{\"cells\":{\"fill\":{\"color\":\"#000038\"},\"font\":{\"color\":\"#000037\"},\"line\":{\"color\":\"#000039\"}},\"header\":{\"fill\":{\"color\":\"#000040\"},\"font\":{\"color\":\"#000036\"},\"line\":{\"color\":\"#000039\"}},\"type\":\"table\"}],\"waterfall\":[{\"connector\":{\"line\":{\"color\":\"#000036\",\"width\":2}},\"decreasing\":{\"marker\":{\"color\":\"#000033\"}},\"increasing\":{\"marker\":{\"color\":\"#000032\"}},\"totals\":{\"marker\":{\"color\":\"#000034\"}},\"type\":\"waterfall\"}]},\"layout\":{\"coloraxis\":{\"colorscale\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]]},\"colorscale\":{\"diverging\":[[0.0,\"#000021\"],[0.1,\"#000022\"],[0.2,\"#000023\"],[0.3,\"#000024\"],[0.4,\"#000025\"],[0.5,\"#000026\"],[0.6,\"#000027\"],[0.7,\"#000028\"],[0.8,\"#000029\"],[0.9,\"#000030\"],[1.0,\"#000031\"]],\"sequential\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]],\"sequentialminus\":[[0.0,\"#000011\"],[0.1111111111111111,\"#000012\"],[0.2222222222222222,\"#000013\"],[0.3333333333333333,\"#000014\"],[0.4444444444444444,\"#000015\"],[0.5555555555555556,\"#000016\"],[0.6666666666666666,\"#000017\"],[0.7777777777777778,\"#000018\"],[0.8888888888888888,\"#000019\"],[1.0,\"#000020\"]]},\"colorway\":[\"#000001\",\"#000002\",\"#000003\",\"#000004\",\"#000005\",\"#000006\",\"#000007\",\"#000008\",\"#000009\",\"#000010\"]}},\"xaxis\":{\"anchor\":\"y\",\"domain\":[0.0,1.0]},\"yaxis\":{\"anchor\":\"x\",\"domain\":[0.0,1.0]},\"title\":{\"text\":\"LLPS Prediction Plot\"},\"height\":800,\"width\":1700},                        {\"responsive\": true}                    ).then(function(){\n",
              "                            \n",
              "var gd = document.getElementById('7442fa78-4f37-4ec3-9cbb-e9278858aaca');\n",
              "var x = new MutationObserver(function (mutations, observer) {{\n",
              "        var display = window.getComputedStyle(gd).display;\n",
              "        if (!display || display === 'none') {{\n",
              "            console.log([gd, 'removed!']);\n",
              "            Plotly.purge(gd);\n",
              "            observer.disconnect();\n",
              "        }}\n",
              "}});\n",
              "\n",
              "// Listen for the removal of the full notebook cells\n",
              "var notebookContainer = gd.closest('#notebook-container');\n",
              "if (notebookContainer) {{\n",
              "    x.observe(notebookContainer, {childList: true});\n",
              "}}\n",
              "\n",
              "// Listen for the clearing of the current output cell\n",
              "var outputEl = gd.closest('.output');\n",
              "if (outputEl) {{\n",
              "    x.observe(outputEl, {childList: true});\n",
              "}}\n",
              "\n",
              "                        })                };                            </script>        </div>\n",
              "</body>\n",
              "</html>"
            ]
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 5,
      "metadata": {
        "cellView": "form",
        "id": "1xeTkJCtV0Wt",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "b55687ea-4c77-4af3-dba4-78a6340a3fea"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Files zipped successfully!\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_12665b4d-6023-4644-a911-16655c0b60fd\", \"zipped_files.zip\", 40630)"
            ]
          },
          "metadata": {}
        }
      ],
      "source": [
        "#@title Downloading the results`\n",
        "\n",
        "if os.path.exists(\"d\"):\n",
        "    !rm -rf \"d\"\n",
        "if os.path.exists(\"text_dir\"):\n",
        "    !rm -rf \"text_dir\"\n",
        "\n",
        "!mkdir text_dir\n",
        "!mkdir d\n",
        "c = PPA(Sequence , model3)\n",
        "if Sequence != None and Sequence !=\"\" :\n",
        "  if ID == None:\n",
        "    c.name = \"Your peptide\"\n",
        "  with open(\"seq.txt\",\"w\") as w:\n",
        "          w.write(\"                                                 <<<<<<  LLPS prediction results  >>>>>>\\n\\n\\n\")\n",
        "          w.write(\"[Sequence id] : \" +c.name+\"\\n\\n\")\n",
        "          w.write(\"[Sequence length] : \"+str(len(c.seq))+\"\\n\\n\")\n",
        "          w.write(\"[Sequence_of_peptide] : \" +c.seq+\"\\n\\n\")\n",
        "          w.write(\"[Predicted_score] : \" +score +\"\\n\\n\")\n",
        "          w.close()\n",
        "\n",
        "\n",
        "\n",
        "  if c.name != None:\n",
        "    img_name = \"Info_of_sequenceID:\"+c.name+\".png\"\n",
        "    fig.write_image(img_name)\n",
        "  else:\n",
        "    name = np.random.randint(1000000)\n",
        "    img_name = \"Info_of_sequenceID:\"+str(name)+\".png\"\n",
        "    fig.write_image(img_name)\n",
        "\n",
        "  shutil.move(img_name, \"d\")\n",
        "  shutil.move(\"seq.txt\", \"text_dir\")\n",
        "  text_files_path = \"text_dir\"\n",
        "  png_files_path = \"d\"\n",
        "\n",
        "  zip_output_path = 'zipped_files.zip'\n",
        "\n",
        "  text_files = [f for f in os.listdir(text_files_path) if f.endswith('.txt')]\n",
        "  png_files = [f for f in os.listdir(png_files_path) if f.endswith('.png')]\n",
        "\n",
        "  with zipfile.ZipFile(zip_output_path, 'w') as zipf:\n",
        "    for text_file in text_files:\n",
        "      zipf.write(os.path.join(text_files_path, text_file), arcname=text_file)\n",
        "\n",
        "    for png_file in png_files:\n",
        "      zipf.write(os.path.join(png_files_path, png_file), arcname=png_file)\n",
        "\n",
        "  print(\"Files zipped successfully!\")\n",
        "\n",
        "\n",
        "else:\n",
        "  h = pd.DataFrame(D)\n",
        "  h.to_json(\"h.json\")\n",
        "  with open(\"results.txt\",\"w\") as w:\n",
        "    w.write(\"                                                 <<<<<<  LLPS prediction results  >>>>>>\\n\\n\\n\")\n",
        "    w.write(\"Number of sequences : \"+str(len(D[\"seq\"]))+\"\\n\\n\")\n",
        "    for i in range(len(D[\"id\"])):\n",
        "      w.write(\" [Sequence {i}]\".format(i = i)+\"\\n\\n\")\n",
        "      w.write(\"   Sequence id : \"+D[\"id\"][i]+\"\\n\\n\")\n",
        "      w.write(\"   Sequence length : \"+str(len(D[\"seq\"][i]))+\"\\n\\n\")\n",
        "      w.write(\"   Sequence : \" + D[\"seq\"][i]+\"\\n\\n\")\n",
        "      w.write(\"   Predicted Score : \" + str(D[\"score\"][i])+\"\\n\\n\")\n",
        "      w.write(\"-\"*150+\"\\n\\n\")\n",
        "    w.close()\n",
        "    shutil.move(\"h.json\", \"d\")\n",
        "    shutil.move(\"results.txt\", \"text_dir\")\n",
        "    text_files_path = \"text_dir\"\n",
        "    png_files_path = \"d\"\n",
        "    zip_output_path = 'zipped_files.zip'\n",
        "\n",
        "    text_files = [f for f in os.listdir(text_files_path) if f.endswith('.txt')]\n",
        "    png_files = [f for f in os.listdir(png_files_path) if f.endswith('.json')]\n",
        "\n",
        "    with zipfile.ZipFile(zip_output_path, 'w') as zipf:\n",
        "      for text_file in text_files:\n",
        "        zipf.write(os.path.join(text_files_path, text_file), arcname=text_file)\n",
        "\n",
        "      for png_file in png_files:\n",
        "        zipf.write(os.path.join(png_files_path, png_file), arcname=png_file)\n",
        "\n",
        "    print(\"Files zipped successfully!\")\n",
        "\n",
        "files.download(\"zipped_files.zip\")\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "##NCBI BLAST+\n",
        "To determine the BLAST database, you can either provide a FASTA file path to create your own database or use the URL link to NCBI's protein databases available on the NCBI page: https://ftp.ncbi.nlm.nih.gov/blast/db/\n",
        "\n",
        "The DrLLPS database has already been provided. By setting the Blast_Database to DrLLPS, the BLAST search will be conducted against the DrLLPS database.\n",
        "\n",
        "To load your sequences, you can either input a single sequence or upload a FASTA format file containing your sequences into the designated box\n",
        "\n",
        "as example :\n",
        "  **MAVRVAKMTEREKLLLKKYGNRRLYDTKRSAYVTLSQVADIIKQGVDVEVVDAKTQEDVTAFILTQIVLEQARRKNALLPAELLHLIIRYGEDLLQEFFQKYLEQTIHGYIQFKKAMDEQFSKWIDMQLNYSDMAHKTLSGLTRFPNIFSPPPGSPGTKKDKG**\n",
        "\n",
        "or\n",
        "\n",
        "**Your_file.fasta**\n",
        "\n"
      ],
      "metadata": {
        "id": "N1t2f4GlUJ7_"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#@title BLAST+ Tool\n",
        "\n",
        "\n",
        "import os\n",
        "#@markdown - Specify the sequence or the path of your FASTA format file in this box : like  `MAVRVAKMTEREKLL`  or `Your_file.fasta`\n",
        "\n",
        "Sequence = \"GIKEEAAAIKWEAAAIKEGASPEPQPKGIKEEAAAIKWEAAAIKEGASPEPQPKPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPKASPEPQPGIKEEAAAIKWEAAAIKEGASPEPQPKPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGKPSGQAKEIQWQAKEIQEQAKGGIKEEAAAIKWEAAAIKEGASPEPQPKPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKG\" #@param {type:\"string\"}\n",
        "#@markdown - Specify the fasta file or URL of your Database here :                    like  `https://ftp.ncbi.nlm.nih.gov/blast/db/swissprot.tar.gz`  or `Your_Database.fasta`\n",
        "\n",
        "Blast_Database = \"DrLLPS\" #@param [\"DrLLPS\",\"https://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/swissprot.gz\"] {allow-input : true}\n",
        "Blast_result_format = \"1\" #@param [1,2,3,4,5,6,7]\n",
        "Blast_result_file_name = \"Blast_result.txt\" #@param {type:\"string\"}\n",
        "E_value_threshold = \"100\" #@param {type:\"string\"}\n",
        "Max_target_Sequences = 6 #@param {type:\"integer\"}\n",
        "\n",
        "if Blast_Database == \"DrLLPS\":\n",
        "  Blast_Database = \"DrLLPS.fasta\"\n",
        "\n",
        "import sys\n",
        "from os import path\n",
        "\n",
        "if path.exists(\"/content/bioinformatics_stockholm/\"):\n",
        "  pass\n",
        "else:\n",
        "  !git clone https://github.com/Intertangler/bioinformatics_stockholm\n",
        "sys.path.insert(0,'/content')\n",
        "\n",
        "import importlib.util\n",
        "spec = importlib.util.spec_from_file_location(\"msa\", \"/content/bioinformatics_stockholm/msa/msa.py\")\n",
        "msa = importlib.util.module_from_spec(spec)\n",
        "spec.loader.exec_module(msa)\n",
        "\n",
        "spec = importlib.util.spec_from_file_location(\"miniblast\", \"/content/bioinformatics_stockholm/Miniblast/miniblast.py\")\n",
        "miniblast = importlib.util.module_from_spec(spec)\n",
        "spec.loader.exec_module(miniblast)\n",
        "\n",
        "def Fasta(fasta_path):\n",
        "  from command_runner import command_runner\n",
        "  command = \"makeblastdb -in {k} -dbtype prot -out blastdb\".format(k =fasta_path )\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  print(\"-- Blast Database has been created -- \\n\")\n",
        "\n",
        "\n",
        "if \".fasta\" in Blast_Database or \".txt\" in Blast_Database:\n",
        "  if os.path.exists(Blast_Database):\n",
        "    Fasta(Blast_Database)\n",
        "  with open(Blast_Database) as w:\n",
        "    j = w.read().split(\">\")[1:]\n",
        "  Dict = {i[:i.index(\"\\n\")].split(\" \")[0] if \"\\n\" in i else i : edit(i[i.index(\"\\n\"):]) if \"\\n\" in i else None for i in j}\n",
        "\n",
        "else:\n",
        "  command = \"wget {k}\".format(k = Blast_Database)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  print(\"-- Blast Database has been downloaded -- \\n\")\n",
        "  exit_code, output = command_runner(\"gunzip {l}\".format(l = Blast_Database.split(\"/\")[-1]), shell=True, live_output=True)\n",
        "  Fasta(Blast_Database.split(\"/\")[-1][:-3])\n",
        "  db = miniblast.read_database_fasta(Blast_Database.split(\"/\")[-1][:-3])\n",
        "  Dict = {i.id :\"\".join(list(i.seq)) for i in db}\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "cellView": "form",
        "id": "HC5GzDMDCOgN",
        "outputId": "76db5a31-3a75-471f-f22a-37758c94112a"
      },
      "execution_count": 37,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "Building a new DB, current time: 06/10/2024 15:19:03\n",
            "New DB name:   /content/blastdb\n",
            "New DB title:  DrLLPS.fasta\n",
            "Sequence type: Protein\n",
            "Deleted existing Protein BLAST database named /content/blastdb\n",
            "Keep MBits: T\n",
            "Maximum file size: 1000000000B\n",
            "Adding sequences from FASTA; added 1272 sequences in 0.0323501 seconds.\n",
            "\n",
            "\n",
            "-- Blast Database has been created -- \n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Blast Result\n",
        "\n",
        "blosum62 = {\n",
        "    ('W', 'F'): 1, ('L', 'R'): -2, ('S', 'P'): -1, ('V', 'T'): 0,\n",
        "    ('Q', 'Q'): 5, ('N', 'A'): -2, ('Z', 'Y'): -2, ('W', 'R'): -3,\n",
        "    ('Q', 'A'): -1, ('S', 'D'): 0, ('H', 'H'): 8, ('S', 'H'): -1,\n",
        "    ('H', 'D'): -1, ('L', 'N'): -3, ('W', 'A'): -3, ('Y', 'M'): -1,\n",
        "    ('G', 'R'): -2, ('Y', 'I'): -1, ('Y', 'E'): -2, ('B', 'Y'): -3,\n",
        "    ('Y', 'A'): -2, ('V', 'D'): -3, ('B', 'S'): 0, ('Y', 'Y'): 7,\n",
        "    ('G', 'N'): 0, ('E', 'C'): -4, ('Y', 'Q'): -1, ('Z', 'Z'): 4,\n",
        "    ('V', 'A'): 0, ('C', 'C'): 9, ('M', 'R'): -1, ('V', 'E'): -2,\n",
        "    ('T', 'N'): 0, ('P', 'P'): 7, ('V', 'I'): 3, ('V', 'S'): -2,\n",
        "    ('Z', 'P'): -1, ('V', 'M'): 1, ('T', 'F'): -2, ('V', 'Q'): -2,\n",
        "    ('K', 'K'): 5, ('P', 'D'): -1, ('I', 'H'): -3, ('I', 'D'): -3,\n",
        "    ('T', 'R'): -1, ('P', 'L'): -3, ('K', 'G'): -2, ('M', 'N'): -2,\n",
        "    ('P', 'H'): -2, ('F', 'Q'): -3, ('Z', 'G'): -2, ('X', 'L'): -1,\n",
        "    ('T', 'M'): -1, ('Z', 'C'): -3, ('X', 'H'): -1, ('D', 'R'): -2,\n",
        "    ('B', 'W'): -4, ('X', 'D'): -1, ('Z', 'K'): 1, ('F', 'A'): -2,\n",
        "    ('Z', 'W'): -3, ('F', 'E'): -3, ('D', 'N'): 1, ('B', 'K'): 0,\n",
        "    ('X', 'X'): -1, ('F', 'I'): 0, ('B', 'G'): -1, ('X', 'T'): 0,\n",
        "    ('F', 'M'): 0, ('B', 'C'): -3, ('Z', 'I'): -3, ('Z', 'V'): -2,\n",
        "    ('S', 'S'): 4, ('L', 'Q'): -2, ('W', 'E'): -3, ('Q', 'R'): 1,\n",
        "    ('N', 'N'): 6, ('W', 'M'): -1, ('Q', 'C'): -3, ('W', 'I'): -3,\n",
        "    ('S', 'C'): -1, ('L', 'A'): -1, ('S', 'G'): 0, ('L', 'E'): -3,\n",
        "    ('W', 'Q'): -2, ('H', 'G'): -2, ('S', 'K'): 0, ('Q', 'N'): 0,\n",
        "    ('N', 'R'): 0, ('H', 'C'): -3, ('Y', 'N'): -2, ('G', 'Q'): -2,\n",
        "    ('Y', 'F'): 3, ('C', 'A'): 0, ('V', 'L'): 1, ('G', 'E'): -2,\n",
        "    ('G', 'A'): 0, ('K', 'R'): 2, ('E', 'D'): 2, ('Y', 'R'): -2,\n",
        "    ('M', 'Q'): 0, ('T', 'I'): -1, ('C', 'D'): -3, ('V', 'F'): -1,\n",
        "    ('T', 'A'): 0, ('T', 'P'): -1, ('B', 'P'): -2, ('T', 'E'): -1,\n",
        "    ('V', 'N'): -3, ('P', 'G'): -2, ('M', 'A'): -1, ('K', 'H'): -1,\n",
        "    ('V', 'R'): -3, ('P', 'C'): -3, ('M', 'E'): -2, ('K', 'L'): -2,\n",
        "    ('V', 'V'): 4, ('M', 'I'): 1, ('T', 'Q'): -1, ('I', 'G'): -4,\n",
        "    ('P', 'K'): -1, ('M', 'M'): 5, ('K', 'D'): -1, ('I', 'C'): -1,\n",
        "    ('Z', 'D'): 1, ('F', 'R'): -3, ('X', 'K'): -1, ('Q', 'D'): 0,\n",
        "    ('X', 'G'): -1, ('Z', 'L'): -3, ('X', 'C'): -2, ('Z', 'H'): 0,\n",
        "    ('B', 'L'): -4, ('B', 'H'): 0, ('F', 'F'): 6, ('X', 'W'): -2,\n",
        "    ('B', 'D'): 4, ('D', 'A'): -2, ('S', 'L'): -2, ('X', 'S'): 0,\n",
        "    ('F', 'N'): -3, ('S', 'R'): -1, ('W', 'D'): -4, ('V', 'Y'): -1,\n",
        "    ('W', 'L'): -2, ('H', 'R'): 0, ('W', 'H'): -2, ('H', 'N'): 1,\n",
        "    ('W', 'T'): -2, ('T', 'T'): 5, ('S', 'F'): -2, ('W', 'P'): -4,\n",
        "    ('L', 'D'): -4, ('B', 'I'): -3, ('L', 'H'): -3, ('S', 'N'): 1,\n",
        "    ('B', 'T'): -1, ('L', 'L'): 4, ('Y', 'K'): -2, ('E', 'Q'): 2,\n",
        "    ('Y', 'G'): -3, ('Z', 'S'): 0, ('Y', 'C'): -2, ('G', 'D'): -1,\n",
        "    ('B', 'V'): -3, ('E', 'A'): -1, ('Y', 'W'): 2, ('E', 'E'): 5,\n",
        "    ('Y', 'S'): -2, ('C', 'N'): -3, ('V', 'C'): -1, ('T', 'H'): -2,\n",
        "    ('P', 'R'): -2, ('V', 'G'): -3, ('T', 'L'): -1, ('V', 'K'): -2,\n",
        "    ('K', 'Q'): 1, ('R', 'A'): -1, ('I', 'R'): -3, ('T', 'D'): -1,\n",
        "    ('P', 'F'): -4, ('I', 'N'): -3, ('K', 'I'): -3, ('M', 'D'): -3,\n",
        "    ('V', 'W'): -3, ('W', 'W'): 11, ('M', 'H'): -2, ('P', 'N'): -2,\n",
        "    ('K', 'A'): -1, ('M', 'L'): 2, ('K', 'E'): 1, ('Z', 'E'): 4,\n",
        "    ('X', 'N'): -1, ('Z', 'A'): -1, ('Z', 'M'): -1, ('X', 'F'): -1,\n",
        "    ('K', 'C'): -3, ('B', 'Q'): 0, ('X', 'B'): -1, ('B', 'M'): -3,\n",
        "    ('F', 'C'): -2, ('Z', 'Q'): 3, ('X', 'Z'): -1, ('F', 'G'): -3,\n",
        "    ('B', 'E'): 1, ('X', 'V'): -1, ('F', 'K'): -3, ('B', 'A'): -2,\n",
        "    ('X', 'R'): -1, ('D', 'D'): 6, ('W', 'G'): -2, ('Z', 'F'): -3,\n",
        "    ('S', 'Q'): 0, ('W', 'C'): -2, ('W', 'K'): -3, ('H', 'Q'): 0,\n",
        "    ('L', 'C'): -1, ('W', 'N'): -4, ('S', 'A'): 1, ('L', 'G'): -4,\n",
        "    ('W', 'S'): -3, ('S', 'E'): 0, ('H', 'E'): 0, ('S', 'I'): -2,\n",
        "    ('H', 'A'): -2, ('S', 'M'): -1, ('Y', 'L'): -1, ('Y', 'H'): 2,\n",
        "    ('Y', 'D'): -3, ('E', 'R'): 0, ('X', 'P'): -2, ('G', 'G'): 6,\n",
        "    ('G', 'C'): -3, ('E', 'N'): 0, ('Y', 'T'): -2, ('Y', 'P'): -3,\n",
        "    ('T', 'K'): -1, ('A', 'A'): 4, ('P', 'Q'): -1, ('T', 'C'): -1,\n",
        "    ('V', 'H'): -3, ('T', 'G'): -2, ('I', 'Q'): -3, ('Z', 'T'): -1,\n",
        "    ('C', 'R'): -3, ('V', 'P'): -2, ('P', 'E'): -1, ('M', 'C'): -1,\n",
        "    ('K', 'N'): 0, ('I', 'I'): 4, ('P', 'A'): -1, ('M', 'G'): -3,\n",
        "    ('T', 'S'): 1, ('I', 'E'): -3, ('P', 'M'): -2, ('M', 'K'): -1,\n",
        "    ('I', 'A'): -1, ('P', 'I'): -3, ('R', 'R'): 5, ('X', 'M'): -1,\n",
        "    ('L', 'I'): 2, ('X', 'I'): -1, ('Z', 'B'): 1, ('X', 'E'): -1,\n",
        "    ('Z', 'N'): 0, ('X', 'A'): 0, ('B', 'R'): -1, ('B', 'N'): 3,\n",
        "    ('F', 'D'): -3, ('X', 'Y'): -1, ('Z', 'R'): 0, ('F', 'H'): -1,\n",
        "    ('B', 'F'): -3, ('F', 'L'): 0, ('X', 'Q'): -1, ('B', 'B'): 4\n",
        "}\n",
        "\n",
        "Blosum62 = {}\n",
        "for i in blosum62:\n",
        "  Blosum62[i] = blosum62[i]\n",
        "  Blosum62[i[::-1]] = blosum62[i]\n",
        "if \".\" not in Sequence:\n",
        "  with open(\"sequence.txt\",\"w\") as w:\n",
        "    w.write(\">seq1\\n{o}\".format(o = Sequence))\n",
        "    w.close()\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = \"sequence.txt\",d = \"blastdb\"\n",
        "  ,f = Blast_result_format,o = Blast_result_file_name , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  files.download(Blast_result_file_name)\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = \"sequence.txt\",d = \"blastdb\"\n",
        "  ,f = 6 ,o = \"results_dict.txt\" , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "\n",
        "  with open(\"results_dict.txt\") as w:\n",
        "    y = w.read()\n",
        "  idx = [i.split(\"\\t\")[1] for i in y.split(\"\\nseq\")]\n",
        "  for i in idx:\n",
        "    target_sequence = Dict[i]\n",
        "    aln = pairwise2.align.localds(Sequence,target_sequence, match_dict = Blosum62, open = -3, extend = -1)[0]\n",
        "    print(\"Subject_Id : \"+i+\"\\n\")\n",
        "    print(pairwise2.format_alignment(align1=aln[0], align2=aln[1], score=aln[2], begin=aln[3], end=aln[4], full_sequences=True))\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "if \".\" in Sequence:\n",
        "  command = \"blastp -query {q} -db blastdb -out {o} -outfmt {f} -max_target_seqs {m} -evalue {e}\".format(q = Sequence,d = \"blastdb\"\n",
        "  ,f = Blast_result_format,o = Blast_result_file_name , m = Max_target_Sequences , e = E_value_threshold)\n",
        "  exit_code, output = command_runner(command, shell=True, live_output=True)\n",
        "  files.download(Blast_result_file_name)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "cellView": "form",
        "id": "Fd2CuI4MHUuM",
        "outputId": "7657aa05-a67e-46a3-b4ed-a813db83a9a4"
      },
      "execution_count": 38,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "\n",
              "    async function download(id, filename, size) {\n",
              "      if (!google.colab.kernel.accessAllowed) {\n",
              "        return;\n",
              "      }\n",
              "      const div = document.createElement('div');\n",
              "      const label = document.createElement('label');\n",
              "      label.textContent = `Downloading \"${filename}\": `;\n",
              "      div.appendChild(label);\n",
              "      const progress = document.createElement('progress');\n",
              "      progress.max = size;\n",
              "      div.appendChild(progress);\n",
              "      document.body.appendChild(div);\n",
              "\n",
              "      const buffers = [];\n",
              "      let downloaded = 0;\n",
              "\n",
              "      const channel = await google.colab.kernel.comms.open(id);\n",
              "      // Send a message to notify the kernel that we're ready.\n",
              "      channel.send({})\n",
              "\n",
              "      for await (const message of channel.messages) {\n",
              "        // Send a message to notify the kernel that we're ready.\n",
              "        channel.send({})\n",
              "        if (message.buffers) {\n",
              "          for (const buffer of message.buffers) {\n",
              "            buffers.push(buffer);\n",
              "            downloaded += buffer.byteLength;\n",
              "            progress.value = downloaded;\n",
              "          }\n",
              "        }\n",
              "      }\n",
              "      const blob = new Blob(buffers, {type: 'application/binary'});\n",
              "      const a = document.createElement('a');\n",
              "      a.href = window.URL.createObjectURL(blob);\n",
              "      a.download = filename;\n",
              "      div.appendChild(a);\n",
              "      a.click();\n",
              "      div.remove();\n",
              "    }\n",
              "  "
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<IPython.core.display.Javascript object>"
            ],
            "application/javascript": [
              "download(\"download_327c2733-f359-46d2-860f-5385197b43a8\", \"Blast_result.txt\", 7998)"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Subject_Id : seq1049\n",
            "\n",
            "--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GIKEEAAAIKW----------EAAA-IKEGASPE-P--QPKG-IKEEAAAIKWEAAAIKEGASP--EPQPKPSG-DPQSKQTPEAS-PR----PQPEPSGKP--QSEQTPKASPEP---Q--------PKPSGQAKEIQ----WQAKEIQEQ---AKGPS-G-DPQ-SKQT---P-----EAS-----PRPQPEPSGKPQSEQTPKASP-EPQPGIKEEAAAIKWEAAAIKEGASPEPQPK--P--SGDPQSKQTPEAS--PRPQPEPS----GKP-QS---EQTP-KAS----PE-P-QPKPSGQAKEIQWQ-AKE--IQEQAKGKPSG-QAKEIQWQAKEIQEQAKGGIKEEAAAIKWEAAAIKEGASPEPQP-KP-S--G-DP---QSKQTPEASPRPQPEP-----------SGKPQSEQTPKAS---PEPQP--K-PSGQAKEIQWQAKEIQEQAKG--------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  ||||   ...|          ||.| ...|.|.| |  | .| .   |........... |  |  ||||   | ..|..   .|| ||    .||.|...|  .|..|  ..|.|   |        |.|| ||....    ..... ..|   .|||| . ||. ...|   |     |.|     .|.| | .|..|....|..|. |     |.     ...... ..| |.|| |.  |  |..|.|   |.|.  |.|.|..|    |.| .|   ..|| ...    |. | .|.||     |... |..  . ..|.|. || .|    ......   .. |..|..|...      ||   |..|| .| |  | ||   | ....|||....||.           ||.  |..|...|   ....|  . |.|                          \n",
            "MVRFGDELGGRYGGTGGGERARGGGAGGAGGPGQGGLPPGQRVLYKQSIAQRARTMALYNPIPVKQNCFTVNRSLFVFSEDNVVRKYAKRITEWPPFEYMILATIIANCIVLALEQHLPDGDKTPMSERLDDTEPYFIGIFCFEAGIKIIALGFVFHKGSYLRNGWNVMDFVVVLTGILATAGTDFDLRTLRAVRVLRPLKLVSGIPSLQVVLKSIMKAMVPLLQIGLLLFFAILMFAIIGLEFYMGKFHKACFPNSTDTEPVGDFPCGKDPPARQCDGDTECREYWPGPNFGITNFDNILFAILTVFQCITMEGWTDILYNTNDAAGNTWNWLYFIPLIIIGSFFMLNLVLGVLSGEFAKERERVENRRAFLKLRRQQQIERELNGYLEWIFKAEEVMLAEEDKNAEEKSPLDVLKRAATKKSRNDLIHAEEGEDRFVDLCAVGSPFARASLKSGKTESSSYFRRKEKMFRFFIRRMVKAQSFYWVVLCVVALNTLCVAMVHYNQPQRLTTALYFAEFVFLGLFLTEMSLKMYGLGPRSYFRSSFNCFDFGVIVGSIFEVVWAAIKPGTSFGISVLRALRLLRIFKVTKYWNSLRNLVVSLLNSMKSIISLLFLLFLFIVVFALLGMQLFGGQFNFQDETPTTNFDTFPAAILTVFQILTGEDWNAVMYHGIESQGGVSKGMFSSFYFIVLTLFGNYTLLNVFLAIAVDNLANAQELTKDEEEMEEAANQKLALQKAKEVAEVSPMSAANISIAARQQNSAKARSVWEQRASQLRLQNLRASCEALYSEMDPEERLRYASTRHVRPDMKTHMDRPLVVEPGRDGLRGPVGSKSKPEGTEATESADLPRRHHRHRDRDKTSATAPAGGEQDRTESTETGAREERARPRRSHSKETPGADTQVRCERSRRHHRRGSPEEATEREPRRHRAHRHAQDSSKEGTAPVLVPKGERRARHRGPRTGPREAENNEEPTRRHRARHKVPPTLQPPEREAAEKESNAVEGDKETRNHQPKEPHCDLEAIAVTGVGPLHMLPSTCLQKVDEQPEDADNQRNVTRMGSQPSDPSTTVHVPVTLTGPPGETPVVPSGNMNLEGQAEGKKEAEADDVLRRGPRPIVPYSSMFCLSPTNLLRRFCHYIVTMRYFEMVILVVIALSSIALAAEDPVRTDSFRNNALKYMDYIFTGVFTFEMVIKMIDLGLLLHPGAYFRDLWNILDFIVVSGALVAFAFSSFMGGSKGKDINTIKSLRVLRVLRPLKTIKRLPKLKAVFDCVVNSLKNVLNILIVYMLFMFIFAVIAVQLFKGKFFYCTDESKELERDCRGQYLDYEKEEVEAQPRQWKKYDFHYDNVLWALLTLFTVSTGEGWPMVLKHSVDATYEEQGPSPGFRMELSIFYVVYFVVFPFFFVNIFVALIIITFQEQGDKVMSECSLEKNERACIDFAISAKPLTRYMPQNKQSFQYKTWTFVVSPPFEYFIMAMIALNTVVLMMKFYDAPYEYELMLKCLNIVFTSMFSMECILKIIAFGVLNYFRDAWNVFDFVTVLGSITDILVTEIANNFINLSFLRLFRAARLIKLLRQGYTIRILLWTFVQSFKALPYVCLLIAMLFFIYAIIGMQVFGNIALDDDTSINRHNNFRTFLQALMLLFRSATGEAWHEIMLSCLGNRACDPHANASECGSDFAYFYFVSFIFLCSFLMLNLFVAVIMDNFEYLTRDSSILGPHHLDEFIRVWAEYDPAACGRISYNDMFEMLKHMSPPLGLGKKCPARVAYKRLVRMNMPISNEDMTVHFTSTLMALIRTALEIKLAPAGTKQHQCDAELRKEISSVWANLPQKTLDLLVPPHKPDEMTVGKVYAALMIFDFYKQNKTTRDQTHQAPGGLSQMGPVSLFHPLKATLEQTQPAVLRGARVFLRQKSATSLSNGGAIQTQESGIKE---SLSWGTQRTQDALYEARAPLERGHSAEIPVGQ-SGTL---AVDVQMQNMTLR-G--PDGEPQP---GLESQGR---AASMPRLAAETQPAPNASPMKRSIST--LAPRPHGTQLCSTVLDRPPPS-QASHHHHHRCHRRRD-KKQRSLEKGPSLSVDPEGAPSTAAGPGLPHGEGSTACRRDRKQ-E-RGRSQERRQPSSSSSE-----KQ-----RFYSCD-RFG-SREP-PQLMPSLSSHPTS---PTAALEPAPHPQGSGSVNGSPLMSTSGASTPGRGGRRQLPQTPLTPRPS-----ITYKTANSSPV-HFAEGQ-SGLPA----FSPGRL---SR-GLSEHNALLQ------KE---PLSQPLAPGSRIGSDPYLGQ-RLDSEASAHTLPEDTLTFEEAVATNSGR--SSRTSYVSSLTSQSHPLRRVPNGYHCTLGLSTGVRARHSYHHPDQDHWC\n",
            "  Score=354\n",
            "\n",
            "Subject_Id : seq1049\n",
            "\n",
            "--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GIKEEAAAIKW----------EAAA-IKEGASPE-P--QPKG-IKEEAAAIKWEAAAIKEGASP--EPQPKPSG-DPQSKQTPEAS-PR----PQPEPSGKP--QSEQTPKASPEP---Q--------PKPSGQAKEIQ----WQAKEIQEQ---AKGPS-G-DPQ-SKQT---P-----EAS-----PRPQPEPSGKPQSEQTPKASP-EPQPGIKEEAAAIKWEAAAIKEGASPEPQPK--P--SGDPQSKQTPEAS--PRPQPEPS----GKP-QS---EQTP-KAS----PE-P-QPKPSGQAKEIQWQ-AKE--IQEQAKGKPSG-QAKEIQWQAKEIQEQAKGGIKEEAAAIKWEAAAIKEGASPEPQP-KP-S--G-DP---QSKQTPEASPRPQPEP-----------SGKPQSEQTPKAS---PEPQP--K-PSGQAKEIQWQAKEIQEQAKG--------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  ||||   ...|          ||.| ...|.|.| |  | .| .   |........... |  |  ||||   | ..|..   .|| ||    .||.|...|  .|..|  ..|.|   |        |.|| ||....    ..... ..|   .|||| . ||. ...|   |     |.|     .|.| | .|..|....|..|. |     |.     ...... ..| |.|| |.  |  |..|.|   |.|.  |.|.|..|    |.| .|   ..|| ...    |. | .|.||     |... |..  . ..|.|. || .|    ......   .. |..|..|...      ||   |..|| .| |  | ||   | ....|||....||.           ||.  |..|...|   ....|  . |.|                          \n",
            "MVRFGDELGGRYGGTGGGERARGGGAGGAGGPGQGGLPPGQRVLYKQSIAQRARTMALYNPIPVKQNCFTVNRSLFVFSEDNVVRKYAKRITEWPPFEYMILATIIANCIVLALEQHLPDGDKTPMSERLDDTEPYFIGIFCFEAGIKIIALGFVFHKGSYLRNGWNVMDFVVVLTGILATAGTDFDLRTLRAVRVLRPLKLVSGIPSLQVVLKSIMKAMVPLLQIGLLLFFAILMFAIIGLEFYMGKFHKACFPNSTDTEPVGDFPCGKDPPARQCDGDTECREYWPGPNFGITNFDNILFAILTVFQCITMEGWTDILYNTNDAAGNTWNWLYFIPLIIIGSFFMLNLVLGVLSGEFAKERERVENRRAFLKLRRQQQIERELNGYLEWIFKAEEVMLAEEDKNAEEKSPLDVLKRAATKKSRNDLIHAEEGEDRFVDLCAVGSPFARASLKSGKTESSSYFRRKEKMFRFFIRRMVKAQSFYWVVLCVVALNTLCVAMVHYNQPQRLTTALYFAEFVFLGLFLTEMSLKMYGLGPRSYFRSSFNCFDFGVIVGSIFEVVWAAIKPGTSFGISVLRALRLLRIFKVTKYWNSLRNLVVSLLNSMKSIISLLFLLFLFIVVFALLGMQLFGGQFNFQDETPTTNFDTFPAAILTVFQILTGEDWNAVMYHGIESQGGVSKGMFSSFYFIVLTLFGNYTLLNVFLAIAVDNLANAQELTKDEEEMEEAANQKLALQKAKEVAEVSPMSAANISIAARQQNSAKARSVWEQRASQLRLQNLRASCEALYSEMDPEERLRYASTRHVRPDMKTHMDRPLVVEPGRDGLRGPVGSKSKPEGTEATESADLPRRHHRHRDRDKTSATAPAGGEQDRTESTETGAREERARPRRSHSKETPGADTQVRCERSRRHHRRGSPEEATEREPRRHRAHRHAQDSSKEGTAPVLVPKGERRARHRGPRTGPREAENNEEPTRRHRARHKVPPTLQPPEREAAEKESNAVEGDKETRNHQPKEPHCDLEAIAVTGVGPLHMLPSTCLQKVDEQPEDADNQRNVTRMGSQPSDPSTTVHVPVTLTGPPGETPVVPSGNMNLEGQAEGKKEAEADDVLRRGPRPIVPYSSMFCLSPTNLLRRFCHYIVTMRYFEMVILVVIALSSIALAAEDPVRTDSFRNNALKYMDYIFTGVFTFEMVIKMIDLGLLLHPGAYFRDLWNILDFIVVSGALVAFAFSSFMGGSKGKDINTIKSLRVLRVLRPLKTIKRLPKLKAVFDCVVNSLKNVLNILIVYMLFMFIFAVIAVQLFKGKFFYCTDESKELERDCRGQYLDYEKEEVEAQPRQWKKYDFHYDNVLWALLTLFTVSTGEGWPMVLKHSVDATYEEQGPSPGFRMELSIFYVVYFVVFPFFFVNIFVALIIITFQEQGDKVMSECSLEKNERACIDFAISAKPLTRYMPQNKQSFQYKTWTFVVSPPFEYFIMAMIALNTVVLMMKFYDAPYEYELMLKCLNIVFTSMFSMECILKIIAFGVLNYFRDAWNVFDFVTVLGSITDILVTEIANNFINLSFLRLFRAARLIKLLRQGYTIRILLWTFVQSFKALPYVCLLIAMLFFIYAIIGMQVFGNIALDDDTSINRHNNFRTFLQALMLLFRSATGEAWHEIMLSCLGNRACDPHANASECGSDFAYFYFVSFIFLCSFLMLNLFVAVIMDNFEYLTRDSSILGPHHLDEFIRVWAEYDPAACGRISYNDMFEMLKHMSPPLGLGKKCPARVAYKRLVRMNMPISNEDMTVHFTSTLMALIRTALEIKLAPAGTKQHQCDAELRKEISSVWANLPQKTLDLLVPPHKPDEMTVGKVYAALMIFDFYKQNKTTRDQTHQAPGGLSQMGPVSLFHPLKATLEQTQPAVLRGARVFLRQKSATSLSNGGAIQTQESGIKE---SLSWGTQRTQDALYEARAPLERGHSAEIPVGQ-SGTL---AVDVQMQNMTLR-G--PDGEPQP---GLESQGR---AASMPRLAAETQPAPNASPMKRSIST--LAPRPHGTQLCSTVLDRPPPS-QASHHHHHRCHRRRD-KKQRSLEKGPSLSVDPEGAPSTAAGPGLPHGEGSTACRRDRKQ-E-RGRSQERRQPSSSSSE-----KQ-----RFYSCD-RFG-SREP-PQLMPSLSSHPTS---PTAALEPAPHPQGSGSVNGSPLMSTSGASTPGRGGRRQLPQTPLTPRPS-----ITYKTANSSPV-HFAEGQ-SGLPA----FSPGRL---SR-GLSEHNALLQ------KE---PLSQPLAPGSRIGSDPYLGQ-RLDSEASAHTLPEDTLTFEEAVATNSGR--SSRTSYVSSLTSQSHPLRRVPNGYHCTLGLSTGVRARHSYHHPDQDHWC\n",
            "  Score=354\n",
            "\n",
            "Subject_Id : seq1049\n",
            "\n",
            "--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GIKEEAAAIKW----------EAAA-IKEGASPE-P--QPKG-IKEEAAAIKWEAAAIKEGASP--EPQPKPSG-DPQSKQTPEAS-PR----PQPEPSGKP--QSEQTPKASPEP---Q--------PKPSGQAKEIQ----WQAKEIQEQ---AKGPS-G-DPQ-SKQT---P-----EAS-----PRPQPEPSGKPQSEQTPKASP-EPQPGIKEEAAAIKWEAAAIKEGASPEPQPK--P--SGDPQSKQTPEAS--PRPQPEPS----GKP-QS---EQTP-KAS----PE-P-QPKPSGQAKEIQWQ-AKE--IQEQAKGKPSG-QAKEIQWQAKEIQEQAKGGIKEEAAAIKWEAAAIKEGASPEPQP-KP-S--G-DP---QSKQTPEASPRPQPEP-----------SGKPQSEQTPKAS---PEPQP--K-PSGQAKEIQWQAKEIQEQAKG--------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  ||||   ...|          ||.| ...|.|.| |  | .| .   |........... |  |  ||||   | ..|..   .|| ||    .||.|...|  .|..|  ..|.|   |        |.|| ||....    ..... ..|   .|||| . ||. ...|   |     |.|     .|.| | .|..|....|..|. |     |.     ...... ..| |.|| |.  |  |..|.|   |.|.  |.|.|..|    |.| .|   ..|| ...    |. | .|.||     |... |..  . ..|.|. || .|    ......   .. |..|..|...      ||   |..|| .| |  | ||   | ....|||....||.           ||.  |..|...|   ....|  . |.|                          \n",
            "MVRFGDELGGRYGGTGGGERARGGGAGGAGGPGQGGLPPGQRVLYKQSIAQRARTMALYNPIPVKQNCFTVNRSLFVFSEDNVVRKYAKRITEWPPFEYMILATIIANCIVLALEQHLPDGDKTPMSERLDDTEPYFIGIFCFEAGIKIIALGFVFHKGSYLRNGWNVMDFVVVLTGILATAGTDFDLRTLRAVRVLRPLKLVSGIPSLQVVLKSIMKAMVPLLQIGLLLFFAILMFAIIGLEFYMGKFHKACFPNSTDTEPVGDFPCGKDPPARQCDGDTECREYWPGPNFGITNFDNILFAILTVFQCITMEGWTDILYNTNDAAGNTWNWLYFIPLIIIGSFFMLNLVLGVLSGEFAKERERVENRRAFLKLRRQQQIERELNGYLEWIFKAEEVMLAEEDKNAEEKSPLDVLKRAATKKSRNDLIHAEEGEDRFVDLCAVGSPFARASLKSGKTESSSYFRRKEKMFRFFIRRMVKAQSFYWVVLCVVALNTLCVAMVHYNQPQRLTTALYFAEFVFLGLFLTEMSLKMYGLGPRSYFRSSFNCFDFGVIVGSIFEVVWAAIKPGTSFGISVLRALRLLRIFKVTKYWNSLRNLVVSLLNSMKSIISLLFLLFLFIVVFALLGMQLFGGQFNFQDETPTTNFDTFPAAILTVFQILTGEDWNAVMYHGIESQGGVSKGMFSSFYFIVLTLFGNYTLLNVFLAIAVDNLANAQELTKDEEEMEEAANQKLALQKAKEVAEVSPMSAANISIAARQQNSAKARSVWEQRASQLRLQNLRASCEALYSEMDPEERLRYASTRHVRPDMKTHMDRPLVVEPGRDGLRGPVGSKSKPEGTEATESADLPRRHHRHRDRDKTSATAPAGGEQDRTESTETGAREERARPRRSHSKETPGADTQVRCERSRRHHRRGSPEEATEREPRRHRAHRHAQDSSKEGTAPVLVPKGERRARHRGPRTGPREAENNEEPTRRHRARHKVPPTLQPPEREAAEKESNAVEGDKETRNHQPKEPHCDLEAIAVTGVGPLHMLPSTCLQKVDEQPEDADNQRNVTRMGSQPSDPSTTVHVPVTLTGPPGETPVVPSGNMNLEGQAEGKKEAEADDVLRRGPRPIVPYSSMFCLSPTNLLRRFCHYIVTMRYFEMVILVVIALSSIALAAEDPVRTDSFRNNALKYMDYIFTGVFTFEMVIKMIDLGLLLHPGAYFRDLWNILDFIVVSGALVAFAFSSFMGGSKGKDINTIKSLRVLRVLRPLKTIKRLPKLKAVFDCVVNSLKNVLNILIVYMLFMFIFAVIAVQLFKGKFFYCTDESKELERDCRGQYLDYEKEEVEAQPRQWKKYDFHYDNVLWALLTLFTVSTGEGWPMVLKHSVDATYEEQGPSPGFRMELSIFYVVYFVVFPFFFVNIFVALIIITFQEQGDKVMSECSLEKNERACIDFAISAKPLTRYMPQNKQSFQYKTWTFVVSPPFEYFIMAMIALNTVVLMMKFYDAPYEYELMLKCLNIVFTSMFSMECILKIIAFGVLNYFRDAWNVFDFVTVLGSITDILVTEIANNFINLSFLRLFRAARLIKLLRQGYTIRILLWTFVQSFKALPYVCLLIAMLFFIYAIIGMQVFGNIALDDDTSINRHNNFRTFLQALMLLFRSATGEAWHEIMLSCLGNRACDPHANASECGSDFAYFYFVSFIFLCSFLMLNLFVAVIMDNFEYLTRDSSILGPHHLDEFIRVWAEYDPAACGRISYNDMFEMLKHMSPPLGLGKKCPARVAYKRLVRMNMPISNEDMTVHFTSTLMALIRTALEIKLAPAGTKQHQCDAELRKEISSVWANLPQKTLDLLVPPHKPDEMTVGKVYAALMIFDFYKQNKTTRDQTHQAPGGLSQMGPVSLFHPLKATLEQTQPAVLRGARVFLRQKSATSLSNGGAIQTQESGIKE---SLSWGTQRTQDALYEARAPLERGHSAEIPVGQ-SGTL---AVDVQMQNMTLR-G--PDGEPQP---GLESQGR---AASMPRLAAETQPAPNASPMKRSIST--LAPRPHGTQLCSTVLDRPPPS-QASHHHHHRCHRRRD-KKQRSLEKGPSLSVDPEGAPSTAAGPGLPHGEGSTACRRDRKQ-E-RGRSQERRQPSSSSSE-----KQ-----RFYSCD-RFG-SREP-PQLMPSLSSHPTS---PTAALEPAPHPQGSGSVNGSPLMSTSGASTPGRGGRRQLPQTPLTPRPS-----ITYKTANSSPV-HFAEGQ-SGLPA----FSPGRL---SR-GLSEHNALLQ------KE---PLSQPLAPGSRIGSDPYLGQ-RLDSEASAHTLPEDTLTFEEAVATNSGR--SSRTSYVSSLTSQSHPLRRVPNGYHCTLGLSTGVRARHSYHHPDQDHWC\n",
            "  Score=354\n",
            "\n",
            "Subject_Id : seq1049\n",
            "\n",
            "--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GIKEEAAAIKW----------EAAA-IKEGASPE-P--QPKG-IKEEAAAIKWEAAAIKEGASP--EPQPKPSG-DPQSKQTPEAS-PR----PQPEPSGKP--QSEQTPKASPEP---Q--------PKPSGQAKEIQ----WQAKEIQEQ---AKGPS-G-DPQ-SKQT---P-----EAS-----PRPQPEPSGKPQSEQTPKASP-EPQPGIKEEAAAIKWEAAAIKEGASPEPQPK--P--SGDPQSKQTPEAS--PRPQPEPS----GKP-QS---EQTP-KAS----PE-P-QPKPSGQAKEIQWQ-AKE--IQEQAKGKPSG-QAKEIQWQAKEIQEQAKGGIKEEAAAIKWEAAAIKEGASPEPQP-KP-S--G-DP---QSKQTPEASPRPQPEP-----------SGKPQSEQTPKAS---PEPQP--K-PSGQAKEIQWQAKEIQEQAKG--------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  ||||   ...|          ||.| ...|.|.| |  | .| .   |........... |  |  ||||   | ..|..   .|| ||    .||.|...|  .|..|  ..|.|   |        |.|| ||....    ..... ..|   .|||| . ||. ...|   |     |.|     .|.| | .|..|....|..|. |     |.     ...... ..| |.|| |.  |  |..|.|   |.|.  |.|.|..|    |.| .|   ..|| ...    |. | .|.||     |... |..  . ..|.|. || .|    ......   .. |..|..|...      ||   |..|| .| |  | ||   | ....|||....||.           ||.  |..|...|   ....|  . |.|                          \n",
            "MVRFGDELGGRYGGTGGGERARGGGAGGAGGPGQGGLPPGQRVLYKQSIAQRARTMALYNPIPVKQNCFTVNRSLFVFSEDNVVRKYAKRITEWPPFEYMILATIIANCIVLALEQHLPDGDKTPMSERLDDTEPYFIGIFCFEAGIKIIALGFVFHKGSYLRNGWNVMDFVVVLTGILATAGTDFDLRTLRAVRVLRPLKLVSGIPSLQVVLKSIMKAMVPLLQIGLLLFFAILMFAIIGLEFYMGKFHKACFPNSTDTEPVGDFPCGKDPPARQCDGDTECREYWPGPNFGITNFDNILFAILTVFQCITMEGWTDILYNTNDAAGNTWNWLYFIPLIIIGSFFMLNLVLGVLSGEFAKERERVENRRAFLKLRRQQQIERELNGYLEWIFKAEEVMLAEEDKNAEEKSPLDVLKRAATKKSRNDLIHAEEGEDRFVDLCAVGSPFARASLKSGKTESSSYFRRKEKMFRFFIRRMVKAQSFYWVVLCVVALNTLCVAMVHYNQPQRLTTALYFAEFVFLGLFLTEMSLKMYGLGPRSYFRSSFNCFDFGVIVGSIFEVVWAAIKPGTSFGISVLRALRLLRIFKVTKYWNSLRNLVVSLLNSMKSIISLLFLLFLFIVVFALLGMQLFGGQFNFQDETPTTNFDTFPAAILTVFQILTGEDWNAVMYHGIESQGGVSKGMFSSFYFIVLTLFGNYTLLNVFLAIAVDNLANAQELTKDEEEMEEAANQKLALQKAKEVAEVSPMSAANISIAARQQNSAKARSVWEQRASQLRLQNLRASCEALYSEMDPEERLRYASTRHVRPDMKTHMDRPLVVEPGRDGLRGPVGSKSKPEGTEATESADLPRRHHRHRDRDKTSATAPAGGEQDRTESTETGAREERARPRRSHSKETPGADTQVRCERSRRHHRRGSPEEATEREPRRHRAHRHAQDSSKEGTAPVLVPKGERRARHRGPRTGPREAENNEEPTRRHRARHKVPPTLQPPEREAAEKESNAVEGDKETRNHQPKEPHCDLEAIAVTGVGPLHMLPSTCLQKVDEQPEDADNQRNVTRMGSQPSDPSTTVHVPVTLTGPPGETPVVPSGNMNLEGQAEGKKEAEADDVLRRGPRPIVPYSSMFCLSPTNLLRRFCHYIVTMRYFEMVILVVIALSSIALAAEDPVRTDSFRNNALKYMDYIFTGVFTFEMVIKMIDLGLLLHPGAYFRDLWNILDFIVVSGALVAFAFSSFMGGSKGKDINTIKSLRVLRVLRPLKTIKRLPKLKAVFDCVVNSLKNVLNILIVYMLFMFIFAVIAVQLFKGKFFYCTDESKELERDCRGQYLDYEKEEVEAQPRQWKKYDFHYDNVLWALLTLFTVSTGEGWPMVLKHSVDATYEEQGPSPGFRMELSIFYVVYFVVFPFFFVNIFVALIIITFQEQGDKVMSECSLEKNERACIDFAISAKPLTRYMPQNKQSFQYKTWTFVVSPPFEYFIMAMIALNTVVLMMKFYDAPYEYELMLKCLNIVFTSMFSMECILKIIAFGVLNYFRDAWNVFDFVTVLGSITDILVTEIANNFINLSFLRLFRAARLIKLLRQGYTIRILLWTFVQSFKALPYVCLLIAMLFFIYAIIGMQVFGNIALDDDTSINRHNNFRTFLQALMLLFRSATGEAWHEIMLSCLGNRACDPHANASECGSDFAYFYFVSFIFLCSFLMLNLFVAVIMDNFEYLTRDSSILGPHHLDEFIRVWAEYDPAACGRISYNDMFEMLKHMSPPLGLGKKCPARVAYKRLVRMNMPISNEDMTVHFTSTLMALIRTALEIKLAPAGTKQHQCDAELRKEISSVWANLPQKTLDLLVPPHKPDEMTVGKVYAALMIFDFYKQNKTTRDQTHQAPGGLSQMGPVSLFHPLKATLEQTQPAVLRGARVFLRQKSATSLSNGGAIQTQESGIKE---SLSWGTQRTQDALYEARAPLERGHSAEIPVGQ-SGTL---AVDVQMQNMTLR-G--PDGEPQP---GLESQGR---AASMPRLAAETQPAPNASPMKRSIST--LAPRPHGTQLCSTVLDRPPPS-QASHHHHHRCHRRRD-KKQRSLEKGPSLSVDPEGAPSTAAGPGLPHGEGSTACRRDRKQ-E-RGRSQERRQPSSSSSE-----KQ-----RFYSCD-RFG-SREP-PQLMPSLSSHPTS---PTAALEPAPHPQGSGSVNGSPLMSTSGASTPGRGGRRQLPQTPLTPRPS-----ITYKTANSSPV-HFAEGQ-SGLPA----FSPGRL---SR-GLSEHNALLQ------KE---PLSQPLAPGSRIGSDPYLGQ-RLDSEASAHTLPEDTLTFEEAVATNSGR--SSRTSYVSSLTSQSHPLRRVPNGYHCTLGLSTGVRARHSYHHPDQDHWC\n",
            "  Score=354\n",
            "\n",
            "Subject_Id : seq1049\n",
            "\n",
            "--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GIKEEAAAIKW----------EAAA-IKEGASPE-P--QPKG-IKEEAAAIKWEAAAIKEGASP--EPQPKPSG-DPQSKQTPEAS-PR----PQPEPSGKP--QSEQTPKASPEP---Q--------PKPSGQAKEIQ----WQAKEIQEQ---AKGPS-G-DPQ-SKQT---P-----EAS-----PRPQPEPSGKPQSEQTPKASP-EPQPGIKEEAAAIKWEAAAIKEGASPEPQPK--P--SGDPQSKQTPEAS--PRPQPEPS----GKP-QS---EQTP-KAS----PE-P-QPKPSGQAKEIQWQ-AKE--IQEQAKGKPSG-QAKEIQWQAKEIQEQAKGGIKEEAAAIKWEAAAIKEGASPEPQP-KP-S--G-DP---QSKQTPEASPRPQPEP-----------SGKPQSEQTPKAS---PEPQP--K-PSGQAKEIQWQAKEIQEQAKG--------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  ||||   ...|          ||.| ...|.|.| |  | .| .   |........... |  |  ||||   | ..|..   .|| ||    .||.|...|  .|..|  ..|.|   |        |.|| ||....    ..... ..|   .|||| . ||. ...|   |     |.|     .|.| | .|..|....|..|. |     |.     ...... ..| |.|| |.  |  |..|.|   |.|.  |.|.|..|    |.| .|   ..|| ...    |. | .|.||     |... |..  . ..|.|. || .|    ......   .. |..|..|...      ||   |..|| .| |  | ||   | ....|||....||.           ||.  |..|...|   ....|  . |.|                          \n",
            "MVRFGDELGGRYGGTGGGERARGGGAGGAGGPGQGGLPPGQRVLYKQSIAQRARTMALYNPIPVKQNCFTVNRSLFVFSEDNVVRKYAKRITEWPPFEYMILATIIANCIVLALEQHLPDGDKTPMSERLDDTEPYFIGIFCFEAGIKIIALGFVFHKGSYLRNGWNVMDFVVVLTGILATAGTDFDLRTLRAVRVLRPLKLVSGIPSLQVVLKSIMKAMVPLLQIGLLLFFAILMFAIIGLEFYMGKFHKACFPNSTDTEPVGDFPCGKDPPARQCDGDTECREYWPGPNFGITNFDNILFAILTVFQCITMEGWTDILYNTNDAAGNTWNWLYFIPLIIIGSFFMLNLVLGVLSGEFAKERERVENRRAFLKLRRQQQIERELNGYLEWIFKAEEVMLAEEDKNAEEKSPLDVLKRAATKKSRNDLIHAEEGEDRFVDLCAVGSPFARASLKSGKTESSSYFRRKEKMFRFFIRRMVKAQSFYWVVLCVVALNTLCVAMVHYNQPQRLTTALYFAEFVFLGLFLTEMSLKMYGLGPRSYFRSSFNCFDFGVIVGSIFEVVWAAIKPGTSFGISVLRALRLLRIFKVTKYWNSLRNLVVSLLNSMKSIISLLFLLFLFIVVFALLGMQLFGGQFNFQDETPTTNFDTFPAAILTVFQILTGEDWNAVMYHGIESQGGVSKGMFSSFYFIVLTLFGNYTLLNVFLAIAVDNLANAQELTKDEEEMEEAANQKLALQKAKEVAEVSPMSAANISIAARQQNSAKARSVWEQRASQLRLQNLRASCEALYSEMDPEERLRYASTRHVRPDMKTHMDRPLVVEPGRDGLRGPVGSKSKPEGTEATESADLPRRHHRHRDRDKTSATAPAGGEQDRTESTETGAREERARPRRSHSKETPGADTQVRCERSRRHHRRGSPEEATEREPRRHRAHRHAQDSSKEGTAPVLVPKGERRARHRGPRTGPREAENNEEPTRRHRARHKVPPTLQPPEREAAEKESNAVEGDKETRNHQPKEPHCDLEAIAVTGVGPLHMLPSTCLQKVDEQPEDADNQRNVTRMGSQPSDPSTTVHVPVTLTGPPGETPVVPSGNMNLEGQAEGKKEAEADDVLRRGPRPIVPYSSMFCLSPTNLLRRFCHYIVTMRYFEMVILVVIALSSIALAAEDPVRTDSFRNNALKYMDYIFTGVFTFEMVIKMIDLGLLLHPGAYFRDLWNILDFIVVSGALVAFAFSSFMGGSKGKDINTIKSLRVLRVLRPLKTIKRLPKLKAVFDCVVNSLKNVLNILIVYMLFMFIFAVIAVQLFKGKFFYCTDESKELERDCRGQYLDYEKEEVEAQPRQWKKYDFHYDNVLWALLTLFTVSTGEGWPMVLKHSVDATYEEQGPSPGFRMELSIFYVVYFVVFPFFFVNIFVALIIITFQEQGDKVMSECSLEKNERACIDFAISAKPLTRYMPQNKQSFQYKTWTFVVSPPFEYFIMAMIALNTVVLMMKFYDAPYEYELMLKCLNIVFTSMFSMECILKIIAFGVLNYFRDAWNVFDFVTVLGSITDILVTEIANNFINLSFLRLFRAARLIKLLRQGYTIRILLWTFVQSFKALPYVCLLIAMLFFIYAIIGMQVFGNIALDDDTSINRHNNFRTFLQALMLLFRSATGEAWHEIMLSCLGNRACDPHANASECGSDFAYFYFVSFIFLCSFLMLNLFVAVIMDNFEYLTRDSSILGPHHLDEFIRVWAEYDPAACGRISYNDMFEMLKHMSPPLGLGKKCPARVAYKRLVRMNMPISNEDMTVHFTSTLMALIRTALEIKLAPAGTKQHQCDAELRKEISSVWANLPQKTLDLLVPPHKPDEMTVGKVYAALMIFDFYKQNKTTRDQTHQAPGGLSQMGPVSLFHPLKATLEQTQPAVLRGARVFLRQKSATSLSNGGAIQTQESGIKE---SLSWGTQRTQDALYEARAPLERGHSAEIPVGQ-SGTL---AVDVQMQNMTLR-G--PDGEPQP---GLESQGR---AASMPRLAAETQPAPNASPMKRSIST--LAPRPHGTQLCSTVLDRPPPS-QASHHHHHRCHRRRD-KKQRSLEKGPSLSVDPEGAPSTAAGPGLPHGEGSTACRRDRKQ-E-RGRSQERRQPSSSSSE-----KQ-----RFYSCD-RFG-SREP-PQLMPSLSSHPTS---PTAALEPAPHPQGSGSVNGSPLMSTSGASTPGRGGRRQLPQTPLTPRPS-----ITYKTANSSPV-HFAEGQ-SGLPA----FSPGRL---SR-GLSEHNALLQ------KE---PLSQPLAPGSRIGSDPYLGQ-RLDSEASAHTLPEDTLTFEEAVATNSGR--SSRTSYVSSLTSQSHPLRRVPNGYHCTLGLSTGVRARHSYHHPDQDHWC\n",
            "  Score=354\n",
            "\n",
            "Subject_Id : seq203\n",
            "\n",
            "GIKEEAAAIKWEAAAIKEGASPEPQPKGIKEEAAAIKWEAAAIKEGASPEPQPKPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPK-ASPEPQPKPSGQAKEIQWQAKEIQEQAKGPS--G-----DPQ-S----KQTPEASPRPQPEPS-----GKPQSEQTPKASPEPQPGIKEEAAA-IKWEAAAIKEGASPEPQPKPSGDPQSKQTP-EASPRPQPEPSG--KPQSEQTPKASPEPQP-KPSG-QAKEIQWQAKEIQEQAKGKPSGQAKEIQWQAKEIQEQAKGGIKEEAAAIKWEAAAIKEG-ASP--EPQPKPS--G-DPQSKQTP--EAS----P--R-P--QPEP----S--GKP---QSE---QTPKASPEPQPKPSGQ-----AKEIQW-QAKEIQEQAKG-----------------\n",
            "                                                                           |||......|| .|.... |.....|....|.|.|.|..|. |  .     |.| .    | .|.|. |   |..     .|..| |  ||       .  |||| .|..   ..|..|          ...| || .|.....   .|  ||..| |    .|..| | |. ..|.   .|| .  .|| ||| .|.  ...| ..     |..   |...| ...  |.| |.|  ..|...|  | .|..  ||  ..|    |  | |  | ||    |  |.|   ..|   ..| |      . .|.     |.|... .......||                   \n",
            "-------------------------------------------------------------------------MASGKKKTSRKPKNRSVKNE-KLASFIKDFDSQVKIITEEMKA-SVVNILKEVDSQYNIEIIK-LPMAI-R---EMCWLDYIAKGGS-Q--KA-------L--EAAATVKVD---MEEITS----------TVTK-TPFKADKKVK---KGKCKPDDE-T----VELNPLK-SVIRTKT---KAK-V--AAK-KPS-TAR--RTRA-SV-----GNV---ANTSK-RTS--KRGRATPSASKQAETSLLGYTPAA--TPRIDTSIFKTPALRTPCMQ-EPVYTFSANGSPLAGMDELFINVP-A------R-DGKIIRLLASEVDGLDINRLDQQAFENIKLLSSRLERLCKKLK\n",
            "  Score=179\n",
            "\n",
            "Subject_Id : seq203\n",
            "\n",
            "GIKEEAAAIKWEAAAIKEGASPEPQPKGIKEEAAAIKWEAAAIKEGASPEPQPKPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPK-ASPEPQPKPSGQAKEIQWQAKEIQEQAKGPS--G-----DPQ-S----KQTPEASPRPQPEPS-----GKPQSEQTPKASPEPQPGIKEEAAA-IKWEAAAIKEGASPEPQPKPSGDPQSKQTP-EASPRPQPEPSG--KPQSEQTPKASPEPQP-KPSG-QAKEIQWQAKEIQEQAKGKPSGQAKEIQWQAKEIQEQAKGGIKEEAAAIKWEAAAIKEG-ASP--EPQPKPS--G-DPQSKQTP--EAS----P--R-P--QPEP----S--GKP---QSE---QTPKASPEPQPKPSGQ-----AKEIQW-QAKEIQEQAKG-----------------\n",
            "                                                                           |||......|| .|.... |.....|....|.|.|.|..|. |  .     |.| .    | .|.|. |   |..     .|..| |  ||       .  |||| .|..   ..|..|          ...| || .|.....   .|  ||..| |    .|..| | |. ..|.   .|| .  .|| ||| .|.  ...| ..     |..   |...| ...  |.| |.|  ..|...|  | .|..  ||  ..|    |  | |  | ||    |  |.|   ..|   ..| |      . .|.     |.|... .......||                   \n",
            "-------------------------------------------------------------------------MASGKKKTSRKPKNRSVKNE-KLASFIKDFDSQVKIITEEMKA-SVVNILKEVDSQYNIEIIK-LPMAI-R---EMCWLDYIAKGGS-Q--KA-------L--EAAATVKVD---MEEITS----------TVTK-TPFKADKKVK---KGKCKPDDE-T----VELNPLK-SVIRTKT---KAK-V--AAK-KPS-TAR--RTRA-SV-----GNV---ANTSK-RTS--KRGRATPSASKQAETSLLGYTPAA--TPRIDTSIFKTPALRTPCMQ-EPVYTFSANGSPLAGMDELFINVP-A------R-DGKIIRLLASEVDGLDINRLDQQAFENIKLLSSRLERLCKKLK\n",
            "  Score=179\n",
            "\n",
            "Subject_Id : seq203\n",
            "\n",
            "GIKEEAAAIKWEAAAIKEGASPEPQPKGIKEEAAAIKWEAAAIKEGASPEPQPKPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPK-ASPEPQPKPSGQAKEIQWQAKEIQEQAKGPS--G-----DPQ-S----KQTPEASPRPQPEPS-----GKPQSEQTPKASPEPQPGIKEEAAA-IKWEAAAIKEGASPEPQPKPSGDPQSKQTP-EASPRPQPEPSG--KPQSEQTPKASPEPQP-KPSG-QAKEIQWQAKEIQEQAKGKPSGQAKEIQWQAKEIQEQAKGGIKEEAAAIKWEAAAIKEG-ASP--EPQPKPS--G-DPQSKQTP--EAS----P--R-P--QPEP----S--GKP---QSE---QTPKASPEPQPKPSGQ-----AKEIQW-QAKEIQEQAKG-----------------\n",
            "                                                                           |||......|| .|.... |.....|....|.|.|.|..|. |  .     |.| .    | .|.|. |   |..     .|..| |  ||       .  |||| .|..   ..|..|          ...| || .|.....   .|  ||..| |    .|..| | |. ..|.   .|| .  .|| ||| .|.  ...| ..     |..   |...| ...  |.| |.|  ..|...|  | .|..  ||  ..|    |  | |  | ||    |  |.|   ..|   ..| |      . .|.     |.|... .......||                   \n",
            "-------------------------------------------------------------------------MASGKKKTSRKPKNRSVKNE-KLASFIKDFDSQVKIITEEMKA-SVVNILKEVDSQYNIEIIK-LPMAI-R---EMCWLDYIAKGGS-Q--KA-------L--EAAATVKVD---MEEITS----------TVTK-TPFKADKKVK---KGKCKPDDE-T----VELNPLK-SVIRTKT---KAK-V--AAK-KPS-TAR--RTRA-SV-----GNV---ANTSK-RTS--KRGRATPSASKQAETSLLGYTPAA--TPRIDTSIFKTPALRTPCMQ-EPVYTFSANGSPLAGMDELFINVP-A------R-DGKIIRLLASEVDGLDINRLDQQAFENIKLLSSRLERLCKKLK\n",
            "  Score=179\n",
            "\n",
            "Subject_Id : seq285\n",
            "\n",
            "------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------G--IKEEAAAIKWEA---AAIK----EG---ASPEPQ-P--KGI-KEEAAAIKWEAA-AIKEGASPE--------P-----QPKPSG---------DPQ---S-KQ-TP--EAS--PRPQPEPSG---KP---QSEQTP-------------KASPEP----QP---KPSGQA-K-EI------------QW--QAK-EIQE--QAKG----P---S--G---DPQSK--------QTP-E-----------A--SP--R-PQPEP--------S--G-KP---------------Q--------SEQTPK-ASP--------E----PQP-G-I---K-EEAAAIKWEAAAIKEGASPEPQPKPS--------GDPQSKQTPEA--SPR-PQP-EPSGKPQSEQTP-KA-----SPEPQPKPSGQAKEIQ-WQAKEIQEQ--------AKGKPSGQAK---EI-----QWQA--K-----EIQEQ-A---K-GGIKEEAA-AIKWEAAAIK-EGAS-PE----PQ-PKP--------SGDPQSKQTPEA----SP--RPQP-EPSG------K-PQ---SE---QTP-KAS-PEP-QPKPSGQA--KEIQWQA---KEIQE--QAKG--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |  ..|. ......|   ...|    .|   ....|| |  ... ...|.|.| |.. .|  |.|.|        |     | |...         |.|   | .. .|  ||.  .| .|| .|   ..   |..|||             |...|.    .|   |..||| . ..            .|  .|. |.||  .|..    |   .  .   |.| |        .|| .           |  .|  | |.| |        |  | |.               |        |....| |..        |    ||. | .   | ..  .. |....||   |    .|||        ||     |.|.  |.. |.| | .|...|  .| .|     |..|...|.    |.. | .......        || .|.|  |   |.     |..|  |     ||... |   | .|.|..|| ..  ||.|.| |... |.    .| ||.        |.|..||| |..    |.  ||.| ..||      | |.   |.   ..| ..| ||. .|||  ||  ||   .|   |.|..  .|.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               \n",
            "MCSIPGLPSKGSNVPVLITRVNLNPSCVLVEFWGNFDEDRKFAYQQLKKEIQYPRECFSESDGNPGDLCLVQVYETWYRARIVSRDSDEYSVFLIDEGRTLRAAVNTLAWGKSDFFYLPPEVEFCILANALPLSPENNWSSMALEFMKTFCGRRVNATVQDVLVAHRTFLLDIPCLSRQMFEMGFAKKLYSDQFMEFVVRSLQASTGTSDLKRISSIRTKPVEIIEQKEKQQAYMFPELQTDTVETVVITEVTSPFRIFCQLKVFSQELKKLTEQITQYYEGRVGSYFARAENLGSPCASRGSDGKWYRSVLQQVMSANNVVEVLHVDYGKKQFVQVENVKPLASEFFRMPVVTYVCSLHGIVDKGVGWTASQIDYLKSLLLNRTVIAKFQYQSLSEGVHYVTLYGEENTNINKLFELKPKCSLDSDMTLADFAVQKSPSSQKSKISRTTESTHINETYSDLKVNKPVFFTETLTPNSTHMAVVQHVDSPGKFWIQTQRYADEFDLLMNGLGNLYSDPTSTESLIRKPVVGLICAAKAQDGVFYRAAVYKVIDKTAEVYFLDYGNTEVVDSFNLRQLPLRFQQLPAVAVKCSLHGVKPRLKLWEERATLFFSKLVRDRIIDLHVQDKQQDTHIVQLVDPSLDGEKDVSKLLCNAGFAVSEK-SIVDYSATRSCGLKTTHASGVFLTGTQPQTPCSSSVVMDSASAFK-EYLFPI--GSSLEVTVSYIENPNDFWCQ-KARNAACLEVLMQDIQRFYSHSEFEPLLEAACVAR-HPE-TGIWYRALVIQKHQTPHVDVLFIDYGQTKKVAIEDLRKITPAFLKMKGQAFRCSLYNLIHPVLHSSSDWSTEATLEFQEFVDAAASMNVPLKCTIFAVMYDSQ-KVVFNVVDLETPFQSICNLLVQRRLADRAPSKRSPLP-PFRLDTYYYSTHGVKTGCDEKVSITCVKGVNQFFCHLARNSDEVEKLAEKVNFLCHQLEATKCPQTFGTVCFAKYTD--GL-WYRGQIK---S----TKPSVVINFVDYGD-----TLEVDKSDLLPVPIE-AGDIMS--VPVQAIECGLSDMPEELPC----EVDNW-FRKFADSHCFTALIVAK-EPAG--KLILELYDGKTQVNALIKQKFHNEIHKNDASTFKIYGLKSRAAESV--EASACKKESSTGPKRDAIDQVPKSRESHAIQRSNDVASKQ-PQSRWGFSTNGRPEPTRDSGTINNCQKQPELRTSQGNLRHPCTSSKPEVVKPKP--QALLKE---SALPIKSIKPGLEAEVFISHCNSPCSFFVQFATDEDDIYSLVEKLNADQSRCRNIDSSDIHEGDLVCAMFPDDSSWYRAVVRKNTNEKIDVEFVDFGNTAVISSKNVCHLGQSFASFPRYSIHCSVHKLNVDSKDQELAPNFKQVLEQNIEKVICTFVKMSGTMWEVRLDVNGVVLGSVCKDHVKPEIAIPDLKDAASEIKVCTYYKNPDISIGQVITGYTSYIKGPQLFWCQYVAMDKLQEISDMLQNIGNASETTLREDCMPVGSACIALFTEDNLWYRAKVTSKDLDTLSITFVDYGNESKVKIGDVKALPPKLSDVPPHAFDCQLEGFDVSEGFWDETADDAFYELVHDKPLNITIEKMGNSEMPHIVKLDCDGVDINTTMKSHWKTRNPETPPAELFNGAEMASDDDYVASKVNIDSVVTFDTDTDPADNETCTSALEMELSEQENLLSSTGVENEAQIDPLKMATENVTLPITESTVLSETHKKLETITEDEPVLGFTGLSDTNQHAVSKETDVGLPQHSEGASSVTIDSFLMNNTDSQLCIVEEPEAPSYEIIQSNLGCLRRATEKKPVGSECVIWSQVRRSWCTARVLKVSEEATLVLLEKYDSEVVVDPINIFEIMPEKPLQIACIEAAIANDDATKETDATLENSASKLYQTEVSDANGIAVALESEDLNGKEETFIDQMAPNDELAGQPQEEESVSCSAFLEDSKAKHMLVEGAQVHDLVQGLCPDDVESKDPQDDLNTSFEEQNDGAKMSTAVDLLLDFLDTAPRDKVQDVSETDALLEEFNIHVTEDLIVLTSDDGAESDTASDGTLHGDAVAMEVGPDTEESSCFQERSNASDCTSAEDSQVTHLTLKVEDASDDVIFVGVLQESQAVVHEPESEKEKRD\n",
            "  Score=287\n",
            "\n",
            "Subject_Id : seq285\n",
            "\n",
            "------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------G--IKEEAAAIKWEA---AAIK----EG---ASPEPQ-P--KGI-KEEAAAIKWEAA-AIKEGASPE--------P-----QPKPSG---------DPQ---S-KQ-TP--EAS--PRPQPEPSG---KP---QSEQTP-------------KASPEP----QP---KPSGQA-K-EI------------QW--QAK-EIQE--QAKG----P---S--G---DPQSK--------QTP-E-----------A--SP--R-PQPEP--------S--G-KP---------------Q--------SEQTPK-ASP--------E----PQP-G-I---K-EEAAAIKWEAAAIKEGASPEPQPKPS--------GDPQSKQTPEA--SPR-PQP-EPSGKPQSEQTP-KA-----SPEPQPKPSGQAKEIQ-WQAKEIQEQ--------AKGKPSGQAK---EI-----QWQA--K-----EIQEQ-A---K-GGIKEEAA-AIKWEAAAIK-EGAS-PE----PQ-PKP--------SGDPQSKQTPEA----SP--RPQP-EPSG------K-PQ---SE---QTP-KAS-PEP-QPKPSGQA--KEIQWQA---KEIQE--QAKG--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |  ..|. ......|   ...|    .|   ....|| |  ... ...|.|.| |.. .|  |.|.|        |     | |...         |.|   | .. .|  ||.  .| .|| .|   ..   |..|||             |...|.    .|   |..||| . ..            .|  .|. |.||  .|..    |   .  .   |.| |        .|| .           |  .|  | |.| |        |  | |.               |        |....| |..        |    ||. | .   | ..  .. |....||   |    .|||        ||     |.|.  |.. |.| | .|...|  .| .|     |..|...|.    |.. | .......        || .|.|  |   |.     |..|  |     ||... |   | .|.|..|| ..  ||.|.| |... |.    .| ||.        |.|..||| |..    |.  ||.| ..||      | |.   |.   ..| ..| ||. .|||  ||  ||   .|   |.|..  .|.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               \n",
            "MCSIPGLPSKGSNVPVLITRVNLNPSCVLVEFWGNFDEDRKFAYQQLKKEIQYPRECFSESDGNPGDLCLVQVYETWYRARIVSRDSDEYSVFLIDEGRTLRAAVNTLAWGKSDFFYLPPEVEFCILANALPLSPENNWSSMALEFMKTFCGRRVNATVQDVLVAHRTFLLDIPCLSRQMFEMGFAKKLYSDQFMEFVVRSLQASTGTSDLKRISSIRTKPVEIIEQKEKQQAYMFPELQTDTVETVVITEVTSPFRIFCQLKVFSQELKKLTEQITQYYEGRVGSYFARAENLGSPCASRGSDGKWYRSVLQQVMSANNVVEVLHVDYGKKQFVQVENVKPLASEFFRMPVVTYVCSLHGIVDKGVGWTASQIDYLKSLLLNRTVIAKFQYQSLSEGVHYVTLYGEENTNINKLFELKPKCSLDSDMTLADFAVQKSPSSQKSKISRTTESTHINETYSDLKVNKPVFFTETLTPNSTHMAVVQHVDSPGKFWIQTQRYADEFDLLMNGLGNLYSDPTSTESLIRKPVVGLICAAKAQDGVFYRAAVYKVIDKTAEVYFLDYGNTEVVDSFNLRQLPLRFQQLPAVAVKCSLHGVKPRLKLWEERATLFFSKLVRDRIIDLHVQDKQQDTHIVQLVDPSLDGEKDVSKLLCNAGFAVSEK-SIVDYSATRSCGLKTTHASGVFLTGTQPQTPCSSSVVMDSASAFK-EYLFPI--GSSLEVTVSYIENPNDFWCQ-KARNAACLEVLMQDIQRFYSHSEFEPLLEAACVAR-HPE-TGIWYRALVIQKHQTPHVDVLFIDYGQTKKVAIEDLRKITPAFLKMKGQAFRCSLYNLIHPVLHSSSDWSTEATLEFQEFVDAAASMNVPLKCTIFAVMYDSQ-KVVFNVVDLETPFQSICNLLVQRRLADRAPSKRSPLP-PFRLDTYYYSTHGVKTGCDEKVSITCVKGVNQFFCHLARNSDEVEKLAEKVNFLCHQLEATKCPQTFGTVCFAKYTD--GL-WYRGQIK---S----TKPSVVINFVDYGD-----TLEVDKSDLLPVPIE-AGDIMS--VPVQAIECGLSDMPEELPC----EVDNW-FRKFADSHCFTALIVAK-EPAG--KLILELYDGKTQVNALIKQKFHNEIHKNDASTFKIYGLKSRAAESV--EASACKKESSTGPKRDAIDQVPKSRESHAIQRSNDVASKQ-PQSRWGFSTNGRPEPTRDSGTINNCQKQPELRTSQGNLRHPCTSSKPEVVKPKP--QALLKE---SALPIKSIKPGLEAEVFISHCNSPCSFFVQFATDEDDIYSLVEKLNADQSRCRNIDSSDIHEGDLVCAMFPDDSSWYRAVVRKNTNEKIDVEFVDFGNTAVISSKNVCHLGQSFASFPRYSIHCSVHKLNVDSKDQELAPNFKQVLEQNIEKVICTFVKMSGTMWEVRLDVNGVVLGSVCKDHVKPEIAIPDLKDAASEIKVCTYYKNPDISIGQVITGYTSYIKGPQLFWCQYVAMDKLQEISDMLQNIGNASETTLREDCMPVGSACIALFTEDNLWYRAKVTSKDLDTLSITFVDYGNESKVKIGDVKALPPKLSDVPPHAFDCQLEGFDVSEGFWDETADDAFYELVHDKPLNITIEKMGNSEMPHIVKLDCDGVDINTTMKSHWKTRNPETPPAELFNGAEMASDDDYVASKVNIDSVVTFDTDTDPADNETCTSALEMELSEQENLLSSTGVENEAQIDPLKMATENVTLPITESTVLSETHKKLETITEDEPVLGFTGLSDTNQHAVSKETDVGLPQHSEGASSVTIDSFLMNNTDSQLCIVEEPEAPSYEIIQSNLGCLRRATEKKPVGSECVIWSQVRRSWCTARVLKVSEEATLVLLEKYDSEVVVDPINIFEIMPEKPLQIACIEAAIANDDATKETDATLENSASKLYQTEVSDANGIAVALESEDLNGKEETFIDQMAPNDELAGQPQEEESVSCSAFLEDSKAKHMLVEGAQVHDLVQGLCPDDVESKDPQDDLNTSFEEQNDGAKMSTAVDLLLDFLDTAPRDKVQDVSETDALLEEFNIHVTEDLIVLTSDDGAESDTASDGTLHGDAVAMEVGPDTEESSCFQERSNASDCTSAEDSQVTHLTLKVEDASDDVIFVGVLQESQAVVHEPESEKEKRD\n",
            "  Score=287\n",
            "\n",
            "Subject_Id : seq285\n",
            "\n",
            "------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------G--IKEEAAAIKWEA---AAIK----EG---ASPEPQ-P--KGI-KEEAAAIKWEAA-AIKEGASPE--------P-----QPKPSG---------DPQ---S-KQ-TP--EAS--PRPQPEPSG---KP---QSEQTP-------------KASPEP----QP---KPSGQA-K-EI------------QW--QAK-EIQE--QAKG----P---S--G---DPQSK--------QTP-E-----------A--SP--R-PQPEP--------S--G-KP---------------Q--------SEQTPK-ASP--------E----PQP-G-I---K-EEAAAIKWEAAAIKEGASPEPQPKPS--------GDPQSKQTPEA--SPR-PQP-EPSGKPQSEQTP-KA-----SPEPQPKPSGQAKEIQ-WQAKEIQEQ--------AKGKPSGQAK---EI-----QWQA--K-----EIQEQ-A---K-GGIKEEAA-AIKWEAAAIK-EGAS-PE----PQ-PKP--------SGDPQSKQTPEA----SP--RPQP-EPSG------K-PQ---SE---QTP-KAS-PEP-QPKPSGQA--KEIQWQA---KEIQE--QAKG--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |  ..|. ......|   ...|    .|   ....|| |  ... ...|.|.| |.. .|  |.|.|        |     | |...         |.|   | .. .|  ||.  .| .|| .|   ..   |..|||             |...|.    .|   |..||| . ..            .|  .|. |.||  .|..    |   .  .   |.| |        .|| .           |  .|  | |.| |        |  | |.               |        |....| |..        |    ||. | .   | ..  .. |....||   |    .|||        ||     |.|.  |.. |.| | .|...|  .| .|     |..|...|.    |.. | .......        || .|.|  |   |.     |..|  |     ||... |   | .|.|..|| ..  ||.|.| |... |.    .| ||.        |.|..||| |..    |.  ||.| ..||      | |.   |.   ..| ..| ||. .|||  ||  ||   .|   |.|..  .|.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               \n",
            "MCSIPGLPSKGSNVPVLITRVNLNPSCVLVEFWGNFDEDRKFAYQQLKKEIQYPRECFSESDGNPGDLCLVQVYETWYRARIVSRDSDEYSVFLIDEGRTLRAAVNTLAWGKSDFFYLPPEVEFCILANALPLSPENNWSSMALEFMKTFCGRRVNATVQDVLVAHRTFLLDIPCLSRQMFEMGFAKKLYSDQFMEFVVRSLQASTGTSDLKRISSIRTKPVEIIEQKEKQQAYMFPELQTDTVETVVITEVTSPFRIFCQLKVFSQELKKLTEQITQYYEGRVGSYFARAENLGSPCASRGSDGKWYRSVLQQVMSANNVVEVLHVDYGKKQFVQVENVKPLASEFFRMPVVTYVCSLHGIVDKGVGWTASQIDYLKSLLLNRTVIAKFQYQSLSEGVHYVTLYGEENTNINKLFELKPKCSLDSDMTLADFAVQKSPSSQKSKISRTTESTHINETYSDLKVNKPVFFTETLTPNSTHMAVVQHVDSPGKFWIQTQRYADEFDLLMNGLGNLYSDPTSTESLIRKPVVGLICAAKAQDGVFYRAAVYKVIDKTAEVYFLDYGNTEVVDSFNLRQLPLRFQQLPAVAVKCSLHGVKPRLKLWEERATLFFSKLVRDRIIDLHVQDKQQDTHIVQLVDPSLDGEKDVSKLLCNAGFAVSEK-SIVDYSATRSCGLKTTHASGVFLTGTQPQTPCSSSVVMDSASAFK-EYLFPI--GSSLEVTVSYIENPNDFWCQ-KARNAACLEVLMQDIQRFYSHSEFEPLLEAACVAR-HPE-TGIWYRALVIQKHQTPHVDVLFIDYGQTKKVAIEDLRKITPAFLKMKGQAFRCSLYNLIHPVLHSSSDWSTEATLEFQEFVDAAASMNVPLKCTIFAVMYDSQ-KVVFNVVDLETPFQSICNLLVQRRLADRAPSKRSPLP-PFRLDTYYYSTHGVKTGCDEKVSITCVKGVNQFFCHLARNSDEVEKLAEKVNFLCHQLEATKCPQTFGTVCFAKYTD--GL-WYRGQIK---S----TKPSVVINFVDYGD-----TLEVDKSDLLPVPIE-AGDIMS--VPVQAIECGLSDMPEELPC----EVDNW-FRKFADSHCFTALIVAK-EPAG--KLILELYDGKTQVNALIKQKFHNEIHKNDASTFKIYGLKSRAAESV--EASACKKESSTGPKRDAIDQVPKSRESHAIQRSNDVASKQ-PQSRWGFSTNGRPEPTRDSGTINNCQKQPELRTSQGNLRHPCTSSKPEVVKPKP--QALLKE---SALPIKSIKPGLEAEVFISHCNSPCSFFVQFATDEDDIYSLVEKLNADQSRCRNIDSSDIHEGDLVCAMFPDDSSWYRAVVRKNTNEKIDVEFVDFGNTAVISSKNVCHLGQSFASFPRYSIHCSVHKLNVDSKDQELAPNFKQVLEQNIEKVICTFVKMSGTMWEVRLDVNGVVLGSVCKDHVKPEIAIPDLKDAASEIKVCTYYKNPDISIGQVITGYTSYIKGPQLFWCQYVAMDKLQEISDMLQNIGNASETTLREDCMPVGSACIALFTEDNLWYRAKVTSKDLDTLSITFVDYGNESKVKIGDVKALPPKLSDVPPHAFDCQLEGFDVSEGFWDETADDAFYELVHDKPLNITIEKMGNSEMPHIVKLDCDGVDINTTMKSHWKTRNPETPPAELFNGAEMASDDDYVASKVNIDSVVTFDTDTDPADNETCTSALEMELSEQENLLSSTGVENEAQIDPLKMATENVTLPITESTVLSETHKKLETITEDEPVLGFTGLSDTNQHAVSKETDVGLPQHSEGASSVTIDSFLMNNTDSQLCIVEEPEAPSYEIIQSNLGCLRRATEKKPVGSECVIWSQVRRSWCTARVLKVSEEATLVLLEKYDSEVVVDPINIFEIMPEKPLQIACIEAAIANDDATKETDATLENSASKLYQTEVSDANGIAVALESEDLNGKEETFIDQMAPNDELAGQPQEEESVSCSAFLEDSKAKHMLVEGAQVHDLVQGLCPDDVESKDPQDDLNTSFEEQNDGAKMSTAVDLLLDFLDTAPRDKVQDVSETDALLEEFNIHVTEDLIVLTSDDGAESDTASDGTLHGDAVAMEVGPDTEESSCFQERSNASDCTSAEDSQVTHLTLKVEDASDDVIFVGVLQESQAVVHEPESEKEKRD\n",
            "  Score=287\n",
            "\n",
            "Subject_Id : seq266\n",
            "\n",
            "---------------------------------------------------------GIKEEAAAIKWEAAAIKEGASPEPQPK---GIKEEAAAI--KWEAAAIKE-GASPEPQPKPSGDPQ-SK---QTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGPSGDPQ-SKQTP---EASPRPQPEPS--GKPQS---EQTP--KASP-EPQPGIKEEAAAIKWEAAAIKEGASPEPQ---------PKPS-GDPQSKQTPEASPRPQ-PEPS-GKPQSEQT--P-----K-----ASP-EPQPKPSG-QAK---EIQWQAKEIQ---EQA-K-----G----------K-------PSGQAKEIQWQAKEIQEQAKGGIK--E-EAA--AIK-WEAAAIKEGASPEP--QPKPSGDPQS---KQTPEA-SPR----PQPE-PSGKPQSE--------QTPK-ASP--EPQPKP-SGQAKE-IQWQAK----E--IQ-EQA-K---G--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                         || .|..|  ||.   .|..    . |   | ||.|...  |.||.|..| | |..     .|... .|   |...|..| .|   |  |..| .|.||        ...|.. |..| |.|..| .|  . || .|   |||.|..| ||  |  .|   ....  .||. .|..||.|.   .. ....|.....||.|         ..|| |..||  |.||...|. ...| |....| |  |     .     ||. |....|.. .||   |.........   |.. |     |          |       ..|.||...|.|....|.||...|  | ...  ..| | |...     | |  ..|...| |.   | ..|. |..    .... |.|.|..|        ..|| ..|  |..|.| ...||| .. | |    |  .| |.. |   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      \n",
            "MATGAATENQWLKGRVKAVTSGDCLVITALTHNRAGPPPEKTITLSSLMAPKMARRGGI-DEPFA--WES---REFL----R-KLCIG-KEVAFKVDYKVEAIAGREFG-SVY-----LGNENLAKLVVQNGWAKVR-RP---G--QQNQ-DKVSP--------YIAELE-QLEE-QAQQEG-FG--RWSK-VPGAAEASIRNLP-PSAVG--DSGNFDAMGLLAASKGKPMEGIVEQ---VR-DGSTIRVYLLPEFQFVQVFVAGLQAPSMGRRQS--TQEAVVDPDVTATSNGDASAE-TRGPLTTAQRLAASAASSVEVSSDPFAMEAKYFTELRVLNRDVRIVLEGVDKFNNLIGSVYYSDGDTVKDLGLELVENGLAKYVEWSANMLDEEAKKKLKATELQCKKNRVKMW-ANYV-----P-PASNSKAIHD-QNFTGK-VVEVVSGDCLVVADDSIPFGSPMAERRVCLSSIRSPKMGNPRREEKPAPYAREAKEFLR-Q-KLIGMEVIVQMEYSRKISPGDGVTTSGAGDRVMDFGSVFLPSPTKGDTAVAAAATPGANIAELIISRGLGTVVRHRDFEERSNHYDALLAAEARAIAGKKNIHSAKDSPALHIADLTVASAKKAKDFLPSLQRINQISAVVEYVLSGHRFKLYIPKESCSIAFAFSGVRCPGRGEPYSEEAIALMRRKIMQRDVEIVVENVDRTGTFLGSMWEKNSKTNAGTYLLEAGLAKMQTGFGADRIPEAHILEMAERSAKNQKLKIWENYVEGEEVVNGSSKVETRQKETLKVVVTEVLGGGRFYVQTVGDQKVASIQNQLAALSLKDAPIIGSFNPKKGDIVLAQFSLDNSWNRAMIVNGPRGAVQSPEEEFEVFYIDYGNQEIVPYSAIRPVDPSVSSAPGLAQLCRLAYIKVPGKEEDFGRDAGEYLHTVTLESGKEFRAVVEERDTSGGKVKGQGTGTELVVTLIAVDDEISVNAAMLQEGIARMEKRRRWEPKDKQAALDALEKFQDEARKSRTGIWEYGDIQSDDEDNVPVRKPGRG\n",
            "  Score=287\n",
            "\n",
            "Subject_Id : seq266\n",
            "\n",
            "---------------------------------------------------------GIKEEAAAIKWEAAAIKEGASPEPQPK---GIKEEAAAI--KWEAAAIKE-GASPEPQPKPSGDPQ-SK---QTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGPSGDPQ-SKQTP---EASPRPQPEPS--GKPQS---EQTP--KASP-EPQPGIKEEAAAIKWEAAAIKEGASPEPQ---------PKPS-GDPQSKQTPEASPRPQ-PEPS-GKPQSEQT--P-----K-----ASP-EPQPKPSG-QAK---EIQWQAKEIQ---EQA-K-----G----------K-------PSGQAKEIQWQAKEIQEQAKGGIK--E-EAA--AIK-WEAAAIKEGASPEP--QPKPSGDPQS---KQTPEA-SPR----PQPE-PSGKPQSE--------QTPK-ASP--EPQPKP-SGQAKE-IQWQAK----E--IQ-EQA-K---G--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                         || .|..|  ||.   .|..    . |   | ||.|...  |.||.|..| | |..     .|... .|   |...|..| .|   |  |..| .|.||        ...|.. |..| |.|..| .|  . || .|   |||.|..| ||  |  .|   ....  .||. .|..||.|.   .. ....|.....||.|         ..|| |..||  |.||...|. ...| |....| |  |     .     ||. |....|.. .||   |.........   |.. |     |          |       ..|.||...|.|....|.||...|  | ...  ..| | |...     | |  ..|...| |.   | ..|. |..    .... |.|.|..|        ..|| ..|  |..|.| ...||| .. | |    |  .| |.. |   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      \n",
            "MATGAATENQWLKGRVKAVTSGDCLVITALTHNRAGPPPEKTITLSSLMAPKMARRGGI-DEPFA--WES---REFL----R-KLCIG-KEVAFKVDYKVEAIAGREFG-SVY-----LGNENLAKLVVQNGWAKVR-RP---G--QQNQ-DKVSP--------YIAELE-QLEE-QAQQEG-FG--RWSK-VPGAAEASIRNLP-PSAVG--DSGNFDAMGLLAASKGKPMEGIVEQ---VR-DGSTIRVYLLPEFQFVQVFVAGLQAPSMGRRQS--TQEAVVDPDVTATSNGDASAE-TRGPLTTAQRLAASAASSVEVSSDPFAMEAKYFTELRVLNRDVRIVLEGVDKFNNLIGSVYYSDGDTVKDLGLELVENGLAKYVEWSANMLDEEAKKKLKATELQCKKNRVKMW-ANYV-----P-PASNSKAIHD-QNFTGK-VVEVVSGDCLVVADDSIPFGSPMAERRVCLSSIRSPKMGNPRREEKPAPYAREAKEFLR-Q-KLIGMEVIVQMEYSRKISPGDGVTTSGAGDRVMDFGSVFLPSPTKGDTAVAAAATPGANIAELIISRGLGTVVRHRDFEERSNHYDALLAAEARAIAGKKNIHSAKDSPALHIADLTVASAKKAKDFLPSLQRINQISAVVEYVLSGHRFKLYIPKESCSIAFAFSGVRCPGRGEPYSEEAIALMRRKIMQRDVEIVVENVDRTGTFLGSMWEKNSKTNAGTYLLEAGLAKMQTGFGADRIPEAHILEMAERSAKNQKLKIWENYVEGEEVVNGSSKVETRQKETLKVVVTEVLGGGRFYVQTVGDQKVASIQNQLAALSLKDAPIIGSFNPKKGDIVLAQFSLDNSWNRAMIVNGPRGAVQSPEEEFEVFYIDYGNQEIVPYSAIRPVDPSVSSAPGLAQLCRLAYIKVPGKEEDFGRDAGEYLHTVTLESGKEFRAVVEERDTSGGKVKGQGTGTELVVTLIAVDDEISVNAAMLQEGIARMEKRRRWEPKDKQAALDALEKFQDEARKSRTGIWEYGDIQSDDEDNVPVRKPGRG\n",
            "  Score=287\n",
            "\n",
            "Subject_Id : seq266\n",
            "\n",
            "---------------------------------------------------------GIKEEAAAIKWEAAAIKEGASPEPQPK---GIKEEAAAI--KWEAAAIKE-GASPEPQPKPSGDPQ-SK---QTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGPSGDPQ-SKQTP---EASPRPQPEPS--GKPQS---EQTP--KASP-EPQPGIKEEAAAIKWEAAAIKEGASPEPQ---------PKPS-GDPQSKQTPEASPRPQ-PEPS-GKPQSEQT--P-----K-----ASP-EPQPKPSG-QAK---EIQWQAKEIQ---EQA-K-----G----------K-------PSGQAKEIQWQAKEIQEQAKGGIK--E-EAA--AIK-WEAAAIKEGASPEP--QPKPSGDPQS---KQTPEA-SPR----PQPE-PSGKPQSE--------QTPK-ASP--EPQPKP-SGQAKE-IQWQAK----E--IQ-EQA-K---G--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                         || .|..|  ||.   .|..    . |   | ||.|...  |.||.|..| | |..     .|... .|   |...|..| .|   |  |..| .|.||        ...|.. |..| |.|..| .|  . || .|   |||.|..| ||  |  .|   ....  .||. .|..||.|.   .. ....|.....||.|         ..|| |..||  |.||...|. ...| |....| |  |     .     ||. |....|.. .||   |.........   |.. |     |          |       ..|.||...|.|....|.||...|  | ...  ..| | |...     | |  ..|...| |.   | ..|. |..    .... |.|.|..|        ..|| ..|  |..|.| ...||| .. | |    |  .| |.. |   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      \n",
            "MATGAATENQWLKGRVKAVTSGDCLVITALTHNRAGPPPEKTITLSSLMAPKMARRGGI-DEPFA--WES---REFL----R-KLCIG-KEVAFKVDYKVEAIAGREFG-SVY-----LGNENLAKLVVQNGWAKVR-RP---G--QQNQ-DKVSP--------YIAELE-QLEE-QAQQEG-FG--RWSK-VPGAAEASIRNLP-PSAVG--DSGNFDAMGLLAASKGKPMEGIVEQ---VR-DGSTIRVYLLPEFQFVQVFVAGLQAPSMGRRQS--TQEAVVDPDVTATSNGDASAE-TRGPLTTAQRLAASAASSVEVSSDPFAMEAKYFTELRVLNRDVRIVLEGVDKFNNLIGSVYYSDGDTVKDLGLELVENGLAKYVEWSANMLDEEAKKKLKATELQCKKNRVKMW-ANYV-----P-PASNSKAIHD-QNFTGK-VVEVVSGDCLVVADDSIPFGSPMAERRVCLSSIRSPKMGNPRREEKPAPYAREAKEFLR-Q-KLIGMEVIVQMEYSRKISPGDGVTTSGAGDRVMDFGSVFLPSPTKGDTAVAAAATPGANIAELIISRGLGTVVRHRDFEERSNHYDALLAAEARAIAGKKNIHSAKDSPALHIADLTVASAKKAKDFLPSLQRINQISAVVEYVLSGHRFKLYIPKESCSIAFAFSGVRCPGRGEPYSEEAIALMRRKIMQRDVEIVVENVDRTGTFLGSMWEKNSKTNAGTYLLEAGLAKMQTGFGADRIPEAHILEMAERSAKNQKLKIWENYVEGEEVVNGSSKVETRQKETLKVVVTEVLGGGRFYVQTVGDQKVASIQNQLAALSLKDAPIIGSFNPKKGDIVLAQFSLDNSWNRAMIVNGPRGAVQSPEEEFEVFYIDYGNQEIVPYSAIRPVDPSVSSAPGLAQLCRLAYIKVPGKEEDFGRDAGEYLHTVTLESGKEFRAVVEERDTSGGKVKGQGTGTELVVTLIAVDDEISVNAAMLQEGIARMEKRRRWEPKDKQAALDALEKFQDEARKSRTGIWEYGDIQSDDEDNVPVRKPGRG\n",
            "  Score=287\n",
            "\n",
            "Subject_Id : seq266\n",
            "\n",
            "---------------------------------------------------------GIKEEAAAIKWEAAAIKEGASPEPQPK---GIKEEAAAI--KWEAAAIKE-GASPEPQPKPSGDPQ-SK---QTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGPSGDPQ-SKQTP---EASPRPQPEPS--GKPQS---EQTP--KASP-EPQPGIKEEAAAIKWEAAAIKEGASPEPQ---------PKPS-GDPQSKQTPEASPRPQ-PEPS-GKPQSEQT--P-----K-----ASP-EPQPKPSG-QAK---EIQWQAKEIQ---EQA-K-----G----------K-------PSGQAKEIQWQAKEIQEQAKGGIK--E-EAA--AIK-WEAAAIKEGASPEP--QPKPSGDPQS---KQTPEA-SPR----PQPE-PSGKPQSE--------QTPK-ASP--EPQPKP-SGQAKE-IQWQAK----E--IQ-EQA-K---G--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                         || .|..|  ||.   .|..    . |   | ||.|...  |.||.|..| | |..     .|... .|   |...|..| .|   |  |..| .|.||        ...|.. |..| |.|..| .|  . || .|   |||.|..| ||  |  .|   ....  .||. .|..||.|.   .. ....|.....||.|         ..|| |..||  |.||...|. ...| |....| |  |     .     ||. |....|.. .||   |.........   |.. |     |          |       ..|.||...|.|....|.||...|  | ...  ..| | |...     | |  ..|...| |.   | ..|. |..    .... |.|.|..|        ..|| ..|  |..|.| ...||| .. | |    |  .| |.. |   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      \n",
            "MATGAATENQWLKGRVKAVTSGDCLVITALTHNRAGPPPEKTITLSSLMAPKMARRGGI-DEPFA--WES---REFL----R-KLCIG-KEVAFKVDYKVEAIAGREFG-SVY-----LGNENLAKLVVQNGWAKVR-RP---G--QQNQ-DKVSP--------YIAELE-QLEE-QAQQEG-FG--RWSK-VPGAAEASIRNLP-PSAVG--DSGNFDAMGLLAASKGKPMEGIVEQ---VR-DGSTIRVYLLPEFQFVQVFVAGLQAPSMGRRQS--TQEAVVDPDVTATSNGDASAE-TRGPLTTAQRLAASAASSVEVSSDPFAMEAKYFTELRVLNRDVRIVLEGVDKFNNLIGSVYYSDGDTVKDLGLELVENGLAKYVEWSANMLDEEAKKKLKATELQCKKNRVKMW-ANYV-----P-PASNSKAIHD-QNFTGK-VVEVVSGDCLVVADDSIPFGSPMAERRVCLSSIRSPKMGNPRREEKPAPYAREAKEFLR-Q-KLIGMEVIVQMEYSRKISPGDGVTTSGAGDRVMDFGSVFLPSPTKGDTAVAAAATPGANIAELIISRGLGTVVRHRDFEERSNHYDALLAAEARAIAGKKNIHSAKDSPALHIADLTVASAKKAKDFLPSLQRINQISAVVEYVLSGHRFKLYIPKESCSIAFAFSGVRCPGRGEPYSEEAIALMRRKIMQRDVEIVVENVDRTGTFLGSMWEKNSKTNAGTYLLEAGLAKMQTGFGADRIPEAHILEMAERSAKNQKLKIWENYVEGEEVVNGSSKVETRQKETLKVVVTEVLGGGRFYVQTVGDQKVASIQNQLAALSLKDAPIIGSFNPKKGDIVLAQFSLDNSWNRAMIVNGPRGAVQSPEEEFEVFYIDYGNQEIVPYSAIRPVDPSVSSAPGLAQLCRLAYIKVPGKEEDFGRDAGEYLHTVTLESGKEFRAVVEERDTSGGKVKGQGTGTELVVTLIAVDDEISVNAAMLQEGIARMEKRRRWEPKDKQAALDALEKFQDEARKSRTGIWEYGDIQSDDEDNVPVRKPGRG\n",
            "  Score=287\n",
            "\n",
            "Subject_Id : seq267\n",
            "\n",
            "-------------------------------------------------------GIKEEAAAIKWEAAAIKEGASPEPQPK---GIKEEAAAI--KWEAAAIKE-GASPEPQPKPS---GDPQ-SK---QTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGPSGDPQ-SKQTP---EASPRPQPEPS--GKPQS---EQTP--KASP-EPQPGIKEEAAAIKWEAAAIKEGASPE---PQPKPSG-D-PQ-SKQTPEASP-RPQP-EPSGKPQSEQT-P-----K-----ASP-EPQPKP-SGQAK---E---------IQWQA--K------EIQ----EQAK--G-K--PSGQAKEIQWQAKEIQEQAKGGIKE-EAAAIKWEAAAIKEGAS--PEP--QPKPSGDPQS---KQTPEA-S-----PRPQPEPSGKPQSE--------QTPK-ASP--EPQPKP-SGQAKE-IQWQ---AKE--IQ-EQA-K---G------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                       || .|..|  ||.   ||..    . |   | ||.|...  |.||.|..| |         |   |... .|   .|..|..|   || |  |..| .|.||        ..||. .|..|...| .| .|  . || .|   |||.|..| ||  |  .|   ....  .|.. .|..||.|.   .. ....|.....||   .|....| . |. ...|...|. ...| ||.|....|.. |     .     ||. |....| ...||   |         |....  |      ...    |..|  | .  ..|.||...|.|....|.||   |. .||.........|..|.  | |  ..|...| |.   | ..|. |     ......|.|.|..|        ..|| ..|  |..|.| ...|.| .. |   .|.  .| |.. |   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                \n",
            "MATGAENQWLKGRVKAVTSGDCLVITALSHNRAGPPPEKTITFSSLMAPKMARRGGI-DEPFA--WES---KEFL----R-KLCIG-KEVAFKVDYKVEAIAGREFG---------SVFLGNENLAKLVVKTGWAKVR---EP-G--QQNQ-DKVSP--------YIKEL-LQLEELAKQ-EG-YG--RWSK-VPGAAEASIRNLP-PSAIG--DSAGFDAMGLLAANKGKPMEGIVEQ---VR-DGSTIRVYLLPEFQFVQVFVAGVQAPSMGRRTTNGSVVETVPDEPNGDVSAESRGPLTTAQRLAASAASSVEVSSDPFATEAKYFTEHRVLSRDVRIVLEGVDKFNNLIGSVHYSDGETVKDLGLELVENGLAKFVEWSANMMEEEAK---KKLKAAELQCKKDKVKMWANYVP-PATNSKAIHD-QNFTGK-VVEVVSGDCLIVADDAVPFGSPAAERRVCLSSIRSPKMGNPRREEKPAPYAREAREFLR-QRLIGKQVIVQMEYSRKVTQGDGPTTSGAADRFMDFGSVFLPSAAKADSDEVTAPPAAAIAGSQPVGVNIAELVLVRGFGNVVRHRDFEERSNHYDALLAAEARALAGKKGIHSAKESPAMHITDLTVSAAKKAKDFLPSLQRIRRIPAVVEYVLSGHRFKLYIPKITCSIAFSFSGVRCPGRGEPYSEEAISVMRRRIMQRDVEIEVETVDRTGTFLGSMWESRTNVATVLLEAGLAKMQTSFGADRIAEAHLLEQAERSAKNQKLKIWENYVEGEEVSNGNTNTVETRQKETLKVVVTEVLGGGRFYVQSAGDQKIASIQNQLASLSIKDAPIIGSFNPKRGDIVLAQFSLDNSWNRAMIVTAPRAAVQSPDEKFEVFYIDYGNQETVPYSAIRPIDPSVSAAPGLAQLCRLAYIKVPSLEDDFGPEAGEYLHTVTLGSGKEFKAVIEERDTSGGKVKGQGTGTEFVVTLIAVDDEISVNAAMLQEGIARMEKRQKWGHKGKQAALDALEKFQEEARKSRIGIWQYGDIESDDEDTGPARKPAGGRR\n",
            "  Score=295\n",
            "\n",
            "Subject_Id : seq267\n",
            "\n",
            "-------------------------------------------------------GIKEEAAAIKWEAAAIKEGASPEPQPK---GIKEEAAAI--KWEAAAIKE-GASPEPQPKPS---GDPQ-SK---QTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGPSGDPQ-SKQTP---EASPRPQPEPS--GKPQS---EQTP--KASP-EPQPGIKEEAAAIKWEAAAIKEGASPE---PQPKPSG-D-PQ-SKQTPEASP-RPQP-EPSGKPQSEQT-P-----K-----ASP-EPQPKP-SGQAK---E---------IQWQA--K------EIQ----EQAK--G-K--PSGQAKEIQWQAKEIQEQAKGGIKE-EAAAIKWEAAAIKEGAS--PEP--QPKPSGDPQS---KQTPEA-S-----PRPQPEPSGKPQSE--------QTPK-ASP--EPQPKP-SGQAKE-IQWQ---AKE--IQ-EQA-K---G------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                       || .|..|  ||.   ||..    . |   | ||.|...  |.||.|..| |         |   |... .|   .|..|..|   || |  |..| .|.||        ..||. .|..|...| .| .|  . || .|   |||.|..| ||  |  .|   ....  .|.. .|..||.|.   .. ....|.....||   .|....| . |. ...|...|. ...| ||.|....|.. |     .     ||. |....| ...||   |         |....  |      ...    |..|  | .  ..|.||...|.|....|.||   |. .||.........|..|.  | |  ..|...| |.   | ..|. |     ......|.|.|..|        ..|| ..|  |..|.| ...|.| .. |   .|.  .| |.. |   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                \n",
            "MATGAENQWLKGRVKAVTSGDCLVITALSHNRAGPPPEKTITFSSLMAPKMARRGGI-DEPFA--WES---KEFL----R-KLCIG-KEVAFKVDYKVEAIAGREFG---------SVFLGNENLAKLVVKTGWAKVR---EP-G--QQNQ-DKVSP--------YIKEL-LQLEELAKQ-EG-YG--RWSK-VPGAAEASIRNLP-PSAIG--DSAGFDAMGLLAANKGKPMEGIVEQ---VR-DGSTIRVYLLPEFQFVQVFVAGVQAPSMGRRTTNGSVVETVPDEPNGDVSAESRGPLTTAQRLAASAASSVEVSSDPFATEAKYFTEHRVLSRDVRIVLEGVDKFNNLIGSVHYSDGETVKDLGLELVENGLAKFVEWSANMMEEEAK---KKLKAAELQCKKDKVKMWANYVP-PATNSKAIHD-QNFTGK-VVEVVSGDCLIVADDAVPFGSPAAERRVCLSSIRSPKMGNPRREEKPAPYAREAREFLR-QRLIGKQVIVQMEYSRKVTQGDGPTTSGAADRFMDFGSVFLPSAAKADSDEVTAPPAAAIAGSQPVGVNIAELVLVRGFGNVVRHRDFEERSNHYDALLAAEARALAGKKGIHSAKESPAMHITDLTVSAAKKAKDFLPSLQRIRRIPAVVEYVLSGHRFKLYIPKITCSIAFSFSGVRCPGRGEPYSEEAISVMRRRIMQRDVEIEVETVDRTGTFLGSMWESRTNVATVLLEAGLAKMQTSFGADRIAEAHLLEQAERSAKNQKLKIWENYVEGEEVSNGNTNTVETRQKETLKVVVTEVLGGGRFYVQSAGDQKIASIQNQLASLSIKDAPIIGSFNPKRGDIVLAQFSLDNSWNRAMIVTAPRAAVQSPDEKFEVFYIDYGNQETVPYSAIRPIDPSVSAAPGLAQLCRLAYIKVPSLEDDFGPEAGEYLHTVTLGSGKEFKAVIEERDTSGGKVKGQGTGTEFVVTLIAVDDEISVNAAMLQEGIARMEKRQKWGHKGKQAALDALEKFQEEARKSRIGIWQYGDIESDDEDTGPARKPAGGRR\n",
            "  Score=295\n",
            "\n",
            "Subject_Id : seq267\n",
            "\n",
            "-------------------------------------------------------GIKEEAAAIKWEAAAIKEGASPEPQPK---GIKEEAAAI--KWEAAAIKE-GASPEPQPKPS---GDPQ-SK---QTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGPSGDPQ-SKQTP---EASPRPQPEPS--GKPQS---EQTP--KASP-EPQPGIKEEAAAIKWEAAAIKEGASPE---PQPKPSG-D-PQ-SKQTPEASP-RPQP-EPSGKPQSEQT-P-----K-----ASP-EPQPKP-SGQAK---E---------IQWQA--K------EIQ----EQAK--G-K--PSGQAKEIQWQAKEIQEQAKGGIKE-EAAAIKWEAAAIKEGAS--PEP--QPKPSGDPQS---KQTPEA-S-----PRPQPEPSGKPQSE--------QTPK-ASP--EPQPKP-SGQAKE-IQWQ---AKE--IQ-EQA-K---G------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                       || .|..|  ||.   ||..    . |   | ||.|...  |.||.|..| |         |   |... .|   .|..|..|   || |  |..| .|.||        ..||. .|..|...| .| .|  . || .|   |||.|..| ||  |  .|   ....  .|.. .|..||.|.   .. ....|.....||   .|....| . |. ...|...|. ...| ||.|....|.. |     .     ||. |....| ...||   |         |....  |      ...    |..|  | .  ..|.||...|.|....|.||   |. .||.........|..|.  | |  ..|...| |.   | ..|. |     ......|.|.|..|        ..|| ..|  |..|.| ...|.| .. |   .|.  .| |.. |   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                \n",
            "MATGAENQWLKGRVKAVTSGDCLVITALSHNRAGPPPEKTITFSSLMAPKMARRGGI-DEPFA--WES---KEFL----R-KLCIG-KEVAFKVDYKVEAIAGREFG---------SVFLGNENLAKLVVKTGWAKVR---EP-G--QQNQ-DKVSP--------YIKEL-LQLEELAKQ-EG-YG--RWSK-VPGAAEASIRNLP-PSAIG--DSAGFDAMGLLAANKGKPMEGIVEQ---VR-DGSTIRVYLLPEFQFVQVFVAGVQAPSMGRRTTNGSVVETVPDEPNGDVSAESRGPLTTAQRLAASAASSVEVSSDPFATEAKYFTEHRVLSRDVRIVLEGVDKFNNLIGSVHYSDGETVKDLGLELVENGLAKFVEWSANMMEEEAK---KKLKAAELQCKKDKVKMWANYVP-PATNSKAIHD-QNFTGK-VVEVVSGDCLIVADDAVPFGSPAAERRVCLSSIRSPKMGNPRREEKPAPYAREAREFLR-QRLIGKQVIVQMEYSRKVTQGDGPTTSGAADRFMDFGSVFLPSAAKADSDEVTAPPAAAIAGSQPVGVNIAELVLVRGFGNVVRHRDFEERSNHYDALLAAEARALAGKKGIHSAKESPAMHITDLTVSAAKKAKDFLPSLQRIRRIPAVVEYVLSGHRFKLYIPKITCSIAFSFSGVRCPGRGEPYSEEAISVMRRRIMQRDVEIEVETVDRTGTFLGSMWESRTNVATVLLEAGLAKMQTSFGADRIAEAHLLEQAERSAKNQKLKIWENYVEGEEVSNGNTNTVETRQKETLKVVVTEVLGGGRFYVQSAGDQKIASIQNQLASLSIKDAPIIGSFNPKRGDIVLAQFSLDNSWNRAMIVTAPRAAVQSPDEKFEVFYIDYGNQETVPYSAIRPIDPSVSAAPGLAQLCRLAYIKVPSLEDDFGPEAGEYLHTVTLGSGKEFKAVIEERDTSGGKVKGQGTGTEFVVTLIAVDDEISVNAAMLQEGIARMEKRQKWGHKGKQAALDALEKFQEEARKSRIGIWQYGDIESDDEDTGPARKPAGGRR\n",
            "  Score=295\n",
            "\n",
            "Subject_Id : seq267\n",
            "\n",
            "-------------------------------------------------------GIKEEAAAIKWEAAAIKEGASPEPQPK---GIKEEAAAI--KWEAAAIKE-GASPEPQPKPS---GDPQ-SK---QTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGPSGDPQ-SKQTP---EASPRPQPEPS--GKPQS---EQTP--KASP-EPQPGIKEEAAAIKWEAAAIKEGASPE---PQPKPSG-D-PQ-SKQTPEASP-RPQP-EPSGKPQSEQT-P-----K-----ASP-EPQPKP-SGQAK---E---------IQWQA--K------EIQ----EQAK--G-K--PSGQAKEIQWQAKEIQEQAKGGIKE-EAAAIKWEAAAIKEGAS--PEP--QPKPSGDPQS---KQTPEA-S-----PRPQPEPSGKPQSE--------QTPK-ASP--EPQPKP-SGQAKE-IQWQ---AKE--IQ-EQA-K---G------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                       || .|..|  ||.   ||..    . |   | ||.|...  |.||.|..| |         |   |... .|   .|..|..|   || |  |..| .|.||        ..||. .|..|...| .| .|  . || .|   |||.|..| ||  |  .|   ....  .|.. .|..||.|.   .. ....|.....||   .|....| . |. ...|...|. ...| ||.|....|.. |     .     ||. |....| ...||   |         |....  |      ...    |..|  | .  ..|.||...|.|....|.||   |. .||.........|..|.  | |  ..|...| |.   | ..|. |     ......|.|.|..|        ..|| ..|  |..|.| ...|.| .. |   .|.  .| |.. |   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                \n",
            "MATGAENQWLKGRVKAVTSGDCLVITALSHNRAGPPPEKTITFSSLMAPKMARRGGI-DEPFA--WES---KEFL----R-KLCIG-KEVAFKVDYKVEAIAGREFG---------SVFLGNENLAKLVVKTGWAKVR---EP-G--QQNQ-DKVSP--------YIKEL-LQLEELAKQ-EG-YG--RWSK-VPGAAEASIRNLP-PSAIG--DSAGFDAMGLLAANKGKPMEGIVEQ---VR-DGSTIRVYLLPEFQFVQVFVAGVQAPSMGRRTTNGSVVETVPDEPNGDVSAESRGPLTTAQRLAASAASSVEVSSDPFATEAKYFTEHRVLSRDVRIVLEGVDKFNNLIGSVHYSDGETVKDLGLELVENGLAKFVEWSANMMEEEAK---KKLKAAELQCKKDKVKMWANYVP-PATNSKAIHD-QNFTGK-VVEVVSGDCLIVADDAVPFGSPAAERRVCLSSIRSPKMGNPRREEKPAPYAREAREFLR-QRLIGKQVIVQMEYSRKVTQGDGPTTSGAADRFMDFGSVFLPSAAKADSDEVTAPPAAAIAGSQPVGVNIAELVLVRGFGNVVRHRDFEERSNHYDALLAAEARALAGKKGIHSAKESPAMHITDLTVSAAKKAKDFLPSLQRIRRIPAVVEYVLSGHRFKLYIPKITCSIAFSFSGVRCPGRGEPYSEEAISVMRRRIMQRDVEIEVETVDRTGTFLGSMWESRTNVATVLLEAGLAKMQTSFGADRIAEAHLLEQAERSAKNQKLKIWENYVEGEEVSNGNTNTVETRQKETLKVVVTEVLGGGRFYVQSAGDQKIASIQNQLASLSIKDAPIIGSFNPKRGDIVLAQFSLDNSWNRAMIVTAPRAAVQSPDEKFEVFYIDYGNQETVPYSAIRPIDPSVSAAPGLAQLCRLAYIKVPSLEDDFGPEAGEYLHTVTLGSGKEFKAVIEERDTSGGKVKGQGTGTEFVVTLIAVDDEISVNAAMLQEGIARMEKRQKWGHKGKQAALDALEKFQEEARKSRIGIWQYGDIESDDEDTGPARKPAGGRR\n",
            "  Score=295\n",
            "\n",
            "Subject_Id : seq1015\n",
            "\n",
            "----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GIKEEAAA--IKWEAA----AIK---EGASPEPQPK-GIKEEA---AAIKWE--AAAIK-EGASPE--------PQPKPSGDPQSK-Q-TPE------------ASPRPQP-----EPSG-K-PQ-SEQ----TPKA---SPEP---QPKP--SGQ-AK--E-IQ-----WQAKEIQ--EQAKGPSGDPQSKQ-TPEASPRPQPEPSGKPQ-SEQ-TPKASPEPQPGIKEEAAAI-KWEAAAIKEGASPEPQPKPSGDPQSKQTPE-ASPRPQP-EPS-GKPQSEQTPKASPEPQPKPS-GQAKEIQWQAKEIQEQAKG--KPSGQ--AKEIQWQAKEI--Q---EQAKGG--I----KEEA---AAIKW---EAAAIKEGASPEPQ---PK-PSGDPQSKQTPEASPRPQPEPSGKPQSE-QTPKASP-E---PQPKPS---GQAK--EI--QWQ-AKEIQEQAKG-----------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |  .|.|.  | ..||    ...   .|      |. |   .|   ||  |.  ...|. | ..||        ....  |||... | .||            ||.|        | .| . .. ||.    | ..   .|.|   .|.|  ||. |.  | ||     |......  |...|||   .||. |..|   |||   .||. .|. |..   |..  ..|  |.. |...|..  |   | ...| |...| .|.. |....|| |.. |.....||  .| |    .. |.|  ...|.  ..|.|.|  .|. |  ..|....|  |  |   |.|.||  .    .|||   |....   |||  .|||... |   .. |.| ....|||.||       |    .. |||..|| .   ||..||   |...  |.  ..| |.|  .||.|                                                                                                                                               \n",
            "MTSTGQDSTTTRQRRSRQNPQSPPQDSSVTSKRNIKKGAVPRSIPNLAEVKKKGKMKKLGQAMEEDLIVGLQGMDLNLEAEALAGTGLVLDEQLNEFHCLWDDSFPEGPERLHAIKEQLIQEGLLDRCVSFQARFAEKEELMLVHSLEYIDLMETTQYMNEGELRVLADTYDSVYLHPNSYSCACLASGSVLRLVDAVLGAEIRNGMAIIRPPGHHAQHSLMDGYCMFNHVAVAARYAQQKHRIRRVLIVDWDVHHGQGTQFTFDQDPSVLYFSIHRYEQGRFWPHLKASNWSTTGFGQGQGYTINVPWNQVGMRDADYIAAFLHVLLPVALEFQPQLVLVAAGFDALQGDPKGEMAATPAGFAQLTHLLMGLAGGKLILSLEGGYNLRALAEGVSASLHTLLGDPCPMLESPGAPCRSAQASVSCALEALEPFWEVLVRSTETVERDNMEEDNVEESEEEGPWEPPVLPILTWPVLQSRTGLVYDQNMMNHCNLWDSHHPEVPQRILRIMCRLEELGLAGRCLTLTPRPATEAELLTCHSAEYVGHLRATEKMKTRELHRESSNFDSIYICPSTFACAQLATGAACRLVEAVLSGEVLNGAAVVRPPGHHAEQDAACGFCFFNSVAVAARHAQTISGHALRILIVDWDVHHGNGTQHMFEDDPSVLYVSLHRYDHGTFFPMG--DEGASSQI-GRAAGTGFTVNVAWNG------PRMG---DADYLAA--WHRLVLPIAYE-FNPELVLVSAGFDAAR--GDPLGGCQVSPEGYAHLTHLLMGLASGR---IILILE-GGYNLTSISESMAACT-RSLLGDPPPLLTLPRPPLSGALASITETIQVHRRYWRSLRVMKVEDREGPS---SSKLVTKKA---PQP---AKPRLAERMTTR---EKK--VLE--AGMGKVTSASF--G---E-ESTP-GQTNS-ETAVVALTQDQPSEAATGGATLAQT--IS-E----AAIGGA--MLGQT--TSEEAVGGATPD-QTTSEETVGGA--ILDQTTSEDAVGGATLGQTTSEEAVGGATLAQTTSEAA--MEGATLD-QTTSEEAPGG-TELIQTPLAS-------S----TDHQTPPTSPVQGTTPQISPSTLIGSLRTLELGSESQGASE--SQAPGEENLLGEAAGGQDMADSMLMQGSRGLTDQAIFYAVTPLPWCPHLVAVCPIPAAGLDVTQPCGDCGTIQENWVCLSCYQVYCGRYINGHMLQHHGNSGHPLVLSYIDLSAWCYYCQAYVHHQALLDVKNIAHQNKFGEDMPHPH\n",
            "  Score=292\n",
            "\n",
            "Subject_Id : seq1015\n",
            "\n",
            "----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GIKEEAAA--IKWEAA----AIK---EGASPEPQPK-GIKEEA---AAIKWE--AAAIK-EGASPE--------PQPKPSGDPQSK-Q-TPE------------ASPRPQP-----EPSG-K-PQ-SEQ----TPKA---SPEP---QPKP--SGQ-AK--E-IQ-----WQAKEIQ--EQAKGPSGDPQSKQ-TPEASPRPQPEPSGKPQ-SEQ-TPKASPEPQPGIKEEAAAI-KWEAAAIKEGASPEPQPKPSGDPQSKQTPE-ASPRPQP-EPS-GKPQSEQTPKASPEPQPKPS-GQAKEIQWQAKEIQEQAKG--KPSGQ--AKEIQWQAKEI--Q---EQAKGG--I----KEEA---AAIKW---EAAAIKEGASPEPQ---PK-PSGDPQSKQTPEASPRPQPEPSGKPQSE-QTPKASP-E---PQPKPS---GQAK--EI--QWQ-AKEIQEQAKG-----------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |  .|.|.  | ..||    ...   .|      |. |   .|   ||  |.  ...|. | ..||        ....  |||... | .||            ||.|        | .| . .. ||.    | ..   .|.|   .|.|  ||. |.  | ||     |......  |...|||   .||. |..|   |||   .||. .|. |..   |..  ..|  |.. |...|..  |   | ...| |...| .|.. |....|| |.. |.....||  .| |    .. |.|  ...|.  ..|.|.|  .|. |  ..|....|  |  |   |.|.||  .    .|||   |....   |||  .|||... |   .. |.| ....|||.||       |    .. |||..|| .   ||..||   |...  |.  ..| |.|  .||.|                                                                                                                                               \n",
            "MTSTGQDSTTTRQRRSRQNPQSPPQDSSVTSKRNIKKGAVPRSIPNLAEVKKKGKMKKLGQAMEEDLIVGLQGMDLNLEAEALAGTGLVLDEQLNEFHCLWDDSFPEGPERLHAIKEQLIQEGLLDRCVSFQARFAEKEELMLVHSLEYIDLMETTQYMNEGELRVLADTYDSVYLHPNSYSCACLASGSVLRLVDAVLGAEIRNGMAIIRPPGHHAQHSLMDGYCMFNHVAVAARYAQQKHRIRRVLIVDWDVHHGQGTQFTFDQDPSVLYFSIHRYEQGRFWPHLKASNWSTTGFGQGQGYTINVPWNQVGMRDADYIAAFLHVLLPVALEFQPQLVLVAAGFDALQGDPKGEMAATPAGFAQLTHLLMGLAGGKLILSLEGGYNLRALAEGVSASLHTLLGDPCPMLESPGAPCRSAQASVSCALEALEPFWEVLVRSTETVERDNMEEDNVEESEEEGPWEPPVLPILTWPVLQSRTGLVYDQNMMNHCNLWDSHHPEVPQRILRIMCRLEELGLAGRCLTLTPRPATEAELLTCHSAEYVGHLRATEKMKTRELHRESSNFDSIYICPSTFACAQLATGAACRLVEAVLSGEVLNGAAVVRPPGHHAEQDAACGFCFFNSVAVAARHAQTISGHALRILIVDWDVHHGNGTQHMFEDDPSVLYVSLHRYDHGTFFPMG--DEGASSQI-GRAAGTGFTVNVAWNG------PRMG---DADYLAA--WHRLVLPIAYE-FNPELVLVSAGFDAAR--GDPLGGCQVSPEGYAHLTHLLMGLASGR---IILILE-GGYNLTSISESMAACT-RSLLGDPPPLLTLPRPPLSGALASITETIQVHRRYWRSLRVMKVEDREGPS---SSKLVTKKA---PQP---AKPRLAERMTTR---EKK--VLE--AGMGKVTSASF--G---E-ESTP-GQTNS-ETAVVALTQDQPSEAATGGATLAQT--IS-E----AAIGGA--MLGQT--TSEEAVGGATPD-QTTSEETVGGA--ILDQTTSEDAVGGATLGQTTSEEAVGGATLAQTTSEAA--MEGATLD-QTTSEEAPGG-TELIQTPLAS-------S----TDHQTPPTSPVQGTTPQISPSTLIGSLRTLELGSESQGASE--SQAPGEENLLGEAAGGQDMADSMLMQGSRGLTDQAIFYAVTPLPWCPHLVAVCPIPAAGLDVTQPCGDCGTIQENWVCLSCYQVYCGRYINGHMLQHHGNSGHPLVLSYIDLSAWCYYCQAYVHHQALLDVKNIAHQNKFGEDMPHPH\n",
            "  Score=292\n",
            "\n",
            "Subject_Id : seq1015\n",
            "\n",
            "----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GIKEEAAA--IKWEAA----AIK---EGASPEPQPK-GIKEEA---AAIKWE--AAAIK-EGASPE--------PQPKPSGDPQSK-Q-TPE------------ASPRPQP-----EPSG-K-PQ-SEQ----TPKA---SPEP---QPKP--SGQ-AK--E-IQ-----WQAKEIQ--EQAKGPSGDPQSKQ-TPEASPRPQPEPSGKPQ-SEQ-TPKASPEPQPGIKEEAAAI-KWEAAAIKEGASPEPQPKPSGDPQSKQTPE-ASPRPQP-EPS-GKPQSEQTPKASPEPQPKPS-GQAKEIQWQAKEIQEQAKG--KPSGQ--AKEIQWQAKEI--Q---EQAKGG--I----KEEA---AAIKW---EAAAIKEGASPEPQ---PK-PSGDPQSKQTPEASPRPQPEPSGKPQSE-QTPKASP-E---PQPKPS---GQAK--EI--QWQ-AKEIQEQAKG-----------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |  .|.|.  | ..||    ...   .|      |. |   .|   ||  |.  ...|. | ..||        ....  |||... | .||            ||.|        | .| . .. ||.    | ..   .|.|   .|.|  ||. |.  | ||     |......  |...|||   .||. |..|   |||   .||. .|. |..   |..  ..|  |.. |...|..  |   | ...| |...| .|.. |....|| |.. |.....||  .| |    .. |.|  ...|.  ..|.|.|  .|. |  ..|....|  |  |   |.|.||  .    .|||   |....   |||  .|||... |   .. |.| ....|||.||       |    .. |||..|| .   ||..||   |...  |.  ..| |.|  .||.|                                                                                                                                               \n",
            "MTSTGQDSTTTRQRRSRQNPQSPPQDSSVTSKRNIKKGAVPRSIPNLAEVKKKGKMKKLGQAMEEDLIVGLQGMDLNLEAEALAGTGLVLDEQLNEFHCLWDDSFPEGPERLHAIKEQLIQEGLLDRCVSFQARFAEKEELMLVHSLEYIDLMETTQYMNEGELRVLADTYDSVYLHPNSYSCACLASGSVLRLVDAVLGAEIRNGMAIIRPPGHHAQHSLMDGYCMFNHVAVAARYAQQKHRIRRVLIVDWDVHHGQGTQFTFDQDPSVLYFSIHRYEQGRFWPHLKASNWSTTGFGQGQGYTINVPWNQVGMRDADYIAAFLHVLLPVALEFQPQLVLVAAGFDALQGDPKGEMAATPAGFAQLTHLLMGLAGGKLILSLEGGYNLRALAEGVSASLHTLLGDPCPMLESPGAPCRSAQASVSCALEALEPFWEVLVRSTETVERDNMEEDNVEESEEEGPWEPPVLPILTWPVLQSRTGLVYDQNMMNHCNLWDSHHPEVPQRILRIMCRLEELGLAGRCLTLTPRPATEAELLTCHSAEYVGHLRATEKMKTRELHRESSNFDSIYICPSTFACAQLATGAACRLVEAVLSGEVLNGAAVVRPPGHHAEQDAACGFCFFNSVAVAARHAQTISGHALRILIVDWDVHHGNGTQHMFEDDPSVLYVSLHRYDHGTFFPMG--DEGASSQI-GRAAGTGFTVNVAWNG------PRMG---DADYLAA--WHRLVLPIAYE-FNPELVLVSAGFDAAR--GDPLGGCQVSPEGYAHLTHLLMGLASGR---IILILE-GGYNLTSISESMAACT-RSLLGDPPPLLTLPRPPLSGALASITETIQVHRRYWRSLRVMKVEDREGPS---SSKLVTKKA---PQP---AKPRLAERMTTR---EKK--VLE--AGMGKVTSASF--G---E-ESTP-GQTNS-ETAVVALTQDQPSEAATGGATLAQT--IS-E----AAIGGA--MLGQT--TSEEAVGGATPD-QTTSEETVGGA--ILDQTTSEDAVGGATLGQTTSEEAVGGATLAQTTSEAA--MEGATLD-QTTSEEAPGG-TELIQTPLAS-------S----TDHQTPPTSPVQGTTPQISPSTLIGSLRTLELGSESQGASE--SQAPGEENLLGEAAGGQDMADSMLMQGSRGLTDQAIFYAVTPLPWCPHLVAVCPIPAAGLDVTQPCGDCGTIQENWVCLSCYQVYCGRYINGHMLQHHGNSGHPLVLSYIDLSAWCYYCQAYVHHQALLDVKNIAHQNKFGEDMPHPH\n",
            "  Score=292\n",
            "\n",
            "Subject_Id : seq1015\n",
            "\n",
            "----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------GIKEEAAA--IKWEAA----AIK---EGASPEPQPK-GIKEEA---AAIKWE--AAAIK-EGASPE--------PQPKPSGDPQSK-Q-TPE------------ASPRPQP-----EPSG-K-PQ-SEQ----TPKA---SPEP---QPKP--SGQ-AK--E-IQ-----WQAKEIQ--EQAKGPSGDPQSKQ-TPEASPRPQPEPSGKPQ-SEQ-TPKASPEPQPGIKEEAAAI-KWEAAAIKEGASPEPQPKPSGDPQSKQTPE-ASPRPQP-EPS-GKPQSEQTPKASPEPQPKPS-GQAKEIQWQAKEIQEQAKG--KPSGQ--AKEIQWQAKEI--Q---EQAKGG--I----KEEA---AAIKW---EAAAIKEGASPEPQ---PK-PSGDPQSKQTPEASPRPQPEPSGKPQSE-QTPKASP-E---PQPKPS---GQAK--EI--QWQ-AKEIQEQAKG-----------------------------------------------------------------------------------------------------------------------------------------------\n",
            "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |  .|.|.  | ..||    ...   .|      |. |   .|   ||  |.  ...|. | ..||        ....  |||... | .||            ||.|        | .| . .. ||.    | ..   .|.|   .|.|  ||. |.  | ||     |......  |...|||   .||. |..|   |||   .||. .|. |..   |..  ..|  |.. |...|..  |   | ...| |...| .|.. |....|| |.. |.....||  .| |    .. |.|  ...|.  ..|.|.|  .|. |  ..|....|  |  |   |.|.||  .    .|||   |....   |||  .|||... |   .. |.| ....|||.||       |    .. |||..|| .   ||..||   |...  |.  ..| |.|  .||.|                                                                                                                                               \n",
            "MTSTGQDSTTTRQRRSRQNPQSPPQDSSVTSKRNIKKGAVPRSIPNLAEVKKKGKMKKLGQAMEEDLIVGLQGMDLNLEAEALAGTGLVLDEQLNEFHCLWDDSFPEGPERLHAIKEQLIQEGLLDRCVSFQARFAEKEELMLVHSLEYIDLMETTQYMNEGELRVLADTYDSVYLHPNSYSCACLASGSVLRLVDAVLGAEIRNGMAIIRPPGHHAQHSLMDGYCMFNHVAVAARYAQQKHRIRRVLIVDWDVHHGQGTQFTFDQDPSVLYFSIHRYEQGRFWPHLKASNWSTTGFGQGQGYTINVPWNQVGMRDADYIAAFLHVLLPVALEFQPQLVLVAAGFDALQGDPKGEMAATPAGFAQLTHLLMGLAGGKLILSLEGGYNLRALAEGVSASLHTLLGDPCPMLESPGAPCRSAQASVSCALEALEPFWEVLVRSTETVERDNMEEDNVEESEEEGPWEPPVLPILTWPVLQSRTGLVYDQNMMNHCNLWDSHHPEVPQRILRIMCRLEELGLAGRCLTLTPRPATEAELLTCHSAEYVGHLRATEKMKTRELHRESSNFDSIYICPSTFACAQLATGAACRLVEAVLSGEVLNGAAVVRPPGHHAEQDAACGFCFFNSVAVAARHAQTISGHALRILIVDWDVHHGNGTQHMFEDDPSVLYVSLHRYDHGTFFPMG--DEGASSQI-GRAAGTGFTVNVAWNG------PRMG---DADYLAA--WHRLVLPIAYE-FNPELVLVSAGFDAAR--GDPLGGCQVSPEGYAHLTHLLMGLASGR---IILILE-GGYNLTSISESMAACT-RSLLGDPPPLLTLPRPPLSGALASITETIQVHRRYWRSLRVMKVEDREGPS---SSKLVTKKA---PQP---AKPRLAERMTTR---EKK--VLE--AGMGKVTSASF--G---E-ESTP-GQTNS-ETAVVALTQDQPSEAATGGATLAQT--IS-E----AAIGGA--MLGQT--TSEEAVGGATPD-QTTSEETVGGA--ILDQTTSEDAVGGATLGQTTSEEAVGGATLAQTTSEAA--MEGATLD-QTTSEEAPGG-TELIQTPLAS-------S----TDHQTPPTSPVQGTTPQISPSTLIGSLRTLELGSESQGASE--SQAPGEENLLGEAAGGQDMADSMLMQGSRGLTDQAIFYAVTPLPWCPHLVAVCPIPAAGLDVTQPCGDCGTIQENWVCLSCYQVYCGRYINGHMLQHHGNSGHPLVLSYIDLSAWCYYCQAYVHHQALLDVKNIAHQNKFGEDMPHPH\n",
            "  Score=292\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title ESM 3D structure Prediction\n",
        "\n",
        "import streamlit as st\n",
        "from stmol import showmol\n",
        "import py3Dmol\n",
        "import requests\n",
        "import biotite.structure.io as bsio\n",
        "import certifi\n",
        "!sudo apt-get install ca-certificates\n",
        "!sudo update-ca-certificates\n",
        "\n",
        "\n",
        "if os.path.exists(\"predicted.pdb\"):\n",
        "    !rm -rf \"predicted.pdb\"\n",
        "\n",
        "# st.set_page_config(layout = 'wide')\n",
        "st.sidebar.title(' ESMFold')\n",
        "st.sidebar.write('[*ESMFold*](https://esmatlas.com/about) is an end-to-end single sequence protein structure predictor based on the ESM-2 language model. For more information, read the [research article](https://www.biorxiv.org/content/10.1101/2022.07.20.500902v2) and the [news article](https://www.nature.com/articles/d41586-022-03539-1) published in *Nature*.')\n",
        "\n",
        "# stmol\n",
        "def render_mol(pdb):\n",
        "    pdbview = py3Dmol.view()\n",
        "    pdbview.addModel(pdb, 'pdb')\n",
        "    pdbview.setStyle({'cartoon': {'color': 'spectrum'}})\n",
        "    pdbview.setBackgroundColor('white')  # ('0xeeeeee')\n",
        "    pdbview.zoomTo()\n",
        "    pdbview.zoom(2, 800)\n",
        "    pdbview.spin(True)\n",
        "    showmol(pdbview, height=500, width=800)\n",
        "\n",
        "# Protein sequence inpu\n",
        "DEFAULT_SEQ = \"GIKEEAAAIKWEAAAIKEGASPEPQPKGIKEEAAAIKWEAAAIKEGASPEPQPKPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPKASPEPQPGIKEEAAAIKWEAAAIKEGASPEPQPKPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKGKPSGQAKEIQWQAKEIQEQAKGGIKEEAAAIKWEAAAIKEGASPEPQPKPSGDPQSKQTPEASPRPQPEPSGKPQSEQTPKASPEPQPKPSGQAKEIQWQAKEIQEQAKG\" #@param {type:\"string\"}\n",
        "show_mainchains = True #@param {type: \"boolean\"}\n",
        "show_sidechains = True #@param {type: \"boolean\"}\n",
        "Color_Style =  True #@param {type: \"boolean\"}\n",
        "txt = st.sidebar.text_area('Input sequence', DEFAULT_SEQ, height=275)\n",
        "\n",
        "def update(sequence=txt):\n",
        "    headers = {\n",
        "        'Content-Type': 'application/x-www-form-urlencoded',\n",
        "    }\n",
        "    response = requests.post('https://api.esmatlas.com/foldSequence/v1/pdb/', headers=headers, data=sequence, verify=False)\n",
        "\n",
        "    if response.status_code != 200:\n",
        "        st.error(f\"Failed to get prediction: {response.status_code}\")\n",
        "        return\n",
        "\n",
        "    pdb_string = response.content.decode('utf-8')\n",
        "\n",
        "    if not pdb_string or \"ATOM\" not in pdb_string:\n",
        "        st.error(\"Received an invalid PDB file.\")\n",
        "        return\n",
        "\n",
        "    with open('predicted.pdb', 'w') as f:\n",
        "        f.write(pdb_string)\n",
        "\n",
        "    try:\n",
        "        struct = bsio.load_structure('predicted.pdb', extra_fields=[\"b_factor\"])\n",
        "        b_value = round(struct.b_factor.mean(), 4)\n",
        "    except Exception as e:\n",
        "        st.error(f\"Error loading structure: {e}\")\n",
        "        return\n",
        "\n",
        "    # Display protein structure\n",
        "    st.subheader('Visualization of predicted protein structure')\n",
        "    render_mol(pdb_string)\n",
        "\n",
        "    # plDDT value is stored in the B-factor field\n",
        "    st.subheader('plDDT')\n",
        "    st.write('plDDT is a per-residue estimate of the confidence in prediction on a scale from 0-100.')\n",
        "    st.info(f'plDDT: {b_value}')\n",
        "\n",
        "    st.download_button(\n",
        "        label=\"Download PDB\",\n",
        "        data=pdb_string,\n",
        "        file_name='predicted.pdb',\n",
        "        mime='text/plain',\n",
        "    )\n",
        "\n",
        "predict = st.sidebar.button('Predict', on_click=update)\n",
        "\n",
        "if not predict:\n",
        "    st.warning(' Enter protein sequence data!')\n",
        "\n",
        "txt = st.sidebar.text_area('Input sequence', DEFAULT_SEQ, height=275)\n",
        "\n",
        "\n",
        "\n",
        "update(txt)\n",
        "\n",
        "\n",
        "with open(\"predicted.pdb\") as ifile:\n",
        "    system = \"\".join([x for x in ifile])\n",
        "\n",
        "view = py3Dmol.view(width=1000, height=800)\n",
        "view.addModelsAsFrames(system)\n",
        "if Color_Style:\n",
        "  view.setStyle({'model': -1}, {\"cartoon\": {'color': 'spectrum'}})\n",
        "if show_sidechains:\n",
        "  BB = ['C','O','N']\n",
        "  view.addStyle({'and':[{'resn':[\"GLY\",\"PRO\"],'invert':True},{'atom':BB,'invert':True}]},\n",
        "                        {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "  view.addStyle({'and':[{'resn':\"GLY\"},{'atom':'CA'}]},\n",
        "                        {'sphere':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "  view.addStyle({'and':[{'resn':\"PRO\"},{'atom':['C','O'],'invert':True}]},\n",
        "                        {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "\n",
        "if show_mainchains:\n",
        "  BB = ['C','O','N','CA']\n",
        "  view.addStyle({'atom':BB},{'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "view.zoomTo()\n",
        "view.show()\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "cellView": "form",
        "id": "hSPz55Uyc2wy",
        "outputId": "194d849e-b97e-437f-b04d-6f443a017340"
      },
      "execution_count": 40,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "ca-certificates is already the newest version (20230311ubuntu0.22.04.1).\n",
            "0 upgraded, 0 newly installed, 0 to remove and 51 not upgraded.\n",
            "Updating certificates in /etc/ssl/certs...\n",
            "0 added, 0 removed; done.\n",
            "Running hooks in /etc/ca-certificates/update.d...\n",
            "\n",
            "done.\n",
            "done.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning:\n",
            "\n",
            "Unverified HTTPS request is being made to host 'api.esmatlas.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_1718032850302779\"  style=\"position: relative; width: 1000px; height: 800px;\">\n        <p id=\"3dmolwarning_1718032850302779\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.1.0/3Dmol-min.js');\n}\n\nvar viewer_1718032850302779 = null;\nvar warn = document.getElementById(\"3dmolwarning_1718032850302779\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_1718032850302779 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1718032850302779\"),{backgroundColor:\"white\"});\nviewer_1718032850302779.zoomTo();\n\tviewer_1718032850302779.addModelsAsFrames(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   GLY A   1      14.054   1.992 -28.510  1.00  0.29           N  \\nATOM      2  CA  GLY A   1      13.927   1.536 -27.135  1.00  0.28           C  \\nATOM      3  C   GLY A   1      12.545   1.768 -26.554  1.00  0.29           C  \\nATOM      4  O   GLY A   1      11.999   0.899 -25.872  1.00  0.28           O  \\nATOM      5  N   ILE A   2      11.969   2.902 -26.632  1.00  0.31           N  \\nATOM      6  CA  ILE A   2      10.578   3.333 -26.552  1.00  0.32           C  \\nATOM      7  C   ILE A   2       9.677   2.294 -27.216  1.00  0.30           C  \\nATOM      8  CB  ILE A   2      10.377   4.716 -27.210  1.00  0.28           C  \\nATOM      9  O   ILE A   2       8.593   1.991 -26.713  1.00  0.29           O  \\nATOM     10  CG1 ILE A   2      11.142   5.794 -26.433  1.00  0.27           C  \\nATOM     11  CG2 ILE A   2       8.887   5.060 -27.299  1.00  0.26           C  \\nATOM     12  CD1 ILE A   2      11.164   7.153 -27.118  1.00  0.24           C  \\nATOM     13  N   LYS A   3      10.221   1.587 -28.210  1.00  0.30           N  \\nATOM     14  CA  LYS A   3       9.438   0.680 -29.045  1.00  0.29           C  \\nATOM     15  C   LYS A   3       9.137  -0.623 -28.310  1.00  0.30           C  \\nATOM     16  CB  LYS A   3      10.173   0.386 -30.353  1.00  0.27           C  \\nATOM     17  O   LYS A   3       8.023  -1.143 -28.390  1.00  0.28           O  \\nATOM     18  CG  LYS A   3      10.138   1.532 -31.354  1.00  0.26           C  \\nATOM     19  CD  LYS A   3      10.744   1.125 -32.691  1.00  0.28           C  \\nATOM     20  CE  LYS A   3      10.696   2.266 -33.698  1.00  0.26           C  \\nATOM     21  NZ  LYS A   3      11.267   1.864 -35.019  1.00  0.26           N  \\nATOM     22  N   GLU A   4      10.053  -1.107 -27.586  1.00  0.33           N  \\nATOM     23  CA  GLU A   4       9.952  -2.354 -26.834  1.00  0.34           C  \\nATOM     24  C   GLU A   4       9.089  -2.178 -25.587  1.00  0.33           C  \\nATOM     25  CB  GLU A   4      11.342  -2.861 -26.444  1.00  0.31           C  \\nATOM     26  O   GLU A   4       8.254  -3.032 -25.278  1.00  0.31           O  \\nATOM     27  CG  GLU A   4      12.118  -3.479 -27.598  1.00  0.29           C  \\nATOM     28  CD  GLU A   4      13.625  -3.422 -27.407  1.00  0.33           C  \\nATOM     29  OE1 GLU A   4      14.085  -2.940 -26.347  1.00  0.42           O  \\nATOM     30  OE2 GLU A   4      14.352  -3.861 -28.325  1.00  0.40           O  \\nATOM     31  N   GLU A   5       9.097  -0.988 -25.017  1.00  0.31           N  \\nATOM     32  CA  GLU A   5       8.297  -0.529 -23.886  1.00  0.33           C  \\nATOM     33  C   GLU A   5       6.835  -0.347 -24.281  1.00  0.31           C  \\nATOM     34  CB  GLU A   5       8.858   0.782 -23.327  1.00  0.30           C  \\nATOM     35  O   GLU A   5       5.934  -0.811 -23.580  1.00  0.30           O  \\nATOM     36  CG  GLU A   5       9.309   0.689 -21.877  1.00  0.29           C  \\nATOM     37  CD  GLU A   5      10.044   1.930 -21.397  1.00  0.35           C  \\nATOM     38  OE1 GLU A   5      10.160   2.906 -22.173  1.00  0.43           O  \\nATOM     39  OE2 GLU A   5      10.510   1.927 -20.235  1.00  0.39           O  \\nATOM     40  N   ALA A   6       6.447   0.365 -25.332  1.00  0.32           N  \\nATOM     41  CA  ALA A   6       5.143   0.456 -25.985  1.00  0.31           C  \\nATOM     42  C   ALA A   6       4.627  -0.927 -26.372  1.00  0.32           C  \\nATOM     43  CB  ALA A   6       5.225   1.354 -27.216  1.00  0.29           C  \\nATOM     44  O   ALA A   6       3.442  -1.224 -26.205  1.00  0.30           O  \\nATOM     45  N   ALA A   7       5.546  -1.886 -26.764  1.00  0.30           N  \\nATOM     46  CA  ALA A   7       5.157  -3.235 -27.170  1.00  0.28           C  \\nATOM     47  C   ALA A   7       4.765  -4.079 -25.960  1.00  0.30           C  \\nATOM     48  CB  ALA A   7       6.291  -3.907 -27.939  1.00  0.27           C  \\nATOM     49  O   ALA A   7       3.751  -4.781 -25.988  1.00  0.28           O  \\nATOM     50  N   ALA A   8       5.551  -4.034 -24.890  1.00  0.32           N  \\nATOM     51  CA  ALA A   8       5.183  -4.760 -23.677  1.00  0.32           C  \\nATOM     52  C   ALA A   8       3.884  -4.218 -23.087  1.00  0.32           C  \\nATOM     53  CB  ALA A   8       6.307  -4.678 -22.647  1.00  0.30           C  \\nATOM     54  O   ALA A   8       3.001  -4.988 -22.702  1.00  0.30           O  \\nATOM     55  N   ILE A   9       3.607  -2.874 -23.042  1.00  0.33           N  \\nATOM     56  CA  ILE A   9       2.394  -2.149 -22.679  1.00  0.33           C  \\nATOM     57  C   ILE A   9       1.249  -2.565 -23.600  1.00  0.33           C  \\nATOM     58  CB  ILE A   9       2.608  -0.620 -22.748  1.00  0.31           C  \\nATOM     59  O   ILE A   9       0.134  -2.818 -23.138  1.00  0.31           O  \\nATOM     60  CG1 ILE A   9       3.600  -0.170 -21.669  1.00  0.29           C  \\nATOM     61  CG2 ILE A   9       1.274   0.119 -22.608  1.00  0.29           C  \\nATOM     62  CD1 ILE A   9       4.100   1.257 -21.844  1.00  0.26           C  \\nATOM     63  N   LYS A  10       1.492  -2.650 -24.954  1.00  0.29           N  \\nATOM     64  CA  LYS A  10       0.531  -3.092 -25.960  1.00  0.28           C  \\nATOM     65  C   LYS A  10       0.133  -4.548 -25.737  1.00  0.29           C  \\nATOM     66  CB  LYS A  10       1.106  -2.916 -27.366  1.00  0.27           C  \\nATOM     67  O   LYS A  10      -1.038  -4.906 -25.880  1.00  0.27           O  \\nATOM     68  CG  LYS A  10       0.688  -1.621 -28.048  1.00  0.26           C  \\nATOM     69  CD  LYS A  10       1.273  -1.513 -29.450  1.00  0.29           C  \\nATOM     70  CE  LYS A  10       0.875  -0.208 -30.124  1.00  0.23           C  \\nATOM     71  NZ  LYS A  10       1.456  -0.092 -31.495  1.00  0.25           N  \\nATOM     72  N   TRP A  11       1.075  -5.423 -25.370  1.00  0.15           N  \\nATOM     73  CA  TRP A  11       0.787  -6.835 -25.137  1.00  0.26           C  \\nATOM     74  C   TRP A  11      -0.048  -7.019 -23.875  1.00  0.20           C  \\nATOM     75  CB  TRP A  11       2.086  -7.638 -25.026  1.00  0.17           C  \\nATOM     76  O   TRP A  11      -1.051  -7.737 -23.886  1.00  0.19           O  \\nATOM     77  CG  TRP A  11       2.185  -8.774 -26.000  1.00  0.17           C  \\nATOM     78  CD1 TRP A  11       2.699  -8.731 -27.266  1.00  0.09           C  \\nATOM     79  CD2 TRP A  11       1.754 -10.121 -25.786  1.00  0.09           C  \\nATOM     80  CE2 TRP A  11       2.037 -10.843 -26.967  1.00  0.08           C  \\nATOM     81  CE3 TRP A  11       1.155 -10.789 -24.710  1.00  0.18           C  \\nATOM     82  NE1 TRP A  11       2.613  -9.973 -27.853  1.00  0.19           N  \\nATOM     83  CH2 TRP A  11       1.156 -12.831 -26.033  1.00  0.17           C  \\nATOM     84  CZ2 TRP A  11       1.742 -12.201 -27.101  1.00  0.17           C  \\nATOM     85  CZ3 TRP A  11       0.861 -12.141 -24.845  1.00  0.17           C  \\nATOM     86  N   GLU A  12       0.247  -6.368 -22.787  1.00  0.32           N  \\nATOM     87  CA  GLU A  12      -0.506  -6.482 -21.542  1.00  0.33           C  \\nATOM     88  C   GLU A  12      -1.869  -5.805 -21.656  1.00  0.32           C  \\nATOM     89  CB  GLU A  12       0.285  -5.881 -20.377  1.00  0.31           C  \\nATOM     90  O   GLU A  12      -2.880  -6.353 -21.213  1.00  0.31           O  \\nATOM     91  CG  GLU A  12       0.732  -6.904 -19.344  1.00  0.30           C  \\nATOM     92  CD  GLU A  12       1.533  -6.295 -18.204  1.00  0.38           C  \\nATOM     93  OE1 GLU A  12       1.792  -5.070 -18.230  1.00  0.44           O  \\nATOM     94  OE2 GLU A  12       1.905  -7.048 -17.277  1.00  0.40           O  \\nATOM     95  N   ALA A  13      -2.054  -4.669 -22.299  1.00  0.32           N  \\nATOM     96  CA  ALA A  13      -3.344  -4.077 -22.641  1.00  0.32           C  \\nATOM     97  C   ALA A  13      -4.130  -4.981 -23.587  1.00  0.32           C  \\nATOM     98  CB  ALA A  13      -3.148  -2.699 -23.269  1.00  0.30           C  \\nATOM     99  O   ALA A  13      -5.337  -5.166 -23.419  1.00  0.30           O  \\nATOM    100  N   ALA A  14      -3.513  -5.635 -24.659  1.00  0.29           N  \\nATOM    101  CA  ALA A  14      -4.132  -6.572 -25.592  1.00  0.27           C  \\nATOM    102  C   ALA A  14      -4.556  -7.855 -24.883  1.00  0.28           C  \\nATOM    103  CB  ALA A  14      -3.175  -6.893 -26.738  1.00  0.26           C  \\nATOM    104  O   ALA A  14      -5.668  -8.347 -25.087  1.00  0.27           O  \\nATOM    105  N   ALA A  15      -3.634  -8.495 -24.148  1.00  0.29           N  \\nATOM    106  CA  ALA A  15      -4.033  -9.686 -23.403  1.00  0.28           C  \\nATOM    107  C   ALA A  15      -5.204  -9.384 -22.472  1.00  0.29           C  \\nATOM    108  CB  ALA A  15      -2.852 -10.236 -22.606  1.00  0.26           C  \\nATOM    109  O   ALA A  15      -6.146 -10.173 -22.373  1.00  0.27           O  \\nATOM    110  N   ILE A  16      -5.333  -8.232 -21.963  1.00  0.32           N  \\nATOM    111  CA  ILE A  16      -6.384  -7.712 -21.095  1.00  0.32           C  \\nATOM    112  C   ILE A  16      -7.640  -7.432 -21.917  1.00  0.32           C  \\nATOM    113  CB  ILE A  16      -5.928  -6.433 -20.359  1.00  0.29           C  \\nATOM    114  O   ILE A  16      -8.746  -7.803 -21.518  1.00  0.30           O  \\nATOM    115  CG1 ILE A  16      -4.812  -6.760 -19.359  1.00  0.29           C  \\nATOM    116  CG2 ILE A  16      -7.112  -5.762 -19.657  1.00  0.28           C  \\nATOM    117  CD1 ILE A  16      -4.124  -5.534 -18.775  1.00  0.25           C  \\nATOM    118  N   LYS A  17      -7.509  -6.714 -23.002  1.00  0.29           N  \\nATOM    119  CA  LYS A  17      -8.629  -6.459 -23.903  1.00  0.29           C  \\nATOM    120  C   LYS A  17      -9.260  -7.765 -24.378  1.00  0.29           C  \\nATOM    121  CB  LYS A  17      -8.174  -5.629 -25.104  1.00  0.27           C  \\nATOM    122  O   LYS A  17     -10.472  -7.833 -24.592  1.00  0.27           O  \\nATOM    123  CG  LYS A  17      -8.875  -4.285 -25.230  1.00  0.26           C  \\nATOM    124  CD  LYS A  17      -8.381  -3.507 -26.442  1.00  0.30           C  \\nATOM    125  CE  LYS A  17      -9.110  -2.179 -26.592  1.00  0.28           C  \\nATOM    126  NZ  LYS A  17      -8.654  -1.429 -27.800  1.00  0.29           N  \\nATOM    127  N   GLU A  18      -8.310  -8.807 -24.381  1.00  0.26           N  \\nATOM    128  CA  GLU A  18      -8.756 -10.100 -24.889  1.00  0.26           C  \\nATOM    129  C   GLU A  18      -9.252 -10.997 -23.758  1.00  0.26           C  \\nATOM    130  CB  GLU A  18      -7.628 -10.795 -25.655  1.00  0.25           C  \\nATOM    131  O   GLU A  18      -9.805 -12.070 -24.007  1.00  0.25           O  \\nATOM    132  CG  GLU A  18      -7.448 -10.289 -27.079  1.00  0.25           C  \\nATOM    133  CD  GLU A  18      -6.359 -11.025 -27.845  1.00  0.31           C  \\nATOM    134  OE1 GLU A  18      -5.770 -11.982 -27.293  1.00  0.41           O  \\nATOM    135  OE2 GLU A  18      -6.093 -10.640 -29.005  1.00  0.34           O  \\nATOM    136  N   GLY A  19      -9.500 -10.337 -22.474  1.00  0.29           N  \\nATOM    137  CA  GLY A  19      -9.920 -11.183 -21.368  1.00  0.29           C  \\nATOM    138  C   GLY A  19      -8.879 -12.215 -20.977  1.00  0.29           C  \\nATOM    139  O   GLY A  19      -9.182 -13.170 -20.260  1.00  0.28           O  \\nATOM    140  N   ALA A  20      -7.678 -12.067 -21.631  1.00  0.26           N  \\nATOM    141  CA  ALA A  20      -6.557 -12.912 -21.231  1.00  0.26           C  \\nATOM    142  C   ALA A  20      -5.891 -12.377 -19.966  1.00  0.26           C  \\nATOM    143  CB  ALA A  20      -5.537 -13.015 -22.362  1.00  0.24           C  \\nATOM    144  O   ALA A  20      -5.494 -11.211 -19.911  1.00  0.25           O  \\nATOM    145  N   SER A  21      -6.316 -12.763 -18.670  1.00  0.31           N  \\nATOM    146  CA  SER A  21      -5.501 -12.766 -17.460  1.00  0.32           C  \\nATOM    147  C   SER A  21      -4.111 -13.333 -17.730  1.00  0.31           C  \\nATOM    148  CB  SER A  21      -6.183 -13.573 -16.355  1.00  0.29           C  \\nATOM    149  O   SER A  21      -3.978 -14.446 -18.242  1.00  0.30           O  \\nATOM    150  OG  SER A  21      -6.884 -12.719 -15.467  1.00  0.30           O  \\nATOM    151  N   PRO A  22      -3.102 -12.474 -18.019  1.00  0.31           N  \\nATOM    152  CA  PRO A  22      -1.757 -13.055 -18.016  1.00  0.31           C  \\nATOM    153  C   PRO A  22      -1.530 -14.011 -16.847  1.00  0.31           C  \\nATOM    154  CB  PRO A  22      -0.844 -11.831 -17.905  1.00  0.29           C  \\nATOM    155  O   PRO A  22      -2.027 -13.772 -15.744  1.00  0.30           O  \\nATOM    156  CG  PRO A  22      -1.758 -10.705 -17.541  1.00  0.28           C  \\nATOM    157  CD  PRO A  22      -3.177 -11.169 -17.711  1.00  0.32           C  \\nATOM    158  N   GLU A  23      -1.577 -15.397 -16.990  1.00  0.29           N  \\nATOM    159  CA  GLU A  23      -1.080 -16.419 -16.074  1.00  0.30           C  \\nATOM    160  C   GLU A  23      -0.081 -15.833 -15.081  1.00  0.29           C  \\nATOM    161  CB  GLU A  23      -0.436 -17.570 -16.851  1.00  0.27           C  \\nATOM    162  O   GLU A  23       0.835 -15.104 -15.471  1.00  0.28           O  \\nATOM    163  CG  GLU A  23      -1.137 -18.907 -16.665  1.00  0.27           C  \\nATOM    164  CD  GLU A  23      -0.475 -20.044 -17.426  1.00  0.32           C  \\nATOM    165  OE1 GLU A  23       0.538 -19.801 -18.120  1.00  0.44           O  \\nATOM    166  OE2 GLU A  23      -0.974 -21.188 -17.329  1.00  0.36           O  \\nATOM    167  N   PRO A  24      -0.412 -15.636 -13.713  1.00  0.31           N  \\nATOM    168  CA  PRO A  24       0.631 -15.268 -12.753  1.00  0.31           C  \\nATOM    169  C   PRO A  24       1.952 -15.989 -13.012  1.00  0.31           C  \\nATOM    170  CB  PRO A  24       0.035 -15.687 -11.406  1.00  0.29           C  \\nATOM    171  O   PRO A  24       1.987 -17.220 -13.070  1.00  0.30           O  \\nATOM    172  CG  PRO A  24      -1.156 -16.519 -11.758  1.00  0.28           C  \\nATOM    173  CD  PRO A  24      -1.420 -16.375 -13.230  1.00  0.32           C  \\nATOM    174  N   GLN A  25       2.851 -15.400 -13.681  1.00  0.30           N  \\nATOM    175  CA  GLN A  25       4.231 -15.857 -13.805  1.00  0.31           C  \\nATOM    176  C   GLN A  25       4.933 -15.861 -12.450  1.00  0.30           C  \\nATOM    177  CB  GLN A  25       5.001 -14.979 -14.793  1.00  0.28           C  \\nATOM    178  O   GLN A  25       4.614 -15.051 -11.577  1.00  0.29           O  \\nATOM    179  CG  GLN A  25       4.619 -15.210 -16.249  1.00  0.28           C  \\nATOM    180  CD  GLN A  25       5.826 -15.324 -17.161  1.00  0.34           C  \\nATOM    181  NE2 GLN A  25       5.632 -15.926 -18.329  1.00  0.32           N  \\nATOM    182  OE1 GLN A  25       6.924 -14.873 -16.819  1.00  0.44           O  \\nATOM    183  N   PRO A  26       5.617 -17.204 -11.914  1.00  0.29           N  \\nATOM    184  CA  PRO A  26       6.365 -17.533 -10.698  1.00  0.28           C  \\nATOM    185  C   PRO A  26       6.950 -16.300 -10.014  1.00  0.29           C  \\nATOM    186  CB  PRO A  26       7.476 -18.455 -11.206  1.00  0.27           C  \\nATOM    187  O   PRO A  26       7.461 -15.400 -10.687  1.00  0.28           O  \\nATOM    188  CG  PRO A  26       7.315 -18.466 -12.691  1.00  0.26           C  \\nATOM    189  CD  PRO A  26       6.102 -17.654 -13.046  1.00  0.30           C  \\nATOM    190  N   LYS A  27       6.378 -15.693  -8.938  1.00  0.29           N  \\nATOM    191  CA  LYS A  27       6.788 -14.831  -7.833  1.00  0.28           C  \\nATOM    192  C   LYS A  27       8.279 -14.516  -7.906  1.00  0.30           C  \\nATOM    193  CB  LYS A  27       6.454 -15.483  -6.490  1.00  0.27           C  \\nATOM    194  O   LYS A  27       9.111 -15.426  -7.925  1.00  0.29           O  \\nATOM    195  CG  LYS A  27       4.977 -15.437  -6.130  1.00  0.26           C  \\nATOM    196  CD  LYS A  27       4.726 -15.983  -4.730  1.00  0.32           C  \\nATOM    197  CE  LYS A  27       3.243 -15.979  -4.385  1.00  0.26           C  \\nATOM    198  NZ  LYS A  27       2.991 -16.523  -3.017  1.00  0.27           N  \\nATOM    199  N   GLY A  28       8.719 -13.633  -8.914  1.00  0.29           N  \\nATOM    200  CA  GLY A  28      10.032 -13.010  -8.964  1.00  0.29           C  \\nATOM    201  C   GLY A  28      10.279 -12.242 -10.249  1.00  0.29           C  \\nATOM    202  O   GLY A  28      10.881 -11.166 -10.229  1.00  0.27           O  \\nATOM    203  N   ILE A  29       9.711 -12.515 -11.326  1.00  0.27           N  \\nATOM    204  CA  ILE A  29      10.127 -11.856 -12.559  1.00  0.27           C  \\nATOM    205  C   ILE A  29       9.187 -10.691 -12.864  1.00  0.27           C  \\nATOM    206  CB  ILE A  29      10.157 -12.845 -13.746  1.00  0.24           C  \\nATOM    207  O   ILE A  29       9.633  -9.614 -13.265  1.00  0.25           O  \\nATOM    208  CG1 ILE A  29      11.240 -13.908 -13.528  1.00  0.15           C  \\nATOM    209  CG2 ILE A  29      10.381 -12.098 -15.065  1.00  0.21           C  \\nATOM    210  CD1 ILE A  29      11.308 -14.957 -14.629  1.00  0.19           C  \\nATOM    211  N   LYS A  30       7.975 -10.677 -12.424  1.00  0.31           N  \\nATOM    212  CA  LYS A  30       7.138  -9.537 -12.789  1.00  0.32           C  \\nATOM    213  C   LYS A  30       7.284  -8.402 -11.779  1.00  0.31           C  \\nATOM    214  CB  LYS A  30       5.672  -9.960 -12.895  1.00  0.28           C  \\nATOM    215  O   LYS A  30       7.231  -7.227 -12.147  1.00  0.29           O  \\nATOM    216  CG  LYS A  30       5.330 -10.692 -14.184  1.00  0.27           C  \\nATOM    217  CD  LYS A  30       3.834 -10.950 -14.301  1.00  0.31           C  \\nATOM    218  CE  LYS A  30       3.490 -11.682 -15.591  1.00  0.26           C  \\nATOM    219  NZ  LYS A  30       2.023 -11.935 -15.711  1.00  0.27           N  \\nATOM    220  N   GLU A  31       7.593  -8.709 -10.567  1.00  0.34           N  \\nATOM    221  CA  GLU A  31       7.854  -7.628  -9.621  1.00  0.35           C  \\nATOM    222  C   GLU A  31       9.167  -6.920  -9.941  1.00  0.34           C  \\nATOM    223  CB  GLU A  31       7.880  -8.163  -8.187  1.00  0.32           C  \\nATOM    224  O   GLU A  31       9.243  -5.691  -9.894  1.00  0.32           O  \\nATOM    225  CG  GLU A  31       6.512  -8.567  -7.657  1.00  0.30           C  \\nATOM    226  CD  GLU A  31       6.576  -9.293  -6.322  1.00  0.35           C  \\nATOM    227  OE1 GLU A  31       7.691  -9.469  -5.780  1.00  0.40           O  \\nATOM    228  OE2 GLU A  31       5.503  -9.689  -5.815  1.00  0.40           O  \\nATOM    229  N   GLU A  32      10.173  -7.705 -10.436  1.00  0.26           N  \\nATOM    230  CA  GLU A  32      11.448  -7.178 -10.912  1.00  0.27           C  \\nATOM    231  C   GLU A  32      11.274  -6.413 -12.221  1.00  0.26           C  \\nATOM    232  CB  GLU A  32      12.463  -8.310 -11.096  1.00  0.24           C  \\nATOM    233  O   GLU A  32      11.800  -5.308 -12.374  1.00  0.25           O  \\nATOM    234  CG  GLU A  32      13.490  -8.402  -9.976  1.00  0.23           C  \\nATOM    235  CD  GLU A  32      14.447  -9.573 -10.135  1.00  0.27           C  \\nATOM    236  OE1 GLU A  32      14.355 -10.298 -11.152  1.00  0.35           O  \\nATOM    237  OE2 GLU A  32      15.294  -9.767  -9.236  1.00  0.28           O  \\nATOM    238  N   ALA A  33      10.473  -6.945 -13.111  1.00  0.28           N  \\nATOM    239  CA  ALA A  33      10.237  -6.255 -14.376  1.00  0.28           C  \\nATOM    240  C   ALA A  33       9.358  -5.024 -14.174  1.00  0.28           C  \\nATOM    241  CB  ALA A  33       9.595  -7.203 -15.387  1.00  0.26           C  \\nATOM    242  O   ALA A  33       9.598  -3.976 -14.779  1.00  0.26           O  \\nATOM    243  N   ALA A  34       8.446  -5.077 -13.204  1.00  0.32           N  \\nATOM    244  CA  ALA A  34       7.665  -3.881 -12.898  1.00  0.32           C  \\nATOM    245  C   ALA A  34       8.522  -2.831 -12.197  1.00  0.32           C  \\nATOM    246  CB  ALA A  34       6.457  -4.240 -12.036  1.00  0.30           C  \\nATOM    247  O   ALA A  34       8.462  -1.646 -12.533  1.00  0.30           O  \\nATOM    248  N   ALA A  35       9.400  -3.264 -11.278  1.00  0.28           N  \\nATOM    249  CA  ALA A  35      10.353  -2.328 -10.686  1.00  0.28           C  \\nATOM    250  C   ALA A  35      11.302  -1.771 -11.743  1.00  0.28           C  \\nATOM    251  CB  ALA A  35      11.143  -3.008  -9.571  1.00  0.26           C  \\nATOM    252  O   ALA A  35      11.583  -0.570 -11.763  1.00  0.27           O  \\nATOM    253  N   ILE A  36      11.901  -2.567 -12.729  1.00  0.26           N  \\nATOM    254  CA  ILE A  36      12.829  -2.167 -13.781  1.00  0.25           C  \\nATOM    255  C   ILE A  36      12.127  -1.227 -14.759  1.00  0.26           C  \\nATOM    256  CB  ILE A  36      13.395  -3.394 -14.531  1.00  0.24           C  \\nATOM    257  O   ILE A  36      12.683  -0.198 -15.148  1.00  0.24           O  \\nATOM    258  CG1 ILE A  36      14.279  -4.229 -13.597  1.00  0.19           C  \\nATOM    259  CG2 ILE A  36      14.175  -2.954 -15.774  1.00  0.22           C  \\nATOM    260  CD1 ILE A  36      14.675  -5.584 -14.168  1.00  0.18           C  \\nATOM    261  N   LYS A  37      10.932  -1.542 -15.151  1.00  0.29           N  \\nATOM    262  CA  LYS A  37      10.252  -0.681 -16.115  1.00  0.29           C  \\nATOM    263  C   LYS A  37       9.972   0.695 -15.519  1.00  0.29           C  \\nATOM    264  CB  LYS A  37       8.947  -1.326 -16.583  1.00  0.27           C  \\nATOM    265  O   LYS A  37      10.032   1.706 -16.223  1.00  0.28           O  \\nATOM    266  CG  LYS A  37       9.104  -2.235 -17.793  1.00  0.26           C  \\nATOM    267  CD  LYS A  37       7.765  -2.807 -18.240  1.00  0.31           C  \\nATOM    268  CE  LYS A  37       7.925  -3.733 -19.439  1.00  0.24           C  \\nATOM    269  NZ  LYS A  37       6.618  -4.319 -19.864  1.00  0.27           N  \\nATOM    270  N   TRP A  38       9.957   0.852 -14.172  1.00  0.16           N  \\nATOM    271  CA  TRP A  38       9.678   2.096 -13.461  1.00  0.27           C  \\nATOM    272  C   TRP A  38      10.912   2.991 -13.426  1.00  0.18           C  \\nATOM    273  CB  TRP A  38       9.202   1.806 -12.035  1.00  0.19           C  \\nATOM    274  O   TRP A  38      10.819   4.198 -13.662  1.00  0.18           O  \\nATOM    275  CG  TRP A  38       7.717   1.634 -11.913  1.00  0.21           C  \\nATOM    276  CD1 TRP A  38       7.035   0.463 -11.733  1.00  0.20           C  \\nATOM    277  CD2 TRP A  38       6.731   2.669 -11.967  1.00  0.18           C  \\nATOM    278  CE2 TRP A  38       5.469   2.053 -11.811  1.00  0.12           C  \\nATOM    279  CE3 TRP A  38       6.791   4.059 -12.131  1.00  0.21           C  \\nATOM    280  NE1 TRP A  38       5.683   0.708 -11.670  1.00  0.21           N  \\nATOM    281  CH2 TRP A  38       4.364   4.139 -11.977  1.00  0.22           C  \\nATOM    282  CZ2 TRP A  38       4.277   2.780 -11.815  1.00  0.21           C  \\nATOM    283  CZ3 TRP A  38       5.603   4.782 -12.134  1.00  0.22           C  \\nATOM    284  N   GLU A  39      11.981   2.475 -13.177  1.00  0.25           N  \\nATOM    285  CA  GLU A  39      13.180   3.301 -13.280  1.00  0.26           C  \\nATOM    286  C   GLU A  39      13.283   3.957 -14.654  1.00  0.25           C  \\nATOM    287  CB  GLU A  39      14.433   2.466 -13.001  1.00  0.23           C  \\nATOM    288  O   GLU A  39      13.572   5.150 -14.757  1.00  0.24           O  \\nATOM    289  CG  GLU A  39      15.127   2.818 -11.693  1.00  0.23           C  \\nATOM    290  CD  GLU A  39      16.349   1.957 -11.411  1.00  0.27           C  \\nATOM    291  OE1 GLU A  39      16.652   1.051 -12.221  1.00  0.36           O  \\nATOM    292  OE2 GLU A  39      17.006   2.189 -10.372  1.00  0.31           O  \\nATOM    293  N   ALA A  40      12.822   3.281 -15.597  1.00  0.25           N  \\nATOM    294  CA  ALA A  40      13.002   3.815 -16.945  1.00  0.25           C  \\nATOM    295  C   ALA A  40      11.956   4.881 -17.257  1.00  0.25           C  \\nATOM    296  CB  ALA A  40      12.935   2.690 -17.976  1.00  0.23           C  \\nATOM    297  O   ALA A  40      12.272   5.915 -17.850  1.00  0.24           O  \\nATOM    298  N   ALA A  41      10.688   4.830 -16.731  1.00  0.29           N  \\nATOM    299  CA  ALA A  41       9.703   5.871 -17.016  1.00  0.28           C  \\nATOM    300  C   ALA A  41       9.908   7.083 -16.111  1.00  0.29           C  \\nATOM    301  CB  ALA A  41       8.288   5.323 -16.853  1.00  0.26           C  \\nATOM    302  O   ALA A  41       9.820   8.226 -16.565  1.00  0.27           O  \\nATOM    303  N   ALA A  42      10.249   6.911 -14.830  1.00  0.27           N  \\nATOM    304  CA  ALA A  42      10.556   8.054 -13.975  1.00  0.27           C  \\nATOM    305  C   ALA A  42      11.792   8.797 -14.474  1.00  0.27           C  \\nATOM    306  CB  ALA A  42      10.759   7.600 -12.531  1.00  0.25           C  \\nATOM    307  O   ALA A  42      11.832  10.029 -14.457  1.00  0.25           O  \\nATOM    308  N   ILE A  43      12.828   8.168 -14.976  1.00  0.25           N  \\nATOM    309  CA  ILE A  43      14.026   8.757 -15.564  1.00  0.24           C  \\nATOM    310  C   ILE A  43      13.660   9.505 -16.844  1.00  0.25           C  \\nATOM    311  CB  ILE A  43      15.098   7.683 -15.858  1.00  0.23           C  \\nATOM    312  O   ILE A  43      14.148  10.611 -17.085  1.00  0.23           O  \\nATOM    313  CG1 ILE A  43      15.629   7.086 -14.549  1.00  0.23           C  \\nATOM    314  CG2 ILE A  43      16.238   8.273 -16.693  1.00  0.22           C  \\nATOM    315  CD1 ILE A  43      16.509   5.859 -14.741  1.00  0.18           C  \\nATOM    316  N   LYS A  44      12.641   9.133 -17.538  1.00  0.26           N  \\nATOM    317  CA  LYS A  44      12.516   9.817 -18.822  1.00  0.27           C  \\nATOM    318  C   LYS A  44      11.576  11.015 -18.718  1.00  0.26           C  \\nATOM    319  CB  LYS A  44      12.018   8.852 -19.899  1.00  0.24           C  \\nATOM    320  O   LYS A  44      11.707  11.981 -19.472  1.00  0.25           O  \\nATOM    321  CG  LYS A  44      13.102   8.382 -20.858  1.00  0.24           C  \\nATOM    322  CD  LYS A  44      12.538   7.457 -21.929  1.00  0.31           C  \\nATOM    323  CE  LYS A  44      13.622   6.986 -22.889  1.00  0.24           C  \\nATOM    324  NZ  LYS A  44      13.076   6.072 -23.936  1.00  0.26           N  \\nATOM    325  N   GLU A  45      10.712  11.024 -17.700  1.00  0.29           N  \\nATOM    326  CA  GLU A  45       9.768  12.136 -17.639  1.00  0.30           C  \\nATOM    327  C   GLU A  45      10.182  13.152 -16.578  1.00  0.29           C  \\nATOM    328  CB  GLU A  45       8.353  11.626 -17.355  1.00  0.27           C  \\nATOM    329  O   GLU A  45       9.426  14.076 -16.268  1.00  0.28           O  \\nATOM    330  CG  GLU A  45       7.627  11.108 -18.588  1.00  0.27           C  \\nATOM    331  CD  GLU A  45       6.188  10.703 -18.311  1.00  0.33           C  \\nATOM    332  OE1 GLU A  45       5.729  10.847 -17.155  1.00  0.43           O  \\nATOM    333  OE2 GLU A  45       5.515  10.238 -19.257  1.00  0.37           O  \\nATOM    334  N   GLY A  46      11.471  13.302 -16.371  1.00  0.30           N  \\nATOM    335  CA  GLY A  46      12.158  14.303 -15.570  1.00  0.31           C  \\nATOM    336  C   GLY A  46      11.848  14.197 -14.089  1.00  0.30           C  \\nATOM    337  O   GLY A  46      12.179  15.097 -13.314  1.00  0.29           O  \\nATOM    338  N   ALA A  47      11.076  13.088 -13.648  1.00  0.31           N  \\nATOM    339  CA  ALA A  47      11.098  12.750 -12.227  1.00  0.32           C  \\nATOM    340  C   ALA A  47      12.291  11.859 -11.894  1.00  0.31           C  \\nATOM    341  CB  ALA A  47       9.795  12.063 -11.823  1.00  0.28           C  \\nATOM    342  O   ALA A  47      12.423  10.759 -12.436  1.00  0.29           O  \\nATOM    343  N   SER A  48      13.477  12.323 -11.780  1.00  0.29           N  \\nATOM    344  CA  SER A  48      14.624  11.690 -11.137  1.00  0.31           C  \\nATOM    345  C   SER A  48      14.247  11.117  -9.775  1.00  0.30           C  \\nATOM    346  CB  SER A  48      15.770  12.690 -10.979  1.00  0.28           C  \\nATOM    347  O   SER A  48      14.042  11.864  -8.816  1.00  0.29           O  \\nATOM    348  OG  SER A  48      16.733  12.518 -12.005  1.00  0.29           O  \\nATOM    349  N   PRO A  49      13.521  10.015  -9.721  1.00  0.35           N  \\nATOM    350  CA  PRO A  49      13.822   9.335  -8.459  1.00  0.34           C  \\nATOM    351  C   PRO A  49      15.319   9.116  -8.250  1.00  0.35           C  \\nATOM    352  CB  PRO A  49      13.088   7.999  -8.595  1.00  0.33           C  \\nATOM    353  O   PRO A  49      16.056   8.902  -9.215  1.00  0.33           O  \\nATOM    354  CG  PRO A  49      12.841   7.846 -10.061  1.00  0.31           C  \\nATOM    355  CD  PRO A  49      13.000   9.189 -10.714  1.00  0.34           C  \\nATOM    356  N   GLU A  50      15.953   9.850  -7.283  1.00  0.36           N  \\nATOM    357  CA  GLU A  50      17.274   9.408  -6.847  1.00  0.37           C  \\nATOM    358  C   GLU A  50      17.444   7.903  -7.037  1.00  0.36           C  \\nATOM    359  CB  GLU A  50      17.510   9.782  -5.381  1.00  0.34           C  \\nATOM    360  O   GLU A  50      16.604   7.117  -6.592  1.00  0.35           O  \\nATOM    361  CG  GLU A  50      18.976   9.775  -4.973  1.00  0.33           C  \\nATOM    362  CD  GLU A  50      19.212  10.330  -3.577  1.00  0.38           C  \\nATOM    363  OE1 GLU A  50      18.227  10.669  -2.884  1.00  0.48           O  \\nATOM    364  OE2 GLU A  50      20.393  10.426  -3.174  1.00  0.43           O  \\nATOM    365  N   PRO A  51      17.825   7.421  -8.338  1.00  0.34           N  \\nATOM    366  CA  PRO A  51      18.100   5.982  -8.361  1.00  0.34           C  \\nATOM    367  C   PRO A  51      18.328   5.403  -6.967  1.00  0.34           C  \\nATOM    368  CB  PRO A  51      19.370   5.880  -9.210  1.00  0.32           C  \\nATOM    369  O   PRO A  51      19.269   5.800  -6.276  1.00  0.33           O  \\nATOM    370  CG  PRO A  51      19.932   7.264  -9.224  1.00  0.31           C  \\nATOM    371  CD  PRO A  51      18.868   8.216  -8.759  1.00  0.34           C  \\nATOM    372  N   GLN A  52      17.356   4.988  -6.251  1.00  0.37           N  \\nATOM    373  CA  GLN A  52      17.717   4.002  -5.237  1.00  0.39           C  \\nATOM    374  C   GLN A  52      18.559   2.880  -5.837  1.00  0.37           C  \\nATOM    375  CB  GLN A  52      16.463   3.423  -4.580  1.00  0.36           C  \\nATOM    376  O   GLN A  52      18.282   2.411  -6.943  1.00  0.37           O  \\nATOM    377  CG  GLN A  52      15.879   4.305  -3.485  1.00  0.36           C  \\nATOM    378  CD  GLN A  52      15.131   3.513  -2.429  1.00  0.42           C  \\nATOM    379  NE2 GLN A  52      15.000   4.088  -1.238  1.00  0.38           N  \\nATOM    380  OE1 GLN A  52      14.673   2.394  -2.682  1.00  0.48           O  \\nATOM    381  N   PRO A  53      19.936   2.898  -5.494  1.00  0.37           N  \\nATOM    382  CA  PRO A  53      20.773   1.780  -5.932  1.00  0.36           C  \\nATOM    383  C   PRO A  53      19.972   0.502  -6.175  1.00  0.37           C  \\nATOM    384  CB  PRO A  53      21.754   1.597  -4.771  1.00  0.35           C  \\nATOM    385  O   PRO A  53      19.132   0.131  -5.352  1.00  0.35           O  \\nATOM    386  CG  PRO A  53      21.106   2.283  -3.612  1.00  0.33           C  \\nATOM    387  CD  PRO A  53      20.069   3.233  -4.140  1.00  0.36           C  \\nATOM    388  N   LYS A  54      19.468   0.190  -7.413  1.00  0.39           N  \\nATOM    389  CA  LYS A  54      19.186  -1.215  -7.689  1.00  0.39           C  \\nATOM    390  C   LYS A  54      19.813  -2.120  -6.633  1.00  0.39           C  \\nATOM    391  CB  LYS A  54      19.694  -1.602  -9.079  1.00  0.37           C  \\nATOM    392  O   LYS A  54      21.001  -1.995  -6.326  1.00  0.37           O  \\nATOM    393  CG  LYS A  54      18.610  -1.644 -10.145  1.00  0.35           C  \\nATOM    394  CD  LYS A  54      19.166  -2.089 -11.492  1.00  0.40           C  \\nATOM    395  CE  LYS A  54      18.083  -2.126 -12.561  1.00  0.36           C  \\nATOM    396  NZ  LYS A  54      18.621  -2.573 -13.880  1.00  0.38           N  \\nATOM    397  N   PRO A  55      19.090  -2.763  -5.652  1.00  0.40           N  \\nATOM    398  CA  PRO A  55      19.791  -3.970  -5.210  1.00  0.39           C  \\nATOM    399  C   PRO A  55      20.148  -4.902  -6.365  1.00  0.40           C  \\nATOM    400  CB  PRO A  55      18.787  -4.634  -4.263  1.00  0.38           C  \\nATOM    401  O   PRO A  55      19.257  -5.431  -7.035  1.00  0.38           O  \\nATOM    402  CG  PRO A  55      17.479  -3.984  -4.578  1.00  0.36           C  \\nATOM    403  CD  PRO A  55      17.743  -2.741  -5.377  1.00  0.40           C  \\nATOM    404  N   SER A  56      20.950  -4.487  -7.307  1.00  0.36           N  \\nATOM    405  CA  SER A  56      21.726  -5.453  -8.077  1.00  0.38           C  \\nATOM    406  C   SER A  56      22.249  -6.577  -7.188  1.00  0.36           C  \\nATOM    407  CB  SER A  56      22.896  -4.764  -8.781  1.00  0.35           C  \\nATOM    408  O   SER A  56      23.151  -6.364  -6.375  1.00  0.35           O  \\nATOM    409  OG  SER A  56      22.434  -3.708  -9.606  1.00  0.36           O  \\nATOM    410  N   GLY A  57      21.409  -7.196  -6.538  1.00  0.36           N  \\nATOM    411  CA  GLY A  57      21.830  -8.481  -6.003  1.00  0.37           C  \\nATOM    412  C   GLY A  57      20.860  -9.604  -6.318  1.00  0.37           C  \\nATOM    413  O   GLY A  57      19.670  -9.508  -6.010  1.00  0.35           O  \\nATOM    414  N   ASP A  58      20.829 -10.239  -7.547  1.00  0.40           N  \\nATOM    415  CA  ASP A  58      20.525 -11.667  -7.535  1.00  0.40           C  \\nATOM    416  C   ASP A  58      20.357 -12.181  -6.107  1.00  0.40           C  \\nATOM    417  CB  ASP A  58      21.622 -12.455  -8.252  1.00  0.38           C  \\nATOM    418  O   ASP A  58      21.239 -11.991  -5.266  1.00  0.39           O  \\nATOM    419  CG  ASP A  58      21.175 -13.008  -9.594  1.00  0.38           C  \\nATOM    420  OD1 ASP A  58      19.971 -13.298  -9.764  1.00  0.42           O  \\nATOM    421  OD2 ASP A  58      22.035 -13.156 -10.490  1.00  0.50           O  \\nATOM    422  N   PRO A  59      19.217 -12.135  -5.426  1.00  0.44           N  \\nATOM    423  CA  PRO A  59      19.291 -12.734  -4.091  1.00  0.44           C  \\nATOM    424  C   PRO A  59      20.336 -13.844  -3.999  1.00  0.44           C  \\nATOM    425  CB  PRO A  59      17.882 -13.291  -3.876  1.00  0.41           C  \\nATOM    426  O   PRO A  59      20.599 -14.361  -2.910  1.00  0.43           O  \\nATOM    427  CG  PRO A  59      17.278 -13.339  -5.242  1.00  0.40           C  \\nATOM    428  CD  PRO A  59      18.114 -12.499  -6.164  1.00  0.45           C  \\nATOM    429  N   GLN A  60      21.034 -14.114  -5.119  1.00  0.37           N  \\nATOM    430  CA  GLN A  60      22.159 -15.023  -4.923  1.00  0.41           C  \\nATOM    431  C   GLN A  60      23.488 -14.275  -4.982  1.00  0.37           C  \\nATOM    432  CB  GLN A  60      22.137 -16.138  -5.970  1.00  0.34           C  \\nATOM    433  O   GLN A  60      24.509 -14.775  -4.505  1.00  0.35           O  \\nATOM    434  CG  GLN A  60      21.120 -17.234  -5.680  1.00  0.35           C  \\nATOM    435  CD  GLN A  60      21.228 -18.403  -6.640  1.00  0.42           C  \\nATOM    436  NE2 GLN A  60      20.294 -19.343  -6.538  1.00  0.35           N  \\nATOM    437  OE1 GLN A  60      22.143 -18.462  -7.468  1.00  0.48           O  \\nATOM    438  N   SER A  61      23.459 -12.932  -4.871  1.00  0.38           N  \\nATOM    439  CA  SER A  61      24.816 -12.413  -4.739  1.00  0.40           C  \\nATOM    440  C   SER A  61      24.874 -11.266  -3.737  1.00  0.39           C  \\nATOM    441  CB  SER A  61      25.346 -11.944  -6.095  1.00  0.36           C  \\nATOM    442  O   SER A  61      24.126 -10.293  -3.855  1.00  0.37           O  \\nATOM    443  OG  SER A  61      25.160 -12.945  -7.081  1.00  0.42           O  \\nATOM    444  N   LYS A  62      25.078 -11.577  -2.469  1.00  0.40           N  \\nATOM    445  CA  LYS A  62      25.951 -10.963  -1.473  1.00  0.44           C  \\nATOM    446  C   LYS A  62      26.846  -9.901  -2.106  1.00  0.40           C  \\nATOM    447  CB  LYS A  62      26.807 -12.026  -0.781  1.00  0.36           C  \\nATOM    448  O   LYS A  62      28.071 -10.038  -2.112  1.00  0.38           O  \\nATOM    449  CG  LYS A  62      26.207 -12.558   0.512  1.00  0.37           C  \\nATOM    450  CD  LYS A  62      27.125 -13.577   1.176  1.00  0.44           C  \\nATOM    451  CE  LYS A  62      26.514 -14.131   2.455  1.00  0.35           C  \\nATOM    452  NZ  LYS A  62      27.399 -15.148   3.097  1.00  0.36           N  \\nATOM    453  N   GLN A  63      26.417  -9.144  -3.008  1.00  0.42           N  \\nATOM    454  CA  GLN A  63      27.253  -7.962  -3.191  1.00  0.46           C  \\nATOM    455  C   GLN A  63      26.408  -6.740  -3.541  1.00  0.43           C  \\nATOM    456  CB  GLN A  63      28.298  -8.207  -4.281  1.00  0.39           C  \\nATOM    457  O   GLN A  63      25.625  -6.771  -4.493  1.00  0.40           O  \\nATOM    458  CG  GLN A  63      29.445  -9.107  -3.842  1.00  0.38           C  \\nATOM    459  CD  GLN A  63      30.458  -9.348  -4.945  1.00  0.44           C  \\nATOM    460  NE2 GLN A  63      31.375 -10.281  -4.715  1.00  0.36           N  \\nATOM    461  OE1 GLN A  63      30.416  -8.701  -5.996  1.00  0.49           O  \\nATOM    462  N   THR A  64      26.018  -5.919  -2.625  1.00  0.47           N  \\nATOM    463  CA  THR A  64      25.855  -4.471  -2.702  1.00  0.48           C  \\nATOM    464  C   THR A  64      26.575  -3.910  -3.925  1.00  0.47           C  \\nATOM    465  CB  THR A  64      26.383  -3.781  -1.431  1.00  0.44           C  \\nATOM    466  O   THR A  64      27.788  -4.076  -4.069  1.00  0.45           O  \\nATOM    467  CG2 THR A  64      25.243  -3.168  -0.624  1.00  0.41           C  \\nATOM    468  OG1 THR A  64      27.065  -4.745  -0.619  1.00  0.41           O  \\nATOM    469  N   PRO A  65      25.947  -3.640  -5.005  1.00  0.42           N  \\nATOM    470  CA  PRO A  65      26.672  -2.718  -5.883  1.00  0.43           C  \\nATOM    471  C   PRO A  65      26.899  -1.350  -5.243  1.00  0.42           C  \\nATOM    472  CB  PRO A  65      25.758  -2.603  -7.106  1.00  0.38           C  \\nATOM    473  O   PRO A  65      25.989  -0.796  -4.622  1.00  0.40           O  \\nATOM    474  CG  PRO A  65      24.496  -3.296  -6.706  1.00  0.38           C  \\nATOM    475  CD  PRO A  65      24.709  -3.941  -5.366  1.00  0.42           C  \\nATOM    476  N   GLU A  66      28.216  -0.909  -4.943  1.00  0.48           N  \\nATOM    477  CA  GLU A  66      28.879   0.391  -4.971  1.00  0.48           C  \\nATOM    478  C   GLU A  66      28.096   1.392  -5.817  1.00  0.48           C  \\nATOM    479  CB  GLU A  66      30.307   0.255  -5.505  1.00  0.44           C  \\nATOM    480  O   GLU A  66      28.011   1.248  -7.038  1.00  0.46           O  \\nATOM    481  CG  GLU A  66      31.164   1.494  -5.292  1.00  0.42           C  \\nATOM    482  CD  GLU A  66      32.653   1.229  -5.449  1.00  0.47           C  \\nATOM    483  OE1 GLU A  66      33.027   0.125  -5.908  1.00  0.53           O  \\nATOM    484  OE2 GLU A  66      33.451   2.131  -5.111  1.00  0.47           O  \\nATOM    485  N   ALA A  67      26.984   1.836  -5.491  1.00  0.49           N  \\nATOM    486  CA  ALA A  67      26.532   3.112  -6.042  1.00  0.51           C  \\nATOM    487  C   ALA A  67      27.715   4.018  -6.369  1.00  0.49           C  \\nATOM    488  CB  ALA A  67      25.587   3.808  -5.065  1.00  0.45           C  \\nATOM    489  O   ALA A  67      28.484   4.391  -5.480  1.00  0.48           O  \\nATOM    490  N   SER A  68      28.344   3.955  -7.502  1.00  0.44           N  \\nATOM    491  CA  SER A  68      29.207   5.014  -8.014  1.00  0.45           C  \\nATOM    492  C   SER A  68      28.548   6.382  -7.873  1.00  0.44           C  \\nATOM    493  CB  SER A  68      29.558   4.757  -9.480  1.00  0.41           C  \\nATOM    494  O   SER A  68      27.403   6.571  -8.288  1.00  0.43           O  \\nATOM    495  OG  SER A  68      30.306   3.561  -9.616  1.00  0.45           O  \\nATOM    496  N   PRO A  69      28.996   7.125  -6.835  1.00  0.51           N  \\nATOM    497  CA  PRO A  69      28.654   8.540  -6.672  1.00  0.49           C  \\nATOM    498  C   PRO A  69      28.504   9.269  -8.005  1.00  0.51           C  \\nATOM    499  CB  PRO A  69      29.835   9.098  -5.874  1.00  0.46           C  \\nATOM    500  O   PRO A  69      29.311   9.067  -8.917  1.00  0.49           O  \\nATOM    501  CG  PRO A  69      30.927   8.094  -6.055  1.00  0.44           C  \\nATOM    502  CD  PRO A  69      30.317   6.806  -6.530  1.00  0.48           C  \\nATOM    503  N   ARG A  70      27.338   9.584  -8.473  1.00  0.52           N  \\nATOM    504  CA  ARG A  70      27.167  10.625  -9.480  1.00  0.53           C  \\nATOM    505  C   ARG A  70      28.358  11.578  -9.492  1.00  0.51           C  \\nATOM    506  CB  ARG A  70      25.874  11.406  -9.233  1.00  0.47           C  \\nATOM    507  O   ARG A  70      28.797  12.044  -8.438  1.00  0.50           O  \\nATOM    508  CG  ARG A  70      25.415  12.231 -10.424  1.00  0.46           C  \\nATOM    509  CD  ARG A  70      24.075  12.904 -10.161  1.00  0.50           C  \\nATOM    510  NE  ARG A  70      23.536  13.523 -11.369  1.00  0.41           N  \\nATOM    511  NH1 ARG A  70      21.514  14.107 -10.417  1.00  0.31           N  \\nATOM    512  NH2 ARG A  70      21.943  14.615 -12.611  1.00  0.28           N  \\nATOM    513  CZ  ARG A  70      22.332  14.080 -11.463  1.00  0.42           C  \\nATOM    514  N   PRO A  71      29.153  11.539 -10.632  1.00  0.48           N  \\nATOM    515  CA  PRO A  71      30.038  12.696 -10.782  1.00  0.47           C  \\nATOM    516  C   PRO A  71      29.363  14.009 -10.392  1.00  0.48           C  \\nATOM    517  CB  PRO A  71      30.386  12.683 -12.273  1.00  0.45           C  \\nATOM    518  O   PRO A  71      28.336  14.375 -10.969  1.00  0.46           O  \\nATOM    519  CG  PRO A  71      29.423  11.716 -12.881  1.00  0.44           C  \\nATOM    520  CD  PRO A  71      28.722  10.981 -11.775  1.00  0.47           C  \\nATOM    521  N   GLN A  72      29.365  14.444  -9.163  1.00  0.50           N  \\nATOM    522  CA  GLN A  72      29.269  15.870  -8.869  1.00  0.51           C  \\nATOM    523  C   GLN A  72      29.851  16.706 -10.005  1.00  0.50           C  \\nATOM    524  CB  GLN A  72      29.982  16.198  -7.556  1.00  0.46           C  \\nATOM    525  O   GLN A  72      30.958  16.436 -10.477  1.00  0.48           O  \\nATOM    526  CG  GLN A  72      29.715  17.607  -7.046  1.00  0.46           C  \\nATOM    527  CD  GLN A  72      30.292  17.851  -5.664  1.00  0.50           C  \\nATOM    528  NE2 GLN A  72      30.193  19.088  -5.190  1.00  0.42           N  \\nATOM    529  OE1 GLN A  72      30.821  16.934  -5.028  1.00  0.56           O  \\nATOM    530  N   PRO A  73      28.950  17.324 -10.891  1.00  0.50           N  \\nATOM    531  CA  PRO A  73      29.579  18.309 -11.775  1.00  0.50           C  \\nATOM    532  C   PRO A  73      30.729  19.055 -11.103  1.00  0.50           C  \\nATOM    533  CB  PRO A  73      28.431  19.264 -12.112  1.00  0.46           C  \\nATOM    534  O   PRO A  73      30.604  19.483  -9.952  1.00  0.48           O  \\nATOM    535  CG  PRO A  73      27.367  18.950 -11.110  1.00  0.45           C  \\nATOM    536  CD  PRO A  73      27.701  17.642 -10.453  1.00  0.48           C  \\nATOM    537  N   GLU A  74      32.007  18.691 -11.374  1.00  0.52           N  \\nATOM    538  CA  GLU A  74      33.218  19.480 -11.168  1.00  0.54           C  \\nATOM    539  C   GLU A  74      32.921  20.976 -11.238  1.00  0.52           C  \\nATOM    540  CB  GLU A  74      34.286  19.107 -12.199  1.00  0.49           C  \\nATOM    541  O   GLU A  74      32.301  21.446 -12.195  1.00  0.49           O  \\nATOM    542  CG  GLU A  74      35.028  17.818 -11.876  1.00  0.48           C  \\nATOM    543  CD  GLU A  74      36.166  17.524 -12.841  1.00  0.52           C  \\nATOM    544  OE1 GLU A  74      36.264  18.202 -13.889  1.00  0.55           O  \\nATOM    545  OE2 GLU A  74      36.965  16.607 -12.547  1.00  0.49           O  \\nATOM    546  N   PRO A  75      32.730  21.669 -10.027  1.00  0.53           N  \\nATOM    547  CA  PRO A  75      32.731  23.132  -9.950  1.00  0.52           C  \\nATOM    548  C   PRO A  75      33.689  23.775 -10.949  1.00  0.53           C  \\nATOM    549  CB  PRO A  75      33.175  23.408  -8.511  1.00  0.48           C  \\nATOM    550  O   PRO A  75      34.908  23.635 -10.819  1.00  0.51           O  \\nATOM    551  CG  PRO A  75      33.945  22.192  -8.108  1.00  0.47           C  \\nATOM    552  CD  PRO A  75      33.504  21.042  -8.969  1.00  0.50           C  \\nATOM    553  N   SER A  76      33.285  23.913 -12.142  1.00  0.49           N  \\nATOM    554  CA  SER A  76      34.006  24.768 -13.080  1.00  0.53           C  \\nATOM    555  C   SER A  76      34.144  26.187 -12.540  1.00  0.49           C  \\nATOM    556  CB  SER A  76      33.296  24.796 -14.434  1.00  0.45           C  \\nATOM    557  O   SER A  76      33.155  26.804 -12.139  1.00  0.47           O  \\nATOM    558  OG  SER A  76      33.189  23.490 -14.973  1.00  0.48           O  \\nATOM    559  N   GLY A  77      35.281  26.445 -11.865  1.00  0.47           N  \\nATOM    560  CA  GLY A  77      36.332  27.426 -11.647  1.00  0.48           C  \\nATOM    561  C   GLY A  77      35.902  28.844 -11.969  1.00  0.47           C  \\nATOM    562  O   GLY A  77      36.733  29.752 -12.028  1.00  0.44           O  \\nATOM    563  N   LYS A  78      34.677  29.238 -11.631  1.00  0.44           N  \\nATOM    564  CA  LYS A  78      34.413  30.665 -11.471  1.00  0.47           C  \\nATOM    565  C   LYS A  78      33.537  30.929 -10.250  1.00  0.43           C  \\nATOM    566  CB  LYS A  78      33.747  31.231 -12.726  1.00  0.40           C  \\nATOM    567  O   LYS A  78      32.541  30.237 -10.033  1.00  0.41           O  \\nATOM    568  CG  LYS A  78      34.702  31.446 -13.891  1.00  0.40           C  \\nATOM    569  CD  LYS A  78      33.996  32.077 -15.084  1.00  0.46           C  \\nATOM    570  CE  LYS A  78      34.952  32.301 -16.247  1.00  0.38           C  \\nATOM    571  NZ  LYS A  78      34.254  32.875 -17.436  1.00  0.38           N  \\nATOM    572  N   PRO A  79      33.995  31.338  -9.041  1.00  0.51           N  \\nATOM    573  CA  PRO A  79      33.077  31.922  -8.060  1.00  0.52           C  \\nATOM    574  C   PRO A  79      31.857  32.573  -8.707  1.00  0.51           C  \\nATOM    575  CB  PRO A  79      33.941  32.968  -7.349  1.00  0.46           C  \\nATOM    576  O   PRO A  79      31.984  33.599  -9.380  1.00  0.48           O  \\nATOM    577  CG  PRO A  79      35.139  33.134  -8.226  1.00  0.45           C  \\nATOM    578  CD  PRO A  79      35.143  32.028  -9.243  1.00  0.50           C  \\nATOM    579  N   GLN A  80      30.922  31.782  -9.200  1.00  0.44           N  \\nATOM    580  CA  GLN A  80      29.572  32.269  -9.465  1.00  0.47           C  \\nATOM    581  C   GLN A  80      28.876  32.689  -8.173  1.00  0.45           C  \\nATOM    582  CB  GLN A  80      28.746  31.200 -10.182  1.00  0.41           C  \\nATOM    583  O   GLN A  80      28.900  31.957  -7.182  1.00  0.43           O  \\nATOM    584  CG  GLN A  80      28.544  31.473 -11.667  1.00  0.40           C  \\nATOM    585  CD  GLN A  80      27.849  30.331 -12.384  1.00  0.46           C  \\nATOM    586  NE2 GLN A  80      27.757  30.429 -13.706  1.00  0.37           N  \\nATOM    587  OE1 GLN A  80      27.399  29.368 -11.755  1.00  0.49           O  \\nATOM    588  N   SER A  81      29.040  33.939  -7.695  1.00  0.49           N  \\nATOM    589  CA  SER A  81      28.131  34.829  -6.980  1.00  0.53           C  \\nATOM    590  C   SER A  81      26.983  34.053  -6.344  1.00  0.49           C  \\nATOM    591  CB  SER A  81      27.575  35.897  -7.923  1.00  0.44           C  \\nATOM    592  O   SER A  81      26.365  33.206  -6.992  1.00  0.46           O  \\nATOM    593  OG  SER A  81      28.598  36.418  -8.753  1.00  0.46           O  \\nATOM    594  N   GLU A  82      27.189  33.386  -5.119  1.00  0.50           N  \\nATOM    595  CA  GLU A  82      26.172  33.163  -4.096  1.00  0.54           C  \\nATOM    596  C   GLU A  82      24.776  33.096  -4.710  1.00  0.50           C  \\nATOM    597  CB  GLU A  82      26.226  34.264  -3.034  1.00  0.46           C  \\nATOM    598  O   GLU A  82      24.099  34.118  -4.837  1.00  0.47           O  \\nATOM    599  CG  GLU A  82      26.857  33.822  -1.721  1.00  0.46           C  \\nATOM    600  CD  GLU A  82      27.041  34.960  -0.731  1.00  0.52           C  \\nATOM    601  OE1 GLU A  82      26.700  36.117  -1.065  1.00  0.53           O  \\nATOM    602  OE2 GLU A  82      27.532  34.693   0.389  1.00  0.48           O  \\nATOM    603  N   GLN A  83      24.489  32.474  -5.831  1.00  0.43           N  \\nATOM    604  CA  GLN A  83      23.155  31.934  -6.072  1.00  0.47           C  \\nATOM    605  C   GLN A  83      22.904  30.690  -5.224  1.00  0.43           C  \\nATOM    606  CB  GLN A  83      22.968  31.606  -7.554  1.00  0.40           C  \\nATOM    607  O   GLN A  83      23.586  29.675  -5.385  1.00  0.41           O  \\nATOM    608  CG  GLN A  83      22.833  32.835  -8.443  1.00  0.40           C  \\nATOM    609  CD  GLN A  83      22.606  32.482  -9.901  1.00  0.46           C  \\nATOM    610  NE2 GLN A  83      22.871  33.434 -10.790  1.00  0.38           N  \\nATOM    611  OE1 GLN A  83      22.196  31.363 -10.226  1.00  0.51           O  \\nATOM    612  N   THR A  84      22.596  30.842  -3.824  1.00  0.48           N  \\nATOM    613  CA  THR A  84      21.772  29.784  -3.251  1.00  0.50           C  \\nATOM    614  C   THR A  84      21.092  28.973  -4.350  1.00  0.48           C  \\nATOM    615  CB  THR A  84      20.706  30.359  -2.300  1.00  0.45           C  \\nATOM    616  O   THR A  84      20.434  29.536  -5.228  1.00  0.46           O  \\nATOM    617  CG2 THR A  84      21.123  30.194  -0.842  1.00  0.42           C  \\nATOM    618  OG1 THR A  84      20.527  31.753  -2.580  1.00  0.41           O  \\nATOM    619  N   PRO A  85      21.597  27.927  -4.820  1.00  0.42           N  \\nATOM    620  CA  PRO A  85      20.581  27.226  -5.608  1.00  0.43           C  \\nATOM    621  C   PRO A  85      19.161  27.491  -5.110  1.00  0.43           C  \\nATOM    622  CB  PRO A  85      20.954  25.752  -5.433  1.00  0.39           C  \\nATOM    623  O   PRO A  85      18.875  27.306  -3.925  1.00  0.41           O  \\nATOM    624  CG  PRO A  85      22.069  25.755  -4.439  1.00  0.38           C  \\nATOM    625  CD  PRO A  85      22.394  27.180  -4.092  1.00  0.44           C  \\nATOM    626  N   LYS A  86      18.334  28.529  -5.460  1.00  0.38           N  \\nATOM    627  CA  LYS A  86      16.894  28.285  -5.464  1.00  0.38           C  \\nATOM    628  C   LYS A  86      16.590  26.790  -5.499  1.00  0.38           C  \\nATOM    629  CB  LYS A  86      16.235  28.982  -6.656  1.00  0.35           C  \\nATOM    630  O   LYS A  86      17.024  26.084  -6.412  1.00  0.36           O  \\nATOM    631  CG  LYS A  86      14.734  29.181  -6.504  1.00  0.35           C  \\nATOM    632  CD  LYS A  86      14.159  29.996  -7.655  1.00  0.39           C  \\nATOM    633  CE  LYS A  86      12.663  30.225  -7.486  1.00  0.35           C  \\nATOM    634  NZ  LYS A  86      12.111  31.099  -8.564  1.00  0.36           N  \\nATOM    635  N   ALA A  87      16.766  25.984  -4.440  1.00  0.41           N  \\nATOM    636  CA  ALA A  87      16.071  24.703  -4.539  1.00  0.43           C  \\nATOM    637  C   ALA A  87      15.090  24.701  -5.708  1.00  0.41           C  \\nATOM    638  CB  ALA A  87      15.341  24.392  -3.235  1.00  0.38           C  \\nATOM    639  O   ALA A  87      14.231  25.579  -5.807  1.00  0.39           O  \\nATOM    640  N   SER A  88      15.287  24.633  -6.941  1.00  0.33           N  \\nATOM    641  CA  SER A  88      14.254  24.224  -7.888  1.00  0.35           C  \\nATOM    642  C   SER A  88      13.038  23.654  -7.166  1.00  0.33           C  \\nATOM    643  CB  SER A  88      14.806  23.189  -8.869  1.00  0.31           C  \\nATOM    644  O   SER A  88      13.179  22.902  -6.199  1.00  0.32           O  \\nATOM    645  OG  SER A  88      15.855  23.742  -9.646  1.00  0.32           O  \\nATOM    646  N   PRO A  89      11.906  24.338  -6.910  1.00  0.36           N  \\nATOM    647  CA  PRO A  89      10.785  23.538  -6.412  1.00  0.35           C  \\nATOM    648  C   PRO A  89      10.990  22.040  -6.622  1.00  0.36           C  \\nATOM    649  CB  PRO A  89       9.597  24.047  -7.233  1.00  0.32           C  \\nATOM    650  O   PRO A  89      11.486  21.622  -7.671  1.00  0.34           O  \\nATOM    651  CG  PRO A  89      10.212  24.785  -8.378  1.00  0.31           C  \\nATOM    652  CD  PRO A  89      11.654  25.051  -8.054  1.00  0.36           C  \\nATOM    653  N   GLU A  90      11.395  21.189  -5.744  1.00  0.39           N  \\nATOM    654  CA  GLU A  90      11.186  19.761  -5.964  1.00  0.40           C  \\nATOM    655  C   GLU A  90      10.198  19.516  -7.100  1.00  0.39           C  \\nATOM    656  CB  GLU A  90      10.692  19.086  -4.681  1.00  0.36           C  \\nATOM    657  O   GLU A  90       9.120  20.114  -7.132  1.00  0.37           O  \\nATOM    658  CG  GLU A  90      10.916  17.581  -4.653  1.00  0.35           C  \\nATOM    659  CD  GLU A  90      10.593  16.952  -3.308  1.00  0.41           C  \\nATOM    660  OE1 GLU A  90      10.162  17.679  -2.385  1.00  0.49           O  \\nATOM    661  OE2 GLU A  90      10.773  15.721  -3.175  1.00  0.44           O  \\nATOM    662  N   PRO A  91      10.476  19.468  -8.400  1.00  0.32           N  \\nATOM    663  CA  PRO A  91       9.376  19.130  -9.306  1.00  0.32           C  \\nATOM    664  C   PRO A  91       8.127  18.654  -8.567  1.00  0.32           C  \\nATOM    665  CB  PRO A  91       9.960  18.009 -10.169  1.00  0.29           C  \\nATOM    666  O   PRO A  91       8.208  17.753  -7.728  1.00  0.29           O  \\nATOM    667  CG  PRO A  91      11.120  17.488  -9.383  1.00  0.28           C  \\nATOM    668  CD  PRO A  91      11.512  18.523  -8.368  1.00  0.32           C  \\nATOM    669  N   GLN A  92       7.049  19.294  -8.231  1.00  0.33           N  \\nATOM    670  CA  GLN A  92       5.711  18.711  -8.224  1.00  0.34           C  \\nATOM    671  C   GLN A  92       5.528  17.738  -9.385  1.00  0.33           C  \\nATOM    672  CB  GLN A  92       4.646  19.807  -8.285  1.00  0.31           C  \\nATOM    673  O   GLN A  92       5.774  18.090 -10.541  1.00  0.32           O  \\nATOM    674  CG  GLN A  92       4.387  20.488  -6.947  1.00  0.30           C  \\nATOM    675  CD  GLN A  92       3.028  21.159  -6.884  1.00  0.37           C  \\nATOM    676  NE2 GLN A  92       2.541  21.398  -5.671  1.00  0.33           N  \\nATOM    677  OE1 GLN A  92       2.422  21.462  -7.916  1.00  0.45           O  \\nATOM    678  N   PRO A  93       5.878  16.461  -9.335  1.00  0.35           N  \\nATOM    679  CA  PRO A  93       5.559  15.563 -10.448  1.00  0.35           C  \\nATOM    680  C   PRO A  93       4.269  15.951 -11.167  1.00  0.35           C  \\nATOM    681  CB  PRO A  93       5.417  14.198  -9.770  1.00  0.32           C  \\nATOM    682  O   PRO A  93       3.270  16.274 -10.519  1.00  0.34           O  \\nATOM    683  CG  PRO A  93       5.180  14.511  -8.328  1.00  0.31           C  \\nATOM    684  CD  PRO A  93       5.691  15.895  -8.049  1.00  0.35           C  \\nATOM    685  N   LYS A  94       4.194  16.578 -12.285  1.00  0.31           N  \\nATOM    686  CA  LYS A  94       3.076  16.661 -13.221  1.00  0.32           C  \\nATOM    687  C   LYS A  94       2.235  15.389 -13.187  1.00  0.31           C  \\nATOM    688  CB  LYS A  94       3.584  16.916 -14.641  1.00  0.29           C  \\nATOM    689  O   LYS A  94       2.775  14.281 -13.181  1.00  0.30           O  \\nATOM    690  CG  LYS A  94       3.720  18.390 -14.995  1.00  0.28           C  \\nATOM    691  CD  LYS A  94       4.158  18.580 -16.441  1.00  0.32           C  \\nATOM    692  CE  LYS A  94       4.324  20.053 -16.787  1.00  0.31           C  \\nATOM    693  NZ  LYS A  94       4.781  20.243 -18.196  1.00  0.32           N  \\nATOM    694  N   PRO A  95       0.987  15.225 -12.600  1.00  0.32           N  \\nATOM    695  CA  PRO A  95       0.075  14.080 -12.562  1.00  0.32           C  \\nATOM    696  C   PRO A  95       0.142  13.229 -13.828  1.00  0.32           C  \\nATOM    697  CB  PRO A  95      -1.303  14.732 -12.416  1.00  0.29           C  \\nATOM    698  O   PRO A  95      -0.529  12.198 -13.920  1.00  0.30           O  \\nATOM    699  CG  PRO A  95      -1.037  16.202 -12.458  1.00  0.28           C  \\nATOM    700  CD  PRO A  95       0.442  16.414 -12.611  1.00  0.33           C  \\nATOM    701  N   SER A  96       0.981  13.258 -14.735  1.00  0.31           N  \\nATOM    702  CA  SER A  96       0.851  12.433 -15.932  1.00  0.32           C  \\nATOM    703  C   SER A  96       1.285  10.996 -15.663  1.00  0.31           C  \\nATOM    704  CB  SER A  96       1.677  13.018 -17.078  1.00  0.28           C  \\nATOM    705  O   SER A  96       0.667  10.053 -16.160  1.00  0.29           O  \\nATOM    706  OG  SER A  96       3.021  13.221 -16.677  1.00  0.28           O  \\nATOM    707  N   GLY A  97       2.104  10.758 -14.703  1.00  0.34           N  \\nATOM    708  CA  GLY A  97       2.527   9.445 -14.244  1.00  0.34           C  \\nATOM    709  C   GLY A  97       1.727   8.941 -13.057  1.00  0.34           C  \\nATOM    710  O   GLY A  97       1.540   7.734 -12.896  1.00  0.32           O  \\nATOM    711  N   GLN A  98       1.100   9.773 -12.310  1.00  0.32           N  \\nATOM    712  CA  GLN A  98       0.420   9.513 -11.045  1.00  0.34           C  \\nATOM    713  C   GLN A  98      -0.864   8.718 -11.264  1.00  0.32           C  \\nATOM    714  CB  GLN A  98       0.109  10.825 -10.323  1.00  0.30           C  \\nATOM    715  O   GLN A  98      -1.151   7.776 -10.522  1.00  0.31           O  \\nATOM    716  CG  GLN A  98       1.300  11.416  -9.581  1.00  0.29           C  \\nATOM    717  CD  GLN A  98       0.957  12.701  -8.850  1.00  0.35           C  \\nATOM    718  NE2 GLN A  98       1.949  13.292  -8.193  1.00  0.33           N  \\nATOM    719  OE1 GLN A  98      -0.190  13.159  -8.876  1.00  0.40           O  \\nATOM    720  N   ALA A  99      -1.694   9.133 -12.301  1.00  0.33           N  \\nATOM    721  CA  ALA A  99      -2.904   8.427 -12.713  1.00  0.32           C  \\nATOM    722  C   ALA A  99      -2.581   7.009 -13.176  1.00  0.32           C  \\nATOM    723  CB  ALA A  99      -3.617   9.196 -13.822  1.00  0.30           C  \\nATOM    724  O   ALA A  99      -3.314   6.066 -12.868  1.00  0.31           O  \\nATOM    725  N   LYS A 100      -1.377   6.879 -13.761  1.00  0.35           N  \\nATOM    726  CA  LYS A 100      -0.957   5.565 -14.238  1.00  0.35           C  \\nATOM    727  C   LYS A 100      -0.514   4.675 -13.081  1.00  0.35           C  \\nATOM    728  CB  LYS A 100       0.174   5.700 -15.259  1.00  0.33           C  \\nATOM    729  O   LYS A 100      -0.738   3.463 -13.102  1.00  0.33           O  \\nATOM    730  CG  LYS A 100      -0.285   6.155 -16.636  1.00  0.31           C  \\nATOM    731  CD  LYS A 100       0.870   6.189 -17.628  1.00  0.35           C  \\nATOM    732  CE  LYS A 100       0.418   6.676 -18.998  1.00  0.31           C  \\nATOM    733  NZ  LYS A 100       1.554   6.743 -19.965  1.00  0.32           N  \\nATOM    734  N   GLU A 101       0.009   5.246 -12.025  1.00  0.36           N  \\nATOM    735  CA  GLU A 101       0.444   4.578 -10.801  1.00  0.37           C  \\nATOM    736  C   GLU A 101      -0.740   3.980 -10.047  1.00  0.36           C  \\nATOM    737  CB  GLU A 101       1.204   5.553  -9.898  1.00  0.35           C  \\nATOM    738  O   GLU A 101      -0.704   2.814  -9.649  1.00  0.35           O  \\nATOM    739  CG  GLU A 101       2.164   4.875  -8.931  1.00  0.33           C  \\nATOM    740  CD  GLU A 101       3.192   5.826  -8.340  1.00  0.39           C  \\nATOM    741  OE1 GLU A 101       3.163   7.033  -8.670  1.00  0.43           O  \\nATOM    742  OE2 GLU A 101       4.034   5.360  -7.540  1.00  0.42           O  \\nATOM    743  N   ILE A 102      -1.847   4.766  -9.752  1.00  0.36           N  \\nATOM    744  CA  ILE A 102      -3.040   4.351  -9.022  1.00  0.36           C  \\nATOM    745  C   ILE A 102      -3.776   3.269  -9.809  1.00  0.36           C  \\nATOM    746  CB  ILE A 102      -3.979   5.547  -8.749  1.00  0.35           C  \\nATOM    747  O   ILE A 102      -4.254   2.290  -9.231  1.00  0.35           O  \\nATOM    748  CG1 ILE A 102      -3.320   6.532  -7.777  1.00  0.32           C  \\nATOM    749  CG2 ILE A 102      -5.327   5.063  -8.207  1.00  0.32           C  \\nATOM    750  CD1 ILE A 102      -4.063   7.854  -7.636  1.00  0.30           C  \\nATOM    751  N   GLN A 103      -3.844   3.396 -11.173  1.00  0.37           N  \\nATOM    752  CA  GLN A 103      -4.492   2.400 -12.021  1.00  0.37           C  \\nATOM    753  C   GLN A 103      -3.801   1.045 -11.905  1.00  0.37           C  \\nATOM    754  CB  GLN A 103      -4.503   2.861 -13.479  1.00  0.36           C  \\nATOM    755  O   GLN A 103      -4.464   0.010 -11.809  1.00  0.36           O  \\nATOM    756  CG  GLN A 103      -5.755   3.635 -13.870  1.00  0.34           C  \\nATOM    757  CD  GLN A 103      -5.750   4.062 -15.326  1.00  0.38           C  \\nATOM    758  NE2 GLN A 103      -6.829   4.705 -15.759  1.00  0.36           N  \\nATOM    759  OE1 GLN A 103      -4.784   3.817 -16.055  1.00  0.43           O  \\nATOM    760  N   TRP A 104      -2.454   1.031 -11.844  1.00  0.23           N  \\nATOM    761  CA  TRP A 104      -1.708  -0.223 -11.806  1.00  0.32           C  \\nATOM    762  C   TRP A 104      -1.837  -0.892 -10.441  1.00  0.24           C  \\nATOM    763  CB  TRP A 104      -0.231   0.020 -12.132  1.00  0.24           C  \\nATOM    764  O   TRP A 104      -2.099  -2.094 -10.355  1.00  0.24           O  \\nATOM    765  CG  TRP A 104       0.500  -1.205 -12.591  1.00  0.25           C  \\nATOM    766  CD1 TRP A 104       0.431  -1.792 -13.824  1.00  0.26           C  \\nATOM    767  CD2 TRP A 104       1.408  -1.997 -11.820  1.00  0.27           C  \\nATOM    768  CE2 TRP A 104       1.854  -3.050 -12.650  1.00  0.19           C  \\nATOM    769  CE3 TRP A 104       1.890  -1.919 -10.506  1.00  0.27           C  \\nATOM    770  NE1 TRP A 104       1.244  -2.901 -13.866  1.00  0.24           N  \\nATOM    771  CH2 TRP A 104       3.215  -3.917 -10.919  1.00  0.24           C  \\nATOM    772  CZ2 TRP A 104       2.759  -4.017 -12.208  1.00  0.23           C  \\nATOM    773  CZ3 TRP A 104       2.791  -2.883 -10.068  1.00  0.25           C  \\nATOM    774  N   GLN A 105      -1.646  -0.189  -9.346  1.00  0.40           N  \\nATOM    775  CA  GLN A 105      -1.756  -0.798  -8.025  1.00  0.40           C  \\nATOM    776  C   GLN A 105      -3.122  -1.449  -7.831  1.00  0.40           C  \\nATOM    777  CB  GLN A 105      -1.510   0.243  -6.932  1.00  0.39           C  \\nATOM    778  O   GLN A 105      -3.215  -2.575  -7.339  1.00  0.39           O  \\nATOM    779  CG  GLN A 105      -0.046   0.626  -6.765  1.00  0.38           C  \\nATOM    780  CD  GLN A 105       0.170   1.666  -5.682  1.00  0.42           C  \\nATOM    781  NE2 GLN A 105       1.428   2.018  -5.440  1.00  0.39           N  \\nATOM    782  OE1 GLN A 105      -0.787   2.149  -5.068  1.00  0.46           O  \\nATOM    783  N   ALA A 106      -4.171  -0.718  -8.237  1.00  0.37           N  \\nATOM    784  CA  ALA A 106      -5.512  -1.292  -8.165  1.00  0.36           C  \\nATOM    785  C   ALA A 106      -5.598  -2.589  -8.965  1.00  0.37           C  \\nATOM    786  CB  ALA A 106      -6.548  -0.290  -8.669  1.00  0.35           C  \\nATOM    787  O   ALA A 106      -6.192  -3.569  -8.509  1.00  0.36           O  \\nATOM    788  N   LYS A 107      -4.870  -2.683 -10.068  1.00  0.38           N  \\nATOM    789  CA  LYS A 107      -4.930  -3.855 -10.936  1.00  0.39           C  \\nATOM    790  C   LYS A 107      -4.075  -4.992 -10.385  1.00  0.38           C  \\nATOM    791  CB  LYS A 107      -4.476  -3.497 -12.352  1.00  0.37           C  \\nATOM    792  O   LYS A 107      -4.495  -6.151 -10.393  1.00  0.37           O  \\nATOM    793  CG  LYS A 107      -5.422  -3.972 -13.445  1.00  0.36           C  \\nATOM    794  CD  LYS A 107      -4.940  -3.547 -14.826  1.00  0.40           C  \\nATOM    795  CE  LYS A 107      -5.851  -4.078 -15.925  1.00  0.37           C  \\nATOM    796  NZ  LYS A 107      -5.325  -3.755 -17.285  1.00  0.38           N  \\nATOM    797  N   GLU A 108      -2.896  -4.745  -9.972  1.00  0.39           N  \\nATOM    798  CA  GLU A 108      -2.054  -5.744  -9.320  1.00  0.40           C  \\nATOM    799  C   GLU A 108      -2.747  -6.337  -8.097  1.00  0.39           C  \\nATOM    800  CB  GLU A 108      -0.709  -5.134  -8.918  1.00  0.39           C  \\nATOM    801  O   GLU A 108      -2.735  -7.554  -7.899  1.00  0.38           O  \\nATOM    802  CG  GLU A 108       0.476  -6.066  -9.127  1.00  0.38           C  \\nATOM    803  CD  GLU A 108       1.770  -5.534  -8.532  1.00  0.43           C  \\nATOM    804  OE1 GLU A 108       1.820  -4.341  -8.156  1.00  0.47           O  \\nATOM    805  OE2 GLU A 108       2.742  -6.316  -8.443  1.00  0.45           O  \\nATOM    806  N   ILE A 109      -3.427  -5.538  -7.401  1.00  0.39           N  \\nATOM    807  CA  ILE A 109      -4.228  -6.036  -6.288  1.00  0.39           C  \\nATOM    808  C   ILE A 109      -5.353  -6.922  -6.817  1.00  0.39           C  \\nATOM    809  CB  ILE A 109      -4.808  -4.878  -5.446  1.00  0.38           C  \\nATOM    810  O   ILE A 109      -5.631  -7.985  -6.257  1.00  0.38           O  \\nATOM    811  CG1 ILE A 109      -3.687  -4.143  -4.703  1.00  0.35           C  \\nATOM    812  CG2 ILE A 109      -5.864  -5.398  -4.467  1.00  0.35           C  \\nATOM    813  CD1 ILE A 109      -4.117  -2.820  -4.085  1.00  0.33           C  \\nATOM    814  N   GLN A 110      -5.952  -6.531  -7.916  1.00  0.39           N  \\nATOM    815  CA  GLN A 110      -6.924  -7.412  -8.556  1.00  0.40           C  \\nATOM    816  C   GLN A 110      -6.294  -8.752  -8.922  1.00  0.39           C  \\nATOM    817  CB  GLN A 110      -7.510  -6.749  -9.803  1.00  0.39           C  \\nATOM    818  O   GLN A 110      -6.896  -9.807  -8.707  1.00  0.39           O  \\nATOM    819  CG  GLN A 110      -8.857  -6.078  -9.568  1.00  0.38           C  \\nATOM    820  CD  GLN A 110      -9.401  -5.401 -10.812  1.00  0.42           C  \\nATOM    821  NE2 GLN A 110     -10.559  -4.763 -10.681  1.00  0.39           N  \\nATOM    822  OE1 GLN A 110      -8.784  -5.450 -11.881  1.00  0.48           O  \\nATOM    823  N   GLU A 111      -5.058  -8.731  -9.439  1.00  0.40           N  \\nATOM    824  CA  GLU A 111      -4.419  -9.953  -9.918  1.00  0.41           C  \\nATOM    825  C   GLU A 111      -3.829 -10.756  -8.763  1.00  0.40           C  \\nATOM    826  CB  GLU A 111      -3.328  -9.624 -10.941  1.00  0.39           C  \\nATOM    827  O   GLU A 111      -3.959 -11.981  -8.721  1.00  0.39           O  \\nATOM    828  CG  GLU A 111      -3.861  -9.335 -12.337  1.00  0.38           C  \\nATOM    829  CD  GLU A 111      -2.798  -9.438 -13.418  1.00  0.43           C  \\nATOM    830  OE1 GLU A 111      -1.601  -9.586 -13.081  1.00  0.48           O  \\nATOM    831  OE2 GLU A 111      -3.164  -9.369 -14.613  1.00  0.46           O  \\nATOM    832  N   GLN A 112      -3.334 -10.090  -7.833  1.00  0.39           N  \\nATOM    833  CA  GLN A 112      -2.922 -10.784  -6.618  1.00  0.40           C  \\nATOM    834  C   GLN A 112      -4.128 -11.336  -5.863  1.00  0.39           C  \\nATOM    835  CB  GLN A 112      -2.119  -9.849  -5.712  1.00  0.38           C  \\nATOM    836  O   GLN A 112      -4.063 -12.427  -5.293  1.00  0.38           O  \\nATOM    837  CG  GLN A 112      -0.662  -9.691  -6.127  1.00  0.37           C  \\nATOM    838  CD  GLN A 112       0.154  -8.905  -5.118  1.00  0.41           C  \\nATOM    839  NE2 GLN A 112       1.386  -8.569  -5.484  1.00  0.38           N  \\nATOM    840  OE1 GLN A 112      -0.320  -8.601  -4.018  1.00  0.45           O  \\nATOM    841  N   ALA A 113      -5.196 -10.581  -5.817  1.00  0.37           N  \\nATOM    842  CA  ALA A 113      -6.435 -11.164  -5.308  1.00  0.37           C  \\nATOM    843  C   ALA A 113      -6.793 -12.439  -6.066  1.00  0.37           C  \\nATOM    844  CB  ALA A 113      -7.576 -10.155  -5.402  1.00  0.35           C  \\nATOM    845  O   ALA A 113      -7.529 -13.287  -5.556  1.00  0.35           O  \\nATOM    846  N   LYS A 114      -6.117 -12.662  -7.235  1.00  0.35           N  \\nATOM    847  CA  LYS A 114      -6.336 -13.902  -7.974  1.00  0.35           C  \\nATOM    848  C   LYS A 114      -5.392 -15.001  -7.494  1.00  0.35           C  \\nATOM    849  CB  LYS A 114      -6.153 -13.673  -9.475  1.00  0.33           C  \\nATOM    850  O   LYS A 114      -4.177 -14.911  -7.686  1.00  0.34           O  \\nATOM    851  CG  LYS A 114      -7.267 -12.858 -10.117  1.00  0.32           C  \\nATOM    852  CD  LYS A 114      -7.011 -12.634 -11.601  1.00  0.35           C  \\nATOM    853  CE  LYS A 114      -8.077 -11.743 -12.225  1.00  0.32           C  \\nATOM    854  NZ  LYS A 114      -7.744 -11.387 -13.637  1.00  0.32           N  \\nATOM    855  N   GLY A 115      -4.748 -14.972  -6.373  1.00  0.38           N  \\nATOM    856  CA  GLY A 115      -3.899 -15.878  -5.616  1.00  0.39           C  \\nATOM    857  C   GLY A 115      -3.620 -17.181  -6.342  1.00  0.38           C  \\nATOM    858  O   GLY A 115      -4.213 -17.455  -7.387  1.00  0.37           O  \\nATOM    859  N   PRO A 116      -2.389 -17.903  -6.226  1.00  0.37           N  \\nATOM    860  CA  PRO A 116      -2.172 -19.262  -6.725  1.00  0.38           C  \\nATOM    861  C   PRO A 116      -3.408 -20.148  -6.583  1.00  0.37           C  \\nATOM    862  CB  PRO A 116      -1.026 -19.780  -5.851  1.00  0.35           C  \\nATOM    863  O   PRO A 116      -4.075 -20.121  -5.546  1.00  0.36           O  \\nATOM    864  CG  PRO A 116      -1.114 -18.978  -4.593  1.00  0.33           C  \\nATOM    865  CD  PRO A 116      -1.792 -17.675  -4.904  1.00  0.37           C  \\nATOM    866  N   SER A 117      -4.295 -20.291  -7.625  1.00  0.32           N  \\nATOM    867  CA  SER A 117      -5.128 -21.485  -7.717  1.00  0.34           C  \\nATOM    868  C   SER A 117      -4.696 -22.541  -6.705  1.00  0.32           C  \\nATOM    869  CB  SER A 117      -5.075 -22.069  -9.130  1.00  0.31           C  \\nATOM    870  O   SER A 117      -3.609 -23.111  -6.821  1.00  0.31           O  \\nATOM    871  OG  SER A 117      -5.878 -21.310 -10.018  1.00  0.31           O  \\nATOM    872  N   GLY A 118      -4.497 -22.196  -5.465  1.00  0.35           N  \\nATOM    873  CA  GLY A 118      -4.474 -23.331  -4.557  1.00  0.36           C  \\nATOM    874  C   GLY A 118      -5.659 -24.261  -4.735  1.00  0.35           C  \\nATOM    875  O   GLY A 118      -6.790 -23.807  -4.918  1.00  0.34           O  \\nATOM    876  N   ASP A 119      -5.439 -25.372  -5.453  1.00  0.35           N  \\nATOM    877  CA  ASP A 119      -6.269 -26.573  -5.438  1.00  0.35           C  \\nATOM    878  C   ASP A 119      -7.237 -26.557  -4.257  1.00  0.35           C  \\nATOM    879  CB  ASP A 119      -5.396 -27.828  -5.387  1.00  0.33           C  \\nATOM    880  O   ASP A 119      -6.819 -26.402  -3.108  1.00  0.34           O  \\nATOM    881  CG  ASP A 119      -5.412 -28.615  -6.685  1.00  0.33           C  \\nATOM    882  OD1 ASP A 119      -6.416 -28.548  -7.427  1.00  0.37           O  \\nATOM    883  OD2 ASP A 119      -4.412 -29.311  -6.968  1.00  0.44           O  \\nATOM    884  N   PRO A 120      -8.491 -26.030  -4.335  1.00  0.40           N  \\nATOM    885  CA  PRO A 120      -9.505 -26.232  -3.298  1.00  0.39           C  \\nATOM    886  C   PRO A 120      -9.334 -27.556  -2.556  1.00  0.39           C  \\nATOM    887  CB  PRO A 120     -10.820 -26.212  -4.082  1.00  0.36           C  \\nATOM    888  O   PRO A 120     -10.027 -27.807  -1.567  1.00  0.37           O  \\nATOM    889  CG  PRO A 120     -10.422 -26.459  -5.502  1.00  0.35           C  \\nATOM    890  CD  PRO A 120      -8.950 -26.194  -5.637  1.00  0.40           C  \\nATOM    891  N   GLN A 121      -8.375 -28.353  -2.937  1.00  0.35           N  \\nATOM    892  CA  GLN A 121      -8.275 -29.560  -2.123  1.00  0.39           C  \\nATOM    893  C   GLN A 121      -7.340 -29.346  -0.936  1.00  0.35           C  \\nATOM    894  CB  GLN A 121      -7.789 -30.739  -2.968  1.00  0.32           C  \\nATOM    895  O   GLN A 121      -7.346 -30.131   0.015  1.00  0.33           O  \\nATOM    896  CG  GLN A 121      -8.886 -31.399  -3.792  1.00  0.32           C  \\nATOM    897  CD  GLN A 121      -8.413 -32.652  -4.503  1.00  0.39           C  \\nATOM    898  NE2 GLN A 121      -9.308 -33.278  -5.261  1.00  0.32           N  \\nATOM    899  OE1 GLN A 121      -7.253 -33.054  -4.374  1.00  0.46           O  \\nATOM    900  N   SER A 122      -6.945 -28.089  -0.563  1.00  0.35           N  \\nATOM    901  CA  SER A 122      -6.322 -28.043   0.756  1.00  0.37           C  \\nATOM    902  C   SER A 122      -7.142 -27.198   1.725  1.00  0.35           C  \\nATOM    903  CB  SER A 122      -4.901 -27.486   0.659  1.00  0.33           C  \\nATOM    904  O   SER A 122      -7.163 -25.969   1.622  1.00  0.34           O  \\nATOM    905  OG  SER A 122      -4.195 -28.094  -0.409  1.00  0.36           O  \\nATOM    906  N   LYS A 123      -8.537 -27.283   1.789  1.00  0.34           N  \\nATOM    907  CA  LYS A 123      -9.409 -27.257   2.960  1.00  0.37           C  \\nATOM    908  C   LYS A 123      -8.724 -27.887   4.169  1.00  0.34           C  \\nATOM    909  CB  LYS A 123     -10.724 -27.979   2.667  1.00  0.31           C  \\nATOM    910  O   LYS A 123      -8.684 -29.112   4.298  1.00  0.32           O  \\nATOM    911  CG  LYS A 123     -11.832 -27.067   2.162  1.00  0.32           C  \\nATOM    912  CD  LYS A 123     -13.136 -27.829   1.964  1.00  0.39           C  \\nATOM    913  CE  LYS A 123     -14.244 -26.919   1.451  1.00  0.33           C  \\nATOM    914  NZ  LYS A 123     -15.522 -27.664   1.245  1.00  0.35           N  \\nATOM    915  N   GLN A 124      -7.636 -27.411   4.717  1.00  0.37           N  \\nATOM    916  CA  GLN A 124      -7.436 -27.424   6.162  1.00  0.40           C  \\nATOM    917  C   GLN A 124      -7.115 -26.027   6.686  1.00  0.37           C  \\nATOM    918  CB  GLN A 124      -6.319 -28.397   6.541  1.00  0.34           C  \\nATOM    919  O   GLN A 124      -6.030 -25.499   6.435  1.00  0.35           O  \\nATOM    920  CG  GLN A 124      -6.707 -29.863   6.403  1.00  0.34           C  \\nATOM    921  CD  GLN A 124      -5.578 -30.806   6.774  1.00  0.41           C  \\nATOM    922  NE2 GLN A 124      -5.716 -32.073   6.399  1.00  0.35           N  \\nATOM    923  OE1 GLN A 124      -4.588 -30.399   7.391  1.00  0.48           O  \\nATOM    924  N   THR A 125      -7.902 -24.999   6.684  1.00  0.41           N  \\nATOM    925  CA  THR A 125      -7.872 -24.166   7.881  1.00  0.42           C  \\nATOM    926  C   THR A 125      -7.643 -25.018   9.126  1.00  0.40           C  \\nATOM    927  CB  THR A 125      -9.177 -23.364   8.037  1.00  0.38           C  \\nATOM    928  O   THR A 125      -8.324 -26.024   9.331  1.00  0.39           O  \\nATOM    929  CG2 THR A 125      -8.924 -21.867   7.894  1.00  0.36           C  \\nATOM    930  OG1 THR A 125     -10.108 -23.778   7.030  1.00  0.35           O  \\nATOM    931  N   PRO A 126      -6.461 -25.326   9.637  1.00  0.38           N  \\nATOM    932  CA  PRO A 126      -6.547 -25.917  10.975  1.00  0.38           C  \\nATOM    933  C   PRO A 126      -7.775 -25.446  11.750  1.00  0.38           C  \\nATOM    934  CB  PRO A 126      -5.259 -25.445  11.653  1.00  0.35           C  \\nATOM    935  O   PRO A 126      -8.083 -24.251  11.759  1.00  0.36           O  \\nATOM    936  CG  PRO A 126      -4.675 -24.443  10.708  1.00  0.34           C  \\nATOM    937  CD  PRO A 126      -5.490 -24.447   9.447  1.00  0.39           C  \\nATOM    938  N   GLU A 127      -9.091 -26.041  11.679  1.00  0.39           N  \\nATOM    939  CA  GLU A 127      -9.903 -26.068  12.892  1.00  0.39           C  \\nATOM    940  C   GLU A 127      -9.292 -25.195  13.984  1.00  0.38           C  \\nATOM    941  CB  GLU A 127     -10.069 -27.503  13.396  1.00  0.35           C  \\nATOM    942  O   GLU A 127      -8.188 -25.470  14.460  1.00  0.35           O  \\nATOM    943  CG  GLU A 127     -11.142 -27.659  14.464  1.00  0.34           C  \\nATOM    944  CD  GLU A 127     -11.581 -29.101  14.668  1.00  0.40           C  \\nATOM    945  OE1 GLU A 127     -10.931 -30.017  14.115  1.00  0.47           O  \\nATOM    946  OE2 GLU A 127     -12.584 -29.315  15.385  1.00  0.44           O  \\nATOM    947  N   ALA A 128      -8.974 -23.915  13.901  1.00  0.41           N  \\nATOM    948  CA  ALA A 128      -8.967 -23.396  15.267  1.00  0.43           C  \\nATOM    949  C   ALA A 128      -9.879 -24.220  16.172  1.00  0.40           C  \\nATOM    950  CB  ALA A 128      -9.392 -21.930  15.281  1.00  0.38           C  \\nATOM    951  O   ALA A 128     -11.088 -24.292  15.944  1.00  0.39           O  \\nATOM    952  N   SER A 129      -9.714 -25.524  16.380  1.00  0.35           N  \\nATOM    953  CA  SER A 129     -10.308 -26.110  17.578  1.00  0.37           C  \\nATOM    954  C   SER A 129     -10.872 -25.032  18.498  1.00  0.35           C  \\nATOM    955  CB  SER A 129      -9.276 -26.948  18.334  1.00  0.33           C  \\nATOM    956  O   SER A 129     -10.186 -24.059  18.817  1.00  0.33           O  \\nATOM    957  OG  SER A 129      -8.854 -28.052  17.551  1.00  0.37           O  \\nATOM    958  N   PRO A 130     -12.137 -24.630  18.349  1.00  0.44           N  \\nATOM    959  CA  PRO A 130     -12.645 -23.910  19.519  1.00  0.43           C  \\nATOM    960  C   PRO A 130     -11.853 -24.215  20.789  1.00  0.44           C  \\nATOM    961  CB  PRO A 130     -14.087 -24.409  19.642  1.00  0.40           C  \\nATOM    962  O   PRO A 130     -11.667 -25.383  21.139  1.00  0.42           O  \\nATOM    963  CG  PRO A 130     -14.122 -25.675  18.849  1.00  0.40           C  \\nATOM    964  CD  PRO A 130     -12.930 -25.699  17.935  1.00  0.44           C  \\nATOM    965  N   ARG A 131     -10.678 -23.673  21.102  1.00  0.44           N  \\nATOM    966  CA  ARG A 131     -10.250 -23.780  22.492  1.00  0.47           C  \\nATOM    967  C   ARG A 131     -11.432 -24.082  23.408  1.00  0.44           C  \\nATOM    968  CB  ARG A 131      -9.554 -22.494  22.942  1.00  0.41           C  \\nATOM    969  O   ARG A 131     -12.474 -23.430  23.319  1.00  0.42           O  \\nATOM    970  CG  ARG A 131      -8.687 -22.662  24.179  1.00  0.41           C  \\nATOM    971  CD  ARG A 131      -7.938 -21.382  24.522  1.00  0.45           C  \\nATOM    972  NE  ARG A 131      -6.911 -21.612  25.533  1.00  0.40           N  \\nATOM    973  NH1 ARG A 131      -6.040 -19.475  25.433  1.00  0.31           N  \\nATOM    974  NH2 ARG A 131      -5.140 -21.018  26.870  1.00  0.27           N  \\nATOM    975  CZ  ARG A 131      -6.033 -20.701  25.943  1.00  0.37           C  \\nATOM    976  N   PRO A 132     -11.616 -25.368  23.761  1.00  0.45           N  \\nATOM    977  CA  PRO A 132     -12.568 -25.686  24.828  1.00  0.45           C  \\nATOM    978  C   PRO A 132     -13.004 -24.453  25.616  1.00  0.45           C  \\nATOM    979  CB  PRO A 132     -11.787 -26.655  25.719  1.00  0.41           C  \\nATOM    980  O   PRO A 132     -12.162 -23.728  26.151  1.00  0.43           O  \\nATOM    981  CG  PRO A 132     -10.372 -26.552  25.248  1.00  0.41           C  \\nATOM    982  CD  PRO A 132     -10.354 -25.782  23.959  1.00  0.44           C  \\nATOM    983  N   GLN A 133     -13.982 -23.728  25.260  1.00  0.47           N  \\nATOM    984  CA  GLN A 133     -14.735 -23.000  26.276  1.00  0.49           C  \\nATOM    985  C   GLN A 133     -14.673 -23.713  27.623  1.00  0.47           C  \\nATOM    986  CB  GLN A 133     -16.191 -22.825  25.841  1.00  0.44           C  \\nATOM    987  O   GLN A 133     -14.926 -24.917  27.707  1.00  0.45           O  \\nATOM    988  CG  GLN A 133     -16.941 -21.754  26.622  1.00  0.44           C  \\nATOM    989  CD  GLN A 133     -18.331 -21.492  26.072  1.00  0.49           C  \\nATOM    990  NE2 GLN A 133     -19.135 -20.748  26.824  1.00  0.40           N  \\nATOM    991  OE1 GLN A 133     -18.679 -21.954  24.981  1.00  0.53           O  \\nATOM    992  N   PRO A 134     -13.689 -23.384  28.488  1.00  0.47           N  \\nATOM    993  CA  PRO A 134     -13.821 -23.915  29.847  1.00  0.48           C  \\nATOM    994  C   PRO A 134     -15.277 -24.115  30.263  1.00  0.47           C  \\nATOM    995  CB  PRO A 134     -13.149 -22.844  30.710  1.00  0.43           C  \\nATOM    996  O   PRO A 134     -16.113 -23.234  30.042  1.00  0.45           O  \\nATOM    997  CG  PRO A 134     -13.108 -21.626  29.846  1.00  0.43           C  \\nATOM    998  CD  PRO A 134     -13.289 -22.049  28.416  1.00  0.46           C  \\nATOM    999  N   GLU A 135     -15.899 -25.361  30.173  1.00  0.53           N  \\nATOM   1000  CA  GLU A 135     -16.998 -25.846  31.002  1.00  0.55           C  \\nATOM   1001  C   GLU A 135     -17.143 -25.011  32.271  1.00  0.53           C  \\nATOM   1002  CB  GLU A 135     -16.789 -27.319  31.363  1.00  0.49           C  \\nATOM   1003  O   GLU A 135     -16.161 -24.762  32.972  1.00  0.50           O  \\nATOM   1004  CG  GLU A 135     -17.106 -28.282  30.228  1.00  0.48           C  \\nATOM   1005  CD  GLU A 135     -16.977 -29.743  30.627  1.00  0.52           C  \\nATOM   1006  OE1 GLU A 135     -16.407 -30.030  31.704  1.00  0.55           O  \\nATOM   1007  OE2 GLU A 135     -17.448 -30.608  29.855  1.00  0.49           O  \\nATOM   1008  N   PRO A 136     -18.135 -24.043  32.286  1.00  0.53           N  \\nATOM   1009  CA  PRO A 136     -18.575 -23.396  33.524  1.00  0.54           C  \\nATOM   1010  C   PRO A 136     -18.474 -24.316  34.739  1.00  0.53           C  \\nATOM   1011  CB  PRO A 136     -20.034 -23.034  33.234  1.00  0.48           C  \\nATOM   1012  O   PRO A 136     -19.292 -25.225  34.899  1.00  0.50           O  \\nATOM   1013  CG  PRO A 136     -20.471 -24.015  32.195  1.00  0.47           C  \\nATOM   1014  CD  PRO A 136     -19.256 -24.507  31.462  1.00  0.51           C  \\nATOM   1015  N   SER A 137     -17.349 -24.609  35.167  1.00  0.49           N  \\nATOM   1016  CA  SER A 137     -17.244 -25.085  36.542  1.00  0.53           C  \\nATOM   1017  C   SER A 137     -17.366 -23.936  37.536  1.00  0.49           C  \\nATOM   1018  CB  SER A 137     -15.918 -25.817  36.757  1.00  0.45           C  \\nATOM   1019  O   SER A 137     -16.683 -22.918  37.404  1.00  0.46           O  \\nATOM   1020  OG  SER A 137     -15.808 -26.926  35.881  1.00  0.50           O  \\nATOM   1021  N   GLY A 138     -18.565 -23.779  37.985  1.00  0.49           N  \\nATOM   1022  CA  GLY A 138     -19.131 -23.471  39.289  1.00  0.50           C  \\nATOM   1023  C   GLY A 138     -18.230 -22.595  40.139  1.00  0.49           C  \\nATOM   1024  O   GLY A 138     -18.465 -22.434  41.338  1.00  0.46           O  \\nATOM   1025  N   LYS A 139     -17.497 -21.625  39.663  1.00  0.46           N  \\nATOM   1026  CA  LYS A 139     -16.990 -20.544  40.504  1.00  0.49           C  \\nATOM   1027  C   LYS A 139     -16.990 -19.217  39.752  1.00  0.46           C  \\nATOM   1028  CB  LYS A 139     -15.580 -20.866  40.999  1.00  0.42           C  \\nATOM   1029  O   LYS A 139     -16.633 -19.165  38.573  1.00  0.44           O  \\nATOM   1030  CG  LYS A 139     -15.530 -21.949  42.066  1.00  0.41           C  \\nATOM   1031  CD  LYS A 139     -14.110 -22.173  42.571  1.00  0.48           C  \\nATOM   1032  CE  LYS A 139     -14.059 -23.260  43.636  1.00  0.38           C  \\nATOM   1033  NZ  LYS A 139     -12.663 -23.515  44.101  1.00  0.38           N  \\nATOM   1034  N   PRO A 140     -17.820 -18.162  40.006  1.00  0.56           N  \\nATOM   1035  CA  PRO A 140     -17.626 -16.769  39.595  1.00  0.57           C  \\nATOM   1036  C   PRO A 140     -16.170 -16.448  39.267  1.00  0.56           C  \\nATOM   1037  CB  PRO A 140     -18.100 -15.973  40.814  1.00  0.50           C  \\nATOM   1038  O   PRO A 140     -15.309 -16.499  40.149  1.00  0.53           O  \\nATOM   1039  CG  PRO A 140     -18.111 -16.962  41.935  1.00  0.49           C  \\nATOM   1040  CD  PRO A 140     -18.001 -18.342  41.355  1.00  0.53           C  \\nATOM   1041  N   GLN A 141     -15.701 -16.812  38.039  1.00  0.46           N  \\nATOM   1042  CA  GLN A 141     -14.521 -16.197  37.440  1.00  0.50           C  \\nATOM   1043  C   GLN A 141     -14.699 -14.687  37.301  1.00  0.47           C  \\nATOM   1044  CB  GLN A 141     -14.226 -16.818  36.074  1.00  0.42           C  \\nATOM   1045  O   GLN A 141     -15.611 -14.225  36.613  1.00  0.46           O  \\nATOM   1046  CG  GLN A 141     -12.983 -17.697  36.053  1.00  0.42           C  \\nATOM   1047  CD  GLN A 141     -12.793 -18.414  34.730  1.00  0.48           C  \\nATOM   1048  NE2 GLN A 141     -11.799 -19.294  34.670  1.00  0.39           N  \\nATOM   1049  OE1 GLN A 141     -13.534 -18.180  33.770  1.00  0.51           O  \\nATOM   1050  N   SER A 142     -14.474 -13.881  38.262  1.00  0.50           N  \\nATOM   1051  CA  SER A 142     -13.851 -12.580  38.481  1.00  0.56           C  \\nATOM   1052  C   SER A 142     -12.709 -12.341  37.500  1.00  0.50           C  \\nATOM   1053  CB  SER A 142     -13.334 -12.467  39.916  1.00  0.46           C  \\nATOM   1054  O   SER A 142     -11.840 -13.198  37.330  1.00  0.47           O  \\nATOM   1055  OG  SER A 142     -13.887 -13.485  40.732  1.00  0.50           O  \\nATOM   1056  N   GLU A 143     -13.028 -11.763  36.260  1.00  0.55           N  \\nATOM   1057  CA  GLU A 143     -12.187 -10.843  35.499  1.00  0.59           C  \\nATOM   1058  C   GLU A 143     -11.481 -11.560  34.352  1.00  0.55           C  \\nATOM   1059  CB  GLU A 143     -11.158 -10.174  36.414  1.00  0.50           C  \\nATOM   1060  O   GLU A 143     -10.371 -12.068  34.521  1.00  0.52           O  \\nATOM   1061  CG  GLU A 143     -11.541  -8.764  36.842  1.00  0.48           C  \\nATOM   1062  CD  GLU A 143     -10.589  -8.170  37.869  1.00  0.53           C  \\nATOM   1063  OE1 GLU A 143      -9.637  -8.866  38.288  1.00  0.55           O  \\nATOM   1064  OE2 GLU A 143     -10.799  -6.999  38.257  1.00  0.49           O  \\nATOM   1065  N   GLN A 144     -12.194 -12.301  33.525  1.00  0.43           N  \\nATOM   1066  CA  GLN A 144     -11.684 -12.487  32.171  1.00  0.49           C  \\nATOM   1067  C   GLN A 144     -11.891 -11.232  31.328  1.00  0.44           C  \\nATOM   1068  CB  GLN A 144     -12.357 -13.686  31.502  1.00  0.41           C  \\nATOM   1069  O   GLN A 144     -13.028 -10.827  31.078  1.00  0.42           O  \\nATOM   1070  CG  GLN A 144     -11.867 -15.033  32.016  1.00  0.42           C  \\nATOM   1071  CD  GLN A 144     -12.522 -16.204  31.310  1.00  0.48           C  \\nATOM   1072  NE2 GLN A 144     -12.464 -17.378  31.930  1.00  0.39           N  \\nATOM   1073  OE1 GLN A 144     -13.077 -16.056  30.216  1.00  0.51           O  \\nATOM   1074  N   THR A 145     -10.915 -10.196  31.356  1.00  0.50           N  \\nATOM   1075  CA  THR A 145     -10.672  -9.333  30.205  1.00  0.53           C  \\nATOM   1076  C   THR A 145     -11.118 -10.015  28.915  1.00  0.50           C  \\nATOM   1077  CB  THR A 145      -9.184  -8.950  30.099  1.00  0.47           C  \\nATOM   1078  O   THR A 145     -10.716 -11.146  28.633  1.00  0.48           O  \\nATOM   1079  CG2 THR A 145      -8.907  -7.623  30.798  1.00  0.45           C  \\nATOM   1080  OG1 THR A 145      -8.388  -9.974  30.709  1.00  0.46           O  \\nATOM   1081  N   PRO A 146     -12.249  -9.691  28.459  1.00  0.43           N  \\nATOM   1082  CA  PRO A 146     -12.471 -10.131  27.080  1.00  0.44           C  \\nATOM   1083  C   PRO A 146     -11.184 -10.174  26.260  1.00  0.43           C  \\nATOM   1084  CB  PRO A 146     -13.436  -9.081  26.525  1.00  0.39           C  \\nATOM   1085  O   PRO A 146     -10.436  -9.193  26.225  1.00  0.42           O  \\nATOM   1086  CG  PRO A 146     -13.535  -8.047  27.600  1.00  0.39           C  \\nATOM   1087  CD  PRO A 146     -12.745  -8.518  28.788  1.00  0.44           C  \\nATOM   1088  N   LYS A 147     -10.453 -11.250  26.062  1.00  0.40           N  \\nATOM   1089  CA  LYS A 147      -9.620 -11.472  24.883  1.00  0.42           C  \\nATOM   1090  C   LYS A 147     -10.169 -10.721  23.673  1.00  0.40           C  \\nATOM   1091  CB  LYS A 147      -9.519 -12.966  24.571  1.00  0.38           C  \\nATOM   1092  O   LYS A 147     -11.315 -10.933  23.272  1.00  0.39           O  \\nATOM   1093  CG  LYS A 147      -8.290 -13.346  23.759  1.00  0.38           C  \\nATOM   1094  CD  LYS A 147      -8.182 -14.855  23.580  1.00  0.43           C  \\nATOM   1095  CE  LYS A 147      -6.940 -15.238  22.788  1.00  0.37           C  \\nATOM   1096  NZ  LYS A 147      -6.800 -16.720  22.659  1.00  0.37           N  \\nATOM   1097  N   ALA A 148      -9.991  -9.376  23.515  1.00  0.39           N  \\nATOM   1098  CA  ALA A 148     -10.248  -8.640  22.279  1.00  0.42           C  \\nATOM   1099  C   ALA A 148     -10.615  -9.590  21.142  1.00  0.39           C  \\nATOM   1100  CB  ALA A 148      -9.030  -7.802  21.896  1.00  0.36           C  \\nATOM   1101  O   ALA A 148      -9.853 -10.502  20.816  1.00  0.38           O  \\nATOM   1102  N   SER A 149     -11.784 -10.079  20.912  1.00  0.33           N  \\nATOM   1103  CA  SER A 149     -12.179 -10.440  19.555  1.00  0.33           C  \\nATOM   1104  C   SER A 149     -11.397  -9.639  18.520  1.00  0.33           C  \\nATOM   1105  CB  SER A 149     -13.679 -10.219  19.356  1.00  0.30           C  \\nATOM   1106  O   SER A 149     -11.106  -8.460  18.731  1.00  0.31           O  \\nATOM   1107  OG  SER A 149     -14.429 -11.001  20.270  1.00  0.33           O  \\nATOM   1108  N   PRO A 150     -10.483 -10.218  17.728  1.00  0.37           N  \\nATOM   1109  CA  PRO A 150      -9.994  -9.308  16.690  1.00  0.37           C  \\nATOM   1110  C   PRO A 150     -10.957  -8.156  16.414  1.00  0.38           C  \\nATOM   1111  CB  PRO A 150      -9.860 -10.213  15.462  1.00  0.34           C  \\nATOM   1112  O   PRO A 150     -12.170  -8.366  16.330  1.00  0.36           O  \\nATOM   1113  CG  PRO A 150     -10.687 -11.416  15.782  1.00  0.33           C  \\nATOM   1114  CD  PRO A 150     -10.926 -11.450  17.264  1.00  0.38           C  \\nATOM   1115  N   GLU A 151     -10.981  -6.964  16.958  1.00  0.36           N  \\nATOM   1116  CA  GLU A 151     -11.632  -5.728  16.534  1.00  0.38           C  \\nATOM   1117  C   GLU A 151     -12.334  -5.906  15.191  1.00  0.36           C  \\nATOM   1118  CB  GLU A 151     -10.616  -4.586  16.449  1.00  0.33           C  \\nATOM   1119  O   GLU A 151     -11.753  -6.445  14.247  1.00  0.35           O  \\nATOM   1120  CG  GLU A 151     -11.213  -3.210  16.706  1.00  0.33           C  \\nATOM   1121  CD  GLU A 151     -10.170  -2.107  16.783  1.00  0.39           C  \\nATOM   1122  OE1 GLU A 151      -8.966  -2.397  16.594  1.00  0.49           O  \\nATOM   1123  OE2 GLU A 151     -10.558  -0.945  17.034  1.00  0.44           O  \\nATOM   1124  N   PRO A 152     -13.732  -6.142  15.090  1.00  0.32           N  \\nATOM   1125  CA  PRO A 152     -14.246  -6.031  13.723  1.00  0.32           C  \\nATOM   1126  C   PRO A 152     -13.501  -4.986  12.895  1.00  0.32           C  \\nATOM   1127  CB  PRO A 152     -15.707  -5.623  13.928  1.00  0.30           C  \\nATOM   1128  O   PRO A 152     -13.395  -3.828  13.307  1.00  0.31           O  \\nATOM   1129  CG  PRO A 152     -15.763  -5.088  15.322  1.00  0.29           C  \\nATOM   1130  CD  PRO A 152     -14.552  -5.573  16.068  1.00  0.34           C  \\nATOM   1131  N   GLN A 153     -12.438  -5.294  12.174  1.00  0.33           N  \\nATOM   1132  CA  GLN A 153     -11.704  -4.556  11.151  1.00  0.35           C  \\nATOM   1133  C   GLN A 153     -12.657  -3.847  10.194  1.00  0.33           C  \\nATOM   1134  CB  GLN A 153     -10.778  -5.492  10.373  1.00  0.31           C  \\nATOM   1135  O   GLN A 153     -13.696  -4.397   9.824  1.00  0.32           O  \\nATOM   1136  CG  GLN A 153      -9.535  -5.911  11.146  1.00  0.31           C  \\nATOM   1137  CD  GLN A 153      -8.402  -6.354  10.240  1.00  0.39           C  \\nATOM   1138  NE2 GLN A 153      -7.174  -6.265  10.739  1.00  0.35           N  \\nATOM   1139  OE1 GLN A 153      -8.629  -6.776   9.102  1.00  0.47           O  \\nATOM   1140  N   PRO A 154     -12.790  -2.520  10.267  1.00  0.33           N  \\nATOM   1141  CA  PRO A 154     -13.665  -1.811   9.330  1.00  0.33           C  \\nATOM   1142  C   PRO A 154     -13.906  -2.593   8.041  1.00  0.33           C  \\nATOM   1143  CB  PRO A 154     -12.902  -0.514   9.048  1.00  0.31           C  \\nATOM   1144  O   PRO A 154     -12.995  -3.254   7.536  1.00  0.32           O  \\nATOM   1145  CG  PRO A 154     -11.568  -0.708   9.692  1.00  0.30           C  \\nATOM   1146  CD  PRO A 154     -11.597  -1.990  10.474  1.00  0.34           C  \\nATOM   1147  N   GLY A 155     -15.129  -3.015   7.830  1.00  0.32           N  \\nATOM   1148  CA  GLY A 155     -15.650  -3.535   6.576  1.00  0.31           C  \\nATOM   1149  C   GLY A 155     -15.223  -2.721   5.369  1.00  0.32           C  \\nATOM   1150  O   GLY A 155     -14.787  -1.577   5.508  1.00  0.31           O  \\nATOM   1151  N   ILE A 156     -14.649  -3.332   4.266  1.00  0.30           N  \\nATOM   1152  CA  ILE A 156     -14.283  -2.856   2.936  1.00  0.31           C  \\nATOM   1153  C   ILE A 156     -14.921  -1.491   2.684  1.00  0.30           C  \\nATOM   1154  CB  ILE A 156     -14.709  -3.858   1.840  1.00  0.28           C  \\nATOM   1155  O   ILE A 156     -14.298  -0.608   2.089  1.00  0.28           O  \\nATOM   1156  CG1 ILE A 156     -13.952  -5.181   2.003  1.00  0.26           C  \\nATOM   1157  CG2 ILE A 156     -14.480  -3.264   0.447  1.00  0.26           C  \\nATOM   1158  CD1 ILE A 156     -14.417  -6.280   1.058  1.00  0.24           C  \\nATOM   1159  N   LYS A 157     -16.120  -1.147   3.184  1.00  0.28           N  \\nATOM   1160  CA  LYS A 157     -16.786   0.128   2.933  1.00  0.28           C  \\nATOM   1161  C   LYS A 157     -16.031   1.280   3.589  1.00  0.28           C  \\nATOM   1162  CB  LYS A 157     -18.228   0.089   3.441  1.00  0.25           C  \\nATOM   1163  O   LYS A 157     -15.885   2.351   2.996  1.00  0.26           O  \\nATOM   1164  CG  LYS A 157     -19.182  -0.678   2.537  1.00  0.24           C  \\nATOM   1165  CD  LYS A 157     -20.619  -0.585   3.034  1.00  0.27           C  \\nATOM   1166  CE  LYS A 157     -21.574  -1.357   2.133  1.00  0.24           C  \\nATOM   1167  NZ  LYS A 157     -22.983  -1.275   2.621  1.00  0.26           N  \\nATOM   1168  N   GLU A 158     -15.586   1.142   4.789  1.00  0.28           N  \\nATOM   1169  CA  GLU A 158     -14.929   2.198   5.553  1.00  0.28           C  \\nATOM   1170  C   GLU A 158     -13.500   2.422   5.068  1.00  0.28           C  \\nATOM   1171  CB  GLU A 158     -14.932   1.863   7.047  1.00  0.26           C  \\nATOM   1172  O   GLU A 158     -13.047   3.564   4.960  1.00  0.26           O  \\nATOM   1173  CG  GLU A 158     -16.287   2.042   7.715  1.00  0.25           C  \\nATOM   1174  CD  GLU A 158     -16.395   1.329   9.053  1.00  0.30           C  \\nATOM   1175  OE1 GLU A 158     -15.396   0.719   9.498  1.00  0.38           O  \\nATOM   1176  OE2 GLU A 158     -17.488   1.379   9.661  1.00  0.31           O  \\nATOM   1177  N   GLU A 159     -12.833   1.457   4.524  1.00  0.27           N  \\nATOM   1178  CA  GLU A 159     -11.497   1.535   3.941  1.00  0.28           C  \\nATOM   1179  C   GLU A 159     -11.536   2.170   2.554  1.00  0.27           C  \\nATOM   1180  CB  GLU A 159     -10.860   0.145   3.866  1.00  0.26           C  \\nATOM   1181  O   GLU A 159     -10.692   3.007   2.225  1.00  0.26           O  \\nATOM   1182  CG  GLU A 159      -9.856  -0.133   4.975  1.00  0.25           C  \\nATOM   1183  CD  GLU A 159      -9.289  -1.543   4.931  1.00  0.31           C  \\nATOM   1184  OE1 GLU A 159      -9.606  -2.296   3.983  1.00  0.39           O  \\nATOM   1185  OE2 GLU A 159      -8.521  -1.897   5.853  1.00  0.34           O  \\nATOM   1186  N   ALA A 160     -12.551   1.930   1.827  1.00  0.29           N  \\nATOM   1187  CA  ALA A 160     -12.795   2.642   0.576  1.00  0.29           C  \\nATOM   1188  C   ALA A 160     -13.043   4.127   0.829  1.00  0.29           C  \\nATOM   1189  CB  ALA A 160     -13.981   2.027  -0.164  1.00  0.27           C  \\nATOM   1190  O   ALA A 160     -12.567   4.979   0.075  1.00  0.28           O  \\nATOM   1191  N   ALA A 161     -13.783   4.510   1.920  1.00  0.27           N  \\nATOM   1192  CA  ALA A 161     -14.008   5.909   2.275  1.00  0.26           C  \\nATOM   1193  C   ALA A 161     -12.700   6.595   2.657  1.00  0.27           C  \\nATOM   1194  CB  ALA A 161     -15.014   6.013   3.418  1.00  0.25           C  \\nATOM   1195  O   ALA A 161     -12.453   7.738   2.264  1.00  0.26           O  \\nATOM   1196  N   ALA A 162     -11.921   5.986   3.433  1.00  0.28           N  \\nATOM   1197  CA  ALA A 162     -10.613   6.554   3.752  1.00  0.27           C  \\nATOM   1198  C   ALA A 162      -9.796   6.792   2.485  1.00  0.28           C  \\nATOM   1199  CB  ALA A 162      -9.854   5.637   4.707  1.00  0.26           C  \\nATOM   1200  O   ALA A 162      -9.153   7.835   2.341  1.00  0.26           O  \\nATOM   1201  N   ILE A 163      -9.940   5.984   1.452  1.00  0.31           N  \\nATOM   1202  CA  ILE A 163      -9.247   6.035   0.170  1.00  0.31           C  \\nATOM   1203  C   ILE A 163      -9.769   7.212  -0.652  1.00  0.31           C  \\nATOM   1204  CB  ILE A 163      -9.414   4.716  -0.617  1.00  0.29           C  \\nATOM   1205  O   ILE A 163      -8.991   7.927  -1.288  1.00  0.29           O  \\nATOM   1206  CG1 ILE A 163      -8.698   3.569   0.106  1.00  0.29           C  \\nATOM   1207  CG2 ILE A 163      -8.893   4.871  -2.048  1.00  0.28           C  \\nATOM   1208  CD1 ILE A 163      -9.040   2.187  -0.434  1.00  0.26           C  \\nATOM   1209  N   LYS A 164     -11.154   7.409  -0.731  1.00  0.28           N  \\nATOM   1210  CA  LYS A 164     -11.760   8.574  -1.368  1.00  0.28           C  \\nATOM   1211  C   LYS A 164     -11.048   9.859  -0.954  1.00  0.28           C  \\nATOM   1212  CB  LYS A 164     -13.247   8.664  -1.021  1.00  0.26           C  \\nATOM   1213  O   LYS A 164     -10.807  10.737  -1.786  1.00  0.26           O  \\nATOM   1214  CG  LYS A 164     -14.171   8.150  -2.115  1.00  0.25           C  \\nATOM   1215  CD  LYS A 164     -15.635   8.277  -1.716  1.00  0.28           C  \\nATOM   1216  CE  LYS A 164     -16.560   7.763  -2.811  1.00  0.24           C  \\nATOM   1217  NZ  LYS A 164     -17.997   7.874  -2.419  1.00  0.26           N  \\nATOM   1218  N   TRP A 165     -10.650  10.016   0.290  1.00  0.17           N  \\nATOM   1219  CA  TRP A 165     -10.208  11.283   0.863  1.00  0.23           C  \\nATOM   1220  C   TRP A 165      -8.698  11.445   0.726  1.00  0.20           C  \\nATOM   1221  CB  TRP A 165     -10.614  11.377   2.337  1.00  0.17           C  \\nATOM   1222  O   TRP A 165      -8.208  12.541   0.444  1.00  0.18           O  \\nATOM   1223  CG  TRP A 165     -11.759  12.310   2.595  1.00  0.16           C  \\nATOM   1224  CD1 TRP A 165     -13.092  12.023   2.500  1.00  0.08           C  \\nATOM   1225  CD2 TRP A 165     -11.671  13.683   2.986  1.00  0.08           C  \\nATOM   1226  CE2 TRP A 165     -12.991  14.168   3.111  1.00  0.08           C  \\nATOM   1227  CE3 TRP A 165     -10.602  14.552   3.243  1.00  0.17           C  \\nATOM   1228  NE1 TRP A 165     -13.839  13.136   2.810  1.00  0.18           N  \\nATOM   1229  CH2 TRP A 165     -12.207  16.314   3.730  1.00  0.15           C  \\nATOM   1230  CZ2 TRP A 165     -13.271  15.485   3.484  1.00  0.16           C  \\nATOM   1231  CZ3 TRP A 165     -10.882  15.862   3.614  1.00  0.16           C  \\nATOM   1232  N   GLU A 166      -7.957  10.459   0.719  1.00  0.29           N  \\nATOM   1233  CA  GLU A 166      -6.524  10.623   0.499  1.00  0.30           C  \\nATOM   1234  C   GLU A 166      -6.224  10.961  -0.959  1.00  0.29           C  \\nATOM   1235  CB  GLU A 166      -5.768   9.357   0.911  1.00  0.28           C  \\nATOM   1236  O   GLU A 166      -5.426  11.856  -1.243  1.00  0.28           O  \\nATOM   1237  CG  GLU A 166      -5.017   9.491   2.228  1.00  0.27           C  \\nATOM   1238  CD  GLU A 166      -4.302   8.215   2.643  1.00  0.34           C  \\nATOM   1239  OE1 GLU A 166      -4.409   7.197   1.922  1.00  0.42           O  \\nATOM   1240  OE2 GLU A 166      -3.629   8.234   3.698  1.00  0.39           O  \\nATOM   1241  N   ALA A 167      -6.935  10.366  -1.865  1.00  0.31           N  \\nATOM   1242  CA  ALA A 167      -6.807  10.767  -3.264  1.00  0.31           C  \\nATOM   1243  C   ALA A 167      -7.252  12.213  -3.463  1.00  0.31           C  \\nATOM   1244  CB  ALA A 167      -7.617   9.836  -4.162  1.00  0.29           C  \\nATOM   1245  O   ALA A 167      -6.604  12.975  -4.185  1.00  0.29           O  \\nATOM   1246  N   ALA A 168      -8.332  12.672  -2.816  1.00  0.28           N  \\nATOM   1247  CA  ALA A 168      -8.800  14.054  -2.877  1.00  0.27           C  \\nATOM   1248  C   ALA A 168      -7.807  14.999  -2.208  1.00  0.28           C  \\nATOM   1249  CB  ALA A 168     -10.174  14.179  -2.222  1.00  0.26           C  \\nATOM   1250  O   ALA A 168      -7.494  16.063  -2.747  1.00  0.27           O  \\nATOM   1251  N   ALA A 169      -7.194  14.669  -1.104  1.00  0.26           N  \\nATOM   1252  CA  ALA A 169      -6.261  15.550  -0.406  1.00  0.26           C  \\nATOM   1253  C   ALA A 169      -4.911  15.595  -1.117  1.00  0.26           C  \\nATOM   1254  CB  ALA A 169      -6.082  15.096   1.041  1.00  0.24           C  \\nATOM   1255  O   ALA A 169      -4.269  16.646  -1.178  1.00  0.25           O  \\nATOM   1256  N   ILE A 170      -4.432  14.532  -1.674  1.00  0.34           N  \\nATOM   1257  CA  ILE A 170      -3.273  14.580  -2.558  1.00  0.34           C  \\nATOM   1258  C   ILE A 170      -3.566  15.497  -3.743  1.00  0.34           C  \\nATOM   1259  CB  ILE A 170      -2.881  13.170  -3.054  1.00  0.32           C  \\nATOM   1260  O   ILE A 170      -2.706  16.276  -4.161  1.00  0.32           O  \\nATOM   1261  CG1 ILE A 170      -2.377  12.314  -1.887  1.00  0.31           C  \\nATOM   1262  CG2 ILE A 170      -1.829  13.261  -4.163  1.00  0.30           C  \\nATOM   1263  CD1 ILE A 170      -2.213  10.839  -2.227  1.00  0.28           C  \\nATOM   1264  N   LYS A 171      -4.782  15.514  -4.173  1.00  0.29           N  \\nATOM   1265  CA  LYS A 171      -5.220  16.424  -5.227  1.00  0.30           C  \\nATOM   1266  C   LYS A 171      -4.980  17.879  -4.832  1.00  0.30           C  \\nATOM   1267  CB  LYS A 171      -6.699  16.205  -5.546  1.00  0.28           C  \\nATOM   1268  O   LYS A 171      -4.598  18.700  -5.669  1.00  0.28           O  \\nATOM   1269  CG  LYS A 171      -6.948  15.453  -6.845  1.00  0.27           C  \\nATOM   1270  CD  LYS A 171      -8.437  15.287  -7.119  1.00  0.33           C  \\nATOM   1271  CE  LYS A 171      -8.687  14.528  -8.415  1.00  0.28           C  \\nATOM   1272  NZ  LYS A 171     -10.146  14.359  -8.686  1.00  0.30           N  \\nATOM   1273  N   GLU A 172      -5.106  18.115  -3.534  1.00  0.26           N  \\nATOM   1274  CA  GLU A 172      -5.135  19.495  -3.060  1.00  0.25           C  \\nATOM   1275  C   GLU A 172      -3.801  19.890  -2.432  1.00  0.26           C  \\nATOM   1276  CB  GLU A 172      -6.270  19.694  -2.053  1.00  0.25           C  \\nATOM   1277  O   GLU A 172      -3.616  21.040  -2.029  1.00  0.25           O  \\nATOM   1278  CG  GLU A 172      -7.641  19.850  -2.695  1.00  0.25           C  \\nATOM   1279  CD  GLU A 172      -8.755  20.073  -1.684  1.00  0.29           C  \\nATOM   1280  OE1 GLU A 172      -8.462  20.175  -0.471  1.00  0.42           O  \\nATOM   1281  OE2 GLU A 172      -9.930  20.146  -2.108  1.00  0.35           O  \\nATOM   1282  N   GLY A 173      -2.624  19.097  -2.861  1.00  0.33           N  \\nATOM   1283  CA  GLY A 173      -1.368  19.426  -2.205  1.00  0.32           C  \\nATOM   1284  C   GLY A 173      -1.465  19.412  -0.691  1.00  0.32           C  \\nATOM   1285  O   GLY A 173      -0.557  19.881  -0.001  1.00  0.31           O  \\nATOM   1286  N   ALA A 174      -2.698  18.955  -0.232  1.00  0.28           N  \\nATOM   1287  CA  ALA A 174      -2.924  18.836   1.206  1.00  0.29           C  \\nATOM   1288  C   ALA A 174      -2.476  17.472   1.721  1.00  0.28           C  \\nATOM   1289  CB  ALA A 174      -4.397  19.068   1.534  1.00  0.26           C  \\nATOM   1290  O   ALA A 174      -2.781  16.442   1.114  1.00  0.27           O  \\nATOM   1291  N   SER A 175      -1.269  17.284   2.382  1.00  0.36           N  \\nATOM   1292  CA  SER A 175      -1.083  16.232   3.377  1.00  0.37           C  \\nATOM   1293  C   SER A 175      -2.399  15.881   4.062  1.00  0.36           C  \\nATOM   1294  CB  SER A 175      -0.052  16.659   4.423  1.00  0.35           C  \\nATOM   1295  O   SER A 175      -3.069  16.754   4.618  1.00  0.35           O  \\nATOM   1296  OG  SER A 175       1.238  16.181   4.083  1.00  0.33           O  \\nATOM   1297  N   PRO A 176      -3.168  14.867   3.520  1.00  0.35           N  \\nATOM   1298  CA  PRO A 176      -4.304  14.532   4.382  1.00  0.36           C  \\nATOM   1299  C   PRO A 176      -3.974  14.653   5.868  1.00  0.35           C  \\nATOM   1300  CB  PRO A 176      -4.615  13.081   4.007  1.00  0.34           C  \\nATOM   1301  O   PRO A 176      -2.872  14.292   6.290  1.00  0.34           O  \\nATOM   1302  CG  PRO A 176      -3.382  12.592   3.319  1.00  0.33           C  \\nATOM   1303  CD  PRO A 176      -2.557  13.780   2.915  1.00  0.36           C  \\nATOM   1304  N   GLU A 177      -4.645  15.562   6.636  1.00  0.36           N  \\nATOM   1305  CA  GLU A 177      -4.826  15.470   8.082  1.00  0.36           C  \\nATOM   1306  C   GLU A 177      -5.307  14.080   8.490  1.00  0.36           C  \\nATOM   1307  CB  GLU A 177      -5.815  16.532   8.569  1.00  0.34           C  \\nATOM   1308  O   GLU A 177      -6.286  13.570   7.941  1.00  0.34           O  \\nATOM   1309  CG  GLU A 177      -5.742  16.801  10.065  1.00  0.33           C  \\nATOM   1310  CD  GLU A 177      -6.587  17.986  10.504  1.00  0.39           C  \\nATOM   1311  OE1 GLU A 177      -7.257  18.604   9.646  1.00  0.48           O  \\nATOM   1312  OE2 GLU A 177      -6.579  18.298  11.716  1.00  0.44           O  \\nATOM   1313  N   PRO A 178      -4.392  13.125   8.903  1.00  0.40           N  \\nATOM   1314  CA  PRO A 178      -4.888  11.860   9.450  1.00  0.40           C  \\nATOM   1315  C   PRO A 178      -6.237  12.009  10.150  1.00  0.40           C  \\nATOM   1316  CB  PRO A 178      -3.798  11.454  10.445  1.00  0.38           C  \\nATOM   1317  O   PRO A 178      -6.375  12.826  11.064  1.00  0.39           O  \\nATOM   1318  CG  PRO A 178      -3.011  12.703  10.680  1.00  0.36           C  \\nATOM   1319  CD  PRO A 178      -3.363  13.697   9.611  1.00  0.39           C  \\nATOM   1320  N   GLN A 179      -7.348  11.648   9.455  1.00  0.35           N  \\nATOM   1321  CA  GLN A 179      -8.491  11.407  10.329  1.00  0.38           C  \\nATOM   1322  C   GLN A 179      -8.234  10.223  11.257  1.00  0.35           C  \\nATOM   1323  CB  GLN A 179      -9.756  11.163   9.505  1.00  0.35           C  \\nATOM   1324  O   GLN A 179      -7.668   9.211  10.838  1.00  0.35           O  \\nATOM   1325  CG  GLN A 179     -10.379  12.434   8.944  1.00  0.35           C  \\nATOM   1326  CD  GLN A 179     -11.889  12.464   9.095  1.00  0.42           C  \\nATOM   1327  NE2 GLN A 179     -12.466  13.659   9.038  1.00  0.38           N  \\nATOM   1328  OE1 GLN A 179     -12.530  11.422   9.262  1.00  0.48           O  \\nATOM   1329  N   PRO A 180      -8.316  10.549  12.649  1.00  0.38           N  \\nATOM   1330  CA  PRO A 180      -8.296   9.499  13.670  1.00  0.37           C  \\nATOM   1331  C   PRO A 180      -9.067   8.251  13.248  1.00  0.38           C  \\nATOM   1332  CB  PRO A 180      -8.960  10.168  14.877  1.00  0.36           C  \\nATOM   1333  O   PRO A 180     -10.199   8.352  12.768  1.00  0.37           O  \\nATOM   1334  CG  PRO A 180      -9.674  11.353  14.312  1.00  0.35           C  \\nATOM   1335  CD  PRO A 180      -9.114  11.642  12.949  1.00  0.37           C  \\nATOM   1336  N   LYS A 181      -8.445   7.239  12.659  1.00  0.44           N  \\nATOM   1337  CA  LYS A 181      -9.028   5.901  12.608  1.00  0.44           C  \\nATOM   1338  C   LYS A 181     -10.087   5.721  13.692  1.00  0.44           C  \\nATOM   1339  CB  LYS A 181      -7.940   4.836  12.756  1.00  0.42           C  \\nATOM   1340  O   LYS A 181      -9.886   6.128  14.838  1.00  0.43           O  \\nATOM   1341  CG  LYS A 181      -7.473   4.241  11.436  1.00  0.41           C  \\nATOM   1342  CD  LYS A 181      -6.436   3.146  11.652  1.00  0.44           C  \\nATOM   1343  CE  LYS A 181      -5.959   2.558  10.331  1.00  0.42           C  \\nATOM   1344  NZ  LYS A 181      -4.945   1.481  10.538  1.00  0.41           N  \\nATOM   1345  N   PRO A 182     -11.433   5.585  13.239  1.00  0.39           N  \\nATOM   1346  CA  PRO A 182     -12.398   5.255  14.290  1.00  0.39           C  \\nATOM   1347  C   PRO A 182     -11.887   4.177  15.243  1.00  0.39           C  \\nATOM   1348  CB  PRO A 182     -13.617   4.759  13.508  1.00  0.37           C  \\nATOM   1349  O   PRO A 182     -11.464   3.106  14.800  1.00  0.37           O  \\nATOM   1350  CG  PRO A 182     -13.113   4.535  12.118  1.00  0.36           C  \\nATOM   1351  CD  PRO A 182     -11.744   5.141  12.004  1.00  0.40           C  \\nATOM   1352  N   SER A 183     -11.029   4.469  16.180  1.00  0.38           N  \\nATOM   1353  CA  SER A 183     -10.896   3.624  17.362  1.00  0.39           C  \\nATOM   1354  C   SER A 183     -12.061   2.646  17.475  1.00  0.38           C  \\nATOM   1355  CB  SER A 183     -10.813   4.479  18.627  1.00  0.37           C  \\nATOM   1356  O   SER A 183     -13.225   3.048  17.410  1.00  0.37           O  \\nATOM   1357  OG  SER A 183      -9.749   5.410  18.535  1.00  0.37           O  \\nATOM   1358  N   GLY A 184     -12.001   1.604  16.705  1.00  0.42           N  \\nATOM   1359  CA  GLY A 184     -12.755   0.362  16.773  1.00  0.42           C  \\nATOM   1360  C   GLY A 184     -13.286   0.061  18.162  1.00  0.42           C  \\nATOM   1361  O   GLY A 184     -13.650  -1.078  18.460  1.00  0.40           O  \\nATOM   1362  N   ASP A 185     -13.616   1.088  18.858  1.00  0.38           N  \\nATOM   1363  CA  ASP A 185     -14.581   0.937  19.943  1.00  0.38           C  \\nATOM   1364  C   ASP A 185     -15.826   1.785  19.693  1.00  0.38           C  \\nATOM   1365  CB  ASP A 185     -13.946   1.316  21.283  1.00  0.36           C  \\nATOM   1366  O   ASP A 185     -15.734   3.006  19.551  1.00  0.37           O  \\nATOM   1367  CG  ASP A 185     -13.963   0.180  22.290  1.00  0.36           C  \\nATOM   1368  OD1 ASP A 185     -14.835  -0.710  22.191  1.00  0.40           O  \\nATOM   1369  OD2 ASP A 185     -13.098   0.177  23.193  1.00  0.47           O  \\nATOM   1370  N   PRO A 186     -16.893   1.388  18.986  1.00  0.41           N  \\nATOM   1371  CA  PRO A 186     -18.113   2.187  19.130  1.00  0.40           C  \\nATOM   1372  C   PRO A 186     -18.289   2.746  20.540  1.00  0.40           C  \\nATOM   1373  CB  PRO A 186     -19.227   1.192  18.794  1.00  0.37           C  \\nATOM   1374  O   PRO A 186     -19.174   3.573  20.774  1.00  0.38           O  \\nATOM   1375  CG  PRO A 186     -18.583  -0.152  18.897  1.00  0.36           C  \\nATOM   1376  CD  PRO A 186     -17.093   0.034  18.935  1.00  0.41           C  \\nATOM   1377  N   GLN A 187     -17.332   2.401  21.478  1.00  0.34           N  \\nATOM   1378  CA  GLN A 187     -17.675   3.069  22.729  1.00  0.39           C  \\nATOM   1379  C   GLN A 187     -16.662   4.160  23.066  1.00  0.35           C  \\nATOM   1380  CB  GLN A 187     -17.757   2.057  23.873  1.00  0.32           C  \\nATOM   1381  O   GLN A 187     -16.907   4.994  23.940  1.00  0.33           O  \\nATOM   1382  CG  GLN A 187     -19.061   1.273  23.908  1.00  0.34           C  \\nATOM   1383  CD  GLN A 187     -19.176   0.378  25.128  1.00  0.42           C  \\nATOM   1384  NE2 GLN A 187     -20.234  -0.424  25.178  1.00  0.33           N  \\nATOM   1385  OE1 GLN A 187     -18.320   0.407  26.018  1.00  0.48           O  \\nATOM   1386  N   SER A 188     -15.797   4.595  22.036  1.00  0.37           N  \\nATOM   1387  CA  SER A 188     -15.185   5.851  22.458  1.00  0.39           C  \\nATOM   1388  C   SER A 188     -15.325   6.923  21.382  1.00  0.38           C  \\nATOM   1389  CB  SER A 188     -13.708   5.642  22.792  1.00  0.35           C  \\nATOM   1390  O   SER A 188     -14.791   6.778  20.281  1.00  0.36           O  \\nATOM   1391  OG  SER A 188     -13.501   4.373  23.389  1.00  0.38           O  \\nATOM   1392  N   LYS A 189     -16.612   7.322  21.049  1.00  0.38           N  \\nATOM   1393  CA  LYS A 189     -17.032   8.680  20.717  1.00  0.41           C  \\nATOM   1394  C   LYS A 189     -16.065   9.711  21.293  1.00  0.38           C  \\nATOM   1395  CB  LYS A 189     -18.448   8.945  21.232  1.00  0.35           C  \\nATOM   1396  O   LYS A 189     -16.444  10.515  22.147  1.00  0.36           O  \\nATOM   1397  CG  LYS A 189     -19.540   8.680  20.205  1.00  0.35           C  \\nATOM   1398  CD  LYS A 189     -20.919   9.013  20.757  1.00  0.42           C  \\nATOM   1399  CE  LYS A 189     -22.014   8.729  19.738  1.00  0.33           C  \\nATOM   1400  NZ  LYS A 189     -23.371   9.038  20.281  1.00  0.34           N  \\nATOM   1401  N   GLN A 190     -14.803   9.486  21.348  1.00  0.40           N  \\nATOM   1402  CA  GLN A 190     -14.023  10.714  21.450  1.00  0.42           C  \\nATOM   1403  C   GLN A 190     -12.956  10.780  20.360  1.00  0.40           C  \\nATOM   1404  CB  GLN A 190     -13.371  10.825  22.829  1.00  0.37           C  \\nATOM   1405  O   GLN A 190     -12.034   9.962  20.337  1.00  0.38           O  \\nATOM   1406  CG  GLN A 190     -14.343  11.198  23.940  1.00  0.36           C  \\nATOM   1407  CD  GLN A 190     -13.673  11.300  25.297  1.00  0.42           C  \\nATOM   1408  NE2 GLN A 190     -14.477  11.390  26.351  1.00  0.36           N  \\nATOM   1409  OE1 GLN A 190     -12.442  11.296  25.398  1.00  0.48           O  \\nATOM   1410  N   THR A 191     -13.251  11.105  19.125  1.00  0.41           N  \\nATOM   1411  CA  THR A 191     -12.376  11.885  18.257  1.00  0.42           C  \\nATOM   1412  C   THR A 191     -11.503  12.831  19.078  1.00  0.41           C  \\nATOM   1413  CB  THR A 191     -13.188  12.695  17.229  1.00  0.39           C  \\nATOM   1414  O   THR A 191     -12.016  13.695  19.791  1.00  0.40           O  \\nATOM   1415  CG2 THR A 191     -13.133  12.045  15.851  1.00  0.38           C  \\nATOM   1416  OG1 THR A 191     -14.554  12.767  17.657  1.00  0.36           O  \\nATOM   1417  N   PRO A 192     -10.272  12.528  19.520  1.00  0.41           N  \\nATOM   1418  CA  PRO A 192      -9.532  13.759  19.804  1.00  0.41           C  \\nATOM   1419  C   PRO A 192      -9.392  14.660  18.579  1.00  0.41           C  \\nATOM   1420  CB  PRO A 192      -8.164  13.250  20.266  1.00  0.38           C  \\nATOM   1421  O   PRO A 192      -9.069  14.179  17.489  1.00  0.39           O  \\nATOM   1422  CG  PRO A 192      -8.201  11.777  20.018  1.00  0.37           C  \\nATOM   1423  CD  PRO A 192      -9.578  11.407  19.547  1.00  0.41           C  \\nATOM   1424  N   GLU A 193     -10.253  15.678  18.355  1.00  0.44           N  \\nATOM   1425  CA  GLU A 193      -9.840  16.921  17.712  1.00  0.43           C  \\nATOM   1426  C   GLU A 193      -8.346  16.912  17.401  1.00  0.44           C  \\nATOM   1427  CB  GLU A 193     -10.187  18.124  18.594  1.00  0.40           C  \\nATOM   1428  O   GLU A 193      -7.519  16.811  18.309  1.00  0.41           O  \\nATOM   1429  CG  GLU A 193     -10.091  19.462  17.875  1.00  0.38           C  \\nATOM   1430  CD  GLU A 193     -10.780  20.594  18.621  1.00  0.44           C  \\nATOM   1431  OE1 GLU A 193     -11.201  20.388  19.781  1.00  0.51           O  \\nATOM   1432  OE2 GLU A 193     -10.899  21.695  18.039  1.00  0.45           O  \\nATOM   1433  N   ALA A 194      -7.809  16.182  16.468  1.00  0.43           N  \\nATOM   1434  CA  ALA A 194      -6.468  16.598  16.066  1.00  0.45           C  \\nATOM   1435  C   ALA A 194      -6.327  18.117  16.114  1.00  0.44           C  \\nATOM   1436  CB  ALA A 194      -6.147  16.081  14.666  1.00  0.40           C  \\nATOM   1437  O   ALA A 194      -6.994  18.832  15.363  1.00  0.42           O  \\nATOM   1438  N   SER A 195      -6.316  18.836  17.241  1.00  0.36           N  \\nATOM   1439  CA  SER A 195      -5.726  20.163  17.386  1.00  0.36           C  \\nATOM   1440  C   SER A 195      -4.640  20.402  16.343  1.00  0.36           C  \\nATOM   1441  CB  SER A 195      -5.145  20.341  18.790  1.00  0.34           C  \\nATOM   1442  O   SER A 195      -3.779  19.546  16.129  1.00  0.35           O  \\nATOM   1443  OG  SER A 195      -6.166  20.262  19.770  1.00  0.38           O  \\nATOM   1444  N   PRO A 196      -4.924  21.131  15.256  1.00  0.46           N  \\nATOM   1445  CA  PRO A 196      -3.738  21.644  14.566  1.00  0.45           C  \\nATOM   1446  C   PRO A 196      -2.506  21.695  15.468  1.00  0.46           C  \\nATOM   1447  CB  PRO A 196      -4.160  23.053  14.139  1.00  0.42           C  \\nATOM   1448  O   PRO A 196      -2.608  22.073  16.638  1.00  0.44           O  \\nATOM   1449  CG  PRO A 196      -5.355  23.365  14.981  1.00  0.41           C  \\nATOM   1450  CD  PRO A 196      -5.869  22.084  15.573  1.00  0.45           C  \\nATOM   1451  N   ARG A 197      -1.563  20.752  15.485  1.00  0.46           N  \\nATOM   1452  CA  ARG A 197      -0.269  20.927  16.137  1.00  0.47           C  \\nATOM   1453  C   ARG A 197       0.040  22.404  16.354  1.00  0.46           C  \\nATOM   1454  CB  ARG A 197       0.842  20.275  15.312  1.00  0.42           C  \\nATOM   1455  O   ARG A 197      -0.042  23.205  15.420  1.00  0.44           O  \\nATOM   1456  CG  ARG A 197       2.080  19.917  16.119  1.00  0.41           C  \\nATOM   1457  CD  ARG A 197       3.113  19.187  15.273  1.00  0.45           C  \\nATOM   1458  NE  ARG A 197       4.107  18.510  16.101  1.00  0.40           N  \\nATOM   1459  NH1 ARG A 197       5.201  17.506  14.331  1.00  0.31           N  \\nATOM   1460  NH2 ARG A 197       5.927  17.160  16.478  1.00  0.27           N  \\nATOM   1461  CZ  ARG A 197       5.076  17.727  15.635  1.00  0.38           C  \\nATOM   1462  N   PRO A 198      -0.234  22.984  17.521  1.00  0.40           N  \\nATOM   1463  CA  PRO A 198       0.412  24.278  17.751  1.00  0.39           C  \\nATOM   1464  C   PRO A 198       1.695  24.449  16.942  1.00  0.40           C  \\nATOM   1465  CB  PRO A 198       0.710  24.261  19.252  1.00  0.37           C  \\nATOM   1466  O   PRO A 198       2.585  23.596  17.002  1.00  0.38           O  \\nATOM   1467  CG  PRO A 198       0.466  22.848  19.674  1.00  0.37           C  \\nATOM   1468  CD  PRO A 198      -0.203  22.121  18.543  1.00  0.41           C  \\nATOM   1469  N   GLN A 199       1.784  24.968  15.712  1.00  0.45           N  \\nATOM   1470  CA  GLN A 199       2.952  25.767  15.360  1.00  0.47           C  \\nATOM   1471  C   GLN A 199       3.719  26.199  16.606  1.00  0.45           C  \\nATOM   1472  CB  GLN A 199       2.538  26.994  14.546  1.00  0.43           C  \\nATOM   1473  O   GLN A 199       3.124  26.684  17.571  1.00  0.43           O  \\nATOM   1474  CG  GLN A 199       3.706  27.726  13.899  1.00  0.42           C  \\nATOM   1475  CD  GLN A 199       3.260  28.837  12.968  1.00  0.46           C  \\nATOM   1476  NE2 GLN A 199       4.219  29.590  12.440  1.00  0.40           N  \\nATOM   1477  OE1 GLN A 199       2.062  29.018  12.725  1.00  0.53           O  \\nATOM   1478  N   PRO A 200       4.834  25.480  17.043  1.00  0.45           N  \\nATOM   1479  CA  PRO A 200       5.587  26.072  18.151  1.00  0.45           C  \\nATOM   1480  C   PRO A 200       5.376  27.580  18.269  1.00  0.45           C  \\nATOM   1481  CB  PRO A 200       7.042  25.750  17.801  1.00  0.41           C  \\nATOM   1482  O   PRO A 200       5.456  28.298  17.269  1.00  0.43           O  \\nATOM   1483  CG  PRO A 200       7.000  25.292  16.378  1.00  0.40           C  \\nATOM   1484  CD  PRO A 200       5.569  25.039  16.001  1.00  0.44           C  \\nATOM   1485  N   GLU A 201       4.474  28.137  19.056  1.00  0.45           N  \\nATOM   1486  CA  GLU A 201       4.616  29.499  19.564  1.00  0.48           C  \\nATOM   1487  C   GLU A 201       6.085  29.901  19.661  1.00  0.45           C  \\nATOM   1488  CB  GLU A 201       3.941  29.636  20.931  1.00  0.42           C  \\nATOM   1489  O   GLU A 201       6.897  29.173  20.234  1.00  0.43           O  \\nATOM   1490  CG  GLU A 201       2.429  29.785  20.859  1.00  0.43           C  \\nATOM   1491  CD  GLU A 201       1.789  30.080  22.206  1.00  0.48           C  \\nATOM   1492  OE1 GLU A 201       2.478  29.959  23.244  1.00  0.53           O  \\nATOM   1493  OE2 GLU A 201       0.589  30.435  22.224  1.00  0.47           O  \\nATOM   1494  N   PRO A 202       6.719  30.634  18.577  1.00  0.49           N  \\nATOM   1495  CA  PRO A 202       7.978  31.341  18.821  1.00  0.49           C  \\nATOM   1496  C   PRO A 202       8.131  31.788  20.273  1.00  0.49           C  \\nATOM   1497  CB  PRO A 202       7.893  32.548  17.884  1.00  0.44           C  \\nATOM   1498  O   PRO A 202       7.415  32.685  20.725  1.00  0.47           O  \\nATOM   1499  CG  PRO A 202       6.432  32.734  17.632  1.00  0.43           C  \\nATOM   1500  CD  PRO A 202       5.729  31.439  17.923  1.00  0.47           C  \\nATOM   1501  N   SER A 203       8.388  30.921  21.170  1.00  0.45           N  \\nATOM   1502  CA  SER A 203       8.837  31.351  22.491  1.00  0.49           C  \\nATOM   1503  C   SER A 203      10.038  32.284  22.389  1.00  0.45           C  \\nATOM   1504  CB  SER A 203       9.191  30.142  23.357  1.00  0.41           C  \\nATOM   1505  O   SER A 203      11.029  31.960  21.731  1.00  0.42           O  \\nATOM   1506  OG  SER A 203       8.084  29.266  23.476  1.00  0.46           O  \\nATOM   1507  N   GLY A 204       9.827  33.603  22.348  1.00  0.45           N  \\nATOM   1508  CA  GLY A 204      10.246  34.828  23.010  1.00  0.48           C  \\nATOM   1509  C   GLY A 204      11.670  34.768  23.530  1.00  0.45           C  \\nATOM   1510  O   GLY A 204      11.926  34.195  24.592  1.00  0.42           O  \\nATOM   1511  N   LYS A 205      12.657  34.509  22.624  1.00  0.42           N  \\nATOM   1512  CA  LYS A 205      13.940  35.169  22.847  1.00  0.45           C  \\nATOM   1513  C   LYS A 205      14.669  35.413  21.530  1.00  0.42           C  \\nATOM   1514  CB  LYS A 205      14.816  34.337  23.785  1.00  0.38           C  \\nATOM   1515  O   LYS A 205      14.766  34.514  20.692  1.00  0.40           O  \\nATOM   1516  CG  LYS A 205      14.417  34.429  25.250  1.00  0.39           C  \\nATOM   1517  CD  LYS A 205      15.370  33.643  26.140  1.00  0.45           C  \\nATOM   1518  CE  LYS A 205      14.975  33.739  27.608  1.00  0.37           C  \\nATOM   1519  NZ  LYS A 205      15.884  32.937  28.480  1.00  0.37           N  \\nATOM   1520  N   PRO A 206      14.785  36.611  20.886  1.00  0.50           N  \\nATOM   1521  CA  PRO A 206      15.854  37.028  19.975  1.00  0.52           C  \\nATOM   1522  C   PRO A 206      17.182  36.334  20.266  1.00  0.50           C  \\nATOM   1523  CB  PRO A 206      15.959  38.535  20.221  1.00  0.46           C  \\nATOM   1524  O   PRO A 206      17.811  36.604  21.293  1.00  0.48           O  \\nATOM   1525  CG  PRO A 206      15.203  38.769  21.489  1.00  0.45           C  \\nATOM   1526  CD  PRO A 206      14.437  37.521  21.824  1.00  0.51           C  \\nATOM   1527  N   GLN A 207      17.364  35.141  19.818  1.00  0.44           N  \\nATOM   1528  CA  GLN A 207      18.723  34.613  19.754  1.00  0.48           C  \\nATOM   1529  C   GLN A 207      19.370  34.920  18.407  1.00  0.45           C  \\nATOM   1530  CB  GLN A 207      18.725  33.104  20.006  1.00  0.40           C  \\nATOM   1531  O   GLN A 207      18.745  34.748  17.359  1.00  0.42           O  \\nATOM   1532  CG  GLN A 207      19.232  32.712  21.388  1.00  0.40           C  \\nATOM   1533  CD  GLN A 207      19.102  31.226  21.661  1.00  0.45           C  \\nATOM   1534  NE2 GLN A 207      19.372  30.824  22.898  1.00  0.37           N  \\nATOM   1535  OE1 GLN A 207      18.762  30.444  20.767  1.00  0.49           O  \\nATOM   1536  N   SER A 208      20.181  35.973  18.316  1.00  0.47           N  \\nATOM   1537  CA  SER A 208      21.444  36.426  17.743  1.00  0.52           C  \\nATOM   1538  C   SER A 208      21.679  35.812  16.367  1.00  0.47           C  \\nATOM   1539  CB  SER A 208      22.609  36.080  18.671  1.00  0.42           C  \\nATOM   1540  O   SER A 208      21.504  34.605  16.182  1.00  0.44           O  \\nATOM   1541  OG  SER A 208      22.157  35.907  20.003  1.00  0.45           O  \\nATOM   1542  N   GLU A 209      21.259  36.542  15.228  1.00  0.47           N  \\nATOM   1543  CA  GLU A 209      21.971  36.711  13.965  1.00  0.53           C  \\nATOM   1544  C   GLU A 209      22.943  35.559  13.721  1.00  0.47           C  \\nATOM   1545  CB  GLU A 209      22.721  38.045  13.944  1.00  0.44           C  \\nATOM   1546  O   GLU A 209      24.095  35.609  14.157  1.00  0.45           O  \\nATOM   1547  CG  GLU A 209      21.908  39.199  13.375  1.00  0.45           C  \\nATOM   1548  CD  GLU A 209      22.634  40.533  13.439  1.00  0.52           C  \\nATOM   1549  OE1 GLU A 209      23.750  40.591  14.005  1.00  0.53           O  \\nATOM   1550  OE2 GLU A 209      22.083  41.529  12.919  1.00  0.48           O  \\nATOM   1551  N   GLN A 210      22.635  34.332  13.905  1.00  0.43           N  \\nATOM   1552  CA  GLN A 210      23.242  33.246  13.144  1.00  0.47           C  \\nATOM   1553  C   GLN A 210      22.387  32.876  11.936  1.00  0.43           C  \\nATOM   1554  CB  GLN A 210      23.454  32.019  14.032  1.00  0.38           C  \\nATOM   1555  O   GLN A 210      21.198  32.583  12.077  1.00  0.40           O  \\nATOM   1556  CG  GLN A 210      24.596  32.171  15.028  1.00  0.39           C  \\nATOM   1557  CD  GLN A 210      24.815  30.925  15.865  1.00  0.44           C  \\nATOM   1558  NE2 GLN A 210      25.558  31.067  16.957  1.00  0.37           N  \\nATOM   1559  OE1 GLN A 210      24.319  29.843  15.533  1.00  0.50           O  \\nATOM   1560  N   THR A 211      22.604  33.544  10.709  1.00  0.45           N  \\nATOM   1561  CA  THR A 211      22.258  32.905   9.444  1.00  0.48           C  \\nATOM   1562  C   THR A 211      21.960  31.423   9.652  1.00  0.45           C  \\nATOM   1563  CB  THR A 211      23.389  33.064   8.411  1.00  0.42           C  \\nATOM   1564  O   THR A 211      22.776  30.693  10.218  1.00  0.43           O  \\nATOM   1565  CG2 THR A 211      23.034  34.115   7.365  1.00  0.40           C  \\nATOM   1566  OG1 THR A 211      24.590  33.461   9.083  1.00  0.39           O  \\nATOM   1567  N   PRO A 212      20.792  31.041   9.893  1.00  0.39           N  \\nATOM   1568  CA  PRO A 212      20.604  29.589   9.838  1.00  0.42           C  \\nATOM   1569  C   PRO A 212      21.550  28.909   8.851  1.00  0.40           C  \\nATOM   1570  CB  PRO A 212      19.148  29.439   9.391  1.00  0.36           C  \\nATOM   1571  O   PRO A 212      21.572  29.261   7.668  1.00  0.38           O  \\nATOM   1572  CG  PRO A 212      18.680  30.836   9.138  1.00  0.37           C  \\nATOM   1573  CD  PRO A 212      19.812  31.778   9.430  1.00  0.44           C  \\nATOM   1574  N   LYS A 213      22.833  28.595   9.116  1.00  0.38           N  \\nATOM   1575  CA  LYS A 213      23.317  27.378   8.470  1.00  0.39           C  \\nATOM   1576  C   LYS A 213      22.157  26.529   7.958  1.00  0.38           C  \\nATOM   1577  CB  LYS A 213      24.177  26.562   9.436  1.00  0.35           C  \\nATOM   1578  O   LYS A 213      21.215  26.244   8.700  1.00  0.35           O  \\nATOM   1579  CG  LYS A 213      25.193  25.660   8.750  1.00  0.34           C  \\nATOM   1580  CD  LYS A 213      26.084  24.952   9.762  1.00  0.39           C  \\nATOM   1581  CE  LYS A 213      27.119  24.070   9.076  1.00  0.33           C  \\nATOM   1582  NZ  LYS A 213      28.040  23.428  10.061  1.00  0.35           N  \\nATOM   1583  N   ALA A 214      21.469  26.795   6.809  1.00  0.34           N  \\nATOM   1584  CA  ALA A 214      20.560  25.805   6.238  1.00  0.37           C  \\nATOM   1585  C   ALA A 214      20.614  24.495   7.019  1.00  0.35           C  \\nATOM   1586  CB  ALA A 214      20.895  25.559   4.769  1.00  0.32           C  \\nATOM   1587  O   ALA A 214      21.653  23.832   7.059  1.00  0.33           O  \\nATOM   1588  N   SER A 215      20.320  24.257   8.295  1.00  0.29           N  \\nATOM   1589  CA  SER A 215      19.913  22.941   8.778  1.00  0.31           C  \\nATOM   1590  C   SER A 215      19.495  22.034   7.626  1.00  0.29           C  \\nATOM   1591  CB  SER A 215      18.765  23.069   9.780  1.00  0.27           C  \\nATOM   1592  O   SER A 215      18.855  22.486   6.674  1.00  0.28           O  \\nATOM   1593  OG  SER A 215      19.183  23.775  10.936  1.00  0.30           O  \\nATOM   1594  N   PRO A 216      20.253  21.141   7.010  1.00  0.33           N  \\nATOM   1595  CA  PRO A 216      19.454  20.326   6.093  1.00  0.33           C  \\nATOM   1596  C   PRO A 216      17.957  20.605   6.207  1.00  0.33           C  \\nATOM   1597  CB  PRO A 216      19.777  18.893   6.523  1.00  0.30           C  \\nATOM   1598  O   PRO A 216      17.455  20.860   7.304  1.00  0.31           O  \\nATOM   1599  CG  PRO A 216      20.389  19.030   7.880  1.00  0.29           C  \\nATOM   1600  CD  PRO A 216      20.799  20.461   8.076  1.00  0.35           C  \\nATOM   1601  N   GLU A 217      17.236  21.363   5.461  1.00  0.30           N  \\nATOM   1602  CA  GLU A 217      15.782  21.358   5.590  1.00  0.31           C  \\nATOM   1603  C   GLU A 217      15.318  20.277   6.562  1.00  0.29           C  \\nATOM   1604  CB  GLU A 217      15.122  21.155   4.224  1.00  0.27           C  \\nATOM   1605  O   GLU A 217      15.737  19.122   6.461  1.00  0.27           O  \\nATOM   1606  CG  GLU A 217      13.735  21.771   4.114  1.00  0.28           C  \\nATOM   1607  CD  GLU A 217      13.137  21.658   2.720  1.00  0.35           C  \\nATOM   1608  OE1 GLU A 217      13.802  21.098   1.818  1.00  0.45           O  \\nATOM   1609  OE2 GLU A 217      11.996  22.133   2.529  1.00  0.38           O  \\nATOM   1610  N   PRO A 218      15.286  20.389   7.908  1.00  0.27           N  \\nATOM   1611  CA  PRO A 218      14.618  19.261   8.561  1.00  0.27           C  \\nATOM   1612  C   PRO A 218      14.039  18.260   7.564  1.00  0.27           C  \\nATOM   1613  CB  PRO A 218      13.505  19.931   9.371  1.00  0.25           C  \\nATOM   1614  O   PRO A 218      13.370  18.655   6.606  1.00  0.25           O  \\nATOM   1615  CG  PRO A 218      13.382  21.302   8.789  1.00  0.25           C  \\nATOM   1616  CD  PRO A 218      14.605  21.582   7.964  1.00  0.29           C  \\nATOM   1617  N   GLN A 219      14.642  17.202   6.981  1.00  0.32           N  \\nATOM   1618  CA  GLN A 219      13.686  16.177   6.575  1.00  0.33           C  \\nATOM   1619  C   GLN A 219      12.292  16.480   7.116  1.00  0.32           C  \\nATOM   1620  CB  GLN A 219      14.144  14.797   7.048  1.00  0.30           C  \\nATOM   1621  O   GLN A 219      12.140  16.851   8.282  1.00  0.30           O  \\nATOM   1622  CG  GLN A 219      15.216  14.169   6.167  1.00  0.29           C  \\nATOM   1623  CD  GLN A 219      15.139  12.654   6.139  1.00  0.36           C  \\nATOM   1624  NE2 GLN A 219      15.700  12.052   5.096  1.00  0.31           N  \\nATOM   1625  OE1 GLN A 219      14.581  12.030   7.047  1.00  0.44           O  \\nATOM   1626  N   PRO A 220      11.430  17.248   6.550  1.00  0.32           N  \\nATOM   1627  CA  PRO A 220      10.155  17.312   7.267  1.00  0.32           C  \\nATOM   1628  C   PRO A 220      10.051  16.274   8.381  1.00  0.32           C  \\nATOM   1629  CB  PRO A 220       9.121  17.042   6.171  1.00  0.30           C  \\nATOM   1630  O   PRO A 220      10.453  15.122   8.196  1.00  0.30           O  \\nATOM   1631  CG  PRO A 220       9.909  16.481   5.032  1.00  0.28           C  \\nATOM   1632  CD  PRO A 220      11.360  16.798   5.254  1.00  0.32           C  \\nATOM   1633  N   LYS A 221      10.469  16.449   9.711  1.00  0.32           N  \\nATOM   1634  CA  LYS A 221       9.855  15.561  10.693  1.00  0.32           C  \\nATOM   1635  C   LYS A 221       8.774  14.696  10.052  1.00  0.31           C  \\nATOM   1636  CB  LYS A 221       9.262  16.367  11.850  1.00  0.29           C  \\nATOM   1637  O   LYS A 221       7.995  15.176   9.226  1.00  0.30           O  \\nATOM   1638  CG  LYS A 221      10.292  16.844  12.863  1.00  0.29           C  \\nATOM   1639  CD  LYS A 221       9.635  17.570  14.030  1.00  0.32           C  \\nATOM   1640  CE  LYS A 221      10.667  18.083  15.024  1.00  0.30           C  \\nATOM   1641  NZ  LYS A 221      10.030  18.831  16.149  1.00  0.32           N  \\nATOM   1642  N   PRO A 222       8.963  13.404   9.584  1.00  0.34           N  \\nATOM   1643  CA  PRO A 222       7.864  12.653   8.973  1.00  0.34           C  \\nATOM   1644  C   PRO A 222       6.512  13.343   9.141  1.00  0.35           C  \\nATOM   1645  CB  PRO A 222       7.891  11.317   9.719  1.00  0.32           C  \\nATOM   1646  O   PRO A 222       6.196  13.833  10.228  1.00  0.33           O  \\nATOM   1647  CG  PRO A 222       8.998  11.458  10.714  1.00  0.31           C  \\nATOM   1648  CD  PRO A 222       9.552  12.850  10.617  1.00  0.35           C  \\nATOM   1649  N   SER A 223       6.138  14.272   8.376  1.00  0.35           N  \\nATOM   1650  CA  SER A 223       4.734  14.669   8.384  1.00  0.36           C  \\nATOM   1651  C   SER A 223       3.873  13.637   9.106  1.00  0.35           C  \\nATOM   1652  CB  SER A 223       4.222  14.862   6.956  1.00  0.34           C  \\nATOM   1653  O   SER A 223       4.273  12.480   9.252  1.00  0.34           O  \\nATOM   1654  OG  SER A 223       4.255  13.640   6.240  1.00  0.31           O  \\nATOM   1655  N   GLY A 224       3.321  13.942  10.354  1.00  0.42           N  \\nATOM   1656  CA  GLY A 224       2.263  13.082  10.860  1.00  0.41           C  \\nATOM   1657  C   GLY A 224       2.044  11.844  10.012  1.00  0.42           C  \\nATOM   1658  O   GLY A 224       1.889  10.742  10.542  1.00  0.40           O  \\nATOM   1659  N   GLN A 225       2.246  12.010   8.687  1.00  0.43           N  \\nATOM   1660  CA  GLN A 225       2.024  10.898   7.769  1.00  0.45           C  \\nATOM   1661  C   GLN A 225       3.084   9.815   7.952  1.00  0.43           C  \\nATOM   1662  CB  GLN A 225       2.019  11.388   6.321  1.00  0.42           C  \\nATOM   1663  O   GLN A 225       2.764   8.625   7.982  1.00  0.43           O  \\nATOM   1664  CG  GLN A 225       0.684  11.965   5.872  1.00  0.42           C  \\nATOM   1665  CD  GLN A 225       0.702  12.433   4.428  1.00  0.48           C  \\nATOM   1666  NE2 GLN A 225      -0.430  12.944   3.956  1.00  0.40           N  \\nATOM   1667  OE1 GLN A 225       1.725  12.337   3.744  1.00  0.48           O  \\nATOM   1668  N   ALA A 226       4.403  10.107   8.075  1.00  0.47           N  \\nATOM   1669  CA  ALA A 226       5.412   9.077   8.308  1.00  0.46           C  \\nATOM   1670  C   ALA A 226       5.163   8.354   9.629  1.00  0.47           C  \\nATOM   1671  CB  ALA A 226       6.810   9.690   8.296  1.00  0.45           C  \\nATOM   1672  O   ALA A 226       5.311   7.133   9.712  1.00  0.46           O  \\nATOM   1673  N   LYS A 227       4.821   9.108  10.718  1.00  0.49           N  \\nATOM   1674  CA  LYS A 227       4.494   8.453  11.981  1.00  0.48           C  \\nATOM   1675  C   LYS A 227       3.301   7.515  11.820  1.00  0.49           C  \\nATOM   1676  CB  LYS A 227       4.201   9.491  13.065  1.00  0.47           C  \\nATOM   1677  O   LYS A 227       3.306   6.400  12.347  1.00  0.48           O  \\nATOM   1678  CG  LYS A 227       5.445  10.158  13.635  1.00  0.45           C  \\nATOM   1679  CD  LYS A 227       5.092  11.148  14.737  1.00  0.46           C  \\nATOM   1680  CE  LYS A 227       6.333  11.834  15.292  1.00  0.41           C  \\nATOM   1681  NZ  LYS A 227       5.989  12.843  16.338  1.00  0.40           N  \\nATOM   1682  N   GLU A 228       2.255   8.081  11.146  1.00  0.51           N  \\nATOM   1683  CA  GLU A 228       1.076   7.260  10.888  1.00  0.51           C  \\nATOM   1684  C   GLU A 228       1.442   5.995  10.117  1.00  0.51           C  \\nATOM   1685  CB  GLU A 228       0.022   8.059  10.117  1.00  0.50           C  \\nATOM   1686  O   GLU A 228       1.015   4.896  10.478  1.00  0.51           O  \\nATOM   1687  CG  GLU A 228      -1.376   7.462  10.190  1.00  0.49           C  \\nATOM   1688  CD  GLU A 228      -2.467   8.438   9.778  1.00  0.51           C  \\nATOM   1689  OE1 GLU A 228      -2.148   9.609   9.468  1.00  0.51           O  \\nATOM   1690  OE2 GLU A 228      -3.649   8.030   9.767  1.00  0.47           O  \\nATOM   1691  N   ILE A 229       2.288   6.170   9.087  1.00  0.54           N  \\nATOM   1692  CA  ILE A 229       2.730   5.033   8.287  1.00  0.54           C  \\nATOM   1693  C   ILE A 229       3.494   4.047   9.169  1.00  0.55           C  \\nATOM   1694  CB  ILE A 229       3.612   5.485   7.101  1.00  0.53           C  \\nATOM   1695  O   ILE A 229       3.282   2.835   9.083  1.00  0.54           O  \\nATOM   1696  CG1 ILE A 229       2.775   6.267   6.082  1.00  0.49           C  \\nATOM   1697  CG2 ILE A 229       4.292   4.281   6.444  1.00  0.50           C  \\nATOM   1698  CD1 ILE A 229       3.601   6.981   5.021  1.00  0.49           C  \\nATOM   1699  N   GLN A 230       4.456   4.549  10.071  1.00  0.55           N  \\nATOM   1700  CA  GLN A 230       5.191   3.690  10.994  1.00  0.55           C  \\nATOM   1701  C   GLN A 230       4.240   2.940  11.923  1.00  0.56           C  \\nATOM   1702  CB  GLN A 230       6.187   4.512  11.815  1.00  0.54           C  \\nATOM   1703  O   GLN A 230       4.427   1.749  12.180  1.00  0.56           O  \\nATOM   1704  CG  GLN A 230       7.510   4.763  11.103  1.00  0.53           C  \\nATOM   1705  CD  GLN A 230       8.458   5.625  11.915  1.00  0.54           C  \\nATOM   1706  NE2 GLN A 230       9.662   5.836  11.393  1.00  0.45           N  \\nATOM   1707  OE1 GLN A 230       8.112   6.098  13.002  1.00  0.55           O  \\nATOM   1708  N   TRP A 231       3.264   3.622  12.526  1.00  0.39           N  \\nATOM   1709  CA  TRP A 231       2.270   2.985  13.383  1.00  0.46           C  \\nATOM   1710  C   TRP A 231       1.519   1.893  12.627  1.00  0.39           C  \\nATOM   1711  CB  TRP A 231       1.281   4.021  13.923  1.00  0.39           C  \\nATOM   1712  O   TRP A 231       1.385   0.770  13.118  1.00  0.39           O  \\nATOM   1713  CG  TRP A 231       0.353   3.489  14.973  1.00  0.38           C  \\nATOM   1714  CD1 TRP A 231       0.657   3.208  16.276  1.00  0.34           C  \\nATOM   1715  CD2 TRP A 231      -1.032   3.167  14.806  1.00  0.37           C  \\nATOM   1716  CE2 TRP A 231      -1.505   2.697  16.051  1.00  0.36           C  \\nATOM   1717  CE3 TRP A 231      -1.919   3.232  13.723  1.00  0.47           C  \\nATOM   1718  NE1 TRP A 231      -0.456   2.732  16.930  1.00  0.44           N  \\nATOM   1719  CH2 TRP A 231      -3.675   2.367  15.168  1.00  0.37           C  \\nATOM   1720  CZ2 TRP A 231      -2.828   2.294  16.243  1.00  0.43           C  \\nATOM   1721  CZ3 TRP A 231      -3.236   2.830  13.916  1.00  0.36           C  \\nATOM   1722  N   GLN A 232       1.057   2.170  11.379  1.00  0.54           N  \\nATOM   1723  CA  GLN A 232       0.341   1.188  10.571  1.00  0.54           C  \\nATOM   1724  C   GLN A 232       1.197  -0.052  10.328  1.00  0.54           C  \\nATOM   1725  CB  GLN A 232      -0.088   1.800   9.237  1.00  0.53           C  \\nATOM   1726  O   GLN A 232       0.716  -1.180  10.455  1.00  0.55           O  \\nATOM   1727  CG  GLN A 232      -1.228   2.802   9.359  1.00  0.52           C  \\nATOM   1728  CD  GLN A 232      -1.599   3.434   8.031  1.00  0.54           C  \\nATOM   1729  NE2 GLN A 232      -2.594   4.315   8.050  1.00  0.46           N  \\nATOM   1730  OE1 GLN A 232      -0.995   3.135   6.996  1.00  0.54           O  \\nATOM   1731  N   ALA A 233       2.525   0.132   9.941  1.00  0.55           N  \\nATOM   1732  CA  ALA A 233       3.436  -0.992   9.743  1.00  0.55           C  \\nATOM   1733  C   ALA A 233       3.541  -1.842  11.006  1.00  0.56           C  \\nATOM   1734  CB  ALA A 233       4.817  -0.491   9.327  1.00  0.54           C  \\nATOM   1735  O   ALA A 233       3.517  -3.074  10.936  1.00  0.57           O  \\nATOM   1736  N   LYS A 234       3.618  -1.189  12.197  1.00  0.57           N  \\nATOM   1737  CA  LYS A 234       3.684  -1.932  13.452  1.00  0.57           C  \\nATOM   1738  C   LYS A 234       2.397  -2.716  13.695  1.00  0.57           C  \\nATOM   1739  CB  LYS A 234       3.949  -0.985  14.623  1.00  0.55           C  \\nATOM   1740  O   LYS A 234       2.440  -3.877  14.107  1.00  0.58           O  \\nATOM   1741  CG  LYS A 234       5.402  -0.949  15.073  1.00  0.54           C  \\nATOM   1742  CD  LYS A 234       5.611   0.042  16.211  1.00  0.55           C  \\nATOM   1743  CE  LYS A 234       7.074   0.117  16.627  1.00  0.52           C  \\nATOM   1744  NZ  LYS A 234       7.284   1.092  17.739  1.00  0.48           N  \\nATOM   1745  N   GLU A 235       1.225  -2.063  13.520  1.00  0.56           N  \\nATOM   1746  CA  GLU A 235      -0.046  -2.763  13.677  1.00  0.56           C  \\nATOM   1747  C   GLU A 235      -0.117  -3.989  12.771  1.00  0.55           C  \\nATOM   1748  CB  GLU A 235      -1.218  -1.823  13.381  1.00  0.55           C  \\nATOM   1749  O   GLU A 235      -0.518  -5.069  13.210  1.00  0.56           O  \\nATOM   1750  CG  GLU A 235      -2.555  -2.321  13.910  1.00  0.54           C  \\nATOM   1751  CD  GLU A 235      -3.646  -1.262  13.875  1.00  0.55           C  \\nATOM   1752  OE1 GLU A 235      -3.335  -0.079  13.611  1.00  0.55           O  \\nATOM   1753  OE2 GLU A 235      -4.821  -1.620  14.115  1.00  0.51           O  \\nATOM   1754  N   ILE A 236       0.298  -3.847  11.523  1.00  0.57           N  \\nATOM   1755  CA  ILE A 236       0.318  -4.954  10.572  1.00  0.56           C  \\nATOM   1756  C   ILE A 236       1.208  -6.074  11.105  1.00  0.57           C  \\nATOM   1757  CB  ILE A 236       0.808  -4.495   9.181  1.00  0.55           C  \\nATOM   1758  O   ILE A 236       0.834  -7.248  11.058  1.00  0.56           O  \\nATOM   1759  CG1 ILE A 236      -0.224  -3.566   8.531  1.00  0.51           C  \\nATOM   1760  CG2 ILE A 236       1.099  -5.703   8.285  1.00  0.53           C  \\nATOM   1761  CD1 ILE A 236       0.297  -2.821   7.309  1.00  0.52           C  \\nATOM   1762  N   GLN A 237       2.336  -5.772  11.649  1.00  0.58           N  \\nATOM   1763  CA  GLN A 237       3.244  -6.762  12.217  1.00  0.59           C  \\nATOM   1764  C   GLN A 237       2.599  -7.491  13.393  1.00  0.59           C  \\nATOM   1765  CB  GLN A 237       4.550  -6.102  12.661  1.00  0.57           C  \\nATOM   1766  O   GLN A 237       2.731  -8.709  13.522  1.00  0.60           O  \\nATOM   1767  CG  GLN A 237       5.540  -5.872  11.528  1.00  0.56           C  \\nATOM   1768  CD  GLN A 237       6.789  -5.138  11.978  1.00  0.57           C  \\nATOM   1769  NE2 GLN A 237       7.746  -4.982  11.070  1.00  0.50           N  \\nATOM   1770  OE1 GLN A 237       6.892  -4.713  13.134  1.00  0.60           O  \\nATOM   1771  N   GLU A 238       1.916  -6.727  14.273  1.00  0.58           N  \\nATOM   1772  CA  GLU A 238       1.272  -7.354  15.423  1.00  0.58           C  \\nATOM   1773  C   GLU A 238       0.165  -8.309  14.983  1.00  0.57           C  \\nATOM   1774  CB  GLU A 238       0.705  -6.292  16.369  1.00  0.56           C  \\nATOM   1775  O   GLU A 238       0.030  -9.405  15.530  1.00  0.57           O  \\nATOM   1776  CG  GLU A 238       1.763  -5.579  17.198  1.00  0.54           C  \\nATOM   1777  CD  GLU A 238       2.501  -6.504  18.153  1.00  0.56           C  \\nATOM   1778  OE1 GLU A 238       1.900  -7.498  18.620  1.00  0.58           O  \\nATOM   1779  OE2 GLU A 238       3.689  -6.231  18.437  1.00  0.55           O  \\nATOM   1780  N   GLN A 239      -0.502  -8.000  13.958  1.00  0.56           N  \\nATOM   1781  CA  GLN A 239      -1.554  -8.848  13.409  1.00  0.56           C  \\nATOM   1782  C   GLN A 239      -0.971 -10.106  12.772  1.00  0.55           C  \\nATOM   1783  CB  GLN A 239      -2.385  -8.077  12.382  1.00  0.54           C  \\nATOM   1784  O   GLN A 239      -1.618 -11.155  12.752  1.00  0.55           O  \\nATOM   1785  CG  GLN A 239      -3.286  -7.013  12.994  1.00  0.54           C  \\nATOM   1786  CD  GLN A 239      -4.201  -6.361  11.975  1.00  0.55           C  \\nATOM   1787  NE2 GLN A 239      -4.882  -5.296  12.384  1.00  0.48           N  \\nATOM   1788  OE1 GLN A 239      -4.294  -6.810  10.828  1.00  0.56           O  \\nATOM   1789  N   ALA A 240       0.225  -9.973  12.238  1.00  0.55           N  \\nATOM   1790  CA  ALA A 240       0.901 -11.140  11.678  1.00  0.55           C  \\nATOM   1791  C   ALA A 240       1.153 -12.195  12.752  1.00  0.56           C  \\nATOM   1792  CB  ALA A 240       2.216 -10.730  11.021  1.00  0.52           C  \\nATOM   1793  O   ALA A 240       1.223 -13.390  12.453  1.00  0.56           O  \\nATOM   1794  N   LYS A 241       1.368 -11.821  14.042  1.00  0.53           N  \\nATOM   1795  CA  LYS A 241       1.817 -12.713  15.108  1.00  0.52           C  \\nATOM   1796  C   LYS A 241       0.768 -13.778  15.412  1.00  0.52           C  \\nATOM   1797  CB  LYS A 241       2.139 -11.918  16.374  1.00  0.49           C  \\nATOM   1798  O   LYS A 241       1.089 -14.834  15.961  1.00  0.50           O  \\nATOM   1799  CG  LYS A 241       3.436 -11.125  16.294  1.00  0.47           C  \\nATOM   1800  CD  LYS A 241       3.687 -10.335  17.571  1.00  0.48           C  \\nATOM   1801  CE  LYS A 241       4.957  -9.500  17.472  1.00  0.52           C  \\nATOM   1802  NZ  LYS A 241       5.115  -8.590  18.646  1.00  0.47           N  \\nATOM   1803  N   GLY A 242      -0.370 -13.983  14.742  1.00  0.49           N  \\nATOM   1804  CA  GLY A 242      -1.403 -14.986  14.948  1.00  0.49           C  \\nATOM   1805  C   GLY A 242      -1.790 -15.713  13.675  1.00  0.49           C  \\nATOM   1806  O   GLY A 242      -2.646 -16.601  13.697  1.00  0.48           O  \\nATOM   1807  N   LYS A 243      -1.108 -15.451  12.601  1.00  0.50           N  \\nATOM   1808  CA  LYS A 243      -1.477 -15.998  11.298  1.00  0.53           C  \\nATOM   1809  C   LYS A 243      -0.603 -17.195  10.937  1.00  0.52           C  \\nATOM   1810  CB  LYS A 243      -1.369 -14.924  10.215  1.00  0.49           C  \\nATOM   1811  O   LYS A 243       0.494 -17.354  11.474  1.00  0.53           O  \\nATOM   1812  CG  LYS A 243      -2.371 -13.789  10.365  1.00  0.47           C  \\nATOM   1813  CD  LYS A 243      -2.743 -13.189   9.015  1.00  0.53           C  \\nATOM   1814  CE  LYS A 243      -3.841 -12.143   9.150  1.00  0.44           C  \\nATOM   1815  NZ  LYS A 243      -4.213 -11.557   7.828  1.00  0.41           N  \\nATOM   1816  N   PRO A 244      -1.173 -18.180  10.166  1.00  0.48           N  \\nATOM   1817  CA  PRO A 244      -0.308 -19.234   9.632  1.00  0.47           C  \\nATOM   1818  C   PRO A 244       0.980 -18.688   9.020  1.00  0.48           C  \\nATOM   1819  CB  PRO A 244      -1.180 -19.900   8.565  1.00  0.43           C  \\nATOM   1820  O   PRO A 244       0.967 -17.624   8.395  1.00  0.47           O  \\nATOM   1821  CG  PRO A 244      -2.576 -19.478   8.895  1.00  0.41           C  \\nATOM   1822  CD  PRO A 244      -2.518 -18.168   9.626  1.00  0.44           C  \\nATOM   1823  N   SER A 245       2.236 -19.327   9.159  1.00  0.43           N  \\nATOM   1824  CA  SER A 245       3.605 -18.885   8.915  1.00  0.45           C  \\nATOM   1825  C   SER A 245       3.736 -18.218   7.550  1.00  0.43           C  \\nATOM   1826  CB  SER A 245       4.575 -20.064   9.010  1.00  0.39           C  \\nATOM   1827  O   SER A 245       4.388 -17.180   7.420  1.00  0.42           O  \\nATOM   1828  OG  SER A 245       4.263 -21.050   8.042  1.00  0.34           O  \\nATOM   1829  N   GLY A 246       3.184 -18.843   6.490  1.00  0.48           N  \\nATOM   1830  CA  GLY A 246       3.319 -18.263   5.164  1.00  0.47           C  \\nATOM   1831  C   GLY A 246       2.684 -16.891   5.045  1.00  0.47           C  \\nATOM   1832  O   GLY A 246       3.299 -15.959   4.523  1.00  0.46           O  \\nATOM   1833  N   GLN A 247       1.483 -16.749   5.553  1.00  0.47           N  \\nATOM   1834  CA  GLN A 247       0.789 -15.465   5.569  1.00  0.48           C  \\nATOM   1835  C   GLN A 247       1.502 -14.465   6.474  1.00  0.47           C  \\nATOM   1836  CB  GLN A 247      -0.661 -15.644   6.023  1.00  0.46           C  \\nATOM   1837  O   GLN A 247       1.678 -13.302   6.106  1.00  0.47           O  \\nATOM   1838  CG  GLN A 247      -1.568 -16.247   4.960  1.00  0.45           C  \\nATOM   1839  CD  GLN A 247      -3.008 -16.374   5.421  1.00  0.49           C  \\nATOM   1840  NE2 GLN A 247      -3.827 -17.056   4.627  1.00  0.42           N  \\nATOM   1841  OE1 GLN A 247      -3.381 -15.865   6.483  1.00  0.52           O  \\nATOM   1842  N   ALA A 248       1.972 -14.924   7.648  1.00  0.49           N  \\nATOM   1843  CA  ALA A 248       2.682 -14.075   8.600  1.00  0.49           C  \\nATOM   1844  C   ALA A 248       3.938 -13.477   7.972  1.00  0.50           C  \\nATOM   1845  CB  ALA A 248       3.044 -14.868   9.854  1.00  0.46           C  \\nATOM   1846  O   ALA A 248       4.210 -12.284   8.128  1.00  0.49           O  \\nATOM   1847  N   LYS A 249       4.712 -14.321   7.220  1.00  0.48           N  \\nATOM   1848  CA  LYS A 249       5.949 -13.858   6.597  1.00  0.48           C  \\nATOM   1849  C   LYS A 249       5.668 -12.784   5.550  1.00  0.48           C  \\nATOM   1850  CB  LYS A 249       6.699 -15.028   5.959  1.00  0.45           C  \\nATOM   1851  O   LYS A 249       6.380 -11.781   5.476  1.00  0.47           O  \\nATOM   1852  CG  LYS A 249       7.450 -15.899   6.955  1.00  0.43           C  \\nATOM   1853  CD  LYS A 249       8.171 -17.048   6.261  1.00  0.44           C  \\nATOM   1854  CE  LYS A 249       8.904 -17.933   7.259  1.00  0.43           C  \\nATOM   1855  NZ  LYS A 249       9.590 -19.077   6.586  1.00  0.42           N  \\nATOM   1856  N   GLU A 250       4.615 -13.087   4.714  1.00  0.52           N  \\nATOM   1857  CA  GLU A 250       4.215 -12.137   3.680  1.00  0.52           C  \\nATOM   1858  C   GLU A 250       3.832 -10.790   4.286  1.00  0.52           C  \\nATOM   1859  CB  GLU A 250       3.050 -12.695   2.859  1.00  0.50           C  \\nATOM   1860  O   GLU A 250       4.295  -9.743   3.829  1.00  0.52           O  \\nATOM   1861  CG  GLU A 250       2.698 -11.853   1.641  1.00  0.49           C  \\nATOM   1862  CD  GLU A 250       2.023 -12.650   0.536  1.00  0.52           C  \\nATOM   1863  OE1 GLU A 250       1.810 -13.871   0.711  1.00  0.53           O  \\nATOM   1864  OE2 GLU A 250       1.707 -12.049  -0.515  1.00  0.50           O  \\nATOM   1865  N   ILE A 251       3.057 -10.815   5.351  1.00  0.55           N  \\nATOM   1866  CA  ILE A 251       2.579  -9.630   6.053  1.00  0.55           C  \\nATOM   1867  C   ILE A 251       3.761  -8.871   6.651  1.00  0.55           C  \\nATOM   1868  CB  ILE A 251       1.565  -9.999   7.159  1.00  0.53           C  \\nATOM   1869  O   ILE A 251       3.830  -7.643   6.559  1.00  0.55           O  \\nATOM   1870  CG1 ILE A 251       0.270 -10.537   6.538  1.00  0.48           C  \\nATOM   1871  CG2 ILE A 251       1.281  -8.792   8.058  1.00  0.50           C  \\nATOM   1872  CD1 ILE A 251      -0.675 -11.184   7.541  1.00  0.49           C  \\nATOM   1873  N   GLN A 252       4.746  -9.613   7.298  1.00  0.54           N  \\nATOM   1874  CA  GLN A 252       5.946  -9.022   7.879  1.00  0.54           C  \\nATOM   1875  C   GLN A 252       6.790  -8.328   6.814  1.00  0.54           C  \\nATOM   1876  CB  GLN A 252       6.778 -10.088   8.594  1.00  0.52           C  \\nATOM   1877  O   GLN A 252       7.310  -7.234   7.042  1.00  0.54           O  \\nATOM   1878  CG  GLN A 252       6.289 -10.409  10.000  1.00  0.50           C  \\nATOM   1879  CD  GLN A 252       7.071 -11.534  10.652  1.00  0.51           C  \\nATOM   1880  NE2 GLN A 252       6.793 -11.790  11.925  1.00  0.46           N  \\nATOM   1881  OE1 GLN A 252       7.919 -12.168  10.015  1.00  0.56           O  \\nATOM   1882  N   TRP A 253       7.021  -8.989   5.660  1.00  0.42           N  \\nATOM   1883  CA  TRP A 253       7.764  -8.401   4.550  1.00  0.48           C  \\nATOM   1884  C   TRP A 253       7.119  -7.096   4.094  1.00  0.41           C  \\nATOM   1885  CB  TRP A 253       7.845  -9.381   3.377  1.00  0.40           C  \\nATOM   1886  O   TRP A 253       7.798  -6.075   3.958  1.00  0.41           O  \\nATOM   1887  CG  TRP A 253       8.696  -8.901   2.240  1.00  0.39           C  \\nATOM   1888  CD1 TRP A 253      10.061  -8.914   2.169  1.00  0.35           C  \\nATOM   1889  CD2 TRP A 253       8.236  -8.332   1.010  1.00  0.38           C  \\nATOM   1890  CE2 TRP A 253       9.378  -8.022   0.240  1.00  0.37           C  \\nATOM   1891  CE3 TRP A 253       6.967  -8.053   0.485  1.00  0.49           C  \\nATOM   1892  NE1 TRP A 253      10.478  -8.387   0.968  1.00  0.49           N  \\nATOM   1893  CH2 TRP A 253       8.036  -7.186  -1.521  1.00  0.44           C  \\nATOM   1894  CZ2 TRP A 253       9.289  -7.448  -1.031  1.00  0.55           C  \\nATOM   1895  CZ3 TRP A 253       6.880  -7.481  -0.779  1.00  0.39           C  \\nATOM   1896  N   GLN A 254       5.878  -7.045   3.913  1.00  0.54           N  \\nATOM   1897  CA  GLN A 254       5.146  -5.868   3.457  1.00  0.54           C  \\nATOM   1898  C   GLN A 254       5.292  -4.713   4.443  1.00  0.54           C  \\nATOM   1899  CB  GLN A 254       3.667  -6.201   3.253  1.00  0.53           C  \\nATOM   1900  O   GLN A 254       5.525  -3.572   4.041  1.00  0.54           O  \\nATOM   1901  CG  GLN A 254       3.393  -7.056   2.023  1.00  0.52           C  \\nATOM   1902  CD  GLN A 254       1.927  -7.417   1.874  1.00  0.54           C  \\nATOM   1903  NE2 GLN A 254       1.601  -8.147   0.812  1.00  0.47           N  \\nATOM   1904  OE1 GLN A 254       1.095  -7.044   2.706  1.00  0.55           O  \\nATOM   1905  N   ALA A 255       5.092  -4.980   5.822  1.00  0.55           N  \\nATOM   1906  CA  ALA A 255       5.286  -3.980   6.869  1.00  0.55           C  \\nATOM   1907  C   ALA A 255       6.681  -3.365   6.788  1.00  0.56           C  \\nATOM   1908  CB  ALA A 255       5.060  -4.601   8.245  1.00  0.53           C  \\nATOM   1909  O   ALA A 255       6.838  -2.150   6.923  1.00  0.55           O  \\nATOM   1910  N   LYS A 256       7.709  -4.219   6.530  1.00  0.56           N  \\nATOM   1911  CA  LYS A 256       9.073  -3.718   6.392  1.00  0.55           C  \\nATOM   1912  C   LYS A 256       9.203  -2.803   5.178  1.00  0.56           C  \\nATOM   1913  CB  LYS A 256      10.063  -4.879   6.281  1.00  0.54           C  \\nATOM   1914  O   LYS A 256       9.839  -1.750   5.253  1.00  0.55           O  \\nATOM   1915  CG  LYS A 256      11.483  -4.524   6.699  1.00  0.52           C  \\nATOM   1916  CD  LYS A 256      12.406  -5.733   6.624  1.00  0.53           C  \\nATOM   1917  CE  LYS A 256      13.842  -5.364   6.973  1.00  0.52           C  \\nATOM   1918  NZ  LYS A 256      14.763  -6.532   6.837  1.00  0.48           N  \\nATOM   1919  N   GLU A 257       8.693  -3.217   3.993  1.00  0.54           N  \\nATOM   1920  CA  GLU A 257       8.714  -2.364   2.808  1.00  0.54           C  \\nATOM   1921  C   GLU A 257       8.079  -1.006   3.095  1.00  0.53           C  \\nATOM   1922  CB  GLU A 257       7.994  -3.045   1.642  1.00  0.53           C  \\nATOM   1923  O   GLU A 257       8.636   0.033   2.736  1.00  0.53           O  \\nATOM   1924  CG  GLU A 257       8.693  -2.871   0.301  1.00  0.53           C  \\nATOM   1925  CD  GLU A 257      10.110  -2.336   0.429  1.00  0.54           C  \\nATOM   1926  OE1 GLU A 257      10.446  -1.753   1.485  1.00  0.54           O  \\nATOM   1927  OE2 GLU A 257      10.891  -2.501  -0.535  1.00  0.51           O  \\nATOM   1928  N   ILE A 258       6.923  -1.020   3.769  1.00  0.55           N  \\nATOM   1929  CA  ILE A 258       6.203   0.192   4.143  1.00  0.55           C  \\nATOM   1930  C   ILE A 258       7.100   1.081   5.002  1.00  0.55           C  \\nATOM   1931  CB  ILE A 258       4.895  -0.137   4.896  1.00  0.53           C  \\nATOM   1932  O   ILE A 258       7.180   2.291   4.781  1.00  0.54           O  \\nATOM   1933  CG1 ILE A 258       3.894  -0.819   3.956  1.00  0.50           C  \\nATOM   1934  CG2 ILE A 258       4.292   1.130   5.511  1.00  0.52           C  \\nATOM   1935  CD1 ILE A 258       2.711  -1.459   4.670  1.00  0.51           C  \\nATOM   1936  N   GLN A 259       7.833   0.534   5.999  1.00  0.57           N  \\nATOM   1937  CA  GLN A 259       8.740   1.266   6.877  1.00  0.57           C  \\nATOM   1938  C   GLN A 259       9.883   1.894   6.085  1.00  0.57           C  \\nATOM   1939  CB  GLN A 259       9.297   0.344   7.963  1.00  0.56           C  \\nATOM   1940  O   GLN A 259      10.256   3.043   6.329  1.00  0.57           O  \\nATOM   1941  CG  GLN A 259       8.387   0.201   9.175  1.00  0.55           C  \\nATOM   1942  CD  GLN A 259       8.939  -0.756  10.214  1.00  0.55           C  \\nATOM   1943  NE2 GLN A 259       8.230  -0.895  11.329  1.00  0.48           N  \\nATOM   1944  OE1 GLN A 259       9.994  -1.366  10.016  1.00  0.57           O  \\nATOM   1945  N   GLU A 260      10.477   1.096   5.162  1.00  0.56           N  \\nATOM   1946  CA  GLU A 260      11.605   1.597   4.383  1.00  0.55           C  \\nATOM   1947  C   GLU A 260      11.184   2.754   3.482  1.00  0.55           C  \\nATOM   1948  CB  GLU A 260      12.220   0.474   3.544  1.00  0.54           C  \\nATOM   1949  O   GLU A 260      11.893   3.758   3.380  1.00  0.54           O  \\nATOM   1950  CG  GLU A 260      13.101  -0.478   4.340  1.00  0.52           C  \\nATOM   1951  CD  GLU A 260      14.536  -0.527   3.840  1.00  0.53           C  \\nATOM   1952  OE1 GLU A 260      14.795  -0.092   2.695  1.00  0.55           O  \\nATOM   1953  OE2 GLU A 260      15.408  -1.005   4.600  1.00  0.52           O  \\nATOM   1954  N   GLN A 261      10.029   2.677   2.985  1.00  0.51           N  \\nATOM   1955  CA  GLN A 261       9.462   3.743   2.164  1.00  0.51           C  \\nATOM   1956  C   GLN A 261       9.163   4.983   3.002  1.00  0.50           C  \\nATOM   1957  CB  GLN A 261       8.189   3.263   1.465  1.00  0.50           C  \\nATOM   1958  O   GLN A 261       9.422   6.108   2.570  1.00  0.50           O  \\nATOM   1959  CG  GLN A 261       8.441   2.258   0.350  1.00  0.50           C  \\nATOM   1960  CD  GLN A 261       7.208   1.994  -0.494  1.00  0.52           C  \\nATOM   1961  NE2 GLN A 261       7.324   1.071  -1.442  1.00  0.44           N  \\nATOM   1962  OE1 GLN A 261       6.159   2.615  -0.293  1.00  0.52           O  \\nATOM   1963  N   ALA A 262       8.500   4.885   4.178  1.00  0.50           N  \\nATOM   1964  CA  ALA A 262       8.241   5.991   5.096  1.00  0.49           C  \\nATOM   1965  C   ALA A 262       9.530   6.729   5.443  1.00  0.49           C  \\nATOM   1966  CB  ALA A 262       7.565   5.480   6.366  1.00  0.47           C  \\nATOM   1967  O   ALA A 262       9.538   7.957   5.562  1.00  0.48           O  \\nATOM   1968  N   LYS A 263      10.656   5.986   5.623  1.00  0.50           N  \\nATOM   1969  CA  LYS A 263      11.964   6.580   5.884  1.00  0.49           C  \\nATOM   1970  C   LYS A 263      12.401   7.477   4.729  1.00  0.49           C  \\nATOM   1971  CB  LYS A 263      13.010   5.491   6.126  1.00  0.48           C  \\nATOM   1972  O   LYS A 263      13.072   8.489   4.942  1.00  0.48           O  \\nATOM   1973  CG  LYS A 263      12.892   4.810   7.481  1.00  0.46           C  \\nATOM   1974  CD  LYS A 263      13.882   3.660   7.617  1.00  0.47           C  \\nATOM   1975  CE  LYS A 263      13.673   2.891   8.914  1.00  0.45           C  \\nATOM   1976  NZ  LYS A 263      14.524   1.665   8.975  1.00  0.44           N  \\nATOM   1977  N   GLY A 264      11.947   7.151   3.526  1.00  0.41           N  \\nATOM   1978  CA  GLY A 264      12.369   7.852   2.324  1.00  0.40           C  \\nATOM   1979  C   GLY A 264      11.647   9.170   2.118  1.00  0.41           C  \\nATOM   1980  O   GLY A 264      11.951   9.911   1.181  1.00  0.39           O  \\nATOM   1981  N   GLY A 265      10.821   9.661   3.203  1.00  0.39           N  \\nATOM   1982  CA  GLY A 265      10.077  10.906   3.093  1.00  0.39           C  \\nATOM   1983  C   GLY A 265       9.096  10.914   1.935  1.00  0.39           C  \\nATOM   1984  O   GLY A 265       8.774  11.974   1.395  1.00  0.37           O  \\nATOM   1985  N   ILE A 266       8.719   9.764   1.305  1.00  0.37           N  \\nATOM   1986  CA  ILE A 266       7.868   9.669   0.124  1.00  0.36           C  \\nATOM   1987  C   ILE A 266       6.400   9.761   0.536  1.00  0.37           C  \\nATOM   1988  CB  ILE A 266       8.125   8.360  -0.655  1.00  0.34           C  \\nATOM   1989  O   ILE A 266       5.895   8.893   1.253  1.00  0.35           O  \\nATOM   1990  CG1 ILE A 266       9.599   8.263  -1.065  1.00  0.33           C  \\nATOM   1991  CG2 ILE A 266       7.209   8.271  -1.879  1.00  0.33           C  \\nATOM   1992  CD1 ILE A 266       9.969   6.950  -1.741  1.00  0.30           C  \\nATOM   1993  N   LYS A 267       5.891  10.978   0.877  1.00  0.37           N  \\nATOM   1994  CA  LYS A 267       4.511  11.367   1.158  1.00  0.37           C  \\nATOM   1995  C   LYS A 267       3.528  10.536   0.338  1.00  0.36           C  \\nATOM   1996  CB  LYS A 267       4.304  12.855   0.873  1.00  0.34           C  \\nATOM   1997  O   LYS A 267       2.442  10.202   0.815  1.00  0.34           O  \\nATOM   1998  CG  LYS A 267       4.899  13.776   1.928  1.00  0.32           C  \\nATOM   1999  CD  LYS A 267       4.567  15.236   1.647  1.00  0.34           C  \\nATOM   2000  CE  LYS A 267       5.178  16.160   2.692  1.00  0.31           C  \\nATOM   2001  NZ  LYS A 267       4.852  17.593   2.423  1.00  0.32           N  \\nATOM   2002  N   GLU A 268       3.886  10.190  -0.927  1.00  0.33           N  \\nATOM   2003  CA  GLU A 268       2.954   9.816  -1.986  1.00  0.34           C  \\nATOM   2004  C   GLU A 268       2.730   8.307  -2.017  1.00  0.33           C  \\nATOM   2005  CB  GLU A 268       3.464  10.300  -3.347  1.00  0.31           C  \\nATOM   2006  O   GLU A 268       1.645   7.842  -2.374  1.00  0.31           O  \\nATOM   2007  CG  GLU A 268       3.385  11.808  -3.532  1.00  0.29           C  \\nATOM   2008  CD  GLU A 268       4.050  12.291  -4.811  1.00  0.34           C  \\nATOM   2009  OE1 GLU A 268       4.594  11.453  -5.566  1.00  0.37           O  \\nATOM   2010  OE2 GLU A 268       4.027  13.517  -5.060  1.00  0.37           O  \\nATOM   2011  N   GLU A 269       3.595   7.468  -1.363  1.00  0.33           N  \\nATOM   2012  CA  GLU A 269       3.516   6.017  -1.224  1.00  0.34           C  \\nATOM   2013  C   GLU A 269       2.674   5.623  -0.014  1.00  0.33           C  \\nATOM   2014  CB  GLU A 269       4.917   5.410  -1.111  1.00  0.32           C  \\nATOM   2015  O   GLU A 269       2.130   4.518   0.036  1.00  0.32           O  \\nATOM   2016  CG  GLU A 269       5.604   5.194  -2.452  1.00  0.30           C  \\nATOM   2017  CD  GLU A 269       7.018   4.650  -2.320  1.00  0.35           C  \\nATOM   2018  OE1 GLU A 269       7.467   4.395  -1.180  1.00  0.40           O  \\nATOM   2019  OE2 GLU A 269       7.682   4.479  -3.367  1.00  0.37           O  \\nATOM   2020  N   ALA A 270       2.270   6.624   0.815  1.00  0.35           N  \\nATOM   2021  CA  ALA A 270       1.403   6.364   1.962  1.00  0.34           C  \\nATOM   2022  C   ALA A 270       0.042   5.840   1.513  1.00  0.35           C  \\nATOM   2023  CB  ALA A 270       1.233   7.629   2.799  1.00  0.33           C  \\nATOM   2024  O   ALA A 270      -0.526   4.946   2.145  1.00  0.33           O  \\nATOM   2025  N   ALA A 271      -0.509   6.262   0.258  1.00  0.35           N  \\nATOM   2026  CA  ALA A 271      -1.797   5.766  -0.220  1.00  0.34           C  \\nATOM   2027  C   ALA A 271      -1.694   4.309  -0.660  1.00  0.35           C  \\nATOM   2028  CB  ALA A 271      -2.306   6.631  -1.371  1.00  0.33           C  \\nATOM   2029  O   ALA A 271      -2.582   3.502  -0.372  1.00  0.33           O  \\nATOM   2030  N   ALA A 272      -0.593   3.910  -1.348  1.00  0.36           N  \\nATOM   2031  CA  ALA A 272      -0.424   2.514  -1.742  1.00  0.35           C  \\nATOM   2032  C   ALA A 272      -0.285   1.612  -0.519  1.00  0.36           C  \\nATOM   2033  CB  ALA A 272       0.791   2.361  -2.653  1.00  0.34           C  \\nATOM   2034  O   ALA A 272      -0.850   0.516  -0.481  1.00  0.34           O  \\nATOM   2035  N   ILE A 273       0.359   2.002   0.578  1.00  0.36           N  \\nATOM   2036  CA  ILE A 273       0.625   1.247   1.797  1.00  0.36           C  \\nATOM   2037  C   ILE A 273      -0.687   0.970   2.528  1.00  0.36           C  \\nATOM   2038  CB  ILE A 273       1.607   1.997   2.724  1.00  0.34           C  \\nATOM   2039  O   ILE A 273      -0.921  -0.148   2.994  1.00  0.35           O  \\nATOM   2040  CG1 ILE A 273       2.988   2.102   2.066  1.00  0.31           C  \\nATOM   2041  CG2 ILE A 273       1.701   1.303   4.086  1.00  0.32           C  \\nATOM   2042  CD1 ILE A 273       3.942   3.051   2.778  1.00  0.29           C  \\nATOM   2043  N   LYS A 274      -1.593   1.956   2.661  1.00  0.36           N  \\nATOM   2044  CA  LYS A 274      -2.878   1.721   3.313  1.00  0.36           C  \\nATOM   2045  C   LYS A 274      -3.666   0.628   2.596  1.00  0.36           C  \\nATOM   2046  CB  LYS A 274      -3.699   3.011   3.363  1.00  0.35           C  \\nATOM   2047  O   LYS A 274      -4.362  -0.164   3.235  1.00  0.35           O  \\nATOM   2048  CG  LYS A 274      -3.340   3.927   4.524  1.00  0.33           C  \\nATOM   2049  CD  LYS A 274      -4.211   5.177   4.539  1.00  0.37           C  \\nATOM   2050  CE  LYS A 274      -3.829   6.112   5.678  1.00  0.32           C  \\nATOM   2051  NZ  LYS A 274      -4.651   7.359   5.671  1.00  0.33           N  \\nATOM   2052  N   TRP A 275      -3.485   0.502   1.228  1.00  0.23           N  \\nATOM   2053  CA  TRP A 275      -4.186  -0.490   0.420  1.00  0.32           C  \\nATOM   2054  C   TRP A 275      -3.751  -1.903   0.794  1.00  0.24           C  \\nATOM   2055  CB  TRP A 275      -3.936  -0.244  -1.071  1.00  0.25           C  \\nATOM   2056  O   TRP A 275      -4.589  -2.787   0.987  1.00  0.24           O  \\nATOM   2057  CG  TRP A 275      -4.912   0.703  -1.703  1.00  0.25           C  \\nATOM   2058  CD1 TRP A 275      -4.708   2.022  -1.996  1.00  0.25           C  \\nATOM   2059  CD2 TRP A 275      -6.249   0.402  -2.115  1.00  0.25           C  \\nATOM   2060  CE2 TRP A 275      -6.798   1.587  -2.652  1.00  0.14           C  \\nATOM   2061  CE3 TRP A 275      -7.036  -0.757  -2.084  1.00  0.22           C  \\nATOM   2062  NE1 TRP A 275      -5.839   2.560  -2.568  1.00  0.22           N  \\nATOM   2063  CH2 TRP A 275      -8.849   0.499  -3.111  1.00  0.22           C  \\nATOM   2064  CZ2 TRP A 275      -8.100   1.647  -3.154  1.00  0.21           C  \\nATOM   2065  CZ3 TRP A 275      -8.332  -0.696  -2.584  1.00  0.22           C  \\nATOM   2066  N   GLU A 276      -2.485  -2.164   0.907  1.00  0.36           N  \\nATOM   2067  CA  GLU A 276      -1.924  -3.479   1.202  1.00  0.37           C  \\nATOM   2068  C   GLU A 276      -2.291  -3.932   2.612  1.00  0.36           C  \\nATOM   2069  CB  GLU A 276      -0.403  -3.465   1.033  1.00  0.35           C  \\nATOM   2070  O   GLU A 276      -2.698  -5.078   2.816  1.00  0.36           O  \\nATOM   2071  CG  GLU A 276       0.101  -4.383  -0.072  1.00  0.34           C  \\nATOM   2072  CD  GLU A 276       1.609  -4.322  -0.261  1.00  0.41           C  \\nATOM   2073  OE1 GLU A 276       2.280  -3.551   0.461  1.00  0.45           O  \\nATOM   2074  OE2 GLU A 276       2.122  -5.050  -1.140  1.00  0.42           O  \\nATOM   2075  N   ALA A 277      -2.302  -3.039   3.564  1.00  0.36           N  \\nATOM   2076  CA  ALA A 277      -2.669  -3.352   4.943  1.00  0.35           C  \\nATOM   2077  C   ALA A 277      -4.136  -3.760   5.041  1.00  0.36           C  \\nATOM   2078  CB  ALA A 277      -2.390  -2.158   5.852  1.00  0.34           C  \\nATOM   2079  O   ALA A 277      -4.478  -4.701   5.761  1.00  0.34           O  \\nATOM   2080  N   ALA A 278      -5.145  -3.171   4.260  1.00  0.35           N  \\nATOM   2081  CA  ALA A 278      -6.554  -3.558   4.248  1.00  0.35           C  \\nATOM   2082  C   ALA A 278      -6.734  -4.952   3.653  1.00  0.35           C  \\nATOM   2083  CB  ALA A 278      -7.379  -2.539   3.468  1.00  0.34           C  \\nATOM   2084  O   ALA A 278      -7.468  -5.777   4.201  1.00  0.34           O  \\nATOM   2085  N   ALA A 279      -6.091  -5.301   2.547  1.00  0.37           N  \\nATOM   2086  CA  ALA A 279      -6.197  -6.631   1.953  1.00  0.37           C  \\nATOM   2087  C   ALA A 279      -5.635  -7.696   2.889  1.00  0.37           C  \\nATOM   2088  CB  ALA A 279      -5.474  -6.673   0.609  1.00  0.35           C  \\nATOM   2089  O   ALA A 279      -6.211  -8.778   3.027  1.00  0.35           O  \\nATOM   2090  N   ILE A 280      -4.592  -7.430   3.616  1.00  0.37           N  \\nATOM   2091  CA  ILE A 280      -3.914  -8.343   4.531  1.00  0.37           C  \\nATOM   2092  C   ILE A 280      -4.759  -8.536   5.788  1.00  0.37           C  \\nATOM   2093  CB  ILE A 280      -2.507  -7.825   4.905  1.00  0.35           C  \\nATOM   2094  O   ILE A 280      -4.939  -9.662   6.256  1.00  0.36           O  \\nATOM   2095  CG1 ILE A 280      -1.586  -7.849   3.680  1.00  0.33           C  \\nATOM   2096  CG2 ILE A 280      -1.915  -8.652   6.051  1.00  0.32           C  \\nATOM   2097  CD1 ILE A 280      -0.263  -7.125   3.886  1.00  0.30           C  \\nATOM   2098  N   LYS A 281      -5.316  -7.510   6.385  1.00  0.37           N  \\nATOM   2099  CA  LYS A 281      -6.065  -7.561   7.637  1.00  0.37           C  \\nATOM   2100  C   LYS A 281      -7.332  -8.399   7.486  1.00  0.37           C  \\nATOM   2101  CB  LYS A 281      -6.423  -6.150   8.106  1.00  0.35           C  \\nATOM   2102  O   LYS A 281      -7.743  -9.085   8.424  1.00  0.36           O  \\nATOM   2103  CG  LYS A 281      -5.413  -5.541   9.067  1.00  0.34           C  \\nATOM   2104  CD  LYS A 281      -5.823  -4.139   9.496  1.00  0.40           C  \\nATOM   2105  CE  LYS A 281      -4.790  -3.511  10.421  1.00  0.33           C  \\nATOM   2106  NZ  LYS A 281      -5.171  -2.122  10.817  1.00  0.34           N  \\nATOM   2107  N   GLU A 282      -7.984  -8.450   6.300  1.00  0.34           N  \\nATOM   2108  CA  GLU A 282      -9.237  -9.182   6.147  1.00  0.34           C  \\nATOM   2109  C   GLU A 282      -8.986 -10.677   5.973  1.00  0.34           C  \\nATOM   2110  CB  GLU A 282     -10.035  -8.640   4.958  1.00  0.32           C  \\nATOM   2111  O   GLU A 282      -9.915 -11.483   6.064  1.00  0.32           O  \\nATOM   2112  CG  GLU A 282     -10.798  -7.359   5.262  1.00  0.30           C  \\nATOM   2113  CD  GLU A 282     -11.663  -6.887   4.103  1.00  0.36           C  \\nATOM   2114  OE1 GLU A 282     -11.705  -7.574   3.058  1.00  0.41           O  \\nATOM   2115  OE2 GLU A 282     -12.302  -5.821   4.242  1.00  0.39           O  \\nATOM   2116  N   GLY A 283      -7.815 -11.207   6.380  1.00  0.35           N  \\nATOM   2117  CA  GLY A 283      -7.767 -12.601   5.970  1.00  0.35           C  \\nATOM   2118  C   GLY A 283      -8.359 -12.840   4.593  1.00  0.35           C  \\nATOM   2119  O   GLY A 283      -9.054 -13.834   4.374  1.00  0.33           O  \\nATOM   2120  N   ALA A 284      -8.797 -11.753   3.935  1.00  0.31           N  \\nATOM   2121  CA  ALA A 284      -9.405 -12.117   2.658  1.00  0.32           C  \\nATOM   2122  C   ALA A 284      -8.460 -12.981   1.827  1.00  0.31           C  \\nATOM   2123  CB  ALA A 284      -9.798 -10.864   1.880  1.00  0.29           C  \\nATOM   2124  O   ALA A 284      -7.349 -12.558   1.500  1.00  0.30           O  \\nATOM   2125  N   SER A 285      -8.155 -14.269   2.072  1.00  0.32           N  \\nATOM   2126  CA  SER A 285      -8.170 -15.249   0.991  1.00  0.32           C  \\nATOM   2127  C   SER A 285      -9.094 -14.811  -0.140  1.00  0.32           C  \\nATOM   2128  CB  SER A 285      -8.606 -16.619   1.514  1.00  0.30           C  \\nATOM   2129  O   SER A 285     -10.268 -14.515   0.090  1.00  0.30           O  \\nATOM   2130  OG  SER A 285      -7.605 -17.592   1.269  1.00  0.29           O  \\nATOM   2131  N   PRO A 286      -8.737 -13.974  -1.087  1.00  0.34           N  \\nATOM   2132  CA  PRO A 286      -9.583 -13.663  -2.241  1.00  0.34           C  \\nATOM   2133  C   PRO A 286     -10.621 -14.747  -2.522  1.00  0.34           C  \\nATOM   2134  CB  PRO A 286      -8.582 -13.554  -3.394  1.00  0.32           C  \\nATOM   2135  O   PRO A 286     -10.327 -15.937  -2.388  1.00  0.33           O  \\nATOM   2136  CG  PRO A 286      -7.327 -14.175  -2.870  1.00  0.31           C  \\nATOM   2137  CD  PRO A 286      -7.458 -14.332  -1.382  1.00  0.34           C  \\nATOM   2138  N   GLU A 287     -11.975 -14.555  -2.241  1.00  0.32           N  \\nATOM   2139  CA  GLU A 287     -13.040 -15.429  -2.724  1.00  0.32           C  \\nATOM   2140  C   GLU A 287     -12.609 -16.180  -3.980  1.00  0.32           C  \\nATOM   2141  CB  GLU A 287     -14.312 -14.624  -3.003  1.00  0.31           C  \\nATOM   2142  O   GLU A 287     -12.096 -15.577  -4.926  1.00  0.31           O  \\nATOM   2143  CG  GLU A 287     -15.559 -15.481  -3.164  1.00  0.30           C  \\nATOM   2144  CD  GLU A 287     -16.841 -14.667  -3.230  1.00  0.36           C  \\nATOM   2145  OE1 GLU A 287     -16.769 -13.418  -3.207  1.00  0.48           O  \\nATOM   2146  OE2 GLU A 287     -17.927 -15.285  -3.305  1.00  0.43           O  \\nATOM   2147  N   PRO A 288     -12.133 -17.503  -3.830  1.00  0.32           N  \\nATOM   2148  CA  PRO A 288     -11.880 -18.303  -5.031  1.00  0.31           C  \\nATOM   2149  C   PRO A 288     -12.780 -17.911  -6.201  1.00  0.32           C  \\nATOM   2150  CB  PRO A 288     -12.175 -19.733  -4.571  1.00  0.29           C  \\nATOM   2151  O   PRO A 288     -14.007 -17.925  -6.073  1.00  0.30           O  \\nATOM   2152  CG  PRO A 288     -12.973 -19.574  -3.317  1.00  0.28           C  \\nATOM   2153  CD  PRO A 288     -12.822 -18.161  -2.831  1.00  0.31           C  \\nATOM   2154  N   GLN A 289     -12.471 -17.017  -7.129  1.00  0.31           N  \\nATOM   2155  CA  GLN A 289     -13.163 -17.018  -8.413  1.00  0.34           C  \\nATOM   2156  C   GLN A 289     -13.317 -18.437  -8.954  1.00  0.31           C  \\nATOM   2157  CB  GLN A 289     -12.418 -16.148  -9.426  1.00  0.31           C  \\nATOM   2158  O   GLN A 289     -12.384 -19.240  -8.878  1.00  0.31           O  \\nATOM   2159  CG  GLN A 289     -12.643 -14.654  -9.237  1.00  0.31           C  \\nATOM   2160  CD  GLN A 289     -12.607 -13.884 -10.544  1.00  0.38           C  \\nATOM   2161  NE2 GLN A 289     -13.118 -12.658 -10.524  1.00  0.36           N  \\nATOM   2162  OE1 GLN A 289     -12.124 -14.388 -11.563  1.00  0.47           O  \\nATOM   2163  N   PRO A 290     -14.674 -18.995  -8.990  1.00  0.33           N  \\nATOM   2164  CA  PRO A 290     -14.937 -20.262  -9.677  1.00  0.32           C  \\nATOM   2165  C   PRO A 290     -14.093 -20.436 -10.937  1.00  0.33           C  \\nATOM   2166  CB  PRO A 290     -16.425 -20.173 -10.024  1.00  0.31           C  \\nATOM   2167  O   PRO A 290     -13.942 -19.494 -11.719  1.00  0.32           O  \\nATOM   2168  CG  PRO A 290     -16.766 -18.725  -9.875  1.00  0.30           C  \\nATOM   2169  CD  PRO A 290     -15.683 -18.060  -9.076  1.00  0.34           C  \\nATOM   2170  N   LYS A 291     -12.934 -21.167 -10.918  1.00  0.34           N  \\nATOM   2171  CA  LYS A 291     -12.446 -21.714 -12.181  1.00  0.33           C  \\nATOM   2172  C   LYS A 291     -13.538 -21.693 -13.247  1.00  0.34           C  \\nATOM   2173  CB  LYS A 291     -11.932 -23.141 -11.986  1.00  0.31           C  \\nATOM   2174  O   LYS A 291     -14.665 -22.126 -12.998  1.00  0.32           O  \\nATOM   2175  CG  LYS A 291     -10.465 -23.325 -12.347  1.00  0.30           C  \\nATOM   2176  CD  LYS A 291     -10.014 -24.765 -12.138  1.00  0.34           C  \\nATOM   2177  CE  LYS A 291      -8.557 -24.959 -12.533  1.00  0.32           C  \\nATOM   2178  NZ  LYS A 291      -8.129 -26.383 -12.393  1.00  0.34           N  \\nATOM   2179  N   PRO A 292     -13.577 -20.719 -14.282  1.00  0.32           N  \\nATOM   2180  CA  PRO A 292     -14.453 -21.048 -15.409  1.00  0.32           C  \\nATOM   2181  C   PRO A 292     -14.463 -22.540 -15.733  1.00  0.32           C  \\nATOM   2182  CB  PRO A 292     -13.856 -20.242 -16.566  1.00  0.30           C  \\nATOM   2183  O   PRO A 292     -13.410 -23.125 -15.998  1.00  0.30           O  \\nATOM   2184  CG  PRO A 292     -12.510 -19.812 -16.078  1.00  0.30           C  \\nATOM   2185  CD  PRO A 292     -12.430 -20.080 -14.602  1.00  0.35           C  \\nATOM   2186  N   SER A 293     -15.106 -23.393 -14.913  1.00  0.32           N  \\nATOM   2187  CA  SER A 293     -15.565 -24.662 -15.469  1.00  0.34           C  \\nATOM   2188  C   SER A 293     -16.170 -24.470 -16.856  1.00  0.32           C  \\nATOM   2189  CB  SER A 293     -16.591 -25.314 -14.542  1.00  0.30           C  \\nATOM   2190  O   SER A 293     -17.263 -23.916 -16.991  1.00  0.31           O  \\nATOM   2191  OG  SER A 293     -16.055 -25.490 -13.242  1.00  0.35           O  \\nATOM   2192  N   GLY A 294     -15.518 -23.667 -17.700  1.00  0.32           N  \\nATOM   2193  CA  GLY A 294     -15.918 -23.711 -19.098  1.00  0.34           C  \\nATOM   2194  C   GLY A 294     -14.749 -23.903 -20.046  1.00  0.32           C  \\nATOM   2195  O   GLY A 294     -13.679 -23.326 -19.845  1.00  0.31           O  \\nATOM   2196  N   ASP A 295     -14.552 -25.176 -20.482  1.00  0.37           N  \\nATOM   2197  CA  ASP A 295     -13.844 -25.609 -21.683  1.00  0.37           C  \\nATOM   2198  C   ASP A 295     -13.287 -24.414 -22.453  1.00  0.37           C  \\nATOM   2199  CB  ASP A 295     -14.768 -26.431 -22.584  1.00  0.35           C  \\nATOM   2200  O   ASP A 295     -14.011 -23.457 -22.733  1.00  0.36           O  \\nATOM   2201  CG  ASP A 295     -14.505 -27.924 -22.499  1.00  0.35           C  \\nATOM   2202  OD1 ASP A 295     -13.341 -28.327 -22.288  1.00  0.39           O  \\nATOM   2203  OD2 ASP A 295     -15.470 -28.705 -22.648  1.00  0.45           O  \\nATOM   2204  N   PRO A 296     -12.043 -23.944 -22.275  1.00  0.39           N  \\nATOM   2205  CA  PRO A 296     -11.355 -22.905 -23.044  1.00  0.39           C  \\nATOM   2206  C   PRO A 296     -11.921 -22.741 -24.453  1.00  0.39           C  \\nATOM   2207  CB  PRO A 296      -9.908 -23.402 -23.092  1.00  0.35           C  \\nATOM   2208  O   PRO A 296     -11.517 -21.832 -25.183  1.00  0.37           O  \\nATOM   2209  CG  PRO A 296      -9.989 -24.849 -22.727  1.00  0.35           C  \\nATOM   2210  CD  PRO A 296     -11.334 -25.112 -22.114  1.00  0.41           C  \\nATOM   2211  N   GLN A 297     -12.892 -23.529 -24.849  1.00  0.32           N  \\nATOM   2212  CA  GLN A 297     -13.440 -23.434 -26.197  1.00  0.37           C  \\nATOM   2213  C   GLN A 297     -14.680 -22.544 -26.225  1.00  0.32           C  \\nATOM   2214  CB  GLN A 297     -13.779 -24.823 -26.739  1.00  0.30           C  \\nATOM   2215  O   GLN A 297     -15.126 -22.125 -27.295  1.00  0.31           O  \\nATOM   2216  CG  GLN A 297     -12.572 -25.591 -27.261  1.00  0.31           C  \\nATOM   2217  CD  GLN A 297     -12.951 -26.905 -27.918  1.00  0.38           C  \\nATOM   2218  NE2 GLN A 297     -11.949 -27.706 -28.264  1.00  0.33           N  \\nATOM   2219  OE1 GLN A 297     -14.135 -27.198 -28.113  1.00  0.47           O  \\nATOM   2220  N   SER A 298     -14.865 -21.641 -25.203  1.00  0.36           N  \\nATOM   2221  CA  SER A 298     -15.868 -20.635 -25.539  1.00  0.39           C  \\nATOM   2222  C   SER A 298     -15.315 -19.225 -25.366  1.00  0.36           C  \\nATOM   2223  CB  SER A 298     -17.116 -20.812 -24.674  1.00  0.34           C  \\nATOM   2224  O   SER A 298     -15.056 -18.787 -24.242  1.00  0.35           O  \\nATOM   2225  OG  SER A 298     -17.534 -22.167 -24.665  1.00  0.37           O  \\nATOM   2226  N   LYS A 299     -14.247 -18.931 -26.085  1.00  0.41           N  \\nATOM   2227  CA  LYS A 299     -13.812 -17.666 -26.670  1.00  0.45           C  \\nATOM   2228  C   LYS A 299     -15.008 -16.812 -27.083  1.00  0.41           C  \\nATOM   2229  CB  LYS A 299     -12.905 -17.915 -27.876  1.00  0.37           C  \\nATOM   2230  O   LYS A 299     -15.580 -17.013 -28.157  1.00  0.39           O  \\nATOM   2231  CG  LYS A 299     -11.441 -18.119 -27.517  1.00  0.37           C  \\nATOM   2232  CD  LYS A 299     -10.575 -18.268 -28.761  1.00  0.44           C  \\nATOM   2233  CE  LYS A 299      -9.115 -18.507 -28.403  1.00  0.35           C  \\nATOM   2234  NZ  LYS A 299      -8.265 -18.673 -29.620  1.00  0.35           N  \\nATOM   2235  N   GLN A 300     -15.887 -16.530 -26.224  1.00  0.41           N  \\nATOM   2236  CA  GLN A 300     -16.747 -15.370 -26.432  1.00  0.44           C  \\nATOM   2237  C   GLN A 300     -16.688 -14.420 -25.239  1.00  0.41           C  \\nATOM   2238  CB  GLN A 300     -18.190 -15.809 -26.682  1.00  0.38           C  \\nATOM   2239  O   GLN A 300     -17.030 -14.802 -24.118  1.00  0.39           O  \\nATOM   2240  CG  GLN A 300     -18.419 -16.414 -28.061  1.00  0.38           C  \\nATOM   2241  CD  GLN A 300     -19.856 -16.850 -28.280  1.00  0.43           C  \\nATOM   2242  NE2 GLN A 300     -20.084 -17.636 -29.326  1.00  0.37           N  \\nATOM   2243  OE1 GLN A 300     -20.753 -16.484 -27.514  1.00  0.50           O  \\nATOM   2244  N   THR A 301     -15.756 -13.447 -25.185  1.00  0.45           N  \\nATOM   2245  CA  THR A 301     -15.913 -12.160 -24.517  1.00  0.47           C  \\nATOM   2246  C   THR A 301     -17.295 -11.573 -24.791  1.00  0.45           C  \\nATOM   2247  CB  THR A 301     -14.830 -11.161 -24.966  1.00  0.42           C  \\nATOM   2248  O   THR A 301     -17.652 -11.325 -25.944  1.00  0.44           O  \\nATOM   2249  CG2 THR A 301     -13.934 -10.760 -23.799  1.00  0.39           C  \\nATOM   2250  OG1 THR A 301     -14.026 -11.765 -25.987  1.00  0.40           O  \\nATOM   2251  N   PRO A 302     -18.247 -11.805 -23.936  1.00  0.39           N  \\nATOM   2252  CA  PRO A 302     -19.393 -10.903 -24.081  1.00  0.39           C  \\nATOM   2253  C   PRO A 302     -19.038  -9.447 -23.790  1.00  0.39           C  \\nATOM   2254  CB  PRO A 302     -20.394 -11.437 -23.054  1.00  0.36           C  \\nATOM   2255  O   PRO A 302     -18.422  -9.151 -22.763  1.00  0.38           O  \\nATOM   2256  CG  PRO A 302     -19.633 -12.461 -22.275  1.00  0.35           C  \\nATOM   2257  CD  PRO A 302     -18.284 -12.638 -22.911  1.00  0.40           C  \\nATOM   2258  N   GLU A 303     -18.645  -8.570 -24.731  1.00  0.48           N  \\nATOM   2259  CA  GLU A 303     -19.060  -7.171 -24.749  1.00  0.48           C  \\nATOM   2260  C   GLU A 303     -20.164  -6.909 -23.728  1.00  0.48           C  \\nATOM   2261  CB  GLU A 303     -19.532  -6.767 -26.148  1.00  0.43           C  \\nATOM   2262  O   GLU A 303     -21.242  -7.502 -23.806  1.00  0.45           O  \\nATOM   2263  CG  GLU A 303     -19.670  -5.264 -26.340  1.00  0.42           C  \\nATOM   2264  CD  GLU A 303     -19.794  -4.854 -27.799  1.00  0.46           C  \\nATOM   2265  OE1 GLU A 303     -19.898  -5.745 -28.673  1.00  0.53           O  \\nATOM   2266  OE2 GLU A 303     -19.787  -3.633 -28.071  1.00  0.48           O  \\nATOM   2267  N   ALA A 304     -19.792  -6.841 -22.405  1.00  0.46           N  \\nATOM   2268  CA  ALA A 304     -20.824  -6.316 -21.515  1.00  0.49           C  \\nATOM   2269  C   ALA A 304     -21.598  -5.182 -22.181  1.00  0.46           C  \\nATOM   2270  CB  ALA A 304     -20.203  -5.835 -20.206  1.00  0.43           C  \\nATOM   2271  O   ALA A 304     -21.138  -4.037 -22.203  1.00  0.45           O  \\nATOM   2272  N   SER A 305     -22.139  -5.260 -23.327  1.00  0.37           N  \\nATOM   2273  CA  SER A 305     -23.361  -4.531 -23.651  1.00  0.39           C  \\nATOM   2274  C   SER A 305     -24.428  -4.735 -22.581  1.00  0.37           C  \\nATOM   2275  CB  SER A 305     -23.902  -4.970 -25.013  1.00  0.35           C  \\nATOM   2276  O   SER A 305     -24.635  -5.856 -22.111  1.00  0.36           O  \\nATOM   2277  OG  SER A 305     -22.999  -4.623 -26.049  1.00  0.39           O  \\nATOM   2278  N   PRO A 306     -24.774  -3.783 -21.740  1.00  0.47           N  \\nATOM   2279  CA  PRO A 306     -26.117  -4.005 -21.202  1.00  0.46           C  \\nATOM   2280  C   PRO A 306     -27.020  -4.774 -22.165  1.00  0.47           C  \\nATOM   2281  CB  PRO A 306     -26.645  -2.586 -20.974  1.00  0.42           C  \\nATOM   2282  O   PRO A 306     -27.355  -4.268 -23.239  1.00  0.44           O  \\nATOM   2283  CG  PRO A 306     -25.751  -1.710 -21.790  1.00  0.41           C  \\nATOM   2284  CD  PRO A 306     -24.517  -2.489 -22.148  1.00  0.46           C  \\nATOM   2285  N   ARG A 307     -26.792  -6.017 -22.453  1.00  0.42           N  \\nATOM   2286  CA  ARG A 307     -27.907  -6.694 -23.107  1.00  0.44           C  \\nATOM   2287  C   ARG A 307     -29.199  -6.508 -22.319  1.00  0.42           C  \\nATOM   2288  CB  ARG A 307     -27.606  -8.184 -23.278  1.00  0.39           C  \\nATOM   2289  O   ARG A 307     -29.223  -6.703 -21.102  1.00  0.40           O  \\nATOM   2290  CG  ARG A 307     -28.372  -8.841 -24.416  1.00  0.39           C  \\nATOM   2291  CD  ARG A 307     -27.964 -10.296 -24.604  1.00  0.45           C  \\nATOM   2292  NE  ARG A 307     -28.422 -10.822 -25.887  1.00  0.39           N  \\nATOM   2293  NH1 ARG A 307     -27.392 -12.875 -25.647  1.00  0.30           N  \\nATOM   2294  NH2 ARG A 307     -28.608 -12.412 -27.534  1.00  0.25           N  \\nATOM   2295  CZ  ARG A 307     -28.140 -12.035 -26.353  1.00  0.35           C  \\nATOM   2296  N   PRO A 308     -30.072  -5.581 -22.811  1.00  0.49           N  \\nATOM   2297  CA  PRO A 308     -31.479  -5.716 -22.424  1.00  0.47           C  \\nATOM   2298  C   PRO A 308     -31.821  -7.116 -21.920  1.00  0.49           C  \\nATOM   2299  CB  PRO A 308     -32.232  -5.402 -23.719  1.00  0.45           C  \\nATOM   2300  O   PRO A 308     -31.505  -8.108 -22.582  1.00  0.47           O  \\nATOM   2301  CG  PRO A 308     -31.182  -5.403 -24.782  1.00  0.44           C  \\nATOM   2302  CD  PRO A 308     -29.834  -5.448 -24.121  1.00  0.48           C  \\nATOM   2303  N   GLN A 309     -31.767  -7.399 -20.683  1.00  0.49           N  \\nATOM   2304  CA  GLN A 309     -32.556  -8.501 -20.143  1.00  0.51           C  \\nATOM   2305  C   GLN A 309     -33.669  -8.904 -21.107  1.00  0.49           C  \\nATOM   2306  CB  GLN A 309     -33.150  -8.122 -18.786  1.00  0.46           C  \\nATOM   2307  O   GLN A 309     -34.483  -8.070 -21.507  1.00  0.47           O  \\nATOM   2308  CG  GLN A 309     -33.774  -9.293 -18.039  1.00  0.46           C  \\nATOM   2309  CD  GLN A 309     -34.202  -8.929 -16.629  1.00  0.50           C  \\nATOM   2310  NE2 GLN A 309     -34.814  -9.879 -15.932  1.00  0.41           N  \\nATOM   2311  OE1 GLN A 309     -33.983  -7.803 -16.171  1.00  0.54           O  \\nATOM   2312  N   PRO A 310     -33.404  -9.877 -22.028  1.00  0.47           N  \\nATOM   2313  CA  PRO A 310     -34.521 -10.434 -22.794  1.00  0.47           C  \\nATOM   2314  C   PRO A 310     -35.751 -10.707 -21.930  1.00  0.47           C  \\nATOM   2315  CB  PRO A 310     -33.948 -11.738 -23.355  1.00  0.43           C  \\nATOM   2316  O   PRO A 310     -35.629 -11.259 -20.833  1.00  0.45           O  \\nATOM   2317  CG  PRO A 310     -32.717 -11.994 -22.547  1.00  0.42           C  \\nATOM   2318  CD  PRO A 310     -32.358 -10.733 -21.814  1.00  0.46           C  \\nATOM   2319  N   GLU A 311     -36.823  -9.859 -22.061  1.00  0.53           N  \\nATOM   2320  CA  GLU A 311     -38.236 -10.171 -21.869  1.00  0.54           C  \\nATOM   2321  C   GLU A 311     -38.499 -11.665 -22.033  1.00  0.52           C  \\nATOM   2322  CB  GLU A 311     -39.101  -9.374 -22.850  1.00  0.49           C  \\nATOM   2323  O   GLU A 311     -38.075 -12.271 -23.019  1.00  0.50           O  \\nATOM   2324  CG  GLU A 311     -39.319  -7.925 -22.441  1.00  0.48           C  \\nATOM   2325  CD  GLU A 311     -40.247  -7.170 -23.379  1.00  0.53           C  \\nATOM   2326  OE1 GLU A 311     -40.584  -7.705 -24.460  1.00  0.56           O  \\nATOM   2327  OE2 GLU A 311     -40.641  -6.034 -23.030  1.00  0.50           O  \\nATOM   2328  N   PRO A 312     -38.635 -12.417 -20.868  1.00  0.55           N  \\nATOM   2329  CA  PRO A 312     -39.170 -13.780 -20.876  1.00  0.56           C  \\nATOM   2330  C   PRO A 312     -40.202 -14.003 -21.979  1.00  0.55           C  \\nATOM   2331  CB  PRO A 312     -39.813 -13.914 -19.493  1.00  0.50           C  \\nATOM   2332  O   PRO A 312     -41.337 -13.530 -21.874  1.00  0.53           O  \\nATOM   2333  CG  PRO A 312     -40.106 -12.511 -19.070  1.00  0.49           C  \\nATOM   2334  CD  PRO A 312     -39.215 -11.583 -19.846  1.00  0.53           C  \\nATOM   2335  N   SER A 313     -39.784 -14.226 -23.182  1.00  0.49           N  \\nATOM   2336  CA  SER A 313     -40.699 -14.749 -24.192  1.00  0.52           C  \\nATOM   2337  C   SER A 313     -40.852 -16.262 -24.069  1.00  0.49           C  \\nATOM   2338  CB  SER A 313     -40.209 -14.391 -25.596  1.00  0.45           C  \\nATOM   2339  O   SER A 313     -39.858 -16.990 -24.029  1.00  0.46           O  \\nATOM   2340  OG  SER A 313     -40.067 -12.988 -25.736  1.00  0.49           O  \\nATOM   2341  N   GLY A 314     -41.940 -16.650 -23.461  1.00  0.50           N  \\nATOM   2342  CA  GLY A 314     -43.081 -17.551 -23.511  1.00  0.51           C  \\nATOM   2343  C   GLY A 314     -42.857 -18.747 -24.417  1.00  0.49           C  \\nATOM   2344  O   GLY A 314     -43.814 -19.354 -24.900  1.00  0.46           O  \\nATOM   2345  N   LYS A 315     -41.730 -19.418 -24.424  1.00  0.44           N  \\nATOM   2346  CA  LYS A 315     -41.830 -20.774 -24.957  1.00  0.47           C  \\nATOM   2347  C   LYS A 315     -40.988 -21.749 -24.139  1.00  0.44           C  \\nATOM   2348  CB  LYS A 315     -41.396 -20.809 -26.423  1.00  0.40           C  \\nATOM   2349  O   LYS A 315     -39.843 -21.450 -23.793  1.00  0.42           O  \\nATOM   2350  CG  LYS A 315     -42.427 -20.240 -27.387  1.00  0.40           C  \\nATOM   2351  CD  LYS A 315     -41.969 -20.368 -28.834  1.00  0.46           C  \\nATOM   2352  CE  LYS A 315     -43.000 -19.801 -29.800  1.00  0.38           C  \\nATOM   2353  NZ  LYS A 315     -42.529 -19.868 -31.216  1.00  0.38           N  \\nATOM   2354  N   PRO A 316     -41.494 -22.717 -23.332  1.00  0.53           N  \\nATOM   2355  CA  PRO A 316     -40.821 -23.932 -22.867  1.00  0.54           C  \\nATOM   2356  C   PRO A 316     -39.792 -24.456 -23.866  1.00  0.53           C  \\nATOM   2357  CB  PRO A 316     -41.970 -24.928 -22.693  1.00  0.48           C  \\nATOM   2358  O   PRO A 316     -40.158 -24.911 -24.953  1.00  0.50           O  \\nATOM   2359  CG  PRO A 316     -43.124 -24.309 -23.413  1.00  0.47           C  \\nATOM   2360  CD  PRO A 316     -42.776 -22.884 -23.737  1.00  0.51           C  \\nATOM   2361  N   GLN A 317     -38.580 -23.913 -23.883  1.00  0.46           N  \\nATOM   2362  CA  GLN A 317     -37.430 -24.548 -24.517  1.00  0.50           C  \\nATOM   2363  C   GLN A 317     -36.936 -25.736 -23.696  1.00  0.47           C  \\nATOM   2364  CB  GLN A 317     -36.298 -23.538 -24.712  1.00  0.42           C  \\nATOM   2365  O   GLN A 317     -36.657 -25.600 -22.504  1.00  0.45           O  \\nATOM   2366  CG  GLN A 317     -36.107 -23.099 -26.157  1.00  0.41           C  \\nATOM   2367  CD  GLN A 317     -35.067 -22.005 -26.306  1.00  0.47           C  \\nATOM   2368  NE2 GLN A 317     -34.930 -21.478 -27.518  1.00  0.39           N  \\nATOM   2369  OE1 GLN A 317     -34.392 -21.636 -25.339  1.00  0.50           O  \\nATOM   2370  N   SER A 318     -37.456 -26.914 -23.922  1.00  0.49           N  \\nATOM   2371  CA  SER A 318     -37.010 -28.303 -23.972  1.00  0.54           C  \\nATOM   2372  C   SER A 318     -35.489 -28.394 -24.027  1.00  0.48           C  \\nATOM   2373  CB  SER A 318     -37.616 -29.017 -25.181  1.00  0.44           C  \\nATOM   2374  O   SER A 318     -34.845 -27.663 -24.783  1.00  0.46           O  \\nATOM   2375  OG  SER A 318     -38.834 -28.405 -25.569  1.00  0.49           O  \\nATOM   2376  N   GLU A 319     -34.890 -28.573 -22.805  1.00  0.49           N  \\nATOM   2377  CA  GLU A 319     -33.659 -29.325 -22.579  1.00  0.55           C  \\nATOM   2378  C   GLU A 319     -32.469 -28.652 -23.256  1.00  0.49           C  \\nATOM   2379  CB  GLU A 319     -33.808 -30.763 -23.083  1.00  0.45           C  \\nATOM   2380  O   GLU A 319     -32.133 -28.975 -24.397  1.00  0.47           O  \\nATOM   2381  CG  GLU A 319     -34.278 -31.743 -22.018  1.00  0.45           C  \\nATOM   2382  CD  GLU A 319     -34.533 -33.141 -22.560  1.00  0.51           C  \\nATOM   2383  OE1 GLU A 319     -34.419 -33.345 -23.789  1.00  0.53           O  \\nATOM   2384  OE2 GLU A 319     -34.851 -34.038 -21.747  1.00  0.47           O  \\nATOM   2385  N   GLN A 320     -32.343 -27.359 -23.226  1.00  0.42           N  \\nATOM   2386  CA  GLN A 320     -31.009 -26.775 -23.306  1.00  0.47           C  \\nATOM   2387  C   GLN A 320     -30.431 -26.529 -21.916  1.00  0.42           C  \\nATOM   2388  CB  GLN A 320     -31.042 -25.467 -24.100  1.00  0.39           C  \\nATOM   2389  O   GLN A 320     -31.037 -25.832 -21.100  1.00  0.40           O  \\nATOM   2390  CG  GLN A 320     -31.191 -25.663 -25.602  1.00  0.40           C  \\nATOM   2391  CD  GLN A 320     -31.182 -24.354 -26.369  1.00  0.46           C  \\nATOM   2392  NE2 GLN A 320     -31.613 -24.399 -27.624  1.00  0.38           N  \\nATOM   2393  OE1 GLN A 320     -30.790 -23.310 -25.836  1.00  0.50           O  \\nATOM   2394  N   THR A 321     -29.734 -27.603 -21.288  1.00  0.47           N  \\nATOM   2395  CA  THR A 321     -28.730 -27.276 -20.282  1.00  0.49           C  \\nATOM   2396  C   THR A 321     -28.374 -25.793 -20.335  1.00  0.47           C  \\nATOM   2397  CB  THR A 321     -27.456 -28.120 -20.469  1.00  0.44           C  \\nATOM   2398  O   THR A 321     -27.998 -25.278 -21.390  1.00  0.45           O  \\nATOM   2399  CG2 THR A 321     -27.467 -29.343 -19.557  1.00  0.42           C  \\nATOM   2400  OG1 THR A 321     -27.373 -28.555 -21.832  1.00  0.41           O  \\nATOM   2401  N   PRO A 322     -29.097 -25.012 -19.663  1.00  0.41           N  \\nATOM   2402  CA  PRO A 322     -28.571 -23.646 -19.627  1.00  0.43           C  \\nATOM   2403  C   PRO A 322     -27.068 -23.585 -19.892  1.00  0.41           C  \\nATOM   2404  CB  PRO A 322     -28.894 -23.181 -18.205  1.00  0.38           C  \\nATOM   2405  O   PRO A 322     -26.294 -24.288 -19.238  1.00  0.40           O  \\nATOM   2406  CG  PRO A 322     -29.460 -24.387 -17.528  1.00  0.38           C  \\nATOM   2407  CD  PRO A 322     -29.530 -25.506 -18.527  1.00  0.43           C  \\nATOM   2408  N   LYS A 323     -26.447 -23.480 -21.160  1.00  0.42           N  \\nATOM   2409  CA  LYS A 323     -25.168 -22.792 -21.310  1.00  0.43           C  \\nATOM   2410  C   LYS A 323     -24.876 -21.906 -20.102  1.00  0.42           C  \\nATOM   2411  CB  LYS A 323     -25.156 -21.954 -22.589  1.00  0.40           C  \\nATOM   2412  O   LYS A 323     -25.672 -21.030 -19.761  1.00  0.40           O  \\nATOM   2413  CG  LYS A 323     -23.761 -21.658 -23.121  1.00  0.39           C  \\nATOM   2414  CD  LYS A 323     -23.817 -20.924 -24.455  1.00  0.44           C  \\nATOM   2415  CE  LYS A 323     -22.422 -20.648 -25.001  1.00  0.39           C  \\nATOM   2416  NZ  LYS A 323     -22.472 -19.977 -26.334  1.00  0.39           N  \\nATOM   2417  N   ALA A 324     -24.541 -22.490 -18.881  1.00  0.43           N  \\nATOM   2418  CA  ALA A 324     -24.068 -21.673 -17.767  1.00  0.46           C  \\nATOM   2419  C   ALA A 324     -23.929 -20.210 -18.180  1.00  0.43           C  \\nATOM   2420  CB  ALA A 324     -22.735 -22.204 -17.245  1.00  0.41           C  \\nATOM   2421  O   ALA A 324     -23.201 -19.891 -19.123  1.00  0.42           O  \\nATOM   2422  N   SER A 325     -24.972 -19.393 -18.417  1.00  0.34           N  \\nATOM   2423  CA  SER A 325     -24.737 -17.956 -18.320  1.00  0.36           C  \\nATOM   2424  C   SER A 325     -23.352 -17.660 -17.754  1.00  0.34           C  \\nATOM   2425  CB  SER A 325     -25.804 -17.295 -17.447  1.00  0.32           C  \\nATOM   2426  O   SER A 325     -22.878 -18.361 -16.858  1.00  0.33           O  \\nATOM   2427  OG  SER A 325     -27.080 -17.385 -18.058  1.00  0.32           O  \\nATOM   2428  N   PRO A 326     -22.327 -17.237 -18.554  1.00  0.38           N  \\nATOM   2429  CA  PRO A 326     -21.217 -16.828 -17.691  1.00  0.38           C  \\nATOM   2430  C   PRO A 326     -21.654 -16.562 -16.252  1.00  0.38           C  \\nATOM   2431  CB  PRO A 326     -20.710 -15.544 -18.353  1.00  0.35           C  \\nATOM   2432  O   PRO A 326     -22.691 -15.935 -16.024  1.00  0.36           O  \\nATOM   2433  CG  PRO A 326     -21.813 -15.126 -19.270  1.00  0.34           C  \\nATOM   2434  CD  PRO A 326     -22.736 -16.294 -19.470  1.00  0.38           C  \\nATOM   2435  N   GLU A 327     -21.643 -17.440 -15.299  1.00  0.35           N  \\nATOM   2436  CA  GLU A 327     -21.800 -17.036 -13.905  1.00  0.37           C  \\nATOM   2437  C   GLU A 327     -21.516 -15.547 -13.726  1.00  0.35           C  \\nATOM   2438  CB  GLU A 327     -20.881 -17.860 -12.999  1.00  0.33           C  \\nATOM   2439  O   GLU A 327     -20.530 -15.030 -14.255  1.00  0.34           O  \\nATOM   2440  CG  GLU A 327     -21.361 -17.953 -11.558  1.00  0.33           C  \\nATOM   2441  CD  GLU A 327     -20.533 -18.906 -10.711  1.00  0.39           C  \\nATOM   2442  OE1 GLU A 327     -19.570 -19.507 -11.239  1.00  0.48           O  \\nATOM   2443  OE2 GLU A 327     -20.850 -19.053  -9.509  1.00  0.44           O  \\nATOM   2444  N   PRO A 328     -22.570 -14.673 -13.748  1.00  0.31           N  \\nATOM   2445  CA  PRO A 328     -22.208 -13.287 -13.441  1.00  0.31           C  \\nATOM   2446  C   PRO A 328     -20.771 -13.148 -12.945  1.00  0.31           C  \\nATOM   2447  CB  PRO A 328     -23.204 -12.898 -12.345  1.00  0.28           C  \\nATOM   2448  O   PRO A 328     -20.342 -13.900 -12.066  1.00  0.29           O  \\nATOM   2449  CG  PRO A 328     -23.698 -14.198 -11.799  1.00  0.28           C  \\nATOM   2450  CD  PRO A 328     -23.414 -15.277 -12.804  1.00  0.32           C  \\nATOM   2451  N   GLN A 329     -19.701 -12.867 -13.708  1.00  0.35           N  \\nATOM   2452  CA  GLN A 329     -18.564 -12.218 -13.063  1.00  0.36           C  \\nATOM   2453  C   GLN A 329     -18.953 -11.655 -11.699  1.00  0.35           C  \\nATOM   2454  CB  GLN A 329     -18.006 -11.105 -13.951  1.00  0.33           C  \\nATOM   2455  O   GLN A 329     -20.034 -11.083 -11.544  1.00  0.34           O  \\nATOM   2456  CG  GLN A 329     -17.151 -11.610 -15.106  1.00  0.33           C  \\nATOM   2457  CD  GLN A 329     -16.052 -10.637 -15.491  1.00  0.40           C  \\nATOM   2458  NE2 GLN A 329     -15.047 -11.130 -16.207  1.00  0.36           N  \\nATOM   2459  OE1 GLN A 329     -16.106  -9.451 -15.149  1.00  0.48           O  \\nATOM   2460  N   PRO A 330     -18.766 -12.257 -10.564  1.00  0.35           N  \\nATOM   2461  CA  PRO A 330     -19.154 -11.412  -9.432  1.00  0.34           C  \\nATOM   2462  C   PRO A 330     -19.301  -9.942  -9.817  1.00  0.35           C  \\nATOM   2463  CB  PRO A 330     -18.005 -11.604  -8.439  1.00  0.32           C  \\nATOM   2464  O   PRO A 330     -18.494  -9.418 -10.590  1.00  0.33           O  \\nATOM   2465  CG  PRO A 330     -16.904 -12.218  -9.243  1.00  0.31           C  \\nATOM   2466  CD  PRO A 330     -17.490 -12.772 -10.509  1.00  0.34           C  \\nATOM   2467  N   LYS A 331     -20.471  -9.328 -10.194  1.00  0.30           N  \\nATOM   2468  CA  LYS A 331     -20.567  -7.874 -10.101  1.00  0.30           C  \\nATOM   2469  C   LYS A 331     -19.323  -7.281  -9.445  1.00  0.30           C  \\nATOM   2470  CB  LYS A 331     -21.816  -7.466  -9.319  1.00  0.28           C  \\nATOM   2471  O   LYS A 331     -18.882  -7.756  -8.397  1.00  0.28           O  \\nATOM   2472  CG  LYS A 331     -23.052  -7.273 -10.185  1.00  0.28           C  \\nATOM   2473  CD  LYS A 331     -24.246  -6.811  -9.359  1.00  0.31           C  \\nATOM   2474  CE  LYS A 331     -25.492  -6.646 -10.219  1.00  0.30           C  \\nATOM   2475  NZ  LYS A 331     -26.675  -6.234  -9.406  1.00  0.32           N  \\nATOM   2476  N   PRO A 332     -18.143  -6.967 -10.064  1.00  0.31           N  \\nATOM   2477  CA  PRO A 332     -17.088  -6.277  -9.318  1.00  0.31           C  \\nATOM   2478  C   PRO A 332     -17.548  -5.810  -7.939  1.00  0.32           C  \\nATOM   2479  CB  PRO A 332     -16.747  -5.083 -10.214  1.00  0.29           C  \\nATOM   2480  O   PRO A 332     -18.657  -5.289  -7.798  1.00  0.30           O  \\nATOM   2481  CG  PRO A 332     -17.687  -5.192 -11.370  1.00  0.28           C  \\nATOM   2482  CD  PRO A 332     -18.594  -6.367 -11.139  1.00  0.33           C  \\nATOM   2483  N   SER A 333     -17.497  -6.582  -6.856  1.00  0.34           N  \\nATOM   2484  CA  SER A 333     -17.707  -5.953  -5.557  1.00  0.34           C  \\nATOM   2485  C   SER A 333     -17.605  -4.434  -5.655  1.00  0.34           C  \\nATOM   2486  CB  SER A 333     -16.694  -6.475  -4.538  1.00  0.32           C  \\nATOM   2487  O   SER A 333     -16.994  -3.907  -6.587  1.00  0.33           O  \\nATOM   2488  OG  SER A 333     -15.368  -6.220  -4.970  1.00  0.32           O  \\nATOM   2489  N   GLY A 334     -18.757  -3.634  -5.674  1.00  0.32           N  \\nATOM   2490  CA  GLY A 334     -18.620  -2.189  -5.584  1.00  0.32           C  \\nATOM   2491  C   GLY A 334     -17.190  -1.714  -5.763  1.00  0.32           C  \\nATOM   2492  O   GLY A 334     -16.950  -0.669  -6.372  1.00  0.30           O  \\nATOM   2493  N   GLN A 335     -16.238  -2.532  -5.345  1.00  0.33           N  \\nATOM   2494  CA  GLN A 335     -14.812  -2.243  -5.452  1.00  0.35           C  \\nATOM   2495  C   GLN A 335     -14.373  -2.171  -6.911  1.00  0.33           C  \\nATOM   2496  CB  GLN A 335     -13.992  -3.299  -4.708  1.00  0.31           C  \\nATOM   2497  O   GLN A 335     -13.648  -1.254  -7.303  1.00  0.32           O  \\nATOM   2498  CG  GLN A 335     -13.860  -3.036  -3.214  1.00  0.29           C  \\nATOM   2499  CD  GLN A 335     -13.068  -4.113  -2.496  1.00  0.35           C  \\nATOM   2500  NE2 GLN A 335     -12.958  -3.988  -1.178  1.00  0.34           N  \\nATOM   2501  OE1 GLN A 335     -12.561  -5.050  -3.121  1.00  0.41           O  \\nATOM   2502  N   ALA A 336     -14.747  -3.037  -7.785  1.00  0.34           N  \\nATOM   2503  CA  ALA A 336     -14.394  -3.011  -9.202  1.00  0.33           C  \\nATOM   2504  C   ALA A 336     -15.085  -1.854  -9.918  1.00  0.34           C  \\nATOM   2505  CB  ALA A 336     -14.759  -4.337  -9.866  1.00  0.32           C  \\nATOM   2506  O   ALA A 336     -14.486  -1.199 -10.774  1.00  0.32           O  \\nATOM   2507  N   LYS A 337     -16.402  -1.554  -9.517  1.00  0.33           N  \\nATOM   2508  CA  LYS A 337     -17.040  -0.378 -10.100  1.00  0.33           C  \\nATOM   2509  C   LYS A 337     -16.324   0.902  -9.680  1.00  0.33           C  \\nATOM   2510  CB  LYS A 337     -18.514  -0.311  -9.696  1.00  0.31           C  \\nATOM   2511  O   LYS A 337     -16.104   1.795 -10.501  1.00  0.31           O  \\nATOM   2512  CG  LYS A 337     -19.426  -1.199 -10.529  1.00  0.30           C  \\nATOM   2513  CD  LYS A 337     -20.887  -1.025 -10.136  1.00  0.33           C  \\nATOM   2514  CE  LYS A 337     -21.801  -1.915 -10.968  1.00  0.29           C  \\nATOM   2515  NZ  LYS A 337     -23.233  -1.761 -10.574  1.00  0.32           N  \\nATOM   2516  N   GLU A 338     -16.007   0.977  -8.393  1.00  0.35           N  \\nATOM   2517  CA  GLU A 338     -15.323   2.148  -7.852  1.00  0.36           C  \\nATOM   2518  C   GLU A 338     -13.927   2.298  -8.450  1.00  0.35           C  \\nATOM   2519  CB  GLU A 338     -15.236   2.062  -6.326  1.00  0.33           C  \\nATOM   2520  O   GLU A 338     -13.530   3.396  -8.845  1.00  0.34           O  \\nATOM   2521  CG  GLU A 338     -15.570   3.366  -5.616  1.00  0.32           C  \\nATOM   2522  CD  GLU A 338     -15.830   3.190  -4.129  1.00  0.38           C  \\nATOM   2523  OE1 GLU A 338     -15.678   2.058  -3.616  1.00  0.44           O  \\nATOM   2524  OE2 GLU A 338     -16.189   4.192  -3.471  1.00  0.40           O  \\nATOM   2525  N   ILE A 339     -13.232   1.236  -8.662  1.00  0.38           N  \\nATOM   2526  CA  ILE A 339     -11.927   1.204  -9.313  1.00  0.38           C  \\nATOM   2527  C   ILE A 339     -12.071   1.611 -10.778  1.00  0.38           C  \\nATOM   2528  CB  ILE A 339     -11.277  -0.194  -9.207  1.00  0.37           C  \\nATOM   2529  O   ILE A 339     -11.274   2.400 -11.291  1.00  0.37           O  \\nATOM   2530  CG1 ILE A 339     -10.911  -0.504  -7.751  1.00  0.34           C  \\nATOM   2531  CG2 ILE A 339     -10.045  -0.286 -10.113  1.00  0.34           C  \\nATOM   2532  CD1 ILE A 339     -10.491  -1.948  -7.512  1.00  0.33           C  \\nATOM   2533  N   GLN A 340     -13.094   1.043 -11.510  1.00  0.38           N  \\nATOM   2534  CA  GLN A 340     -13.383   1.381 -12.900  1.00  0.39           C  \\nATOM   2535  C   GLN A 340     -13.741   2.857 -13.044  1.00  0.38           C  \\nATOM   2536  CB  GLN A 340     -14.520   0.510 -13.439  1.00  0.37           C  \\nATOM   2537  O   GLN A 340     -13.294   3.521 -13.982  1.00  0.37           O  \\nATOM   2538  CG  GLN A 340     -14.044  -0.731 -14.182  1.00  0.36           C  \\nATOM   2539  CD  GLN A 340     -15.190  -1.566 -14.722  1.00  0.41           C  \\nATOM   2540  NE2 GLN A 340     -14.859  -2.677 -15.371  1.00  0.37           N  \\nATOM   2541  OE1 GLN A 340     -16.363  -1.217 -14.556  1.00  0.46           O  \\nATOM   2542  N   TRP A 341     -14.509   3.431 -12.110  1.00  0.23           N  \\nATOM   2543  CA  TRP A 341     -14.868   4.845 -12.120  1.00  0.30           C  \\nATOM   2544  C   TRP A 341     -13.640   5.720 -11.897  1.00  0.23           C  \\nATOM   2545  CB  TRP A 341     -15.922   5.140 -11.049  1.00  0.23           C  \\nATOM   2546  O   TRP A 341     -13.402   6.670 -12.647  1.00  0.23           O  \\nATOM   2547  CG  TRP A 341     -16.681   6.413 -11.276  1.00  0.25           C  \\nATOM   2548  CD1 TRP A 341     -17.649   6.639 -12.215  1.00  0.23           C  \\nATOM   2549  CD2 TRP A 341     -16.532   7.637 -10.551  1.00  0.23           C  \\nATOM   2550  CE2 TRP A 341     -17.442   8.565 -11.105  1.00  0.17           C  \\nATOM   2551  CE3 TRP A 341     -15.716   8.041  -9.485  1.00  0.33           C  \\nATOM   2552  NE1 TRP A 341     -18.110   7.931 -12.118  1.00  0.27           N  \\nATOM   2553  CH2 TRP A 341     -16.749  10.241  -9.585  1.00  0.24           C  \\nATOM   2554  CZ2 TRP A 341     -17.559   9.872 -10.628  1.00  0.24           C  \\nATOM   2555  CZ3 TRP A 341     -15.834   9.343  -9.012  1.00  0.27           C  \\nATOM   2556  N   GLN A 342     -12.810   5.413 -10.959  1.00  0.39           N  \\nATOM   2557  CA  GLN A 342     -11.590   6.176 -10.719  1.00  0.40           C  \\nATOM   2558  C   GLN A 342     -10.664   6.127 -11.931  1.00  0.39           C  \\nATOM   2559  CB  GLN A 342     -10.862   5.652  -9.481  1.00  0.38           C  \\nATOM   2560  O   GLN A 342     -10.101   7.149 -12.331  1.00  0.39           O  \\nATOM   2561  CG  GLN A 342     -11.517   6.054  -8.166  1.00  0.37           C  \\nATOM   2562  CD  GLN A 342     -10.793   5.497  -6.955  1.00  0.42           C  \\nATOM   2563  NE2 GLN A 342     -11.285   5.829  -5.766  1.00  0.39           N  \\nATOM   2564  OE1 GLN A 342      -9.801   4.774  -7.087  1.00  0.48           O  \\nATOM   2565  N   ALA A 343     -10.459   4.965 -12.526  1.00  0.40           N  \\nATOM   2566  CA  ALA A 343      -9.656   4.854 -13.742  1.00  0.41           C  \\nATOM   2567  C   ALA A 343     -10.214   5.743 -14.850  1.00  0.40           C  \\nATOM   2568  CB  ALA A 343      -9.595   3.402 -14.209  1.00  0.40           C  \\nATOM   2569  O   ALA A 343      -9.459   6.425 -15.547  1.00  0.41           O  \\nATOM   2570  N   LYS A 344     -11.540   5.775 -15.027  1.00  0.40           N  \\nATOM   2571  CA  LYS A 344     -12.179   6.593 -16.054  1.00  0.41           C  \\nATOM   2572  C   LYS A 344     -12.029   8.079 -15.745  1.00  0.40           C  \\nATOM   2573  CB  LYS A 344     -13.659   6.231 -16.184  1.00  0.39           C  \\nATOM   2574  O   LYS A 344     -11.723   8.876 -16.635  1.00  0.40           O  \\nATOM   2575  CG  LYS A 344     -14.095   5.909 -17.606  1.00  0.39           C  \\nATOM   2576  CD  LYS A 344     -15.570   5.536 -17.667  1.00  0.42           C  \\nATOM   2577  CE  LYS A 344     -16.022   5.269 -19.097  1.00  0.42           C  \\nATOM   2578  NZ  LYS A 344     -17.494   5.029 -19.178  1.00  0.40           N  \\nATOM   2579  N   GLU A 345     -12.283   8.514 -14.499  1.00  0.38           N  \\nATOM   2580  CA  GLU A 345     -12.109   9.916 -14.133  1.00  0.40           C  \\nATOM   2581  C   GLU A 345     -10.671  10.373 -14.365  1.00  0.38           C  \\nATOM   2582  CB  GLU A 345     -12.504  10.144 -12.672  1.00  0.38           C  \\nATOM   2583  O   GLU A 345     -10.438  11.451 -14.916  1.00  0.38           O  \\nATOM   2584  CG  GLU A 345     -13.351  11.388 -12.450  1.00  0.37           C  \\nATOM   2585  CD  GLU A 345     -13.419  11.816 -10.993  1.00  0.43           C  \\nATOM   2586  OE1 GLU A 345     -13.025  11.023 -10.108  1.00  0.51           O  \\nATOM   2587  OE2 GLU A 345     -13.871  12.954 -10.734  1.00  0.47           O  \\nATOM   2588  N   ILE A 346      -9.726   9.589 -14.028  1.00  0.41           N  \\nATOM   2589  CA  ILE A 346      -8.324   9.907 -14.274  1.00  0.42           C  \\nATOM   2590  C   ILE A 346      -8.061   9.952 -15.778  1.00  0.42           C  \\nATOM   2591  CB  ILE A 346      -7.385   8.883 -13.597  1.00  0.41           C  \\nATOM   2592  O   ILE A 346      -7.361  10.842 -16.265  1.00  0.42           O  \\nATOM   2593  CG1 ILE A 346      -7.454   9.024 -12.072  1.00  0.38           C  \\nATOM   2594  CG2 ILE A 346      -5.948   9.054 -14.098  1.00  0.37           C  \\nATOM   2595  CD1 ILE A 346      -6.797   7.878 -11.314  1.00  0.37           C  \\nATOM   2596  N   GLN A 347      -8.599   9.032 -16.635  1.00  0.43           N  \\nATOM   2597  CA  GLN A 347      -8.491   9.068 -18.089  1.00  0.48           C  \\nATOM   2598  C   GLN A 347      -9.083  10.356 -18.654  1.00  0.44           C  \\nATOM   2599  CB  GLN A 347      -9.187   7.855 -18.710  1.00  0.43           C  \\nATOM   2600  O   GLN A 347      -8.511  10.962 -19.562  1.00  0.45           O  \\nATOM   2601  CG  GLN A 347      -8.232   6.739 -19.113  1.00  0.42           C  \\nATOM   2602  CD  GLN A 347      -8.950   5.532 -19.689  1.00  0.47           C  \\nATOM   2603  NE2 GLN A 347      -8.192   4.491 -20.015  1.00  0.43           N  \\nATOM   2604  OE1 GLN A 347     -10.176   5.535 -19.838  1.00  0.57           O  \\nATOM   2605  N   GLU A 348     -10.248  10.767 -18.163  1.00  0.40           N  \\nATOM   2606  CA  GLU A 348     -10.930  11.958 -18.662  1.00  0.43           C  \\nATOM   2607  C   GLU A 348     -10.177  13.228 -18.277  1.00  0.40           C  \\nATOM   2608  CB  GLU A 348     -12.365  12.016 -18.133  1.00  0.39           C  \\nATOM   2609  O   GLU A 348     -10.046  14.147 -19.087  1.00  0.40           O  \\nATOM   2610  CG  GLU A 348     -13.340  11.134 -18.899  1.00  0.38           C  \\nATOM   2611  CD  GLU A 348     -14.783  11.601 -18.793  1.00  0.42           C  \\nATOM   2612  OE1 GLU A 348     -15.076  12.482 -17.953  1.00  0.55           O  \\nATOM   2613  OE2 GLU A 348     -15.628  11.080 -19.554  1.00  0.51           O  \\nATOM   2614  N   GLN A 349      -9.493  13.176 -17.154  1.00  0.41           N  \\nATOM   2615  CA  GLN A 349      -8.688  14.321 -16.741  1.00  0.44           C  \\nATOM   2616  C   GLN A 349      -7.382  14.388 -17.528  1.00  0.41           C  \\nATOM   2617  CB  GLN A 349      -8.395  14.259 -15.241  1.00  0.40           C  \\nATOM   2618  O   GLN A 349      -6.870  15.476 -17.798  1.00  0.41           O  \\nATOM   2619  CG  GLN A 349      -9.564  14.695 -14.368  1.00  0.38           C  \\nATOM   2620  CD  GLN A 349      -9.221  14.703 -12.890  1.00  0.44           C  \\nATOM   2621  NE2 GLN A 349     -10.225  14.927 -12.049  1.00  0.41           N  \\nATOM   2622  OE1 GLN A 349      -8.063  14.510 -12.507  1.00  0.55           O  \\nATOM   2623  N   ALA A 350      -6.813  13.260 -17.963  1.00  0.42           N  \\nATOM   2624  CA  ALA A 350      -5.626  13.244 -18.814  1.00  0.48           C  \\nATOM   2625  C   ALA A 350      -5.947  13.762 -20.213  1.00  0.44           C  \\nATOM   2626  CB  ALA A 350      -5.046  11.834 -18.892  1.00  0.42           C  \\nATOM   2627  O   ALA A 350      -5.047  14.157 -20.957  1.00  0.45           O  \\nATOM   2628  N   LYS A 351      -7.161  13.685 -20.522  1.00  0.43           N  \\nATOM   2629  CA  LYS A 351      -7.582  14.131 -21.847  1.00  0.49           C  \\nATOM   2630  C   LYS A 351      -7.837  15.635 -21.866  1.00  0.42           C  \\nATOM   2631  CB  LYS A 351      -8.838  13.380 -22.292  1.00  0.43           C  \\nATOM   2632  O   LYS A 351      -7.902  16.246 -22.934  1.00  0.42           O  \\nATOM   2633  CG  LYS A 351      -8.584  11.938 -22.704  1.00  0.41           C  \\nATOM   2634  CD  LYS A 351      -9.883  11.210 -23.022  1.00  0.45           C  \\nATOM   2635  CE  LYS A 351      -9.637   9.747 -23.365  1.00  0.51           C  \\nATOM   2636  NZ  LYS A 351     -10.914   8.982 -23.479  1.00  0.47           N  \\nATOM   2637  N   GLY A 352      -8.005  16.356 -20.663  1.00  0.35           N  \\nATOM   2638  CA  GLY A 352      -8.202  17.797 -20.630  1.00  0.40           C  \\nATOM   2639  C   GLY A 352      -6.909  18.572 -20.460  1.00  0.34           C  \\nATOM   2640  O   GLY A 352      -6.694  19.585 -21.129  1.00  0.34           O  \");\n\tviewer_1718032850302779.setStyle({\"model\": -1},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_1718032850302779.addStyle({\"and\": [{\"resn\": [\"GLY\", \"PRO\"], \"invert\": true}, {\"atom\": [\"C\", \"O\", \"N\"], \"invert\": true}]},{\"stick\": {\"colorscheme\": \"WhiteCarbon\", \"radius\": 0.3}});\n\tviewer_1718032850302779.addStyle({\"and\": [{\"resn\": \"GLY\"}, {\"atom\": \"CA\"}]},{\"sphere\": {\"colorscheme\": \"WhiteCarbon\", \"radius\": 0.3}});\n\tviewer_1718032850302779.addStyle({\"and\": [{\"resn\": \"PRO\"}, {\"atom\": [\"C\", \"O\"], \"invert\": true}]},{\"stick\": {\"colorscheme\": \"WhiteCarbon\", \"radius\": 0.3}});\n\tviewer_1718032850302779.addStyle({\"atom\": [\"C\", \"O\", \"N\", \"CA\"]},{\"stick\": {\"colorscheme\": \"WhiteCarbon\", \"radius\": 0.3}});\n\tviewer_1718032850302779.zoomTo();\nviewer_1718032850302779.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_1718032850302779\"  style=\"position: relative; width: 1000px; height: 800px;\">\n",
              "        <p id=\"3dmolwarning_1718032850302779\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.1.0/3Dmol-min.js');\n",
              "}\n",
              "\n",
              "var viewer_1718032850302779 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_1718032850302779\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_1718032850302779 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1718032850302779\"),{backgroundColor:\"white\"});\n",
              "viewer_1718032850302779.zoomTo();\n",
              "\tviewer_1718032850302779.addModelsAsFrames(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   GLY A   1      14.054   1.992 -28.510  1.00  0.29           N  \\nATOM      2  CA  GLY A   1      13.927   1.536 -27.135  1.00  0.28           C  \\nATOM      3  C   GLY A   1      12.545   1.768 -26.554  1.00  0.29           C  \\nATOM      4  O   GLY A   1      11.999   0.899 -25.872  1.00  0.28           O  \\nATOM      5  N   ILE A   2      11.969   2.902 -26.632  1.00  0.31           N  \\nATOM      6  CA  ILE A   2      10.578   3.333 -26.552  1.00  0.32           C  \\nATOM      7  C   ILE A   2       9.677   2.294 -27.216  1.00  0.30           C  \\nATOM      8  CB  ILE A   2      10.377   4.716 -27.210  1.00  0.28           C  \\nATOM      9  O   ILE A   2       8.593   1.991 -26.713  1.00  0.29           O  \\nATOM     10  CG1 ILE A   2      11.142   5.794 -26.433  1.00  0.27           C  \\nATOM     11  CG2 ILE A   2       8.887   5.060 -27.299  1.00  0.26           C  \\nATOM     12  CD1 ILE A   2      11.164   7.153 -27.118  1.00  0.24           C  \\nATOM     13  N   LYS A   3      10.221   1.587 -28.210  1.00  0.30           N  \\nATOM     14  CA  LYS A   3       9.438   0.680 -29.045  1.00  0.29           C  \\nATOM     15  C   LYS A   3       9.137  -0.623 -28.310  1.00  0.30           C  \\nATOM     16  CB  LYS A   3      10.173   0.386 -30.353  1.00  0.27           C  \\nATOM     17  O   LYS A   3       8.023  -1.143 -28.390  1.00  0.28           O  \\nATOM     18  CG  LYS A   3      10.138   1.532 -31.354  1.00  0.26           C  \\nATOM     19  CD  LYS A   3      10.744   1.125 -32.691  1.00  0.28           C  \\nATOM     20  CE  LYS A   3      10.696   2.266 -33.698  1.00  0.26           C  \\nATOM     21  NZ  LYS A   3      11.267   1.864 -35.019  1.00  0.26           N  \\nATOM     22  N   GLU A   4      10.053  -1.107 -27.586  1.00  0.33           N  \\nATOM     23  CA  GLU A   4       9.952  -2.354 -26.834  1.00  0.34           C  \\nATOM     24  C   GLU A   4       9.089  -2.178 -25.587  1.00  0.33           C  \\nATOM     25  CB  GLU A   4      11.342  -2.861 -26.444  1.00  0.31           C  \\nATOM     26  O   GLU A   4       8.254  -3.032 -25.278  1.00  0.31           O  \\nATOM     27  CG  GLU A   4      12.118  -3.479 -27.598  1.00  0.29           C  \\nATOM     28  CD  GLU A   4      13.625  -3.422 -27.407  1.00  0.33           C  \\nATOM     29  OE1 GLU A   4      14.085  -2.940 -26.347  1.00  0.42           O  \\nATOM     30  OE2 GLU A   4      14.352  -3.861 -28.325  1.00  0.40           O  \\nATOM     31  N   GLU A   5       9.097  -0.988 -25.017  1.00  0.31           N  \\nATOM     32  CA  GLU A   5       8.297  -0.529 -23.886  1.00  0.33           C  \\nATOM     33  C   GLU A   5       6.835  -0.347 -24.281  1.00  0.31           C  \\nATOM     34  CB  GLU A   5       8.858   0.782 -23.327  1.00  0.30           C  \\nATOM     35  O   GLU A   5       5.934  -0.811 -23.580  1.00  0.30           O  \\nATOM     36  CG  GLU A   5       9.309   0.689 -21.877  1.00  0.29           C  \\nATOM     37  CD  GLU A   5      10.044   1.930 -21.397  1.00  0.35           C  \\nATOM     38  OE1 GLU A   5      10.160   2.906 -22.173  1.00  0.43           O  \\nATOM     39  OE2 GLU A   5      10.510   1.927 -20.235  1.00  0.39           O  \\nATOM     40  N   ALA A   6       6.447   0.365 -25.332  1.00  0.32           N  \\nATOM     41  CA  ALA A   6       5.143   0.456 -25.985  1.00  0.31           C  \\nATOM     42  C   ALA A   6       4.627  -0.927 -26.372  1.00  0.32           C  \\nATOM     43  CB  ALA A   6       5.225   1.354 -27.216  1.00  0.29           C  \\nATOM     44  O   ALA A   6       3.442  -1.224 -26.205  1.00  0.30           O  \\nATOM     45  N   ALA A   7       5.546  -1.886 -26.764  1.00  0.30           N  \\nATOM     46  CA  ALA A   7       5.157  -3.235 -27.170  1.00  0.28           C  \\nATOM     47  C   ALA A   7       4.765  -4.079 -25.960  1.00  0.30           C  \\nATOM     48  CB  ALA A   7       6.291  -3.907 -27.939  1.00  0.27           C  \\nATOM     49  O   ALA A   7       3.751  -4.781 -25.988  1.00  0.28           O  \\nATOM     50  N   ALA A   8       5.551  -4.034 -24.890  1.00  0.32           N  \\nATOM     51  CA  ALA A   8       5.183  -4.760 -23.677  1.00  0.32           C  \\nATOM     52  C   ALA A   8       3.884  -4.218 -23.087  1.00  0.32           C  \\nATOM     53  CB  ALA A   8       6.307  -4.678 -22.647  1.00  0.30           C  \\nATOM     54  O   ALA A   8       3.001  -4.988 -22.702  1.00  0.30           O  \\nATOM     55  N   ILE A   9       3.607  -2.874 -23.042  1.00  0.33           N  \\nATOM     56  CA  ILE A   9       2.394  -2.149 -22.679  1.00  0.33           C  \\nATOM     57  C   ILE A   9       1.249  -2.565 -23.600  1.00  0.33           C  \\nATOM     58  CB  ILE A   9       2.608  -0.620 -22.748  1.00  0.31           C  \\nATOM     59  O   ILE A   9       0.134  -2.818 -23.138  1.00  0.31           O  \\nATOM     60  CG1 ILE A   9       3.600  -0.170 -21.669  1.00  0.29           C  \\nATOM     61  CG2 ILE A   9       1.274   0.119 -22.608  1.00  0.29           C  \\nATOM     62  CD1 ILE A   9       4.100   1.257 -21.844  1.00  0.26           C  \\nATOM     63  N   LYS A  10       1.492  -2.650 -24.954  1.00  0.29           N  \\nATOM     64  CA  LYS A  10       0.531  -3.092 -25.960  1.00  0.28           C  \\nATOM     65  C   LYS A  10       0.133  -4.548 -25.737  1.00  0.29           C  \\nATOM     66  CB  LYS A  10       1.106  -2.916 -27.366  1.00  0.27           C  \\nATOM     67  O   LYS A  10      -1.038  -4.906 -25.880  1.00  0.27           O  \\nATOM     68  CG  LYS A  10       0.688  -1.621 -28.048  1.00  0.26           C  \\nATOM     69  CD  LYS A  10       1.273  -1.513 -29.450  1.00  0.29           C  \\nATOM     70  CE  LYS A  10       0.875  -0.208 -30.124  1.00  0.23           C  \\nATOM     71  NZ  LYS A  10       1.456  -0.092 -31.495  1.00  0.25           N  \\nATOM     72  N   TRP A  11       1.075  -5.423 -25.370  1.00  0.15           N  \\nATOM     73  CA  TRP A  11       0.787  -6.835 -25.137  1.00  0.26           C  \\nATOM     74  C   TRP A  11      -0.048  -7.019 -23.875  1.00  0.20           C  \\nATOM     75  CB  TRP A  11       2.086  -7.638 -25.026  1.00  0.17           C  \\nATOM     76  O   TRP A  11      -1.051  -7.737 -23.886  1.00  0.19           O  \\nATOM     77  CG  TRP A  11       2.185  -8.774 -26.000  1.00  0.17           C  \\nATOM     78  CD1 TRP A  11       2.699  -8.731 -27.266  1.00  0.09           C  \\nATOM     79  CD2 TRP A  11       1.754 -10.121 -25.786  1.00  0.09           C  \\nATOM     80  CE2 TRP A  11       2.037 -10.843 -26.967  1.00  0.08           C  \\nATOM     81  CE3 TRP A  11       1.155 -10.789 -24.710  1.00  0.18           C  \\nATOM     82  NE1 TRP A  11       2.613  -9.973 -27.853  1.00  0.19           N  \\nATOM     83  CH2 TRP A  11       1.156 -12.831 -26.033  1.00  0.17           C  \\nATOM     84  CZ2 TRP A  11       1.742 -12.201 -27.101  1.00  0.17           C  \\nATOM     85  CZ3 TRP A  11       0.861 -12.141 -24.845  1.00  0.17           C  \\nATOM     86  N   GLU A  12       0.247  -6.368 -22.787  1.00  0.32           N  \\nATOM     87  CA  GLU A  12      -0.506  -6.482 -21.542  1.00  0.33           C  \\nATOM     88  C   GLU A  12      -1.869  -5.805 -21.656  1.00  0.32           C  \\nATOM     89  CB  GLU A  12       0.285  -5.881 -20.377  1.00  0.31           C  \\nATOM     90  O   GLU A  12      -2.880  -6.353 -21.213  1.00  0.31           O  \\nATOM     91  CG  GLU A  12       0.732  -6.904 -19.344  1.00  0.30           C  \\nATOM     92  CD  GLU A  12       1.533  -6.295 -18.204  1.00  0.38           C  \\nATOM     93  OE1 GLU A  12       1.792  -5.070 -18.230  1.00  0.44           O  \\nATOM     94  OE2 GLU A  12       1.905  -7.048 -17.277  1.00  0.40           O  \\nATOM     95  N   ALA A  13      -2.054  -4.669 -22.299  1.00  0.32           N  \\nATOM     96  CA  ALA A  13      -3.344  -4.077 -22.641  1.00  0.32           C  \\nATOM     97  C   ALA A  13      -4.130  -4.981 -23.587  1.00  0.32           C  \\nATOM     98  CB  ALA A  13      -3.148  -2.699 -23.269  1.00  0.30           C  \\nATOM     99  O   ALA A  13      -5.337  -5.166 -23.419  1.00  0.30           O  \\nATOM    100  N   ALA A  14      -3.513  -5.635 -24.659  1.00  0.29           N  \\nATOM    101  CA  ALA A  14      -4.132  -6.572 -25.592  1.00  0.27           C  \\nATOM    102  C   ALA A  14      -4.556  -7.855 -24.883  1.00  0.28           C  \\nATOM    103  CB  ALA A  14      -3.175  -6.893 -26.738  1.00  0.26           C  \\nATOM    104  O   ALA A  14      -5.668  -8.347 -25.087  1.00  0.27           O  \\nATOM    105  N   ALA A  15      -3.634  -8.495 -24.148  1.00  0.29           N  \\nATOM    106  CA  ALA A  15      -4.033  -9.686 -23.403  1.00  0.28           C  \\nATOM    107  C   ALA A  15      -5.204  -9.384 -22.472  1.00  0.29           C  \\nATOM    108  CB  ALA A  15      -2.852 -10.236 -22.606  1.00  0.26           C  \\nATOM    109  O   ALA A  15      -6.146 -10.173 -22.373  1.00  0.27           O  \\nATOM    110  N   ILE A  16      -5.333  -8.232 -21.963  1.00  0.32           N  \\nATOM    111  CA  ILE A  16      -6.384  -7.712 -21.095  1.00  0.32           C  \\nATOM    112  C   ILE A  16      -7.640  -7.432 -21.917  1.00  0.32           C  \\nATOM    113  CB  ILE A  16      -5.928  -6.433 -20.359  1.00  0.29           C  \\nATOM    114  O   ILE A  16      -8.746  -7.803 -21.518  1.00  0.30           O  \\nATOM    115  CG1 ILE A  16      -4.812  -6.760 -19.359  1.00  0.29           C  \\nATOM    116  CG2 ILE A  16      -7.112  -5.762 -19.657  1.00  0.28           C  \\nATOM    117  CD1 ILE A  16      -4.124  -5.534 -18.775  1.00  0.25           C  \\nATOM    118  N   LYS A  17      -7.509  -6.714 -23.002  1.00  0.29           N  \\nATOM    119  CA  LYS A  17      -8.629  -6.459 -23.903  1.00  0.29           C  \\nATOM    120  C   LYS A  17      -9.260  -7.765 -24.378  1.00  0.29           C  \\nATOM    121  CB  LYS A  17      -8.174  -5.629 -25.104  1.00  0.27           C  \\nATOM    122  O   LYS A  17     -10.472  -7.833 -24.592  1.00  0.27           O  \\nATOM    123  CG  LYS A  17      -8.875  -4.285 -25.230  1.00  0.26           C  \\nATOM    124  CD  LYS A  17      -8.381  -3.507 -26.442  1.00  0.30           C  \\nATOM    125  CE  LYS A  17      -9.110  -2.179 -26.592  1.00  0.28           C  \\nATOM    126  NZ  LYS A  17      -8.654  -1.429 -27.800  1.00  0.29           N  \\nATOM    127  N   GLU A  18      -8.310  -8.807 -24.381  1.00  0.26           N  \\nATOM    128  CA  GLU A  18      -8.756 -10.100 -24.889  1.00  0.26           C  \\nATOM    129  C   GLU A  18      -9.252 -10.997 -23.758  1.00  0.26           C  \\nATOM    130  CB  GLU A  18      -7.628 -10.795 -25.655  1.00  0.25           C  \\nATOM    131  O   GLU A  18      -9.805 -12.070 -24.007  1.00  0.25           O  \\nATOM    132  CG  GLU A  18      -7.448 -10.289 -27.079  1.00  0.25           C  \\nATOM    133  CD  GLU A  18      -6.359 -11.025 -27.845  1.00  0.31           C  \\nATOM    134  OE1 GLU A  18      -5.770 -11.982 -27.293  1.00  0.41           O  \\nATOM    135  OE2 GLU A  18      -6.093 -10.640 -29.005  1.00  0.34           O  \\nATOM    136  N   GLY A  19      -9.500 -10.337 -22.474  1.00  0.29           N  \\nATOM    137  CA  GLY A  19      -9.920 -11.183 -21.368  1.00  0.29           C  \\nATOM    138  C   GLY A  19      -8.879 -12.215 -20.977  1.00  0.29           C  \\nATOM    139  O   GLY A  19      -9.182 -13.170 -20.260  1.00  0.28           O  \\nATOM    140  N   ALA A  20      -7.678 -12.067 -21.631  1.00  0.26           N  \\nATOM    141  CA  ALA A  20      -6.557 -12.912 -21.231  1.00  0.26           C  \\nATOM    142  C   ALA A  20      -5.891 -12.377 -19.966  1.00  0.26           C  \\nATOM    143  CB  ALA A  20      -5.537 -13.015 -22.362  1.00  0.24           C  \\nATOM    144  O   ALA A  20      -5.494 -11.211 -19.911  1.00  0.25           O  \\nATOM    145  N   SER A  21      -6.316 -12.763 -18.670  1.00  0.31           N  \\nATOM    146  CA  SER A  21      -5.501 -12.766 -17.460  1.00  0.32           C  \\nATOM    147  C   SER A  21      -4.111 -13.333 -17.730  1.00  0.31           C  \\nATOM    148  CB  SER A  21      -6.183 -13.573 -16.355  1.00  0.29           C  \\nATOM    149  O   SER A  21      -3.978 -14.446 -18.242  1.00  0.30           O  \\nATOM    150  OG  SER A  21      -6.884 -12.719 -15.467  1.00  0.30           O  \\nATOM    151  N   PRO A  22      -3.102 -12.474 -18.019  1.00  0.31           N  \\nATOM    152  CA  PRO A  22      -1.757 -13.055 -18.016  1.00  0.31           C  \\nATOM    153  C   PRO A  22      -1.530 -14.011 -16.847  1.00  0.31           C  \\nATOM    154  CB  PRO A  22      -0.844 -11.831 -17.905  1.00  0.29           C  \\nATOM    155  O   PRO A  22      -2.027 -13.772 -15.744  1.00  0.30           O  \\nATOM    156  CG  PRO A  22      -1.758 -10.705 -17.541  1.00  0.28           C  \\nATOM    157  CD  PRO A  22      -3.177 -11.169 -17.711  1.00  0.32           C  \\nATOM    158  N   GLU A  23      -1.577 -15.397 -16.990  1.00  0.29           N  \\nATOM    159  CA  GLU A  23      -1.080 -16.419 -16.074  1.00  0.30           C  \\nATOM    160  C   GLU A  23      -0.081 -15.833 -15.081  1.00  0.29           C  \\nATOM    161  CB  GLU A  23      -0.436 -17.570 -16.851  1.00  0.27           C  \\nATOM    162  O   GLU A  23       0.835 -15.104 -15.471  1.00  0.28           O  \\nATOM    163  CG  GLU A  23      -1.137 -18.907 -16.665  1.00  0.27           C  \\nATOM    164  CD  GLU A  23      -0.475 -20.044 -17.426  1.00  0.32           C  \\nATOM    165  OE1 GLU A  23       0.538 -19.801 -18.120  1.00  0.44           O  \\nATOM    166  OE2 GLU A  23      -0.974 -21.188 -17.329  1.00  0.36           O  \\nATOM    167  N   PRO A  24      -0.412 -15.636 -13.713  1.00  0.31           N  \\nATOM    168  CA  PRO A  24       0.631 -15.268 -12.753  1.00  0.31           C  \\nATOM    169  C   PRO A  24       1.952 -15.989 -13.012  1.00  0.31           C  \\nATOM    170  CB  PRO A  24       0.035 -15.687 -11.406  1.00  0.29           C  \\nATOM    171  O   PRO A  24       1.987 -17.220 -13.070  1.00  0.30           O  \\nATOM    172  CG  PRO A  24      -1.156 -16.519 -11.758  1.00  0.28           C  \\nATOM    173  CD  PRO A  24      -1.420 -16.375 -13.230  1.00  0.32           C  \\nATOM    174  N   GLN A  25       2.851 -15.400 -13.681  1.00  0.30           N  \\nATOM    175  CA  GLN A  25       4.231 -15.857 -13.805  1.00  0.31           C  \\nATOM    176  C   GLN A  25       4.933 -15.861 -12.450  1.00  0.30           C  \\nATOM    177  CB  GLN A  25       5.001 -14.979 -14.793  1.00  0.28           C  \\nATOM    178  O   GLN A  25       4.614 -15.051 -11.577  1.00  0.29           O  \\nATOM    179  CG  GLN A  25       4.619 -15.210 -16.249  1.00  0.28           C  \\nATOM    180  CD  GLN A  25       5.826 -15.324 -17.161  1.00  0.34           C  \\nATOM    181  NE2 GLN A  25       5.632 -15.926 -18.329  1.00  0.32           N  \\nATOM    182  OE1 GLN A  25       6.924 -14.873 -16.819  1.00  0.44           O  \\nATOM    183  N   PRO A  26       5.617 -17.204 -11.914  1.00  0.29           N  \\nATOM    184  CA  PRO A  26       6.365 -17.533 -10.698  1.00  0.28           C  \\nATOM    185  C   PRO A  26       6.950 -16.300 -10.014  1.00  0.29           C  \\nATOM    186  CB  PRO A  26       7.476 -18.455 -11.206  1.00  0.27           C  \\nATOM    187  O   PRO A  26       7.461 -15.400 -10.687  1.00  0.28           O  \\nATOM    188  CG  PRO A  26       7.315 -18.466 -12.691  1.00  0.26           C  \\nATOM    189  CD  PRO A  26       6.102 -17.654 -13.046  1.00  0.30           C  \\nATOM    190  N   LYS A  27       6.378 -15.693  -8.938  1.00  0.29           N  \\nATOM    191  CA  LYS A  27       6.788 -14.831  -7.833  1.00  0.28           C  \\nATOM    192  C   LYS A  27       8.279 -14.516  -7.906  1.00  0.30           C  \\nATOM    193  CB  LYS A  27       6.454 -15.483  -6.490  1.00  0.27           C  \\nATOM    194  O   LYS A  27       9.111 -15.426  -7.925  1.00  0.29           O  \\nATOM    195  CG  LYS A  27       4.977 -15.437  -6.130  1.00  0.26           C  \\nATOM    196  CD  LYS A  27       4.726 -15.983  -4.730  1.00  0.32           C  \\nATOM    197  CE  LYS A  27       3.243 -15.979  -4.385  1.00  0.26           C  \\nATOM    198  NZ  LYS A  27       2.991 -16.523  -3.017  1.00  0.27           N  \\nATOM    199  N   GLY A  28       8.719 -13.633  -8.914  1.00  0.29           N  \\nATOM    200  CA  GLY A  28      10.032 -13.010  -8.964  1.00  0.29           C  \\nATOM    201  C   GLY A  28      10.279 -12.242 -10.249  1.00  0.29           C  \\nATOM    202  O   GLY A  28      10.881 -11.166 -10.229  1.00  0.27           O  \\nATOM    203  N   ILE A  29       9.711 -12.515 -11.326  1.00  0.27           N  \\nATOM    204  CA  ILE A  29      10.127 -11.856 -12.559  1.00  0.27           C  \\nATOM    205  C   ILE A  29       9.187 -10.691 -12.864  1.00  0.27           C  \\nATOM    206  CB  ILE A  29      10.157 -12.845 -13.746  1.00  0.24           C  \\nATOM    207  O   ILE A  29       9.633  -9.614 -13.265  1.00  0.25           O  \\nATOM    208  CG1 ILE A  29      11.240 -13.908 -13.528  1.00  0.15           C  \\nATOM    209  CG2 ILE A  29      10.381 -12.098 -15.065  1.00  0.21           C  \\nATOM    210  CD1 ILE A  29      11.308 -14.957 -14.629  1.00  0.19           C  \\nATOM    211  N   LYS A  30       7.975 -10.677 -12.424  1.00  0.31           N  \\nATOM    212  CA  LYS A  30       7.138  -9.537 -12.789  1.00  0.32           C  \\nATOM    213  C   LYS A  30       7.284  -8.402 -11.779  1.00  0.31           C  \\nATOM    214  CB  LYS A  30       5.672  -9.960 -12.895  1.00  0.28           C  \\nATOM    215  O   LYS A  30       7.231  -7.227 -12.147  1.00  0.29           O  \\nATOM    216  CG  LYS A  30       5.330 -10.692 -14.184  1.00  0.27           C  \\nATOM    217  CD  LYS A  30       3.834 -10.950 -14.301  1.00  0.31           C  \\nATOM    218  CE  LYS A  30       3.490 -11.682 -15.591  1.00  0.26           C  \\nATOM    219  NZ  LYS A  30       2.023 -11.935 -15.711  1.00  0.27           N  \\nATOM    220  N   GLU A  31       7.593  -8.709 -10.567  1.00  0.34           N  \\nATOM    221  CA  GLU A  31       7.854  -7.628  -9.621  1.00  0.35           C  \\nATOM    222  C   GLU A  31       9.167  -6.920  -9.941  1.00  0.34           C  \\nATOM    223  CB  GLU A  31       7.880  -8.163  -8.187  1.00  0.32           C  \\nATOM    224  O   GLU A  31       9.243  -5.691  -9.894  1.00  0.32           O  \\nATOM    225  CG  GLU A  31       6.512  -8.567  -7.657  1.00  0.30           C  \\nATOM    226  CD  GLU A  31       6.576  -9.293  -6.322  1.00  0.35           C  \\nATOM    227  OE1 GLU A  31       7.691  -9.469  -5.780  1.00  0.40           O  \\nATOM    228  OE2 GLU A  31       5.503  -9.689  -5.815  1.00  0.40           O  \\nATOM    229  N   GLU A  32      10.173  -7.705 -10.436  1.00  0.26           N  \\nATOM    230  CA  GLU A  32      11.448  -7.178 -10.912  1.00  0.27           C  \\nATOM    231  C   GLU A  32      11.274  -6.413 -12.221  1.00  0.26           C  \\nATOM    232  CB  GLU A  32      12.463  -8.310 -11.096  1.00  0.24           C  \\nATOM    233  O   GLU A  32      11.800  -5.308 -12.374  1.00  0.25           O  \\nATOM    234  CG  GLU A  32      13.490  -8.402  -9.976  1.00  0.23           C  \\nATOM    235  CD  GLU A  32      14.447  -9.573 -10.135  1.00  0.27           C  \\nATOM    236  OE1 GLU A  32      14.355 -10.298 -11.152  1.00  0.35           O  \\nATOM    237  OE2 GLU A  32      15.294  -9.767  -9.236  1.00  0.28           O  \\nATOM    238  N   ALA A  33      10.473  -6.945 -13.111  1.00  0.28           N  \\nATOM    239  CA  ALA A  33      10.237  -6.255 -14.376  1.00  0.28           C  \\nATOM    240  C   ALA A  33       9.358  -5.024 -14.174  1.00  0.28           C  \\nATOM    241  CB  ALA A  33       9.595  -7.203 -15.387  1.00  0.26           C  \\nATOM    242  O   ALA A  33       9.598  -3.976 -14.779  1.00  0.26           O  \\nATOM    243  N   ALA A  34       8.446  -5.077 -13.204  1.00  0.32           N  \\nATOM    244  CA  ALA A  34       7.665  -3.881 -12.898  1.00  0.32           C  \\nATOM    245  C   ALA A  34       8.522  -2.831 -12.197  1.00  0.32           C  \\nATOM    246  CB  ALA A  34       6.457  -4.240 -12.036  1.00  0.30           C  \\nATOM    247  O   ALA A  34       8.462  -1.646 -12.533  1.00  0.30           O  \\nATOM    248  N   ALA A  35       9.400  -3.264 -11.278  1.00  0.28           N  \\nATOM    249  CA  ALA A  35      10.353  -2.328 -10.686  1.00  0.28           C  \\nATOM    250  C   ALA A  35      11.302  -1.771 -11.743  1.00  0.28           C  \\nATOM    251  CB  ALA A  35      11.143  -3.008  -9.571  1.00  0.26           C  \\nATOM    252  O   ALA A  35      11.583  -0.570 -11.763  1.00  0.27           O  \\nATOM    253  N   ILE A  36      11.901  -2.567 -12.729  1.00  0.26           N  \\nATOM    254  CA  ILE A  36      12.829  -2.167 -13.781  1.00  0.25           C  \\nATOM    255  C   ILE A  36      12.127  -1.227 -14.759  1.00  0.26           C  \\nATOM    256  CB  ILE A  36      13.395  -3.394 -14.531  1.00  0.24           C  \\nATOM    257  O   ILE A  36      12.683  -0.198 -15.148  1.00  0.24           O  \\nATOM    258  CG1 ILE A  36      14.279  -4.229 -13.597  1.00  0.19           C  \\nATOM    259  CG2 ILE A  36      14.175  -2.954 -15.774  1.00  0.22           C  \\nATOM    260  CD1 ILE A  36      14.675  -5.584 -14.168  1.00  0.18           C  \\nATOM    261  N   LYS A  37      10.932  -1.542 -15.151  1.00  0.29           N  \\nATOM    262  CA  LYS A  37      10.252  -0.681 -16.115  1.00  0.29           C  \\nATOM    263  C   LYS A  37       9.972   0.695 -15.519  1.00  0.29           C  \\nATOM    264  CB  LYS A  37       8.947  -1.326 -16.583  1.00  0.27           C  \\nATOM    265  O   LYS A  37      10.032   1.706 -16.223  1.00  0.28           O  \\nATOM    266  CG  LYS A  37       9.104  -2.235 -17.793  1.00  0.26           C  \\nATOM    267  CD  LYS A  37       7.765  -2.807 -18.240  1.00  0.31           C  \\nATOM    268  CE  LYS A  37       7.925  -3.733 -19.439  1.00  0.24           C  \\nATOM    269  NZ  LYS A  37       6.618  -4.319 -19.864  1.00  0.27           N  \\nATOM    270  N   TRP A  38       9.957   0.852 -14.172  1.00  0.16           N  \\nATOM    271  CA  TRP A  38       9.678   2.096 -13.461  1.00  0.27           C  \\nATOM    272  C   TRP A  38      10.912   2.991 -13.426  1.00  0.18           C  \\nATOM    273  CB  TRP A  38       9.202   1.806 -12.035  1.00  0.19           C  \\nATOM    274  O   TRP A  38      10.819   4.198 -13.662  1.00  0.18           O  \\nATOM    275  CG  TRP A  38       7.717   1.634 -11.913  1.00  0.21           C  \\nATOM    276  CD1 TRP A  38       7.035   0.463 -11.733  1.00  0.20           C  \\nATOM    277  CD2 TRP A  38       6.731   2.669 -11.967  1.00  0.18           C  \\nATOM    278  CE2 TRP A  38       5.469   2.053 -11.811  1.00  0.12           C  \\nATOM    279  CE3 TRP A  38       6.791   4.059 -12.131  1.00  0.21           C  \\nATOM    280  NE1 TRP A  38       5.683   0.708 -11.670  1.00  0.21           N  \\nATOM    281  CH2 TRP A  38       4.364   4.139 -11.977  1.00  0.22           C  \\nATOM    282  CZ2 TRP A  38       4.277   2.780 -11.815  1.00  0.21           C  \\nATOM    283  CZ3 TRP A  38       5.603   4.782 -12.134  1.00  0.22           C  \\nATOM    284  N   GLU A  39      11.981   2.475 -13.177  1.00  0.25           N  \\nATOM    285  CA  GLU A  39      13.180   3.301 -13.280  1.00  0.26           C  \\nATOM    286  C   GLU A  39      13.283   3.957 -14.654  1.00  0.25           C  \\nATOM    287  CB  GLU A  39      14.433   2.466 -13.001  1.00  0.23           C  \\nATOM    288  O   GLU A  39      13.572   5.150 -14.757  1.00  0.24           O  \\nATOM    289  CG  GLU A  39      15.127   2.818 -11.693  1.00  0.23           C  \\nATOM    290  CD  GLU A  39      16.349   1.957 -11.411  1.00  0.27           C  \\nATOM    291  OE1 GLU A  39      16.652   1.051 -12.221  1.00  0.36           O  \\nATOM    292  OE2 GLU A  39      17.006   2.189 -10.372  1.00  0.31           O  \\nATOM    293  N   ALA A  40      12.822   3.281 -15.597  1.00  0.25           N  \\nATOM    294  CA  ALA A  40      13.002   3.815 -16.945  1.00  0.25           C  \\nATOM    295  C   ALA A  40      11.956   4.881 -17.257  1.00  0.25           C  \\nATOM    296  CB  ALA A  40      12.935   2.690 -17.976  1.00  0.23           C  \\nATOM    297  O   ALA A  40      12.272   5.915 -17.850  1.00  0.24           O  \\nATOM    298  N   ALA A  41      10.688   4.830 -16.731  1.00  0.29           N  \\nATOM    299  CA  ALA A  41       9.703   5.871 -17.016  1.00  0.28           C  \\nATOM    300  C   ALA A  41       9.908   7.083 -16.111  1.00  0.29           C  \\nATOM    301  CB  ALA A  41       8.288   5.323 -16.853  1.00  0.26           C  \\nATOM    302  O   ALA A  41       9.820   8.226 -16.565  1.00  0.27           O  \\nATOM    303  N   ALA A  42      10.249   6.911 -14.830  1.00  0.27           N  \\nATOM    304  CA  ALA A  42      10.556   8.054 -13.975  1.00  0.27           C  \\nATOM    305  C   ALA A  42      11.792   8.797 -14.474  1.00  0.27           C  \\nATOM    306  CB  ALA A  42      10.759   7.600 -12.531  1.00  0.25           C  \\nATOM    307  O   ALA A  42      11.832  10.029 -14.457  1.00  0.25           O  \\nATOM    308  N   ILE A  43      12.828   8.168 -14.976  1.00  0.25           N  \\nATOM    309  CA  ILE A  43      14.026   8.757 -15.564  1.00  0.24           C  \\nATOM    310  C   ILE A  43      13.660   9.505 -16.844  1.00  0.25           C  \\nATOM    311  CB  ILE A  43      15.098   7.683 -15.858  1.00  0.23           C  \\nATOM    312  O   ILE A  43      14.148  10.611 -17.085  1.00  0.23           O  \\nATOM    313  CG1 ILE A  43      15.629   7.086 -14.549  1.00  0.23           C  \\nATOM    314  CG2 ILE A  43      16.238   8.273 -16.693  1.00  0.22           C  \\nATOM    315  CD1 ILE A  43      16.509   5.859 -14.741  1.00  0.18           C  \\nATOM    316  N   LYS A  44      12.641   9.133 -17.538  1.00  0.26           N  \\nATOM    317  CA  LYS A  44      12.516   9.817 -18.822  1.00  0.27           C  \\nATOM    318  C   LYS A  44      11.576  11.015 -18.718  1.00  0.26           C  \\nATOM    319  CB  LYS A  44      12.018   8.852 -19.899  1.00  0.24           C  \\nATOM    320  O   LYS A  44      11.707  11.981 -19.472  1.00  0.25           O  \\nATOM    321  CG  LYS A  44      13.102   8.382 -20.858  1.00  0.24           C  \\nATOM    322  CD  LYS A  44      12.538   7.457 -21.929  1.00  0.31           C  \\nATOM    323  CE  LYS A  44      13.622   6.986 -22.889  1.00  0.24           C  \\nATOM    324  NZ  LYS A  44      13.076   6.072 -23.936  1.00  0.26           N  \\nATOM    325  N   GLU A  45      10.712  11.024 -17.700  1.00  0.29           N  \\nATOM    326  CA  GLU A  45       9.768  12.136 -17.639  1.00  0.30           C  \\nATOM    327  C   GLU A  45      10.182  13.152 -16.578  1.00  0.29           C  \\nATOM    328  CB  GLU A  45       8.353  11.626 -17.355  1.00  0.27           C  \\nATOM    329  O   GLU A  45       9.426  14.076 -16.268  1.00  0.28           O  \\nATOM    330  CG  GLU A  45       7.627  11.108 -18.588  1.00  0.27           C  \\nATOM    331  CD  GLU A  45       6.188  10.703 -18.311  1.00  0.33           C  \\nATOM    332  OE1 GLU A  45       5.729  10.847 -17.155  1.00  0.43           O  \\nATOM    333  OE2 GLU A  45       5.515  10.238 -19.257  1.00  0.37           O  \\nATOM    334  N   GLY A  46      11.471  13.302 -16.371  1.00  0.30           N  \\nATOM    335  CA  GLY A  46      12.158  14.303 -15.570  1.00  0.31           C  \\nATOM    336  C   GLY A  46      11.848  14.197 -14.089  1.00  0.30           C  \\nATOM    337  O   GLY A  46      12.179  15.097 -13.314  1.00  0.29           O  \\nATOM    338  N   ALA A  47      11.076  13.088 -13.648  1.00  0.31           N  \\nATOM    339  CA  ALA A  47      11.098  12.750 -12.227  1.00  0.32           C  \\nATOM    340  C   ALA A  47      12.291  11.859 -11.894  1.00  0.31           C  \\nATOM    341  CB  ALA A  47       9.795  12.063 -11.823  1.00  0.28           C  \\nATOM    342  O   ALA A  47      12.423  10.759 -12.436  1.00  0.29           O  \\nATOM    343  N   SER A  48      13.477  12.323 -11.780  1.00  0.29           N  \\nATOM    344  CA  SER A  48      14.624  11.690 -11.137  1.00  0.31           C  \\nATOM    345  C   SER A  48      14.247  11.117  -9.775  1.00  0.30           C  \\nATOM    346  CB  SER A  48      15.770  12.690 -10.979  1.00  0.28           C  \\nATOM    347  O   SER A  48      14.042  11.864  -8.816  1.00  0.29           O  \\nATOM    348  OG  SER A  48      16.733  12.518 -12.005  1.00  0.29           O  \\nATOM    349  N   PRO A  49      13.521  10.015  -9.721  1.00  0.35           N  \\nATOM    350  CA  PRO A  49      13.822   9.335  -8.459  1.00  0.34           C  \\nATOM    351  C   PRO A  49      15.319   9.116  -8.250  1.00  0.35           C  \\nATOM    352  CB  PRO A  49      13.088   7.999  -8.595  1.00  0.33           C  \\nATOM    353  O   PRO A  49      16.056   8.902  -9.215  1.00  0.33           O  \\nATOM    354  CG  PRO A  49      12.841   7.846 -10.061  1.00  0.31           C  \\nATOM    355  CD  PRO A  49      13.000   9.189 -10.714  1.00  0.34           C  \\nATOM    356  N   GLU A  50      15.953   9.850  -7.283  1.00  0.36           N  \\nATOM    357  CA  GLU A  50      17.274   9.408  -6.847  1.00  0.37           C  \\nATOM    358  C   GLU A  50      17.444   7.903  -7.037  1.00  0.36           C  \\nATOM    359  CB  GLU A  50      17.510   9.782  -5.381  1.00  0.34           C  \\nATOM    360  O   GLU A  50      16.604   7.117  -6.592  1.00  0.35           O  \\nATOM    361  CG  GLU A  50      18.976   9.775  -4.973  1.00  0.33           C  \\nATOM    362  CD  GLU A  50      19.212  10.330  -3.577  1.00  0.38           C  \\nATOM    363  OE1 GLU A  50      18.227  10.669  -2.884  1.00  0.48           O  \\nATOM    364  OE2 GLU A  50      20.393  10.426  -3.174  1.00  0.43           O  \\nATOM    365  N   PRO A  51      17.825   7.421  -8.338  1.00  0.34           N  \\nATOM    366  CA  PRO A  51      18.100   5.982  -8.361  1.00  0.34           C  \\nATOM    367  C   PRO A  51      18.328   5.403  -6.967  1.00  0.34           C  \\nATOM    368  CB  PRO A  51      19.370   5.880  -9.210  1.00  0.32           C  \\nATOM    369  O   PRO A  51      19.269   5.800  -6.276  1.00  0.33           O  \\nATOM    370  CG  PRO A  51      19.932   7.264  -9.224  1.00  0.31           C  \\nATOM    371  CD  PRO A  51      18.868   8.216  -8.759  1.00  0.34           C  \\nATOM    372  N   GLN A  52      17.356   4.988  -6.251  1.00  0.37           N  \\nATOM    373  CA  GLN A  52      17.717   4.002  -5.237  1.00  0.39           C  \\nATOM    374  C   GLN A  52      18.559   2.880  -5.837  1.00  0.37           C  \\nATOM    375  CB  GLN A  52      16.463   3.423  -4.580  1.00  0.36           C  \\nATOM    376  O   GLN A  52      18.282   2.411  -6.943  1.00  0.37           O  \\nATOM    377  CG  GLN A  52      15.879   4.305  -3.485  1.00  0.36           C  \\nATOM    378  CD  GLN A  52      15.131   3.513  -2.429  1.00  0.42           C  \\nATOM    379  NE2 GLN A  52      15.000   4.088  -1.238  1.00  0.38           N  \\nATOM    380  OE1 GLN A  52      14.673   2.394  -2.682  1.00  0.48           O  \\nATOM    381  N   PRO A  53      19.936   2.898  -5.494  1.00  0.37           N  \\nATOM    382  CA  PRO A  53      20.773   1.780  -5.932  1.00  0.36           C  \\nATOM    383  C   PRO A  53      19.972   0.502  -6.175  1.00  0.37           C  \\nATOM    384  CB  PRO A  53      21.754   1.597  -4.771  1.00  0.35           C  \\nATOM    385  O   PRO A  53      19.132   0.131  -5.352  1.00  0.35           O  \\nATOM    386  CG  PRO A  53      21.106   2.283  -3.612  1.00  0.33           C  \\nATOM    387  CD  PRO A  53      20.069   3.233  -4.140  1.00  0.36           C  \\nATOM    388  N   LYS A  54      19.468   0.190  -7.413  1.00  0.39           N  \\nATOM    389  CA  LYS A  54      19.186  -1.215  -7.689  1.00  0.39           C  \\nATOM    390  C   LYS A  54      19.813  -2.120  -6.633  1.00  0.39           C  \\nATOM    391  CB  LYS A  54      19.694  -1.602  -9.079  1.00  0.37           C  \\nATOM    392  O   LYS A  54      21.001  -1.995  -6.326  1.00  0.37           O  \\nATOM    393  CG  LYS A  54      18.610  -1.644 -10.145  1.00  0.35           C  \\nATOM    394  CD  LYS A  54      19.166  -2.089 -11.492  1.00  0.40           C  \\nATOM    395  CE  LYS A  54      18.083  -2.126 -12.561  1.00  0.36           C  \\nATOM    396  NZ  LYS A  54      18.621  -2.573 -13.880  1.00  0.38           N  \\nATOM    397  N   PRO A  55      19.090  -2.763  -5.652  1.00  0.40           N  \\nATOM    398  CA  PRO A  55      19.791  -3.970  -5.210  1.00  0.39           C  \\nATOM    399  C   PRO A  55      20.148  -4.902  -6.365  1.00  0.40           C  \\nATOM    400  CB  PRO A  55      18.787  -4.634  -4.263  1.00  0.38           C  \\nATOM    401  O   PRO A  55      19.257  -5.431  -7.035  1.00  0.38           O  \\nATOM    402  CG  PRO A  55      17.479  -3.984  -4.578  1.00  0.36           C  \\nATOM    403  CD  PRO A  55      17.743  -2.741  -5.377  1.00  0.40           C  \\nATOM    404  N   SER A  56      20.950  -4.487  -7.307  1.00  0.36           N  \\nATOM    405  CA  SER A  56      21.726  -5.453  -8.077  1.00  0.38           C  \\nATOM    406  C   SER A  56      22.249  -6.577  -7.188  1.00  0.36           C  \\nATOM    407  CB  SER A  56      22.896  -4.764  -8.781  1.00  0.35           C  \\nATOM    408  O   SER A  56      23.151  -6.364  -6.375  1.00  0.35           O  \\nATOM    409  OG  SER A  56      22.434  -3.708  -9.606  1.00  0.36           O  \\nATOM    410  N   GLY A  57      21.409  -7.196  -6.538  1.00  0.36           N  \\nATOM    411  CA  GLY A  57      21.830  -8.481  -6.003  1.00  0.37           C  \\nATOM    412  C   GLY A  57      20.860  -9.604  -6.318  1.00  0.37           C  \\nATOM    413  O   GLY A  57      19.670  -9.508  -6.010  1.00  0.35           O  \\nATOM    414  N   ASP A  58      20.829 -10.239  -7.547  1.00  0.40           N  \\nATOM    415  CA  ASP A  58      20.525 -11.667  -7.535  1.00  0.40           C  \\nATOM    416  C   ASP A  58      20.357 -12.181  -6.107  1.00  0.40           C  \\nATOM    417  CB  ASP A  58      21.622 -12.455  -8.252  1.00  0.38           C  \\nATOM    418  O   ASP A  58      21.239 -11.991  -5.266  1.00  0.39           O  \\nATOM    419  CG  ASP A  58      21.175 -13.008  -9.594  1.00  0.38           C  \\nATOM    420  OD1 ASP A  58      19.971 -13.298  -9.764  1.00  0.42           O  \\nATOM    421  OD2 ASP A  58      22.035 -13.156 -10.490  1.00  0.50           O  \\nATOM    422  N   PRO A  59      19.217 -12.135  -5.426  1.00  0.44           N  \\nATOM    423  CA  PRO A  59      19.291 -12.734  -4.091  1.00  0.44           C  \\nATOM    424  C   PRO A  59      20.336 -13.844  -3.999  1.00  0.44           C  \\nATOM    425  CB  PRO A  59      17.882 -13.291  -3.876  1.00  0.41           C  \\nATOM    426  O   PRO A  59      20.599 -14.361  -2.910  1.00  0.43           O  \\nATOM    427  CG  PRO A  59      17.278 -13.339  -5.242  1.00  0.40           C  \\nATOM    428  CD  PRO A  59      18.114 -12.499  -6.164  1.00  0.45           C  \\nATOM    429  N   GLN A  60      21.034 -14.114  -5.119  1.00  0.37           N  \\nATOM    430  CA  GLN A  60      22.159 -15.023  -4.923  1.00  0.41           C  \\nATOM    431  C   GLN A  60      23.488 -14.275  -4.982  1.00  0.37           C  \\nATOM    432  CB  GLN A  60      22.137 -16.138  -5.970  1.00  0.34           C  \\nATOM    433  O   GLN A  60      24.509 -14.775  -4.505  1.00  0.35           O  \\nATOM    434  CG  GLN A  60      21.120 -17.234  -5.680  1.00  0.35           C  \\nATOM    435  CD  GLN A  60      21.228 -18.403  -6.640  1.00  0.42           C  \\nATOM    436  NE2 GLN A  60      20.294 -19.343  -6.538  1.00  0.35           N  \\nATOM    437  OE1 GLN A  60      22.143 -18.462  -7.468  1.00  0.48           O  \\nATOM    438  N   SER A  61      23.459 -12.932  -4.871  1.00  0.38           N  \\nATOM    439  CA  SER A  61      24.816 -12.413  -4.739  1.00  0.40           C  \\nATOM    440  C   SER A  61      24.874 -11.266  -3.737  1.00  0.39           C  \\nATOM    441  CB  SER A  61      25.346 -11.944  -6.095  1.00  0.36           C  \\nATOM    442  O   SER A  61      24.126 -10.293  -3.855  1.00  0.37           O  \\nATOM    443  OG  SER A  61      25.160 -12.945  -7.081  1.00  0.42           O  \\nATOM    444  N   LYS A  62      25.078 -11.577  -2.469  1.00  0.40           N  \\nATOM    445  CA  LYS A  62      25.951 -10.963  -1.473  1.00  0.44           C  \\nATOM    446  C   LYS A  62      26.846  -9.901  -2.106  1.00  0.40           C  \\nATOM    447  CB  LYS A  62      26.807 -12.026  -0.781  1.00  0.36           C  \\nATOM    448  O   LYS A  62      28.071 -10.038  -2.112  1.00  0.38           O  \\nATOM    449  CG  LYS A  62      26.207 -12.558   0.512  1.00  0.37           C  \\nATOM    450  CD  LYS A  62      27.125 -13.577   1.176  1.00  0.44           C  \\nATOM    451  CE  LYS A  62      26.514 -14.131   2.455  1.00  0.35           C  \\nATOM    452  NZ  LYS A  62      27.399 -15.148   3.097  1.00  0.36           N  \\nATOM    453  N   GLN A  63      26.417  -9.144  -3.008  1.00  0.42           N  \\nATOM    454  CA  GLN A  63      27.253  -7.962  -3.191  1.00  0.46           C  \\nATOM    455  C   GLN A  63      26.408  -6.740  -3.541  1.00  0.43           C  \\nATOM    456  CB  GLN A  63      28.298  -8.207  -4.281  1.00  0.39           C  \\nATOM    457  O   GLN A  63      25.625  -6.771  -4.493  1.00  0.40           O  \\nATOM    458  CG  GLN A  63      29.445  -9.107  -3.842  1.00  0.38           C  \\nATOM    459  CD  GLN A  63      30.458  -9.348  -4.945  1.00  0.44           C  \\nATOM    460  NE2 GLN A  63      31.375 -10.281  -4.715  1.00  0.36           N  \\nATOM    461  OE1 GLN A  63      30.416  -8.701  -5.996  1.00  0.49           O  \\nATOM    462  N   THR A  64      26.018  -5.919  -2.625  1.00  0.47           N  \\nATOM    463  CA  THR A  64      25.855  -4.471  -2.702  1.00  0.48           C  \\nATOM    464  C   THR A  64      26.575  -3.910  -3.925  1.00  0.47           C  \\nATOM    465  CB  THR A  64      26.383  -3.781  -1.431  1.00  0.44           C  \\nATOM    466  O   THR A  64      27.788  -4.076  -4.069  1.00  0.45           O  \\nATOM    467  CG2 THR A  64      25.243  -3.168  -0.624  1.00  0.41           C  \\nATOM    468  OG1 THR A  64      27.065  -4.745  -0.619  1.00  0.41           O  \\nATOM    469  N   PRO A  65      25.947  -3.640  -5.005  1.00  0.42           N  \\nATOM    470  CA  PRO A  65      26.672  -2.718  -5.883  1.00  0.43           C  \\nATOM    471  C   PRO A  65      26.899  -1.350  -5.243  1.00  0.42           C  \\nATOM    472  CB  PRO A  65      25.758  -2.603  -7.106  1.00  0.38           C  \\nATOM    473  O   PRO A  65      25.989  -0.796  -4.622  1.00  0.40           O  \\nATOM    474  CG  PRO A  65      24.496  -3.296  -6.706  1.00  0.38           C  \\nATOM    475  CD  PRO A  65      24.709  -3.941  -5.366  1.00  0.42           C  \\nATOM    476  N   GLU A  66      28.216  -0.909  -4.943  1.00  0.48           N  \\nATOM    477  CA  GLU A  66      28.879   0.391  -4.971  1.00  0.48           C  \\nATOM    478  C   GLU A  66      28.096   1.392  -5.817  1.00  0.48           C  \\nATOM    479  CB  GLU A  66      30.307   0.255  -5.505  1.00  0.44           C  \\nATOM    480  O   GLU A  66      28.011   1.248  -7.038  1.00  0.46           O  \\nATOM    481  CG  GLU A  66      31.164   1.494  -5.292  1.00  0.42           C  \\nATOM    482  CD  GLU A  66      32.653   1.229  -5.449  1.00  0.47           C  \\nATOM    483  OE1 GLU A  66      33.027   0.125  -5.908  1.00  0.53           O  \\nATOM    484  OE2 GLU A  66      33.451   2.131  -5.111  1.00  0.47           O  \\nATOM    485  N   ALA A  67      26.984   1.836  -5.491  1.00  0.49           N  \\nATOM    486  CA  ALA A  67      26.532   3.112  -6.042  1.00  0.51           C  \\nATOM    487  C   ALA A  67      27.715   4.018  -6.369  1.00  0.49           C  \\nATOM    488  CB  ALA A  67      25.587   3.808  -5.065  1.00  0.45           C  \\nATOM    489  O   ALA A  67      28.484   4.391  -5.480  1.00  0.48           O  \\nATOM    490  N   SER A  68      28.344   3.955  -7.502  1.00  0.44           N  \\nATOM    491  CA  SER A  68      29.207   5.014  -8.014  1.00  0.45           C  \\nATOM    492  C   SER A  68      28.548   6.382  -7.873  1.00  0.44           C  \\nATOM    493  CB  SER A  68      29.558   4.757  -9.480  1.00  0.41           C  \\nATOM    494  O   SER A  68      27.403   6.571  -8.288  1.00  0.43           O  \\nATOM    495  OG  SER A  68      30.306   3.561  -9.616  1.00  0.45           O  \\nATOM    496  N   PRO A  69      28.996   7.125  -6.835  1.00  0.51           N  \\nATOM    497  CA  PRO A  69      28.654   8.540  -6.672  1.00  0.49           C  \\nATOM    498  C   PRO A  69      28.504   9.269  -8.005  1.00  0.51           C  \\nATOM    499  CB  PRO A  69      29.835   9.098  -5.874  1.00  0.46           C  \\nATOM    500  O   PRO A  69      29.311   9.067  -8.917  1.00  0.49           O  \\nATOM    501  CG  PRO A  69      30.927   8.094  -6.055  1.00  0.44           C  \\nATOM    502  CD  PRO A  69      30.317   6.806  -6.530  1.00  0.48           C  \\nATOM    503  N   ARG A  70      27.338   9.584  -8.473  1.00  0.52           N  \\nATOM    504  CA  ARG A  70      27.167  10.625  -9.480  1.00  0.53           C  \\nATOM    505  C   ARG A  70      28.358  11.578  -9.492  1.00  0.51           C  \\nATOM    506  CB  ARG A  70      25.874  11.406  -9.233  1.00  0.47           C  \\nATOM    507  O   ARG A  70      28.797  12.044  -8.438  1.00  0.50           O  \\nATOM    508  CG  ARG A  70      25.415  12.231 -10.424  1.00  0.46           C  \\nATOM    509  CD  ARG A  70      24.075  12.904 -10.161  1.00  0.50           C  \\nATOM    510  NE  ARG A  70      23.536  13.523 -11.369  1.00  0.41           N  \\nATOM    511  NH1 ARG A  70      21.514  14.107 -10.417  1.00  0.31           N  \\nATOM    512  NH2 ARG A  70      21.943  14.615 -12.611  1.00  0.28           N  \\nATOM    513  CZ  ARG A  70      22.332  14.080 -11.463  1.00  0.42           C  \\nATOM    514  N   PRO A  71      29.153  11.539 -10.632  1.00  0.48           N  \\nATOM    515  CA  PRO A  71      30.038  12.696 -10.782  1.00  0.47           C  \\nATOM    516  C   PRO A  71      29.363  14.009 -10.392  1.00  0.48           C  \\nATOM    517  CB  PRO A  71      30.386  12.683 -12.273  1.00  0.45           C  \\nATOM    518  O   PRO A  71      28.336  14.375 -10.969  1.00  0.46           O  \\nATOM    519  CG  PRO A  71      29.423  11.716 -12.881  1.00  0.44           C  \\nATOM    520  CD  PRO A  71      28.722  10.981 -11.775  1.00  0.47           C  \\nATOM    521  N   GLN A  72      29.365  14.444  -9.163  1.00  0.50           N  \\nATOM    522  CA  GLN A  72      29.269  15.870  -8.869  1.00  0.51           C  \\nATOM    523  C   GLN A  72      29.851  16.706 -10.005  1.00  0.50           C  \\nATOM    524  CB  GLN A  72      29.982  16.198  -7.556  1.00  0.46           C  \\nATOM    525  O   GLN A  72      30.958  16.436 -10.477  1.00  0.48           O  \\nATOM    526  CG  GLN A  72      29.715  17.607  -7.046  1.00  0.46           C  \\nATOM    527  CD  GLN A  72      30.292  17.851  -5.664  1.00  0.50           C  \\nATOM    528  NE2 GLN A  72      30.193  19.088  -5.190  1.00  0.42           N  \\nATOM    529  OE1 GLN A  72      30.821  16.934  -5.028  1.00  0.56           O  \\nATOM    530  N   PRO A  73      28.950  17.324 -10.891  1.00  0.50           N  \\nATOM    531  CA  PRO A  73      29.579  18.309 -11.775  1.00  0.50           C  \\nATOM    532  C   PRO A  73      30.729  19.055 -11.103  1.00  0.50           C  \\nATOM    533  CB  PRO A  73      28.431  19.264 -12.112  1.00  0.46           C  \\nATOM    534  O   PRO A  73      30.604  19.483  -9.952  1.00  0.48           O  \\nATOM    535  CG  PRO A  73      27.367  18.950 -11.110  1.00  0.45           C  \\nATOM    536  CD  PRO A  73      27.701  17.642 -10.453  1.00  0.48           C  \\nATOM    537  N   GLU A  74      32.007  18.691 -11.374  1.00  0.52           N  \\nATOM    538  CA  GLU A  74      33.218  19.480 -11.168  1.00  0.54           C  \\nATOM    539  C   GLU A  74      32.921  20.976 -11.238  1.00  0.52           C  \\nATOM    540  CB  GLU A  74      34.286  19.107 -12.199  1.00  0.49           C  \\nATOM    541  O   GLU A  74      32.301  21.446 -12.195  1.00  0.49           O  \\nATOM    542  CG  GLU A  74      35.028  17.818 -11.876  1.00  0.48           C  \\nATOM    543  CD  GLU A  74      36.166  17.524 -12.841  1.00  0.52           C  \\nATOM    544  OE1 GLU A  74      36.264  18.202 -13.889  1.00  0.55           O  \\nATOM    545  OE2 GLU A  74      36.965  16.607 -12.547  1.00  0.49           O  \\nATOM    546  N   PRO A  75      32.730  21.669 -10.027  1.00  0.53           N  \\nATOM    547  CA  PRO A  75      32.731  23.132  -9.950  1.00  0.52           C  \\nATOM    548  C   PRO A  75      33.689  23.775 -10.949  1.00  0.53           C  \\nATOM    549  CB  PRO A  75      33.175  23.408  -8.511  1.00  0.48           C  \\nATOM    550  O   PRO A  75      34.908  23.635 -10.819  1.00  0.51           O  \\nATOM    551  CG  PRO A  75      33.945  22.192  -8.108  1.00  0.47           C  \\nATOM    552  CD  PRO A  75      33.504  21.042  -8.969  1.00  0.50           C  \\nATOM    553  N   SER A  76      33.285  23.913 -12.142  1.00  0.49           N  \\nATOM    554  CA  SER A  76      34.006  24.768 -13.080  1.00  0.53           C  \\nATOM    555  C   SER A  76      34.144  26.187 -12.540  1.00  0.49           C  \\nATOM    556  CB  SER A  76      33.296  24.796 -14.434  1.00  0.45           C  \\nATOM    557  O   SER A  76      33.155  26.804 -12.139  1.00  0.47           O  \\nATOM    558  OG  SER A  76      33.189  23.490 -14.973  1.00  0.48           O  \\nATOM    559  N   GLY A  77      35.281  26.445 -11.865  1.00  0.47           N  \\nATOM    560  CA  GLY A  77      36.332  27.426 -11.647  1.00  0.48           C  \\nATOM    561  C   GLY A  77      35.902  28.844 -11.969  1.00  0.47           C  \\nATOM    562  O   GLY A  77      36.733  29.752 -12.028  1.00  0.44           O  \\nATOM    563  N   LYS A  78      34.677  29.238 -11.631  1.00  0.44           N  \\nATOM    564  CA  LYS A  78      34.413  30.665 -11.471  1.00  0.47           C  \\nATOM    565  C   LYS A  78      33.537  30.929 -10.250  1.00  0.43           C  \\nATOM    566  CB  LYS A  78      33.747  31.231 -12.726  1.00  0.40           C  \\nATOM    567  O   LYS A  78      32.541  30.237 -10.033  1.00  0.41           O  \\nATOM    568  CG  LYS A  78      34.702  31.446 -13.891  1.00  0.40           C  \\nATOM    569  CD  LYS A  78      33.996  32.077 -15.084  1.00  0.46           C  \\nATOM    570  CE  LYS A  78      34.952  32.301 -16.247  1.00  0.38           C  \\nATOM    571  NZ  LYS A  78      34.254  32.875 -17.436  1.00  0.38           N  \\nATOM    572  N   PRO A  79      33.995  31.338  -9.041  1.00  0.51           N  \\nATOM    573  CA  PRO A  79      33.077  31.922  -8.060  1.00  0.52           C  \\nATOM    574  C   PRO A  79      31.857  32.573  -8.707  1.00  0.51           C  \\nATOM    575  CB  PRO A  79      33.941  32.968  -7.349  1.00  0.46           C  \\nATOM    576  O   PRO A  79      31.984  33.599  -9.380  1.00  0.48           O  \\nATOM    577  CG  PRO A  79      35.139  33.134  -8.226  1.00  0.45           C  \\nATOM    578  CD  PRO A  79      35.143  32.028  -9.243  1.00  0.50           C  \\nATOM    579  N   GLN A  80      30.922  31.782  -9.200  1.00  0.44           N  \\nATOM    580  CA  GLN A  80      29.572  32.269  -9.465  1.00  0.47           C  \\nATOM    581  C   GLN A  80      28.876  32.689  -8.173  1.00  0.45           C  \\nATOM    582  CB  GLN A  80      28.746  31.200 -10.182  1.00  0.41           C  \\nATOM    583  O   GLN A  80      28.900  31.957  -7.182  1.00  0.43           O  \\nATOM    584  CG  GLN A  80      28.544  31.473 -11.667  1.00  0.40           C  \\nATOM    585  CD  GLN A  80      27.849  30.331 -12.384  1.00  0.46           C  \\nATOM    586  NE2 GLN A  80      27.757  30.429 -13.706  1.00  0.37           N  \\nATOM    587  OE1 GLN A  80      27.399  29.368 -11.755  1.00  0.49           O  \\nATOM    588  N   SER A  81      29.040  33.939  -7.695  1.00  0.49           N  \\nATOM    589  CA  SER A  81      28.131  34.829  -6.980  1.00  0.53           C  \\nATOM    590  C   SER A  81      26.983  34.053  -6.344  1.00  0.49           C  \\nATOM    591  CB  SER A  81      27.575  35.897  -7.923  1.00  0.44           C  \\nATOM    592  O   SER A  81      26.365  33.206  -6.992  1.00  0.46           O  \\nATOM    593  OG  SER A  81      28.598  36.418  -8.753  1.00  0.46           O  \\nATOM    594  N   GLU A  82      27.189  33.386  -5.119  1.00  0.50           N  \\nATOM    595  CA  GLU A  82      26.172  33.163  -4.096  1.00  0.54           C  \\nATOM    596  C   GLU A  82      24.776  33.096  -4.710  1.00  0.50           C  \\nATOM    597  CB  GLU A  82      26.226  34.264  -3.034  1.00  0.46           C  \\nATOM    598  O   GLU A  82      24.099  34.118  -4.837  1.00  0.47           O  \\nATOM    599  CG  GLU A  82      26.857  33.822  -1.721  1.00  0.46           C  \\nATOM    600  CD  GLU A  82      27.041  34.960  -0.731  1.00  0.52           C  \\nATOM    601  OE1 GLU A  82      26.700  36.117  -1.065  1.00  0.53           O  \\nATOM    602  OE2 GLU A  82      27.532  34.693   0.389  1.00  0.48           O  \\nATOM    603  N   GLN A  83      24.489  32.474  -5.831  1.00  0.43           N  \\nATOM    604  CA  GLN A  83      23.155  31.934  -6.072  1.00  0.47           C  \\nATOM    605  C   GLN A  83      22.904  30.690  -5.224  1.00  0.43           C  \\nATOM    606  CB  GLN A  83      22.968  31.606  -7.554  1.00  0.40           C  \\nATOM    607  O   GLN A  83      23.586  29.675  -5.385  1.00  0.41           O  \\nATOM    608  CG  GLN A  83      22.833  32.835  -8.443  1.00  0.40           C  \\nATOM    609  CD  GLN A  83      22.606  32.482  -9.901  1.00  0.46           C  \\nATOM    610  NE2 GLN A  83      22.871  33.434 -10.790  1.00  0.38           N  \\nATOM    611  OE1 GLN A  83      22.196  31.363 -10.226  1.00  0.51           O  \\nATOM    612  N   THR A  84      22.596  30.842  -3.824  1.00  0.48           N  \\nATOM    613  CA  THR A  84      21.772  29.784  -3.251  1.00  0.50           C  \\nATOM    614  C   THR A  84      21.092  28.973  -4.350  1.00  0.48           C  \\nATOM    615  CB  THR A  84      20.706  30.359  -2.300  1.00  0.45           C  \\nATOM    616  O   THR A  84      20.434  29.536  -5.228  1.00  0.46           O  \\nATOM    617  CG2 THR A  84      21.123  30.194  -0.842  1.00  0.42           C  \\nATOM    618  OG1 THR A  84      20.527  31.753  -2.580  1.00  0.41           O  \\nATOM    619  N   PRO A  85      21.597  27.927  -4.820  1.00  0.42           N  \\nATOM    620  CA  PRO A  85      20.581  27.226  -5.608  1.00  0.43           C  \\nATOM    621  C   PRO A  85      19.161  27.491  -5.110  1.00  0.43           C  \\nATOM    622  CB  PRO A  85      20.954  25.752  -5.433  1.00  0.39           C  \\nATOM    623  O   PRO A  85      18.875  27.306  -3.925  1.00  0.41           O  \\nATOM    624  CG  PRO A  85      22.069  25.755  -4.439  1.00  0.38           C  \\nATOM    625  CD  PRO A  85      22.394  27.180  -4.092  1.00  0.44           C  \\nATOM    626  N   LYS A  86      18.334  28.529  -5.460  1.00  0.38           N  \\nATOM    627  CA  LYS A  86      16.894  28.285  -5.464  1.00  0.38           C  \\nATOM    628  C   LYS A  86      16.590  26.790  -5.499  1.00  0.38           C  \\nATOM    629  CB  LYS A  86      16.235  28.982  -6.656  1.00  0.35           C  \\nATOM    630  O   LYS A  86      17.024  26.084  -6.412  1.00  0.36           O  \\nATOM    631  CG  LYS A  86      14.734  29.181  -6.504  1.00  0.35           C  \\nATOM    632  CD  LYS A  86      14.159  29.996  -7.655  1.00  0.39           C  \\nATOM    633  CE  LYS A  86      12.663  30.225  -7.486  1.00  0.35           C  \\nATOM    634  NZ  LYS A  86      12.111  31.099  -8.564  1.00  0.36           N  \\nATOM    635  N   ALA A  87      16.766  25.984  -4.440  1.00  0.41           N  \\nATOM    636  CA  ALA A  87      16.071  24.703  -4.539  1.00  0.43           C  \\nATOM    637  C   ALA A  87      15.090  24.701  -5.708  1.00  0.41           C  \\nATOM    638  CB  ALA A  87      15.341  24.392  -3.235  1.00  0.38           C  \\nATOM    639  O   ALA A  87      14.231  25.579  -5.807  1.00  0.39           O  \\nATOM    640  N   SER A  88      15.287  24.633  -6.941  1.00  0.33           N  \\nATOM    641  CA  SER A  88      14.254  24.224  -7.888  1.00  0.35           C  \\nATOM    642  C   SER A  88      13.038  23.654  -7.166  1.00  0.33           C  \\nATOM    643  CB  SER A  88      14.806  23.189  -8.869  1.00  0.31           C  \\nATOM    644  O   SER A  88      13.179  22.902  -6.199  1.00  0.32           O  \\nATOM    645  OG  SER A  88      15.855  23.742  -9.646  1.00  0.32           O  \\nATOM    646  N   PRO A  89      11.906  24.338  -6.910  1.00  0.36           N  \\nATOM    647  CA  PRO A  89      10.785  23.538  -6.412  1.00  0.35           C  \\nATOM    648  C   PRO A  89      10.990  22.040  -6.622  1.00  0.36           C  \\nATOM    649  CB  PRO A  89       9.597  24.047  -7.233  1.00  0.32           C  \\nATOM    650  O   PRO A  89      11.486  21.622  -7.671  1.00  0.34           O  \\nATOM    651  CG  PRO A  89      10.212  24.785  -8.378  1.00  0.31           C  \\nATOM    652  CD  PRO A  89      11.654  25.051  -8.054  1.00  0.36           C  \\nATOM    653  N   GLU A  90      11.395  21.189  -5.744  1.00  0.39           N  \\nATOM    654  CA  GLU A  90      11.186  19.761  -5.964  1.00  0.40           C  \\nATOM    655  C   GLU A  90      10.198  19.516  -7.100  1.00  0.39           C  \\nATOM    656  CB  GLU A  90      10.692  19.086  -4.681  1.00  0.36           C  \\nATOM    657  O   GLU A  90       9.120  20.114  -7.132  1.00  0.37           O  \\nATOM    658  CG  GLU A  90      10.916  17.581  -4.653  1.00  0.35           C  \\nATOM    659  CD  GLU A  90      10.593  16.952  -3.308  1.00  0.41           C  \\nATOM    660  OE1 GLU A  90      10.162  17.679  -2.385  1.00  0.49           O  \\nATOM    661  OE2 GLU A  90      10.773  15.721  -3.175  1.00  0.44           O  \\nATOM    662  N   PRO A  91      10.476  19.468  -8.400  1.00  0.32           N  \\nATOM    663  CA  PRO A  91       9.376  19.130  -9.306  1.00  0.32           C  \\nATOM    664  C   PRO A  91       8.127  18.654  -8.567  1.00  0.32           C  \\nATOM    665  CB  PRO A  91       9.960  18.009 -10.169  1.00  0.29           C  \\nATOM    666  O   PRO A  91       8.208  17.753  -7.728  1.00  0.29           O  \\nATOM    667  CG  PRO A  91      11.120  17.488  -9.383  1.00  0.28           C  \\nATOM    668  CD  PRO A  91      11.512  18.523  -8.368  1.00  0.32           C  \\nATOM    669  N   GLN A  92       7.049  19.294  -8.231  1.00  0.33           N  \\nATOM    670  CA  GLN A  92       5.711  18.711  -8.224  1.00  0.34           C  \\nATOM    671  C   GLN A  92       5.528  17.738  -9.385  1.00  0.33           C  \\nATOM    672  CB  GLN A  92       4.646  19.807  -8.285  1.00  0.31           C  \\nATOM    673  O   GLN A  92       5.774  18.090 -10.541  1.00  0.32           O  \\nATOM    674  CG  GLN A  92       4.387  20.488  -6.947  1.00  0.30           C  \\nATOM    675  CD  GLN A  92       3.028  21.159  -6.884  1.00  0.37           C  \\nATOM    676  NE2 GLN A  92       2.541  21.398  -5.671  1.00  0.33           N  \\nATOM    677  OE1 GLN A  92       2.422  21.462  -7.916  1.00  0.45           O  \\nATOM    678  N   PRO A  93       5.878  16.461  -9.335  1.00  0.35           N  \\nATOM    679  CA  PRO A  93       5.559  15.563 -10.448  1.00  0.35           C  \\nATOM    680  C   PRO A  93       4.269  15.951 -11.167  1.00  0.35           C  \\nATOM    681  CB  PRO A  93       5.417  14.198  -9.770  1.00  0.32           C  \\nATOM    682  O   PRO A  93       3.270  16.274 -10.519  1.00  0.34           O  \\nATOM    683  CG  PRO A  93       5.180  14.511  -8.328  1.00  0.31           C  \\nATOM    684  CD  PRO A  93       5.691  15.895  -8.049  1.00  0.35           C  \\nATOM    685  N   LYS A  94       4.194  16.578 -12.285  1.00  0.31           N  \\nATOM    686  CA  LYS A  94       3.076  16.661 -13.221  1.00  0.32           C  \\nATOM    687  C   LYS A  94       2.235  15.389 -13.187  1.00  0.31           C  \\nATOM    688  CB  LYS A  94       3.584  16.916 -14.641  1.00  0.29           C  \\nATOM    689  O   LYS A  94       2.775  14.281 -13.181  1.00  0.30           O  \\nATOM    690  CG  LYS A  94       3.720  18.390 -14.995  1.00  0.28           C  \\nATOM    691  CD  LYS A  94       4.158  18.580 -16.441  1.00  0.32           C  \\nATOM    692  CE  LYS A  94       4.324  20.053 -16.787  1.00  0.31           C  \\nATOM    693  NZ  LYS A  94       4.781  20.243 -18.196  1.00  0.32           N  \\nATOM    694  N   PRO A  95       0.987  15.225 -12.600  1.00  0.32           N  \\nATOM    695  CA  PRO A  95       0.075  14.080 -12.562  1.00  0.32           C  \\nATOM    696  C   PRO A  95       0.142  13.229 -13.828  1.00  0.32           C  \\nATOM    697  CB  PRO A  95      -1.303  14.732 -12.416  1.00  0.29           C  \\nATOM    698  O   PRO A  95      -0.529  12.198 -13.920  1.00  0.30           O  \\nATOM    699  CG  PRO A  95      -1.037  16.202 -12.458  1.00  0.28           C  \\nATOM    700  CD  PRO A  95       0.442  16.414 -12.611  1.00  0.33           C  \\nATOM    701  N   SER A  96       0.981  13.258 -14.735  1.00  0.31           N  \\nATOM    702  CA  SER A  96       0.851  12.433 -15.932  1.00  0.32           C  \\nATOM    703  C   SER A  96       1.285  10.996 -15.663  1.00  0.31           C  \\nATOM    704  CB  SER A  96       1.677  13.018 -17.078  1.00  0.28           C  \\nATOM    705  O   SER A  96       0.667  10.053 -16.160  1.00  0.29           O  \\nATOM    706  OG  SER A  96       3.021  13.221 -16.677  1.00  0.28           O  \\nATOM    707  N   GLY A  97       2.104  10.758 -14.703  1.00  0.34           N  \\nATOM    708  CA  GLY A  97       2.527   9.445 -14.244  1.00  0.34           C  \\nATOM    709  C   GLY A  97       1.727   8.941 -13.057  1.00  0.34           C  \\nATOM    710  O   GLY A  97       1.540   7.734 -12.896  1.00  0.32           O  \\nATOM    711  N   GLN A  98       1.100   9.773 -12.310  1.00  0.32           N  \\nATOM    712  CA  GLN A  98       0.420   9.513 -11.045  1.00  0.34           C  \\nATOM    713  C   GLN A  98      -0.864   8.718 -11.264  1.00  0.32           C  \\nATOM    714  CB  GLN A  98       0.109  10.825 -10.323  1.00  0.30           C  \\nATOM    715  O   GLN A  98      -1.151   7.776 -10.522  1.00  0.31           O  \\nATOM    716  CG  GLN A  98       1.300  11.416  -9.581  1.00  0.29           C  \\nATOM    717  CD  GLN A  98       0.957  12.701  -8.850  1.00  0.35           C  \\nATOM    718  NE2 GLN A  98       1.949  13.292  -8.193  1.00  0.33           N  \\nATOM    719  OE1 GLN A  98      -0.190  13.159  -8.876  1.00  0.40           O  \\nATOM    720  N   ALA A  99      -1.694   9.133 -12.301  1.00  0.33           N  \\nATOM    721  CA  ALA A  99      -2.904   8.427 -12.713  1.00  0.32           C  \\nATOM    722  C   ALA A  99      -2.581   7.009 -13.176  1.00  0.32           C  \\nATOM    723  CB  ALA A  99      -3.617   9.196 -13.822  1.00  0.30           C  \\nATOM    724  O   ALA A  99      -3.314   6.066 -12.868  1.00  0.31           O  \\nATOM    725  N   LYS A 100      -1.377   6.879 -13.761  1.00  0.35           N  \\nATOM    726  CA  LYS A 100      -0.957   5.565 -14.238  1.00  0.35           C  \\nATOM    727  C   LYS A 100      -0.514   4.675 -13.081  1.00  0.35           C  \\nATOM    728  CB  LYS A 100       0.174   5.700 -15.259  1.00  0.33           C  \\nATOM    729  O   LYS A 100      -0.738   3.463 -13.102  1.00  0.33           O  \\nATOM    730  CG  LYS A 100      -0.285   6.155 -16.636  1.00  0.31           C  \\nATOM    731  CD  LYS A 100       0.870   6.189 -17.628  1.00  0.35           C  \\nATOM    732  CE  LYS A 100       0.418   6.676 -18.998  1.00  0.31           C  \\nATOM    733  NZ  LYS A 100       1.554   6.743 -19.965  1.00  0.32           N  \\nATOM    734  N   GLU A 101       0.009   5.246 -12.025  1.00  0.36           N  \\nATOM    735  CA  GLU A 101       0.444   4.578 -10.801  1.00  0.37           C  \\nATOM    736  C   GLU A 101      -0.740   3.980 -10.047  1.00  0.36           C  \\nATOM    737  CB  GLU A 101       1.204   5.553  -9.898  1.00  0.35           C  \\nATOM    738  O   GLU A 101      -0.704   2.814  -9.649  1.00  0.35           O  \\nATOM    739  CG  GLU A 101       2.164   4.875  -8.931  1.00  0.33           C  \\nATOM    740  CD  GLU A 101       3.192   5.826  -8.340  1.00  0.39           C  \\nATOM    741  OE1 GLU A 101       3.163   7.033  -8.670  1.00  0.43           O  \\nATOM    742  OE2 GLU A 101       4.034   5.360  -7.540  1.00  0.42           O  \\nATOM    743  N   ILE A 102      -1.847   4.766  -9.752  1.00  0.36           N  \\nATOM    744  CA  ILE A 102      -3.040   4.351  -9.022  1.00  0.36           C  \\nATOM    745  C   ILE A 102      -3.776   3.269  -9.809  1.00  0.36           C  \\nATOM    746  CB  ILE A 102      -3.979   5.547  -8.749  1.00  0.35           C  \\nATOM    747  O   ILE A 102      -4.254   2.290  -9.231  1.00  0.35           O  \\nATOM    748  CG1 ILE A 102      -3.320   6.532  -7.777  1.00  0.32           C  \\nATOM    749  CG2 ILE A 102      -5.327   5.063  -8.207  1.00  0.32           C  \\nATOM    750  CD1 ILE A 102      -4.063   7.854  -7.636  1.00  0.30           C  \\nATOM    751  N   GLN A 103      -3.844   3.396 -11.173  1.00  0.37           N  \\nATOM    752  CA  GLN A 103      -4.492   2.400 -12.021  1.00  0.37           C  \\nATOM    753  C   GLN A 103      -3.801   1.045 -11.905  1.00  0.37           C  \\nATOM    754  CB  GLN A 103      -4.503   2.861 -13.479  1.00  0.36           C  \\nATOM    755  O   GLN A 103      -4.464   0.010 -11.809  1.00  0.36           O  \\nATOM    756  CG  GLN A 103      -5.755   3.635 -13.870  1.00  0.34           C  \\nATOM    757  CD  GLN A 103      -5.750   4.062 -15.326  1.00  0.38           C  \\nATOM    758  NE2 GLN A 103      -6.829   4.705 -15.759  1.00  0.36           N  \\nATOM    759  OE1 GLN A 103      -4.784   3.817 -16.055  1.00  0.43           O  \\nATOM    760  N   TRP A 104      -2.454   1.031 -11.844  1.00  0.23           N  \\nATOM    761  CA  TRP A 104      -1.708  -0.223 -11.806  1.00  0.32           C  \\nATOM    762  C   TRP A 104      -1.837  -0.892 -10.441  1.00  0.24           C  \\nATOM    763  CB  TRP A 104      -0.231   0.020 -12.132  1.00  0.24           C  \\nATOM    764  O   TRP A 104      -2.099  -2.094 -10.355  1.00  0.24           O  \\nATOM    765  CG  TRP A 104       0.500  -1.205 -12.591  1.00  0.25           C  \\nATOM    766  CD1 TRP A 104       0.431  -1.792 -13.824  1.00  0.26           C  \\nATOM    767  CD2 TRP A 104       1.408  -1.997 -11.820  1.00  0.27           C  \\nATOM    768  CE2 TRP A 104       1.854  -3.050 -12.650  1.00  0.19           C  \\nATOM    769  CE3 TRP A 104       1.890  -1.919 -10.506  1.00  0.27           C  \\nATOM    770  NE1 TRP A 104       1.244  -2.901 -13.866  1.00  0.24           N  \\nATOM    771  CH2 TRP A 104       3.215  -3.917 -10.919  1.00  0.24           C  \\nATOM    772  CZ2 TRP A 104       2.759  -4.017 -12.208  1.00  0.23           C  \\nATOM    773  CZ3 TRP A 104       2.791  -2.883 -10.068  1.00  0.25           C  \\nATOM    774  N   GLN A 105      -1.646  -0.189  -9.346  1.00  0.40           N  \\nATOM    775  CA  GLN A 105      -1.756  -0.798  -8.025  1.00  0.40           C  \\nATOM    776  C   GLN A 105      -3.122  -1.449  -7.831  1.00  0.40           C  \\nATOM    777  CB  GLN A 105      -1.510   0.243  -6.932  1.00  0.39           C  \\nATOM    778  O   GLN A 105      -3.215  -2.575  -7.339  1.00  0.39           O  \\nATOM    779  CG  GLN A 105      -0.046   0.626  -6.765  1.00  0.38           C  \\nATOM    780  CD  GLN A 105       0.170   1.666  -5.682  1.00  0.42           C  \\nATOM    781  NE2 GLN A 105       1.428   2.018  -5.440  1.00  0.39           N  \\nATOM    782  OE1 GLN A 105      -0.787   2.149  -5.068  1.00  0.46           O  \\nATOM    783  N   ALA A 106      -4.171  -0.718  -8.237  1.00  0.37           N  \\nATOM    784  CA  ALA A 106      -5.512  -1.292  -8.165  1.00  0.36           C  \\nATOM    785  C   ALA A 106      -5.598  -2.589  -8.965  1.00  0.37           C  \\nATOM    786  CB  ALA A 106      -6.548  -0.290  -8.669  1.00  0.35           C  \\nATOM    787  O   ALA A 106      -6.192  -3.569  -8.509  1.00  0.36           O  \\nATOM    788  N   LYS A 107      -4.870  -2.683 -10.068  1.00  0.38           N  \\nATOM    789  CA  LYS A 107      -4.930  -3.855 -10.936  1.00  0.39           C  \\nATOM    790  C   LYS A 107      -4.075  -4.992 -10.385  1.00  0.38           C  \\nATOM    791  CB  LYS A 107      -4.476  -3.497 -12.352  1.00  0.37           C  \\nATOM    792  O   LYS A 107      -4.495  -6.151 -10.393  1.00  0.37           O  \\nATOM    793  CG  LYS A 107      -5.422  -3.972 -13.445  1.00  0.36           C  \\nATOM    794  CD  LYS A 107      -4.940  -3.547 -14.826  1.00  0.40           C  \\nATOM    795  CE  LYS A 107      -5.851  -4.078 -15.925  1.00  0.37           C  \\nATOM    796  NZ  LYS A 107      -5.325  -3.755 -17.285  1.00  0.38           N  \\nATOM    797  N   GLU A 108      -2.896  -4.745  -9.972  1.00  0.39           N  \\nATOM    798  CA  GLU A 108      -2.054  -5.744  -9.320  1.00  0.40           C  \\nATOM    799  C   GLU A 108      -2.747  -6.337  -8.097  1.00  0.39           C  \\nATOM    800  CB  GLU A 108      -0.709  -5.134  -8.918  1.00  0.39           C  \\nATOM    801  O   GLU A 108      -2.735  -7.554  -7.899  1.00  0.38           O  \\nATOM    802  CG  GLU A 108       0.476  -6.066  -9.127  1.00  0.38           C  \\nATOM    803  CD  GLU A 108       1.770  -5.534  -8.532  1.00  0.43           C  \\nATOM    804  OE1 GLU A 108       1.820  -4.341  -8.156  1.00  0.47           O  \\nATOM    805  OE2 GLU A 108       2.742  -6.316  -8.443  1.00  0.45           O  \\nATOM    806  N   ILE A 109      -3.427  -5.538  -7.401  1.00  0.39           N  \\nATOM    807  CA  ILE A 109      -4.228  -6.036  -6.288  1.00  0.39           C  \\nATOM    808  C   ILE A 109      -5.353  -6.922  -6.817  1.00  0.39           C  \\nATOM    809  CB  ILE A 109      -4.808  -4.878  -5.446  1.00  0.38           C  \\nATOM    810  O   ILE A 109      -5.631  -7.985  -6.257  1.00  0.38           O  \\nATOM    811  CG1 ILE A 109      -3.687  -4.143  -4.703  1.00  0.35           C  \\nATOM    812  CG2 ILE A 109      -5.864  -5.398  -4.467  1.00  0.35           C  \\nATOM    813  CD1 ILE A 109      -4.117  -2.820  -4.085  1.00  0.33           C  \\nATOM    814  N   GLN A 110      -5.952  -6.531  -7.916  1.00  0.39           N  \\nATOM    815  CA  GLN A 110      -6.924  -7.412  -8.556  1.00  0.40           C  \\nATOM    816  C   GLN A 110      -6.294  -8.752  -8.922  1.00  0.39           C  \\nATOM    817  CB  GLN A 110      -7.510  -6.749  -9.803  1.00  0.39           C  \\nATOM    818  O   GLN A 110      -6.896  -9.807  -8.707  1.00  0.39           O  \\nATOM    819  CG  GLN A 110      -8.857  -6.078  -9.568  1.00  0.38           C  \\nATOM    820  CD  GLN A 110      -9.401  -5.401 -10.812  1.00  0.42           C  \\nATOM    821  NE2 GLN A 110     -10.559  -4.763 -10.681  1.00  0.39           N  \\nATOM    822  OE1 GLN A 110      -8.784  -5.450 -11.881  1.00  0.48           O  \\nATOM    823  N   GLU A 111      -5.058  -8.731  -9.439  1.00  0.40           N  \\nATOM    824  CA  GLU A 111      -4.419  -9.953  -9.918  1.00  0.41           C  \\nATOM    825  C   GLU A 111      -3.829 -10.756  -8.763  1.00  0.40           C  \\nATOM    826  CB  GLU A 111      -3.328  -9.624 -10.941  1.00  0.39           C  \\nATOM    827  O   GLU A 111      -3.959 -11.981  -8.721  1.00  0.39           O  \\nATOM    828  CG  GLU A 111      -3.861  -9.335 -12.337  1.00  0.38           C  \\nATOM    829  CD  GLU A 111      -2.798  -9.438 -13.418  1.00  0.43           C  \\nATOM    830  OE1 GLU A 111      -1.601  -9.586 -13.081  1.00  0.48           O  \\nATOM    831  OE2 GLU A 111      -3.164  -9.369 -14.613  1.00  0.46           O  \\nATOM    832  N   GLN A 112      -3.334 -10.090  -7.833  1.00  0.39           N  \\nATOM    833  CA  GLN A 112      -2.922 -10.784  -6.618  1.00  0.40           C  \\nATOM    834  C   GLN A 112      -4.128 -11.336  -5.863  1.00  0.39           C  \\nATOM    835  CB  GLN A 112      -2.119  -9.849  -5.712  1.00  0.38           C  \\nATOM    836  O   GLN A 112      -4.063 -12.427  -5.293  1.00  0.38           O  \\nATOM    837  CG  GLN A 112      -0.662  -9.691  -6.127  1.00  0.37           C  \\nATOM    838  CD  GLN A 112       0.154  -8.905  -5.118  1.00  0.41           C  \\nATOM    839  NE2 GLN A 112       1.386  -8.569  -5.484  1.00  0.38           N  \\nATOM    840  OE1 GLN A 112      -0.320  -8.601  -4.018  1.00  0.45           O  \\nATOM    841  N   ALA A 113      -5.196 -10.581  -5.817  1.00  0.37           N  \\nATOM    842  CA  ALA A 113      -6.435 -11.164  -5.308  1.00  0.37           C  \\nATOM    843  C   ALA A 113      -6.793 -12.439  -6.066  1.00  0.37           C  \\nATOM    844  CB  ALA A 113      -7.576 -10.155  -5.402  1.00  0.35           C  \\nATOM    845  O   ALA A 113      -7.529 -13.287  -5.556  1.00  0.35           O  \\nATOM    846  N   LYS A 114      -6.117 -12.662  -7.235  1.00  0.35           N  \\nATOM    847  CA  LYS A 114      -6.336 -13.902  -7.974  1.00  0.35           C  \\nATOM    848  C   LYS A 114      -5.392 -15.001  -7.494  1.00  0.35           C  \\nATOM    849  CB  LYS A 114      -6.153 -13.673  -9.475  1.00  0.33           C  \\nATOM    850  O   LYS A 114      -4.177 -14.911  -7.686  1.00  0.34           O  \\nATOM    851  CG  LYS A 114      -7.267 -12.858 -10.117  1.00  0.32           C  \\nATOM    852  CD  LYS A 114      -7.011 -12.634 -11.601  1.00  0.35           C  \\nATOM    853  CE  LYS A 114      -8.077 -11.743 -12.225  1.00  0.32           C  \\nATOM    854  NZ  LYS A 114      -7.744 -11.387 -13.637  1.00  0.32           N  \\nATOM    855  N   GLY A 115      -4.748 -14.972  -6.373  1.00  0.38           N  \\nATOM    856  CA  GLY A 115      -3.899 -15.878  -5.616  1.00  0.39           C  \\nATOM    857  C   GLY A 115      -3.620 -17.181  -6.342  1.00  0.38           C  \\nATOM    858  O   GLY A 115      -4.213 -17.455  -7.387  1.00  0.37           O  \\nATOM    859  N   PRO A 116      -2.389 -17.903  -6.226  1.00  0.37           N  \\nATOM    860  CA  PRO A 116      -2.172 -19.262  -6.725  1.00  0.38           C  \\nATOM    861  C   PRO A 116      -3.408 -20.148  -6.583  1.00  0.37           C  \\nATOM    862  CB  PRO A 116      -1.026 -19.780  -5.851  1.00  0.35           C  \\nATOM    863  O   PRO A 116      -4.075 -20.121  -5.546  1.00  0.36           O  \\nATOM    864  CG  PRO A 116      -1.114 -18.978  -4.593  1.00  0.33           C  \\nATOM    865  CD  PRO A 116      -1.792 -17.675  -4.904  1.00  0.37           C  \\nATOM    866  N   SER A 117      -4.295 -20.291  -7.625  1.00  0.32           N  \\nATOM    867  CA  SER A 117      -5.128 -21.485  -7.717  1.00  0.34           C  \\nATOM    868  C   SER A 117      -4.696 -22.541  -6.705  1.00  0.32           C  \\nATOM    869  CB  SER A 117      -5.075 -22.069  -9.130  1.00  0.31           C  \\nATOM    870  O   SER A 117      -3.609 -23.111  -6.821  1.00  0.31           O  \\nATOM    871  OG  SER A 117      -5.878 -21.310 -10.018  1.00  0.31           O  \\nATOM    872  N   GLY A 118      -4.497 -22.196  -5.465  1.00  0.35           N  \\nATOM    873  CA  GLY A 118      -4.474 -23.331  -4.557  1.00  0.36           C  \\nATOM    874  C   GLY A 118      -5.659 -24.261  -4.735  1.00  0.35           C  \\nATOM    875  O   GLY A 118      -6.790 -23.807  -4.918  1.00  0.34           O  \\nATOM    876  N   ASP A 119      -5.439 -25.372  -5.453  1.00  0.35           N  \\nATOM    877  CA  ASP A 119      -6.269 -26.573  -5.438  1.00  0.35           C  \\nATOM    878  C   ASP A 119      -7.237 -26.557  -4.257  1.00  0.35           C  \\nATOM    879  CB  ASP A 119      -5.396 -27.828  -5.387  1.00  0.33           C  \\nATOM    880  O   ASP A 119      -6.819 -26.402  -3.108  1.00  0.34           O  \\nATOM    881  CG  ASP A 119      -5.412 -28.615  -6.685  1.00  0.33           C  \\nATOM    882  OD1 ASP A 119      -6.416 -28.548  -7.427  1.00  0.37           O  \\nATOM    883  OD2 ASP A 119      -4.412 -29.311  -6.968  1.00  0.44           O  \\nATOM    884  N   PRO A 120      -8.491 -26.030  -4.335  1.00  0.40           N  \\nATOM    885  CA  PRO A 120      -9.505 -26.232  -3.298  1.00  0.39           C  \\nATOM    886  C   PRO A 120      -9.334 -27.556  -2.556  1.00  0.39           C  \\nATOM    887  CB  PRO A 120     -10.820 -26.212  -4.082  1.00  0.36           C  \\nATOM    888  O   PRO A 120     -10.027 -27.807  -1.567  1.00  0.37           O  \\nATOM    889  CG  PRO A 120     -10.422 -26.459  -5.502  1.00  0.35           C  \\nATOM    890  CD  PRO A 120      -8.950 -26.194  -5.637  1.00  0.40           C  \\nATOM    891  N   GLN A 121      -8.375 -28.353  -2.937  1.00  0.35           N  \\nATOM    892  CA  GLN A 121      -8.275 -29.560  -2.123  1.00  0.39           C  \\nATOM    893  C   GLN A 121      -7.340 -29.346  -0.936  1.00  0.35           C  \\nATOM    894  CB  GLN A 121      -7.789 -30.739  -2.968  1.00  0.32           C  \\nATOM    895  O   GLN A 121      -7.346 -30.131   0.015  1.00  0.33           O  \\nATOM    896  CG  GLN A 121      -8.886 -31.399  -3.792  1.00  0.32           C  \\nATOM    897  CD  GLN A 121      -8.413 -32.652  -4.503  1.00  0.39           C  \\nATOM    898  NE2 GLN A 121      -9.308 -33.278  -5.261  1.00  0.32           N  \\nATOM    899  OE1 GLN A 121      -7.253 -33.054  -4.374  1.00  0.46           O  \\nATOM    900  N   SER A 122      -6.945 -28.089  -0.563  1.00  0.35           N  \\nATOM    901  CA  SER A 122      -6.322 -28.043   0.756  1.00  0.37           C  \\nATOM    902  C   SER A 122      -7.142 -27.198   1.725  1.00  0.35           C  \\nATOM    903  CB  SER A 122      -4.901 -27.486   0.659  1.00  0.33           C  \\nATOM    904  O   SER A 122      -7.163 -25.969   1.622  1.00  0.34           O  \\nATOM    905  OG  SER A 122      -4.195 -28.094  -0.409  1.00  0.36           O  \\nATOM    906  N   LYS A 123      -8.537 -27.283   1.789  1.00  0.34           N  \\nATOM    907  CA  LYS A 123      -9.409 -27.257   2.960  1.00  0.37           C  \\nATOM    908  C   LYS A 123      -8.724 -27.887   4.169  1.00  0.34           C  \\nATOM    909  CB  LYS A 123     -10.724 -27.979   2.667  1.00  0.31           C  \\nATOM    910  O   LYS A 123      -8.684 -29.112   4.298  1.00  0.32           O  \\nATOM    911  CG  LYS A 123     -11.832 -27.067   2.162  1.00  0.32           C  \\nATOM    912  CD  LYS A 123     -13.136 -27.829   1.964  1.00  0.39           C  \\nATOM    913  CE  LYS A 123     -14.244 -26.919   1.451  1.00  0.33           C  \\nATOM    914  NZ  LYS A 123     -15.522 -27.664   1.245  1.00  0.35           N  \\nATOM    915  N   GLN A 124      -7.636 -27.411   4.717  1.00  0.37           N  \\nATOM    916  CA  GLN A 124      -7.436 -27.424   6.162  1.00  0.40           C  \\nATOM    917  C   GLN A 124      -7.115 -26.027   6.686  1.00  0.37           C  \\nATOM    918  CB  GLN A 124      -6.319 -28.397   6.541  1.00  0.34           C  \\nATOM    919  O   GLN A 124      -6.030 -25.499   6.435  1.00  0.35           O  \\nATOM    920  CG  GLN A 124      -6.707 -29.863   6.403  1.00  0.34           C  \\nATOM    921  CD  GLN A 124      -5.578 -30.806   6.774  1.00  0.41           C  \\nATOM    922  NE2 GLN A 124      -5.716 -32.073   6.399  1.00  0.35           N  \\nATOM    923  OE1 GLN A 124      -4.588 -30.399   7.391  1.00  0.48           O  \\nATOM    924  N   THR A 125      -7.902 -24.999   6.684  1.00  0.41           N  \\nATOM    925  CA  THR A 125      -7.872 -24.166   7.881  1.00  0.42           C  \\nATOM    926  C   THR A 125      -7.643 -25.018   9.126  1.00  0.40           C  \\nATOM    927  CB  THR A 125      -9.177 -23.364   8.037  1.00  0.38           C  \\nATOM    928  O   THR A 125      -8.324 -26.024   9.331  1.00  0.39           O  \\nATOM    929  CG2 THR A 125      -8.924 -21.867   7.894  1.00  0.36           C  \\nATOM    930  OG1 THR A 125     -10.108 -23.778   7.030  1.00  0.35           O  \\nATOM    931  N   PRO A 126      -6.461 -25.326   9.637  1.00  0.38           N  \\nATOM    932  CA  PRO A 126      -6.547 -25.917  10.975  1.00  0.38           C  \\nATOM    933  C   PRO A 126      -7.775 -25.446  11.750  1.00  0.38           C  \\nATOM    934  CB  PRO A 126      -5.259 -25.445  11.653  1.00  0.35           C  \\nATOM    935  O   PRO A 126      -8.083 -24.251  11.759  1.00  0.36           O  \\nATOM    936  CG  PRO A 126      -4.675 -24.443  10.708  1.00  0.34           C  \\nATOM    937  CD  PRO A 126      -5.490 -24.447   9.447  1.00  0.39           C  \\nATOM    938  N   GLU A 127      -9.091 -26.041  11.679  1.00  0.39           N  \\nATOM    939  CA  GLU A 127      -9.903 -26.068  12.892  1.00  0.39           C  \\nATOM    940  C   GLU A 127      -9.292 -25.195  13.984  1.00  0.38           C  \\nATOM    941  CB  GLU A 127     -10.069 -27.503  13.396  1.00  0.35           C  \\nATOM    942  O   GLU A 127      -8.188 -25.470  14.460  1.00  0.35           O  \\nATOM    943  CG  GLU A 127     -11.142 -27.659  14.464  1.00  0.34           C  \\nATOM    944  CD  GLU A 127     -11.581 -29.101  14.668  1.00  0.40           C  \\nATOM    945  OE1 GLU A 127     -10.931 -30.017  14.115  1.00  0.47           O  \\nATOM    946  OE2 GLU A 127     -12.584 -29.315  15.385  1.00  0.44           O  \\nATOM    947  N   ALA A 128      -8.974 -23.915  13.901  1.00  0.41           N  \\nATOM    948  CA  ALA A 128      -8.967 -23.396  15.267  1.00  0.43           C  \\nATOM    949  C   ALA A 128      -9.879 -24.220  16.172  1.00  0.40           C  \\nATOM    950  CB  ALA A 128      -9.392 -21.930  15.281  1.00  0.38           C  \\nATOM    951  O   ALA A 128     -11.088 -24.292  15.944  1.00  0.39           O  \\nATOM    952  N   SER A 129      -9.714 -25.524  16.380  1.00  0.35           N  \\nATOM    953  CA  SER A 129     -10.308 -26.110  17.578  1.00  0.37           C  \\nATOM    954  C   SER A 129     -10.872 -25.032  18.498  1.00  0.35           C  \\nATOM    955  CB  SER A 129      -9.276 -26.948  18.334  1.00  0.33           C  \\nATOM    956  O   SER A 129     -10.186 -24.059  18.817  1.00  0.33           O  \\nATOM    957  OG  SER A 129      -8.854 -28.052  17.551  1.00  0.37           O  \\nATOM    958  N   PRO A 130     -12.137 -24.630  18.349  1.00  0.44           N  \\nATOM    959  CA  PRO A 130     -12.645 -23.910  19.519  1.00  0.43           C  \\nATOM    960  C   PRO A 130     -11.853 -24.215  20.789  1.00  0.44           C  \\nATOM    961  CB  PRO A 130     -14.087 -24.409  19.642  1.00  0.40           C  \\nATOM    962  O   PRO A 130     -11.667 -25.383  21.139  1.00  0.42           O  \\nATOM    963  CG  PRO A 130     -14.122 -25.675  18.849  1.00  0.40           C  \\nATOM    964  CD  PRO A 130     -12.930 -25.699  17.935  1.00  0.44           C  \\nATOM    965  N   ARG A 131     -10.678 -23.673  21.102  1.00  0.44           N  \\nATOM    966  CA  ARG A 131     -10.250 -23.780  22.492  1.00  0.47           C  \\nATOM    967  C   ARG A 131     -11.432 -24.082  23.408  1.00  0.44           C  \\nATOM    968  CB  ARG A 131      -9.554 -22.494  22.942  1.00  0.41           C  \\nATOM    969  O   ARG A 131     -12.474 -23.430  23.319  1.00  0.42           O  \\nATOM    970  CG  ARG A 131      -8.687 -22.662  24.179  1.00  0.41           C  \\nATOM    971  CD  ARG A 131      -7.938 -21.382  24.522  1.00  0.45           C  \\nATOM    972  NE  ARG A 131      -6.911 -21.612  25.533  1.00  0.40           N  \\nATOM    973  NH1 ARG A 131      -6.040 -19.475  25.433  1.00  0.31           N  \\nATOM    974  NH2 ARG A 131      -5.140 -21.018  26.870  1.00  0.27           N  \\nATOM    975  CZ  ARG A 131      -6.033 -20.701  25.943  1.00  0.37           C  \\nATOM    976  N   PRO A 132     -11.616 -25.368  23.761  1.00  0.45           N  \\nATOM    977  CA  PRO A 132     -12.568 -25.686  24.828  1.00  0.45           C  \\nATOM    978  C   PRO A 132     -13.004 -24.453  25.616  1.00  0.45           C  \\nATOM    979  CB  PRO A 132     -11.787 -26.655  25.719  1.00  0.41           C  \\nATOM    980  O   PRO A 132     -12.162 -23.728  26.151  1.00  0.43           O  \\nATOM    981  CG  PRO A 132     -10.372 -26.552  25.248  1.00  0.41           C  \\nATOM    982  CD  PRO A 132     -10.354 -25.782  23.959  1.00  0.44           C  \\nATOM    983  N   GLN A 133     -13.982 -23.728  25.260  1.00  0.47           N  \\nATOM    984  CA  GLN A 133     -14.735 -23.000  26.276  1.00  0.49           C  \\nATOM    985  C   GLN A 133     -14.673 -23.713  27.623  1.00  0.47           C  \\nATOM    986  CB  GLN A 133     -16.191 -22.825  25.841  1.00  0.44           C  \\nATOM    987  O   GLN A 133     -14.926 -24.917  27.707  1.00  0.45           O  \\nATOM    988  CG  GLN A 133     -16.941 -21.754  26.622  1.00  0.44           C  \\nATOM    989  CD  GLN A 133     -18.331 -21.492  26.072  1.00  0.49           C  \\nATOM    990  NE2 GLN A 133     -19.135 -20.748  26.824  1.00  0.40           N  \\nATOM    991  OE1 GLN A 133     -18.679 -21.954  24.981  1.00  0.53           O  \\nATOM    992  N   PRO A 134     -13.689 -23.384  28.488  1.00  0.47           N  \\nATOM    993  CA  PRO A 134     -13.821 -23.915  29.847  1.00  0.48           C  \\nATOM    994  C   PRO A 134     -15.277 -24.115  30.263  1.00  0.47           C  \\nATOM    995  CB  PRO A 134     -13.149 -22.844  30.710  1.00  0.43           C  \\nATOM    996  O   PRO A 134     -16.113 -23.234  30.042  1.00  0.45           O  \\nATOM    997  CG  PRO A 134     -13.108 -21.626  29.846  1.00  0.43           C  \\nATOM    998  CD  PRO A 134     -13.289 -22.049  28.416  1.00  0.46           C  \\nATOM    999  N   GLU A 135     -15.899 -25.361  30.173  1.00  0.53           N  \\nATOM   1000  CA  GLU A 135     -16.998 -25.846  31.002  1.00  0.55           C  \\nATOM   1001  C   GLU A 135     -17.143 -25.011  32.271  1.00  0.53           C  \\nATOM   1002  CB  GLU A 135     -16.789 -27.319  31.363  1.00  0.49           C  \\nATOM   1003  O   GLU A 135     -16.161 -24.762  32.972  1.00  0.50           O  \\nATOM   1004  CG  GLU A 135     -17.106 -28.282  30.228  1.00  0.48           C  \\nATOM   1005  CD  GLU A 135     -16.977 -29.743  30.627  1.00  0.52           C  \\nATOM   1006  OE1 GLU A 135     -16.407 -30.030  31.704  1.00  0.55           O  \\nATOM   1007  OE2 GLU A 135     -17.448 -30.608  29.855  1.00  0.49           O  \\nATOM   1008  N   PRO A 136     -18.135 -24.043  32.286  1.00  0.53           N  \\nATOM   1009  CA  PRO A 136     -18.575 -23.396  33.524  1.00  0.54           C  \\nATOM   1010  C   PRO A 136     -18.474 -24.316  34.739  1.00  0.53           C  \\nATOM   1011  CB  PRO A 136     -20.034 -23.034  33.234  1.00  0.48           C  \\nATOM   1012  O   PRO A 136     -19.292 -25.225  34.899  1.00  0.50           O  \\nATOM   1013  CG  PRO A 136     -20.471 -24.015  32.195  1.00  0.47           C  \\nATOM   1014  CD  PRO A 136     -19.256 -24.507  31.462  1.00  0.51           C  \\nATOM   1015  N   SER A 137     -17.349 -24.609  35.167  1.00  0.49           N  \\nATOM   1016  CA  SER A 137     -17.244 -25.085  36.542  1.00  0.53           C  \\nATOM   1017  C   SER A 137     -17.366 -23.936  37.536  1.00  0.49           C  \\nATOM   1018  CB  SER A 137     -15.918 -25.817  36.757  1.00  0.45           C  \\nATOM   1019  O   SER A 137     -16.683 -22.918  37.404  1.00  0.46           O  \\nATOM   1020  OG  SER A 137     -15.808 -26.926  35.881  1.00  0.50           O  \\nATOM   1021  N   GLY A 138     -18.565 -23.779  37.985  1.00  0.49           N  \\nATOM   1022  CA  GLY A 138     -19.131 -23.471  39.289  1.00  0.50           C  \\nATOM   1023  C   GLY A 138     -18.230 -22.595  40.139  1.00  0.49           C  \\nATOM   1024  O   GLY A 138     -18.465 -22.434  41.338  1.00  0.46           O  \\nATOM   1025  N   LYS A 139     -17.497 -21.625  39.663  1.00  0.46           N  \\nATOM   1026  CA  LYS A 139     -16.990 -20.544  40.504  1.00  0.49           C  \\nATOM   1027  C   LYS A 139     -16.990 -19.217  39.752  1.00  0.46           C  \\nATOM   1028  CB  LYS A 139     -15.580 -20.866  40.999  1.00  0.42           C  \\nATOM   1029  O   LYS A 139     -16.633 -19.165  38.573  1.00  0.44           O  \\nATOM   1030  CG  LYS A 139     -15.530 -21.949  42.066  1.00  0.41           C  \\nATOM   1031  CD  LYS A 139     -14.110 -22.173  42.571  1.00  0.48           C  \\nATOM   1032  CE  LYS A 139     -14.059 -23.260  43.636  1.00  0.38           C  \\nATOM   1033  NZ  LYS A 139     -12.663 -23.515  44.101  1.00  0.38           N  \\nATOM   1034  N   PRO A 140     -17.820 -18.162  40.006  1.00  0.56           N  \\nATOM   1035  CA  PRO A 140     -17.626 -16.769  39.595  1.00  0.57           C  \\nATOM   1036  C   PRO A 140     -16.170 -16.448  39.267  1.00  0.56           C  \\nATOM   1037  CB  PRO A 140     -18.100 -15.973  40.814  1.00  0.50           C  \\nATOM   1038  O   PRO A 140     -15.309 -16.499  40.149  1.00  0.53           O  \\nATOM   1039  CG  PRO A 140     -18.111 -16.962  41.935  1.00  0.49           C  \\nATOM   1040  CD  PRO A 140     -18.001 -18.342  41.355  1.00  0.53           C  \\nATOM   1041  N   GLN A 141     -15.701 -16.812  38.039  1.00  0.46           N  \\nATOM   1042  CA  GLN A 141     -14.521 -16.197  37.440  1.00  0.50           C  \\nATOM   1043  C   GLN A 141     -14.699 -14.687  37.301  1.00  0.47           C  \\nATOM   1044  CB  GLN A 141     -14.226 -16.818  36.074  1.00  0.42           C  \\nATOM   1045  O   GLN A 141     -15.611 -14.225  36.613  1.00  0.46           O  \\nATOM   1046  CG  GLN A 141     -12.983 -17.697  36.053  1.00  0.42           C  \\nATOM   1047  CD  GLN A 141     -12.793 -18.414  34.730  1.00  0.48           C  \\nATOM   1048  NE2 GLN A 141     -11.799 -19.294  34.670  1.00  0.39           N  \\nATOM   1049  OE1 GLN A 141     -13.534 -18.180  33.770  1.00  0.51           O  \\nATOM   1050  N   SER A 142     -14.474 -13.881  38.262  1.00  0.50           N  \\nATOM   1051  CA  SER A 142     -13.851 -12.580  38.481  1.00  0.56           C  \\nATOM   1052  C   SER A 142     -12.709 -12.341  37.500  1.00  0.50           C  \\nATOM   1053  CB  SER A 142     -13.334 -12.467  39.916  1.00  0.46           C  \\nATOM   1054  O   SER A 142     -11.840 -13.198  37.330  1.00  0.47           O  \\nATOM   1055  OG  SER A 142     -13.887 -13.485  40.732  1.00  0.50           O  \\nATOM   1056  N   GLU A 143     -13.028 -11.763  36.260  1.00  0.55           N  \\nATOM   1057  CA  GLU A 143     -12.187 -10.843  35.499  1.00  0.59           C  \\nATOM   1058  C   GLU A 143     -11.481 -11.560  34.352  1.00  0.55           C  \\nATOM   1059  CB  GLU A 143     -11.158 -10.174  36.414  1.00  0.50           C  \\nATOM   1060  O   GLU A 143     -10.371 -12.068  34.521  1.00  0.52           O  \\nATOM   1061  CG  GLU A 143     -11.541  -8.764  36.842  1.00  0.48           C  \\nATOM   1062  CD  GLU A 143     -10.589  -8.170  37.869  1.00  0.53           C  \\nATOM   1063  OE1 GLU A 143      -9.637  -8.866  38.288  1.00  0.55           O  \\nATOM   1064  OE2 GLU A 143     -10.799  -6.999  38.257  1.00  0.49           O  \\nATOM   1065  N   GLN A 144     -12.194 -12.301  33.525  1.00  0.43           N  \\nATOM   1066  CA  GLN A 144     -11.684 -12.487  32.171  1.00  0.49           C  \\nATOM   1067  C   GLN A 144     -11.891 -11.232  31.328  1.00  0.44           C  \\nATOM   1068  CB  GLN A 144     -12.357 -13.686  31.502  1.00  0.41           C  \\nATOM   1069  O   GLN A 144     -13.028 -10.827  31.078  1.00  0.42           O  \\nATOM   1070  CG  GLN A 144     -11.867 -15.033  32.016  1.00  0.42           C  \\nATOM   1071  CD  GLN A 144     -12.522 -16.204  31.310  1.00  0.48           C  \\nATOM   1072  NE2 GLN A 144     -12.464 -17.378  31.930  1.00  0.39           N  \\nATOM   1073  OE1 GLN A 144     -13.077 -16.056  30.216  1.00  0.51           O  \\nATOM   1074  N   THR A 145     -10.915 -10.196  31.356  1.00  0.50           N  \\nATOM   1075  CA  THR A 145     -10.672  -9.333  30.205  1.00  0.53           C  \\nATOM   1076  C   THR A 145     -11.118 -10.015  28.915  1.00  0.50           C  \\nATOM   1077  CB  THR A 145      -9.184  -8.950  30.099  1.00  0.47           C  \\nATOM   1078  O   THR A 145     -10.716 -11.146  28.633  1.00  0.48           O  \\nATOM   1079  CG2 THR A 145      -8.907  -7.623  30.798  1.00  0.45           C  \\nATOM   1080  OG1 THR A 145      -8.388  -9.974  30.709  1.00  0.46           O  \\nATOM   1081  N   PRO A 146     -12.249  -9.691  28.459  1.00  0.43           N  \\nATOM   1082  CA  PRO A 146     -12.471 -10.131  27.080  1.00  0.44           C  \\nATOM   1083  C   PRO A 146     -11.184 -10.174  26.260  1.00  0.43           C  \\nATOM   1084  CB  PRO A 146     -13.436  -9.081  26.525  1.00  0.39           C  \\nATOM   1085  O   PRO A 146     -10.436  -9.193  26.225  1.00  0.42           O  \\nATOM   1086  CG  PRO A 146     -13.535  -8.047  27.600  1.00  0.39           C  \\nATOM   1087  CD  PRO A 146     -12.745  -8.518  28.788  1.00  0.44           C  \\nATOM   1088  N   LYS A 147     -10.453 -11.250  26.062  1.00  0.40           N  \\nATOM   1089  CA  LYS A 147      -9.620 -11.472  24.883  1.00  0.42           C  \\nATOM   1090  C   LYS A 147     -10.169 -10.721  23.673  1.00  0.40           C  \\nATOM   1091  CB  LYS A 147      -9.519 -12.966  24.571  1.00  0.38           C  \\nATOM   1092  O   LYS A 147     -11.315 -10.933  23.272  1.00  0.39           O  \\nATOM   1093  CG  LYS A 147      -8.290 -13.346  23.759  1.00  0.38           C  \\nATOM   1094  CD  LYS A 147      -8.182 -14.855  23.580  1.00  0.43           C  \\nATOM   1095  CE  LYS A 147      -6.940 -15.238  22.788  1.00  0.37           C  \\nATOM   1096  NZ  LYS A 147      -6.800 -16.720  22.659  1.00  0.37           N  \\nATOM   1097  N   ALA A 148      -9.991  -9.376  23.515  1.00  0.39           N  \\nATOM   1098  CA  ALA A 148     -10.248  -8.640  22.279  1.00  0.42           C  \\nATOM   1099  C   ALA A 148     -10.615  -9.590  21.142  1.00  0.39           C  \\nATOM   1100  CB  ALA A 148      -9.030  -7.802  21.896  1.00  0.36           C  \\nATOM   1101  O   ALA A 148      -9.853 -10.502  20.816  1.00  0.38           O  \\nATOM   1102  N   SER A 149     -11.784 -10.079  20.912  1.00  0.33           N  \\nATOM   1103  CA  SER A 149     -12.179 -10.440  19.555  1.00  0.33           C  \\nATOM   1104  C   SER A 149     -11.397  -9.639  18.520  1.00  0.33           C  \\nATOM   1105  CB  SER A 149     -13.679 -10.219  19.356  1.00  0.30           C  \\nATOM   1106  O   SER A 149     -11.106  -8.460  18.731  1.00  0.31           O  \\nATOM   1107  OG  SER A 149     -14.429 -11.001  20.270  1.00  0.33           O  \\nATOM   1108  N   PRO A 150     -10.483 -10.218  17.728  1.00  0.37           N  \\nATOM   1109  CA  PRO A 150      -9.994  -9.308  16.690  1.00  0.37           C  \\nATOM   1110  C   PRO A 150     -10.957  -8.156  16.414  1.00  0.38           C  \\nATOM   1111  CB  PRO A 150      -9.860 -10.213  15.462  1.00  0.34           C  \\nATOM   1112  O   PRO A 150     -12.170  -8.366  16.330  1.00  0.36           O  \\nATOM   1113  CG  PRO A 150     -10.687 -11.416  15.782  1.00  0.33           C  \\nATOM   1114  CD  PRO A 150     -10.926 -11.450  17.264  1.00  0.38           C  \\nATOM   1115  N   GLU A 151     -10.981  -6.964  16.958  1.00  0.36           N  \\nATOM   1116  CA  GLU A 151     -11.632  -5.728  16.534  1.00  0.38           C  \\nATOM   1117  C   GLU A 151     -12.334  -5.906  15.191  1.00  0.36           C  \\nATOM   1118  CB  GLU A 151     -10.616  -4.586  16.449  1.00  0.33           C  \\nATOM   1119  O   GLU A 151     -11.753  -6.445  14.247  1.00  0.35           O  \\nATOM   1120  CG  GLU A 151     -11.213  -3.210  16.706  1.00  0.33           C  \\nATOM   1121  CD  GLU A 151     -10.170  -2.107  16.783  1.00  0.39           C  \\nATOM   1122  OE1 GLU A 151      -8.966  -2.397  16.594  1.00  0.49           O  \\nATOM   1123  OE2 GLU A 151     -10.558  -0.945  17.034  1.00  0.44           O  \\nATOM   1124  N   PRO A 152     -13.732  -6.142  15.090  1.00  0.32           N  \\nATOM   1125  CA  PRO A 152     -14.246  -6.031  13.723  1.00  0.32           C  \\nATOM   1126  C   PRO A 152     -13.501  -4.986  12.895  1.00  0.32           C  \\nATOM   1127  CB  PRO A 152     -15.707  -5.623  13.928  1.00  0.30           C  \\nATOM   1128  O   PRO A 152     -13.395  -3.828  13.307  1.00  0.31           O  \\nATOM   1129  CG  PRO A 152     -15.763  -5.088  15.322  1.00  0.29           C  \\nATOM   1130  CD  PRO A 152     -14.552  -5.573  16.068  1.00  0.34           C  \\nATOM   1131  N   GLN A 153     -12.438  -5.294  12.174  1.00  0.33           N  \\nATOM   1132  CA  GLN A 153     -11.704  -4.556  11.151  1.00  0.35           C  \\nATOM   1133  C   GLN A 153     -12.657  -3.847  10.194  1.00  0.33           C  \\nATOM   1134  CB  GLN A 153     -10.778  -5.492  10.373  1.00  0.31           C  \\nATOM   1135  O   GLN A 153     -13.696  -4.397   9.824  1.00  0.32           O  \\nATOM   1136  CG  GLN A 153      -9.535  -5.911  11.146  1.00  0.31           C  \\nATOM   1137  CD  GLN A 153      -8.402  -6.354  10.240  1.00  0.39           C  \\nATOM   1138  NE2 GLN A 153      -7.174  -6.265  10.739  1.00  0.35           N  \\nATOM   1139  OE1 GLN A 153      -8.629  -6.776   9.102  1.00  0.47           O  \\nATOM   1140  N   PRO A 154     -12.790  -2.520  10.267  1.00  0.33           N  \\nATOM   1141  CA  PRO A 154     -13.665  -1.811   9.330  1.00  0.33           C  \\nATOM   1142  C   PRO A 154     -13.906  -2.593   8.041  1.00  0.33           C  \\nATOM   1143  CB  PRO A 154     -12.902  -0.514   9.048  1.00  0.31           C  \\nATOM   1144  O   PRO A 154     -12.995  -3.254   7.536  1.00  0.32           O  \\nATOM   1145  CG  PRO A 154     -11.568  -0.708   9.692  1.00  0.30           C  \\nATOM   1146  CD  PRO A 154     -11.597  -1.990  10.474  1.00  0.34           C  \\nATOM   1147  N   GLY A 155     -15.129  -3.015   7.830  1.00  0.32           N  \\nATOM   1148  CA  GLY A 155     -15.650  -3.535   6.576  1.00  0.31           C  \\nATOM   1149  C   GLY A 155     -15.223  -2.721   5.369  1.00  0.32           C  \\nATOM   1150  O   GLY A 155     -14.787  -1.577   5.508  1.00  0.31           O  \\nATOM   1151  N   ILE A 156     -14.649  -3.332   4.266  1.00  0.30           N  \\nATOM   1152  CA  ILE A 156     -14.283  -2.856   2.936  1.00  0.31           C  \\nATOM   1153  C   ILE A 156     -14.921  -1.491   2.684  1.00  0.30           C  \\nATOM   1154  CB  ILE A 156     -14.709  -3.858   1.840  1.00  0.28           C  \\nATOM   1155  O   ILE A 156     -14.298  -0.608   2.089  1.00  0.28           O  \\nATOM   1156  CG1 ILE A 156     -13.952  -5.181   2.003  1.00  0.26           C  \\nATOM   1157  CG2 ILE A 156     -14.480  -3.264   0.447  1.00  0.26           C  \\nATOM   1158  CD1 ILE A 156     -14.417  -6.280   1.058  1.00  0.24           C  \\nATOM   1159  N   LYS A 157     -16.120  -1.147   3.184  1.00  0.28           N  \\nATOM   1160  CA  LYS A 157     -16.786   0.128   2.933  1.00  0.28           C  \\nATOM   1161  C   LYS A 157     -16.031   1.280   3.589  1.00  0.28           C  \\nATOM   1162  CB  LYS A 157     -18.228   0.089   3.441  1.00  0.25           C  \\nATOM   1163  O   LYS A 157     -15.885   2.351   2.996  1.00  0.26           O  \\nATOM   1164  CG  LYS A 157     -19.182  -0.678   2.537  1.00  0.24           C  \\nATOM   1165  CD  LYS A 157     -20.619  -0.585   3.034  1.00  0.27           C  \\nATOM   1166  CE  LYS A 157     -21.574  -1.357   2.133  1.00  0.24           C  \\nATOM   1167  NZ  LYS A 157     -22.983  -1.275   2.621  1.00  0.26           N  \\nATOM   1168  N   GLU A 158     -15.586   1.142   4.789  1.00  0.28           N  \\nATOM   1169  CA  GLU A 158     -14.929   2.198   5.553  1.00  0.28           C  \\nATOM   1170  C   GLU A 158     -13.500   2.422   5.068  1.00  0.28           C  \\nATOM   1171  CB  GLU A 158     -14.932   1.863   7.047  1.00  0.26           C  \\nATOM   1172  O   GLU A 158     -13.047   3.564   4.960  1.00  0.26           O  \\nATOM   1173  CG  GLU A 158     -16.287   2.042   7.715  1.00  0.25           C  \\nATOM   1174  CD  GLU A 158     -16.395   1.329   9.053  1.00  0.30           C  \\nATOM   1175  OE1 GLU A 158     -15.396   0.719   9.498  1.00  0.38           O  \\nATOM   1176  OE2 GLU A 158     -17.488   1.379   9.661  1.00  0.31           O  \\nATOM   1177  N   GLU A 159     -12.833   1.457   4.524  1.00  0.27           N  \\nATOM   1178  CA  GLU A 159     -11.497   1.535   3.941  1.00  0.28           C  \\nATOM   1179  C   GLU A 159     -11.536   2.170   2.554  1.00  0.27           C  \\nATOM   1180  CB  GLU A 159     -10.860   0.145   3.866  1.00  0.26           C  \\nATOM   1181  O   GLU A 159     -10.692   3.007   2.225  1.00  0.26           O  \\nATOM   1182  CG  GLU A 159      -9.856  -0.133   4.975  1.00  0.25           C  \\nATOM   1183  CD  GLU A 159      -9.289  -1.543   4.931  1.00  0.31           C  \\nATOM   1184  OE1 GLU A 159      -9.606  -2.296   3.983  1.00  0.39           O  \\nATOM   1185  OE2 GLU A 159      -8.521  -1.897   5.853  1.00  0.34           O  \\nATOM   1186  N   ALA A 160     -12.551   1.930   1.827  1.00  0.29           N  \\nATOM   1187  CA  ALA A 160     -12.795   2.642   0.576  1.00  0.29           C  \\nATOM   1188  C   ALA A 160     -13.043   4.127   0.829  1.00  0.29           C  \\nATOM   1189  CB  ALA A 160     -13.981   2.027  -0.164  1.00  0.27           C  \\nATOM   1190  O   ALA A 160     -12.567   4.979   0.075  1.00  0.28           O  \\nATOM   1191  N   ALA A 161     -13.783   4.510   1.920  1.00  0.27           N  \\nATOM   1192  CA  ALA A 161     -14.008   5.909   2.275  1.00  0.26           C  \\nATOM   1193  C   ALA A 161     -12.700   6.595   2.657  1.00  0.27           C  \\nATOM   1194  CB  ALA A 161     -15.014   6.013   3.418  1.00  0.25           C  \\nATOM   1195  O   ALA A 161     -12.453   7.738   2.264  1.00  0.26           O  \\nATOM   1196  N   ALA A 162     -11.921   5.986   3.433  1.00  0.28           N  \\nATOM   1197  CA  ALA A 162     -10.613   6.554   3.752  1.00  0.27           C  \\nATOM   1198  C   ALA A 162      -9.796   6.792   2.485  1.00  0.28           C  \\nATOM   1199  CB  ALA A 162      -9.854   5.637   4.707  1.00  0.26           C  \\nATOM   1200  O   ALA A 162      -9.153   7.835   2.341  1.00  0.26           O  \\nATOM   1201  N   ILE A 163      -9.940   5.984   1.452  1.00  0.31           N  \\nATOM   1202  CA  ILE A 163      -9.247   6.035   0.170  1.00  0.31           C  \\nATOM   1203  C   ILE A 163      -9.769   7.212  -0.652  1.00  0.31           C  \\nATOM   1204  CB  ILE A 163      -9.414   4.716  -0.617  1.00  0.29           C  \\nATOM   1205  O   ILE A 163      -8.991   7.927  -1.288  1.00  0.29           O  \\nATOM   1206  CG1 ILE A 163      -8.698   3.569   0.106  1.00  0.29           C  \\nATOM   1207  CG2 ILE A 163      -8.893   4.871  -2.048  1.00  0.28           C  \\nATOM   1208  CD1 ILE A 163      -9.040   2.187  -0.434  1.00  0.26           C  \\nATOM   1209  N   LYS A 164     -11.154   7.409  -0.731  1.00  0.28           N  \\nATOM   1210  CA  LYS A 164     -11.760   8.574  -1.368  1.00  0.28           C  \\nATOM   1211  C   LYS A 164     -11.048   9.859  -0.954  1.00  0.28           C  \\nATOM   1212  CB  LYS A 164     -13.247   8.664  -1.021  1.00  0.26           C  \\nATOM   1213  O   LYS A 164     -10.807  10.737  -1.786  1.00  0.26           O  \\nATOM   1214  CG  LYS A 164     -14.171   8.150  -2.115  1.00  0.25           C  \\nATOM   1215  CD  LYS A 164     -15.635   8.277  -1.716  1.00  0.28           C  \\nATOM   1216  CE  LYS A 164     -16.560   7.763  -2.811  1.00  0.24           C  \\nATOM   1217  NZ  LYS A 164     -17.997   7.874  -2.419  1.00  0.26           N  \\nATOM   1218  N   TRP A 165     -10.650  10.016   0.290  1.00  0.17           N  \\nATOM   1219  CA  TRP A 165     -10.208  11.283   0.863  1.00  0.23           C  \\nATOM   1220  C   TRP A 165      -8.698  11.445   0.726  1.00  0.20           C  \\nATOM   1221  CB  TRP A 165     -10.614  11.377   2.337  1.00  0.17           C  \\nATOM   1222  O   TRP A 165      -8.208  12.541   0.444  1.00  0.18           O  \\nATOM   1223  CG  TRP A 165     -11.759  12.310   2.595  1.00  0.16           C  \\nATOM   1224  CD1 TRP A 165     -13.092  12.023   2.500  1.00  0.08           C  \\nATOM   1225  CD2 TRP A 165     -11.671  13.683   2.986  1.00  0.08           C  \\nATOM   1226  CE2 TRP A 165     -12.991  14.168   3.111  1.00  0.08           C  \\nATOM   1227  CE3 TRP A 165     -10.602  14.552   3.243  1.00  0.17           C  \\nATOM   1228  NE1 TRP A 165     -13.839  13.136   2.810  1.00  0.18           N  \\nATOM   1229  CH2 TRP A 165     -12.207  16.314   3.730  1.00  0.15           C  \\nATOM   1230  CZ2 TRP A 165     -13.271  15.485   3.484  1.00  0.16           C  \\nATOM   1231  CZ3 TRP A 165     -10.882  15.862   3.614  1.00  0.16           C  \\nATOM   1232  N   GLU A 166      -7.957  10.459   0.719  1.00  0.29           N  \\nATOM   1233  CA  GLU A 166      -6.524  10.623   0.499  1.00  0.30           C  \\nATOM   1234  C   GLU A 166      -6.224  10.961  -0.959  1.00  0.29           C  \\nATOM   1235  CB  GLU A 166      -5.768   9.357   0.911  1.00  0.28           C  \\nATOM   1236  O   GLU A 166      -5.426  11.856  -1.243  1.00  0.28           O  \\nATOM   1237  CG  GLU A 166      -5.017   9.491   2.228  1.00  0.27           C  \\nATOM   1238  CD  GLU A 166      -4.302   8.215   2.643  1.00  0.34           C  \\nATOM   1239  OE1 GLU A 166      -4.409   7.197   1.922  1.00  0.42           O  \\nATOM   1240  OE2 GLU A 166      -3.629   8.234   3.698  1.00  0.39           O  \\nATOM   1241  N   ALA A 167      -6.935  10.366  -1.865  1.00  0.31           N  \\nATOM   1242  CA  ALA A 167      -6.807  10.767  -3.264  1.00  0.31           C  \\nATOM   1243  C   ALA A 167      -7.252  12.213  -3.463  1.00  0.31           C  \\nATOM   1244  CB  ALA A 167      -7.617   9.836  -4.162  1.00  0.29           C  \\nATOM   1245  O   ALA A 167      -6.604  12.975  -4.185  1.00  0.29           O  \\nATOM   1246  N   ALA A 168      -8.332  12.672  -2.816  1.00  0.28           N  \\nATOM   1247  CA  ALA A 168      -8.800  14.054  -2.877  1.00  0.27           C  \\nATOM   1248  C   ALA A 168      -7.807  14.999  -2.208  1.00  0.28           C  \\nATOM   1249  CB  ALA A 168     -10.174  14.179  -2.222  1.00  0.26           C  \\nATOM   1250  O   ALA A 168      -7.494  16.063  -2.747  1.00  0.27           O  \\nATOM   1251  N   ALA A 169      -7.194  14.669  -1.104  1.00  0.26           N  \\nATOM   1252  CA  ALA A 169      -6.261  15.550  -0.406  1.00  0.26           C  \\nATOM   1253  C   ALA A 169      -4.911  15.595  -1.117  1.00  0.26           C  \\nATOM   1254  CB  ALA A 169      -6.082  15.096   1.041  1.00  0.24           C  \\nATOM   1255  O   ALA A 169      -4.269  16.646  -1.178  1.00  0.25           O  \\nATOM   1256  N   ILE A 170      -4.432  14.532  -1.674  1.00  0.34           N  \\nATOM   1257  CA  ILE A 170      -3.273  14.580  -2.558  1.00  0.34           C  \\nATOM   1258  C   ILE A 170      -3.566  15.497  -3.743  1.00  0.34           C  \\nATOM   1259  CB  ILE A 170      -2.881  13.170  -3.054  1.00  0.32           C  \\nATOM   1260  O   ILE A 170      -2.706  16.276  -4.161  1.00  0.32           O  \\nATOM   1261  CG1 ILE A 170      -2.377  12.314  -1.887  1.00  0.31           C  \\nATOM   1262  CG2 ILE A 170      -1.829  13.261  -4.163  1.00  0.30           C  \\nATOM   1263  CD1 ILE A 170      -2.213  10.839  -2.227  1.00  0.28           C  \\nATOM   1264  N   LYS A 171      -4.782  15.514  -4.173  1.00  0.29           N  \\nATOM   1265  CA  LYS A 171      -5.220  16.424  -5.227  1.00  0.30           C  \\nATOM   1266  C   LYS A 171      -4.980  17.879  -4.832  1.00  0.30           C  \\nATOM   1267  CB  LYS A 171      -6.699  16.205  -5.546  1.00  0.28           C  \\nATOM   1268  O   LYS A 171      -4.598  18.700  -5.669  1.00  0.28           O  \\nATOM   1269  CG  LYS A 171      -6.948  15.453  -6.845  1.00  0.27           C  \\nATOM   1270  CD  LYS A 171      -8.437  15.287  -7.119  1.00  0.33           C  \\nATOM   1271  CE  LYS A 171      -8.687  14.528  -8.415  1.00  0.28           C  \\nATOM   1272  NZ  LYS A 171     -10.146  14.359  -8.686  1.00  0.30           N  \\nATOM   1273  N   GLU A 172      -5.106  18.115  -3.534  1.00  0.26           N  \\nATOM   1274  CA  GLU A 172      -5.135  19.495  -3.060  1.00  0.25           C  \\nATOM   1275  C   GLU A 172      -3.801  19.890  -2.432  1.00  0.26           C  \\nATOM   1276  CB  GLU A 172      -6.270  19.694  -2.053  1.00  0.25           C  \\nATOM   1277  O   GLU A 172      -3.616  21.040  -2.029  1.00  0.25           O  \\nATOM   1278  CG  GLU A 172      -7.641  19.850  -2.695  1.00  0.25           C  \\nATOM   1279  CD  GLU A 172      -8.755  20.073  -1.684  1.00  0.29           C  \\nATOM   1280  OE1 GLU A 172      -8.462  20.175  -0.471  1.00  0.42           O  \\nATOM   1281  OE2 GLU A 172      -9.930  20.146  -2.108  1.00  0.35           O  \\nATOM   1282  N   GLY A 173      -2.624  19.097  -2.861  1.00  0.33           N  \\nATOM   1283  CA  GLY A 173      -1.368  19.426  -2.205  1.00  0.32           C  \\nATOM   1284  C   GLY A 173      -1.465  19.412  -0.691  1.00  0.32           C  \\nATOM   1285  O   GLY A 173      -0.557  19.881  -0.001  1.00  0.31           O  \\nATOM   1286  N   ALA A 174      -2.698  18.955  -0.232  1.00  0.28           N  \\nATOM   1287  CA  ALA A 174      -2.924  18.836   1.206  1.00  0.29           C  \\nATOM   1288  C   ALA A 174      -2.476  17.472   1.721  1.00  0.28           C  \\nATOM   1289  CB  ALA A 174      -4.397  19.068   1.534  1.00  0.26           C  \\nATOM   1290  O   ALA A 174      -2.781  16.442   1.114  1.00  0.27           O  \\nATOM   1291  N   SER A 175      -1.269  17.284   2.382  1.00  0.36           N  \\nATOM   1292  CA  SER A 175      -1.083  16.232   3.377  1.00  0.37           C  \\nATOM   1293  C   SER A 175      -2.399  15.881   4.062  1.00  0.36           C  \\nATOM   1294  CB  SER A 175      -0.052  16.659   4.423  1.00  0.35           C  \\nATOM   1295  O   SER A 175      -3.069  16.754   4.618  1.00  0.35           O  \\nATOM   1296  OG  SER A 175       1.238  16.181   4.083  1.00  0.33           O  \\nATOM   1297  N   PRO A 176      -3.168  14.867   3.520  1.00  0.35           N  \\nATOM   1298  CA  PRO A 176      -4.304  14.532   4.382  1.00  0.36           C  \\nATOM   1299  C   PRO A 176      -3.974  14.653   5.868  1.00  0.35           C  \\nATOM   1300  CB  PRO A 176      -4.615  13.081   4.007  1.00  0.34           C  \\nATOM   1301  O   PRO A 176      -2.872  14.292   6.290  1.00  0.34           O  \\nATOM   1302  CG  PRO A 176      -3.382  12.592   3.319  1.00  0.33           C  \\nATOM   1303  CD  PRO A 176      -2.557  13.780   2.915  1.00  0.36           C  \\nATOM   1304  N   GLU A 177      -4.645  15.562   6.636  1.00  0.36           N  \\nATOM   1305  CA  GLU A 177      -4.826  15.470   8.082  1.00  0.36           C  \\nATOM   1306  C   GLU A 177      -5.307  14.080   8.490  1.00  0.36           C  \\nATOM   1307  CB  GLU A 177      -5.815  16.532   8.569  1.00  0.34           C  \\nATOM   1308  O   GLU A 177      -6.286  13.570   7.941  1.00  0.34           O  \\nATOM   1309  CG  GLU A 177      -5.742  16.801  10.065  1.00  0.33           C  \\nATOM   1310  CD  GLU A 177      -6.587  17.986  10.504  1.00  0.39           C  \\nATOM   1311  OE1 GLU A 177      -7.257  18.604   9.646  1.00  0.48           O  \\nATOM   1312  OE2 GLU A 177      -6.579  18.298  11.716  1.00  0.44           O  \\nATOM   1313  N   PRO A 178      -4.392  13.125   8.903  1.00  0.40           N  \\nATOM   1314  CA  PRO A 178      -4.888  11.860   9.450  1.00  0.40           C  \\nATOM   1315  C   PRO A 178      -6.237  12.009  10.150  1.00  0.40           C  \\nATOM   1316  CB  PRO A 178      -3.798  11.454  10.445  1.00  0.38           C  \\nATOM   1317  O   PRO A 178      -6.375  12.826  11.064  1.00  0.39           O  \\nATOM   1318  CG  PRO A 178      -3.011  12.703  10.680  1.00  0.36           C  \\nATOM   1319  CD  PRO A 178      -3.363  13.697   9.611  1.00  0.39           C  \\nATOM   1320  N   GLN A 179      -7.348  11.648   9.455  1.00  0.35           N  \\nATOM   1321  CA  GLN A 179      -8.491  11.407  10.329  1.00  0.38           C  \\nATOM   1322  C   GLN A 179      -8.234  10.223  11.257  1.00  0.35           C  \\nATOM   1323  CB  GLN A 179      -9.756  11.163   9.505  1.00  0.35           C  \\nATOM   1324  O   GLN A 179      -7.668   9.211  10.838  1.00  0.35           O  \\nATOM   1325  CG  GLN A 179     -10.379  12.434   8.944  1.00  0.35           C  \\nATOM   1326  CD  GLN A 179     -11.889  12.464   9.095  1.00  0.42           C  \\nATOM   1327  NE2 GLN A 179     -12.466  13.659   9.038  1.00  0.38           N  \\nATOM   1328  OE1 GLN A 179     -12.530  11.422   9.262  1.00  0.48           O  \\nATOM   1329  N   PRO A 180      -8.316  10.549  12.649  1.00  0.38           N  \\nATOM   1330  CA  PRO A 180      -8.296   9.499  13.670  1.00  0.37           C  \\nATOM   1331  C   PRO A 180      -9.067   8.251  13.248  1.00  0.38           C  \\nATOM   1332  CB  PRO A 180      -8.960  10.168  14.877  1.00  0.36           C  \\nATOM   1333  O   PRO A 180     -10.199   8.352  12.768  1.00  0.37           O  \\nATOM   1334  CG  PRO A 180      -9.674  11.353  14.312  1.00  0.35           C  \\nATOM   1335  CD  PRO A 180      -9.114  11.642  12.949  1.00  0.37           C  \\nATOM   1336  N   LYS A 181      -8.445   7.239  12.659  1.00  0.44           N  \\nATOM   1337  CA  LYS A 181      -9.028   5.901  12.608  1.00  0.44           C  \\nATOM   1338  C   LYS A 181     -10.087   5.721  13.692  1.00  0.44           C  \\nATOM   1339  CB  LYS A 181      -7.940   4.836  12.756  1.00  0.42           C  \\nATOM   1340  O   LYS A 181      -9.886   6.128  14.838  1.00  0.43           O  \\nATOM   1341  CG  LYS A 181      -7.473   4.241  11.436  1.00  0.41           C  \\nATOM   1342  CD  LYS A 181      -6.436   3.146  11.652  1.00  0.44           C  \\nATOM   1343  CE  LYS A 181      -5.959   2.558  10.331  1.00  0.42           C  \\nATOM   1344  NZ  LYS A 181      -4.945   1.481  10.538  1.00  0.41           N  \\nATOM   1345  N   PRO A 182     -11.433   5.585  13.239  1.00  0.39           N  \\nATOM   1346  CA  PRO A 182     -12.398   5.255  14.290  1.00  0.39           C  \\nATOM   1347  C   PRO A 182     -11.887   4.177  15.243  1.00  0.39           C  \\nATOM   1348  CB  PRO A 182     -13.617   4.759  13.508  1.00  0.37           C  \\nATOM   1349  O   PRO A 182     -11.464   3.106  14.800  1.00  0.37           O  \\nATOM   1350  CG  PRO A 182     -13.113   4.535  12.118  1.00  0.36           C  \\nATOM   1351  CD  PRO A 182     -11.744   5.141  12.004  1.00  0.40           C  \\nATOM   1352  N   SER A 183     -11.029   4.469  16.180  1.00  0.38           N  \\nATOM   1353  CA  SER A 183     -10.896   3.624  17.362  1.00  0.39           C  \\nATOM   1354  C   SER A 183     -12.061   2.646  17.475  1.00  0.38           C  \\nATOM   1355  CB  SER A 183     -10.813   4.479  18.627  1.00  0.37           C  \\nATOM   1356  O   SER A 183     -13.225   3.048  17.410  1.00  0.37           O  \\nATOM   1357  OG  SER A 183      -9.749   5.410  18.535  1.00  0.37           O  \\nATOM   1358  N   GLY A 184     -12.001   1.604  16.705  1.00  0.42           N  \\nATOM   1359  CA  GLY A 184     -12.755   0.362  16.773  1.00  0.42           C  \\nATOM   1360  C   GLY A 184     -13.286   0.061  18.162  1.00  0.42           C  \\nATOM   1361  O   GLY A 184     -13.650  -1.078  18.460  1.00  0.40           O  \\nATOM   1362  N   ASP A 185     -13.616   1.088  18.858  1.00  0.38           N  \\nATOM   1363  CA  ASP A 185     -14.581   0.937  19.943  1.00  0.38           C  \\nATOM   1364  C   ASP A 185     -15.826   1.785  19.693  1.00  0.38           C  \\nATOM   1365  CB  ASP A 185     -13.946   1.316  21.283  1.00  0.36           C  \\nATOM   1366  O   ASP A 185     -15.734   3.006  19.551  1.00  0.37           O  \\nATOM   1367  CG  ASP A 185     -13.963   0.180  22.290  1.00  0.36           C  \\nATOM   1368  OD1 ASP A 185     -14.835  -0.710  22.191  1.00  0.40           O  \\nATOM   1369  OD2 ASP A 185     -13.098   0.177  23.193  1.00  0.47           O  \\nATOM   1370  N   PRO A 186     -16.893   1.388  18.986  1.00  0.41           N  \\nATOM   1371  CA  PRO A 186     -18.113   2.187  19.130  1.00  0.40           C  \\nATOM   1372  C   PRO A 186     -18.289   2.746  20.540  1.00  0.40           C  \\nATOM   1373  CB  PRO A 186     -19.227   1.192  18.794  1.00  0.37           C  \\nATOM   1374  O   PRO A 186     -19.174   3.573  20.774  1.00  0.38           O  \\nATOM   1375  CG  PRO A 186     -18.583  -0.152  18.897  1.00  0.36           C  \\nATOM   1376  CD  PRO A 186     -17.093   0.034  18.935  1.00  0.41           C  \\nATOM   1377  N   GLN A 187     -17.332   2.401  21.478  1.00  0.34           N  \\nATOM   1378  CA  GLN A 187     -17.675   3.069  22.729  1.00  0.39           C  \\nATOM   1379  C   GLN A 187     -16.662   4.160  23.066  1.00  0.35           C  \\nATOM   1380  CB  GLN A 187     -17.757   2.057  23.873  1.00  0.32           C  \\nATOM   1381  O   GLN A 187     -16.907   4.994  23.940  1.00  0.33           O  \\nATOM   1382  CG  GLN A 187     -19.061   1.273  23.908  1.00  0.34           C  \\nATOM   1383  CD  GLN A 187     -19.176   0.378  25.128  1.00  0.42           C  \\nATOM   1384  NE2 GLN A 187     -20.234  -0.424  25.178  1.00  0.33           N  \\nATOM   1385  OE1 GLN A 187     -18.320   0.407  26.018  1.00  0.48           O  \\nATOM   1386  N   SER A 188     -15.797   4.595  22.036  1.00  0.37           N  \\nATOM   1387  CA  SER A 188     -15.185   5.851  22.458  1.00  0.39           C  \\nATOM   1388  C   SER A 188     -15.325   6.923  21.382  1.00  0.38           C  \\nATOM   1389  CB  SER A 188     -13.708   5.642  22.792  1.00  0.35           C  \\nATOM   1390  O   SER A 188     -14.791   6.778  20.281  1.00  0.36           O  \\nATOM   1391  OG  SER A 188     -13.501   4.373  23.389  1.00  0.38           O  \\nATOM   1392  N   LYS A 189     -16.612   7.322  21.049  1.00  0.38           N  \\nATOM   1393  CA  LYS A 189     -17.032   8.680  20.717  1.00  0.41           C  \\nATOM   1394  C   LYS A 189     -16.065   9.711  21.293  1.00  0.38           C  \\nATOM   1395  CB  LYS A 189     -18.448   8.945  21.232  1.00  0.35           C  \\nATOM   1396  O   LYS A 189     -16.444  10.515  22.147  1.00  0.36           O  \\nATOM   1397  CG  LYS A 189     -19.540   8.680  20.205  1.00  0.35           C  \\nATOM   1398  CD  LYS A 189     -20.919   9.013  20.757  1.00  0.42           C  \\nATOM   1399  CE  LYS A 189     -22.014   8.729  19.738  1.00  0.33           C  \\nATOM   1400  NZ  LYS A 189     -23.371   9.038  20.281  1.00  0.34           N  \\nATOM   1401  N   GLN A 190     -14.803   9.486  21.348  1.00  0.40           N  \\nATOM   1402  CA  GLN A 190     -14.023  10.714  21.450  1.00  0.42           C  \\nATOM   1403  C   GLN A 190     -12.956  10.780  20.360  1.00  0.40           C  \\nATOM   1404  CB  GLN A 190     -13.371  10.825  22.829  1.00  0.37           C  \\nATOM   1405  O   GLN A 190     -12.034   9.962  20.337  1.00  0.38           O  \\nATOM   1406  CG  GLN A 190     -14.343  11.198  23.940  1.00  0.36           C  \\nATOM   1407  CD  GLN A 190     -13.673  11.300  25.297  1.00  0.42           C  \\nATOM   1408  NE2 GLN A 190     -14.477  11.390  26.351  1.00  0.36           N  \\nATOM   1409  OE1 GLN A 190     -12.442  11.296  25.398  1.00  0.48           O  \\nATOM   1410  N   THR A 191     -13.251  11.105  19.125  1.00  0.41           N  \\nATOM   1411  CA  THR A 191     -12.376  11.885  18.257  1.00  0.42           C  \\nATOM   1412  C   THR A 191     -11.503  12.831  19.078  1.00  0.41           C  \\nATOM   1413  CB  THR A 191     -13.188  12.695  17.229  1.00  0.39           C  \\nATOM   1414  O   THR A 191     -12.016  13.695  19.791  1.00  0.40           O  \\nATOM   1415  CG2 THR A 191     -13.133  12.045  15.851  1.00  0.38           C  \\nATOM   1416  OG1 THR A 191     -14.554  12.767  17.657  1.00  0.36           O  \\nATOM   1417  N   PRO A 192     -10.272  12.528  19.520  1.00  0.41           N  \\nATOM   1418  CA  PRO A 192      -9.532  13.759  19.804  1.00  0.41           C  \\nATOM   1419  C   PRO A 192      -9.392  14.660  18.579  1.00  0.41           C  \\nATOM   1420  CB  PRO A 192      -8.164  13.250  20.266  1.00  0.38           C  \\nATOM   1421  O   PRO A 192      -9.069  14.179  17.489  1.00  0.39           O  \\nATOM   1422  CG  PRO A 192      -8.201  11.777  20.018  1.00  0.37           C  \\nATOM   1423  CD  PRO A 192      -9.578  11.407  19.547  1.00  0.41           C  \\nATOM   1424  N   GLU A 193     -10.253  15.678  18.355  1.00  0.44           N  \\nATOM   1425  CA  GLU A 193      -9.840  16.921  17.712  1.00  0.43           C  \\nATOM   1426  C   GLU A 193      -8.346  16.912  17.401  1.00  0.44           C  \\nATOM   1427  CB  GLU A 193     -10.187  18.124  18.594  1.00  0.40           C  \\nATOM   1428  O   GLU A 193      -7.519  16.811  18.309  1.00  0.41           O  \\nATOM   1429  CG  GLU A 193     -10.091  19.462  17.875  1.00  0.38           C  \\nATOM   1430  CD  GLU A 193     -10.780  20.594  18.621  1.00  0.44           C  \\nATOM   1431  OE1 GLU A 193     -11.201  20.388  19.781  1.00  0.51           O  \\nATOM   1432  OE2 GLU A 193     -10.899  21.695  18.039  1.00  0.45           O  \\nATOM   1433  N   ALA A 194      -7.809  16.182  16.468  1.00  0.43           N  \\nATOM   1434  CA  ALA A 194      -6.468  16.598  16.066  1.00  0.45           C  \\nATOM   1435  C   ALA A 194      -6.327  18.117  16.114  1.00  0.44           C  \\nATOM   1436  CB  ALA A 194      -6.147  16.081  14.666  1.00  0.40           C  \\nATOM   1437  O   ALA A 194      -6.994  18.832  15.363  1.00  0.42           O  \\nATOM   1438  N   SER A 195      -6.316  18.836  17.241  1.00  0.36           N  \\nATOM   1439  CA  SER A 195      -5.726  20.163  17.386  1.00  0.36           C  \\nATOM   1440  C   SER A 195      -4.640  20.402  16.343  1.00  0.36           C  \\nATOM   1441  CB  SER A 195      -5.145  20.341  18.790  1.00  0.34           C  \\nATOM   1442  O   SER A 195      -3.779  19.546  16.129  1.00  0.35           O  \\nATOM   1443  OG  SER A 195      -6.166  20.262  19.770  1.00  0.38           O  \\nATOM   1444  N   PRO A 196      -4.924  21.131  15.256  1.00  0.46           N  \\nATOM   1445  CA  PRO A 196      -3.738  21.644  14.566  1.00  0.45           C  \\nATOM   1446  C   PRO A 196      -2.506  21.695  15.468  1.00  0.46           C  \\nATOM   1447  CB  PRO A 196      -4.160  23.053  14.139  1.00  0.42           C  \\nATOM   1448  O   PRO A 196      -2.608  22.073  16.638  1.00  0.44           O  \\nATOM   1449  CG  PRO A 196      -5.355  23.365  14.981  1.00  0.41           C  \\nATOM   1450  CD  PRO A 196      -5.869  22.084  15.573  1.00  0.45           C  \\nATOM   1451  N   ARG A 197      -1.563  20.752  15.485  1.00  0.46           N  \\nATOM   1452  CA  ARG A 197      -0.269  20.927  16.137  1.00  0.47           C  \\nATOM   1453  C   ARG A 197       0.040  22.404  16.354  1.00  0.46           C  \\nATOM   1454  CB  ARG A 197       0.842  20.275  15.312  1.00  0.42           C  \\nATOM   1455  O   ARG A 197      -0.042  23.205  15.420  1.00  0.44           O  \\nATOM   1456  CG  ARG A 197       2.080  19.917  16.119  1.00  0.41           C  \\nATOM   1457  CD  ARG A 197       3.113  19.187  15.273  1.00  0.45           C  \\nATOM   1458  NE  ARG A 197       4.107  18.510  16.101  1.00  0.40           N  \\nATOM   1459  NH1 ARG A 197       5.201  17.506  14.331  1.00  0.31           N  \\nATOM   1460  NH2 ARG A 197       5.927  17.160  16.478  1.00  0.27           N  \\nATOM   1461  CZ  ARG A 197       5.076  17.727  15.635  1.00  0.38           C  \\nATOM   1462  N   PRO A 198      -0.234  22.984  17.521  1.00  0.40           N  \\nATOM   1463  CA  PRO A 198       0.412  24.278  17.751  1.00  0.39           C  \\nATOM   1464  C   PRO A 198       1.695  24.449  16.942  1.00  0.40           C  \\nATOM   1465  CB  PRO A 198       0.710  24.261  19.252  1.00  0.37           C  \\nATOM   1466  O   PRO A 198       2.585  23.596  17.002  1.00  0.38           O  \\nATOM   1467  CG  PRO A 198       0.466  22.848  19.674  1.00  0.37           C  \\nATOM   1468  CD  PRO A 198      -0.203  22.121  18.543  1.00  0.41           C  \\nATOM   1469  N   GLN A 199       1.784  24.968  15.712  1.00  0.45           N  \\nATOM   1470  CA  GLN A 199       2.952  25.767  15.360  1.00  0.47           C  \\nATOM   1471  C   GLN A 199       3.719  26.199  16.606  1.00  0.45           C  \\nATOM   1472  CB  GLN A 199       2.538  26.994  14.546  1.00  0.43           C  \\nATOM   1473  O   GLN A 199       3.124  26.684  17.571  1.00  0.43           O  \\nATOM   1474  CG  GLN A 199       3.706  27.726  13.899  1.00  0.42           C  \\nATOM   1475  CD  GLN A 199       3.260  28.837  12.968  1.00  0.46           C  \\nATOM   1476  NE2 GLN A 199       4.219  29.590  12.440  1.00  0.40           N  \\nATOM   1477  OE1 GLN A 199       2.062  29.018  12.725  1.00  0.53           O  \\nATOM   1478  N   PRO A 200       4.834  25.480  17.043  1.00  0.45           N  \\nATOM   1479  CA  PRO A 200       5.587  26.072  18.151  1.00  0.45           C  \\nATOM   1480  C   PRO A 200       5.376  27.580  18.269  1.00  0.45           C  \\nATOM   1481  CB  PRO A 200       7.042  25.750  17.801  1.00  0.41           C  \\nATOM   1482  O   PRO A 200       5.456  28.298  17.269  1.00  0.43           O  \\nATOM   1483  CG  PRO A 200       7.000  25.292  16.378  1.00  0.40           C  \\nATOM   1484  CD  PRO A 200       5.569  25.039  16.001  1.00  0.44           C  \\nATOM   1485  N   GLU A 201       4.474  28.137  19.056  1.00  0.45           N  \\nATOM   1486  CA  GLU A 201       4.616  29.499  19.564  1.00  0.48           C  \\nATOM   1487  C   GLU A 201       6.085  29.901  19.661  1.00  0.45           C  \\nATOM   1488  CB  GLU A 201       3.941  29.636  20.931  1.00  0.42           C  \\nATOM   1489  O   GLU A 201       6.897  29.173  20.234  1.00  0.43           O  \\nATOM   1490  CG  GLU A 201       2.429  29.785  20.859  1.00  0.43           C  \\nATOM   1491  CD  GLU A 201       1.789  30.080  22.206  1.00  0.48           C  \\nATOM   1492  OE1 GLU A 201       2.478  29.959  23.244  1.00  0.53           O  \\nATOM   1493  OE2 GLU A 201       0.589  30.435  22.224  1.00  0.47           O  \\nATOM   1494  N   PRO A 202       6.719  30.634  18.577  1.00  0.49           N  \\nATOM   1495  CA  PRO A 202       7.978  31.341  18.821  1.00  0.49           C  \\nATOM   1496  C   PRO A 202       8.131  31.788  20.273  1.00  0.49           C  \\nATOM   1497  CB  PRO A 202       7.893  32.548  17.884  1.00  0.44           C  \\nATOM   1498  O   PRO A 202       7.415  32.685  20.725  1.00  0.47           O  \\nATOM   1499  CG  PRO A 202       6.432  32.734  17.632  1.00  0.43           C  \\nATOM   1500  CD  PRO A 202       5.729  31.439  17.923  1.00  0.47           C  \\nATOM   1501  N   SER A 203       8.388  30.921  21.170  1.00  0.45           N  \\nATOM   1502  CA  SER A 203       8.837  31.351  22.491  1.00  0.49           C  \\nATOM   1503  C   SER A 203      10.038  32.284  22.389  1.00  0.45           C  \\nATOM   1504  CB  SER A 203       9.191  30.142  23.357  1.00  0.41           C  \\nATOM   1505  O   SER A 203      11.029  31.960  21.731  1.00  0.42           O  \\nATOM   1506  OG  SER A 203       8.084  29.266  23.476  1.00  0.46           O  \\nATOM   1507  N   GLY A 204       9.827  33.603  22.348  1.00  0.45           N  \\nATOM   1508  CA  GLY A 204      10.246  34.828  23.010  1.00  0.48           C  \\nATOM   1509  C   GLY A 204      11.670  34.768  23.530  1.00  0.45           C  \\nATOM   1510  O   GLY A 204      11.926  34.195  24.592  1.00  0.42           O  \\nATOM   1511  N   LYS A 205      12.657  34.509  22.624  1.00  0.42           N  \\nATOM   1512  CA  LYS A 205      13.940  35.169  22.847  1.00  0.45           C  \\nATOM   1513  C   LYS A 205      14.669  35.413  21.530  1.00  0.42           C  \\nATOM   1514  CB  LYS A 205      14.816  34.337  23.785  1.00  0.38           C  \\nATOM   1515  O   LYS A 205      14.766  34.514  20.692  1.00  0.40           O  \\nATOM   1516  CG  LYS A 205      14.417  34.429  25.250  1.00  0.39           C  \\nATOM   1517  CD  LYS A 205      15.370  33.643  26.140  1.00  0.45           C  \\nATOM   1518  CE  LYS A 205      14.975  33.739  27.608  1.00  0.37           C  \\nATOM   1519  NZ  LYS A 205      15.884  32.937  28.480  1.00  0.37           N  \\nATOM   1520  N   PRO A 206      14.785  36.611  20.886  1.00  0.50           N  \\nATOM   1521  CA  PRO A 206      15.854  37.028  19.975  1.00  0.52           C  \\nATOM   1522  C   PRO A 206      17.182  36.334  20.266  1.00  0.50           C  \\nATOM   1523  CB  PRO A 206      15.959  38.535  20.221  1.00  0.46           C  \\nATOM   1524  O   PRO A 206      17.811  36.604  21.293  1.00  0.48           O  \\nATOM   1525  CG  PRO A 206      15.203  38.769  21.489  1.00  0.45           C  \\nATOM   1526  CD  PRO A 206      14.437  37.521  21.824  1.00  0.51           C  \\nATOM   1527  N   GLN A 207      17.364  35.141  19.818  1.00  0.44           N  \\nATOM   1528  CA  GLN A 207      18.723  34.613  19.754  1.00  0.48           C  \\nATOM   1529  C   GLN A 207      19.370  34.920  18.407  1.00  0.45           C  \\nATOM   1530  CB  GLN A 207      18.725  33.104  20.006  1.00  0.40           C  \\nATOM   1531  O   GLN A 207      18.745  34.748  17.359  1.00  0.42           O  \\nATOM   1532  CG  GLN A 207      19.232  32.712  21.388  1.00  0.40           C  \\nATOM   1533  CD  GLN A 207      19.102  31.226  21.661  1.00  0.45           C  \\nATOM   1534  NE2 GLN A 207      19.372  30.824  22.898  1.00  0.37           N  \\nATOM   1535  OE1 GLN A 207      18.762  30.444  20.767  1.00  0.49           O  \\nATOM   1536  N   SER A 208      20.181  35.973  18.316  1.00  0.47           N  \\nATOM   1537  CA  SER A 208      21.444  36.426  17.743  1.00  0.52           C  \\nATOM   1538  C   SER A 208      21.679  35.812  16.367  1.00  0.47           C  \\nATOM   1539  CB  SER A 208      22.609  36.080  18.671  1.00  0.42           C  \\nATOM   1540  O   SER A 208      21.504  34.605  16.182  1.00  0.44           O  \\nATOM   1541  OG  SER A 208      22.157  35.907  20.003  1.00  0.45           O  \\nATOM   1542  N   GLU A 209      21.259  36.542  15.228  1.00  0.47           N  \\nATOM   1543  CA  GLU A 209      21.971  36.711  13.965  1.00  0.53           C  \\nATOM   1544  C   GLU A 209      22.943  35.559  13.721  1.00  0.47           C  \\nATOM   1545  CB  GLU A 209      22.721  38.045  13.944  1.00  0.44           C  \\nATOM   1546  O   GLU A 209      24.095  35.609  14.157  1.00  0.45           O  \\nATOM   1547  CG  GLU A 209      21.908  39.199  13.375  1.00  0.45           C  \\nATOM   1548  CD  GLU A 209      22.634  40.533  13.439  1.00  0.52           C  \\nATOM   1549  OE1 GLU A 209      23.750  40.591  14.005  1.00  0.53           O  \\nATOM   1550  OE2 GLU A 209      22.083  41.529  12.919  1.00  0.48           O  \\nATOM   1551  N   GLN A 210      22.635  34.332  13.905  1.00  0.43           N  \\nATOM   1552  CA  GLN A 210      23.242  33.246  13.144  1.00  0.47           C  \\nATOM   1553  C   GLN A 210      22.387  32.876  11.936  1.00  0.43           C  \\nATOM   1554  CB  GLN A 210      23.454  32.019  14.032  1.00  0.38           C  \\nATOM   1555  O   GLN A 210      21.198  32.583  12.077  1.00  0.40           O  \\nATOM   1556  CG  GLN A 210      24.596  32.171  15.028  1.00  0.39           C  \\nATOM   1557  CD  GLN A 210      24.815  30.925  15.865  1.00  0.44           C  \\nATOM   1558  NE2 GLN A 210      25.558  31.067  16.957  1.00  0.37           N  \\nATOM   1559  OE1 GLN A 210      24.319  29.843  15.533  1.00  0.50           O  \\nATOM   1560  N   THR A 211      22.604  33.544  10.709  1.00  0.45           N  \\nATOM   1561  CA  THR A 211      22.258  32.905   9.444  1.00  0.48           C  \\nATOM   1562  C   THR A 211      21.960  31.423   9.652  1.00  0.45           C  \\nATOM   1563  CB  THR A 211      23.389  33.064   8.411  1.00  0.42           C  \\nATOM   1564  O   THR A 211      22.776  30.693  10.218  1.00  0.43           O  \\nATOM   1565  CG2 THR A 211      23.034  34.115   7.365  1.00  0.40           C  \\nATOM   1566  OG1 THR A 211      24.590  33.461   9.083  1.00  0.39           O  \\nATOM   1567  N   PRO A 212      20.792  31.041   9.893  1.00  0.39           N  \\nATOM   1568  CA  PRO A 212      20.604  29.589   9.838  1.00  0.42           C  \\nATOM   1569  C   PRO A 212      21.550  28.909   8.851  1.00  0.40           C  \\nATOM   1570  CB  PRO A 212      19.148  29.439   9.391  1.00  0.36           C  \\nATOM   1571  O   PRO A 212      21.572  29.261   7.668  1.00  0.38           O  \\nATOM   1572  CG  PRO A 212      18.680  30.836   9.138  1.00  0.37           C  \\nATOM   1573  CD  PRO A 212      19.812  31.778   9.430  1.00  0.44           C  \\nATOM   1574  N   LYS A 213      22.833  28.595   9.116  1.00  0.38           N  \\nATOM   1575  CA  LYS A 213      23.317  27.378   8.470  1.00  0.39           C  \\nATOM   1576  C   LYS A 213      22.157  26.529   7.958  1.00  0.38           C  \\nATOM   1577  CB  LYS A 213      24.177  26.562   9.436  1.00  0.35           C  \\nATOM   1578  O   LYS A 213      21.215  26.244   8.700  1.00  0.35           O  \\nATOM   1579  CG  LYS A 213      25.193  25.660   8.750  1.00  0.34           C  \\nATOM   1580  CD  LYS A 213      26.084  24.952   9.762  1.00  0.39           C  \\nATOM   1581  CE  LYS A 213      27.119  24.070   9.076  1.00  0.33           C  \\nATOM   1582  NZ  LYS A 213      28.040  23.428  10.061  1.00  0.35           N  \\nATOM   1583  N   ALA A 214      21.469  26.795   6.809  1.00  0.34           N  \\nATOM   1584  CA  ALA A 214      20.560  25.805   6.238  1.00  0.37           C  \\nATOM   1585  C   ALA A 214      20.614  24.495   7.019  1.00  0.35           C  \\nATOM   1586  CB  ALA A 214      20.895  25.559   4.769  1.00  0.32           C  \\nATOM   1587  O   ALA A 214      21.653  23.832   7.059  1.00  0.33           O  \\nATOM   1588  N   SER A 215      20.320  24.257   8.295  1.00  0.29           N  \\nATOM   1589  CA  SER A 215      19.913  22.941   8.778  1.00  0.31           C  \\nATOM   1590  C   SER A 215      19.495  22.034   7.626  1.00  0.29           C  \\nATOM   1591  CB  SER A 215      18.765  23.069   9.780  1.00  0.27           C  \\nATOM   1592  O   SER A 215      18.855  22.486   6.674  1.00  0.28           O  \\nATOM   1593  OG  SER A 215      19.183  23.775  10.936  1.00  0.30           O  \\nATOM   1594  N   PRO A 216      20.253  21.141   7.010  1.00  0.33           N  \\nATOM   1595  CA  PRO A 216      19.454  20.326   6.093  1.00  0.33           C  \\nATOM   1596  C   PRO A 216      17.957  20.605   6.207  1.00  0.33           C  \\nATOM   1597  CB  PRO A 216      19.777  18.893   6.523  1.00  0.30           C  \\nATOM   1598  O   PRO A 216      17.455  20.860   7.304  1.00  0.31           O  \\nATOM   1599  CG  PRO A 216      20.389  19.030   7.880  1.00  0.29           C  \\nATOM   1600  CD  PRO A 216      20.799  20.461   8.076  1.00  0.35           C  \\nATOM   1601  N   GLU A 217      17.236  21.363   5.461  1.00  0.30           N  \\nATOM   1602  CA  GLU A 217      15.782  21.358   5.590  1.00  0.31           C  \\nATOM   1603  C   GLU A 217      15.318  20.277   6.562  1.00  0.29           C  \\nATOM   1604  CB  GLU A 217      15.122  21.155   4.224  1.00  0.27           C  \\nATOM   1605  O   GLU A 217      15.737  19.122   6.461  1.00  0.27           O  \\nATOM   1606  CG  GLU A 217      13.735  21.771   4.114  1.00  0.28           C  \\nATOM   1607  CD  GLU A 217      13.137  21.658   2.720  1.00  0.35           C  \\nATOM   1608  OE1 GLU A 217      13.802  21.098   1.818  1.00  0.45           O  \\nATOM   1609  OE2 GLU A 217      11.996  22.133   2.529  1.00  0.38           O  \\nATOM   1610  N   PRO A 218      15.286  20.389   7.908  1.00  0.27           N  \\nATOM   1611  CA  PRO A 218      14.618  19.261   8.561  1.00  0.27           C  \\nATOM   1612  C   PRO A 218      14.039  18.260   7.564  1.00  0.27           C  \\nATOM   1613  CB  PRO A 218      13.505  19.931   9.371  1.00  0.25           C  \\nATOM   1614  O   PRO A 218      13.370  18.655   6.606  1.00  0.25           O  \\nATOM   1615  CG  PRO A 218      13.382  21.302   8.789  1.00  0.25           C  \\nATOM   1616  CD  PRO A 218      14.605  21.582   7.964  1.00  0.29           C  \\nATOM   1617  N   GLN A 219      14.642  17.202   6.981  1.00  0.32           N  \\nATOM   1618  CA  GLN A 219      13.686  16.177   6.575  1.00  0.33           C  \\nATOM   1619  C   GLN A 219      12.292  16.480   7.116  1.00  0.32           C  \\nATOM   1620  CB  GLN A 219      14.144  14.797   7.048  1.00  0.30           C  \\nATOM   1621  O   GLN A 219      12.140  16.851   8.282  1.00  0.30           O  \\nATOM   1622  CG  GLN A 219      15.216  14.169   6.167  1.00  0.29           C  \\nATOM   1623  CD  GLN A 219      15.139  12.654   6.139  1.00  0.36           C  \\nATOM   1624  NE2 GLN A 219      15.700  12.052   5.096  1.00  0.31           N  \\nATOM   1625  OE1 GLN A 219      14.581  12.030   7.047  1.00  0.44           O  \\nATOM   1626  N   PRO A 220      11.430  17.248   6.550  1.00  0.32           N  \\nATOM   1627  CA  PRO A 220      10.155  17.312   7.267  1.00  0.32           C  \\nATOM   1628  C   PRO A 220      10.051  16.274   8.381  1.00  0.32           C  \\nATOM   1629  CB  PRO A 220       9.121  17.042   6.171  1.00  0.30           C  \\nATOM   1630  O   PRO A 220      10.453  15.122   8.196  1.00  0.30           O  \\nATOM   1631  CG  PRO A 220       9.909  16.481   5.032  1.00  0.28           C  \\nATOM   1632  CD  PRO A 220      11.360  16.798   5.254  1.00  0.32           C  \\nATOM   1633  N   LYS A 221      10.469  16.449   9.711  1.00  0.32           N  \\nATOM   1634  CA  LYS A 221       9.855  15.561  10.693  1.00  0.32           C  \\nATOM   1635  C   LYS A 221       8.774  14.696  10.052  1.00  0.31           C  \\nATOM   1636  CB  LYS A 221       9.262  16.367  11.850  1.00  0.29           C  \\nATOM   1637  O   LYS A 221       7.995  15.176   9.226  1.00  0.30           O  \\nATOM   1638  CG  LYS A 221      10.292  16.844  12.863  1.00  0.29           C  \\nATOM   1639  CD  LYS A 221       9.635  17.570  14.030  1.00  0.32           C  \\nATOM   1640  CE  LYS A 221      10.667  18.083  15.024  1.00  0.30           C  \\nATOM   1641  NZ  LYS A 221      10.030  18.831  16.149  1.00  0.32           N  \\nATOM   1642  N   PRO A 222       8.963  13.404   9.584  1.00  0.34           N  \\nATOM   1643  CA  PRO A 222       7.864  12.653   8.973  1.00  0.34           C  \\nATOM   1644  C   PRO A 222       6.512  13.343   9.141  1.00  0.35           C  \\nATOM   1645  CB  PRO A 222       7.891  11.317   9.719  1.00  0.32           C  \\nATOM   1646  O   PRO A 222       6.196  13.833  10.228  1.00  0.33           O  \\nATOM   1647  CG  PRO A 222       8.998  11.458  10.714  1.00  0.31           C  \\nATOM   1648  CD  PRO A 222       9.552  12.850  10.617  1.00  0.35           C  \\nATOM   1649  N   SER A 223       6.138  14.272   8.376  1.00  0.35           N  \\nATOM   1650  CA  SER A 223       4.734  14.669   8.384  1.00  0.36           C  \\nATOM   1651  C   SER A 223       3.873  13.637   9.106  1.00  0.35           C  \\nATOM   1652  CB  SER A 223       4.222  14.862   6.956  1.00  0.34           C  \\nATOM   1653  O   SER A 223       4.273  12.480   9.252  1.00  0.34           O  \\nATOM   1654  OG  SER A 223       4.255  13.640   6.240  1.00  0.31           O  \\nATOM   1655  N   GLY A 224       3.321  13.942  10.354  1.00  0.42           N  \\nATOM   1656  CA  GLY A 224       2.263  13.082  10.860  1.00  0.41           C  \\nATOM   1657  C   GLY A 224       2.044  11.844  10.012  1.00  0.42           C  \\nATOM   1658  O   GLY A 224       1.889  10.742  10.542  1.00  0.40           O  \\nATOM   1659  N   GLN A 225       2.246  12.010   8.687  1.00  0.43           N  \\nATOM   1660  CA  GLN A 225       2.024  10.898   7.769  1.00  0.45           C  \\nATOM   1661  C   GLN A 225       3.084   9.815   7.952  1.00  0.43           C  \\nATOM   1662  CB  GLN A 225       2.019  11.388   6.321  1.00  0.42           C  \\nATOM   1663  O   GLN A 225       2.764   8.625   7.982  1.00  0.43           O  \\nATOM   1664  CG  GLN A 225       0.684  11.965   5.872  1.00  0.42           C  \\nATOM   1665  CD  GLN A 225       0.702  12.433   4.428  1.00  0.48           C  \\nATOM   1666  NE2 GLN A 225      -0.430  12.944   3.956  1.00  0.40           N  \\nATOM   1667  OE1 GLN A 225       1.725  12.337   3.744  1.00  0.48           O  \\nATOM   1668  N   ALA A 226       4.403  10.107   8.075  1.00  0.47           N  \\nATOM   1669  CA  ALA A 226       5.412   9.077   8.308  1.00  0.46           C  \\nATOM   1670  C   ALA A 226       5.163   8.354   9.629  1.00  0.47           C  \\nATOM   1671  CB  ALA A 226       6.810   9.690   8.296  1.00  0.45           C  \\nATOM   1672  O   ALA A 226       5.311   7.133   9.712  1.00  0.46           O  \\nATOM   1673  N   LYS A 227       4.821   9.108  10.718  1.00  0.49           N  \\nATOM   1674  CA  LYS A 227       4.494   8.453  11.981  1.00  0.48           C  \\nATOM   1675  C   LYS A 227       3.301   7.515  11.820  1.00  0.49           C  \\nATOM   1676  CB  LYS A 227       4.201   9.491  13.065  1.00  0.47           C  \\nATOM   1677  O   LYS A 227       3.306   6.400  12.347  1.00  0.48           O  \\nATOM   1678  CG  LYS A 227       5.445  10.158  13.635  1.00  0.45           C  \\nATOM   1679  CD  LYS A 227       5.092  11.148  14.737  1.00  0.46           C  \\nATOM   1680  CE  LYS A 227       6.333  11.834  15.292  1.00  0.41           C  \\nATOM   1681  NZ  LYS A 227       5.989  12.843  16.338  1.00  0.40           N  \\nATOM   1682  N   GLU A 228       2.255   8.081  11.146  1.00  0.51           N  \\nATOM   1683  CA  GLU A 228       1.076   7.260  10.888  1.00  0.51           C  \\nATOM   1684  C   GLU A 228       1.442   5.995  10.117  1.00  0.51           C  \\nATOM   1685  CB  GLU A 228       0.022   8.059  10.117  1.00  0.50           C  \\nATOM   1686  O   GLU A 228       1.015   4.896  10.478  1.00  0.51           O  \\nATOM   1687  CG  GLU A 228      -1.376   7.462  10.190  1.00  0.49           C  \\nATOM   1688  CD  GLU A 228      -2.467   8.438   9.778  1.00  0.51           C  \\nATOM   1689  OE1 GLU A 228      -2.148   9.609   9.468  1.00  0.51           O  \\nATOM   1690  OE2 GLU A 228      -3.649   8.030   9.767  1.00  0.47           O  \\nATOM   1691  N   ILE A 229       2.288   6.170   9.087  1.00  0.54           N  \\nATOM   1692  CA  ILE A 229       2.730   5.033   8.287  1.00  0.54           C  \\nATOM   1693  C   ILE A 229       3.494   4.047   9.169  1.00  0.55           C  \\nATOM   1694  CB  ILE A 229       3.612   5.485   7.101  1.00  0.53           C  \\nATOM   1695  O   ILE A 229       3.282   2.835   9.083  1.00  0.54           O  \\nATOM   1696  CG1 ILE A 229       2.775   6.267   6.082  1.00  0.49           C  \\nATOM   1697  CG2 ILE A 229       4.292   4.281   6.444  1.00  0.50           C  \\nATOM   1698  CD1 ILE A 229       3.601   6.981   5.021  1.00  0.49           C  \\nATOM   1699  N   GLN A 230       4.456   4.549  10.071  1.00  0.55           N  \\nATOM   1700  CA  GLN A 230       5.191   3.690  10.994  1.00  0.55           C  \\nATOM   1701  C   GLN A 230       4.240   2.940  11.923  1.00  0.56           C  \\nATOM   1702  CB  GLN A 230       6.187   4.512  11.815  1.00  0.54           C  \\nATOM   1703  O   GLN A 230       4.427   1.749  12.180  1.00  0.56           O  \\nATOM   1704  CG  GLN A 230       7.510   4.763  11.103  1.00  0.53           C  \\nATOM   1705  CD  GLN A 230       8.458   5.625  11.915  1.00  0.54           C  \\nATOM   1706  NE2 GLN A 230       9.662   5.836  11.393  1.00  0.45           N  \\nATOM   1707  OE1 GLN A 230       8.112   6.098  13.002  1.00  0.55           O  \\nATOM   1708  N   TRP A 231       3.264   3.622  12.526  1.00  0.39           N  \\nATOM   1709  CA  TRP A 231       2.270   2.985  13.383  1.00  0.46           C  \\nATOM   1710  C   TRP A 231       1.519   1.893  12.627  1.00  0.39           C  \\nATOM   1711  CB  TRP A 231       1.281   4.021  13.923  1.00  0.39           C  \\nATOM   1712  O   TRP A 231       1.385   0.770  13.118  1.00  0.39           O  \\nATOM   1713  CG  TRP A 231       0.353   3.489  14.973  1.00  0.38           C  \\nATOM   1714  CD1 TRP A 231       0.657   3.208  16.276  1.00  0.34           C  \\nATOM   1715  CD2 TRP A 231      -1.032   3.167  14.806  1.00  0.37           C  \\nATOM   1716  CE2 TRP A 231      -1.505   2.697  16.051  1.00  0.36           C  \\nATOM   1717  CE3 TRP A 231      -1.919   3.232  13.723  1.00  0.47           C  \\nATOM   1718  NE1 TRP A 231      -0.456   2.732  16.930  1.00  0.44           N  \\nATOM   1719  CH2 TRP A 231      -3.675   2.367  15.168  1.00  0.37           C  \\nATOM   1720  CZ2 TRP A 231      -2.828   2.294  16.243  1.00  0.43           C  \\nATOM   1721  CZ3 TRP A 231      -3.236   2.830  13.916  1.00  0.36           C  \\nATOM   1722  N   GLN A 232       1.057   2.170  11.379  1.00  0.54           N  \\nATOM   1723  CA  GLN A 232       0.341   1.188  10.571  1.00  0.54           C  \\nATOM   1724  C   GLN A 232       1.197  -0.052  10.328  1.00  0.54           C  \\nATOM   1725  CB  GLN A 232      -0.088   1.800   9.237  1.00  0.53           C  \\nATOM   1726  O   GLN A 232       0.716  -1.180  10.455  1.00  0.55           O  \\nATOM   1727  CG  GLN A 232      -1.228   2.802   9.359  1.00  0.52           C  \\nATOM   1728  CD  GLN A 232      -1.599   3.434   8.031  1.00  0.54           C  \\nATOM   1729  NE2 GLN A 232      -2.594   4.315   8.050  1.00  0.46           N  \\nATOM   1730  OE1 GLN A 232      -0.995   3.135   6.996  1.00  0.54           O  \\nATOM   1731  N   ALA A 233       2.525   0.132   9.941  1.00  0.55           N  \\nATOM   1732  CA  ALA A 233       3.436  -0.992   9.743  1.00  0.55           C  \\nATOM   1733  C   ALA A 233       3.541  -1.842  11.006  1.00  0.56           C  \\nATOM   1734  CB  ALA A 233       4.817  -0.491   9.327  1.00  0.54           C  \\nATOM   1735  O   ALA A 233       3.517  -3.074  10.936  1.00  0.57           O  \\nATOM   1736  N   LYS A 234       3.618  -1.189  12.197  1.00  0.57           N  \\nATOM   1737  CA  LYS A 234       3.684  -1.932  13.452  1.00  0.57           C  \\nATOM   1738  C   LYS A 234       2.397  -2.716  13.695  1.00  0.57           C  \\nATOM   1739  CB  LYS A 234       3.949  -0.985  14.623  1.00  0.55           C  \\nATOM   1740  O   LYS A 234       2.440  -3.877  14.107  1.00  0.58           O  \\nATOM   1741  CG  LYS A 234       5.402  -0.949  15.073  1.00  0.54           C  \\nATOM   1742  CD  LYS A 234       5.611   0.042  16.211  1.00  0.55           C  \\nATOM   1743  CE  LYS A 234       7.074   0.117  16.627  1.00  0.52           C  \\nATOM   1744  NZ  LYS A 234       7.284   1.092  17.739  1.00  0.48           N  \\nATOM   1745  N   GLU A 235       1.225  -2.063  13.520  1.00  0.56           N  \\nATOM   1746  CA  GLU A 235      -0.046  -2.763  13.677  1.00  0.56           C  \\nATOM   1747  C   GLU A 235      -0.117  -3.989  12.771  1.00  0.55           C  \\nATOM   1748  CB  GLU A 235      -1.218  -1.823  13.381  1.00  0.55           C  \\nATOM   1749  O   GLU A 235      -0.518  -5.069  13.210  1.00  0.56           O  \\nATOM   1750  CG  GLU A 235      -2.555  -2.321  13.910  1.00  0.54           C  \\nATOM   1751  CD  GLU A 235      -3.646  -1.262  13.875  1.00  0.55           C  \\nATOM   1752  OE1 GLU A 235      -3.335  -0.079  13.611  1.00  0.55           O  \\nATOM   1753  OE2 GLU A 235      -4.821  -1.620  14.115  1.00  0.51           O  \\nATOM   1754  N   ILE A 236       0.298  -3.847  11.523  1.00  0.57           N  \\nATOM   1755  CA  ILE A 236       0.318  -4.954  10.572  1.00  0.56           C  \\nATOM   1756  C   ILE A 236       1.208  -6.074  11.105  1.00  0.57           C  \\nATOM   1757  CB  ILE A 236       0.808  -4.495   9.181  1.00  0.55           C  \\nATOM   1758  O   ILE A 236       0.834  -7.248  11.058  1.00  0.56           O  \\nATOM   1759  CG1 ILE A 236      -0.224  -3.566   8.531  1.00  0.51           C  \\nATOM   1760  CG2 ILE A 236       1.099  -5.703   8.285  1.00  0.53           C  \\nATOM   1761  CD1 ILE A 236       0.297  -2.821   7.309  1.00  0.52           C  \\nATOM   1762  N   GLN A 237       2.336  -5.772  11.649  1.00  0.58           N  \\nATOM   1763  CA  GLN A 237       3.244  -6.762  12.217  1.00  0.59           C  \\nATOM   1764  C   GLN A 237       2.599  -7.491  13.393  1.00  0.59           C  \\nATOM   1765  CB  GLN A 237       4.550  -6.102  12.661  1.00  0.57           C  \\nATOM   1766  O   GLN A 237       2.731  -8.709  13.522  1.00  0.60           O  \\nATOM   1767  CG  GLN A 237       5.540  -5.872  11.528  1.00  0.56           C  \\nATOM   1768  CD  GLN A 237       6.789  -5.138  11.978  1.00  0.57           C  \\nATOM   1769  NE2 GLN A 237       7.746  -4.982  11.070  1.00  0.50           N  \\nATOM   1770  OE1 GLN A 237       6.892  -4.713  13.134  1.00  0.60           O  \\nATOM   1771  N   GLU A 238       1.916  -6.727  14.273  1.00  0.58           N  \\nATOM   1772  CA  GLU A 238       1.272  -7.354  15.423  1.00  0.58           C  \\nATOM   1773  C   GLU A 238       0.165  -8.309  14.983  1.00  0.57           C  \\nATOM   1774  CB  GLU A 238       0.705  -6.292  16.369  1.00  0.56           C  \\nATOM   1775  O   GLU A 238       0.030  -9.405  15.530  1.00  0.57           O  \\nATOM   1776  CG  GLU A 238       1.763  -5.579  17.198  1.00  0.54           C  \\nATOM   1777  CD  GLU A 238       2.501  -6.504  18.153  1.00  0.56           C  \\nATOM   1778  OE1 GLU A 238       1.900  -7.498  18.620  1.00  0.58           O  \\nATOM   1779  OE2 GLU A 238       3.689  -6.231  18.437  1.00  0.55           O  \\nATOM   1780  N   GLN A 239      -0.502  -8.000  13.958  1.00  0.56           N  \\nATOM   1781  CA  GLN A 239      -1.554  -8.848  13.409  1.00  0.56           C  \\nATOM   1782  C   GLN A 239      -0.971 -10.106  12.772  1.00  0.55           C  \\nATOM   1783  CB  GLN A 239      -2.385  -8.077  12.382  1.00  0.54           C  \\nATOM   1784  O   GLN A 239      -1.618 -11.155  12.752  1.00  0.55           O  \\nATOM   1785  CG  GLN A 239      -3.286  -7.013  12.994  1.00  0.54           C  \\nATOM   1786  CD  GLN A 239      -4.201  -6.361  11.975  1.00  0.55           C  \\nATOM   1787  NE2 GLN A 239      -4.882  -5.296  12.384  1.00  0.48           N  \\nATOM   1788  OE1 GLN A 239      -4.294  -6.810  10.828  1.00  0.56           O  \\nATOM   1789  N   ALA A 240       0.225  -9.973  12.238  1.00  0.55           N  \\nATOM   1790  CA  ALA A 240       0.901 -11.140  11.678  1.00  0.55           C  \\nATOM   1791  C   ALA A 240       1.153 -12.195  12.752  1.00  0.56           C  \\nATOM   1792  CB  ALA A 240       2.216 -10.730  11.021  1.00  0.52           C  \\nATOM   1793  O   ALA A 240       1.223 -13.390  12.453  1.00  0.56           O  \\nATOM   1794  N   LYS A 241       1.368 -11.821  14.042  1.00  0.53           N  \\nATOM   1795  CA  LYS A 241       1.817 -12.713  15.108  1.00  0.52           C  \\nATOM   1796  C   LYS A 241       0.768 -13.778  15.412  1.00  0.52           C  \\nATOM   1797  CB  LYS A 241       2.139 -11.918  16.374  1.00  0.49           C  \\nATOM   1798  O   LYS A 241       1.089 -14.834  15.961  1.00  0.50           O  \\nATOM   1799  CG  LYS A 241       3.436 -11.125  16.294  1.00  0.47           C  \\nATOM   1800  CD  LYS A 241       3.687 -10.335  17.571  1.00  0.48           C  \\nATOM   1801  CE  LYS A 241       4.957  -9.500  17.472  1.00  0.52           C  \\nATOM   1802  NZ  LYS A 241       5.115  -8.590  18.646  1.00  0.47           N  \\nATOM   1803  N   GLY A 242      -0.370 -13.983  14.742  1.00  0.49           N  \\nATOM   1804  CA  GLY A 242      -1.403 -14.986  14.948  1.00  0.49           C  \\nATOM   1805  C   GLY A 242      -1.790 -15.713  13.675  1.00  0.49           C  \\nATOM   1806  O   GLY A 242      -2.646 -16.601  13.697  1.00  0.48           O  \\nATOM   1807  N   LYS A 243      -1.108 -15.451  12.601  1.00  0.50           N  \\nATOM   1808  CA  LYS A 243      -1.477 -15.998  11.298  1.00  0.53           C  \\nATOM   1809  C   LYS A 243      -0.603 -17.195  10.937  1.00  0.52           C  \\nATOM   1810  CB  LYS A 243      -1.369 -14.924  10.215  1.00  0.49           C  \\nATOM   1811  O   LYS A 243       0.494 -17.354  11.474  1.00  0.53           O  \\nATOM   1812  CG  LYS A 243      -2.371 -13.789  10.365  1.00  0.47           C  \\nATOM   1813  CD  LYS A 243      -2.743 -13.189   9.015  1.00  0.53           C  \\nATOM   1814  CE  LYS A 243      -3.841 -12.143   9.150  1.00  0.44           C  \\nATOM   1815  NZ  LYS A 243      -4.213 -11.557   7.828  1.00  0.41           N  \\nATOM   1816  N   PRO A 244      -1.173 -18.180  10.166  1.00  0.48           N  \\nATOM   1817  CA  PRO A 244      -0.308 -19.234   9.632  1.00  0.47           C  \\nATOM   1818  C   PRO A 244       0.980 -18.688   9.020  1.00  0.48           C  \\nATOM   1819  CB  PRO A 244      -1.180 -19.900   8.565  1.00  0.43           C  \\nATOM   1820  O   PRO A 244       0.967 -17.624   8.395  1.00  0.47           O  \\nATOM   1821  CG  PRO A 244      -2.576 -19.478   8.895  1.00  0.41           C  \\nATOM   1822  CD  PRO A 244      -2.518 -18.168   9.626  1.00  0.44           C  \\nATOM   1823  N   SER A 245       2.236 -19.327   9.159  1.00  0.43           N  \\nATOM   1824  CA  SER A 245       3.605 -18.885   8.915  1.00  0.45           C  \\nATOM   1825  C   SER A 245       3.736 -18.218   7.550  1.00  0.43           C  \\nATOM   1826  CB  SER A 245       4.575 -20.064   9.010  1.00  0.39           C  \\nATOM   1827  O   SER A 245       4.388 -17.180   7.420  1.00  0.42           O  \\nATOM   1828  OG  SER A 245       4.263 -21.050   8.042  1.00  0.34           O  \\nATOM   1829  N   GLY A 246       3.184 -18.843   6.490  1.00  0.48           N  \\nATOM   1830  CA  GLY A 246       3.319 -18.263   5.164  1.00  0.47           C  \\nATOM   1831  C   GLY A 246       2.684 -16.891   5.045  1.00  0.47           C  \\nATOM   1832  O   GLY A 246       3.299 -15.959   4.523  1.00  0.46           O  \\nATOM   1833  N   GLN A 247       1.483 -16.749   5.553  1.00  0.47           N  \\nATOM   1834  CA  GLN A 247       0.789 -15.465   5.569  1.00  0.48           C  \\nATOM   1835  C   GLN A 247       1.502 -14.465   6.474  1.00  0.47           C  \\nATOM   1836  CB  GLN A 247      -0.661 -15.644   6.023  1.00  0.46           C  \\nATOM   1837  O   GLN A 247       1.678 -13.302   6.106  1.00  0.47           O  \\nATOM   1838  CG  GLN A 247      -1.568 -16.247   4.960  1.00  0.45           C  \\nATOM   1839  CD  GLN A 247      -3.008 -16.374   5.421  1.00  0.49           C  \\nATOM   1840  NE2 GLN A 247      -3.827 -17.056   4.627  1.00  0.42           N  \\nATOM   1841  OE1 GLN A 247      -3.381 -15.865   6.483  1.00  0.52           O  \\nATOM   1842  N   ALA A 248       1.972 -14.924   7.648  1.00  0.49           N  \\nATOM   1843  CA  ALA A 248       2.682 -14.075   8.600  1.00  0.49           C  \\nATOM   1844  C   ALA A 248       3.938 -13.477   7.972  1.00  0.50           C  \\nATOM   1845  CB  ALA A 248       3.044 -14.868   9.854  1.00  0.46           C  \\nATOM   1846  O   ALA A 248       4.210 -12.284   8.128  1.00  0.49           O  \\nATOM   1847  N   LYS A 249       4.712 -14.321   7.220  1.00  0.48           N  \\nATOM   1848  CA  LYS A 249       5.949 -13.858   6.597  1.00  0.48           C  \\nATOM   1849  C   LYS A 249       5.668 -12.784   5.550  1.00  0.48           C  \\nATOM   1850  CB  LYS A 249       6.699 -15.028   5.959  1.00  0.45           C  \\nATOM   1851  O   LYS A 249       6.380 -11.781   5.476  1.00  0.47           O  \\nATOM   1852  CG  LYS A 249       7.450 -15.899   6.955  1.00  0.43           C  \\nATOM   1853  CD  LYS A 249       8.171 -17.048   6.261  1.00  0.44           C  \\nATOM   1854  CE  LYS A 249       8.904 -17.933   7.259  1.00  0.43           C  \\nATOM   1855  NZ  LYS A 249       9.590 -19.077   6.586  1.00  0.42           N  \\nATOM   1856  N   GLU A 250       4.615 -13.087   4.714  1.00  0.52           N  \\nATOM   1857  CA  GLU A 250       4.215 -12.137   3.680  1.00  0.52           C  \\nATOM   1858  C   GLU A 250       3.832 -10.790   4.286  1.00  0.52           C  \\nATOM   1859  CB  GLU A 250       3.050 -12.695   2.859  1.00  0.50           C  \\nATOM   1860  O   GLU A 250       4.295  -9.743   3.829  1.00  0.52           O  \\nATOM   1861  CG  GLU A 250       2.698 -11.853   1.641  1.00  0.49           C  \\nATOM   1862  CD  GLU A 250       2.023 -12.650   0.536  1.00  0.52           C  \\nATOM   1863  OE1 GLU A 250       1.810 -13.871   0.711  1.00  0.53           O  \\nATOM   1864  OE2 GLU A 250       1.707 -12.049  -0.515  1.00  0.50           O  \\nATOM   1865  N   ILE A 251       3.057 -10.815   5.351  1.00  0.55           N  \\nATOM   1866  CA  ILE A 251       2.579  -9.630   6.053  1.00  0.55           C  \\nATOM   1867  C   ILE A 251       3.761  -8.871   6.651  1.00  0.55           C  \\nATOM   1868  CB  ILE A 251       1.565  -9.999   7.159  1.00  0.53           C  \\nATOM   1869  O   ILE A 251       3.830  -7.643   6.559  1.00  0.55           O  \\nATOM   1870  CG1 ILE A 251       0.270 -10.537   6.538  1.00  0.48           C  \\nATOM   1871  CG2 ILE A 251       1.281  -8.792   8.058  1.00  0.50           C  \\nATOM   1872  CD1 ILE A 251      -0.675 -11.184   7.541  1.00  0.49           C  \\nATOM   1873  N   GLN A 252       4.746  -9.613   7.298  1.00  0.54           N  \\nATOM   1874  CA  GLN A 252       5.946  -9.022   7.879  1.00  0.54           C  \\nATOM   1875  C   GLN A 252       6.790  -8.328   6.814  1.00  0.54           C  \\nATOM   1876  CB  GLN A 252       6.778 -10.088   8.594  1.00  0.52           C  \\nATOM   1877  O   GLN A 252       7.310  -7.234   7.042  1.00  0.54           O  \\nATOM   1878  CG  GLN A 252       6.289 -10.409  10.000  1.00  0.50           C  \\nATOM   1879  CD  GLN A 252       7.071 -11.534  10.652  1.00  0.51           C  \\nATOM   1880  NE2 GLN A 252       6.793 -11.790  11.925  1.00  0.46           N  \\nATOM   1881  OE1 GLN A 252       7.919 -12.168  10.015  1.00  0.56           O  \\nATOM   1882  N   TRP A 253       7.021  -8.989   5.660  1.00  0.42           N  \\nATOM   1883  CA  TRP A 253       7.764  -8.401   4.550  1.00  0.48           C  \\nATOM   1884  C   TRP A 253       7.119  -7.096   4.094  1.00  0.41           C  \\nATOM   1885  CB  TRP A 253       7.845  -9.381   3.377  1.00  0.40           C  \\nATOM   1886  O   TRP A 253       7.798  -6.075   3.958  1.00  0.41           O  \\nATOM   1887  CG  TRP A 253       8.696  -8.901   2.240  1.00  0.39           C  \\nATOM   1888  CD1 TRP A 253      10.061  -8.914   2.169  1.00  0.35           C  \\nATOM   1889  CD2 TRP A 253       8.236  -8.332   1.010  1.00  0.38           C  \\nATOM   1890  CE2 TRP A 253       9.378  -8.022   0.240  1.00  0.37           C  \\nATOM   1891  CE3 TRP A 253       6.967  -8.053   0.485  1.00  0.49           C  \\nATOM   1892  NE1 TRP A 253      10.478  -8.387   0.968  1.00  0.49           N  \\nATOM   1893  CH2 TRP A 253       8.036  -7.186  -1.521  1.00  0.44           C  \\nATOM   1894  CZ2 TRP A 253       9.289  -7.448  -1.031  1.00  0.55           C  \\nATOM   1895  CZ3 TRP A 253       6.880  -7.481  -0.779  1.00  0.39           C  \\nATOM   1896  N   GLN A 254       5.878  -7.045   3.913  1.00  0.54           N  \\nATOM   1897  CA  GLN A 254       5.146  -5.868   3.457  1.00  0.54           C  \\nATOM   1898  C   GLN A 254       5.292  -4.713   4.443  1.00  0.54           C  \\nATOM   1899  CB  GLN A 254       3.667  -6.201   3.253  1.00  0.53           C  \\nATOM   1900  O   GLN A 254       5.525  -3.572   4.041  1.00  0.54           O  \\nATOM   1901  CG  GLN A 254       3.393  -7.056   2.023  1.00  0.52           C  \\nATOM   1902  CD  GLN A 254       1.927  -7.417   1.874  1.00  0.54           C  \\nATOM   1903  NE2 GLN A 254       1.601  -8.147   0.812  1.00  0.47           N  \\nATOM   1904  OE1 GLN A 254       1.095  -7.044   2.706  1.00  0.55           O  \\nATOM   1905  N   ALA A 255       5.092  -4.980   5.822  1.00  0.55           N  \\nATOM   1906  CA  ALA A 255       5.286  -3.980   6.869  1.00  0.55           C  \\nATOM   1907  C   ALA A 255       6.681  -3.365   6.788  1.00  0.56           C  \\nATOM   1908  CB  ALA A 255       5.060  -4.601   8.245  1.00  0.53           C  \\nATOM   1909  O   ALA A 255       6.838  -2.150   6.923  1.00  0.55           O  \\nATOM   1910  N   LYS A 256       7.709  -4.219   6.530  1.00  0.56           N  \\nATOM   1911  CA  LYS A 256       9.073  -3.718   6.392  1.00  0.55           C  \\nATOM   1912  C   LYS A 256       9.203  -2.803   5.178  1.00  0.56           C  \\nATOM   1913  CB  LYS A 256      10.063  -4.879   6.281  1.00  0.54           C  \\nATOM   1914  O   LYS A 256       9.839  -1.750   5.253  1.00  0.55           O  \\nATOM   1915  CG  LYS A 256      11.483  -4.524   6.699  1.00  0.52           C  \\nATOM   1916  CD  LYS A 256      12.406  -5.733   6.624  1.00  0.53           C  \\nATOM   1917  CE  LYS A 256      13.842  -5.364   6.973  1.00  0.52           C  \\nATOM   1918  NZ  LYS A 256      14.763  -6.532   6.837  1.00  0.48           N  \\nATOM   1919  N   GLU A 257       8.693  -3.217   3.993  1.00  0.54           N  \\nATOM   1920  CA  GLU A 257       8.714  -2.364   2.808  1.00  0.54           C  \\nATOM   1921  C   GLU A 257       8.079  -1.006   3.095  1.00  0.53           C  \\nATOM   1922  CB  GLU A 257       7.994  -3.045   1.642  1.00  0.53           C  \\nATOM   1923  O   GLU A 257       8.636   0.033   2.736  1.00  0.53           O  \\nATOM   1924  CG  GLU A 257       8.693  -2.871   0.301  1.00  0.53           C  \\nATOM   1925  CD  GLU A 257      10.110  -2.336   0.429  1.00  0.54           C  \\nATOM   1926  OE1 GLU A 257      10.446  -1.753   1.485  1.00  0.54           O  \\nATOM   1927  OE2 GLU A 257      10.891  -2.501  -0.535  1.00  0.51           O  \\nATOM   1928  N   ILE A 258       6.923  -1.020   3.769  1.00  0.55           N  \\nATOM   1929  CA  ILE A 258       6.203   0.192   4.143  1.00  0.55           C  \\nATOM   1930  C   ILE A 258       7.100   1.081   5.002  1.00  0.55           C  \\nATOM   1931  CB  ILE A 258       4.895  -0.137   4.896  1.00  0.53           C  \\nATOM   1932  O   ILE A 258       7.180   2.291   4.781  1.00  0.54           O  \\nATOM   1933  CG1 ILE A 258       3.894  -0.819   3.956  1.00  0.50           C  \\nATOM   1934  CG2 ILE A 258       4.292   1.130   5.511  1.00  0.52           C  \\nATOM   1935  CD1 ILE A 258       2.711  -1.459   4.670  1.00  0.51           C  \\nATOM   1936  N   GLN A 259       7.833   0.534   5.999  1.00  0.57           N  \\nATOM   1937  CA  GLN A 259       8.740   1.266   6.877  1.00  0.57           C  \\nATOM   1938  C   GLN A 259       9.883   1.894   6.085  1.00  0.57           C  \\nATOM   1939  CB  GLN A 259       9.297   0.344   7.963  1.00  0.56           C  \\nATOM   1940  O   GLN A 259      10.256   3.043   6.329  1.00  0.57           O  \\nATOM   1941  CG  GLN A 259       8.387   0.201   9.175  1.00  0.55           C  \\nATOM   1942  CD  GLN A 259       8.939  -0.756  10.214  1.00  0.55           C  \\nATOM   1943  NE2 GLN A 259       8.230  -0.895  11.329  1.00  0.48           N  \\nATOM   1944  OE1 GLN A 259       9.994  -1.366  10.016  1.00  0.57           O  \\nATOM   1945  N   GLU A 260      10.477   1.096   5.162  1.00  0.56           N  \\nATOM   1946  CA  GLU A 260      11.605   1.597   4.383  1.00  0.55           C  \\nATOM   1947  C   GLU A 260      11.184   2.754   3.482  1.00  0.55           C  \\nATOM   1948  CB  GLU A 260      12.220   0.474   3.544  1.00  0.54           C  \\nATOM   1949  O   GLU A 260      11.893   3.758   3.380  1.00  0.54           O  \\nATOM   1950  CG  GLU A 260      13.101  -0.478   4.340  1.00  0.52           C  \\nATOM   1951  CD  GLU A 260      14.536  -0.527   3.840  1.00  0.53           C  \\nATOM   1952  OE1 GLU A 260      14.795  -0.092   2.695  1.00  0.55           O  \\nATOM   1953  OE2 GLU A 260      15.408  -1.005   4.600  1.00  0.52           O  \\nATOM   1954  N   GLN A 261      10.029   2.677   2.985  1.00  0.51           N  \\nATOM   1955  CA  GLN A 261       9.462   3.743   2.164  1.00  0.51           C  \\nATOM   1956  C   GLN A 261       9.163   4.983   3.002  1.00  0.50           C  \\nATOM   1957  CB  GLN A 261       8.189   3.263   1.465  1.00  0.50           C  \\nATOM   1958  O   GLN A 261       9.422   6.108   2.570  1.00  0.50           O  \\nATOM   1959  CG  GLN A 261       8.441   2.258   0.350  1.00  0.50           C  \\nATOM   1960  CD  GLN A 261       7.208   1.994  -0.494  1.00  0.52           C  \\nATOM   1961  NE2 GLN A 261       7.324   1.071  -1.442  1.00  0.44           N  \\nATOM   1962  OE1 GLN A 261       6.159   2.615  -0.293  1.00  0.52           O  \\nATOM   1963  N   ALA A 262       8.500   4.885   4.178  1.00  0.50           N  \\nATOM   1964  CA  ALA A 262       8.241   5.991   5.096  1.00  0.49           C  \\nATOM   1965  C   ALA A 262       9.530   6.729   5.443  1.00  0.49           C  \\nATOM   1966  CB  ALA A 262       7.565   5.480   6.366  1.00  0.47           C  \\nATOM   1967  O   ALA A 262       9.538   7.957   5.562  1.00  0.48           O  \\nATOM   1968  N   LYS A 263      10.656   5.986   5.623  1.00  0.50           N  \\nATOM   1969  CA  LYS A 263      11.964   6.580   5.884  1.00  0.49           C  \\nATOM   1970  C   LYS A 263      12.401   7.477   4.729  1.00  0.49           C  \\nATOM   1971  CB  LYS A 263      13.010   5.491   6.126  1.00  0.48           C  \\nATOM   1972  O   LYS A 263      13.072   8.489   4.942  1.00  0.48           O  \\nATOM   1973  CG  LYS A 263      12.892   4.810   7.481  1.00  0.46           C  \\nATOM   1974  CD  LYS A 263      13.882   3.660   7.617  1.00  0.47           C  \\nATOM   1975  CE  LYS A 263      13.673   2.891   8.914  1.00  0.45           C  \\nATOM   1976  NZ  LYS A 263      14.524   1.665   8.975  1.00  0.44           N  \\nATOM   1977  N   GLY A 264      11.947   7.151   3.526  1.00  0.41           N  \\nATOM   1978  CA  GLY A 264      12.369   7.852   2.324  1.00  0.40           C  \\nATOM   1979  C   GLY A 264      11.647   9.170   2.118  1.00  0.41           C  \\nATOM   1980  O   GLY A 264      11.951   9.911   1.181  1.00  0.39           O  \\nATOM   1981  N   GLY A 265      10.821   9.661   3.203  1.00  0.39           N  \\nATOM   1982  CA  GLY A 265      10.077  10.906   3.093  1.00  0.39           C  \\nATOM   1983  C   GLY A 265       9.096  10.914   1.935  1.00  0.39           C  \\nATOM   1984  O   GLY A 265       8.774  11.974   1.395  1.00  0.37           O  \\nATOM   1985  N   ILE A 266       8.719   9.764   1.305  1.00  0.37           N  \\nATOM   1986  CA  ILE A 266       7.868   9.669   0.124  1.00  0.36           C  \\nATOM   1987  C   ILE A 266       6.400   9.761   0.536  1.00  0.37           C  \\nATOM   1988  CB  ILE A 266       8.125   8.360  -0.655  1.00  0.34           C  \\nATOM   1989  O   ILE A 266       5.895   8.893   1.253  1.00  0.35           O  \\nATOM   1990  CG1 ILE A 266       9.599   8.263  -1.065  1.00  0.33           C  \\nATOM   1991  CG2 ILE A 266       7.209   8.271  -1.879  1.00  0.33           C  \\nATOM   1992  CD1 ILE A 266       9.969   6.950  -1.741  1.00  0.30           C  \\nATOM   1993  N   LYS A 267       5.891  10.978   0.877  1.00  0.37           N  \\nATOM   1994  CA  LYS A 267       4.511  11.367   1.158  1.00  0.37           C  \\nATOM   1995  C   LYS A 267       3.528  10.536   0.338  1.00  0.36           C  \\nATOM   1996  CB  LYS A 267       4.304  12.855   0.873  1.00  0.34           C  \\nATOM   1997  O   LYS A 267       2.442  10.202   0.815  1.00  0.34           O  \\nATOM   1998  CG  LYS A 267       4.899  13.776   1.928  1.00  0.32           C  \\nATOM   1999  CD  LYS A 267       4.567  15.236   1.647  1.00  0.34           C  \\nATOM   2000  CE  LYS A 267       5.178  16.160   2.692  1.00  0.31           C  \\nATOM   2001  NZ  LYS A 267       4.852  17.593   2.423  1.00  0.32           N  \\nATOM   2002  N   GLU A 268       3.886  10.190  -0.927  1.00  0.33           N  \\nATOM   2003  CA  GLU A 268       2.954   9.816  -1.986  1.00  0.34           C  \\nATOM   2004  C   GLU A 268       2.730   8.307  -2.017  1.00  0.33           C  \\nATOM   2005  CB  GLU A 268       3.464  10.300  -3.347  1.00  0.31           C  \\nATOM   2006  O   GLU A 268       1.645   7.842  -2.374  1.00  0.31           O  \\nATOM   2007  CG  GLU A 268       3.385  11.808  -3.532  1.00  0.29           C  \\nATOM   2008  CD  GLU A 268       4.050  12.291  -4.811  1.00  0.34           C  \\nATOM   2009  OE1 GLU A 268       4.594  11.453  -5.566  1.00  0.37           O  \\nATOM   2010  OE2 GLU A 268       4.027  13.517  -5.060  1.00  0.37           O  \\nATOM   2011  N   GLU A 269       3.595   7.468  -1.363  1.00  0.33           N  \\nATOM   2012  CA  GLU A 269       3.516   6.017  -1.224  1.00  0.34           C  \\nATOM   2013  C   GLU A 269       2.674   5.623  -0.014  1.00  0.33           C  \\nATOM   2014  CB  GLU A 269       4.917   5.410  -1.111  1.00  0.32           C  \\nATOM   2015  O   GLU A 269       2.130   4.518   0.036  1.00  0.32           O  \\nATOM   2016  CG  GLU A 269       5.604   5.194  -2.452  1.00  0.30           C  \\nATOM   2017  CD  GLU A 269       7.018   4.650  -2.320  1.00  0.35           C  \\nATOM   2018  OE1 GLU A 269       7.467   4.395  -1.180  1.00  0.40           O  \\nATOM   2019  OE2 GLU A 269       7.682   4.479  -3.367  1.00  0.37           O  \\nATOM   2020  N   ALA A 270       2.270   6.624   0.815  1.00  0.35           N  \\nATOM   2021  CA  ALA A 270       1.403   6.364   1.962  1.00  0.34           C  \\nATOM   2022  C   ALA A 270       0.042   5.840   1.513  1.00  0.35           C  \\nATOM   2023  CB  ALA A 270       1.233   7.629   2.799  1.00  0.33           C  \\nATOM   2024  O   ALA A 270      -0.526   4.946   2.145  1.00  0.33           O  \\nATOM   2025  N   ALA A 271      -0.509   6.262   0.258  1.00  0.35           N  \\nATOM   2026  CA  ALA A 271      -1.797   5.766  -0.220  1.00  0.34           C  \\nATOM   2027  C   ALA A 271      -1.694   4.309  -0.660  1.00  0.35           C  \\nATOM   2028  CB  ALA A 271      -2.306   6.631  -1.371  1.00  0.33           C  \\nATOM   2029  O   ALA A 271      -2.582   3.502  -0.372  1.00  0.33           O  \\nATOM   2030  N   ALA A 272      -0.593   3.910  -1.348  1.00  0.36           N  \\nATOM   2031  CA  ALA A 272      -0.424   2.514  -1.742  1.00  0.35           C  \\nATOM   2032  C   ALA A 272      -0.285   1.612  -0.519  1.00  0.36           C  \\nATOM   2033  CB  ALA A 272       0.791   2.361  -2.653  1.00  0.34           C  \\nATOM   2034  O   ALA A 272      -0.850   0.516  -0.481  1.00  0.34           O  \\nATOM   2035  N   ILE A 273       0.359   2.002   0.578  1.00  0.36           N  \\nATOM   2036  CA  ILE A 273       0.625   1.247   1.797  1.00  0.36           C  \\nATOM   2037  C   ILE A 273      -0.687   0.970   2.528  1.00  0.36           C  \\nATOM   2038  CB  ILE A 273       1.607   1.997   2.724  1.00  0.34           C  \\nATOM   2039  O   ILE A 273      -0.921  -0.148   2.994  1.00  0.35           O  \\nATOM   2040  CG1 ILE A 273       2.988   2.102   2.066  1.00  0.31           C  \\nATOM   2041  CG2 ILE A 273       1.701   1.303   4.086  1.00  0.32           C  \\nATOM   2042  CD1 ILE A 273       3.942   3.051   2.778  1.00  0.29           C  \\nATOM   2043  N   LYS A 274      -1.593   1.956   2.661  1.00  0.36           N  \\nATOM   2044  CA  LYS A 274      -2.878   1.721   3.313  1.00  0.36           C  \\nATOM   2045  C   LYS A 274      -3.666   0.628   2.596  1.00  0.36           C  \\nATOM   2046  CB  LYS A 274      -3.699   3.011   3.363  1.00  0.35           C  \\nATOM   2047  O   LYS A 274      -4.362  -0.164   3.235  1.00  0.35           O  \\nATOM   2048  CG  LYS A 274      -3.340   3.927   4.524  1.00  0.33           C  \\nATOM   2049  CD  LYS A 274      -4.211   5.177   4.539  1.00  0.37           C  \\nATOM   2050  CE  LYS A 274      -3.829   6.112   5.678  1.00  0.32           C  \\nATOM   2051  NZ  LYS A 274      -4.651   7.359   5.671  1.00  0.33           N  \\nATOM   2052  N   TRP A 275      -3.485   0.502   1.228  1.00  0.23           N  \\nATOM   2053  CA  TRP A 275      -4.186  -0.490   0.420  1.00  0.32           C  \\nATOM   2054  C   TRP A 275      -3.751  -1.903   0.794  1.00  0.24           C  \\nATOM   2055  CB  TRP A 275      -3.936  -0.244  -1.071  1.00  0.25           C  \\nATOM   2056  O   TRP A 275      -4.589  -2.787   0.987  1.00  0.24           O  \\nATOM   2057  CG  TRP A 275      -4.912   0.703  -1.703  1.00  0.25           C  \\nATOM   2058  CD1 TRP A 275      -4.708   2.022  -1.996  1.00  0.25           C  \\nATOM   2059  CD2 TRP A 275      -6.249   0.402  -2.115  1.00  0.25           C  \\nATOM   2060  CE2 TRP A 275      -6.798   1.587  -2.652  1.00  0.14           C  \\nATOM   2061  CE3 TRP A 275      -7.036  -0.757  -2.084  1.00  0.22           C  \\nATOM   2062  NE1 TRP A 275      -5.839   2.560  -2.568  1.00  0.22           N  \\nATOM   2063  CH2 TRP A 275      -8.849   0.499  -3.111  1.00  0.22           C  \\nATOM   2064  CZ2 TRP A 275      -8.100   1.647  -3.154  1.00  0.21           C  \\nATOM   2065  CZ3 TRP A 275      -8.332  -0.696  -2.584  1.00  0.22           C  \\nATOM   2066  N   GLU A 276      -2.485  -2.164   0.907  1.00  0.36           N  \\nATOM   2067  CA  GLU A 276      -1.924  -3.479   1.202  1.00  0.37           C  \\nATOM   2068  C   GLU A 276      -2.291  -3.932   2.612  1.00  0.36           C  \\nATOM   2069  CB  GLU A 276      -0.403  -3.465   1.033  1.00  0.35           C  \\nATOM   2070  O   GLU A 276      -2.698  -5.078   2.816  1.00  0.36           O  \\nATOM   2071  CG  GLU A 276       0.101  -4.383  -0.072  1.00  0.34           C  \\nATOM   2072  CD  GLU A 276       1.609  -4.322  -0.261  1.00  0.41           C  \\nATOM   2073  OE1 GLU A 276       2.280  -3.551   0.461  1.00  0.45           O  \\nATOM   2074  OE2 GLU A 276       2.122  -5.050  -1.140  1.00  0.42           O  \\nATOM   2075  N   ALA A 277      -2.302  -3.039   3.564  1.00  0.36           N  \\nATOM   2076  CA  ALA A 277      -2.669  -3.352   4.943  1.00  0.35           C  \\nATOM   2077  C   ALA A 277      -4.136  -3.760   5.041  1.00  0.36           C  \\nATOM   2078  CB  ALA A 277      -2.390  -2.158   5.852  1.00  0.34           C  \\nATOM   2079  O   ALA A 277      -4.478  -4.701   5.761  1.00  0.34           O  \\nATOM   2080  N   ALA A 278      -5.145  -3.171   4.260  1.00  0.35           N  \\nATOM   2081  CA  ALA A 278      -6.554  -3.558   4.248  1.00  0.35           C  \\nATOM   2082  C   ALA A 278      -6.734  -4.952   3.653  1.00  0.35           C  \\nATOM   2083  CB  ALA A 278      -7.379  -2.539   3.468  1.00  0.34           C  \\nATOM   2084  O   ALA A 278      -7.468  -5.777   4.201  1.00  0.34           O  \\nATOM   2085  N   ALA A 279      -6.091  -5.301   2.547  1.00  0.37           N  \\nATOM   2086  CA  ALA A 279      -6.197  -6.631   1.953  1.00  0.37           C  \\nATOM   2087  C   ALA A 279      -5.635  -7.696   2.889  1.00  0.37           C  \\nATOM   2088  CB  ALA A 279      -5.474  -6.673   0.609  1.00  0.35           C  \\nATOM   2089  O   ALA A 279      -6.211  -8.778   3.027  1.00  0.35           O  \\nATOM   2090  N   ILE A 280      -4.592  -7.430   3.616  1.00  0.37           N  \\nATOM   2091  CA  ILE A 280      -3.914  -8.343   4.531  1.00  0.37           C  \\nATOM   2092  C   ILE A 280      -4.759  -8.536   5.788  1.00  0.37           C  \\nATOM   2093  CB  ILE A 280      -2.507  -7.825   4.905  1.00  0.35           C  \\nATOM   2094  O   ILE A 280      -4.939  -9.662   6.256  1.00  0.36           O  \\nATOM   2095  CG1 ILE A 280      -1.586  -7.849   3.680  1.00  0.33           C  \\nATOM   2096  CG2 ILE A 280      -1.915  -8.652   6.051  1.00  0.32           C  \\nATOM   2097  CD1 ILE A 280      -0.263  -7.125   3.886  1.00  0.30           C  \\nATOM   2098  N   LYS A 281      -5.316  -7.510   6.385  1.00  0.37           N  \\nATOM   2099  CA  LYS A 281      -6.065  -7.561   7.637  1.00  0.37           C  \\nATOM   2100  C   LYS A 281      -7.332  -8.399   7.486  1.00  0.37           C  \\nATOM   2101  CB  LYS A 281      -6.423  -6.150   8.106  1.00  0.35           C  \\nATOM   2102  O   LYS A 281      -7.743  -9.085   8.424  1.00  0.36           O  \\nATOM   2103  CG  LYS A 281      -5.413  -5.541   9.067  1.00  0.34           C  \\nATOM   2104  CD  LYS A 281      -5.823  -4.139   9.496  1.00  0.40           C  \\nATOM   2105  CE  LYS A 281      -4.790  -3.511  10.421  1.00  0.33           C  \\nATOM   2106  NZ  LYS A 281      -5.171  -2.122  10.817  1.00  0.34           N  \\nATOM   2107  N   GLU A 282      -7.984  -8.450   6.300  1.00  0.34           N  \\nATOM   2108  CA  GLU A 282      -9.237  -9.182   6.147  1.00  0.34           C  \\nATOM   2109  C   GLU A 282      -8.986 -10.677   5.973  1.00  0.34           C  \\nATOM   2110  CB  GLU A 282     -10.035  -8.640   4.958  1.00  0.32           C  \\nATOM   2111  O   GLU A 282      -9.915 -11.483   6.064  1.00  0.32           O  \\nATOM   2112  CG  GLU A 282     -10.798  -7.359   5.262  1.00  0.30           C  \\nATOM   2113  CD  GLU A 282     -11.663  -6.887   4.103  1.00  0.36           C  \\nATOM   2114  OE1 GLU A 282     -11.705  -7.574   3.058  1.00  0.41           O  \\nATOM   2115  OE2 GLU A 282     -12.302  -5.821   4.242  1.00  0.39           O  \\nATOM   2116  N   GLY A 283      -7.815 -11.207   6.380  1.00  0.35           N  \\nATOM   2117  CA  GLY A 283      -7.767 -12.601   5.970  1.00  0.35           C  \\nATOM   2118  C   GLY A 283      -8.359 -12.840   4.593  1.00  0.35           C  \\nATOM   2119  O   GLY A 283      -9.054 -13.834   4.374  1.00  0.33           O  \\nATOM   2120  N   ALA A 284      -8.797 -11.753   3.935  1.00  0.31           N  \\nATOM   2121  CA  ALA A 284      -9.405 -12.117   2.658  1.00  0.32           C  \\nATOM   2122  C   ALA A 284      -8.460 -12.981   1.827  1.00  0.31           C  \\nATOM   2123  CB  ALA A 284      -9.798 -10.864   1.880  1.00  0.29           C  \\nATOM   2124  O   ALA A 284      -7.349 -12.558   1.500  1.00  0.30           O  \\nATOM   2125  N   SER A 285      -8.155 -14.269   2.072  1.00  0.32           N  \\nATOM   2126  CA  SER A 285      -8.170 -15.249   0.991  1.00  0.32           C  \\nATOM   2127  C   SER A 285      -9.094 -14.811  -0.140  1.00  0.32           C  \\nATOM   2128  CB  SER A 285      -8.606 -16.619   1.514  1.00  0.30           C  \\nATOM   2129  O   SER A 285     -10.268 -14.515   0.090  1.00  0.30           O  \\nATOM   2130  OG  SER A 285      -7.605 -17.592   1.269  1.00  0.29           O  \\nATOM   2131  N   PRO A 286      -8.737 -13.974  -1.087  1.00  0.34           N  \\nATOM   2132  CA  PRO A 286      -9.583 -13.663  -2.241  1.00  0.34           C  \\nATOM   2133  C   PRO A 286     -10.621 -14.747  -2.522  1.00  0.34           C  \\nATOM   2134  CB  PRO A 286      -8.582 -13.554  -3.394  1.00  0.32           C  \\nATOM   2135  O   PRO A 286     -10.327 -15.937  -2.388  1.00  0.33           O  \\nATOM   2136  CG  PRO A 286      -7.327 -14.175  -2.870  1.00  0.31           C  \\nATOM   2137  CD  PRO A 286      -7.458 -14.332  -1.382  1.00  0.34           C  \\nATOM   2138  N   GLU A 287     -11.975 -14.555  -2.241  1.00  0.32           N  \\nATOM   2139  CA  GLU A 287     -13.040 -15.429  -2.724  1.00  0.32           C  \\nATOM   2140  C   GLU A 287     -12.609 -16.180  -3.980  1.00  0.32           C  \\nATOM   2141  CB  GLU A 287     -14.312 -14.624  -3.003  1.00  0.31           C  \\nATOM   2142  O   GLU A 287     -12.096 -15.577  -4.926  1.00  0.31           O  \\nATOM   2143  CG  GLU A 287     -15.559 -15.481  -3.164  1.00  0.30           C  \\nATOM   2144  CD  GLU A 287     -16.841 -14.667  -3.230  1.00  0.36           C  \\nATOM   2145  OE1 GLU A 287     -16.769 -13.418  -3.207  1.00  0.48           O  \\nATOM   2146  OE2 GLU A 287     -17.927 -15.285  -3.305  1.00  0.43           O  \\nATOM   2147  N   PRO A 288     -12.133 -17.503  -3.830  1.00  0.32           N  \\nATOM   2148  CA  PRO A 288     -11.880 -18.303  -5.031  1.00  0.31           C  \\nATOM   2149  C   PRO A 288     -12.780 -17.911  -6.201  1.00  0.32           C  \\nATOM   2150  CB  PRO A 288     -12.175 -19.733  -4.571  1.00  0.29           C  \\nATOM   2151  O   PRO A 288     -14.007 -17.925  -6.073  1.00  0.30           O  \\nATOM   2152  CG  PRO A 288     -12.973 -19.574  -3.317  1.00  0.28           C  \\nATOM   2153  CD  PRO A 288     -12.822 -18.161  -2.831  1.00  0.31           C  \\nATOM   2154  N   GLN A 289     -12.471 -17.017  -7.129  1.00  0.31           N  \\nATOM   2155  CA  GLN A 289     -13.163 -17.018  -8.413  1.00  0.34           C  \\nATOM   2156  C   GLN A 289     -13.317 -18.437  -8.954  1.00  0.31           C  \\nATOM   2157  CB  GLN A 289     -12.418 -16.148  -9.426  1.00  0.31           C  \\nATOM   2158  O   GLN A 289     -12.384 -19.240  -8.878  1.00  0.31           O  \\nATOM   2159  CG  GLN A 289     -12.643 -14.654  -9.237  1.00  0.31           C  \\nATOM   2160  CD  GLN A 289     -12.607 -13.884 -10.544  1.00  0.38           C  \\nATOM   2161  NE2 GLN A 289     -13.118 -12.658 -10.524  1.00  0.36           N  \\nATOM   2162  OE1 GLN A 289     -12.124 -14.388 -11.563  1.00  0.47           O  \\nATOM   2163  N   PRO A 290     -14.674 -18.995  -8.990  1.00  0.33           N  \\nATOM   2164  CA  PRO A 290     -14.937 -20.262  -9.677  1.00  0.32           C  \\nATOM   2165  C   PRO A 290     -14.093 -20.436 -10.937  1.00  0.33           C  \\nATOM   2166  CB  PRO A 290     -16.425 -20.173 -10.024  1.00  0.31           C  \\nATOM   2167  O   PRO A 290     -13.942 -19.494 -11.719  1.00  0.32           O  \\nATOM   2168  CG  PRO A 290     -16.766 -18.725  -9.875  1.00  0.30           C  \\nATOM   2169  CD  PRO A 290     -15.683 -18.060  -9.076  1.00  0.34           C  \\nATOM   2170  N   LYS A 291     -12.934 -21.167 -10.918  1.00  0.34           N  \\nATOM   2171  CA  LYS A 291     -12.446 -21.714 -12.181  1.00  0.33           C  \\nATOM   2172  C   LYS A 291     -13.538 -21.693 -13.247  1.00  0.34           C  \\nATOM   2173  CB  LYS A 291     -11.932 -23.141 -11.986  1.00  0.31           C  \\nATOM   2174  O   LYS A 291     -14.665 -22.126 -12.998  1.00  0.32           O  \\nATOM   2175  CG  LYS A 291     -10.465 -23.325 -12.347  1.00  0.30           C  \\nATOM   2176  CD  LYS A 291     -10.014 -24.765 -12.138  1.00  0.34           C  \\nATOM   2177  CE  LYS A 291      -8.557 -24.959 -12.533  1.00  0.32           C  \\nATOM   2178  NZ  LYS A 291      -8.129 -26.383 -12.393  1.00  0.34           N  \\nATOM   2179  N   PRO A 292     -13.577 -20.719 -14.282  1.00  0.32           N  \\nATOM   2180  CA  PRO A 292     -14.453 -21.048 -15.409  1.00  0.32           C  \\nATOM   2181  C   PRO A 292     -14.463 -22.540 -15.733  1.00  0.32           C  \\nATOM   2182  CB  PRO A 292     -13.856 -20.242 -16.566  1.00  0.30           C  \\nATOM   2183  O   PRO A 292     -13.410 -23.125 -15.998  1.00  0.30           O  \\nATOM   2184  CG  PRO A 292     -12.510 -19.812 -16.078  1.00  0.30           C  \\nATOM   2185  CD  PRO A 292     -12.430 -20.080 -14.602  1.00  0.35           C  \\nATOM   2186  N   SER A 293     -15.106 -23.393 -14.913  1.00  0.32           N  \\nATOM   2187  CA  SER A 293     -15.565 -24.662 -15.469  1.00  0.34           C  \\nATOM   2188  C   SER A 293     -16.170 -24.470 -16.856  1.00  0.32           C  \\nATOM   2189  CB  SER A 293     -16.591 -25.314 -14.542  1.00  0.30           C  \\nATOM   2190  O   SER A 293     -17.263 -23.916 -16.991  1.00  0.31           O  \\nATOM   2191  OG  SER A 293     -16.055 -25.490 -13.242  1.00  0.35           O  \\nATOM   2192  N   GLY A 294     -15.518 -23.667 -17.700  1.00  0.32           N  \\nATOM   2193  CA  GLY A 294     -15.918 -23.711 -19.098  1.00  0.34           C  \\nATOM   2194  C   GLY A 294     -14.749 -23.903 -20.046  1.00  0.32           C  \\nATOM   2195  O   GLY A 294     -13.679 -23.326 -19.845  1.00  0.31           O  \\nATOM   2196  N   ASP A 295     -14.552 -25.176 -20.482  1.00  0.37           N  \\nATOM   2197  CA  ASP A 295     -13.844 -25.609 -21.683  1.00  0.37           C  \\nATOM   2198  C   ASP A 295     -13.287 -24.414 -22.453  1.00  0.37           C  \\nATOM   2199  CB  ASP A 295     -14.768 -26.431 -22.584  1.00  0.35           C  \\nATOM   2200  O   ASP A 295     -14.011 -23.457 -22.733  1.00  0.36           O  \\nATOM   2201  CG  ASP A 295     -14.505 -27.924 -22.499  1.00  0.35           C  \\nATOM   2202  OD1 ASP A 295     -13.341 -28.327 -22.288  1.00  0.39           O  \\nATOM   2203  OD2 ASP A 295     -15.470 -28.705 -22.648  1.00  0.45           O  \\nATOM   2204  N   PRO A 296     -12.043 -23.944 -22.275  1.00  0.39           N  \\nATOM   2205  CA  PRO A 296     -11.355 -22.905 -23.044  1.00  0.39           C  \\nATOM   2206  C   PRO A 296     -11.921 -22.741 -24.453  1.00  0.39           C  \\nATOM   2207  CB  PRO A 296      -9.908 -23.402 -23.092  1.00  0.35           C  \\nATOM   2208  O   PRO A 296     -11.517 -21.832 -25.183  1.00  0.37           O  \\nATOM   2209  CG  PRO A 296      -9.989 -24.849 -22.727  1.00  0.35           C  \\nATOM   2210  CD  PRO A 296     -11.334 -25.112 -22.114  1.00  0.41           C  \\nATOM   2211  N   GLN A 297     -12.892 -23.529 -24.849  1.00  0.32           N  \\nATOM   2212  CA  GLN A 297     -13.440 -23.434 -26.197  1.00  0.37           C  \\nATOM   2213  C   GLN A 297     -14.680 -22.544 -26.225  1.00  0.32           C  \\nATOM   2214  CB  GLN A 297     -13.779 -24.823 -26.739  1.00  0.30           C  \\nATOM   2215  O   GLN A 297     -15.126 -22.125 -27.295  1.00  0.31           O  \\nATOM   2216  CG  GLN A 297     -12.572 -25.591 -27.261  1.00  0.31           C  \\nATOM   2217  CD  GLN A 297     -12.951 -26.905 -27.918  1.00  0.38           C  \\nATOM   2218  NE2 GLN A 297     -11.949 -27.706 -28.264  1.00  0.33           N  \\nATOM   2219  OE1 GLN A 297     -14.135 -27.198 -28.113  1.00  0.47           O  \\nATOM   2220  N   SER A 298     -14.865 -21.641 -25.203  1.00  0.36           N  \\nATOM   2221  CA  SER A 298     -15.868 -20.635 -25.539  1.00  0.39           C  \\nATOM   2222  C   SER A 298     -15.315 -19.225 -25.366  1.00  0.36           C  \\nATOM   2223  CB  SER A 298     -17.116 -20.812 -24.674  1.00  0.34           C  \\nATOM   2224  O   SER A 298     -15.056 -18.787 -24.242  1.00  0.35           O  \\nATOM   2225  OG  SER A 298     -17.534 -22.167 -24.665  1.00  0.37           O  \\nATOM   2226  N   LYS A 299     -14.247 -18.931 -26.085  1.00  0.41           N  \\nATOM   2227  CA  LYS A 299     -13.812 -17.666 -26.670  1.00  0.45           C  \\nATOM   2228  C   LYS A 299     -15.008 -16.812 -27.083  1.00  0.41           C  \\nATOM   2229  CB  LYS A 299     -12.905 -17.915 -27.876  1.00  0.37           C  \\nATOM   2230  O   LYS A 299     -15.580 -17.013 -28.157  1.00  0.39           O  \\nATOM   2231  CG  LYS A 299     -11.441 -18.119 -27.517  1.00  0.37           C  \\nATOM   2232  CD  LYS A 299     -10.575 -18.268 -28.761  1.00  0.44           C  \\nATOM   2233  CE  LYS A 299      -9.115 -18.507 -28.403  1.00  0.35           C  \\nATOM   2234  NZ  LYS A 299      -8.265 -18.673 -29.620  1.00  0.35           N  \\nATOM   2235  N   GLN A 300     -15.887 -16.530 -26.224  1.00  0.41           N  \\nATOM   2236  CA  GLN A 300     -16.747 -15.370 -26.432  1.00  0.44           C  \\nATOM   2237  C   GLN A 300     -16.688 -14.420 -25.239  1.00  0.41           C  \\nATOM   2238  CB  GLN A 300     -18.190 -15.809 -26.682  1.00  0.38           C  \\nATOM   2239  O   GLN A 300     -17.030 -14.802 -24.118  1.00  0.39           O  \\nATOM   2240  CG  GLN A 300     -18.419 -16.414 -28.061  1.00  0.38           C  \\nATOM   2241  CD  GLN A 300     -19.856 -16.850 -28.280  1.00  0.43           C  \\nATOM   2242  NE2 GLN A 300     -20.084 -17.636 -29.326  1.00  0.37           N  \\nATOM   2243  OE1 GLN A 300     -20.753 -16.484 -27.514  1.00  0.50           O  \\nATOM   2244  N   THR A 301     -15.756 -13.447 -25.185  1.00  0.45           N  \\nATOM   2245  CA  THR A 301     -15.913 -12.160 -24.517  1.00  0.47           C  \\nATOM   2246  C   THR A 301     -17.295 -11.573 -24.791  1.00  0.45           C  \\nATOM   2247  CB  THR A 301     -14.830 -11.161 -24.966  1.00  0.42           C  \\nATOM   2248  O   THR A 301     -17.652 -11.325 -25.944  1.00  0.44           O  \\nATOM   2249  CG2 THR A 301     -13.934 -10.760 -23.799  1.00  0.39           C  \\nATOM   2250  OG1 THR A 301     -14.026 -11.765 -25.987  1.00  0.40           O  \\nATOM   2251  N   PRO A 302     -18.247 -11.805 -23.936  1.00  0.39           N  \\nATOM   2252  CA  PRO A 302     -19.393 -10.903 -24.081  1.00  0.39           C  \\nATOM   2253  C   PRO A 302     -19.038  -9.447 -23.790  1.00  0.39           C  \\nATOM   2254  CB  PRO A 302     -20.394 -11.437 -23.054  1.00  0.36           C  \\nATOM   2255  O   PRO A 302     -18.422  -9.151 -22.763  1.00  0.38           O  \\nATOM   2256  CG  PRO A 302     -19.633 -12.461 -22.275  1.00  0.35           C  \\nATOM   2257  CD  PRO A 302     -18.284 -12.638 -22.911  1.00  0.40           C  \\nATOM   2258  N   GLU A 303     -18.645  -8.570 -24.731  1.00  0.48           N  \\nATOM   2259  CA  GLU A 303     -19.060  -7.171 -24.749  1.00  0.48           C  \\nATOM   2260  C   GLU A 303     -20.164  -6.909 -23.728  1.00  0.48           C  \\nATOM   2261  CB  GLU A 303     -19.532  -6.767 -26.148  1.00  0.43           C  \\nATOM   2262  O   GLU A 303     -21.242  -7.502 -23.806  1.00  0.45           O  \\nATOM   2263  CG  GLU A 303     -19.670  -5.264 -26.340  1.00  0.42           C  \\nATOM   2264  CD  GLU A 303     -19.794  -4.854 -27.799  1.00  0.46           C  \\nATOM   2265  OE1 GLU A 303     -19.898  -5.745 -28.673  1.00  0.53           O  \\nATOM   2266  OE2 GLU A 303     -19.787  -3.633 -28.071  1.00  0.48           O  \\nATOM   2267  N   ALA A 304     -19.792  -6.841 -22.405  1.00  0.46           N  \\nATOM   2268  CA  ALA A 304     -20.824  -6.316 -21.515  1.00  0.49           C  \\nATOM   2269  C   ALA A 304     -21.598  -5.182 -22.181  1.00  0.46           C  \\nATOM   2270  CB  ALA A 304     -20.203  -5.835 -20.206  1.00  0.43           C  \\nATOM   2271  O   ALA A 304     -21.138  -4.037 -22.203  1.00  0.45           O  \\nATOM   2272  N   SER A 305     -22.139  -5.260 -23.327  1.00  0.37           N  \\nATOM   2273  CA  SER A 305     -23.361  -4.531 -23.651  1.00  0.39           C  \\nATOM   2274  C   SER A 305     -24.428  -4.735 -22.581  1.00  0.37           C  \\nATOM   2275  CB  SER A 305     -23.902  -4.970 -25.013  1.00  0.35           C  \\nATOM   2276  O   SER A 305     -24.635  -5.856 -22.111  1.00  0.36           O  \\nATOM   2277  OG  SER A 305     -22.999  -4.623 -26.049  1.00  0.39           O  \\nATOM   2278  N   PRO A 306     -24.774  -3.783 -21.740  1.00  0.47           N  \\nATOM   2279  CA  PRO A 306     -26.117  -4.005 -21.202  1.00  0.46           C  \\nATOM   2280  C   PRO A 306     -27.020  -4.774 -22.165  1.00  0.47           C  \\nATOM   2281  CB  PRO A 306     -26.645  -2.586 -20.974  1.00  0.42           C  \\nATOM   2282  O   PRO A 306     -27.355  -4.268 -23.239  1.00  0.44           O  \\nATOM   2283  CG  PRO A 306     -25.751  -1.710 -21.790  1.00  0.41           C  \\nATOM   2284  CD  PRO A 306     -24.517  -2.489 -22.148  1.00  0.46           C  \\nATOM   2285  N   ARG A 307     -26.792  -6.017 -22.453  1.00  0.42           N  \\nATOM   2286  CA  ARG A 307     -27.907  -6.694 -23.107  1.00  0.44           C  \\nATOM   2287  C   ARG A 307     -29.199  -6.508 -22.319  1.00  0.42           C  \\nATOM   2288  CB  ARG A 307     -27.606  -8.184 -23.278  1.00  0.39           C  \\nATOM   2289  O   ARG A 307     -29.223  -6.703 -21.102  1.00  0.40           O  \\nATOM   2290  CG  ARG A 307     -28.372  -8.841 -24.416  1.00  0.39           C  \\nATOM   2291  CD  ARG A 307     -27.964 -10.296 -24.604  1.00  0.45           C  \\nATOM   2292  NE  ARG A 307     -28.422 -10.822 -25.887  1.00  0.39           N  \\nATOM   2293  NH1 ARG A 307     -27.392 -12.875 -25.647  1.00  0.30           N  \\nATOM   2294  NH2 ARG A 307     -28.608 -12.412 -27.534  1.00  0.25           N  \\nATOM   2295  CZ  ARG A 307     -28.140 -12.035 -26.353  1.00  0.35           C  \\nATOM   2296  N   PRO A 308     -30.072  -5.581 -22.811  1.00  0.49           N  \\nATOM   2297  CA  PRO A 308     -31.479  -5.716 -22.424  1.00  0.47           C  \\nATOM   2298  C   PRO A 308     -31.821  -7.116 -21.920  1.00  0.49           C  \\nATOM   2299  CB  PRO A 308     -32.232  -5.402 -23.719  1.00  0.45           C  \\nATOM   2300  O   PRO A 308     -31.505  -8.108 -22.582  1.00  0.47           O  \\nATOM   2301  CG  PRO A 308     -31.182  -5.403 -24.782  1.00  0.44           C  \\nATOM   2302  CD  PRO A 308     -29.834  -5.448 -24.121  1.00  0.48           C  \\nATOM   2303  N   GLN A 309     -31.767  -7.399 -20.683  1.00  0.49           N  \\nATOM   2304  CA  GLN A 309     -32.556  -8.501 -20.143  1.00  0.51           C  \\nATOM   2305  C   GLN A 309     -33.669  -8.904 -21.107  1.00  0.49           C  \\nATOM   2306  CB  GLN A 309     -33.150  -8.122 -18.786  1.00  0.46           C  \\nATOM   2307  O   GLN A 309     -34.483  -8.070 -21.507  1.00  0.47           O  \\nATOM   2308  CG  GLN A 309     -33.774  -9.293 -18.039  1.00  0.46           C  \\nATOM   2309  CD  GLN A 309     -34.202  -8.929 -16.629  1.00  0.50           C  \\nATOM   2310  NE2 GLN A 309     -34.814  -9.879 -15.932  1.00  0.41           N  \\nATOM   2311  OE1 GLN A 309     -33.983  -7.803 -16.171  1.00  0.54           O  \\nATOM   2312  N   PRO A 310     -33.404  -9.877 -22.028  1.00  0.47           N  \\nATOM   2313  CA  PRO A 310     -34.521 -10.434 -22.794  1.00  0.47           C  \\nATOM   2314  C   PRO A 310     -35.751 -10.707 -21.930  1.00  0.47           C  \\nATOM   2315  CB  PRO A 310     -33.948 -11.738 -23.355  1.00  0.43           C  \\nATOM   2316  O   PRO A 310     -35.629 -11.259 -20.833  1.00  0.45           O  \\nATOM   2317  CG  PRO A 310     -32.717 -11.994 -22.547  1.00  0.42           C  \\nATOM   2318  CD  PRO A 310     -32.358 -10.733 -21.814  1.00  0.46           C  \\nATOM   2319  N   GLU A 311     -36.823  -9.859 -22.061  1.00  0.53           N  \\nATOM   2320  CA  GLU A 311     -38.236 -10.171 -21.869  1.00  0.54           C  \\nATOM   2321  C   GLU A 311     -38.499 -11.665 -22.033  1.00  0.52           C  \\nATOM   2322  CB  GLU A 311     -39.101  -9.374 -22.850  1.00  0.49           C  \\nATOM   2323  O   GLU A 311     -38.075 -12.271 -23.019  1.00  0.50           O  \\nATOM   2324  CG  GLU A 311     -39.319  -7.925 -22.441  1.00  0.48           C  \\nATOM   2325  CD  GLU A 311     -40.247  -7.170 -23.379  1.00  0.53           C  \\nATOM   2326  OE1 GLU A 311     -40.584  -7.705 -24.460  1.00  0.56           O  \\nATOM   2327  OE2 GLU A 311     -40.641  -6.034 -23.030  1.00  0.50           O  \\nATOM   2328  N   PRO A 312     -38.635 -12.417 -20.868  1.00  0.55           N  \\nATOM   2329  CA  PRO A 312     -39.170 -13.780 -20.876  1.00  0.56           C  \\nATOM   2330  C   PRO A 312     -40.202 -14.003 -21.979  1.00  0.55           C  \\nATOM   2331  CB  PRO A 312     -39.813 -13.914 -19.493  1.00  0.50           C  \\nATOM   2332  O   PRO A 312     -41.337 -13.530 -21.874  1.00  0.53           O  \\nATOM   2333  CG  PRO A 312     -40.106 -12.511 -19.070  1.00  0.49           C  \\nATOM   2334  CD  PRO A 312     -39.215 -11.583 -19.846  1.00  0.53           C  \\nATOM   2335  N   SER A 313     -39.784 -14.226 -23.182  1.00  0.49           N  \\nATOM   2336  CA  SER A 313     -40.699 -14.749 -24.192  1.00  0.52           C  \\nATOM   2337  C   SER A 313     -40.852 -16.262 -24.069  1.00  0.49           C  \\nATOM   2338  CB  SER A 313     -40.209 -14.391 -25.596  1.00  0.45           C  \\nATOM   2339  O   SER A 313     -39.858 -16.990 -24.029  1.00  0.46           O  \\nATOM   2340  OG  SER A 313     -40.067 -12.988 -25.736  1.00  0.49           O  \\nATOM   2341  N   GLY A 314     -41.940 -16.650 -23.461  1.00  0.50           N  \\nATOM   2342  CA  GLY A 314     -43.081 -17.551 -23.511  1.00  0.51           C  \\nATOM   2343  C   GLY A 314     -42.857 -18.747 -24.417  1.00  0.49           C  \\nATOM   2344  O   GLY A 314     -43.814 -19.354 -24.900  1.00  0.46           O  \\nATOM   2345  N   LYS A 315     -41.730 -19.418 -24.424  1.00  0.44           N  \\nATOM   2346  CA  LYS A 315     -41.830 -20.774 -24.957  1.00  0.47           C  \\nATOM   2347  C   LYS A 315     -40.988 -21.749 -24.139  1.00  0.44           C  \\nATOM   2348  CB  LYS A 315     -41.396 -20.809 -26.423  1.00  0.40           C  \\nATOM   2349  O   LYS A 315     -39.843 -21.450 -23.793  1.00  0.42           O  \\nATOM   2350  CG  LYS A 315     -42.427 -20.240 -27.387  1.00  0.40           C  \\nATOM   2351  CD  LYS A 315     -41.969 -20.368 -28.834  1.00  0.46           C  \\nATOM   2352  CE  LYS A 315     -43.000 -19.801 -29.800  1.00  0.38           C  \\nATOM   2353  NZ  LYS A 315     -42.529 -19.868 -31.216  1.00  0.38           N  \\nATOM   2354  N   PRO A 316     -41.494 -22.717 -23.332  1.00  0.53           N  \\nATOM   2355  CA  PRO A 316     -40.821 -23.932 -22.867  1.00  0.54           C  \\nATOM   2356  C   PRO A 316     -39.792 -24.456 -23.866  1.00  0.53           C  \\nATOM   2357  CB  PRO A 316     -41.970 -24.928 -22.693  1.00  0.48           C  \\nATOM   2358  O   PRO A 316     -40.158 -24.911 -24.953  1.00  0.50           O  \\nATOM   2359  CG  PRO A 316     -43.124 -24.309 -23.413  1.00  0.47           C  \\nATOM   2360  CD  PRO A 316     -42.776 -22.884 -23.737  1.00  0.51           C  \\nATOM   2361  N   GLN A 317     -38.580 -23.913 -23.883  1.00  0.46           N  \\nATOM   2362  CA  GLN A 317     -37.430 -24.548 -24.517  1.00  0.50           C  \\nATOM   2363  C   GLN A 317     -36.936 -25.736 -23.696  1.00  0.47           C  \\nATOM   2364  CB  GLN A 317     -36.298 -23.538 -24.712  1.00  0.42           C  \\nATOM   2365  O   GLN A 317     -36.657 -25.600 -22.504  1.00  0.45           O  \\nATOM   2366  CG  GLN A 317     -36.107 -23.099 -26.157  1.00  0.41           C  \\nATOM   2367  CD  GLN A 317     -35.067 -22.005 -26.306  1.00  0.47           C  \\nATOM   2368  NE2 GLN A 317     -34.930 -21.478 -27.518  1.00  0.39           N  \\nATOM   2369  OE1 GLN A 317     -34.392 -21.636 -25.339  1.00  0.50           O  \\nATOM   2370  N   SER A 318     -37.456 -26.914 -23.922  1.00  0.49           N  \\nATOM   2371  CA  SER A 318     -37.010 -28.303 -23.972  1.00  0.54           C  \\nATOM   2372  C   SER A 318     -35.489 -28.394 -24.027  1.00  0.48           C  \\nATOM   2373  CB  SER A 318     -37.616 -29.017 -25.181  1.00  0.44           C  \\nATOM   2374  O   SER A 318     -34.845 -27.663 -24.783  1.00  0.46           O  \\nATOM   2375  OG  SER A 318     -38.834 -28.405 -25.569  1.00  0.49           O  \\nATOM   2376  N   GLU A 319     -34.890 -28.573 -22.805  1.00  0.49           N  \\nATOM   2377  CA  GLU A 319     -33.659 -29.325 -22.579  1.00  0.55           C  \\nATOM   2378  C   GLU A 319     -32.469 -28.652 -23.256  1.00  0.49           C  \\nATOM   2379  CB  GLU A 319     -33.808 -30.763 -23.083  1.00  0.45           C  \\nATOM   2380  O   GLU A 319     -32.133 -28.975 -24.397  1.00  0.47           O  \\nATOM   2381  CG  GLU A 319     -34.278 -31.743 -22.018  1.00  0.45           C  \\nATOM   2382  CD  GLU A 319     -34.533 -33.141 -22.560  1.00  0.51           C  \\nATOM   2383  OE1 GLU A 319     -34.419 -33.345 -23.789  1.00  0.53           O  \\nATOM   2384  OE2 GLU A 319     -34.851 -34.038 -21.747  1.00  0.47           O  \\nATOM   2385  N   GLN A 320     -32.343 -27.359 -23.226  1.00  0.42           N  \\nATOM   2386  CA  GLN A 320     -31.009 -26.775 -23.306  1.00  0.47           C  \\nATOM   2387  C   GLN A 320     -30.431 -26.529 -21.916  1.00  0.42           C  \\nATOM   2388  CB  GLN A 320     -31.042 -25.467 -24.100  1.00  0.39           C  \\nATOM   2389  O   GLN A 320     -31.037 -25.832 -21.100  1.00  0.40           O  \\nATOM   2390  CG  GLN A 320     -31.191 -25.663 -25.602  1.00  0.40           C  \\nATOM   2391  CD  GLN A 320     -31.182 -24.354 -26.369  1.00  0.46           C  \\nATOM   2392  NE2 GLN A 320     -31.613 -24.399 -27.624  1.00  0.38           N  \\nATOM   2393  OE1 GLN A 320     -30.790 -23.310 -25.836  1.00  0.50           O  \\nATOM   2394  N   THR A 321     -29.734 -27.603 -21.288  1.00  0.47           N  \\nATOM   2395  CA  THR A 321     -28.730 -27.276 -20.282  1.00  0.49           C  \\nATOM   2396  C   THR A 321     -28.374 -25.793 -20.335  1.00  0.47           C  \\nATOM   2397  CB  THR A 321     -27.456 -28.120 -20.469  1.00  0.44           C  \\nATOM   2398  O   THR A 321     -27.998 -25.278 -21.390  1.00  0.45           O  \\nATOM   2399  CG2 THR A 321     -27.467 -29.343 -19.557  1.00  0.42           C  \\nATOM   2400  OG1 THR A 321     -27.373 -28.555 -21.832  1.00  0.41           O  \\nATOM   2401  N   PRO A 322     -29.097 -25.012 -19.663  1.00  0.41           N  \\nATOM   2402  CA  PRO A 322     -28.571 -23.646 -19.627  1.00  0.43           C  \\nATOM   2403  C   PRO A 322     -27.068 -23.585 -19.892  1.00  0.41           C  \\nATOM   2404  CB  PRO A 322     -28.894 -23.181 -18.205  1.00  0.38           C  \\nATOM   2405  O   PRO A 322     -26.294 -24.288 -19.238  1.00  0.40           O  \\nATOM   2406  CG  PRO A 322     -29.460 -24.387 -17.528  1.00  0.38           C  \\nATOM   2407  CD  PRO A 322     -29.530 -25.506 -18.527  1.00  0.43           C  \\nATOM   2408  N   LYS A 323     -26.447 -23.480 -21.160  1.00  0.42           N  \\nATOM   2409  CA  LYS A 323     -25.168 -22.792 -21.310  1.00  0.43           C  \\nATOM   2410  C   LYS A 323     -24.876 -21.906 -20.102  1.00  0.42           C  \\nATOM   2411  CB  LYS A 323     -25.156 -21.954 -22.589  1.00  0.40           C  \\nATOM   2412  O   LYS A 323     -25.672 -21.030 -19.761  1.00  0.40           O  \\nATOM   2413  CG  LYS A 323     -23.761 -21.658 -23.121  1.00  0.39           C  \\nATOM   2414  CD  LYS A 323     -23.817 -20.924 -24.455  1.00  0.44           C  \\nATOM   2415  CE  LYS A 323     -22.422 -20.648 -25.001  1.00  0.39           C  \\nATOM   2416  NZ  LYS A 323     -22.472 -19.977 -26.334  1.00  0.39           N  \\nATOM   2417  N   ALA A 324     -24.541 -22.490 -18.881  1.00  0.43           N  \\nATOM   2418  CA  ALA A 324     -24.068 -21.673 -17.767  1.00  0.46           C  \\nATOM   2419  C   ALA A 324     -23.929 -20.210 -18.180  1.00  0.43           C  \\nATOM   2420  CB  ALA A 324     -22.735 -22.204 -17.245  1.00  0.41           C  \\nATOM   2421  O   ALA A 324     -23.201 -19.891 -19.123  1.00  0.42           O  \\nATOM   2422  N   SER A 325     -24.972 -19.393 -18.417  1.00  0.34           N  \\nATOM   2423  CA  SER A 325     -24.737 -17.956 -18.320  1.00  0.36           C  \\nATOM   2424  C   SER A 325     -23.352 -17.660 -17.754  1.00  0.34           C  \\nATOM   2425  CB  SER A 325     -25.804 -17.295 -17.447  1.00  0.32           C  \\nATOM   2426  O   SER A 325     -22.878 -18.361 -16.858  1.00  0.33           O  \\nATOM   2427  OG  SER A 325     -27.080 -17.385 -18.058  1.00  0.32           O  \\nATOM   2428  N   PRO A 326     -22.327 -17.237 -18.554  1.00  0.38           N  \\nATOM   2429  CA  PRO A 326     -21.217 -16.828 -17.691  1.00  0.38           C  \\nATOM   2430  C   PRO A 326     -21.654 -16.562 -16.252  1.00  0.38           C  \\nATOM   2431  CB  PRO A 326     -20.710 -15.544 -18.353  1.00  0.35           C  \\nATOM   2432  O   PRO A 326     -22.691 -15.935 -16.024  1.00  0.36           O  \\nATOM   2433  CG  PRO A 326     -21.813 -15.126 -19.270  1.00  0.34           C  \\nATOM   2434  CD  PRO A 326     -22.736 -16.294 -19.470  1.00  0.38           C  \\nATOM   2435  N   GLU A 327     -21.643 -17.440 -15.299  1.00  0.35           N  \\nATOM   2436  CA  GLU A 327     -21.800 -17.036 -13.905  1.00  0.37           C  \\nATOM   2437  C   GLU A 327     -21.516 -15.547 -13.726  1.00  0.35           C  \\nATOM   2438  CB  GLU A 327     -20.881 -17.860 -12.999  1.00  0.33           C  \\nATOM   2439  O   GLU A 327     -20.530 -15.030 -14.255  1.00  0.34           O  \\nATOM   2440  CG  GLU A 327     -21.361 -17.953 -11.558  1.00  0.33           C  \\nATOM   2441  CD  GLU A 327     -20.533 -18.906 -10.711  1.00  0.39           C  \\nATOM   2442  OE1 GLU A 327     -19.570 -19.507 -11.239  1.00  0.48           O  \\nATOM   2443  OE2 GLU A 327     -20.850 -19.053  -9.509  1.00  0.44           O  \\nATOM   2444  N   PRO A 328     -22.570 -14.673 -13.748  1.00  0.31           N  \\nATOM   2445  CA  PRO A 328     -22.208 -13.287 -13.441  1.00  0.31           C  \\nATOM   2446  C   PRO A 328     -20.771 -13.148 -12.945  1.00  0.31           C  \\nATOM   2447  CB  PRO A 328     -23.204 -12.898 -12.345  1.00  0.28           C  \\nATOM   2448  O   PRO A 328     -20.342 -13.900 -12.066  1.00  0.29           O  \\nATOM   2449  CG  PRO A 328     -23.698 -14.198 -11.799  1.00  0.28           C  \\nATOM   2450  CD  PRO A 328     -23.414 -15.277 -12.804  1.00  0.32           C  \\nATOM   2451  N   GLN A 329     -19.701 -12.867 -13.708  1.00  0.35           N  \\nATOM   2452  CA  GLN A 329     -18.564 -12.218 -13.063  1.00  0.36           C  \\nATOM   2453  C   GLN A 329     -18.953 -11.655 -11.699  1.00  0.35           C  \\nATOM   2454  CB  GLN A 329     -18.006 -11.105 -13.951  1.00  0.33           C  \\nATOM   2455  O   GLN A 329     -20.034 -11.083 -11.544  1.00  0.34           O  \\nATOM   2456  CG  GLN A 329     -17.151 -11.610 -15.106  1.00  0.33           C  \\nATOM   2457  CD  GLN A 329     -16.052 -10.637 -15.491  1.00  0.40           C  \\nATOM   2458  NE2 GLN A 329     -15.047 -11.130 -16.207  1.00  0.36           N  \\nATOM   2459  OE1 GLN A 329     -16.106  -9.451 -15.149  1.00  0.48           O  \\nATOM   2460  N   PRO A 330     -18.766 -12.257 -10.564  1.00  0.35           N  \\nATOM   2461  CA  PRO A 330     -19.154 -11.412  -9.432  1.00  0.34           C  \\nATOM   2462  C   PRO A 330     -19.301  -9.942  -9.817  1.00  0.35           C  \\nATOM   2463  CB  PRO A 330     -18.005 -11.604  -8.439  1.00  0.32           C  \\nATOM   2464  O   PRO A 330     -18.494  -9.418 -10.590  1.00  0.33           O  \\nATOM   2465  CG  PRO A 330     -16.904 -12.218  -9.243  1.00  0.31           C  \\nATOM   2466  CD  PRO A 330     -17.490 -12.772 -10.509  1.00  0.34           C  \\nATOM   2467  N   LYS A 331     -20.471  -9.328 -10.194  1.00  0.30           N  \\nATOM   2468  CA  LYS A 331     -20.567  -7.874 -10.101  1.00  0.30           C  \\nATOM   2469  C   LYS A 331     -19.323  -7.281  -9.445  1.00  0.30           C  \\nATOM   2470  CB  LYS A 331     -21.816  -7.466  -9.319  1.00  0.28           C  \\nATOM   2471  O   LYS A 331     -18.882  -7.756  -8.397  1.00  0.28           O  \\nATOM   2472  CG  LYS A 331     -23.052  -7.273 -10.185  1.00  0.28           C  \\nATOM   2473  CD  LYS A 331     -24.246  -6.811  -9.359  1.00  0.31           C  \\nATOM   2474  CE  LYS A 331     -25.492  -6.646 -10.219  1.00  0.30           C  \\nATOM   2475  NZ  LYS A 331     -26.675  -6.234  -9.406  1.00  0.32           N  \\nATOM   2476  N   PRO A 332     -18.143  -6.967 -10.064  1.00  0.31           N  \\nATOM   2477  CA  PRO A 332     -17.088  -6.277  -9.318  1.00  0.31           C  \\nATOM   2478  C   PRO A 332     -17.548  -5.810  -7.939  1.00  0.32           C  \\nATOM   2479  CB  PRO A 332     -16.747  -5.083 -10.214  1.00  0.29           C  \\nATOM   2480  O   PRO A 332     -18.657  -5.289  -7.798  1.00  0.30           O  \\nATOM   2481  CG  PRO A 332     -17.687  -5.192 -11.370  1.00  0.28           C  \\nATOM   2482  CD  PRO A 332     -18.594  -6.367 -11.139  1.00  0.33           C  \\nATOM   2483  N   SER A 333     -17.497  -6.582  -6.856  1.00  0.34           N  \\nATOM   2484  CA  SER A 333     -17.707  -5.953  -5.557  1.00  0.34           C  \\nATOM   2485  C   SER A 333     -17.605  -4.434  -5.655  1.00  0.34           C  \\nATOM   2486  CB  SER A 333     -16.694  -6.475  -4.538  1.00  0.32           C  \\nATOM   2487  O   SER A 333     -16.994  -3.907  -6.587  1.00  0.33           O  \\nATOM   2488  OG  SER A 333     -15.368  -6.220  -4.970  1.00  0.32           O  \\nATOM   2489  N   GLY A 334     -18.757  -3.634  -5.674  1.00  0.32           N  \\nATOM   2490  CA  GLY A 334     -18.620  -2.189  -5.584  1.00  0.32           C  \\nATOM   2491  C   GLY A 334     -17.190  -1.714  -5.763  1.00  0.32           C  \\nATOM   2492  O   GLY A 334     -16.950  -0.669  -6.372  1.00  0.30           O  \\nATOM   2493  N   GLN A 335     -16.238  -2.532  -5.345  1.00  0.33           N  \\nATOM   2494  CA  GLN A 335     -14.812  -2.243  -5.452  1.00  0.35           C  \\nATOM   2495  C   GLN A 335     -14.373  -2.171  -6.911  1.00  0.33           C  \\nATOM   2496  CB  GLN A 335     -13.992  -3.299  -4.708  1.00  0.31           C  \\nATOM   2497  O   GLN A 335     -13.648  -1.254  -7.303  1.00  0.32           O  \\nATOM   2498  CG  GLN A 335     -13.860  -3.036  -3.214  1.00  0.29           C  \\nATOM   2499  CD  GLN A 335     -13.068  -4.113  -2.496  1.00  0.35           C  \\nATOM   2500  NE2 GLN A 335     -12.958  -3.988  -1.178  1.00  0.34           N  \\nATOM   2501  OE1 GLN A 335     -12.561  -5.050  -3.121  1.00  0.41           O  \\nATOM   2502  N   ALA A 336     -14.747  -3.037  -7.785  1.00  0.34           N  \\nATOM   2503  CA  ALA A 336     -14.394  -3.011  -9.202  1.00  0.33           C  \\nATOM   2504  C   ALA A 336     -15.085  -1.854  -9.918  1.00  0.34           C  \\nATOM   2505  CB  ALA A 336     -14.759  -4.337  -9.866  1.00  0.32           C  \\nATOM   2506  O   ALA A 336     -14.486  -1.199 -10.774  1.00  0.32           O  \\nATOM   2507  N   LYS A 337     -16.402  -1.554  -9.517  1.00  0.33           N  \\nATOM   2508  CA  LYS A 337     -17.040  -0.378 -10.100  1.00  0.33           C  \\nATOM   2509  C   LYS A 337     -16.324   0.902  -9.680  1.00  0.33           C  \\nATOM   2510  CB  LYS A 337     -18.514  -0.311  -9.696  1.00  0.31           C  \\nATOM   2511  O   LYS A 337     -16.104   1.795 -10.501  1.00  0.31           O  \\nATOM   2512  CG  LYS A 337     -19.426  -1.199 -10.529  1.00  0.30           C  \\nATOM   2513  CD  LYS A 337     -20.887  -1.025 -10.136  1.00  0.33           C  \\nATOM   2514  CE  LYS A 337     -21.801  -1.915 -10.968  1.00  0.29           C  \\nATOM   2515  NZ  LYS A 337     -23.233  -1.761 -10.574  1.00  0.32           N  \\nATOM   2516  N   GLU A 338     -16.007   0.977  -8.393  1.00  0.35           N  \\nATOM   2517  CA  GLU A 338     -15.323   2.148  -7.852  1.00  0.36           C  \\nATOM   2518  C   GLU A 338     -13.927   2.298  -8.450  1.00  0.35           C  \\nATOM   2519  CB  GLU A 338     -15.236   2.062  -6.326  1.00  0.33           C  \\nATOM   2520  O   GLU A 338     -13.530   3.396  -8.845  1.00  0.34           O  \\nATOM   2521  CG  GLU A 338     -15.570   3.366  -5.616  1.00  0.32           C  \\nATOM   2522  CD  GLU A 338     -15.830   3.190  -4.129  1.00  0.38           C  \\nATOM   2523  OE1 GLU A 338     -15.678   2.058  -3.616  1.00  0.44           O  \\nATOM   2524  OE2 GLU A 338     -16.189   4.192  -3.471  1.00  0.40           O  \\nATOM   2525  N   ILE A 339     -13.232   1.236  -8.662  1.00  0.38           N  \\nATOM   2526  CA  ILE A 339     -11.927   1.204  -9.313  1.00  0.38           C  \\nATOM   2527  C   ILE A 339     -12.071   1.611 -10.778  1.00  0.38           C  \\nATOM   2528  CB  ILE A 339     -11.277  -0.194  -9.207  1.00  0.37           C  \\nATOM   2529  O   ILE A 339     -11.274   2.400 -11.291  1.00  0.37           O  \\nATOM   2530  CG1 ILE A 339     -10.911  -0.504  -7.751  1.00  0.34           C  \\nATOM   2531  CG2 ILE A 339     -10.045  -0.286 -10.113  1.00  0.34           C  \\nATOM   2532  CD1 ILE A 339     -10.491  -1.948  -7.512  1.00  0.33           C  \\nATOM   2533  N   GLN A 340     -13.094   1.043 -11.510  1.00  0.38           N  \\nATOM   2534  CA  GLN A 340     -13.383   1.381 -12.900  1.00  0.39           C  \\nATOM   2535  C   GLN A 340     -13.741   2.857 -13.044  1.00  0.38           C  \\nATOM   2536  CB  GLN A 340     -14.520   0.510 -13.439  1.00  0.37           C  \\nATOM   2537  O   GLN A 340     -13.294   3.521 -13.982  1.00  0.37           O  \\nATOM   2538  CG  GLN A 340     -14.044  -0.731 -14.182  1.00  0.36           C  \\nATOM   2539  CD  GLN A 340     -15.190  -1.566 -14.722  1.00  0.41           C  \\nATOM   2540  NE2 GLN A 340     -14.859  -2.677 -15.371  1.00  0.37           N  \\nATOM   2541  OE1 GLN A 340     -16.363  -1.217 -14.556  1.00  0.46           O  \\nATOM   2542  N   TRP A 341     -14.509   3.431 -12.110  1.00  0.23           N  \\nATOM   2543  CA  TRP A 341     -14.868   4.845 -12.120  1.00  0.30           C  \\nATOM   2544  C   TRP A 341     -13.640   5.720 -11.897  1.00  0.23           C  \\nATOM   2545  CB  TRP A 341     -15.922   5.140 -11.049  1.00  0.23           C  \\nATOM   2546  O   TRP A 341     -13.402   6.670 -12.647  1.00  0.23           O  \\nATOM   2547  CG  TRP A 341     -16.681   6.413 -11.276  1.00  0.25           C  \\nATOM   2548  CD1 TRP A 341     -17.649   6.639 -12.215  1.00  0.23           C  \\nATOM   2549  CD2 TRP A 341     -16.532   7.637 -10.551  1.00  0.23           C  \\nATOM   2550  CE2 TRP A 341     -17.442   8.565 -11.105  1.00  0.17           C  \\nATOM   2551  CE3 TRP A 341     -15.716   8.041  -9.485  1.00  0.33           C  \\nATOM   2552  NE1 TRP A 341     -18.110   7.931 -12.118  1.00  0.27           N  \\nATOM   2553  CH2 TRP A 341     -16.749  10.241  -9.585  1.00  0.24           C  \\nATOM   2554  CZ2 TRP A 341     -17.559   9.872 -10.628  1.00  0.24           C  \\nATOM   2555  CZ3 TRP A 341     -15.834   9.343  -9.012  1.00  0.27           C  \\nATOM   2556  N   GLN A 342     -12.810   5.413 -10.959  1.00  0.39           N  \\nATOM   2557  CA  GLN A 342     -11.590   6.176 -10.719  1.00  0.40           C  \\nATOM   2558  C   GLN A 342     -10.664   6.127 -11.931  1.00  0.39           C  \\nATOM   2559  CB  GLN A 342     -10.862   5.652  -9.481  1.00  0.38           C  \\nATOM   2560  O   GLN A 342     -10.101   7.149 -12.331  1.00  0.39           O  \\nATOM   2561  CG  GLN A 342     -11.517   6.054  -8.166  1.00  0.37           C  \\nATOM   2562  CD  GLN A 342     -10.793   5.497  -6.955  1.00  0.42           C  \\nATOM   2563  NE2 GLN A 342     -11.285   5.829  -5.766  1.00  0.39           N  \\nATOM   2564  OE1 GLN A 342      -9.801   4.774  -7.087  1.00  0.48           O  \\nATOM   2565  N   ALA A 343     -10.459   4.965 -12.526  1.00  0.40           N  \\nATOM   2566  CA  ALA A 343      -9.656   4.854 -13.742  1.00  0.41           C  \\nATOM   2567  C   ALA A 343     -10.214   5.743 -14.850  1.00  0.40           C  \\nATOM   2568  CB  ALA A 343      -9.595   3.402 -14.209  1.00  0.40           C  \\nATOM   2569  O   ALA A 343      -9.459   6.425 -15.547  1.00  0.41           O  \\nATOM   2570  N   LYS A 344     -11.540   5.775 -15.027  1.00  0.40           N  \\nATOM   2571  CA  LYS A 344     -12.179   6.593 -16.054  1.00  0.41           C  \\nATOM   2572  C   LYS A 344     -12.029   8.079 -15.745  1.00  0.40           C  \\nATOM   2573  CB  LYS A 344     -13.659   6.231 -16.184  1.00  0.39           C  \\nATOM   2574  O   LYS A 344     -11.723   8.876 -16.635  1.00  0.40           O  \\nATOM   2575  CG  LYS A 344     -14.095   5.909 -17.606  1.00  0.39           C  \\nATOM   2576  CD  LYS A 344     -15.570   5.536 -17.667  1.00  0.42           C  \\nATOM   2577  CE  LYS A 344     -16.022   5.269 -19.097  1.00  0.42           C  \\nATOM   2578  NZ  LYS A 344     -17.494   5.029 -19.178  1.00  0.40           N  \\nATOM   2579  N   GLU A 345     -12.283   8.514 -14.499  1.00  0.38           N  \\nATOM   2580  CA  GLU A 345     -12.109   9.916 -14.133  1.00  0.40           C  \\nATOM   2581  C   GLU A 345     -10.671  10.373 -14.365  1.00  0.38           C  \\nATOM   2582  CB  GLU A 345     -12.504  10.144 -12.672  1.00  0.38           C  \\nATOM   2583  O   GLU A 345     -10.438  11.451 -14.916  1.00  0.38           O  \\nATOM   2584  CG  GLU A 345     -13.351  11.388 -12.450  1.00  0.37           C  \\nATOM   2585  CD  GLU A 345     -13.419  11.816 -10.993  1.00  0.43           C  \\nATOM   2586  OE1 GLU A 345     -13.025  11.023 -10.108  1.00  0.51           O  \\nATOM   2587  OE2 GLU A 345     -13.871  12.954 -10.734  1.00  0.47           O  \\nATOM   2588  N   ILE A 346      -9.726   9.589 -14.028  1.00  0.41           N  \\nATOM   2589  CA  ILE A 346      -8.324   9.907 -14.274  1.00  0.42           C  \\nATOM   2590  C   ILE A 346      -8.061   9.952 -15.778  1.00  0.42           C  \\nATOM   2591  CB  ILE A 346      -7.385   8.883 -13.597  1.00  0.41           C  \\nATOM   2592  O   ILE A 346      -7.361  10.842 -16.265  1.00  0.42           O  \\nATOM   2593  CG1 ILE A 346      -7.454   9.024 -12.072  1.00  0.38           C  \\nATOM   2594  CG2 ILE A 346      -5.948   9.054 -14.098  1.00  0.37           C  \\nATOM   2595  CD1 ILE A 346      -6.797   7.878 -11.314  1.00  0.37           C  \\nATOM   2596  N   GLN A 347      -8.599   9.032 -16.635  1.00  0.43           N  \\nATOM   2597  CA  GLN A 347      -8.491   9.068 -18.089  1.00  0.48           C  \\nATOM   2598  C   GLN A 347      -9.083  10.356 -18.654  1.00  0.44           C  \\nATOM   2599  CB  GLN A 347      -9.187   7.855 -18.710  1.00  0.43           C  \\nATOM   2600  O   GLN A 347      -8.511  10.962 -19.562  1.00  0.45           O  \\nATOM   2601  CG  GLN A 347      -8.232   6.739 -19.113  1.00  0.42           C  \\nATOM   2602  CD  GLN A 347      -8.950   5.532 -19.689  1.00  0.47           C  \\nATOM   2603  NE2 GLN A 347      -8.192   4.491 -20.015  1.00  0.43           N  \\nATOM   2604  OE1 GLN A 347     -10.176   5.535 -19.838  1.00  0.57           O  \\nATOM   2605  N   GLU A 348     -10.248  10.767 -18.163  1.00  0.40           N  \\nATOM   2606  CA  GLU A 348     -10.930  11.958 -18.662  1.00  0.43           C  \\nATOM   2607  C   GLU A 348     -10.177  13.228 -18.277  1.00  0.40           C  \\nATOM   2608  CB  GLU A 348     -12.365  12.016 -18.133  1.00  0.39           C  \\nATOM   2609  O   GLU A 348     -10.046  14.147 -19.087  1.00  0.40           O  \\nATOM   2610  CG  GLU A 348     -13.340  11.134 -18.899  1.00  0.38           C  \\nATOM   2611  CD  GLU A 348     -14.783  11.601 -18.793  1.00  0.42           C  \\nATOM   2612  OE1 GLU A 348     -15.076  12.482 -17.953  1.00  0.55           O  \\nATOM   2613  OE2 GLU A 348     -15.628  11.080 -19.554  1.00  0.51           O  \\nATOM   2614  N   GLN A 349      -9.493  13.176 -17.154  1.00  0.41           N  \\nATOM   2615  CA  GLN A 349      -8.688  14.321 -16.741  1.00  0.44           C  \\nATOM   2616  C   GLN A 349      -7.382  14.388 -17.528  1.00  0.41           C  \\nATOM   2617  CB  GLN A 349      -8.395  14.259 -15.241  1.00  0.40           C  \\nATOM   2618  O   GLN A 349      -6.870  15.476 -17.798  1.00  0.41           O  \\nATOM   2619  CG  GLN A 349      -9.564  14.695 -14.368  1.00  0.38           C  \\nATOM   2620  CD  GLN A 349      -9.221  14.703 -12.890  1.00  0.44           C  \\nATOM   2621  NE2 GLN A 349     -10.225  14.927 -12.049  1.00  0.41           N  \\nATOM   2622  OE1 GLN A 349      -8.063  14.510 -12.507  1.00  0.55           O  \\nATOM   2623  N   ALA A 350      -6.813  13.260 -17.963  1.00  0.42           N  \\nATOM   2624  CA  ALA A 350      -5.626  13.244 -18.814  1.00  0.48           C  \\nATOM   2625  C   ALA A 350      -5.947  13.762 -20.213  1.00  0.44           C  \\nATOM   2626  CB  ALA A 350      -5.046  11.834 -18.892  1.00  0.42           C  \\nATOM   2627  O   ALA A 350      -5.047  14.157 -20.957  1.00  0.45           O  \\nATOM   2628  N   LYS A 351      -7.161  13.685 -20.522  1.00  0.43           N  \\nATOM   2629  CA  LYS A 351      -7.582  14.131 -21.847  1.00  0.49           C  \\nATOM   2630  C   LYS A 351      -7.837  15.635 -21.866  1.00  0.42           C  \\nATOM   2631  CB  LYS A 351      -8.838  13.380 -22.292  1.00  0.43           C  \\nATOM   2632  O   LYS A 351      -7.902  16.246 -22.934  1.00  0.42           O  \\nATOM   2633  CG  LYS A 351      -8.584  11.938 -22.704  1.00  0.41           C  \\nATOM   2634  CD  LYS A 351      -9.883  11.210 -23.022  1.00  0.45           C  \\nATOM   2635  CE  LYS A 351      -9.637   9.747 -23.365  1.00  0.51           C  \\nATOM   2636  NZ  LYS A 351     -10.914   8.982 -23.479  1.00  0.47           N  \\nATOM   2637  N   GLY A 352      -8.005  16.356 -20.663  1.00  0.35           N  \\nATOM   2638  CA  GLY A 352      -8.202  17.797 -20.630  1.00  0.40           C  \\nATOM   2639  C   GLY A 352      -6.909  18.572 -20.460  1.00  0.34           C  \\nATOM   2640  O   GLY A 352      -6.694  19.585 -21.129  1.00  0.34           O  \");\n",
              "\tviewer_1718032850302779.setStyle({\"model\": -1},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
              "\tviewer_1718032850302779.addStyle({\"and\": [{\"resn\": [\"GLY\", \"PRO\"], \"invert\": true}, {\"atom\": [\"C\", \"O\", \"N\"], \"invert\": true}]},{\"stick\": {\"colorscheme\": \"WhiteCarbon\", \"radius\": 0.3}});\n",
              "\tviewer_1718032850302779.addStyle({\"and\": [{\"resn\": \"GLY\"}, {\"atom\": \"CA\"}]},{\"sphere\": {\"colorscheme\": \"WhiteCarbon\", \"radius\": 0.3}});\n",
              "\tviewer_1718032850302779.addStyle({\"and\": [{\"resn\": \"PRO\"}, {\"atom\": [\"C\", \"O\"], \"invert\": true}]},{\"stick\": {\"colorscheme\": \"WhiteCarbon\", \"radius\": 0.3}});\n",
              "\tviewer_1718032850302779.addStyle({\"atom\": [\"C\", \"O\", \"N\", \"CA\"]},{\"stick\": {\"colorscheme\": \"WhiteCarbon\", \"radius\": 0.3}});\n",
              "\tviewer_1718032850302779.zoomTo();\n",
              "viewer_1718032850302779.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        }
      ]
    }
  ],
  "metadata": {
    "accelerator": "GPU",
    "colab": {
      "gpuType": "A100",
      "machine_shape": "hm",
      "provenance": [],
      "include_colab_link": true
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}